# Cynthia

| Date | Help request | Notes |
| ---- | ------------ | ----- |
| 2025-03-14 | Feedback previous to presentation | I provided feedback on both technical and aesthetic aspects of the presentation. On the technical side, I pointed out areas where additional context might help fill potential gaps. Aesthetically, I suggested  enhance clarity, such as adding axis labels to plots, maintaining consistent terminology across slides, and including descriptive titles or annotations to make visual elements more self-explanatory. |
| 2025-03-06 | PCA plots interpretation | I provided a brief explaination of PCA used to visualize variance patterns such as clustering by cell type, condition, or batch. I highlighed diferent situation, for example, points close together represent similar profiles, while separation along PC1 or PC2 often indicates major sources of variation; also explained that  coloring by metadata (like sample ID or treatment) helps identify biological or technical drivers, that could be rare populations or quality issues. |
| 2025-02-27 | Feedback for her recent presentation in spatial meeting | I explained the Harmony plots before and after batch correction, highlighting differences between the Harmony pure plot and Harmony UMAP plot. We also discuss quick tests to assess whether low statistical power was affecting DEG identification. I suggested applying a power function (as in WGCNA) and duplicating samples to evaluate dataset limitations, while also discussing the constraints of these approaches. |
| 2025-02-13 | Check previous steps to DEG process | We reviewed the steps leading up to DEG identification, as she wanted to confirm the correctness of her process. I shared a couple of tests I typically run to verify range and summarize various metrics, enabling her to check if her target thresholds are present in the matrix data. |
| 2024-11-07 | Adding manual annotation | I explained to him from scratch how to make a manual annotation using his own SpatialLIBD application. I explained to him how to select slides, select spots with lasso or one to one, and how to import a csv file for further analysis. I also explained to him the conceptual use of the annotation, how he can change the names of the labels and share the file with other people to work collaboratively. |
| 2024-10-25 | spaceranger metrics | I explained to him where he can find the results summary report generated for Spaceranger. We focused mainly on the CSV file as it is more detailed. I explained to him the specific metrics he was interested in and where he can find on the 10x website the technical notes for 10x Genomics products for further questions. |
| 2024-09-24 | AtlaxXomics interpretation results | I explained in more detail the FRiP score and TSS quality control metrics for AtlasXomics sequenced data for the pilot samples provided by Lieber for Hb and DLPFC. We compared the recommended thresholds with those obtained in the pilot samples. I emphasized that the FRiP should be greater than 0.3, but values ​​greater than 0.2 are also acceptable. |
| 2024-09-20 | Visium Training Steps III and Visium Training, JHPCE commands and configuration | Using as an example our https://libd.shinyapps.io/Habenula_Visium_raw/  app, I conitnued explaining her how to explore data from discrete variables, for example the 10x graph-clusters and PCs. I also explained her how explore a particular cluster of interest and a glimse of how to check the gene expresion using multiple features (multi-gene function). |
| 2024-09-17 | AtlaxXomics interpretation results | I helped her understand the results of the AtlasXomics pilot review for the samples provided by Lieber for Hb and DLPFC. I explained the basic concepts and quality controls expected according to the ENCODE standard, however not all of them apply to this type of data. I also pointed out that the sequenced samples were bulk ATAC and could be another metric to consider outside my field of expertise. |
| 2024-09-16 | Visium Training Steps II | I explained her the main layout and variables included on the spatialLIBD shiny app, Using as an example our https://libd.shinyapps.io/Habenula_Visium_raw/   app I showed her many examples using continuos and discrete variables. As well I provided her a glimse of the 10x graph-clustering  wrapped in the app. |
| 2024-09-13 | Visium Training Steps | I explained her main steps to build a spatialLIBD shiny app with raw data. I quickly provided her an overview of spaceranger, and then I explained her how to build a basic spe object and the settings of the ShinnyApps on the cloud. We use as example  our https://libd.shinyapps.io/Habenula_Visium_raw/   app. |
| 2024-09-06 | Visium overview | I explained her relevant points when working with visium samples. We focus on the general concepts and main workflow. I also show her resources to deep into this subjects, such as presentations and the use of the shinny up to reproduce some results. |
| 2024-09-04 | template_project (GitHub, JHPCE) | I showed him the template we currently use. I showed him many examples of how the various directories are expected to be related. |
| 2024-08-30 | Laptop config: RStudio on laptop + local git clone + work on RStudio terminal | I help her to navigate in the resources to setup an appropiate configuration for her laptop. We installed some programs. |
| 2024-08-29 | Team website self-description | I showed her the team repository so she could update her profile. I helped her clone the repository and showed her the sections she needed to update. I presented her with different examples so she could create her own profile. |
| 2024-08-26 | Slack etiquette (questions on channels, avoid DMs, when to tag) and Google Slides weekly update setup | I showed her how to use Slack to have efficient communication, I also showed her examples of what kind of assumptions can be made in the different channels. I help her to organize her threads. I also show her my Google Slides weekly update setup, and ways to improve the current report. |
| 2024-08-22 | LIBD rstats club Google Calendar | I explained the purpose of the RStat Club, showed her where the calendar is, how to schedule her participations in the shared Excel sheet, and how to send an email to invite other subscribers, as well as how to receive these notifications in the Google Slack thread. |
| 2024-08-21 | Team presentations Google Sheet and  How to sign up for a DSgs | I explained the purpose of the team presentations, showed her where the calendar is, how to schedule her participations in the shared Excel sheet, and how to follow the notifications in the Google Slack thread. I also explained her how to schedule a Dsgs calendar for ask help. |
| 2024-04-18 | WGCNA network selection | I explained the difference between signed and unsigned networks, as well as the statistical values that describe an eigengene module. I also explained the network representation in terms of a gene expression data frame versus a differential gene expression data frame. I also explained some functions to validate the genetic significance and module membership of eigengenes under this methodological approach for rna-seq data. |
| 2024-04-11 | Loupe Browser re-analizing and advanced options | In the session, we explored reanalyzing cloupe objects with the Loupe browser for scRNAseq assays. This involved using multiple projections like UMAP and t-SNE, splitting clusters, and filtering data based on UMI count and mitochondrial percentage. While there was interest in analyzing integrated datasets, the Loupe browser currently only supports exploring one Cloupe object at a time. |
| 2024-04-05 | Inspect multimome objects with Browse Loupe | I explained where is located the cloupe output object generated by cell ranger-arc. Then, I explained how to use the tool to explore gene expression and atac assays by separate and also together. I also explained how to inspect clustering and re-label clusters and sub clusters; plus how to check DEG and look at their gene distributions. Finally, I explained how to re-run an analysis and use the most classic formats to save the analysis performed. |
| 2024-04-05 | Support with Config Files | I explained how to prepare the configuration files for working R and JHPCE.  I showed all the section that need to completed plus the configuration to access JHPCE, with links pointing specifically to Windows system. I explained the purpose of each section in the chapter 17 of https://lcolladotor.github.io/bioc_team_ds/config-files.html. I also provided the updated link (https://jhpce.jhu.edu/access/ssh/)  for the JHPCE configuration due the past link is broken because of the change to the new web version. Finally, I provided the most recent video made by Leo  https://youtu.be/qtn3eTwFlZI  (minute 17:50) to do this specific task for windows OS. |
| 2024-03-27 | Question about the correct number of reads in sequencing for scRNAseq and scATACseq multiome strategy | I explained what the key metrics are for each assay in a Multiome experiment from 10x scRNAseq and scATACseq data sets. I showed several HTML reports of data sets processed with cell ranger-arc to illustrate different scenarios targeting different numbers of reads for each assay, to help you understand the relevance of quality beyond the number of reads in sequencing. I showed how with a dataset with a moderate number of good quality reads you can perform a good analysis, while having many reads with compromised quality it is possible to have more challenges during the analysis. Finally, I explained the minimum recommended number and that it would be a good idea to search the literature for technical data from other experiments similar to his as a reference. |
| 2024-03-25 | Cell Ranger-ARC html report | I explained in detail the metrics reported by cellranger arc and helped you download a multiome public sample to run the pipeline. |
| 2024-03-22 | GEX and ATAC clustering | I explained the previous steps to perform clustering with GEX and ATAC 10x multiome data using Seurat and Signac. |
| 2024-02-07 | Discuss Seurat, cell type markers, DE | I showed him to the 2 approaches available in Seurat to perform DGE, including the FindMarkers() function to find the genes that are different between control and treatments. I also show him code to perform quality controls in the main covariates related with these data-type. I shared with him consulted resources and links with public-code for his reference. |
| 2023-04-10 | Feedback the SmokingMouse Project websites | Project links reviewed: http://research.libd.org/smokingMouse/ and http://research.libd.org/smokingMouse/articles/smokingMouse.html |
| 2023-04-06 | Alternatives for RNA-seq analysis without replicates | Website tools to look for public rna-seq samples & packages to make synthetic replicates [as shown here](https://support.bioconductor.org/p/9150681/#9150698). |
| 2023-03-22 | Introduction to CellRanger RNAseq ATAC analysis | We reviewed the overflow to analyse RNA-Seq and ATAC data with the Seurat and Signac tools. I provided learning resources, such as data file organization, funtions and methos available in the tools. I also provided a complete explanation about QC for preprocessing data integrated. |

