# Leo

| Date | Help request | Notes |
| ---- | ------------ | ----- |
| 2025-02-28 | Project goals discussion | We refreshed our memory on where the project stands and what would be some of the next steps that need to be performed. |
| 2025-02-27 | How to build a RSE combining megadepth counts and recount3 colData | I showed how you can obtain the sample information (colData) from an RSE gene-level `recount3` object, then by sample ID, match with the colData you have on a small RSE made manually, and replace the colData info. The small RSE has counts computed with `megadepth`. This could end up being a `recount3` or `megadepth` helper function. In any case, having the full `recount3` colData enables then the use of functions such as `transform_counts()` to convert the `recount3` / `megadepth` base-pair level raw counts into read counts for downstream analyses. |
| 2025-02-27 | How to speed up SPEAQeasy runs | I mentioned that one option involves contacting your HPC admins (JHPCE BitSupport for us) and asking for an increased allocation of computing cores. That way you can run more jobs in parallel. Though ultimately it's best to meet with Nick Eagles for a DSgs as Nick is the author and maintainer of SPEAQeasy. |
| 2025-02-21 | Identification of specific marker genes for a cluster in a published dataset | We discussed the links at https://www.one-tab.com/page/oKFdo7_aRT-3ffoVddJ0sQ. I suggested using `DeconvoBuddies::get_mean_ratio()` on the public data available from GEO. Then checking if that cluster should be merged with another one, and if so, then repeating the Mean Ratio analysis. |
| 2025-02-20 | Replication rate using qvalue | We reviewed code for using `qvalue::qvalue()` to assesss replication rate of a set of DEGs using p-values from a replication dataset. This used code written by Gabriel Hoffman. |
| 2025-02-18 | Biostats prospective student | Discussed our work, life in Baltimore, and other aspects related to choosing a lab to work for during a PhD. |
| 2025-02-13 | smoking-nicotine-mouse paper reviews | We discussed a strategy for responding to reviewers. |
| 2025-02-12 | Accessing Gandal et al 2018 files, other BrainSEQ re-analyses | I showed how to access the no qSV results from the `outGene0` object listed at https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/casectrl_DLPFC.R#L43 that is part of https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/casectrl_DLPFC.R#L59. I also suggested that they could use the BSP1 data aligned with HISAT2 as part of the BSP2 paper by re-runnign the code at https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase1_qsv/casectrl_DLPFC_allFeatures.R#L43 and basically not adding the qSVs if they don't want them. I also showed them the BSP3 code from the habenulaPilot paper at https://github.com/LieberInstitute/Habenula_Pilot/blob/99b5d8b0d5486b475d8b2e21920e9ddb1b71ffc4/code/15_compare_vs_BrainSeq/01_Hb_vs_BSP2_BSP3_DG.R#L32. |
| 2025-02-06 | EB2-NIW | NA |
| 2025-02-06 | JHPCE new PI | We got him setup at JHU Genomics Collective's Slack workspace, talked about DSgs system, LIBD rstats club, and then also JHPCE new PI form. We contacted Francesca and John for more details needed for the new PI form. |
| 2025-02-05 | EB2-NIW | NA |
| 2025-01-23 | recount2/recount3 base-pair coverage questions | Check that the input coordinates are in hg38 and not in hg19 positions. Use `disjoin()` on the exons of a given gene to find the disjoint exon pieces, quantify those, then add them up to get the gene expression. Do this instead of quantifying start to end (aka introns might be noisy). |
| 2025-01-17 | Debugging FUSION TWAS | See links from today's session [here](https://www.one-tab.com/page/UkWtmL03STGtP2Y1Gojdig). To debug the issue at hand, I advised manually running https://github.com/LieberInstitute/fusion_twas/blob/jhpce/FUSION.compute_weights.R for one input gene set of files. |
| 2024-12-10 | Help debugging bioc devel (3.21) issues | See https://community-bioc.slack.com/archives/CLUJWDQF4/p1733847223678279?thread_ts=1733845668.397749&cid=CLUJWDQF4 for more information. Basically, the `bib` object was not well made and this caused a "subscript out of bounds" error on R devel (4.5) and bioc devel (3.21) that wasn't an error before. |
| 2024-12-10 | Help updating a GitHub pages website | I helped update https://github.com/LieberInstitute/spatial_hpc/tree/gh-pages. Note that the html file has a base64 representation of the image, so commit 1 is to update the image, commit 2 is to re-render the html file (once the new image image is available). Also, the image wasn't been displayed at https://github.com/LieberInstitute/spatial_hpc since it was too big. So I resized it so the max dimension would be 1,200 pixels (in this case, the width). |
| 2024-12-10 | Debugging curl installation | Ultimately this person needs to update their macOS computer. They had avoided doing so for too long. See also https://github.com/jeroen/curl/issues/358 for more details from someone with a nearly identical issue. |
| 2024-10-11 | Letter of support for a grant application | We discussed their grant and how my letter of support should be framed. I sent the letter at the end of the day. |
| 2024-10-11 | Debugging a GitHub Actions workflow error on a BioC 3.20 ExperimentHub package under development | 1. Check with `lobstr::obj_size()` how big your objects are. They might not be compatible with [memory limits provided by free GHA workers](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners) (GitHub Actions). 2. `biocthis` 's GHA workflow tries to emulate the Biocondutor SPB (Single Package Builder). People can submit packages to BioC without having tested them on GHA, though then they might run into R CMD build / check / BiocCheck issues for the first time there and have a slower peer review process. 3. Consider using `rm()` once you are done with the first large object. 4. In general, it's best to break long code chunks into smaller ones (and name them). That way, you can get more specific locations of the errors. Hence why we added that `stopifnot()` from my PR to formally check that the snapshot date is valid. 5. For debugging, it's useful to have a local environment where you can reproduce in detail the issues. I didn't have one ready before the DSgs for BioC 3.20, but if we meet again, I could be ready. |
| 2024-10-11 | Reviewing BioC package contribution process | We went over the process of synching a GitHub package with the Bioconductor git server, what is the Bioconductor Single Package Builder (SBP), when we can tell that the SBP has been configured (and is thus ready to access new versions via bumps in the DESCRIPTION), and how biocthis's GHA simulates part of what the SBP does but how they still have some differences. Ultimately in this case, GHA doesn't have a timeout limit (15 min) that SBP does have, which is why SBP is erroring out while GHA is ok. We also ran into an issue with the development version of pkgdown. |
| 2024-10-02 | visiumStitched revision | We went over the response to the reviewers (from the journal) and clarified how to address some of the comments. |
| 2024-09-27 | Team support overview and introduction | I provided an overview of what the team does, how we can help the OP, added them to Slack, the LIBD rstats club, and other intro first steps. |
| 2024-09-27 | Paper draft structure | We reviewed https://lcolladotor.github.io/bioc_team_ds/writing-papers.html and started a Google Doc, SciWheel, LIBD OneDrive for this paper. |
| 2024-09-25 | Debugging an .Rbuildignore file | I helped debug an issue with a new R package. Basically, some files were not being included in the .Rbuildignore that should have been. |
| 2024-09-19 | Project handoff checks and project updates | We discussed the general setup we will move ahead with now that we have to wait a few months before new data is generated to complete this study. We brainstormed options moving forward on how to best collaborate. |
| 2024-09-16 | Cont. PhD application plans | Ibidem |
| 2024-09-13 | smoking-nicotine-mouse paper checks | We reviewed responses from the OP to recent comments by a co-author of this paper. |
| 2024-09-11 | PhD application plans | We reviewed some information about applying for PhD programs: how to decide, application strategy in general, information that is best to provide for those writing letters, and other related info. |
| 2024-09-09 | smoking-nicotine-mouse paper checks | We reviewed recent changes made by OP to the paper draft and ultimately accepted / resolved current open comments. |
| 2024-09-06 | Hb_VTA_projectors updates | We discussed recent results with our colleagues and made a list of pending items to clarify. |
| 2024-09-06 | LIBD internship | We set up the logistics for performing a remote LIBD internship with my team by re-using materials from the CDSB 2021 snRNA-seq OSCA-based workshop materials and videos. |
| 2024-09-05 | spatialLIBD apps deployment review | We reviewed how to deploy spatialLIBD shiny apps on shinyapps.io and why lines such as `library("markdown")` are important to keep in the `app.R` R script file. |
| 2024-08-28 | GO / KEGG ORA analysis on cell-type specific DEG results | We reviewed how the universe of background genes (for GO/KEGG ORA) is different for each cluster when using `pseudoBulkDGE()` due to the `filterByExpr()` internal step. |
| 2024-08-26 | Single-cell genomics webinar Latin America | We went over the preparation for my invited speaker presentation for this webinar series. |
| 2024-08-19 | Hb_VTA_projectors updates | Discussed recent updates. |
| 2024-08-15 | LIBD cloud computing | We discussed some future options for LIBD cloud computing to make sure that we were on the same page. |
| 2024-08-13 | Cont. on team options | Further discussed options for working with the team. |
| 2024-08-09 | Clustering vs sub-clustering results comparison | We interpreted and made annotations on the results comparing the clustering vs sub-clustering analyses. |
| 2024-08-05 | edgeR 4.0 and clusterProfiler questions | We briefly mentioned the edgeR 4.0 paper at https://doi.org/10.1101/2024.01.21.576131 and then mostly spent time reviewing the documentation for `clusterProfiler` at https://yulab-smu.top/biomedical-knowledge-mining-book/clusterprofiler-comparecluster.html. |
| 2024-08-01 | Features from JHPCE storage tracker website | We discussed the preliminary version of the website for visualizing storage use over time at JHPCE. |
| 2024-07-30 | Random seed usage for PCA/UMAP/tSNE | We reviewed when to use `set.seed()` when using `runPCA()`, `runUMAP()`, `runTSNE()` and other related analysis steps. |
| 2024-07-29 | Smoking-nicotine-mouse paper checks | Ibidem. |
| 2024-07-26 | Smoking-nicotine-mouse paper checks | We discussed feedback from collaborators and how to best address it. |
| 2024-07-25 | Cell type sub-clustering | We discussed why a sub-clustering analysis would be of interest for this dataset. |
| 2024-07-24 | InterJad | We talked about data science education and the InterJad platform. |
| 2024-07-22 | Review on how to write method sections | We reviewed https://lcolladotor.github.io/bioc_team_ds/writing-papers.html and other guidelines for writing method sections of papers. |
| 2024-07-18 | Continuation on DEG analysis at the cell cluster level | We reviewed https://rdrr.io/bioc/scran/man/pseudoBulkDGE.html and then also talked about how to perform ORA (Over Representation Analysis) on GO / KEGG. We also reviewed how to make volcano plots with https://www.bioconductor.org/packages/release/bioc/vignettes/ideal/inst/doc/ideal-usersguide.html#75_plot_volcano and https://bioconductor.org/packages/release/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html#plot-the-most-basic-volcano-plot. |
| 2024-07-18 | Viruses and recount2/3 | We discussed options for how to use `derfinder`, `recount2`, `recount3`, `megadepth` and other related packages and tools (like `recount.bwtool`) to quantify expression from BigWig files from the recount2/3 projects. There is a need for a `megadepth` function that is similar to `recount.bwtool::coverage_matrix()` that works via the web given the current limitations with `rtracklayer`. |
| 2024-07-11 | DEG analysis at the cell cluster level | We reviewed material from OSCA on `pseudoBulkDGE()` documented at https://bioconductor.org/books/3.19/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis for performing DEG analysis within each cell type cluster. |
| 2024-07-11 | LIBD opportunities | We discussed opportunities within my team and LIBD in general. |
| 2024-07-09 | 2024 opportunities | We discussed options to extend our collaboration in 2024. |
| 2024-07-08 | SpatialExperiment and other related R objects | We reviewed the properties of SpatialExperiment objects and other type of R objects (like a `GRangesList`) we used in Huuki-Myers et al, Science, 2024. The goal was to promote interoperability with planned work by the OP. We also went over https://research.libd.org/DeconvoBuddies/reference/findMarkers_1vAll.html, https://research.libd.org/spatialLIBD/reference/registration_pseudobulk.html, https://bioconductor.org/books/3.19/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis, and https://research.libd.org/spatialLIBD/reference/registration_wrapper.html. |
| 2024-07-03 | Team opportunities | We discussed opportunities within my team and options for collaborating moving forward. |
| 2024-06-26 | LIBD fundraising | I provided an overview of what my team does and the type of projects we work on. We brainstormed a bit about fundraising. |
| 2024-06-25 | GitHub profile review | We discussed their GitHub profile and other evidence of coding skills. |
| 2024-05-24 | Questions about the "McCarroll lab dlPFC web application" | We went over some documentation and features of https://dlpfc.mccarrolllab.org/app/dlpfc. |
| 2024-05-22 | 10x Genomics reference GTF annotation files | We revisited [documentation by 10x Genomics on how the reference GTFs were made](https://kb.10xgenomics.com/hc/en-us/articles/115003616023-Files-and-criteria-used-to-generate-10x-references) such as the [mouse 2020 ref](https://www.10xgenomics.com/support/software/cell-ranger/downloads/cr-ref-build-steps#mouse-ref-2020-a). |
| 2024-05-21 | Zotero setup | We went over how to configure Zotero https://www.zotero.org/ and Zotfile https://github.com/jlegewie/zotfile for organizing PDF files of papers in your laptop. I keep the PDFs on my Dropbox and use Zotfile for automatically naming the PDFs when I download them via the [Zotero Google Chrome plugin](https://chromewebstore.google.com/detail/zotero-connector/ekhagklcjbdpajgpjgmbionohlpdbjgc?pli=1). |
| 2024-05-15 | qsvaR questions | We discussed some questions about the application of qsvaR to the habenulaPilot bulk RNA-seq dataset. |
| 2024-05-14 | Career development | We checked their LinkedIn, website, and CV. I provided a list of changes to improve all 3. |
| 2024-05-02 | How to check indices used from 10x FASTQ files | We used commands such as `for i in *Cg*/*.fastq.gz; do echo $i; zcat $i | head; done` to show the first few lines of 10x Genomics FASTQ files (Illumina sequencing), then checked the indexes vs those that our colleagues used for based on their wet lab notes. |
| 2024-05-01 | DGE analysis of a data subset questions | We discussed a few things including `jaffelab::expression_cutoff()` logic and briefly mentioned `edgeR::filterByExpr()` as a more conservative alternative. In any case, when working on a sample subset I do recommend starting with the raw data and filtering out lowly-expressed genes (features) as which genes are expressed in a subset can be very different from the full dataset. We talked about ranges of logFC and t-statistics seen in DGE results from different projects, which can change depending on the biological covariate of interest. We also discussed our current strategy for selecting covariates in a DGE analysis when working on a small(er) dataset. |
| 2024-05-01 | Filtering lowly expressed features prior for replication analyses | We discussed that on a discovery dataset, we apply an FDR filter, and thus filtering out lowly-expressed features (genes) can affect the results. It doesn't impact the target dataset at all since in the target we are using nominal p-value (aka not FDR-adjusted p-values) and sign(logFC). |
| 2024-04-30 | Updating broken Globus endpoints | We went over how to update globus https://research.libd.org/globus/ by updating https://github.com/LieberInstitute/globus/blob/master/index.Rmd, then clicking the "knit" button. We also went over how to create Globus endpoints (collections that you share), how to set the right permissions, add other administrators, and get the link for sharing which is what we need for the index.Rmd script. |
| 2024-04-25 | Query about some LIBD bulk RNA-seq and WGBS datasets | We discussed the nuances of some of LIBD's internal datasets, questions that need to be asked, whom to ask them to, and potential reasons for including/not including myself or others as Co-Is on a grant application. |
| 2024-04-24 | Impact of CellRanger versions over time on single-exon genes embedded inside introns of longer genes | We discussed how it could be useful to compare CellRanger v3 with the pre-mRNA modified GTF, v6 without the include-introns flag, v6 with the include-introns flag, Alevin, and STARsolo. Alevin and STARsolo might have similar include-introns flags to CellRanger by 10x Genomics. We discussed how this could potentially affect the class of single exon genes that are embededded within introns of longer genes. Violin plots by cell type across the different computational pipelines and alignment options could be useful to learn more how this affects gene quantification of this special gene class. |
| 2024-04-23 | SCZD DEG models | We discussed in detail the DEG models for the BSP1, BSP2 (combined regions, one region at a time), BSP3, Habenula, and CMC Phase 1 datasets. All should be age > 17. BSP2 overall should adjust for brain region (a large source of biological variation) and also drop the 43 HPC RiboZeroGold samples. BSP2 HPC only should also drop those same 43 RiboZeroGold samples. Habenula should adjust for the tot.Hb and tot.Thal estimated fractions: here's the [published model](https://github.com/LieberInstitute/Habenula_Pilot/blob/ca6dba5f629a8ec580297ab6d56b36b4f0cb66da/code/10_DEA/04_DEA.R#L124-L126). CMC Phase 1 should adjust for BrainBank. Here's the BSP1 hg38 (part of the BSP2 paper) [model](https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase1_qsv/casectrl_DLPFC_allFeatures.R#L40). |
| 2024-04-18 | Reviewing covariates to use in a DE model for snRNA-seq pseudobulked data by cell type with dreamlet | They taught me that `dreamlet` can produce `mito_percent` and `ribo_percent` for each cell, and thus you can summarize them with the mean across all cells of a given cell type for a given donor. This expands the options for covariates in a DEG model. We discussed why it could be interesting to run a model with and without adjusting for `mean(mito_percent)` given that in spatial data (such as the data from `HumanPilot`) `mito_percent` is related to the spatial location of WM and L1 (richer in oligodendrocytes and other glia cells, low presence of neurons). We also went over a part of the [LC eLife paper](https://doi.org/10.7554/eLife.84628.3) and revised some steps from [OSCA on multi-sample DE](https://bioconductor.org/books/3.18/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis). An equivalent of [`totalAssignedGene` produced by `SPEAQeasy`](https://research.libd.org/SPEAQeasy/outputs.html#quality-metrics) could be available at the donor (individual) level, but maybe it wouldn't be that useful as a covariate in these cell-type specific analyses. |
| 2024-04-18 | Mean ratio application to HumanPilot data and spatialDLPFC re-analysis | Re-organized code to run with a shell script from slurm_jobs. We solved a bug with args index, and talked about adding catches for unexpected inputs. Future goal is to use map to replace for loops when handling list of data objects |
| 2024-04-18 | Setting up a website similar to the [spatialDLPFC one](https://research.libd.org/spatialDLPFC/) | After making a new `git clone`, we created the `gh-pages` branch with commands from https://jiafulow.github.io/blog/2020/07/09/create-gh-pages-branch-in-existing-repo/. For more on GitHub pages check [this page](https://pages.github.com/). We created a favicon using https://favicon.io/favicon-converter/. We created a [`README.Rmd` file re-using some code from the `spatialDLPFC` one](https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd), and also adapted the [`update_website.R` script](https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R). We also did a few other manual setup steps like creating the `img` directory and the `.nojekyll` file on the `gh-pages` branch, plus version controlling them. I might write a blog post about this full process in detail. |
| 2024-04-16 | Review of SCZD vs NTC DE models | I explained why `age > 17` is a must for SCZD vs NTC DGE analyses: we have no SCZD cases with `age <= 17`, so if you don't filter the NTC donors with `age <=17`, you end up basically comparing older (SCZD) vs younger (NTC) donors, which is not what you want to do. We also talked about how in BSP2 there are some HPC (HIPPO) samples from 2 RiboZero protocols (Gold and HMR), how the Gold HPC samples cluster with the DLPFC ones on PC1 vs PC2, and thus why we ended up dropping the 43 Gold HPC samples in previous work (see https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/make_qSVs.R#L62-L69). This information is reflected by `analysis_sczd_casecontrol_hippo` at https://s3.us-east-2.amazonaws.com/libd-brainseq2/methprop_pd.Rdata listed at https://eqtl.brainseq.org/phase2/. Finally, I emphasized that self-reported `race` is not a trustworthy variable and we instead rely on `snpPC`s. |
| 2024-04-15 | Options for creating links to use in manuscripts | I explained why using something like https://research.libd.org/spatialDLPFC/#interactive-websites or https://research.libd.org/globus/ give us control over links that we might need to update later on, instead of including other less stable links on manuscripts (domains we don't control, tools that might change their URL syntax, etc). I provided links to how to make each option such as https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R and https://github.com/LieberInstitute/globus/blob/98dc479ad2cde0a047cf7a46769a85c32969b7b1/index.Rmd#L8-L37. |
| 2024-04-12 | Review steps for a new snRNA-seq project | We went over the data that we have for this project, worked on making a soft link to the output of cellranger that was generated in a different repository (and thus JHPCE location) to enable the use of `here::here()` in future scripts, and discussed potential issues we might observe in this new dataset. |
| 2024-04-08 | Reference letter request | We discussed the specifics of the [KavliNDI Distinguished Graduate Fellowship](https://kavlijhu.org/funding/4) and revised information that I can use while writing the reference letter (project title, assays proposed, that this involved new software, how long we've worked together, education history of the applicant, etc). |
| 2024-04-05 | Updating ranger_metrics information | We added the `LFF_spatial_ERC` and `LFF_spatial_LC` datasets to https://github.com/lieberinstitute/ranger_metrics. This involved debugging the code a bit to handle updates on the Excel sheet files describing samples prior to being sent for sequencing. |
| 2024-04-03 | Avoid using jsonlite when editing spaceranger json files | We found out that `spaceranger count` did not like the JSON files we created with `jsonlite::write_json()` so instead, we edited them manually using `gsub()` and `read/writeLines()` as documented [here](https://github.com/LieberInstitute/LFF_spatial_ERC/commit/bbf810ad39012ba6ee4e0403063bf83bf844d007). Note that we had to manuallly revert the JSON files to their original state as documented [here](https://github.com/LieberInstitute/LFF_spatial_ERC/commit/79ed5479d502cd41d51f3aa8ee1467bd8100a08a). |
| 2024-04-02 | Write a script to manually edit spaceranger json files | We wrote a script to manually update the json files and add the Visium Slide ID as well as the capture area. This will enable us to not have to use `--unknown-slide=visium-1` https://github.com/LieberInstitute/LFF_spatial_ERC/blob/6e85c05ce9f531c555569e616f8d5c351b427b4f/code/01_spaceranger/spaceranger_SLURM.sh#L63 that 10x Genomics support had recommended using. Doing so led to an error at https://github.com/LieberInstitute/LFF_spatial_ERC/blob/b5b92a8bb9744ee424ec4a0642ae692423a91d56/code/01_spaceranger/logs/spaceranger_17.txt#L22 which stated that `--slide` cannot be used as the same time as `--unknown-slide`. This is a workaround that was previously manually implemented at https://github.com/LieberInstitute/Habenula_Visium. Since we have lots of samples, we wrote a script to avoid introducing copy-paste errors. |
| 2024-04-01 | Debugging spaceranger count | The issue on this [script](https://github.com/LieberInstitute/LFF_spatial_ERC/blob/e913ff28fffbb17335d3844d312ec6486a7b182f/code/01_spaceranger/spaceranger_SLURM.sh#L55) and related ones is that there is a missing `\ ` (note the space) before the `#` comment symbol. Thus the chained command is breaking into two, and the second half is not being recognized as options to the first half. |
| 2024-03-29 | Understanding a spaceranger error | We worked on the following error message from `spaceranger count`: _ERROR: We detected a mixture of different R1 lengths ([26-28]), which breaks assumptions in how UMIs are tabulated and corrected. To process these data, you will need to truncate to the shortest observed R1 length by providing 26 to the --r1-length argument if are running count/vdj, or via the r1-length parameter in each of the following tables of your multi config CSV if you are running multi: [gene-expression]_. We inspected the soft links for the FASTQ files and used `zcat fastq.gz | head` to inspect the first few lines of FASTQ files. We confirmed that the `R1` FASTQs were indeed of varying lenghts and will attempt a new `spaceranger count` run using `--r1-length = 26` though it's likely that we'll next get an error with the `R2` FASTQ files as we observed that those files also have varying lengths. |
| 2024-03-25 | LIBD Data Portal for BSP1 and BSP2 DLPFC SCZD vs NTC data | I explained that BSP1 is polyA, aligned vs hg19, older annotation, aligned with TopHat2 (an older aligner). BSP2 is RiboZeroGold for DLPFC and RiboZeroHMR for HIPPO (with 43 RiboZeroGold samples that were dropped from the DE analysis), aligned vs hg38, against Gencode v25 (from July 2016), aligned with HISAT2. I also explained the naming structure of a Supplementary Table for BSP1's paper where `adj`, `qSVA`, and `PCA` are labels specifying whether it was a model `adj`usting for covariates, whether it adjusts for qSVs using `qSVA` (the better model to use), or adjusting for gene-level expression PCs (`PCA`). These names are related to the [`qSVA` 2017 paper](https://www.pnas.org/doi/full/10.1073/pnas.1617384114) Figure 2. I also explained how in the BSP2 paper we did re-align the BSP1 data against hg38, though there is no Supplementary Table showing which of the 243 DLPFC DEGs replicated in BSP1 (though the data does exist since it was used to make Figure S28 and S29). We would need another DSgs or two to actually locate the data and construct an CSV file or table that others can use if they are interested in replication of BSP2 results using BSP1 and/or CommonMind Consortium (CMC) Phase 1 data (that was aligned vs hg19 with TopHat2). I also used [this slide](https://speakerdeck.com/lcolladotor/libd-ds-tldr?slide=56) to explain all the pieces that go into a DEG model and why the visualizations provided by the LIBD Data Portal (boxplots for SCZD vs NTC) are just a visual guide using normalized gene expression data and not the full answer (as the LIBD Data Portal doesn't use the full details we include in a DEG model). |
| 2024-03-22 | Polishing details on how to ask for help effectively | We disscussed specific details on how to ask for help. For example, it's ok to ask those around you in person but you should also ask on the `#libd_helpdesk` channel (and update the channel if someone else gave you an answer). That is, don't assume that only those around you (in person) know the answer. We went over details on how to preview GitHub messages (to verify that they are correctly formatted), why GitHub issues are useful to the authors, what information is needed for others to help you answer a question (R session info, `reprex::reprex()` output, link to code on a public GitHub repo; private is ok when asking internally for help). We also discussed the overall idea of why learning how to make an R package will be useful to then improve one's tooltik for debugging R packages made by others. |
| 2024-03-20 | Cg_Hb_projector_profiling progress update | We went over how to update a GitHub remote following [information from the documentation](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories), finishing stylistic design touches on an `iSEE` app, and setting up a new repository for the next project. |
| 2024-03-20 | Overview of use cases for JHPCE disk usage data over time | We discussed details of the use cases I have in mind for how to search and visualize the JHPCE disk usage data over time and across different JHPCE paths. This information could be useful in forecasting data growth for individual PIs or projects, identifying who has exceeded their storage allocation, identifying static projects that could potentially can be moved off JHPCE / deleted / or moved to non-production disk space. |
| 2024-03-19 | snRNA-seq data analysis steps overview | We discussed the general overview of snRNA-seq data analysis steps and how the code at [`Habenula_Pilot`](https://github.com/LieberInstitute/Habenula_Pilot) could be a good starting place. I also recommended signing up for a Data Science guidance session with Louise as she is quite familiar with the code in that repository prior to diving into it. |
| 2024-03-19 | Cloud computing vs JHPCE | I explained details about how data is organized at JHPCE, data sharing limitations, and several suggestions we have been working on that could be useful to implement at a larger scale across LIBD teams. |
| 2024-03-15 | Advice prior to a faculty interview | Discussed collaborations, personalities, and other details of the work environment with some of our collaborators with whom this person was going to interview for a job with. |
| 2024-03-14 | Derive code for how to merge ERCCsumLogErr values across sequencing lanes for a given sample | See https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1710454284251579 for the full code details. We might add this to solution to https://github.com/LieberInstitute/jaffelab/blob/devel/R/merge_rse_metrics.R, although it's only an approximation as you need to go back to the raw ERCC output files (typically generated with `kallisto`) to actually compute the ERCCsumLogErr accurately across different sequencing lanes for a given sample. |
| 2024-03-14 | Help identifying differences between two RSE objects containing data from the same project | We have RSE objects from the LIBD Data Portal (or those that helped build that portal) as well as RSE objects made prior to the LIBD Data Portal. They have some differences, mostly on the `colData()` slot. Note how https://github.com/LieberInstitute/jaffelab/blob/21e65745a5b0852fc13ebdb5e5d23c6f1f66010b/R/merge_rse_metrics.R#L83-L97 and related code merges `colData()` information for samples that were sequenced in more than one lane and whose FASTQ files were not merged prior to aligning with `SPEAQeasy`. |
| 2024-03-12 | spatialDLPFC and deconvolution-benchmark project details on how snRNA-seq data was used for each of them | I explained details about the spatialDLPFC and deconvolution-benchmark papers. I also went over how we addressed feedback from the editors for the spatialDLPFC paper. |
| 2024-03-07 | Grant related discussion | We discussed how to address feedback from a recent grant application, how our teams can get to know each other better, cleared up misunderstandings related to my team's role as well as the DSgs system, and decided next steps to respond to the grant reviewers. |
| 2024-03-05 | Cg_Hb_projector_profiling progress update | We discussed more details about `iSEE` and shinyapps.io deployment details, such as memory settings and other configuration needed to actually customize and share an `iSEE` app. We also asked our collaborator for the list of genes they want to have plotted outside of `iSEE`. |
| 2024-03-01 | Feedback on the nascentRNA project | I sent a long and detailed email explaining different components of the LIBD BrainSEQ projects, background information on several covariates we typically have used in DEG models such as RIN and age at time of death, as well as other ones we have not used such as postmortem interval (PMI). I also explained the DSgs system and how it could be used by our JHU collaborators. |
| 2024-02-28 | How to extract normalized expression values with limma | We went over internal code from the limma package using https://code.bioconductor.org/, particularly for the `voom()` function. Doing so, we discussed that's actually returned in the `voom()$E` slot. We also went over how to model a ratio using a linear regression model. |
| 2024-02-26 | Prospective JHU PhD student | We went topics such as live in Baltimore, research at JHU, overview of the different JHU campi, how to compare PhD programs across universities, and other questions this future PhD student had. |
| 2024-02-23 | Cg_Hb_projector_profiling progress update | Discussed feedback from collaborators and detailed next steps to wrap up this project. |
| 2024-02-16 | Cg_Hb_projector_profiling progress update | Went over recent updated plots and results, as well as slides for a presentation with our collaborators for 2024-02-21. |
| 2024-02-16 | qsvaR bug report resolution | We discussed the updates implemented in qsvaR to handle input with ENSEMBL transcript IDs instead of GENCODE ones, updates to automatically detect if the input is at the gene level instead of the transcript level, how to check for correlation between qSVs and covariates, among other related issues. |
| 2024-02-08 | Cg_Hb_projector_profiling progress update | Discussed updates to violing plots and how to organize result files prior to sharing them with our collaborators. |
| 2024-01-29 | Career development | Caught up and talked about career development next steps. |
| 2024-01-25 | Permutation analysis | Went over the details of how to compute a permutation analysis with R. |
| 2024-01-24 | Review response to qsvaR bug report from last year | Clarified updated automatic checks and information for users who attempt to use qsvaR with gene-level data instead of transcript-level data. |
| 2024-01-11 | Reviewing updates to the qsvaR package | We checked udpates to the `getDegTx()` and `DEqual()` functions, as well as discussed the response to a bug report from last year. |
| 2023-12-01 | JHPCE R modules | I explained the different modules such as `module load conda_R/4.3.x` versus `conda_R/4.3`, how each is tied to a different version of Bioconductor. I also linked to https://youtu.be/D6t2GR7xaTE and https://docs.google.com/document/d/1DD9MUI2e3jp5S8C3PWRulZ4jwbXo94D3UTCQNJl_-Qo/edit?usp=sharing from the LIBD rstats club. |
| 2023-11-29 | CV writing advice | Provided feedback on CV writing for data science job applications. |
| 2023-11-16 | Cg_Hb_projector_profiling progress update | We discussed the [GLM-PCA paper](https://doi.org/10.1186/s13059-019-1861-6) and decided to use GLM-PCA instead of logcounts by `quickCluster()` + `calculateSumFactors()` as described at [the CDSB 2021 snRNA-seq workshop](https://comunidadbioinfo.github.io/cdsb2021_scRNAseq/normalizaci%C3%B3n-de-datos.html#normalizaci%C3%B3n-por-decircunvoluci%C3%B3n-deconvolution). We also discussed the development version of `Harmony`, in particular with the `lambda = NULL` setting. |
| 2023-11-16 | Review how to respond to a user's bug report | We went over how to build a reprex for a user's bug report, how to simplify the process of debugging, and the actual results from this particular bug report. We also discussed how to relay this information back to the person who initially asked the question. |
| 2023-11-15 | Job application support | I provided background information on a job ad from a colleague at Boston's Children Hospital that I had shared. I also provided suggestions for preparing their GitHub account, CV, and other information useful for job interviewers. |
| 2023-11-08 | Job application support | I provided background information on a job ad from a colleague at FHCRC that I had shared. |
| 2023-11-08 | Cg_Hb_projector_profiling progress update | I gave feedback on scripts, how to organize them, the order of some analytical steps, which gene annotation files to use, and things like that. We don’t have results on the QC steps to share just yet. |
| 2023-11-03 | Review replication analysis R script | Continued reviewing qsvaR replication analysis code. |
| 2023-11-01 | Cg_Hb_projector_profiling project overview | Detailed analysis steps for [this project](https://github.com/LieberInstitute/Cg_Hb_projector_profiling) using snRNA-seq data and analyzing it following OSCA-based steps from the CDSB 2021 snRNA-seq workshop. |
| 2023-11-01 | Review Visium SPG AD blog post | Reviewed draft blog post on the Visium SPG AD project and published it at https://research.libd.org/rstatsclub/2023/11/01/our-first-adventure-with-visium-spatial-proteogenomics/. |
| 2023-10-27 | Interaction modeling with limma-voom | Explained how interaction models work with limma-voom. |
| 2023-10-27 | snRNA-seq OSCA-based analysis | Ibidem to 2023-09-29 session. |
| 2023-10-26 | Review replication analysis R script | Reviewed [this script](https://github.com/LieberInstitute/QSVA_2020/blob/914e3b58f7fb97c204654493c9f9dcb2e48d112b/code/06_Replication/02_DE_and_replication_plot.R#L220-L313) for the replication analysis performed as part of the qsvaR project. |
| 2023-10-26 | Debugging a ranger_metrics R script | Paired programming session to debug `/dcs04/lieber/lcolladotor/with10x_LIBD001/ranger_metrics/code/spaceranger/03_merged_metrics/02_explore_metrics.R`. Further explained the original intent of the code in this script and steps for updating it. |
| 2023-10-20 | Review replication analysis scripts for the qsvaR project | Reviewed some scripts related to replication analyses. I explained the different thresholds used on the discovery as well as the target datasets, as well as the different levels for assessing replication. |
| 2023-10-18 | snRNA-seq OSCA-based analysis | Ibidem to 2023-09-29 session. |
| 2023-10-12 | qsvaR paper planning | We went over the changes needed to take the current qsvaR paper draft by Josh Stolz to a production-level result (aka for pre-printing / publishing it). |
| 2023-10-06 | snRNA-seq OSCA-based analysis | Ibidem to 2023-09-29 session. |
| 2023-09-29 | snRNA-seq OSCA-based analysis | We continued working with the OSCA-based CDSB 2021 intro to snRNA-seq workshop to further understand snRNA-seq data analysis steps, packages, and how they all relate to each other. The person who requested this DSgs was working on the workshop contents and had questions that we reviewed weekly (or so). Once they completed this OSCA-based training, we moved to analyzing an actual unpublished snRNA-seq dataset. |
| 2023-09-27 | Onboarding questions for a snRNA-seq dataset | We went over the background for this project (lateral septum) and an overview of the DSgs system, GitHub setup, and other onboarding material. |
| 2023-09-22 | snRNA-seq training | Resolved some questions related to https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq and made two pull requests to fix typos. |
| 2023-09-22 | How to interpret terms on an interaction model | We went over the math derivations for interpreting beta1, beta2, and beta3 on an interaction model where X1 is a continuous variable, X2 is a discrete variable with 2 options and we model `Y ~ X1 * X2`. We also used `ExploreModelMatrix`. |
| 2023-09-21 | Guidance on expression heatmaps | I explained the benefits of centering and scaling and referred to both the short video on this topic https://youtu.be/c8ffeFVUzk8 as well as the material we developed for the CSHL 2023 course https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html. |
| 2023-09-14 | snRNA-seq training | Continued reviewing material from https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq |
| 2023-09-13 | Social media ideas | Discussed a few ideas about LIBD and social media |
| 2023-08-21 | snRNA-seq training | Answered questions related to material at https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq |
| 2023-08-18 | ranger_metrics | We talked about the first scripts in https://github.com/LieberInstitute/ranger_metrics |
| 2023-08-14 | Getting started with snRNA-seq | We talked about material at https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq |
| 2023-08-10 | Introduction to what we do as a team and future plans | We talked about how I became an Investigator at LIBD, the overall type of work we do as a team, some of our main collaborations, and overall plans for applying for funding in the future. In particular, I highlighted how I want to train team members on how to write an NIH biosketch, though not all are going to be "key personnel". I sent a link to the DSgs, team meeting schedule, rstats club schedule, and our http://lcolladotor.github.io/talk/teamtldr2022/ summary from April 20, 2022. |
| 2023-07-27 | Guidance on reference data for deconvolution and snRNA-seq analysis | For deconvolution, we talked about finding an appropriate reference dataset and we talked about an expert we can contact to ask which dataset(s) might be the most relevant. For snRNA-seq data analysis, I mentioned OSCA (Orchestrating Single Cell Analysis with Bioconductor) and how Louise could help guide this person on our code based off OSCA through a series of DSgs sessions. |
| 2023-07-27 | Feedback on an R package + GO/KEGG enrichment analyses | We transferred https://github.com/LieberInstitute/qsvaRPkgDemo to the LieberInstitute account. I asked to include some citations for sources (like the image source) and some other small edits. For GO / KEGG enrichment with clusterProfiler I asked to not filter results (pvalueCutoff / qvalueCutoff parameters), so we can see more results prior to choosing which ones to plot. |
| 2023-07-26 | Evaluate a potential external contractor | We discussed the current status of the team, the project, what skills are required, boundaries in terms of time and who would be involved (and who wouldn't). We ended the session with me relaying a list of resources to check and read more carefully. Then in about 2 weeks we'll meet again to determine the next steps. |
| 2023-07-26 | How to best document what skills I can perform or need to learn | We made a Google sheet with columns such as `study_type`, `skill_name`, `learning_priority`, etc. I also worked on it after the DSgs session to help complete the current information. See [Slack](https://neuralplasticity.slack.com/archives/C04Q8MX99FY/p1690397027111449) for more details. |
| 2023-07-26 | Need help debugging iSEE app errors/warnings | We resolved this issue by making sure that the variable names for which they wanted to specify the correct colors were actually part of the colData(SE). See [Slack](https://jhu-genomics.slack.com/archives/C01KSLPCGAC/p1689858182290369) for the actual code links. I also asked them to make sure to use soft links / full paths instead of using setwd() for files outside the project + to keep all related files at JHPCE instead of on a laptop, to make it easier to rule out any sources of differences (aka for reproducibility purposes). |
| 2023-07-20 | Input on a course design | We discussed what would be the appropriate material for us to teach in a course. We will re-use some of our materials from https://github.com/lcolladotor/cshl_rstats_genome_scale_2023. |
| 2023-07-20 | Feedback on responding to reviewers | We went over the proposed response to reviewers, clarified some sentences, and requested co-authors to approve the response to the reviewers. |
| 2023-06-29 | How to make a project documentation website | We adapted code from https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R and https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd in order to make the files at https://github.com/LieberInstitute/septum_lateral. See https://github.com/LieberInstitute/septum_lateral/commits/main from https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50 to https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c as well as related commits on the `gh-pages` branch. |
| 2023-06-28 | clusterProfiler, set.seed, and BiocParallel questions | Summarized at https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659. This involved explaining why we don't use as a background universe all genes in the annotation in gene ontology / KEGG pathway enrichment analyses. Instead, we use those that pass the expression filters and for which we computed differential expression statistics. |
| 2023-06-28 | Resolving a permissions issue | We resolved https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199 at https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&cid=C02RG5VGL6R, which was due to a user incorrectly having a Linux group assigned to files at JHPCE when they are not part of the user group. Uploads through Cyberduck can trigger this. The solution is to run https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example again on a qrsh session. |
| 2023-06-27 | How to make dynamic error messages | We worked on qsvaR debugging an issue raised by Renee and Louise. We then improved the error messages on qsvaR as noted at https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400. |
| 2023-06-23 | How to make a new Globus endpoint for a publication. | Using the septum lateral data, I trained Ryan on how to make this type of Globus endpoint, how to set up the right permissions, how to edit and update http://research.libd.org/globus/ (see [this](https://github.com/LieberInstitute/globus/blob/b10e0fb89c40489d23028e8ed3f7812cdea76a6b/index.Rmd#L57) for details), and how to document the Globus information on a manuscript. |
| 2023-06-22 | How to filter an SCE object prior to making an iSEE app | Using the lateral septum snRNA-seq data, I trained Heena on how to filter a SingleCellExperiment object to reduce the required memory from 7GB to less than 1GB in order to be able to deploy an iSEE app at shinyapps.io, whose computers have a maximum of 8 GB of RAM. See https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98 and related commits. |
| 2023-06-21 | How to debug an R package function from a user's report | We worked with the script where Renee had noticed issues with qsvaR. We adapted it to work on qsvaR local updates through devtools::load_all(). We also resolved https://github.com/LieberInstitute/qsvaR/issues/29 and https://github.com/LieberInstitute/qsvaR/issues/28. |
| 2023-06-08 | Options for cell type specific eQTLs | Discussed how to best pseudobulk the data prior to doing this type of analysis. In particular, how to adapt https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148 and related code, as well as plots to make such as boxplots of the min_ncells by cell type (without any filtering) and cutoffs found by `isOutlier()` (different cutoff for each cell type) + PCA plots colored by min_ncells. |
| 2023-06-08 | Overview of what we do as a team and outreach options | Discussed a few different areas of growth, including a change in direction related to our outreach activities. |
| 2023-03-13 | Updating spatialLIBD::read10xVisiumAnalysis | See https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099 for more. Led to https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6 and related commits. spatialLIBD version 1.11.10 now supports spaceranger version 2023.0208.0 output files. |
| 2023-03-13 | Verify GitHub ssh key setup and team overview | We used https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection and also went over the JHPCE and R config files from the team website. We also talked about the DSgs and how it would be beneficial for Kinnary to meet 1 on 1 with all DSgs guides to build familiarity and make it easier to ask for help in the future. |
| 2023-03-10 | Debugging spatialLIBD::read10xVisiumAnalysis | See https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179 for more details. Heena is working with spaceranger output files from 2 different versions, and in particular, version 2023.0208.0 (internal 10x Genomics version) changed their output files structure |
| 2023-03-09 | Overview on "R/Bioconductor-powered Team Data Science" | Described what we do, the DSgs system, potential for visitors at our team meeting, and research ideas. |
| 2023-03-02 | Help resolve JHPCE permission issues on the lifespan_hpc_spatial project | The solution involved asking BitSupport to login as a specific user and run https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example. Knowing the exact file paths that triggered the permission issues are important to help check with `ls -lh` and `nfs4_getfacl` under a `qrsh` session. |
| 2023-03-02 | Debug spaceranger 2023.0208.0 (likely an alpha for 2.0.1) with SpatialExperiment 1.8.0 | See the [Bioconductor Slack message](https://community-bioc.slack.com/archives/CR1BLV821/p1677783580326559) I wrote for more details. |
| 2023-03-02 | How to write a tweetorial | End result: https://twitter.com/HeenaDivecha/status/1631360771342098433 |
| 2023-03-02 | read10xVisium() in version 1.7.1 (and now 1.8.0) creates a dangerous silent error by mismatching tissue positions across multiple samples | See https://github.com/drighelli/SpatialExperiment/issues/135 and the pull request at https://github.com/drighelli/SpatialExperiment/pull/136 that resolves this issue. |
| 2023-02-28 | How to write a tweetorial | No public result just yet. |
| 2023-01-11 | Getting started with snRNA-seq | See [Slack notes](https://jhu-genomics.slack.com/archives/C0498763CUC/p1673451228034439) for further details. They involved https://github.com/LieberInstitute/template_project, https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h, and other specific instructions for setting up a project. |
| 2022-11-01 | Introduction to LIBD and our team | https://lcolladotor.github.io/bioc_team_ds for my team website, https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing for the LIBD rstats club schedule, https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials could be of interest also, https://github.com/LieberInstitute/template_project for a template for organizing projects, learn about the DSgs https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs, https://rstudio.github.io/reticulate/ for using Python from R |
| 2022-09-29 | Overview on using GitHub for Matlab projects | We talked about https://github.com/LieberInstitute/template_project, https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/, http://research.libd.org/sgejobs/reference/job_single.html, https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7, https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1, https://happygitwithr.com/, https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent. |
| 2022-09-23 | Introduction to DSgs, R, Bioconductor, JHPCE | LIBD rstats club schedule https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing. Bioconductor overview video https://youtu.be/c0Ch_sXiGDQ and related notes in case you want to skim them https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing (we typically have this for each video entry). R and RStudio intro https://youtu.be/XTuJ8vGnzBU. Example vignette https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html from https://bioconductor.org/packages/spatialLIBD/. Data Science guidance sessions https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0. JHPCE new user request https://jhpce.jhu.edu/register/user/. Making your personal website with postcards https://youtu.be/9LUCZAx3SlU. Examples include https://lahuuki.github.io/ by Louise, https://jtleek.com/, http://aejaffe.com/. |
| 2022-09-06 | How to organize OneDrive files | Related to how we organize files for a publication. |
| 2022-08-24 | How can I interpret the y-axis from `plotExplanatoryVariables()`? | Density is related to the probability density function, see [wiki](https://en.wikipedia.org/wiki/Kernel_density_estimation for) more details. [This link](https://stats.stackexchange.com/questions/4220/can-a-probability-distribution-value-exceeding-1-be-ok) is also useful for understanding the y-axis in more detail. Violin plots are just density plots rotated 90 degrees and mirrored. They are compared against boxplots frequently, which while easier to understand, are not good for non-unimodal data. See [this link](https://mode.com/blog/violin-plot-examples/#:~:text=What%20is%20a%20violin%20plot%3F,the%20density%20of%20each%20variable) for more. I like `plotExplanatoryVariables()` because it's fast to run and gives a general idea of things. See [Figure 1b](https://bioconductor.org/packages/release/bioc/vignettes/variancePartition/inst/doc/variancePartition.pdf) from `variancePartition` for a violin plot of similar data. `variancePartition` can handle mixed effects models (random effects) and in that scenario is more precise, but will also take longer to compute. |
| 2022-08-17 | DE analysis introduction | Check [EnchancedVolcano](https://bioconductor.org/packages/devel/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html) which shows a quick DE guide and finishes with some Volcano plots, which can be good enough for a grant application. If you want to use limma, you could follow [the SPEAQeasy example](http://research.libd.org/SPEAQeasy-example/de_analysis_speaqeasy.html#16_Modeling) or [our DE bootcamp](http://research.libd.org/SPEAQeasy-example/bootcamp_intro#5_Day_3) a bit. Or dive deeper into [limma's documentation](http://bioconductor.org/packages/release/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html). |
| 2022-07-19 | Help with spatialLIBD colors setup | Note that `spe$var_colors` works for `spe$var` where the `var` part has to be the same in both variable names. Also `spe$var_colors` should be a named vector such that `unique(names(spe$var_colors))` matches `unique(spe$var)`. |
| 2022-07-15 | R, RStudio, git, SSH key setup | NA |
| 2022-07-12 | LIBD Data Science | NA |
| 2022-07-06 | Data science coordination | NA |
| 2022-06-03 | Tools and resources related to recount2 and recount3 | Talked about recount2, recount3, snaptron, snapcount, recount.bwtool, recount-brain, recount predictions, megadepth, the recount3 study explorer. I also sent them all the links to them, though you can find many of them at https://jhubiostatistics.shinyapps.io/recount/, https://jhubiostatistics.shinyapps.io/recount-brain/, and http://rna.recount.bio/. |
| 2022-05-27 | Debug `spatialLIBD::sig_genes_extract_all()`. | This had already been fixed, but Abby had an older version of spatialLIBD installed. We also went over some other steps needed to deploy an `spatialLIBD` app with pseudo-bulked data and modeling results. |
| 2022-05-27 | How to debug an R package code | Resulted in a fix for [this issue](https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1137544893). |
| 2022-05-27 | Data science organization. | Brainstorming and planning session. |
| 2022-05-25 | MatLab questions. | Resolved via email. |
| 2022-05-20 | Brainomics meeting with collaborators | Review of statistical models as well as access to data |
| 2022-05-18 | Discussion on how to recruit members | Twitter, LinkedIn, jobRxiv |
| 2022-05-17 | BrainSeq Phase II development browser | Clarified what the data is showing and explained a bit the goals of the LIBD data portal that Geo Pertea is building. |
| 2022-05-06 | Career development | PhD programs and other options. |
| 2022-05-03 | How to make a tweetorial for a pre-print | See [this Twitter thread](https://twitter.com/lahuuki/status/1521581339903873024) for the result. |
| 2022-04-29 | File organization for a paper submission | Provided guidance on how to share images and how to organize supplementary files. Resulting in this [Figshare](https://doi.org/10.6084/m9.figshare.19729567.v1) zip file. |
| 2022-04-28 | In promptu meeting | NA |
| 2022-04-27 | Grant proposal feedback | NA |
| 2022-04-21 | CV feedback | NA |
| 2022-04-20 | Cover letter feedback | NA |
| 2022-04-15 | Debugged code from https://github.com/LieberInstitute/spatialLIBD/issues/33 | See https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012. |
| 2022-04-15 | Brainomics meeting with collaborators | Clarified a few questions about LIBD's SPEAQeasy RSE objects and a problem with some colData() that was missing, that we'll provide early next week. Send links to our deconvolution work with Louise A Huuki-Myers. |
| 2022-04-08 | How to delete large files from GitHub history | If the situation had been simple, that is, that the latest commit was the one that commited the large files, we could have used the solution described at http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/. Instead, we had to use https://rtyley.github.io/bfg-repo-cleaner/. |
| 2022-04-08 | macOS and JHPCE laptop set | JHPCE ssh keys (to avoid typing passwords when using qrsh), .Rprofile (both), .bashrc, .zshrc (laptop), iTerm2 (laptop), Textmate (laptop), remote port fowading for rmate, xquartz (laptop) installation and configuration. There are several LIBD rstats club videos on these topics already, but we just did all the setup fast. In particular, the .bashrc configuration was important to resolve a umask setting problem that was interfering with the permissions on the Visium IF AD project. |
| 2022-04-06 | Debugging JHPCE configuration files | See [this Slack thread](https://neuralplasticity.slack.com/archives/C010NFSQS48/p1649249130880349). Ultimately, this situation involved teaching Tony to share his code. |
| 2022-03-18 | The authenticity of host 'github.com' | I explained how the first time that you ssh into a new computer (which happens also with git pull/git push), you need to accept that the new host is authentic, which will then edit your ~/.ssh/known_hosts file. |
| 2022-03-16 | Plan how to make spatial registration R functions | This is challenging since it involves putting together code from multiple scripts/projects. We summarized the steps to do [on this GitHub issue thread](https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1069493294). |
| 2022-03-15 | Clarifying what role deconvolution can play on LCM-seq | Louise and I discussed the [Dentate Gyrus, Jaffe at al, 2020](https://doi.org/10.1038/s41593-020-0604-z) paper which used LCM-seq, what deconvolution using Bisque can do, and what role it can play at different analytical stages. |
| 2022-03-11 | How to log in to JHU-controlled laptop | We had some trouble logging in because the WiFi wasn't working without logging in. We solved this by using an ethernet cable at LIBD (which is part of the JHU network). Once we logged in once, then we could use the WiFi. |
| 2022-03-04 | How to guide others learning DS | I described different strateguies for helping others navigate the world of data science. In particular, how weekly coding sessions can be really useful. |
| 2022-03-04 | "Megadepth usage to add a new annotation to hg38 file" | Created an R script that reads in the transcripts FASTA file from GENCODE and uses BSgenome from Bioconductor to edit it and save it as a larger .fasta.gz file. Nick Eagles helped us by pointing us to http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation. |
| 2022-03-03 | How to build a cov_rse object using qSVA v1 | Linked to https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13, `/dcs04/lieber/lcolladotor/qSVA_LIBD3080/degradation_experiments/*/bed`, https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855, https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5, https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R, https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R, https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253, https://github.com/CRG-Barcelona/bwtool/wiki#installation, http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html, and http://research.libd.org/qsvaR/. Andrew Jaffe linked `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/degradation_mat_CMC_ribozero.rda`, `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/run_de_analysis_CMC.R`, `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/geneLevel_CMC_qSVsAndMod.rda` and `/users/ajaffe/Lieber/Projects/RNAseq/CMC/`. |
| 2022-03-03 | Debradation matrix for LIBD riboZ | Linked to https://github.com/LieberInstitute/qsvaR, http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28. Made public the file `/dcl01/ajaffe/data/lab/qsva_brain/brainseq_phase2_qsv/rdas/degradation_rse_phase2_usingJoint_justFirst.rda` and other ones. See https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5 for details. |
| 2022-02-24 | WGBS data analysis overview | Using BiocMAP http://research.libd.org/BiocMAP/ which Nick is finishing up, you could process the data and have R objects you could load and analyze. So, the data processing would be "easy" in a sense. Next, you might want to check the bsseq documentation https://bioconductor.org/packages/bsseq/ if you are interested in finding differentially methylated regions (DMRs) and in any case, because BiocMAP provides that type of R object as noted at http://research.libd.org/BiocMAP/outputs.html. https://jhubiostatistics.shinyapps.io/wgbsExprs/ and the companion paper is the last WGBS data analysis project I did myself with code at https://github.com/LieberInstitute/brain-epigenomics. Kira Perzel-Mandell, who worked with Shizhong, wrote https://github.com/LieberInstitute/wgbs_meqtl_analysis (link to paper there) and did other projects. If you are plotting methylation levels at some locations, then that's very direct. If you are trying to find DMRs, it can take a bit of time to get used to the large objects and the related complications (once you filter the data, it'll be more manageable). If you go into CpH instead of CpG, then memory constraints become harder to deal with. |
| 2022-02-18 | Ideas for possible future analyses | NA |
| 2022-02-16 | JHPCE array jobs and fasthplus | Related to JHPCE https://github.com/ntdyjack/fasthplus and http://research.libd.org/sgejobs/ |
| 2022-02-15 | snRNA-seq and deconvolution overview | Brainstormed several ideas related to deconvolution of bulk data as well as uses of snRNA-seq. Refered to OSCA https://bioconductor.org/books/release/OSCA/ at time. |
| 2021-12-06 | JHPCE file permissions and ACLs | Motivated /dcs04/lieber/lcolladotor/_jhpce_org_LIBD001/update_permissions_spatialteam.sh |
| 2021-10-20 | Transfering files between JHPCE disks | Explained some scripts such as /dcs04/lieber/lcolladotor/BrainSEQ_LIBD001/_jhpce_org |
| 2021-09-27 | Presentation preparation feedback | NA |
| 2021-08-06 | How to write a Twitter thread for a paper | We revised key features of Brenda Pardo's Twitter thread https://twitter.com/PardoBree/status/1388253938391175173?s=20 and some features that you would want your Twitter thread to have. |
| 2021-08-03 | Refining an abstract submission | We talked about all the details that go into writing an abstract for a conference. These details can take longer to complete than initially anticipated. |
| 2021-07-30 | Git branches | We revised how to make a new branch in Git working with https://github.com/lmweber/OSTA-book |
| 2021-07-28 | DNA genotyping overview | We discussed the current DNA genotype pipeline and potential changes others might want. |
| 2021-07-20 | Overview of the RNA-seq universe | From SPEAQeasy, to qSVA, deconvolution, TWAS, etc. It might be best to give a talk about all these topics to whoever is interested at LIBD since we know about all these pieces but others might not. |
| 2021-07-13 | Setting up a team at LIBD | We talked about https://lcolladotor.github.io/bioc_team_ds/ among other topics. |
| 2021-07-01 | How to set up a GitHub actions-powered website | This has changed over time. The most recent setup (as of 2021-08-06) is at https://github.com/lmweber/OSTA-book |
| 2021-06-29 | Planning session for getting up to speed with SpatialExperiment, spatialLIBD, here and ggplot2 | We talked about the set of videos and resources publicly available to get to learn about R, Bioconductor and these packages. It was a lot of information and I'm sure that we'll have more sessions later on to clarify questions as they arise. |
| 2021-06-29 | How to choose and add a licence to a GitHub repo | We talked about https://choosealicense.com/, https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository, https://creativecommons.org/choose/. |
| 2021-06-29 | Designing an MS-level class | We discussed the posibility of proposing a class that has a mix of biology lectures and some practical coding sessions. The coding sessions could be based after the CDSB 2021 mini courses series. |
| 2021-06-21 | Locating public data at JHPCE + bamtofastq workflow | We talked about the public data from Synapse and worked on locating some of it at JHPCE. I explained also how we have used bamtofastq when others don't share their FASTQ files. Basically, we use bamtofastq to reconstruct the original FASTQ files (you might have to append the unmapped FASTQs if they are provided by the original authors) and then run FASTQ-processing tools like our SPEAQeasy pipeline. |
| 2021-06-17 | How to transfer recount3 data | We met to figure out how to transfer the data for this collaboration, which ended up being quite tricky and involved using an SSH tunnel for rsync. Basically, a command like this one: `rsync -avzh -e 'ssh -o ProxyJump=user@server_somewhere_for_the_tunnel' *_files_of_interest user@final_server:/final_path/` |
| 2021-06-17 | How to get started with R | We discussed the different possible goals and ways to get to them and determined a path forward that relies on the rest of the DSgs system as well as existing LIBD rstats club videos to get started. It'll be supported by a meeting with Leo and others involved in the spatial transcriptomics projects from the R side. |
| 2021-06-17 | Brainstorming session on collaboration | I showed the initial version of the BrainSeq Phase II developmental expression explorer developed by Bill Ulrich and Aaron Goldman. We discussed other possibilities for the collaboration including helping set up SPEAQeasy in their servers. |
| 2021-06-02 | How to prepare for a LIBD seminar part 4 | We discussed the changes that we should make and the ones we shouldn't based on the feedback received from the practice talk with Keri's group. The feedback was excellent and very useful, however, we discussed which suggestions were maybe complicating the presentation too much and how to best incorporate them. |
| 2021-05-26 | How to prepare for a LIBD seminar part 3 | Similar to part 2. The presentation was looking very polished by this meeting! |
| 2021-05-24 | Grant application review | Helped review the grant application and revise the text. |
| 2021-05-21 | PredictDB weights | Further updates on asking for help with this complicated project. Suggested reaching out to other LIBD members who have tried running the software before. |
| 2021-05-20 | Help preparing a figure for a grant application | Discussed with Josh what figures we needed to prepare and how to generate them. This also involved base_size = 30 with ggplot2. Otherwise the font was too small to read it on the grant. |
| 2021-05-20 | How to prepare for a LIBD seminar part 2 | Now that a first major draft of the presentation was complete, we revised it and proposed some changes to it. Some of them involved remaking the figures using base_size = 20 or even 30 with ggplot2. For future work, it might be best to save the images with that font size from the beginning instead of having to update them later. |
| 2021-05-10 | Setting up covr using GitHub actions in an R package | We went over the details of the GitHub actions workflow created by biocthis, the parts that you need to run from biocthis::use_bioc_pkg_templates(), and resolving issues prompted by R CMD check + BiocCheck |
| 2021-05-07 | How to get started with computational biology | I gave an overview of the resources and materials we have created as well as the LIBD bootcamps that we have publicly available to help others get started with computational biology. This is likely the first of several meetings. |
| 2021-05-07 | How to insert a figure using the Oxford Bioinfomratics Overleaf LaTeX template | We spent a significant amount troubleshooting the problem here (nearly two hours?). In the end, using the insert image "here" LaTeX syntax resolved the problem. |
| 2021-05-06 | PredictDB weights | General advice on how to ask for help from open source projects |
| 2021-05-06 | How to prepare for a LIBD seminar | Initial planning session for the LIBD seminar. We talked about the different slides we planned to show and the flow of the presentation. We also scheduled 3 practice talks with different groups of people. |
| 2021-05-03 | TWAS prep | Overview of the TWAS pipeline |
| 2021-04-08 | General guidance on how to learn how to do differential expression analyses | Created a learning plan involving several team members that is now documented at a channel in JHU Genomics Slack. That way everyone can keep track of what others are teaching the person who asked the question, and we can have a common communication channel among everyone involved. Sessions include learning how to interact with JHPCE through RStudio, how to run SPEAQeasy, key R/Bioconductor packages (limma, SummarizedExperiment), and explore possible visualizations of interest. |
| 2021-03-29 | spatialLIBD support | Discussion about features that spatialLIBD could support to make it easier to use for newer data. |
| 2021-03-25 | Creating a documentation website using bookdown | Help with understanding [bookdown](https://bookdown.org/yihui/bookdown/) and ideas on how to organize a documentation website (chapters and information to consider). |
| 2021-03-25 | Clarification of LIBD support | Discussed the research project proposal and how we can help. |
| 2021-03-25 | General advice | General advice. Received feedback regarding file backups. |
| 2021-03-24 | Download SRA data and run SPEAQeasy | I identified that https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829 contains both miRNA and RNA-seq data. Using data from the SRA Run selector at https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&o=acc_s%3Aa I was able to filter the data to the RNA-seq samples and subset the input samples.manifest file for SPEAQeasy. |
| 2021-03-24 | Explain the feasibility of an on-demand end user interface for DEG and eQTLs | To identify DEGs or eQTLs, the user has to choose what covariates to adjust for. Some studies might require more complicated models than others, as they might be from more than one brain region or present some batch effect. In addition, in newer projects we are considering adjusting for cell type fractions derived from deconvolution. Even if all the covariates were fixed across datasets, in DEGs we have typically adjusted for qSVs which are computed in a dataset-dependent fashion and in essence are PCs derived from degradation-associated genomic regions. Similarly, for eQTLs we adjust for expression PCs. In both we adjust for ancestry PCs. Computing PCs is input-dependent: you won't get the same values or PCs if you change which samples you are using. So, computing DEGs and eQTLs is not as simple as say a map-reduce problem where you compute the same thing for every input sample, then aggregate the per-sample results. We have steps that depend on the whole sample set. Then there's also steps for filtering expression features based on some minimum expression cutoffs. Plus, differences between say polyA and RiboZero data, which don't make sense to process together. For these reasons, I don't think that an on-demand computation pipeline is feasible based on a user selecting a set of input samples. Constructing RNA-seq objects (RangedSummarizedExperiment objects) as well as DNA genotyping VCF files based on a set of input samples is feasible and that's what others are building. After that, I think that it's best to adapt code we have (or even use newer tools) after carefully exploring the data. |
| 2021-03-11 | BSP2 eQTLs with a subset of SNPs | Adapted code and shared it at https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock |
| 2021-03-10 | How to write a small methods paper | Created GitHub issues at https://github.com/LieberInstitute/VisiumImgProcessing/issues |
| 2021-02-05 | BSP2 LIBD eQTLs request | NA |
| 2021-01-29 | Help determining what data is available and what can be used for an analysis | Use of galaxybrain for identifying samples with DNAm, DNA genotype and RNA-seq data |
| 2021-01-26 | Guidance on how to download data from a study in GEO | NA |
| 2021-01-17 | Career development | Practiced questions for interviews. |
| 2021-01-14 | biocthis macOS GHA bug | [BioC Support website answer](https://support.bioconductor.org/p/p134069/#p134141) |
| 2021-01-11 | Questions related to accessing results from the DG paper by Andrew Jaffe | NA |
| 2020-11-25 | Accesing CMC RNA-seq data | Started PsychENCODE access request process |
| 2020-11-11 | Differences between recount2's "bp_length" and SPEAQeasy's "Length" | [Check Slack helpdesk channel](https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1605123296024100) |
| 2020-11-11 | GitHub actions workflow issue with macOS on R-devel | [GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/15) |
| 2020-11-03 | Fixed two bugs on recount3 related to locate_url() for the BigWig files. | Resulted in recount3 versions 1.1.1 and 1.1.2 on bioc-devel (and 1.0.2 on bioc-release) |
| 2020-11-03 | Fixed a bug on biocthis for release branches | [GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/13) |
| 2020-11-03 | Issue with knitcitations | [GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/14) |
| 2020-10-30 | Issue with JHPCE file permissions | Explained the `chgrp` and `umask` commands plus `~/.bashrc` file configuration. I also mentioned why we don't use `chmod 777` at JHPCE. Details on [Slack](https://jhu-genomics.slack.com/archives/CR9NYA0BF/p1604086263148400?thread_ts=1603826219.094600&cid=CR9NYA0BF) (private channel). |
| 2020-10-30 | recount3 GTEx data access | [BioC Support website answer](https://support.bioconductor.org/p/p132487/#p132521) |
| 2020-10-28 | exceRpt pipeline counts | [BioC Support website answer](https://support.bioconductor.org/p/p132450/#p132464) |
| 2020-10-21 | Ballgown stattest | [BioC Support website answer](https://support.bioconductor.org/p/134914/#134915) |

