<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Full logs | Data Science guidance session logs</title>
  <meta name="description" content="Full logs | Data Science guidance session logs" />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="Full logs | Data Science guidance session logs" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="lcolladotor/dsgs_logs" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Full logs | Data Science guidance session logs" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="icon_32.png" type="image/x-icon" />
<link rel="prev" href="index.html"/>
<link rel="next" href="r-session-information.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.4/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>
<script src="libs/core-js-2.5.3/shim.min.js"></script>
<script src="libs/react-18.2.0/react.min.js"></script>
<script src="libs/react-18.2.0/react-dom.min.js"></script>
<script src="libs/reactwidget-2.0.0/react-tools.js"></script>
<link href="libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="libs/reactable-binding-0.4.4/reactable.js"></script>


<style type="text/css">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#dsgs-guides"><i class="fa fa-check"></i>DSgs-guides</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#graphs"><i class="fa fa-check"></i>Graphs</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="full-logs.html"><a href="full-logs.html"><i class="fa fa-check"></i>Full logs</a></li>
<li class="chapter" data-level="" data-path="r-session-information.html"><a href="r-session-information.html"><i class="fa fa-check"></i>R session information</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data Science guidance session logs</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="full-logs" class="section level1 unnumbered hasAnchor">
<h1>Full logs<a href="full-logs.html#full-logs" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Here you an explore the full logs. The name of the person asking the question(s) is not displayed for privacy.</p>
<p>The idea is that this information could be useful for people who have similar questions, as well as for DSgs guides who want to re-use material from previous answers when helping someone.</p>
<div id="egkyysggow" class=".gt_table" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#egkyysggow table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#egkyysggow thead, #egkyysggow tbody, #egkyysggow tfoot, #egkyysggow tr, #egkyysggow td, #egkyysggow th {
  border-style: none;
}

#egkyysggow p {
  margin: 0;
  padding: 0;
}

#egkyysggow .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 1200px;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#egkyysggow .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#egkyysggow .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#egkyysggow .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#egkyysggow .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#egkyysggow .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#egkyysggow .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#egkyysggow .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#egkyysggow .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#egkyysggow .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#egkyysggow .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#egkyysggow .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#egkyysggow .gt_spanner_row {
  border-bottom-style: hidden;
}

#egkyysggow .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#egkyysggow .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#egkyysggow .gt_from_md > :first-child {
  margin-top: 0;
}

#egkyysggow .gt_from_md > :last-child {
  margin-bottom: 0;
}

#egkyysggow .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#egkyysggow .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#egkyysggow .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#egkyysggow .gt_row_group_first td {
  border-top-width: 2px;
}

#egkyysggow .gt_row_group_first th {
  border-top-width: 2px;
}

#egkyysggow .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#egkyysggow .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#egkyysggow .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#egkyysggow .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#egkyysggow .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#egkyysggow .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#egkyysggow .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#egkyysggow .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#egkyysggow .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#egkyysggow .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#egkyysggow .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#egkyysggow .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#egkyysggow .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#egkyysggow .gt_left {
  text-align: left;
}

#egkyysggow .gt_center {
  text-align: center;
}

#egkyysggow .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#egkyysggow .gt_font_normal {
  font-weight: normal;
}

#egkyysggow .gt_font_bold {
  font-weight: bold;
}

#egkyysggow .gt_font_italic {
  font-style: italic;
}

#egkyysggow .gt_super {
  font-size: 65%;
}

#egkyysggow .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#egkyysggow .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#egkyysggow .gt_indent_1 {
  text-indent: 5px;
}

#egkyysggow .gt_indent_2 {
  text-indent: 10px;
}

#egkyysggow .gt_indent_3 {
  text-indent: 15px;
}

#egkyysggow .gt_indent_4 {
  text-indent: 20px;
}

#egkyysggow .gt_indent_5 {
  text-indent: 25px;
}

#egkyysggow .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#egkyysggow div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>
<div id="egkyysggow" class="reactable html-widget" style="width:1200px;height:auto;"></div>
<script type="application/json" data-for="egkyysggow">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"date":["2025-04-21","2025-04-21","2025-04-18","2025-04-17","2025-04-16","2025-04-09","2025-04-08","2025-04-04","2025-04-03","2025-04-03","2025-04-02","2025-03-27","2025-03-26","2025-03-26","2025-03-25","2025-03-24","2025-03-21","2025-03-20","2025-03-19","2025-03-14","2025-03-13","2025-03-13","2025-03-13","2025-03-07","2025-03-06","2025-03-05","2025-03-04","2025-02-28","2025-02-28","2025-02-27","2025-02-27","2025-02-27","2025-02-27","2025-02-25","2025-02-21","2025-02-20","2025-02-20","2025-02-18","2025-02-18","2025-02-13","2025-02-13","2025-02-13","2025-02-12","2025-02-06","2025-02-06","2025-02-05","2025-01-30","2025-01-28","2025-01-24","2025-01-24","2025-01-23","2025-01-22","2025-01-17","2025-01-17","2025-01-15","2025-01-15","2025-01-10","2025-01-06","2024-12-19","2024-12-10","2024-12-10","2024-12-10","2024-12-05","2024-12-04","2024-12-04","2024-11-27","2024-11-26","2024-11-25","2024-11-22","2024-11-21","2024-11-20","2024-11-19","2024-11-18","2024-11-14","2024-11-07","2024-11-01","2024-10-31","2024-10-25","2024-10-23","2024-10-18","2024-10-18","2024-10-16","2024-10-11","2024-10-11","2024-10-11","2024-10-10","2024-10-02","2024-10-02","2024-10-01","2024-09-27","2024-09-27","2024-09-25","2024-09-24","2024-09-24","2024-09-23","2024-09-20","2024-09-20","2024-09-19","2024-09-17","2024-09-16","2024-09-16","2024-09-16","2024-09-13","2024-09-13","2024-09-11","2024-09-09","2024-09-09","2024-09-06","2024-09-06","2024-09-06","2024-09-06","2024-09-05","2024-09-05","2024-09-04","2024-08-30","2024-08-29","2024-08-28","2024-08-28","2024-08-26","2024-08-26","2024-08-22","2024-08-21","2024-08-20","2024-08-19","2024-08-16","2024-08-15","2024-08-13","2024-08-12","2024-08-09","2024-08-08","2024-08-06","2024-08-05","2024-08-02","2024-08-01","2024-07-31","2024-07-30","2024-07-29","2024-07-26","2024-07-26","2024-07-25","2024-07-24","2024-07-22","2024-07-18","2024-07-18","2024-07-18","2024-07-15","2024-07-11","2024-07-11","2024-07-11","2024-07-09","2024-07-08","2024-07-03","2024-06-26","2024-06-26","2024-06-25","2024-06-21","2024-06-11","2024-06-11","2024-06-08","2024-06-03","2024-06-03","2024-05-30","2024-05-24","2024-05-22","2024-05-21","2024-05-15","2024-05-14","2024-05-14","2024-05-14","2024-05-14","2024-05-13","2024-05-08","2024-05-02","2024-05-01","2024-05-01","2024-05-01","2024-05-01","2024-04-30","2024-04-29","2024-04-29","2024-04-28","2024-04-28","2024-04-28","2024-04-26","2024-04-25","2024-04-25","2024-04-25","2024-04-24","2024-04-24","2024-04-23","2024-04-23","2024-04-19","2024-04-18","2024-04-18","2024-04-18","2024-04-18","2024-04-16","2024-04-15","2024-04-15","2024-04-12","2024-04-11","2024-04-10","2024-04-08","2024-04-05","2024-04-05","2024-04-05","2024-04-03","2024-04-02","2024-04-01","2024-04-01","2024-03-29","2024-03-27","2024-03-27","2024-03-27","2024-03-25","2024-03-25","2024-03-25","2024-03-23","2024-03-22","2024-03-22","2024-03-20","2024-03-20","2024-03-20","2024-03-19","2024-03-19","2024-03-19","2024-03-15","2024-03-15","2024-03-14","2024-03-14","2024-03-14","2024-03-13","2024-03-12","2024-03-11","2024-03-11","2024-03-07","2024-03-07","2024-03-05","2024-03-05","2024-03-04","2024-03-01","2024-03-01","2024-02-28","2024-02-28","2024-02-26","2024-02-23","2024-02-16","2024-02-16","2024-02-16","2024-02-12","2024-02-09","2024-02-08","2024-02-07","2024-02-05","2024-02-02","2024-02-01","2024-01-30","2024-01-29","2024-01-26","2024-01-26","2024-01-25","2024-01-25","2024-01-24","2024-01-24","2024-01-22","2024-01-22","2024-01-18","2024-01-11","2024-01-08","2024-01-05","2023-12-12","2023-12-11","2023-12-08","2023-12-08","2023-12-08","2023-12-08","2023-12-05","2023-12-01","2023-11-30","2023-11-29","2023-11-29","2023-11-21","2023-11-16","2023-11-16","2023-11-15","2023-11-15","2023-11-08","2023-11-08","2023-11-08","2023-11-03","2023-11-01","2023-11-01","2023-11-01","2023-10-30","2023-10-30","2023-10-27","2023-10-27","2023-10-26","2023-10-26","2023-10-25","2023-10-25","2023-10-24","2023-10-20","2023-10-19","2023-10-18","2023-10-17","2023-10-16","2023-10-13","2023-10-13","2023-10-12","2023-10-12","2023-10-11","2023-10-11","2023-10-06","2023-10-05","2023-10-04","2023-09-29","2023-09-29","2023-09-28","2023-09-27","2023-09-27","2023-09-26","2023-09-25","2023-09-22","2023-09-22","2023-09-22","2023-09-21","2023-09-21","2023-09-21","2023-09-19","2023-09-19","2023-09-14","2023-09-13","2023-09-12","2023-09-06","2023-08-28","2023-08-23","2023-08-22","2023-08-21","2023-08-21","2023-08-18","2023-08-17","2023-08-16","2023-08-16","2023-08-14","2023-08-14","2023-08-11","2023-08-11","2023-08-10","2023-08-10","2023-08-09","2023-08-09","2023-08-09","2023-08-09","2023-08-07","2023-08-03","2023-08-02","2023-07-27","2023-07-27","2023-07-27","2023-07-27","2023-07-26","2023-07-26","2023-07-26","2023-07-26","2023-07-26","2023-07-25","2023-07-25","2023-07-24","2023-07-21","2023-07-20","2023-07-20","2023-07-20","2023-07-19","2023-07-18","2023-07-17","2023-07-17","2023-07-12","2023-07-07","2023-07-01","2023-06-29","2023-06-28","2023-06-28","2023-06-28","2023-06-27","2023-06-23","2023-06-22","2023-06-21","2023-06-15","2023-06-14","2023-06-09","2023-06-08","2023-06-08","2023-06-08","2023-06-08","2023-06-07","2023-06-02","2023-06-02","2023-06-01","2023-05-26","2023-05-25","2023-05-25","2023-05-24","2023-05-18","2023-05-18","2023-05-17","2023-05-16","2023-05-09","2023-05-02","2023-04-25","2023-04-21","2023-04-19","2023-04-17","2023-04-14","2023-04-12","2023-04-11","2023-04-10","2023-04-10","2023-04-07","2023-04-06","2023-04-06","2023-04-04","2023-03-31","2023-03-28","2023-03-24","2023-03-23","2023-03-23","2023-03-22","2023-03-22","2023-03-22","2023-03-21","2023-03-21","2023-03-21","2023-03-20","2023-03-15","2023-03-13","2023-03-13","2023-03-10","2023-03-09","2023-03-08","2023-03-02","2023-03-02","2023-03-02","2023-03-02","2023-03-02","2023-03-01","2023-03-01","2023-03-01","2023-02-28","2023-02-21","2023-02-17","2023-02-15","2023-02-10","2023-02-09","2023-02-06","2023-02-03","2023-02-01","2023-01-25","2023-01-25","2023-01-25","2023-01-20","2023-01-19","2023-01-18","2023-01-18","2023-01-18","2023-01-13","2023-01-11","2023-01-11","2023-01-10","2023-01-04","2023-01-03","2022-12-14","2022-12-14","2022-12-06","2022-12-05","2022-12-01","2022-11-30","2022-11-30","2022-11-30","2022-11-23","2022-11-16","2022-11-15","2022-11-03","2022-11-01","2022-10-28","2022-10-27","2022-10-26","2022-10-17","2022-10-14","2022-10-13","2022-10-11","2022-10-10","2022-10-05","2022-10-05","2022-09-30","2022-09-30","2022-09-29","2022-09-26","2022-09-23","2022-09-23","2022-09-16","2022-09-16","2022-09-12","2022-09-09","2022-09-06","2022-08-24","2022-08-24","2022-08-22","2022-08-22","2022-08-18","2022-08-17","2022-08-17","2022-08-10","2022-08-09","2022-08-09","2022-08-09","2022-08-08","2022-08-03","2022-07-28","2022-07-20","2022-07-19","2022-07-19","2022-07-18","2022-07-15","2022-07-15","2022-07-13","2022-07-12","2022-07-07","2022-07-07","2022-07-06","2022-07-06","2022-07-05","2022-07-01","2022-06-29","2022-06-29","2022-06-28","2022-06-23","2022-06-23","2022-06-21","2022-06-21","2022-06-17","2022-06-15","2022-06-15","2022-06-15","2022-06-15","2022-06-10","2022-06-03","2022-06-03","2022-06-02","2022-06-01","2022-05-27","2022-05-27","2022-05-27","2022-05-27","2022-05-25","2022-05-20","2022-05-20","2022-05-18","2022-05-18","2022-05-17","2022-05-17","2022-05-17","2022-05-17","2022-05-10","2022-05-09","2022-05-06","2022-05-05","2022-05-05","2022-05-03","2022-04-29","2022-04-29","2022-04-28","2022-04-27","2022-04-27","2022-04-27","2022-04-21","2022-04-20","2022-04-15","2022-04-15","2022-04-08","2022-04-08","2022-04-08","2022-04-07","2022-04-07","2022-04-07","2022-04-06","2022-04-06","2022-04-06","2022-04-05","2022-04-04","2022-04-04","2022-04-01","2022-03-27","2022-03-25","2022-03-24","2022-03-22","2022-03-18","2022-03-18","2022-03-17","2022-03-17","2022-03-16","2022-03-16","2022-03-15","2022-03-15","2022-03-14","2022-03-11","2022-03-09","2022-03-09","2022-03-09","2022-03-09","2022-03-09","2022-03-04","2022-03-04","2022-03-04","2022-03-04","2022-03-03","2022-03-03","2022-03-02","2022-03-02","2022-02-25","2022-02-24","2022-02-24","2022-02-24","2022-02-23","2022-02-18","2022-02-17","2022-02-16","2022-02-16","2022-02-15","2022-02-15","2022-02-15","2022-02-14","2022-02-11","2022-02-11","2022-02-10","2022-02-07","2022-02-04","2022-02-04","2022-01-25","2022-01-21","2022-01-21","2022-01-21","2022-01-21","2022-01-21","2022-01-20","2022-01-20","2022-01-19","2021-12-15","2021-12-09","2021-12-08","2021-12-06","2021-11-18","2021-11-18","2021-11-18","2021-11-17","2021-11-16","2021-11-15","2021-11-04","2021-11-03","2021-10-29","2021-10-25","2021-10-23","2021-10-20","2021-10-14","2021-10-07","2021-10-06","2021-10-04","2021-09-27","2021-09-22","2021-09-20","2021-09-14","2021-09-14","2021-09-10","2021-09-09","2021-09-09","2021-09-09","2021-09-09","2021-09-09","2021-09-08","2021-09-07","2021-09-03","2021-09-03","2021-09-02","2021-09-02","2021-08-30","2021-08-30","2021-08-30","2021-08-26","2021-08-26","2021-08-25","2021-08-23","2021-08-23","2021-08-23","2021-08-21","2021-08-20","2021-08-18","2021-08-11","2021-08-06","2021-08-06","2021-08-03","2021-07-30","2021-07-28","2021-07-28","2021-07-20","2021-07-20","2021-07-15","2021-07-15","2021-07-14","2021-07-14","2021-07-14","2021-07-14","2021-07-14","2021-07-13","2021-07-13","2021-07-09","2021-07-09","2021-07-09","2021-07-02","2021-07-01","2021-07-01","2021-06-30","2021-06-30","2021-06-29","2021-06-29","2021-06-29","2021-06-24","2021-06-24","2021-06-24","2021-06-21","2021-06-21","2021-06-17","2021-06-17","2021-06-17","2021-06-17","2021-06-16","2021-06-16","2021-06-15","2021-06-10","2021-06-05","2021-06-04","2021-06-03","2021-06-02","2021-06-02","2021-06-01","2021-06-01","2021-05-28","2021-05-26","2021-05-24","2021-05-21","2021-05-20","2021-05-20","2021-05-20","2021-05-19","2021-05-19","2021-05-14","2021-05-13","2021-05-12","2021-05-10","2021-05-07","2021-05-07","2021-05-06","2021-05-06","2021-05-03","2021-05-03","2021-04-28","2021-04-28","2021-04-26","2021-04-22","2021-04-22","2021-04-20","2021-04-19","2021-04-19","2021-04-15","2021-04-15","2021-04-12","2021-04-09","2021-04-09","2021-04-08","2021-04-07","2021-04-01","2021-03-30","2021-03-29","2021-03-26","2021-03-26","2021-03-25","2021-03-25","2021-03-25","2021-03-24","2021-03-24","2021-03-24","2021-03-23","2021-03-22","2021-03-19","2021-03-18","2021-03-18","2021-03-17","2021-03-17","2021-03-16","2021-03-12","2021-03-12","2021-03-11","2021-03-10","2021-03-10","2021-03-06","2021-03-05","2021-03-05","2021-03-03","2021-03-02","2021-03-02","2021-02-26","2021-02-26","2021-02-25","2021-02-23","2021-02-19","2021-02-12","2021-02-09","2021-02-05","2021-02-05","2021-02-05","2021-02-04","2021-02-04","2021-01-29","2021-01-27","2021-01-26","2021-01-17","2021-01-14","2021-01-11","2021-01-06","2020-12-09","2020-12-08","2020-12-07","2020-12-03","2020-12-03","2020-12-03","2020-12-02","2020-12-01","2020-11-25","2020-11-23","2020-11-20","2020-11-19","2020-11-19","2020-11-18","2020-11-13","2020-11-12","2020-11-12","2020-11-11","2020-11-11","2020-11-11","2020-11-05","2020-11-04","2020-11-04","2020-11-03","2020-11-03","2020-11-03","2020-11-02","2020-10-30","2020-10-30","2020-10-28","2020-10-28","2020-10-28","2020-10-28","2020-10-28","2020-10-27","2020-10-26","2020-10-21","2020-10-21","2020-10-21","2020-10-21","2020-10-20"],"guide":["Leo","Louise","Louise","Nick","Nick","Leo","Louise","Leo","Leo","Leo","Louise","Leo","Nick","Leo","Manisha","Nick","Leo","Leo","Louise","Cynthia","Leo","Leo","Louise","Nick","Cynthia","Louise","Nick","Nick","Leo","Nick","Leo","Leo","Cynthia","Nick","Leo","Nick","Leo","Nick","Leo","Cynthia","Nick","Leo","Leo","Leo","Leo","Leo","Louise","Louise","Nick","Louise","Leo","Nick","Nick","Leo","Nick","Louise","Nick","Louise","Louise","Leo","Leo","Leo","Nick","Louise","Nick","Louise","Nick","Nick","Louise","Nick","Nick","Nick","Nick","Nick","Cynthia","Louise","Louise","Cynthia","Nick","Nick","Louise","Nick","Leo","Leo","Leo","Nick","Louise","Leo","Nick","Leo","Leo","Leo","Cynthia","Nick","Louise","Cynthia","Nick","Leo","Cynthia","Cynthia","Leo","Louise","Cynthia","Leo","Leo","Nick","Leo","Cynthia","Leo","Leo","Louise","Leo","Louise","Cynthia","Cynthia","Cynthia","Louise","Leo","Cynthia","Leo","Cynthia","Cynthia","Nick","Leo","Nick","Leo","Leo","Louise","Leo","Nick","Nick","Leo","Nick","Leo","Louise","Leo","Leo","Leo","Nick","Leo","Leo","Leo","Leo","Leo","Louise","Louise","Leo","Leo","Louise","Leo","Leo","Leo","Louise","Leo","Leo","Nick","Nick","Nick","Nick","Nick","Louise","Nick","Leo","Leo","Leo","Leo","Nick","Nick","Leo","Louise","Louise","Nick","Leo","Nick","Leo","Louise","Leo","Leo","Nick","Louise","Nick","Nick","Louise","Nick","Louise","Louise","Leo","Nick","Leo","Nick","Leo","Nick","Leo","Leo","Leo","Cynthia","Leo","Nick","Leo","Leo","Cynthia","Nick","Leo","Cynthia","Cynthia","Leo","Leo","Leo","Leo","Louise","Leo","Louise","Maddy","Cynthia","Maddy","Leo","Cynthia","Louise","Leo","Cynthia","Nick","Leo","Leo","Leo","Leo","Nick","Leo","Nick","Leo","Leo","Louise","Nick","Leo","Maddy","Louise","Leo","Nick","Leo","Nick","Nick","Maddy","Leo","Louise","Leo","Leo","Leo","Leo","Leo","Nick","Nick","Hedia","Leo","Cynthia","Nick","Nick","Nick","Nick","Leo","Nick","Nick","Leo","Louise","Leo","Nick","Louise","Louise","Hedia","Leo","Nick","Nick","Nick","Nick","Nick","Hedia","Nick","Nick","Nick","Leo","Louise","Leo","Louise","Nick","Leo","Leo","Leo","Louise","Leo","Leo","Nick","Leo","Leo","Leo","Nick","Nick","Nick","Leo","Leo","Leo","Leo","Nick","Louise","Nick","Leo","Louise","Leo","Nick","Nick","Nick","Nick","Leo","Louise","Nick","Nick","Leo","Louise","Nick","Leo","Louise","Nick","Leo","Nick","Nick","Nick","Louise","Leo","Leo","Leo","Louise","Louise","Nick","Nick","Leo","Leo","Nick","Louise","Nick","Nick","Nick","Louise","Leo","Leo","Nick","Nick","Nick","Nick","Leo","Nick","Nick","Louise","Leo","Nick","Louise","Maddy","Maddy","Nick","Maddy","Maddy","Leo","Maddy","Nick","Leo","Louise","Louise","Leo","Leo","Leo","Louise","Nick","Maddy","Maddy","Maddy","Leo","Leo","Nick","Nick","Maddy","Nick","Nick","Maddy","Maddy","Leo","Leo","Leo","Louise","Leo","Leo","Leo","Leo","Louise","Louise","Nick","Nick","Louise","Leo","Leo","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Louise","Nick","Nick","Nick","Nick","Nick","Nick","Hedia","Ryan","Nick","Cynthia","Nick","Nick","Cynthia","Nick","Hedia","Nick","Louise","Hedia","Louise","Cynthia","Nick","Nick","Hedia","Hedia","Ryan","Louise","Nick","Leo","Leo","Leo","Leo","Nick","Hedia","Leo","Leo","Leo","Leo","Nick","Nick","Louise","Leo","Louise","Louise","Nick","Louise","Louise","Nick","Nick","Nick","Nick","Nick","Hedia","Nick","Louise","Louise","Nick","Hedia","Louise","Leo","Louise","Nick","Louise","Nick","Louise","Louise","Nick","Nick","Nick","Louise","Louise","Nick","Nick","Nick","Nick","Louise","Leo","Nick","Louise","Louise","Louise","Louise","Nick","Louise","Louise","Louise","Louise","Louise","Louise","Leo","Nick","Nick","Leo","Nick","Nick","Nick","Maddy","Leo","Leo","Nick","Nick","Louise","Louise","Louise","Leo","Louise","Maddy","Maddy","Maddy","Maddy","Louise","Maddy","Abby","Leo","Maddy","Nick","Louise","Leo","Nick","Leo","Louise","Louise","Nick","Leo","Louise","Nick","Abby","Nick","Louise","Nick","Maddy","Nick","Abby","Louise","Abby","Abby","Louise","Louise","Maddy","Leo","Louise","Louise","Louise","Leo","Leo","Abby","Leo","Leo","Abby","Leo","Louise","Leo","Abby","Abby","Leo","Maddy","Abby","Louise","Leo","Louise","Nick","Leo","Leo","Nick","Leo","Louise","Leo","Abby","Leo","Leo","Leo","Leo","Leo","Abby","Leo","Louise","Maddy","Maddy","Maddy","Leo","Louise","Maddy","Josh","Abby","Nick","Abby","Nick","Louise","Maddy","Abby","Leo","Abby","Nick","Leo","Nick","Leo","Louise","Nick","Leo","Nick","Maddy","Nick","Maddy","Maddy","Nick","Leo","Louise","Leo","Leo","Leo","Josh","Nick","Maddy","Leo","Louise","Louise","Maddy","Leo","Maddy","Leo","Maddy","Leo","Louise","Maddy","Maddy","Maddy","Maddy","Maddy","Nick","Nick","Nick","Maddy","Nick","Maddy","Maddy","Maddy","Maddy","Louise","Maddy","Louise","Nick","Maddy","Maddy","Leo","Maddy","Maddy","Maddy","Nick","Maddy","Maddy","Maddy","Louise","Louise","Louise","Louise","Leo","Maddy","Maddy","Nick","Nick","Leo","Nick","Maddy","Maddy","Maddy","Nick","Maddy","Maddy","Maddy","Maddy","Maddy","Maddy","Maddy","Arta","Nick","Maddy","Maddy","Maddy","Maddy","Maddy","Louise","Nick","Nick","Maddy","Maddy","Louise","Maddy","Louise","Maddy","Maddy","Leo","Nick","Leo","Leo","Leo","Josh","Maddy","Leo","Maddy","Maddy","Maddy","Maddy","Maddy","Nick","Nick","Nick","Leo","Maddy","Maddy","Maddy","Nick","Arta","Leo","Nick","Louise","Leo","Leo","Leo","Maddy","Maddy","Maddy","Leo","Arta","Leo","Leo","Leo","Louise","Nick","Josh","Arta","Josh","Josh","Arta","Louise","Arta","Leo","Maddy","Josh","Louise","Leo","Leo","Leo","Leo","Leo","Louise","Nick","Nick","Josh","Nick","Louise","Leo","Leo","Leo","Leo","Leo","Nick","Leo","Nick","Arta","Nick","Nick","Nick","Arta","Louise","Nick","Maddy","Louise","Nick","Maddy","Maddy","Leo","Nick","Maddy","Maddy","Leo","Louise","Josh","Leo","Leo","Leo","Maddy","Leo","Leo","Maddy","Nick","Louise","Nick","Arta","Maddy","Maddy","Josh","Maddy","Maddy","Leo","Nick","Leo","Maddy","Louise","Maddy","Nick","Nick","Nick","Maddy","Maddy","Nick","Nick","Maddy","Maddy","Louise","Leo","Maddy","Maddy","Louise","Louise","Leo","Louise","Leo","Leo","Leo","Leo","Nick","Nick","Louise","Louise","Arta","Maddy","Maddy","Nick","Maddy","Leo","Arta","Nick","Maddy","Maddy","Louise","Louise","Louise","Arta","Leo","Nick","Leo","Louise","Nick","Maddy","Leo","Leo","Leo","Nick","Leo","Leo","Maddy","Josh","Nick","Leo","Nick","Louise","Nick","Louise","Leo","Nick","Arta","Louise"],"help_request":["log2 CPM vs log2 normalized counts","Guidance to add spatial registration to shiny app.","ways to explore domain specific marker genes compared to other domain genes","Resources for processing methylation data","git help: accidentally pushing large file","NIH K08 grant","i have a question about visualization: when i calculate the logcounts matrix of the pseudobulked SRT data, i get some negative values because there are some extremely low counts. under the hood, the function is using edgeR:cpm() by adding an average count of 1 (https://github.com/LieberInstitute/spatialLIBD/blob/35fe720741b0bf44e19475f80b5f3475f347d6b6/R/registration_pseudobulk.R#L163). i think this is fine for DE analysis, but it is hard to interpret when visualizing boxplots of gene expression on the log scale. i tested this out on the DLPFC n=30 SRT data as well, and i also get some negatives when lognormalizing the pseudobulked matrix.\nwhat is the preferred way to visualize this? one option would be to just leave the negatives as is. another option would be to add a slightly larger prior count, like 3, to prevent negatives while preserving the distribution of the data.","How to write an NIH R01","Learning how to work with a PhD student","JHU PhD info","Review scripts containing pseudobulk visium data to compute DEG data.","Team culture overview","formatting data for box plots","PhD vs masters career paths","DE strategies, pseudobulk process and using pseudoBulkDGE()","curl error in SPEAQeasy","PCA plots with a batch effect","Updates on projector datasets","Reviewing code to quantify mRNA expression","Feedback previous to presentation","Cloud computing trial","Donor-level pseudobulk analysis","I'm trying to use the spatialDLPFC_snRNAseq dataset from spatialLIBD. I'm not sure how much QC has been done. I'm also wondering what the UMAP was run on.","Formatting inputs for `spatialLIBD::layer_stat_cor()`","PCA plots interpretation","registration wrapper debug","Spatial registration process","SPEAQeasy preprocessing error","Project goals discussion","Advice for running SPEAQeasy efficiently and estimating memory use","How to build a RSE combining megadepth counts and recount3 colData","How to speed up SPEAQeasy runs","Feedback for her recent presentation in spatial meeting","The qvalue and using the pi1 statistic for replication","Identification of specific marker genes for a cluster in a published dataset","Interpretation of pi1 statistic","Replication rate using qvalue","job_loop questions and run_all scripts advice","Biostats prospective student","Check previous steps to DEG process","SPEAQeasy preprocessing failure","smoking-nicotine-mouse paper reviews","Accessing Gandal et al 2018 files, other BrainSEQ re-analyses","EB2-NIW","JHPCE new PI","EB2-NIW","discuss about writing methods","Issue with several plots in a  PDF file","Setting Paylocity goals","setting goals for the year- to complete paylocity profile","recount2/recount3 base-pair coverage questions","Concurrent task limit at JHPCE","Shiny error without message","Debugging FUSION TWAS","SLURM environment variables","identifying files in MDDseq project","FICTURE Visium HD results","I am doing DEG analysis on DLPFC PNN project, I got CAMK2A as the top upregulated DEG in neuropil+ spots. But when I plot the expression of CAMK2A in neuropil+ vs neuropil- spots as bar plot, the bar in neuropil- category is higher which contradicts the DEG result.","Git pull fails","Help debugging bioc devel (3.21) issues","Help updating a GitHub pages website","Debugging curl installation","Package clashes in conda_R/devel","discuss about qtune results, spatial registration script. completed running add clusters and model psuedobulk","Running SPEQeasy with mixed-species samples","Discuss about next steps spatial registration, Rscripts.","Unexpected prompting of credentials when logging into JHPCE","read10xVisium error","Running BayesSpace, can we go over the preparatory steps","Building SPE and plotting with Visium HD","Debugging R installation in 'r_nac' module","Running SPEAQeasy at JHPCE","Samui troubleshooting continued","Troubleshooting Samui","Adding manual annotation","debug deploy.R","BayesSpace clustering","spaceranger metrics","Using sciwheel and OneDrive","Adding Samui annotations to an SPE object","Merging HALO data","Using git at JHPCE and ignoring files","Letter of support for a grant application","Debugging a GitHub Actions workflow error on a BioC 3.20 ExperimentHub package under development","Reviewing BioC package contribution process","Loading old scikit-learn model","spatial registration, Automatic annotation of spatial registration results, spatialLIBD cluster import()/export()","visiumStitched revision","PRECAST recommendations","Team support overview and introduction","Paper draft structure","Debugging an .Rbuildignore file","AtlaxXomics interpretation results","SpatialExperiment::rotateObject() with stitched data","facing issues with cyberduck - it keeps asking me for the authentication key always","Visium Training Steps III and Visium Training, JHPCE commands and configuration","Addressing NDA submission errors","Project handoff checks and project updates","AtlaxXomics interpretation results","Visium Training Steps II","Cont. PhD application plans","Can you please point me to a table (from Maynard et al. 2021, Huuki-Myers et al. 2024, or other?) that lists the necessary marker genes important for delineating each cortical layer (WM, layer 1-6) in clustered 3’ Visium DLPFC datasets?","Visium Training Steps","smoking-nicotine-mouse paper checks","PhD application plans","Using modules, slurmjobs, and certain partitions","smoking-nicotine-mouse paper checks","Visium overview","Hb_VTA_projectors updates","LIBD internship","using GitHub to sync laptop & JHPCE","spatialLIBD apps deployment review","using GitHub to sync laptop & JHPCE","template_project (GitHub, JHPCE)","Laptop config: RStudio on laptop + local git clone + work on RStudio terminal","Team website self-description","Manisha Training- Laptop config: Cyberduck","GO / KEGG ORA analysis on cell-type specific DEG results","Slack etiquette (questions on channels, avoid DMs, when to tag) and Google Slides weekly update setup","Single-cell genomics webinar Latin America","LIBD rstats club Google Calendar","Team presentations Google Sheet and  How to sign up for a DSgs","Getting an integer vector from factor of integers","Hb_VTA_projectors updates","pseudobulking with Seurat","LIBD cloud computing","Cont. on team options","we find that we have thousands of upregulated DEGs and the volcano plot looks odd. We are following the code in the best practices st book and am not sure if the plot we are getting is just the nature of the data and this brain region, or something about how we are running the DEG testing","Clustering vs sub-clustering results comparison","Trimming options with SPEAQeasy","Adapting scripts to SLURM and using the 'fusion_twas' module","edgeR 4.0 and clusterProfiler questions","Running nnSVG and visualizing PRECAST results","Features from JHPCE storage tracker website","I have questions regarding the leafcutter pipeline, which is used in the MDD project to produce the clustered files.","Random seed usage for PCA/UMAP/tSNE","Smoking-nicotine-mouse paper checks","Smoking-nicotine-mouse paper checks","NDA- submission and authentication","Cell type sub-clustering","InterJad","Review on how to write method sections","Continuation on DEG analysis at the cell cluster level","Viruses and recount2/3","spatialLIBD app results and interpretation for pseudobulk specific bayes space k","single cell markers","DEG analysis at the cell cluster level","LIBD opportunities","I am using spatialLIBD::registration_wrapper() for enrichment DE testing. The dimensions of my spe are 29720 x 77553, but the modeling results only represent 12326 genes. I'm not sure what happens to the remaining genes or if this behavior is intentional (because not all the genes have significant adjusted p-values). Why do all the modeling types result in the same number of genes?","2024 opportunities","SpatialExperiment and other related R objects","Team opportunities","Spatial-Registration workflow. I am particularly confused with the use of the reference, as it is coming from DLFPC","LIBD fundraising","GitHub profile review","Overview of running PRECAST","Overview of using NDA","vis_clus issues","Shiny issue: reading previous manual annotation","Scripts for log normalization and GLM-PCA","Review the habenula snRNA-seq data to assess the enrichment of habenula DEGs for fentanyl administration in cell type-specific marker genes","Getting SPEAQeasy started","Questions about the \"McCarroll lab dlPFC web application\"","10x Genomics reference GTF annotation files","Zotero setup","qsvaR questions","Log normalizing counts in spatial data","Resolving merge conflicts","Career development","Questions about article","deconvolution results (cell type proportions) to be used as covariates for differential expression analysis in Brainseq DLPFC, caudate and hippocampus","Understanding misalignment in Visium stitching","How to check indices used from 10x FASTQ files","Deploying and testing Shiny app locally","DGE analysis of a data subset questions","Basic stuff to run R scripts in the cluster","Filtering lowly expressed features prior for replication analyses","Updating broken Globus endpoints","Trimming error in BiocMAP","Making RNAscope plots using HALO csv files","Reviewing NDA upload scripts","Understanding cleaningY()","Help finding odds ratios for PGC3 SNPs to determine risk allele, and general discussion about plot results and aesthetics","Module-related JHPCE files","Understand how to run a MAGMA analysis for habenula DEGs","We want advice as to *how best* to do the deconvolution","Query about some LIBD bulk RNA-seq and WGBS datasets","Getting BiocMAP running at JHPCE","Impact of CellRanger versions over time on single-exon genes embedded inside introns of longer genes","Running nextflow based pipeline with singularity","SCZD DEG models","Resolving merge conflicts","Reviewing covariates to use in a DE model for snRNA-seq pseudobulked data by cell type with dreamlet","Mean ratio application to HumanPilot data and spatialDLPFC re-analysis","Setting up a website similar to the [spatialDLPFC one](https://research.libd.org/spatialDLPFC/)","WGCNA network selection","Review of SCZD vs NTC DE models","Understanding SPEAQeasy and using correct annotation","Options for creating links to use in manuscripts","Review steps for a new snRNA-seq project","Loupe Browser re-analizing and advanced options","Running SPEAQeasy with conda","Reference letter request","Inspect multimome objects with Browse Loupe","Support with Config Files","Updating ranger_metrics information","Avoid using jsonlite when editing spaceranger json files","Write a script to manually edit spaceranger json files","Debugging spaceranger count","feedback on figures","Understanding a spaceranger error","access bulk RNAseq from samples cataloged in the Lieber-internal bulk rnaseq database","debugging samui script for HPC HE data","Question about the correct number of reads in sequencing for scRNAseq and scATACseq multiome strategy","debugging samui script for HPC HE data","LIBD Data Portal for BSP1 and BSP2 DLPFC SCZD vs NTC data","Cell Ranger-ARC html report","slurm job loops","Polishing details on how to ask for help effectively","GEX and ATAC clustering","More troubleshooting misaligned image","Cg_Hb_projector_profiling progress update","Overview of use cases for JHPCE disk usage data over time","snRNA-seq data analysis steps overview","Cloud computing vs JHPCE","Troubleshooting misaligned image in Samui","Advice prior to a faculty interview","Plotting to a multi-page PDF with 3 plots per page","Derive code for how to merge ERCCsumLogErr values across sequencing lanes for a given sample","Help identifying differences between two RSE objects containing data from the same project","I tried to apply \"qSV_model_DE_analysis.R\" in lithium-exposed samples ( from Dpseq data), but I was confused by some of the code, especially \"run_DE.R.\"","Installing cellprofiler at JHPCE","spatialDLPFC and deconvolution-benchmark project details on how snRNA-seq data was used for each of them","JHPCE login questions","quick big fix. Its the same eQTL code -- just one header in a file that's always missing","Grant related discussion","Adding notes to Samui","Cg_Hb_projector_profiling progress update","Better algorithm for dealing with overlapping images in Samui","Setting up SSH keypair for passwordless login to JHPCE","Vistoseg parameter tuning","Feedback on the nascentRNA project","Deconvolution","How to extract normalized expression values with limma","Prospective JHU PhD student","Cg_Hb_projector_profiling progress update","Cg_Hb_projector_profiling progress update","qsvaR bug report resolution","Adding phenotype data to coldata","Adding packages to a custom R module","Rownames not well formatted","Cg_Hb_projector_profiling progress update","Discuss Seurat, cell type markers, DE","Follow-up: removing large files from version control","Troubleshooting GitHub authentication issues","Removing large files from version control","Working with SLURM","Career development","Running python on cluster","Creating a module","Permutation analysis","making a corrplot with a lot of data points .","Review response to qsvaR bug report from last year","Follow up on stitching for Samui","Maybe divinding them by sample name may help.","when I try to rerun R dreamlet pseudobulk on the new JHPCE, I keep getting errors... (provided code example)","Reviewing analysis using qsvaR package","Reviewing updates to the qsvaR package","Creating stitched image in Samui from ImageJ estimates","Investigating visual problems in Samui image","Help adding phenotype info to colData","Investigating R package installation conflicts","Investigating error in qSVA funtion call","Fix misleading getDegTx error message","Performing 'jaffelab::s' on large tibble efficiently","Follow up on stitching for Samui","Creating stitched image in Samui from ImageJ estimates","JHPCE R modules","Regarding the transcriptome assay from the multiome approach, I have selected canonical gene markers from the different samples; Moving further, I would like to integrate to the analysis external gene-markers shared by S.Page. I am looking the best way to do this.","CV writing advice","Help with qSVA overview schematic","Overview of image stitching process","Cg_Hb_projector_profiling progress update","Review how to respond to a user's bug report","Job application support","I would like to discuss with you the correlation issue related to the cellular_component model of the qsvaR package and get your feedback.","Job application support","Cg_Hb_projector_profiling progress update","Modifying a global-scope variable inside a function","Review replication analysis R script","Cg_Hb_projector_profiling project overview","Review Visium SPG AD blog post","Issues linking repo to github","Interpreting decision tree and spot-deconvo plots","Issues with tensorqtl module","Interaction modeling with limma-voom","snRNA-seq OSCA-based analysis","Review replication analysis R script","Debugging a ranger_metrics R script","Interpretation of \"replication\" plots and performance of QSVA models","Hi! We would love an opinion on our (now cell-type assigned) single nuclei rna-seq data, i.e. how they look - if anything is off, advice for improvement...","Example datasets and getting started with data science","Review replication analysis scripts for the qsvaR project","Help migrating to SLURM setup","snRNA-seq OSCA-based analysis","Nextflow issue when running SPEAQeasy","Setting up and running SPEAQeasy on a new cluster","Installing SPEAQeasy and setting up main script","Troubleshooting a SPEAQeasy error","qsvaR paper planning","Prepare expression bed files for tensorQTL","sbatch issue with SPEAQeasy","Interpreting and refactoring \"concordance\" formulas in QSVA results","snRNA-seq OSCA-based analysis","Prepare covariate input files for tensorQTL","Refactoring QSVA code","snRNA-seq OSCA-based analysis","Interpretation of results plotted in inflection plots of Cellranger-arc data sets (GEX assays). For example, how do you know which genes are representing neurons or glia?","Dropping extra factor levels","Onboarding questions for a snRNA-seq dataset","Confidence metric in spot deconvo model","Array jobs over variables with SLURM","SGE vs. SLURM R package conflicts","help get normalized single cell data from a single cell rse for downstream eQTL","snRNA-seq training","How to interpret terms on an interaction model","Guidance on expression heatmaps","Discuss deconvolution for some future work","help get normalized single cell data from a single cell rse for downstream eQTL","Customizing the home page for spatialLIBD apps","Overview of the stitching process for split Visium samples","snRNA-seq training","Social media ideas","SLURM array jobs and X11","snRNAseq celltype grouping in DLPFC data","Creating documentation sites with bookdown","Displaying info about cells, not spots in Samui","Interpreting spot deconvolution results","I performed the whole analysis as previously established and also made an alternative version based on knowledge of OSCA book for single nuclei (with adaptive thresholds for QC and less tolerance for mito UMIs). Would you help me checking my QC thresholds and seeing if they make sense?","snRNA-seq training","ranger_metrics","Missing genes in Samui","My coding set up","Recommendations for learning more data science skills","Misc spot deconvolution questions","Getting started with snRNA-seq","recount 3 permissions","Using modules at SLURM and configuring bashrc","writing tweets efficiently","Introduction to what we do as a team and future plans","Using Samui from start to finish","Running DeconvoBuddies on the Mathy's data","Adding ceil function to splitSlide to split arrays covering fiducials","Explain the SPE organization and subsetting an SPE","Normalizing spot deconvolution results to answer biological questions of interest","memory error with VistoSeg::InFormStitch function","How to check if a visium slide data is already entered into REDCap","Guidance on reference data for deconvolution and snRNA-seq analysis","insert offset in VistoSeg::splitSlide_IF","Quantifying spatial promiscuity of genes across clusters","Feedback on an R package + GO/KEGG enrichment analyses","Feeback on spatial data + code organization","Feedback on workshop vignette","Evaluate a potential external contractor","How to best document what skills I can perform or need to learn","Need help debugging iSEE app errors/warnings","Practice BioC talk","Troubleshooting GHA workflow errors when building workshop","REDCap build table error","NM pigement segmentation overview for LC samples","Overview of image processing Visium and VSPG","Input on a course design","Feedback on responding to reviewers","Spot deconvolution workflow overview","Loading large files in R","tif2mat conversion","Sharing conda environment","Performing QSVA with genes or with different annotation","mat2tif conversion","0% Vs 5% overlap and repeated signal issues in VSPG images","How to make a project documentation website","clusterProfiler, set.seed, and BiocParallel questions","Resolving a permissions issue","About empty drops run with GEX cellranger-arc data","How to make dynamic error messages","How to make a new Globus endpoint for a publication.","How to filter an SCE object prior to making an iSEE app","How to debug an R package function from a user's report","1/Learn about the roxygen2 syntax for examples. 2/Run devtools::document() like in dev/04_update.R (maybe also auto-style code). 3/Edit the news.md file + bump version on the DESCRIPTION (and change the date). 4/Learn about how to sync her local repo with steps like http://contributions.bioconductor.org/git-version-control.html#new-package-workflow (aka register to https://git.bioconductor.org/BiocCredentials/login/?next=/BiocCredentials/ like @Daianna González + add the upstream remote + git push to both upstream and origin).","I created this github repo: https://github.com/LieberInstitute/neurovasculature-pilot/tree/main. Also, integrated it with slack, but instead of integrating the main branch, I linked the devel branch, and I am unable to see the commits that I make on slack. They are updated on github, though. So, I need help with the integration.","Displaying spot coordinates in Samui","Using the Samui API to display data locally","Options for cell type specific eQTLs","Options for cell type specific eQTLs","Overview of what we do as a team and outreach options","Troubleshooting SPEAQeasy error","Ignoring a directory with gitignore","Debugging rewritten code","Troubleshooting unexpected SPEAQeasy output","Interpreting spot-deconvolution plots","Subsetting RSE by chromosome","SPEAQeasy defaults","Using Samui's API to visualize spatial data","Creating many configuration files differing by one variable","Merging two dataframes","Renaming many files by a pattern","Help with SSH keys with GitHub","Overfitting in qSVA","Interpreting SPEAQeasy metrics","Creating a blog post on the rstats website","Ensembl IDs vs. gene symbols with cell2location","Modules and reproducible software set-up for python","Properly tracking git contributions","When and how to merge git branches","code review","user had problem using import() function in R","Information needed for Samui","Feedback the SmokingMouse Project websites","Properly transferring from dcl01 to dcs05,dcs04","Git-LFS error messages","Alternatives for RNA-seq analysis without replicates","Configuring X11 for plotting at JHPCE","Using the qsvaR with the new annotation","Creating modules at JHPCE","tensorQTL code overview","seurat + signac","Check some issues with GitHub Actions","Introduction to CellRanger RNAseq ATAC analysis","Using git and RStudio to work with R Markdown files","SPEAQeasy error","Understanding API authentication","the user asked how to change the dot size in the plot_pca( )","user had question about generating rarefaction curves in R","tensorQTL pipeline overview","Quantifying a single gene not in SPEAQeasy's default reference","Updating spatialLIBD::read10xVisiumAnalysis","Verify GitHub ssh key setup and team overview","Debugging spatialLIBD::read10xVisiumAnalysis","Overview on \"R/Bioconductor-powered Team Data Science\"","Finalizing genotyping + expression script","Understanding the template project repository","Help resolve JHPCE permission issues on the lifespan_hpc_spatial project","Debug spaceranger 2023.0208.0 (likely an alpha for 2.0.1) with SpatialExperiment 1.8.0","How to write a tweetorial","read10xVisium() in version 1.7.1 (and now 1.8.0) creates a dangerous silent error by mismatching tissue positions across multiple samples","Version controlling renamed directories","Miscellaneous R questions","Deconvolu code where the bisque doesnt show any differences","How to write a tweetorial","Creating marker heatmaps with pseudobulk data","Setting up .bashrc file for folder access permission for DLPFC-SCZ project","Using a gitignore to not version control specific files","I have questions on GitHub and submitting SGE jobs.","Moving forward with multiple cluster methods","Adding figures to OneDrive and google docs with CrossReference","Organizing figure components for Illustrator","Lining up gene expression and genotyping data","Setting up GitHub SSH keys","Managing software dependencies with Nextflow","Nextflow with Conda and Github actions","SPEAQeasy error","GitHub for a beginner","snRNA-seq GLM PCA","Integrating R scripts with Nextflow","Making custom R script work with Netxflow","snRNA-seq QC","Getting started with snRNA-seq","Plotting Gene Expression with SCE","Help identify the source of unexpected NA values","Help building SCE with downloaded data","Adapting spot-deconvolution code to new data","Discuss Mean Ratio marker gene finding","Deconvolution study with rnaseq bulk data","resolving a merge conflict","Identifying the source of an error","coloring plots by a continuous variable","Understanding PCA for Habenula project","Review deconvolition script for qSVA","wrangling and reformatting expression data","preprocessing and reformatting genotype and expression data","organizing plots and output files","understanding linear mixed-effects models","help submitting jobs with R parallelization in jhpce. For instance, I would like to use scater::runPCA with the BPPARAM=BiocParallel::MulticoreParam(8)) parameter","Introduction to LIBD and our team","formatting genotype data into a matrix for each gene","I have this single nucleus RNA sequencing experiment with the mouse LS that Matt analyzed and we have a final single cell experiment, but I don't know how to do anything with this data set and want to learn how to plot heatmaps and possibly other things for this data set. Can we maybe schedule an hour to start this learning process and then I can schedule future time as needed?","Check the DEG heatmaps","Review TMM paper","csvs that I want to merge together","Lifting results between hg19 and hg38","GO and KEGG boxplots with the modifications you suggested","QC approach for hebenula","How to check technical effects for RNAseq data","Reviewing GO results and dotplots","Working on making a violin plot for just one gene. So by brain region rather than gene type","Question on the feasibility of some approach with HALO (for the AD project)","Overview on using GitHub for Matlab projects","Setting up SSH keys between login and compute nodes","Setting up SSH keys for JHPCE","Introduction to DSgs, R, Bioconductor, JHPCE","Interaction between .bashrc and 'module load' in 'qsub'd scripts","Modifying a shell script to segment a VCF by genomic region","Setting up SPEAQeasy at JHPCE","Code to threshold small sized nuclei from the NaCC visium images","How to organize OneDrive files","How can I interpret the y-axis from `plotExplanatoryVariables()`?","Use of git from multiple local locations","Moving and correctly accessing HDF5-backed SummarizedExperiments","DE analysis introduction","Git push help","check the models and the results I have obtained so far from the DEA","DE analysis introduction","Check the model design of the smoking mouse project","VNS error","Strategies to pick threshold for cell size in segmentations","open multichannel image in imagej","code to ectract x and y coordinates of randomnly slected inform tiles for PNN project","Review results of Varaince Parition for smoking mouse","Pick randomn tiles for the PNN project","go over pseudobulk code","Help with spatialLIBD colors setup","Extracting final output into table","CUDA issues with cellpose module","QC update for smoking mouse","R, RStudio, git, SSH key setup","Addressing SPEAQeasy error messages","LIBD Data Science","help row annotation colors of a pheatmap","Subsetting eQTL results with in risk loci","Setting up SPEAQeasy","Data science coordination","QC update for smoking mouse","Question about how 'mitoRate' is calculated in SPEAQeasy","Help troubleshooting error messge when running BayesSpace","zellkonverter error and guidance using SpaGCN","Review progress on smoking mouse DE project","pulling in external changes to a git repository","Adjusting thresholds for event detection using CaPTure","help installing 'jaffelab'","help with vis_grid_clus()","JHPCE Github setup","Help troubleshooting shiny app that crashes on gene interactive tab","getting total UMI's for each sample","Github setup","Revisited the sc analysis and could use some help","explain spaceranger output files","Tools and resources related to recount2 and recount3","I might need a review of how to go about selecting marker genes for the spatialDLPFC data. Also, I'm curious what code you have for making heatmaps showing marker gene expression across cell types.","Single cell rna seq analysis","I have a mouse single-cell RNAseq data that I am aiming to identify cell types in different clusters.","Debug `spatialLIBD::sig_genes_extract_all()`.","How to debug an R package code","Help with BayesSpace clustering","Data science organization.","MatLab questions.","Subsetting spe object for shiny app","Brainomics meeting with collaborators","Color bar for heatmap","Discussion on how to recruit members","VisiumIF DLPFC building spe","OneDrive","BrainSeq Phase II development browser","make modified tissue_positions_list from DLPFC IF samples for running spotsopcheck on mock PNN visium sample","Help Deploying Shiny App for DLPFC_IF","Cross Reference help","Career development","Need some help to make a plot","Guide to creating a module at JHPCE","How to make a tweetorial for a pre-print","File organization for a paper submission","creating a nextflow pipeline and supporting docker","In promptu meeting","Help with purrr loops for WGCNA funtions","Grant proposal feedback","help with html output of spaceranger","CV feedback","Cover letter feedback","Debugged code from https://github.com/LieberInstitute/spatialLIBD/issues/33","Brainomics meeting with collaborators","How to delete large files from GitHub history","help with read10xVisiumWrapper()","macOS and JHPCE laptop set","writing papers in google docs and setting up links for figures","Hey Maddy, I need something called \"spot size\"  on the 12 published DLPFC samples","https://github.com/LieberInstitute/VistoSeg/issues/12","applying size threshold to vistoseg outputs to get rid of smaller regions that are not nuclei","Debugging JHPCE configuration files","I'd like to consult with Louise about how to make a spatial plot in R using a data set from the HALO FISH-IF module.","directed the person to the VistoSeg website for steps to run VistoSeg","using toptable and duplicate correlation","help with clustering portion of th build_spe","Using the \"redstone\" software","help with build_spe","Input files for custom annotation with SPEAQeasy","Advice on how to increase code coverate with testthat","Help run trial shiny app deployment","set up git repo for long read project","The authenticity of host 'github.com'","Assistance with analyzing spatial data","Follow-up on custom annotation with SPEAQeasy","Plan how to make spatial registration R functions","Adding lines to a GTF for use in SPEAQeasy with custom annotation","Clarifying what role deconvolution can play on LCM-seq","TODO","Error message running SPEAQeasy","How to log in to JHU-controlled laptop","Setting up passwordless login","VisiumIF loupe alignment","Using custom annotation with SPEAQeasy","Masking of blood vessels from PNN channel in PNN project","How are InForm tiles arranged","Using the caracol queue","How to guide others learning DS","Understand the covariates used in scRNA-seq cell type analysis for a project we are doing with striatal organoids","\"Megadepth usage to add a new annotation to hg38 file\"","How to build a cov_rse object using qSVA v1","Debradation matrix for LIBD riboZ","Work through qsvaR vignette together","Generating bigWigs with SPEAQeasy","Data type change of image data when loading into Python","WGBS data analysis overview","Help building for loops to analyze WGCNA ouput data","I would like to learn how you ran tensorQTL on the MDD data, maybe with using the GPU nodes for it.","contrast adjustment, intensity normalization, channel/gene description in PNN images","Ideas for possible future analyses","rename files on JHPCE through command line","JHPCE array jobs and fasthplus","Provide comparison table for different metrics obtained from image meta data in MATLAB, for Visium IF workflow","snRNA-seq and deconvolution overview","Access to the gene info in the rowData of the Summarized Experiment objects to annotate eqtl results","Explain Tangram, scRNAseq count matrix, spe object","Explain the concept of Squidpy paper","tiling and metadata changes in PNN project","explaination on Image metadata, bit depth","Need for rotation of visium array images and its spatial coordinnnate in spe object","Debugging seg fault with tensorqtl module","Using tensorqtl with a GPU","X11 and SSH issues","VistoSegIF pipeline","accessing the 'gpu' node at JHPCE correctly","Transfer data from JHPCE to Neural_Plasticity","Transfer data Neural_Plasticity to JHPCE","JHPCE directory organization, repo linking, making commits","opening python, R, MATLAB console interactive session on JHPCE","Troubleshooting ExprssionSet conversion","glitch in CAPTure tutorial about input data types to a function","Help using BisqueRNA, getting snRNA-seq into correct input format, chosing refrence vs. marker based deconvolution","creating a 'samples.manifest' and choosing trimming settings","Removing smaller objects from VistoSeg segmentations","applying kmeans to CA3 mouse RNAscope data to obtain gene expression levels","JHPCE file permissions and ACLs","running loupe browser for HPC images","countSpots from VistoSeg throwing errors for one DLPFC sample from round4","Resolve issues with comments","using 'list.files' in R","batch processing - dotdotdot, on local machine","error in transferring Visuim HPC data to JHPCE","using awk","Debugging help with rse_tx object","qc general scripts, rnaseq scripts general, ggplot histogram with breaks","We would love an opinion on our (now cell-type assigned) single nuclei rna-seq data, i.e. how they look - if anything is off, advice for improvement...","I am trying to compute DEGs from thefunction registration_stats_enrichment... \nError in limma::makeContrasts(contrasts = z, levels = registration_model) :\nThe levels must by syntactically valid names in R","Transfering files between JHPCE disks","running spotspotcheck for SFN","doubts on VistoSeg","help with git","debugging SPEAQeasy errors","Presentation preparation feedback","help with 'umask' and git","issues with regioprops function in dotdotdot","Fiducial frame enhancement for Visium IF images using preview on MAC","running Loupe browser for mannual alignment","merging FASTQ files automatically via SPEAQeasy","Creating cyberduck bookmarks for JHPCE paths","writing shell script for countNuclei function in VistoSeg","Explaining Github issues, history and version control","Explaining SPE (Spatial Experiment) SCE (Single Cell Experiment) objects","syntax error in shell script","CellProfiler H&E segmentations","Trouble with implementing RNAscope function for humans","Discussion about rerunning FUSION/TWAS with new GWAS with extra attention paid to the number of samples for each brain region","identifying SPEAQeasy error","Error running VistoSeg::spospotcheck (Error in spotspotcheck>start (line 60))","Error while running rnascope_mouse - line 159","parameter tuning for dotdotdot to split the nuclei clusters","improve lipofuscin masking in dotdotdot","dynamically name structures in matlab based on user inputs","using map to cbind a list of six to its corresponding list of lists. The second is a list of 6, each a list of 3.","resuming SPEAQeasy after an issue in the counts step","downloading FASTQ files from SRA","Explain how to merge 2 data tables in R","Explain public keys and the authorized_keys file on JHPCE, removed duplicate public keys for the local machine to resolve ssh key login","Using purrr and map to loop over datasets and apply qsva","resolving issues with multiple keys in the authorized_keys file","Help with JHPCE login issues","explanation of the methods and algorithm of a paper we are reviewing together","steps to resolve image segmentation of LC heated samples round 2","How to write a Twitter thread for a paper","Forming a data frame from data in multiple files","Refining an abstract submission","Git branches","DNA genotyping overview","DNA genotyping overview","Optimizing DAPI segmentation in dotdotdot","Overview of the RNA-seq universe","Explain how awk works","Issues with SUFI","Resolve errors produced by dotdotdot on SUFI output","R Code to create list of files to be processed through VisiumLIBD","Setup aliases in bashrc","debugging SPEAQeasy errors","getting SPEAQeasy running","formatting a LaTeX document and bookdown website","Setting up a team at LIBD","recreating spatstat plots with more features","Helping understand errors from VisiumLIBD","Helping understand errors from VisiumLIBD","Help indexing a possibly corrupt VCF file","Running Genome Studio on n = 200, generating PLINK outputs","How to set up a GitHub actions-powered website","Specifying output logs on SGE","Debugging ggplot code","Planning session for getting up to speed with SpatialExperiment, spatialLIBD, here and ggplot2","How to choose and add a licence to a GitHub repo","Designing an MS-level class","making a list of filenames in a specific folder and subfolder in R","Rectified issues for batch running emma's RNAscope data","create shell script for batch running images on VisiumLIBD","Locating public data at JHPCE + bamtofastq workflow","TWAS questions","How to transfer recount3 data","How to get started with R","Brainstorming session on collaboration","Manhattan plot making and tstatistic","addressing SPEAQeasy error","got mds score for the vcf she presented using plink","FUSION / TWAS Walkthrough","mds for vcfs","checking plots and data structure in clusterprofiler","Havent got my mds yet so making the meeting for Friday","questions about expression_cut and pca plots","Using GenomeStudio","How to prepare for a LIBD seminar part 4","troubleshoot SUFI output and figured it was not compatible with dotdotdot","gene ontology for genesets","Differential Expression help (continuation of 5/20)","How to prepare for a LIBD seminar part 3","Grant application review","PredictDB weights","Help preparing a figure for a grant application","How to prepare for a LIBD seminar part 2","Making plots for DE studies","Making a module","Configuring a parallel series of bash scripts","questions about processing genotype data","Setting up a SPEAQeasy run","Making plots for DE studies","Setting up covr using GitHub actions in an R package","How to get started with computational biology","How to insert a figure using the Oxford Bioinfomratics Overleaf LaTeX template","PredictDB weights","How to prepare for a LIBD seminar","Running SPEAQeasy- follow up","TWAS prep","Constructing an array job from an existing sequential shell script","Setting up FUSION / TWAS for practice","Running SPEAQeasy- follow up","Running SPEAQeasy","Running SPEAQeasy- follow up","Running FUSION / TWAS","JHPCE set up on mac","Running SPEAQeasy","running and trouble shooting VisiumIF image splitting code","JHPCE set up on mac","Running the old RNA-seq pipeline on some SRA data","Resolving issues with running dotdotdot on 20X RNAscope images","runnning Hiplex registration","General guidance on how to learn how to do differential expression analyses","Nextflow overview for work with imputation pipeline","creating logs for issues from running SUFI (spectral unmixing of fluorescent biological images)","steps to troubleshoot issues with stitched multi tile RNAscope images","spatialLIBD support","Need help understanding the sce - single cell object","help doing a power analysis for RNA-seq data","Creating a documentation website using bookdown","Clarification of LIBD support","General advice","running nucleiCount for spatial DLPFC samples","Download SRA data and run SPEAQeasy","Explain the feasibility of an on-demand end user interface for DEG and eQTLs","extracting p-values from linear mixed effect model for Calcium imaging data","Using an LIBD module","Change point size in vis_clust_grid","Download SRA data and run SPEAQeasy","Attempted SPEAQeasy installation on MacOS","visium image segmentation","troubleshooting memory usage for VNS function from VisiumLIBD","get effect size for calcium imaging","split histology images into respective capture areas","x-quartz installation","BSP2 eQTLs with a subset of SNPs","qSVA explanation and code review","How to write a small methods paper","resolve figure generation error in human rnascope script from dotdotdot","Maker finding for cell types in single cell data","image analysis on QuPath","Advice for writing papers","Clarification about how recent CMC data was processed with SPEAQeasy","Our group's set-up and approach for writing papers with google docs, and managing citations","data transfer from LIBD server to JHPCE","Submitting/debugging bash job for dotdotdot on cluster","I need help in: downloading data from SRA to JHPCE, running SPEAQeasy to create rse object (one session will likely not be enough).","I need help with setting up Mobaexterm. Unable to open new terminals and folder permission issues","explaination of Owner/Group/Other permissions for files on JHPCe","download repository from github to cluster and run dotdotdot on cluster","Visualizing RPKM valus for genes in Humna and Primarte data","BSP2 LIBD eQTLs request","JHPCE ssh keypair set up","dotdotdot parameter tuning for Ema's rat RNAscope data","Github ssh key set up","Work on habenula deconvolution and debug the deconvolution package","Help determining what data is available and what can be used for an analysis","group_by bug for get_mean_ratio","Guidance on how to download data from a study in GEO","Career development","biocthis macOS GHA bug","Questions related to accessing results from the DG paper by Andrew Jaffe","Correlation values from labeledHeatmap","Help with importing HDF5 file","Resolving issues in phenotype data between genotype, expression, and LIBDpheno","jhpce set up on mac","Inline formula throwing an error for interaction term","run dotdotdot pipeline local computer and cluster","how to open matlab (GUI and command line) on JHPCE","Matching Variants Between Two VCF Objects","how to connect to LIBD servers on windows and mac","Accesing CMC RNA-seq data","Ran Genome Studio, installed PLINK input plugin","ChipQC 'names' attribute [9] must be the same length as the vector [2]'","data transfer from LIBD server to JHPCE","download matlab on local computer through JHU software portal","Help clustering sce data","Using reshape package to transform table for plotting","Help adapting ggplot script for TWAS Z-score","RDavid Error","Differences between recount2's \"bp_length\" and SPEAQeasy's \"Length\"","How does one modify a single value of a field in a seqinfo object?","GitHub actions workflow issue with macOS on R-devel","LC repo organization","Extracting Sequences from Fasta Files with Rsamtools","3D volume extraction in Matlab","Fixed two bugs on recount3 related to locate_url() for the BigWig files.","Fixed a bug on biocthis for release branches","Issue with knitcitations","Help with SpaceRanger","Issue with JHPCE file permissions","recount3 GTEx data access","customize dotdotdot to his data","Learning Lexical Scoping Rules","Changing Row Names of MultiAssayExperiment Assay","exceRpt pipeline counts","issue running spaceranger","Plotly help if possible? I'm trying to export an html file that allows the user to look at each snp/gene on the plot, and I am almost successful. Leo recommended that I talk to you on this matter.","Help running speakeasy on MARCC and JHPCE please","Question: Subset center regions of a set of different length sequences","Ballgown stattest","Question: Cannot Install gmapR Bioconductor Package","copy number variation analysis (CNV)","Help with spatstat"],"public_notes":["We worked on <https://github.com/LieberInstitute/spatialLIBD/issues/106> and eventually resolved it thanks to answers from Gordon Smyth and James W. MacDonald at <https://support.bioconductor.org/p/9161754>. See the GitHub issue thread for more details about the differences between `edgeR::cpm(log = TRUE)` and `scuttle::logNormCounts()`. The documentation at `spatialLIBD::registration_pseudobulk()` has been expanded to further explain this difference.","spatialLIBD::app_server seems to be breaking for layer_stat_cor_plot when retriving colors, this looked more like an issues with the spatialLIBD code than the users application. Will investagate further","Researcher is Intrested in definind what makes a spatial domain unique from the others. I recomended to review the top enrichemnt and parwise genes from the modeling results. Running MeanRatio may also be informative. I also recomended reviewing litature to see if there is any other work that may help caracterize this domain.","I pointed to the public rstats google sheet in case there were helpful sessions, then recommended BiocMAP as our pipeline for methylation data","Since the commit with the large file was recent, I recommended using `git log` to identify when the commit occurred, then use `git reset --soft` to undo commits (without modifying files) after that point","We discussed how I can help on a K08 grant application, the support structure my team provides via the DSgs system, and I shared a draft support letter from another grant application.","The negative values are probably 0s in the counts matrix, I agreed that the negative values in vaisualizations are tricky to expain, it would be more intutaive if they were also zeros. I am not sure if changeing the prior.count to > 1 would help. I suggested plotting the pseudobulk counts as a sanity check and exploring the count values that end up as negtive in logcounts. Researcher may leave an issue on spatialLIBD about this.","We talked about the process of writing an NIH R01 grant, how to organize some grant related files, options for how to collaborate and the related budget implications.","We talked about how to train a JHU PhD student, how to organize learning tasks, and updates on a mouse snRNA-seq project.","We talked about applying to JHU PhD programs at BSPH, how to best communicate via email with busy faculty (short emails asking specific questions), and other aspects of studying in Baltimore.","We debugged code for pseudobulkDGE, we found several silent errors that were leading to empty output, but not errors. After correcting some of the variables we got closer to expected output, but still need to further debug. I think the function could be improved, prehaps we'll create a github issue","We talked about LIBD, the team history, team culture, how we are organized, how we train people, and other related topics.","For a specific set of boxplots, I described how grouping and summarization should work in order to produce the expected number of points at each x value. I also emphasized keeping data in long format for plotting","We discussed some career options and what math background classes would be useful to take before applying to a PhD in biostatistics.","We discussed the DE analysis steps, focusing on pseudobulking. I explained the overall workflow I am using, including the use of pseudoBulkDGE(), the covariates to consider, and how to set coefficient in the design matrix for analysis.","Since we couldn't directly access the SPEAQeasy configuration file (it was in another user's home directory), I recommended reinstalling SPEAQeasy in a new location. The error was indeed due to being unable to read configuration variables","We talked about results from `multiBatchPCA()`, and ultimately preferredd the results from computing PCs in each batch separately, then we interpreted the plots to evaluate whether biological or technical signal dominated PC1 and PC2 within each batch.","We talked about new data being generated, the questions we have of interest, and the overall timeline.","Feedback on how to document code for working with HALO/RNAScope data. From example scripts I think it would be best to build an R package. I provided some examples of what that looks like. They are also thinking a shiny element would be good, that adds additional, but managable, complexity.","I provided feedback on both technical and aesthetic aspects of the presentation. On the technical side, I pointed out areas where additional context might help fill potential gaps. Aesthetically, I suggested  enhance clarity, such as adding axis labels to plots, maintaining consistent terminology across slides, and including descriptive titles or annotations to make visual elements more self-explanatory.","We discussed ideas for how to proceed with a trial.","We went over how to use `pseudoBulkDGE()` for performing a DGE analysis at the donor level, and what covariates we would need to compute.","We reviewed the key details for the spatialDLPFC snRNA-seq data. Key colums are cellType_layer, and cellType_broad_hc, also dropping sce$cellType_hc == \"Ambiguous\". I noted our documentation for fetch_data() could benifit from more details about this dataset.","I described how one key input needs a dataframe and the other needs a list of dataframes. I showed how to loop over various reference datasets with `lapply`","I provided a brief explaination of PCA used to visualize variance patterns such as clustering by cell type, condition, or batch. I highlighed diferent situation, for example, points close together represent similar profiles, while separation along PC1 or PC2 often indicates major sources of variation; also explained that  coloring by metadata (like sample ID or treatment) helps identify biological or technical drivers, that could be rare populations or quality issues.","we worked on filling the rowData of a sce object after converting from seurat, as the gene ids are key for running layer_stat_cor later in the spatial registration process","I described the general concept of spatial registration, pointed to `spatialLIBD::registration_wrapper()` as a key function, then helped prepare input data and parameters for running that function, starting with a Seurat object. This seemed to involve manually adding `rowData()` columns to the converted SingleCellExperiment.","They noted the exit code of 137, which I agreed (often) indicated a job being killed due to exceeding  memory limits. I suggested doubling or greatly increasing the memory for the `PreprocessInputs` process in `conf/jhpce.config`.","We refreshed our memory on where the project stands and what would be some of the next steps that need to be performed.","I noted that as a Nextflow pipeline, parallelism is built in to the design of SPEAQeasy; processes will execute as soon as they have input data ready, and many SLURM jobs execute at once. I noted that most processes shouldn't need memory modifications for large datasets, since they usually operate on one sample at a time, but I mentioned the `CountObjects` process as a possible exception and noted it could be edited in `conf/jhpce.config`.","I showed how you can obtain the sample information (colData) from an RSE gene-level `recount3` object, then by sample ID, match with the colData you have on a small RSE made manually, and replace the colData info. The small RSE has counts computed with `megadepth`. This could end up being a `recount3` or `megadepth` helper function. In any case, having the full `recount3` colData enables then the use of functions such as `transform_counts()` to convert the `recount3` / `megadepth` base-pair level raw counts into read counts for downstream analyses.","I mentioned that one option involves contacting your HPC admins (JHPCE BitSupport for us) and asking for an increased allocation of computing cores. That way you can run more jobs in parallel. Though ultimately it's best to meet with Nick Eagles for a DSgs as Nick is the author and maintainer of SPEAQeasy.","I explained the Harmony plots before and after batch correction, highlighting differences between the Harmony pure plot and Harmony UMAP plot. We also discuss quick tests to assess whether low statistical power was affecting DEG identification. I suggested applying a power function (as in WGCNA) and duplicating samples to evaluate dataset limitations, while also discussing the constraints of these approaches.","We reviewed the concepts in a blog post about the meaning of different p-value distributions in multiple-testing scenarios, and I explained how p-value distributions connect to the pi1 statistic as used to estimate replication rate","We discussed the links at https://www.one-tab.com/page/oKFdo7_aRT-3ffoVddJ0sQ. I suggested using `DeconvoBuddies::get_mean_ratio()` on the public data available from GEO. Then checking if that cluster should be merged with another one, and if so, then repeating the Mean Ratio analysis.","I looked over some code and clarified that the pi1 statistic from the qvalue package can be used to estimate replication (the proportion of DE genes from one dataset expected to be DE in another)","We reviewed code for using `qvalue::qvalue()` to assesss replication rate of a set of DEGs using p-values from a replication dataset. This used code written by Gabriel Hoffman.","I described the meaning of the syntax used to redirect output to a custom log name as internally created by `slurmjobs::job_loop()`. I described the `-f` parameter to `rm` for removing output files in a `run_all.sh` script without failing if they don't exist","Discussed our work, life in Baltimore, and other aspects related to choosing a lab to work for during a PhD.","We reviewed the steps leading up to DEG identification, as she wanted to confirm the correctness of her process. I shared a couple of tests I typically run to verify range and summarize various metrics, enabling her to check if her target thresholds are present in the matrix data.","I investigated the source of the issue and recommended checking for typo in samples.manifest, which ended up being the issue","We discussed a strategy for responding to reviewers.","I showed how to access the no qSV results from the `outGene0` object listed at https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/casectrl_DLPFC.R#L43 that is part of https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/casectrl_DLPFC.R#L59. I also suggested that they could use the BSP1 data aligned with HISAT2 as part of the BSP2 paper by re-runnign the code at https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase1_qsv/casectrl_DLPFC_allFeatures.R#L43 and basically not adding the qSVs if they don't want them. I also showed them the BSP3 code from the habenulaPilot paper at https://github.com/LieberInstitute/Habenula_Pilot/blob/99b5d8b0d5486b475d8b2e21920e9ddb1b71ffc4/code/15_compare_vs_BrainSeq/01_Hb_vs_BSP2_BSP3_DG.R#L32.","NA","We got him setup at JHU Genomics Collective's Slack workspace, talked about DSgs system, LIBD rstats club, and then also JHPCE new PI form. We contacted Francesca and John for more details needed for the new PI form.","NA","Demoed how to use sciwheel and cross ref in google docs for witing paper","We found that par does not interact well with patchwork plot objects. To arrange a list of patchwork objects in to 4 rows use Reduce(\"/\", plot_list)","I mentioned specific goals I made and the types/ categories of goals that could be appropriate (specific progress towards a manuscript, specific presention-related goals, etc)","We discussed strategies set qunatiative goals, such as number of presentations or breaking large projects in to individual, completable steps","Check that the input coordinates are in hg38 and not in hg19 positions. Use `disjoin()` on the exons of a given gene to find the disjoint exon pieces, quantify those, then add them up to get the gene expression. Do this instead of quantifying start to end (aka introns might be noisy).","I noted the 100 CPU and 1TB RAM limit per user, though noted the number of array tasks for a user can be quite high","I guessed that the issue was a memory limit and we confirmed by loading the object in R and using `gc()` (`lobstr::obj_size()` would've been more precise). I recommended using HDF5-backed assays","See links from today's session [here](https://www.one-tab.com/page/UkWtmL03STGtP2Y1Gojdig). To debug the issue at hand, I advised manually running https://github.com/LieberInstitute/fusion_twas/blob/jhpce/FUSION.compute_weights.R for one input gene set of files.","I corrected a few environment variables in a partially working script to sbatch and noted the '#SBATCH --array=' syntax for job arrays","We revieded the project history and  orgainzation to help find needed scripts","I shared FICTURE results for the LC dataset and we discussed","Tools like scater::ploExpression or DeconvoBuddies::plotGeneExpress() will help plot gene expressio vs. catagorey. jafflab::cleaningY() will regress out varaibles controled for in DEG and provide the best representation of what the \"model\" sees. We could consider adding the \"clean\" expression as an assay durring the \"modeling\" pipeline from spatialLIBD.","debugging git pull with conflicts","See https://community-bioc.slack.com/archives/CLUJWDQF4/p1733847223678279?thread_ts=1733845668.397749&cid=CLUJWDQF4 for more information. Basically, the `bib` object was not well made and this caused a \"subscript out of bounds\" error on R devel (4.5) and bioc devel (3.21) that wasn't an error before.","I helped update https://github.com/LieberInstitute/spatial_hpc/tree/gh-pages. Note that the html file has a base64 representation of the image, so commit 1 is to update the image, commit 2 is to re-render the html file (once the new image image is available). Also, the image wasn't been displayed at https://github.com/LieberInstitute/spatial_hpc since it was too big. So I resized it so the max dimension would be 1,200 pixels (in this case, the width).","Ultimately this person needs to update their macOS computer. They had avoided doing so for too long. See also https://github.com/jeroen/curl/issues/358 for more details from someone with a nearly identical issue.","She ran into a large amount of package installation and loading failures while using the conda_R/devel module. I noted that while the R version periodically updates for this module, the user directory for packages stays the same. leading to conflicts. I noted deleting the user-specific directories was one failproof but extreme option","Review results from q-tune, looks like it might not align with the desired resolution for the data. Review output from modelining analysis.","I suggested using '--strand_mode \"declare\"' to cirumvent possible issues with poor transcriptome pseudoalignment, and noted HISAT2 can be configured to penalize alignment mismatches more aggressively with the '--mp' parameter","Set up dataset for spatial registration, review code, plan for how to interpret output","I noted that somehow the ~/.ssh directory had overly open permissions, leading to issues with git and SSHing into the cluster. We ran 'chmod -R 700 ~/.ssh', then later discovered the home directory as a whole needed a similar permissions fix","The error hinted at a clash of gene names, which I traced back to having multiple samples using different spaceranger reference files","We discussed the quality control, feature seelction, and batch corrections steps needed before clustering visium data","I mentioned tricks I used to build a SpatialExperiment for Visium HD data and hacks for using 'vis_gene' and similar","I suggested a 'conda install' command that might fix a missing system dependency seen when trying to install a partcular R package. I also noted that python dependencies could be installed in the same module (conda environment) if desired","I described the installation process for SPEAQeasy and typical command-line parameters one might typically change","We discovered one source of odd behavior stemming from not fully deleting the directory for a given Samui sample before reconstructing it","I walked through Python code to check when building a Samui sample to properly align coordinates and the image, while showing gene expression","I explained to him from scratch how to make a manual annotation using his own SpatialLIBD application. I explained to him how to select slides, select spots with lasso or one to one, and how to import a csv file for further analysis. I also explained to him the conceptual use of the annotation, how he can change the names of the labels and share the file with other people to work collaboratively.","We comapred the app.R and deploy.R scripts to working ones, corrected some issues with the here function. This did not succesfully deploy the app (local version works). Likewise we matched the settings to the ERC spp settings, still failed to run. We may need a second set of eyes to help finetune the shiny.io settings or diagnose the issue here.","We discussed the steps needed to prepare for spatial clustering: quality control, normalization, feature selection, demension reduction, batch correction THEN clustering w/ BayesSpace. We went over the clustering code from LFF ERC","I explained to him where he can find the results summary report generated for Spaceranger. We focused mainly on the CSV file as it is more detailed. I explained to him the specific metrics he was interested in and where he can find on the 10x website the technical notes for 10x Genomics products for further questions.","I described how we put any component of each manuscript figure in a directory to be stored on OneDrive (often consisting of individual plots in PDF form), and described how Sciwheel integrates with a google document","I referenced https://github.com/LieberInstitute/spatialdACC/blob/main/code/09_samui/09_find_unannotated.R and described that the 'id' column in the Samui CSV refers to the index of the 'coords.csv' file for the Samui sample.","Discussed tips for working with HALO data, and how to visualize","I described how .gitignore files work and mentioned the best practice of putting them in the directory where files to ignore are present, not at the top level","We discussed their grant and how my letter of support should be framed. I sent the letter at the end of the day.","1. Check with `lobstr::obj_size()` how big your objects are. They might not be compatible with [memory limits provided by free GHA workers](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners) (GitHub Actions). 2. `biocthis` 's GHA workflow tries to emulate the Biocondutor SPB (Single Package Builder). People can submit packages to BioC without having tested them on GHA, though then they might run into R CMD build / check / BiocCheck issues for the first time there and have a slower peer review process. 3. Consider using `rm()` once you are done with the first large object. 4. In general, it's best to break long code chunks into smaller ones (and name them). That way, you can get more specific locations of the errors. Hence why we added that `stopifnot()` from my PR to formally check that the snapshot date is valid. 5. For debugging, it's useful to have a local environment where you can reproduce in detail the issues. I didn't have one ready before the DSgs for BioC 3.20, but if we meet again, I could be ready.","We went over the process of synching a GitHub package with the Bioconductor git server, what is the Bioconductor Single Package Builder (SBP), when we can tell that the SBP has been configured (and is thus ready to access new versions via bumps in the DESCRIPTION), and how biocthis's GHA simulates part of what the SBP does but how they still have some differences. Ultimately in this case, GHA doesn't have a timeout limit (15 min) that SBP does have, which is why SBP is erroring out while GHA is ok. We also ran into an issue with the development version of pkgdown.","I noted that you must in general load scikit-learn models in using the same scikit-learn version they were trained in, and mentioned the cellpose 2.2 module in particular for an old model I trained.","Reviewed the spatial registration pipeline: pseudobulking, modeling, correlation, and annotation","We went over the response to the reviewers (from the journal) and clarified how to address some of the comments.","I pointed to https://github.com/LieberInstitute/visiumStitched_brain/blob/devel/code/03_stitching/07_precast_unstitched.R and mentioned that some evidence from stitching analysis (in particular, https://github.com/LieberInstitute/visiumStitched_brain/blob/devel/plots/03_stitching/agreement_at_overlaps.pdf) suggested SVGs were slightly favorable to HVGs as input","I provided an overview of what the team does, how we can help the OP, added them to Slack, the LIBD rstats club, and other intro first steps.","We reviewed https://lcolladotor.github.io/bioc_team_ds/writing-papers.html and started a Google Doc, SciWheel, LIBD OneDrive for this paper.","I helped debug an issue with a new R package. Basically, some files were not being included in the .Rbuildignore that should have been.","I explained in more detail the FRiP score and TSS quality control metrics for AtlasXomics sequenced data for the pilot samples provided by Lieber for Hb and DLPFC. We compared the recommended thresholds with those obtained in the pilot samples. I emphasized that the FRiP should be greater than 0.3, but values ​​greater than 0.2 are also acceptable.","She encountered an open bug with this function that I described. I noted a workaround was to rotate in increments (e.g. 90 degrees 3 times instead of 270 degress once).","Trouble shot cyberduck so login works automatically with ssh keys","Using as an example our https://libd.shinyapps.io/Habenula_Visium_raw/  app, I conitnued explaining her how to explore data from discrete variables, for example the 10x graph-clusters and PCs. I also explained her how explore a particular cluster of interest and a glimse of how to check the gene expresion using multiple features (multi-gene function).","I recommended using a GUID-mapping table I generated in the spatialDLPFC dataset, since the issue was related to subject-GUID mapping.","We discussed the general setup we will move ahead with now that we have to wait a few months before new data is generated to complete this study. We brainstormed options moving forward on how to best collaborate.","I helped her understand the results of the AtlasXomics pilot review for the samples provided by Lieber for Hb and DLPFC. I explained the basic concepts and quality controls expected according to the ENCODE standard, however not all of them apply to this type of data. I also pointed out that the sequenced samples were bulk ATAC and could be another metric to consider outside my field of expertise.","I explained her the main layout and variables included on the spatialLIBD shiny app, Using as an example our https://libd.shinyapps.io/Habenula_Visium_raw/   app I showed her many examples using continuos and discrete variables. As well I provided her a glimse of the 10x graph-clustering  wrapped in the app.","Ibidem","Pointed to the different ways we have to access modeling data from the fiffernt DLPFC studies","I explained her main steps to build a spatialLIBD shiny app with raw data. I quickly provided her an overview of spaceranger, and then I explained her how to build a basic spe object and the settings of the ShinnyApps on the cloud. We use as example  our https://libd.shinyapps.io/Habenula_Visium_raw/   app.","We reviewed responses from the OP to recent comments by a co-author of this paper.","We reviewed some information about applying for PhD programs: how to decide, application strategy in general, information that is best to provide for those writing letters, and other related info.","I demonstrated to use of modules for command-line software, R, and python. I pointed to slurmjobs vignettes and the 'job_single()' function in particular. I noted katun or caracol are reserved partitions that can be accessed with '-p katun' when using srun or sbatch.","We reviewed recent changes made by OP to the paper draft and ultimately accepted / resolved current open comments.","I explained her relevant points when working with visium samples. We focus on the general concepts and main workflow. I also show her resources to deep into this subjects, such as presentations and the use of the shinny up to reproduce some results.","We discussed recent results with our colleagues and made a list of pending items to clarify.","We set up the logistics for performing a remote LIBD internship with my team by re-using materials from the CDSB 2021 snRNA-seq OSCA-based workshop materials and videos.","Setting up and testing github access on laptop and JHPCE, continued","We reviewed how to deploy spatialLIBD shiny apps on shinyapps.io and why lines such as `library(\"markdown\")` are important to keep in the `app.R` R script file.","Setting up and testing github access on laptop and JHPCE","I showed him the template we currently use. I showed him many examples of how the various directories are expected to be related.","I help her to navigate in the resources to setup an appropiate configuration for her laptop. We installed some programs.","I showed her the team repository so she could update her profile. I helped her clone the repository and showed her the sections she needed to update. I presented her with different examples so she could create her own profile.","We setup cyberduck and ssh keys for the new laptop","We reviewed how the universe of background genes (for GO/KEGG ORA) is different for each cluster when using `pseudoBulkDGE()` due to the `filterByExpr()` internal step.","I showed her how to use Slack to have efficient communication, I also showed her examples of what kind of assumptions can be made in the different channels. I help her to organize her threads. I also show her my Google Slides weekly update setup, and ways to improve the current report.","We went over the preparation for my invited speaker presentation for this webinar series.","I explained the purpose of the RStat Club, showed her where the calendar is, how to schedule her participations in the shared Excel sheet, and how to send an email to invite other subscribers, as well as how to receive these notifications in the Google Slack thread.","I explained the purpose of the team presentations, showed her where the calendar is, how to schedule her participations in the shared Excel sheet, and how to follow the notifications in the Google Slack thread. I also explained her how to schedule a Dsgs calendar for ask help.","I recommended the code 'as.integer(levels(clusters)[as.integer(clusters)])' to address the particular use case, for a factor of integers 'clusters'","Discussed recent updates.","She encountered a cryptic error when pseudobulking on particular data. I recommended trying the code with simpler data to identify whether the data or code was a problem.","We discussed some future options for LIBD cloud computing to make sure that we were on the same page.","Further discussed options for working with the team.",null,"We interpreted and made annotations on the results comparing the clustering vs sub-clustering analyses.","I explained that often samples aren't trimmed by default, as the adapter content metric from FastQC must fail to trigger trimming. I mentioned the '--trim_mode force' option to force trimming","I described several common sbatch options and their syntax (and SGE equivalent). We ultimately discovered that the 'fusion_twas' module was never rebuilt, so I did that afterward","We briefly mentioned the edgeR 4.0 paper at https://doi.org/10.1101/2024.01.21.576131 and then mostly spent time reviewing the documentation for `clusterProfiler` at https://yulab-smu.top/biomedical-knowledge-mining-book/clusterprofiler-comparecluster.html.","I found a basic example script where I ran nnSVG and described the array job I use to run it for each sample, followed by an ordinary job to aggregate results by averaging gene ranks","We discussed the preliminary version of the website for visualizing storage use over time at JHPCE.","We walked thorught the code for leafcutter and the output for each stage","We reviewed when to use `set.seed()` when using `runPCA()`, `runUMAP()`, `runTSNE()` and other related analysis steps.","Ibidem.","We discussed feedback from collaborators and how to best address it.","I showed the specific scripts I used to perform the upload, described the process for resetting your password to obtain one usable for the CLI, and the python library for authenticating with NDA","We discussed why a sub-clustering analysis would be of interest for this dataset.","We talked about data science education and the InterJad platform.","We reviewed https://lcolladotor.github.io/bioc_team_ds/writing-papers.html and other guidelines for writing method sections of papers.","We reviewed https://rdrr.io/bioc/scran/man/pseudoBulkDGE.html and then also talked about how to perform ORA (Over Representation Analysis) on GO / KEGG. We also reviewed how to make volcano plots with https://www.bioconductor.org/packages/release/bioc/vignettes/ideal/inst/doc/ideal-usersguide.html#75_plot_volcano and https://bioconductor.org/packages/release/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html#plot-the-most-basic-volcano-plot.","We discussed options for how to use `derfinder`, `recount2`, `recount3`, `megadepth` and other related packages and tools (like `recount.bwtool`) to quantify expression from BigWig files from the recount2/3 projects. There is a need for a `megadepth` function that is similar to `recount.bwtool::coverage_matrix()` that works via the web given the current limitations with `rtracklayer`.","We de-bugged some issues with launching the app, and made some observations about the clustering at different ks, utimatley the optimal k will be based on the biological question at hand","We discueed MeanRatio and 1vAll methods for finding marker genes","We reviewed material from OSCA on `pseudoBulkDGE()` documented at https://bioconductor.org/books/3.19/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis for performing DEG analysis within each cell type cluster.","We discussed opportunities within my team and LIBD in general.","registation wrapper filters the pseudobulked object,dropping lowly expressed genes. Often resutlsing in ~12k genes (comapred to ~30lk genes in the original object). This is expected behaviour.","We discussed options to extend our collaboration in 2024.","We reviewed the properties of SpatialExperiment objects and other type of R objects (like a `GRangesList`) we used in Huuki-Myers et al, Science, 2024. The goal was to promote interoperability with planned work by the OP. We also went over https://research.libd.org/DeconvoBuddies/reference/findMarkers_1vAll.html, https://research.libd.org/spatialLIBD/reference/registration_pseudobulk.html, https://bioconductor.org/books/3.19/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis, and https://research.libd.org/spatialLIBD/reference/registration_wrapper.html.","We discussed opportunities within my team and options for collaborating moving forward.","We discussced refernce vs. querery in spatial registation and the required modeing to prep the data for the registration anlaysis.","I provided an overview of what my team does and the type of projects we work on. We brainstormed a bit about fundraising.","We discussed their GitHub profile and other evidence of coding skills.","I discussed possible choices of input genes, the fact that raw counts are used, and pointed to some example code where I had run PRECAST","I reviewed the general process and location of code I used for the spatialDLPFC project for uploading to NDA","Demoed issues with 'vis_clus' when plotting logical variables, which we ultimately determined to be a spatialLIBD bug (wrote a GH issue as a result)","I showed the source code inducing the error message (suggesting a mismatch in keys between previous annotation and the current SPE object), and we increasingly tested simpler annotation examples until something worked. Ultimately, we think the editor that was used in manually modifying the annotation CSV may have been the culprit","I pointed to an initial version of my code which had a simpler workflow than the reference code she was modeling hers after, and gave a brief explanation of the purpose of GLM-PCA","Identified the correct data and organization for this analysis","I helped with building the samples.manifest file and addressed a permissions issue on annotation files that was halting the first SPEAQeasy run","We went over some documentation and features of https://dlpfc.mccarrolllab.org/app/dlpfc.","We revisited [documentation by 10x Genomics on how the reference GTFs were made](https://kb.10xgenomics.com/hc/en-us/articles/115003616023-Files-and-criteria-used-to-generate-10x-references) such as the [mouse 2020 ref](https://www.10xgenomics.com/support/software/cell-ranger/downloads/cr-ref-build-steps#mouse-ref-2020-a).","We went over how to configure Zotero https://www.zotero.org/ and Zotfile https://github.com/jlegewie/zotfile for organizing PDF files of papers in your laptop. I keep the PDFs on my Dropbox and use Zotfile for automatically naming the PDFs when I download them via the [Zotero Google Chrome plugin](https://chromewebstore.google.com/detail/zotero-connector/ekhagklcjbdpajgpjgmbionohlpdbjgc?pli=1).","We discussed some questions about the application of qsvaR to the habenulaPilot bulk RNA-seq dataset.","I showed the code I've used for log normalizing spatial data counts, and linked to OSCA's section on log normalization via deconvolution, after which the code was based","I showed the syntax git uses when a merge conflict occurs, and how to resolve conflicted files","We checked their LinkedIn, website, and CV. I provided a list of changes to improve all 3.","We discussed the spatialDLPFC paper, how we integrated the sn and visium datasets","We have old results for the brainseq DLPFC, but i described the method we chose from deconvolution benchmark to get most accurate deconvolution results","She showed that capture area positions from images stitched with ImageJ disagreed with those stitched in the python-and-Samui-based workflow. I went through suspicious areas of code without success. Later, she identified that it was an issue related to differences in scalefactors and image resolutions, and differences in underlying assumptions between the stitching steps","We used commands such as `for i in *Cg*/*.fastq.gz; do echo $i; zcat $i | head; done` to show the first few lines of 10x Genomics FASTQ files (Illumina sequencing), then checked the indexes vs those that our colleagues used for based on their wet lab notes.","I went through the scripts involved in locally deploying a Shiny app, as well as which files to download from JHPCE to interactively test","We discussed a few things including `jaffelab::expression_cutoff()` logic and briefly mentioned `edgeR::filterByExpr()` as a more conservative alternative. In any case, when working on a sample subset I do recommend starting with the raw data and filtering out lowly-expressed genes (features) as which genes are expressed in a subset can be very different from the full dataset. We talked about ranges of logFC and t-statistics seen in DGE results from different projects, which can change depending on the biological covariate of interest. We also discussed our current strategy for selecting covariates in a DGE analysis when working on a small(er) dataset.","We discussed tools to manage files on JHPCE, and stradagies to get code to the cluster (either edit remote code, or use git)","We discussed that on a discovery dataset, we apply an FDR filter, and thus filtering out lowly-expressed features (genes) can affect the results. It doesn't impact the target dataset at all since in the target we are using nominal p-value (aka not FDR-adjusted p-values) and sign(logFC).","We went over how to update globus https://research.libd.org/globus/ by updating https://github.com/LieberInstitute/globus/blob/master/index.Rmd, then clicking the \"knit\" button. We also went over how to create Globus endpoints (collections that you share), how to set the right permissions, add other administrators, and get the link for sharing which is what we need for the index.Rmd script.","I noted the error about termination by the external system, which suggested to me enforcement of time or memory limits. Looking in the working directory for the failing Trimming process, we saw a message about OOM, so I suggested bumping memory up for Trimming in the config file","I showed the three kinds of plots we made for the habenula RNAScope data, geom_hex/stat_summary_hex for continuious variables across the tissue, geom_rect for specific cells, and geom_rect + geom_point for highlighting rare cell types.","I jogged my memory of the NDA upload process to provide guidance about creating an NDA study. I then described the order, purpose, and inputs to some scripts for actually uploading the data","I described the overall idea: fitting a linear model and dropping specific terms, keeping just the residuals. I recommended setting `P = 2` for case-control analyses where the diagnosis term is first, in order to explore if dropping the effect of other terms, such as batch, could clean up the signal","We double chekecked the stats for eQTL results, refined results and plots accordingly","He noted that an ldpaths and Makeconf file were created with bad permissions sometimes when working with the r_nac module at JHPCE. We identified that switching from one conda-based module to r_nac triggered the issue, and I suggested tricks for changing permissions on the files to prevent the issue from recurring","I walked Daianna through the 3 steps of MAGMA, and the needed input from GWAS data and gene sets. There are many detials to consider, especally in preparing this data. Daianna will start with preparing the GWAS data and we will revisit the next steps.","We talked thorough their proposed analysis. THery Decided Bisque may be the test boption I pointed them to my tuttorial for Bisque Bisque https://research.libd.org/DeconvoBuddies/articles/Deconvolution_with_Bisque.htm","We discussed the nuances of some of LIBD's internal datasets, questions that need to be asked, whom to ask them to, and potential reasons for including/not including myself or others as Co-Is on a grant application.","He noted some configuration was out of date, and I pushed updated configuration (related to the transition to SLURM). I also recommended using Arioc 1.51, since the 1.43 module was failing to install","We discussed how it could be useful to compare CellRanger v3 with the pre-mRNA modified GTF, v6 without the include-introns flag, v6 with the include-introns flag, Alevin, and STARsolo. Alevin and STARsolo might have similar include-introns flags to CellRanger by 10x Genomics. We discussed how this could potentially affect the class of single exon genes that are embededded within introns of longer genes. Violin plots by cell type across the different computational pipelines and alignment options could be useful to learn more how this affects gene quantification of this special gene class.","I found a config file for the particular software that supported singularity, and noted which module to use at JHPCE (also warning that it may require a good amount of memory on the first run)","We discussed in detail the DEG models for the BSP1, BSP2 (combined regions, one region at a time), BSP3, Habenula, and CMC Phase 1 datasets. All should be age > 17. BSP2 overall should adjust for brain region (a large source of biological variation) and also drop the 43 HPC RiboZeroGold samples. BSP2 HPC only should also drop those same 43 RiboZeroGold samples. Habenula should adjust for the tot.Hb and tot.Thal estimated fractions: here's the [published model](https://github.com/LieberInstitute/Habenula_Pilot/blob/ca6dba5f629a8ec580297ab6d56b36b4f0cb66da/code/10_DEA/04_DEA.R#L124-L126). CMC Phase 1 should adjust for BrainBank. Here's the BSP1 hg38 (part of the BSP2 paper) [model](https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase1_qsv/casectrl_DLPFC_allFeatures.R#L40).","I outlined that staged but not committed changes prevent merging, and how to resolve merge conflicts in files modified in different ways from two different sources","They taught me that `dreamlet` can produce `mito_percent` and `ribo_percent` for each cell, and thus you can summarize them with the mean across all cells of a given cell type for a given donor. This expands the options for covariates in a DEG model. We discussed why it could be interesting to run a model with and without adjusting for `mean(mito_percent)` given that in spatial data (such as the data from `HumanPilot`) `mito_percent` is related to the spatial location of WM and L1 (richer in oligodendrocytes and other glia cells, low presence of neurons). We also went over a part of the [LC eLife paper](https://doi.org/10.7554/eLife.84628.3) and revised some steps from [OSCA on multi-sample DE](https://bioconductor.org/books/3.18/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis). An equivalent of [`totalAssignedGene` produced by `SPEAQeasy`](https://research.libd.org/SPEAQeasy/outputs.html#quality-metrics) could be available at the donor (individual) level, but maybe it wouldn't be that useful as a covariate in these cell-type specific analyses.","Re-organized code to run with a shell script from slurm_jobs. We solved a bug with args index, and talked about adding catches for unexpected inputs. Future goal is to use map to replace for loops when handling list of data objects","After making a new `git clone`, we created the `gh-pages` branch with commands from https://jiafulow.github.io/blog/2020/07/09/create-gh-pages-branch-in-existing-repo/. For more on GitHub pages check [this page](https://pages.github.com/). We created a favicon using https://favicon.io/favicon-converter/. We created a [`README.Rmd` file re-using some code from the `spatialDLPFC` one](https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd), and also adapted the [`update_website.R` script](https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R). We also did a few other manual setup steps like creating the `img` directory and the `.nojekyll` file on the `gh-pages` branch, plus version controlling them. I might write a blog post about this full process in detail.","I explained the difference between signed and unsigned networks, as well as the statistical values that describe an eigengene module. I also explained the network representation in terms of a gene expression data frame versus a differential gene expression data frame. I also explained some functions to validate the genetic significance and module membership of eigengenes under this methodological approach for rna-seq data.","I explained why `age > 17` is a must for SCZD vs NTC DGE analyses: we have no SCZD cases with `age <= 17`, so if you don't filter the NTC donors with `age <=17`, you end up basically comparing older (SCZD) vs younger (NTC) donors, which is not what you want to do. We also talked about how in BSP2 there are some HPC (HIPPO) samples from 2 RiboZero protocols (Gold and HMR), how the Gold HPC samples cluster with the DLPFC ones on PC1 vs PC2, and thus why we ended up dropping the 43 Gold HPC samples in previous work (see https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/make_qSVs.R#L62-L69). This information is reflected by `analysis_sczd_casecontrol_hippo` at https://s3.us-east-2.amazonaws.com/libd-brainseq2/methprop_pd.Rdata listed at https://eqtl.brainseq.org/phase2/. Finally, I emphasized that self-reported `race` is not a trustworthy variable and we instead rely on `snpPC`s.","I outlined the process of running SPEAQeasy as a whole, and showed how to find which annotation settings were used on a particular project","I explained why using something like https://research.libd.org/spatialDLPFC/#interactive-websites or https://research.libd.org/globus/ give us control over links that we might need to update later on, instead of including other less stable links on manuscripts (domains we don't control, tools that might change their URL syntax, etc). I provided links to how to make each option such as https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R and https://github.com/LieberInstitute/globus/blob/98dc479ad2cde0a047cf7a46769a85c32969b7b1/index.Rmd#L8-L37.","We went over the data that we have for this project, worked on making a soft link to the output of cellranger that was generated in a different repository (and thus JHPCE location) to enable the use of `here::here()` in future scripts, and discussed potential issues we might observe in this new dataset.","In the session, we explored reanalyzing cloupe objects with the Loupe browser for scRNAseq assays. This involved using multiple projections like UMAP and t-SNE, splitting clusters, and filtering data based on UMI count and mitochondrial percentage. While there was interest in analyzing integrated datasets, the Loupe browser currently only supports exploring one Cloupe object at a time.","I suggested some temporary workarounds to get SPEAQeasy to recognize a conda environment with the necessary dependencies, but noted that we didn't officially support using conda for this purpose","We discussed the specifics of the [KavliNDI Distinguished Graduate Fellowship](https://kavlijhu.org/funding/4) and revised information that I can use while writing the reference letter (project title, assays proposed, that this involved new software, how long we've worked together, education history of the applicant, etc).","I explained where is located the cloupe output object generated by cell ranger-arc. Then, I explained how to use the tool to explore gene expression and atac assays by separate and also together. I also explained how to inspect clustering and re-label clusters and sub clusters; plus how to check DEG and look at their gene distributions. Finally, I explained how to re-run an analysis and use the most classic formats to save the analysis performed.","I explained how to prepare the configuration files for working R and JHPCE.  I showed all the section that need to completed plus the configuration to access JHPCE, with links pointing specifically to Windows system. I explained the purpose of each section in the chapter 17 of https://lcolladotor.github.io/bioc_team_ds/config-files.html. I also provided the updated link (https://jhpce.jhu.edu/access/ssh/)  for the JHPCE configuration due the past link is broken because of the change to the new web version. Finally, I provided the most recent video made by Leo  https://youtu.be/qtn3eTwFlZI  (minute 17:50) to do this specific task for windows OS.","We added the `LFF_spatial_ERC` and `LFF_spatial_LC` datasets to https://github.com/lieberinstitute/ranger_metrics. This involved debugging the code a bit to handle updates on the Excel sheet files describing samples prior to being sent for sequencing.","We found out that `spaceranger count` did not like the JSON files we created with `jsonlite::write_json()` so instead, we edited them manually using `gsub()` and `read/writeLines()` as documented [here](https://github.com/LieberInstitute/LFF_spatial_ERC/commit/bbf810ad39012ba6ee4e0403063bf83bf844d007). Note that we had to manuallly revert the JSON files to their original state as documented [here](https://github.com/LieberInstitute/LFF_spatial_ERC/commit/79ed5479d502cd41d51f3aa8ee1467bd8100a08a).","We wrote a script to manually update the json files and add the Visium Slide ID as well as the capture area. This will enable us to not have to use `--unknown-slide=visium-1` https://github.com/LieberInstitute/LFF_spatial_ERC/blob/6e85c05ce9f531c555569e616f8d5c351b427b4f/code/01_spaceranger/spaceranger_SLURM.sh#L63 that 10x Genomics support had recommended using. Doing so led to an error at https://github.com/LieberInstitute/LFF_spatial_ERC/blob/b5b92a8bb9744ee424ec4a0642ae692423a91d56/code/01_spaceranger/logs/spaceranger_17.txt#L22 which stated that `--slide` cannot be used as the same time as `--unknown-slide`. This is a workaround that was previously manually implemented at https://github.com/LieberInstitute/Habenula_Visium. Since we have lots of samples, we wrote a script to avoid introducing copy-paste errors.","The issue on this [script](https://github.com/LieberInstitute/LFF_spatial_ERC/blob/e913ff28fffbb17335d3844d312ec6486a7b182f/code/01_spaceranger/spaceranger_SLURM.sh#L55) and related ones is that there is a missing `\\ ` (note the space) before the `#` comment symbol. Thus the chained command is breaking into two, and the second half is not being recognized as options to the first half.","We discussed tools for spacing and aligning objects in BioRender. I suggested maximizing font sizes and reducing white space","We worked on the following error message from `spaceranger count`: _ERROR: We detected a mixture of different R1 lengths ([26-28]), which breaks assumptions in how UMIs are tabulated and corrected. To process these data, you will need to truncate to the shortest observed R1 length by providing 26 to the --r1-length argument if are running count/vdj, or via the r1-length parameter in each of the following tables of your multi config CSV if you are running multi: [gene-expression]_. We inspected the soft links for the FASTQ files and used `zcat fastq.gz | head` to inspect the first few lines of FASTQ files. We confirmed that the `R1` FASTQs were indeed of varying lenghts and will attempt a new `spaceranger count` run using `--r1-length = 26` though it's likely that we'll next get an error with the `R2` FASTQ files as we observed that those files also have varying lengths.","Able to access the data throught the LIBD dataportal, I also pointed to the combined LIBD rse_gene in brain_swap (which I think might be the source of the portal data...would need to check)","Updated script by commenting out unwanted lines and providing correct paths to the input files","I explained what the key metrics are for each assay in a Multiome experiment from 10x scRNAseq and scATACseq data sets. I showed several HTML reports of data sets processed with cell ranger-arc to illustrate different scenarios targeting different numbers of reads for each assay, to help you understand the relevance of quality beyond the number of reads in sequencing. I showed how with a dataset with a moderate number of good quality reads you can perform a good analysis, while having many reads with compromised quality it is possible to have more challenges during the analysis. Finally, I explained the minimum recommended number and that it would be a good idea to search the literature for technical data from other experiments similar to his as a reference.","Cross checked the script and rectified filename/samplename processing","I explained that BSP1 is polyA, aligned vs hg19, older annotation, aligned with TopHat2 (an older aligner). BSP2 is RiboZeroGold for DLPFC and RiboZeroHMR for HIPPO (with 43 RiboZeroGold samples that were dropped from the DE analysis), aligned vs hg38, against Gencode v25 (from July 2016), aligned with HISAT2. I also explained the naming structure of a Supplementary Table for BSP1's paper where `adj`, `qSVA`, and `PCA` are labels specifying whether it was a model `adj`usting for covariates, whether it adjusts for qSVs using `qSVA` (the better model to use), or adjusting for gene-level expression PCs (`PCA`). These names are related to the [`qSVA` 2017 paper](https://www.pnas.org/doi/full/10.1073/pnas.1617384114) Figure 2. I also explained how in the BSP2 paper we did re-align the BSP1 data against hg38, though there is no Supplementary Table showing which of the 243 DLPFC DEGs replicated in BSP1 (though the data does exist since it was used to make Figure S28 and S29). We would need another DSgs or two to actually locate the data and construct an CSV file or table that others can use if they are interested in replication of BSP2 results using BSP1 and/or CommonMind Consortium (CMC) Phase 1 data (that was aligned vs hg19 with TopHat2). I also used [this slide](https://speakerdeck.com/lcolladotor/libd-ds-tldr?slide=56) to explain all the pieces that go into a DEG model and why the visualizations provided by the LIBD Data Portal (boxplots for SCZD vs NTC) are just a visual guide using normalized gene expression data and not the full answer (as the LIBD Data Portal doesn't use the full details we include in a DEG model).","I explained in detail the metrics reported by cellranger arc and helped you download a multiome public sample to run the pipeline.","Re-organized code to run with a shell script from slurm_jobs. We solved a bug with args index, and talked about adding catches for unexpected inputs. Future goal is to use map to replace for loops when handling list of data objects","We disscussed specific details on how to ask for help. For example, it's ok to ask those around you in person but you should also ask on the `#libd_helpdesk` channel (and update the channel if someone else gave you an answer). That is, don't assume that only those around you (in person) know the answer. We went over details on how to preview GitHub messages (to verify that they are correctly formatted), why GitHub issues are useful to the authors, what information is needed for others to help you answer a question (R session info, `reprex::reprex()` output, link to code on a public GitHub repo; private is ok when asking internally for help). We also discussed the overall idea of why learning how to make an R package will be useful to then improve one's tooltik for debugging R packages made by others.","I explained the previous steps to perform clustering with GEX and ATAC 10x multiome data using Seurat and Signac.","We interactively ran the python code for reading in ImageJ transforms to rule out an issue where the wrong data was read in. Ultimately, we couldn't find issues with the data, but the Samui full-res images still seemed misaligned relative to the low-res images","We went over how to update a GitHub remote following [information from the documentation](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories), finishing stylistic design touches on an `iSEE` app, and setting up a new repository for the next project.","We discussed details of the use cases I have in mind for how to search and visualize the JHPCE disk usage data over time and across different JHPCE paths. This information could be useful in forecasting data growth for individual PIs or projects, identifying who has exceeded their storage allocation, identifying static projects that could potentially can be moved off JHPCE / deleted / or moved to non-production disk space.","We discussed the general overview of snRNA-seq data analysis steps and how the code at [`Habenula_Pilot`](https://github.com/LieberInstitute/Habenula_Pilot) could be a good starting place. I also recommended signing up for a Data Science guidance session with Louise as she is quite familiar with the code in that repository prior to diving into it.","I explained details about how data is organized at JHPCE, data sharing limitations, and several suggestions we have been working on that could be useful to implement at a larger scale across LIBD teams.","I pointed to suspicious areas of code where dataset-specific regexs were originally used, though the regexs looked good for his data as well. We decided to interactively walk line by line at a later date","Discussed collaborations, personalities, and other details of the work environment with some of our collaborators with whom this person was going to interview for a job with.","I suggested building a list() where each element was a 3-piece plot (combined with a function like cowplot::plot_grid), then using print(plot_list)","See https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1710454284251579 for the full code details. We might add this to solution to https://github.com/LieberInstitute/jaffelab/blob/devel/R/merge_rse_metrics.R, although it's only an approximation as you need to go back to the raw ERCC output files (typically generated with `kallisto`) to actually compute the ERCCsumLogErr accurately across different sequencing lanes for a given sample.","We have RSE objects from the LIBD Data Portal (or those that helped build that portal) as well as RSE objects made prior to the LIBD Data Portal. They have some differences, mostly on the `colData()` slot. Note how https://github.com/LieberInstitute/jaffelab/blob/21e65745a5b0852fc13ebdb5e5d23c6f1f66010b/R/merge_rse_metrics.R#L83-L97 and related code merges `colData()` information for samples that were sequenced in more than one lane and whose FASTQ files were not merged prior to aligning with `SPEAQeasy`.","I walked throught the run_DE function I wrote for MDDseq (a wrapper for the limma-voom) pipline. I pointed out what parts of the MDDseq code that could adapted and simplified for this use case (its less complicated than the many DE runs we completed for MDDseq)","I suggested using conda after pip failed (missing system dependencies). Ultimately, my troubleshooting was enough to just build a module for everyone","I explained details about the spatialDLPFC and deconvolution-benchmark papers. I also went over how we addressed feedback from the editors for the spatialDLPFC paper.","Explained how to access authorized keys and exit JHPCE","Having some conectivity issues with JHPCE, so we didn't get a full bug fix in I suggested some de-buggin strategies  [Update 3/15] We met again breifly to confirm the functions were working as intended. Initial issue was caused by un-intentinally overwriting the function","We discussed how to address feedback from a recent grant application, how our teams can get to know each other better, cleared up misunderstandings related to my team's role as well as the DSgs system, and decided next steps to respond to the grant reviewers.","I described how a markdown of notes could be added using the Samui python API, though this is not how I had done it in a previous project ( I was unable to trace the original method I used)","We discussed more details about `iSEE` and shinyapps.io deployment details, such as memory settings and other configuration needed to actually customize and share an `iSEE` app. We also asked our collaborator for the list of genes they want to have plotted outside of `iSEE`.","I helped implement an algorithm taking the minimum pixel intensity in case of overlapping images to display an H&E image in Samui","I diagnosed several potential issues, ultimately missing one detail that was the culprit: permissions were too open on some .ssh files on her laptop","Explained what parts of VNS and refineVNS can be tuned to get appropriate segmentations for Nac project","I sent a long and detailed email explaining different components of the LIBD BrainSEQ projects, background information on several covariates we typically have used in DEG models such as RIN and age at time of death, as well as other ones we have not used such as postmortem interval (PMI). I also explained the DSgs system and how it could be used by our JHU collaborators.","Giovanna and Jisu are working with some bulk RNA-seq data as a some pilot data in a new brain region prior to preforming saptail. They want to use deconvolution to help QC the samples, selecting for good disections. I walked them through our current recomndations for deconolition to help with this goal.","We went over internal code from the limma package using https://code.bioconductor.org/, particularly for the `voom()` function. Doing so, we discussed that's actually returned in the `voom()$E` slot. We also went over how to model a ratio using a linear regression model.","We went topics such as live in Baltimore, research at JHU, overview of the different JHU campi, how to compare PhD programs across universities, and other questions this future PhD student had.","Discussed feedback from collaborators and detailed next steps to wrap up this project.","Went over recent updated plots and results, as well as slides for a presentation with our collaborators for 2024-02-21.","We discussed the updates implemented in qsvaR to handle input with ENSEMBL transcript IDs instead of GENCODE ones, updates to automatically detect if the input is at the gene level instead of the transcript level, how to check for correlation between qSVs and covariates, among other related issues.","I described my typical strategy of using left_join by sample ID, as well as ensuring rownames don't get lost from conversion to a tibble (thus dropping colnames from the SummarizedExperiment unintentionally)","I noted that the r_nac module was built similarly to conda_R, where the user doesn't have to do anything special to install more packages. We ultimately traced the issue to permissions problems on files created as part of the proper use of the module by other users, likely due to problematic 'umask' settings","Sreya went through an error issue in qsvaR due to rownames not well formatted. The error message was misleading. This problem has been fixed in the latest updates of the package. I advised Sreya to get the latest version of the package.","Discussed updates to violing plots and how to organize result files prior to sharing them with our collaborators.","I showed him to the 2 approaches available in Seurat to perform DGE, including the FindMarkers() function to find the genes that are different between control and treatments. I also show him code to perform quality controls in the main covariates related with these data-type. I shared with him consulted resources and links with public-code for his reference.","I ultimately recommended 'git reset --soft' to the commit ID before the recent mistaken addition of the large files","I advised using the SSH address (not HTTPS) for the 'git remote', ensuring the 'git config' email matches the primary GitHub email, and using the up-to-date SSH key (not RSA, which is no longer supported/ recommended)","We explored options: BFG cleaner, git rebase and git reset --soft and discussed which was best for her situation","I described the use of 'srun' (including with X11) vs. 'sbatch', and generally recommended 'sbatch' where possible to document how analyses were run/ ensure reproducibility","Caught up and talked about career development next steps.","I mostly focused on my setup using VS code with its remote SSH extension, and custom keyboard shortcuts for running python code interactively","I mostly confirmed the general process he had started for a particular module made from software packages as a precompiled binary","Went over the details of how to compute a permutation analysis with R.","Sreya wants to compare the correlation of 5 transcipts with 2k degradation transcripts. She had the data in the right format but needed help appllying the cor function to the table using group_by. Corrrplot was not the correct function for what she wanted to achive here.","Clarified updated automatic checks and information for users who attempt to use qsvaR with gene-level data instead of transcript-level data.","I described how code might change when using IF images rather than H&E, for which the code was originally developed","Michael and Peter are working on a BPD vs. MDD DE analysis. They were getting more DEGs than were reported in an early version of the MDDseq project. I could not find the code for the data they were refrencing, with out the code it will be impossible to diagnosis what is differnt in the two analysis. I recomended they use the more up to date data from MDDseq, and pointed them to the correct data & scripts.","Tricia and Fernando are re-running serval analysis they previously had working on the sge cluster, with the update to slurm they are now having issues. They are also having differnt errors in the same scrips + data. This could be a multi-facet issue, but firstly sounds like a R package version issue. I advisded the first step should be to have the same set-up, work in the same version of R and the same version (most updated) packages. See what errors persist and go from there. I also recomended Frenando start using git repos and version control with his team to help keep projects organized and consistent.","During the meeting Sreya showed me her progress on the analysis she's doing taking the 2976 txs from the cell-component model and she wanted to compute the correlation between those and selected transcripts she's interested in. She thinks those transcripts are involved in the degradation process. We couldn't find those transcripts as part of the 2976 txs from the cell-component model.","We checked udpates to the `getDegTx()` and `DEqual()` functions, as well as discussed the response to a bug report from last year.","I described how to adapt a script I wrote for NAc to her dataset, where the goal was simply to display a stitched-together image in Samui","I traced the issue to the use of 8-bit integers, where an overflow occured during normal addition of pixel values (the solution was to use floats while computing the mean, and then convert back to uint8)","I described how I typically use dplyr::left_join, as well as how to add back missing rownames to the colData (or colnames of the SummarizedExperiment)","I described the conflict between SGE and SLURM R packages, as well as identified lock files preventing re-installation of problematic packages","We traced the offending line of R code in the qsvar package and I described how unit testing could identify problematic function arguments and give more informative error messages","I met with Sreya because she's having an issue with the getDegTx function from the qsvaR package. I tried to solve the issue but I wasn't successful so I asked Sreya to give me more time and access to the rse object to have a more deeper look to understand the issue. After using her data I realized that the error message was misleading  and improved the function and unit tests.","I noted that the function could be applied before the data was converted to long format, where it was impractically large to work with","We identified issues causing incorrect coloring in the Samui outputs","I described how to adapt a script I wrote for NAc to his dataset, where the goal was simply to display a stitched-together image in Samui","I explained the different modules such as `module load conda_R/4.3.x` versus `conda_R/4.3`, how each is tied to a different version of Bioconductor. I also linked to https://youtu.be/D6t2GR7xaTE and https://docs.google.com/document/d/1DD9MUI2e3jp5S8C3PWRulZ4jwbXo94D3UTCQNJl_-Qo/edit?usp=sharing from the LIBD rstats club.","Researchers are starting the annotation/cluster integration of the multiome data. The data is currently clusted indiviudally by sample, sn, ATAC only, and combined sn+ATAC. findMarker has be used to calc marker stats for each cluster in each sample (there is a ton of data to digest). I am unfamilar with the sample independent cluster, consered marker gene integration from seurat. But I gave advice to check the current clusters for high expression of any of the known marker genes to see if we're picking up the expected biological signal, and do some prelimary annotations as a sanity check. Also to keep following the workflow and see if the conserved marker genes match with any of the know markers. Also to start recoring annotations for the clusters in a csv file for easy editing by hand as well as easy reading in to R","Provided feedback on CV writing for data science job applications.","I advised on how to adress some issues with the qSVA overview plot, both minor issues like aliginig and resising objects with biorender tools, and major rearngment of the table and DE application part.","I described the workflow for stitching Visium capture areas with Samui, and various Samui API functions like 'add_chunked_feature' that are helpful","We discussed the [GLM-PCA paper](https://doi.org/10.1186/s13059-019-1861-6) and decided to use GLM-PCA instead of logcounts by `quickCluster()` + `calculateSumFactors()` as described at [the CDSB 2021 snRNA-seq workshop](https://comunidadbioinfo.github.io/cdsb2021_scRNAseq/normalizaci%C3%B3n-de-datos.html#normalizaci%C3%B3n-por-decircunvoluci%C3%B3n-deconvolution). We also discussed the development version of `Harmony`, in particular with the `lambda = NULL` setting.","We went over how to build a reprex for a user's bug report, how to simplify the process of debugging, and the actual results from this particular bug report. We also discussed how to relay this information back to the person who initially asked the question.","I provided background information on a job ad from a colleague at Boston's Children Hospital that I had shared. I also provided suggestions for preparing their GitHub account, CV, and other information useful for job interviewers.","Adressing feedback on some unexpected behavior from qSVAR package. The qSV values show no significant correlation with other phenotype metrics (they have in the past), while this is not impossible it is strange. There are many factors that could have lead to this issue. We talked about some debugging strategies, starting with re-running the workflow on familar data to see if we can re-create the issue or detirmine if it is unique the external dataset.","I provided background information on a job ad from a colleague at FHCRC that I had shared.","I gave feedback on scripts, how to organize them, the order of some analytical steps, which gene annotation files to use, and things like that. We don’t have results on the QC steps to share just yet.","I showed a workaround for making sure a variable has global scope before editing it inside a function, though I noted it's better not to use a function in her particular case","Continued reviewing qsvaR replication analysis code.","Detailed analysis steps for [this project](https://github.com/LieberInstitute/Cg_Hb_projector_profiling) using snRNA-seq data and analyzing it following OSCA-based steps from the CDSB 2021 snRNA-seq workshop.","Reviewed draft blog post on the Visium SPG AD project and published it at https://research.libd.org/rstatsclub/2023/11/01/our-first-adventure-with-visium-spatial-proteogenomics/.","I identified two separate issues: the repository name and using HTTPS remote instead of SSH","I walked through how a decision tree works and can be interpreted, then described how correlation and RMSE are computed in a couple spatialDLPFC spot deconvo plots","He noted that the wrong version of python was being used, and we eventually traced this to user-specific conda settings interfering with the module","Explained how interaction models work with limma-voom.","Ibidem to 2023-09-29 session.","Reviewed [this script](https://github.com/LieberInstitute/QSVA_2020/blob/914e3b58f7fb97c204654493c9f9dcb2e48d112b/code/06_Replication/02_DE_and_replication_plot.R#L220-L313) for the replication analysis performed as part of the qsvaR project.","Paired programming session to debug `/dcs04/lieber/lcolladotor/with10x_LIBD001/ranger_metrics/code/spaceranger/03_merged_metrics/02_explore_metrics.R`. Further explained the original intent of the code in this script and steps for updating it.","We interpreted the intuitive meaning of a formula from some old code, which attempted to quantify the similarity of a set of significant genes between datasets. I also helped interpret the meaning of a plot based on this metric","This data has one cluster that is unqique to one donor, post batch correction. This donor also has some unique clinical background as well, I advised to drop the cluster as their batch corection seemed to be correct, and it possible that this is a rare biologically real cluster but there is not enough data to confirm.","I recommended 'spatialLIBD::fetch_data' for easy retrieval of public data to get started with spatial computational analysis. I also recommended pushing for an account at JHPCE, where computational resources are more plentiful, and getting used to submitting jobs, accessing software, etc is valuable to learn","Reviewed some scripts related to replication analyses. I explained the different thresholds used on the discovery as well as the target datasets, as well as the different levels for assessing replication.","I walked through the new login commands for SLURM + updating his bashrc file so they can work with the slurm cluser","Ibidem to 2023-09-29 session.","I noted that conda_R seemed to be changing the version of java loaded, which affected how the nextflow module worked. I recommended unloading conda_R and avoiding the loading of conda_R via the .bashrc","I described the general process of setting up SPEAQeasy and gave recommendations for his cluster: using singularity and SLURM-based installation/ set-up","I mostly covered documentation but also the appropriate options for his particular data","I investigated a \"failure to submit process to scheduler\" error using 'sacct' and similar commands, ultimately discovering the issue was related to SLURM configuration. There was an email chain with bithelp to follow up","We went over the changes needed to take the current qsvaR paper draft by Josh Stolz to a production-level result (aka for pre-printing / publishing it).","Adapted mode code for tensorQTL prep, had to add genomics ranges to the sce object, and deal with bugs around rse_to_bed function","I followed up with bitsupport, as the issue seemed to involve some failed DNS query internal to SLURM, rather than SPEAQeasy","We walked through old code which aimed to quantify the similarity between significant DE results, interpreting formulas from the code and constructing more intuitive formulas","Ibidem to 2023-09-29 session.","Worked through addapting code for tensorQTL prep for the MDD single cell data - had to deal with bugs relating to cell type names","I gave general advice on re-writing code-- keeping separate scripts, chunking code into segments and testing equality of outputs between old and new code after each chunk, and other strategies","We continued working with the OSCA-based CDSB 2021 intro to snRNA-seq workshop to further understand snRNA-seq data analysis steps, packages, and how they all relate to each other. The person who requested this DSgs was working on the workshop contents and had questions that we reviewed weekly (or so). Once they completed this OSCA-based training, we moved to analyzing an actual unpublished snRNA-seq dataset.","We looked at the rank vs. total UMI curves relevant for emptyDrops for the atac-seq samples. We have discussed these samples before. We went over how the emptyDrops function works again and what we can guesss is happening in the curve (neurons vs. glia vs. empty droplets).  There is concerned beause for the 42-4 sample the wetbench target number of nuc was 7k, emptyDrops retrurns 9k. I thought that that sounded in the right ball park","I described the 'droplevels()' function, designed to remove unused factor levels","We went over the background for this project (lateral septum) and an overview of the DSgs system, GitHub setup, and other onboarding material.","I discussed how we selected training data for the spot-deconvolution image model-- using logistic regression to compute a confidence metric across the original training examples, that we then binned into 4 quartiles to re-select new examples to label in a less biased way","I mentioned slurmjobs::job_loop and also described how to use it in conjunction with getopt in R for looping over several variables","She had an issue installing a particular R package, which we pinpointed to which modules were loaded (including via bashrc). I also described the general problem of SLURM and SGE versions of JHPCE sharing a library path, which could result in subtle problems. I recommended deleting SGE packages (better would've been backing up!) and reinstalling on SLURM","I helped modify my code for creating input files to work witht the new pseudobulk data. We got the gene PCs done, he needs to get the genotype files together for this data before we can move to the next steps. Planning to meet more next week to continue this process.","Resolved some questions related to https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq and made two pull requests to fix typos.","We went over the math derivations for interpreting beta1, beta2, and beta3 on an interaction model where X1 is a continuous variable, X2 is a discrete variable with 2 options and we model `Y ~ X1 * X2`. We also used `ExploreModelMatrix`.","I explained the benefits of centering and scaling and referred to both the short video on this topic https://youtu.be/c8ffeFVUzk8 as well as the material we developed for the CSHL 2023 course https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html.","They were interested in running deconvolution on the bulk MDDseq with the new large MDD snRNA-seq dataset. The MDD snRNA-seq dataset is only sACC, we concluded that only haveing one region might make this analysis a bit awkward for the Amyg data. I them to the current deconvolution results and how we applied those results in downstream analysis.","I walked through using registration_pseudobulk() on the large MDD single cell dataset and went over how the function works. We are planning to meet agian to go over running PCA to prep for eQTL analysis.","I noted the 'www' directory (or the directory passed to the 'docs_path' parameter) controls the home page, and we typically use a relative soft link to the main repository README for the bulk of the content.","I described the general workflow, involving ImageJ, Samui, VisiumStitcher, and custom R + python code needed to eventually produce SpatialExperiment objetcs with one donor per sample. I focused on the high level of what the Samui piece looked like/ how much work is involved","Continued reviewing material from https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq","Discussed a few ideas about LIBD and social media","I showed code examples of writing array jobs with SLURM (and how it compares to SGE), then diagnosed an X11 forwarding issue where the '-x11' srun option was insufficient. Adding X11-related configuration in the SSH config fixed the issue (i.e. ForwardX11 and ForwardX11Trusted lines)","I discussed diagnosing marker quality for the spatial hypocampus project. They are on the right track, and have created violin + hockystick plots for the different iterations of the dataset. I advised they elimate a noisy cluster and re-compute markers.","I described the pieces that go into a bookdown-powered documentation site, showing RMD code examples and the GHA workflow that deploys the site","I noted that Samui supports many types of features, and the spatial coordinates of cells can be supplied instead of spot coordinates to display features related to cells","I described the meaning of cell2location outputs and how to compare outputs between software tools. I also described where plots came from in R","Used MAD cutoffs on total UMIs, genes detected, mito %, and a metric we haven't worked with yet: genes per UMI. They evaluated the data all together (not by sample) and chose #Mads by what seemed reasonable. The values  used look good, but perhaps a bit strict. We looked at the distributions, which are slightly different between samples, I advised to use scater::isOutlier to see what the cutoffs would be by sample. One step lacking is doublet detection, I suggested to apply scDblFinder::computeDoubletDensity. The dataset already has clusteres and is annotated  (lots of work) I suggested that a great double check is to calcluate the median value of the QC metrics for each final cluster and see if any are much higher than the others as a good double check that his initial QC process worked well.","Answered questions related to material at https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq","We talked about the first scripts in https://github.com/LieberInstitute/ranger_metrics","He noted that not all measured genes were displayed in the Visium_SPG_AD project. I found in the code why I subsetted and made a GH issue to fix it","I shared my favored set up for getting computational work done: working directly on JHPCE via the SSH extension for VSCode, using system-wide JHPCE modules for consistent software versioning, and X11 for interactive plotting (with a note I tend to just create and view PDFs for complex plots)","My major recommendation was learning version control with git, though I also mentioned learning SGE/SLURM also is a valuable skill for larger-scale coding projects. I helped him create a git repository for his current project (including account set up) and described the basics of git add, commit, and push","I described how the various scripts connect in spatialDLPFC for my work with deconvolution, to help share and re-use them for her project","We talked about material at https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq","I explained the status of the recount-pump transfer, where files were, and noted that chris or JHPCE admins would have to change permissions to allow her access to some files","I noted that modules at SGE vs. SLURM are totally disjoint, and that the HOSTNAME trick originally used in the team recommended bashrc didn't (at the time, though this has recently changed) work anymore, and the SLURMD_NODENAME variable can instead be used to infer where the user is","I gave some pointers about writing tweets. Mostly keep it simple, give people a call to action \"check out this link!\", add something eye catching. Also to spend sometime reading tweets and pay attention to what you like that other people tweet? What caught your interest? We also talked some stradgedy about tweeting around confrences, calling peoples attention to your talk/making it easy for people to find you to get in touch","We talked about how I became an Investigator at LIBD, the overall type of work we do as a team, some of our main collaborations, and overall plans for applying for funding in the future. In particular, I highlighted how I want to train team members on how to write an NIH biosketch, though not all are going to be \"key personnel\". I sent a link to the DSgs, team meeting schedule, rstats club schedule, and our http://lcolladotor.github.io/talk/teamtldr2022/ summary from April 20, 2022.","I did a complete tutorial of using the Samui API to create Samui-importable directories, including how to preprocess image files and how to host results publicly","We talked thorough the thigns to think about for the Mathy's dataset (this dataset has case + control paitents and found DEGs w/in cell types). I advised to start simple with the broad cell tytpes and not consider Dx at first. Check if the DEG they idetified are in our marker set. I think this will need some discussion with the project team if Dx is a concern for spot deconvolution.","https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/5e84f5ce32e6723b1a1eea20dc47ee320d55ca10",null,"He noted that just taking the most plentiful cell type in any cluster was leading to too many clusters with excitatory layer-5 cells. I mentioned that the raw outputs of spot deconvolution tools are fairly unreliable, but they still carry useful information when normalized. For his example, I mentioned dividing by the total number of cells for each cell type when making comparisons between cell types","Suggested deleting all old files as the code might be appending data instead of creating new files and also increasing the h_fsize, as the error relates disk space","guided him to the search engine on REDCap","For deconvolution, we talked about finding an appropriate reference dataset and we talked about an expert we can contact to ask which dataset(s) might be the most relevant. For snRNA-seq data analysis, I mentioned OSCA (Orchestrating Single Cell Analysis with Bioconductor) and how Louise could help guide this person on our code based off OSCA through a series of DSgs sessions.","removing the top 2500 pixels before splitting the image corrects the offset","I mentioned (information) entropy as a metric that would determine how well a gene is confined to a particular cluster (using average counts of a gene across all spots in each cluster, then normalizing the sum to add to 1), but cautioned that I wasn't familiar with favored metrics within the field","We transferred https://github.com/LieberInstitute/qsvaRPkgDemo to the LieberInstitute account. I asked to include some citations for sources (like the image source) and some other small edits. For GO / KEGG enrichment with clusterProfiler I asked to not filter results (pvalueCutoff / qvalueCutoff parameters), so we can see more results prior to choosing which ones to plot.","We took a look at the changes in the clustering in the spatial data before and after cropping out a area of questionalble quality. To me it looks like the QC params are too strict leaving spots floating in space, which I'm guessing is contributing to weird BayesSpace results (but this is a guess). We also talked code organization strtagedy - mostly keep it simple and consistent","I helped improve the formatting & content for a BioC qSVA workshop, main feedback ins to focus on and highlight the qsvaR steps in the workflow (rather than SummarizedExperiment overvew & DEG analysis)","We discussed the current status of the team, the project, what skills are required, boundaries in terms of time and who would be involved (and who wouldn't). We ended the session with me relaying a list of resources to check and read more carefully. Then in about 2 weeks we'll meet again to determine the next steps.","We made a Google sheet with columns such as `study_type`, `skill_name`, `learning_priority`, etc. I also worked on it after the DSgs session to help complete the current information. See [Slack](https://neuralplasticity.slack.com/archives/C04Q8MX99FY/p1690397027111449) for more details.","We resolved this issue by making sure that the variable names for which they wanted to specify the correct colors were actually part of the colData(SE). See [Slack](https://jhu-genomics.slack.com/archives/C01KSLPCGAC/p1689858182290369) for the actual code links. I also asked them to make sure to use soft links / full paths instead of using setwd() for files outside the project + to keep all related files at JHPCE instead of on a laptop, to make it easier to rule out any sources of differences (aka for reproducibility purposes).","I helped review a practice BioC short talk, Mostly the talk needs to give more slides to discussing the results & include more actual plots not just icons (this is much improved this presentation for the 7/26 meeting :) )","I helped discover the source and solutions to errors and warnings (mostly from BiocCheck and R CMD check), and found how to specify a single author in in-text citations",null,"reffered to this script /dcs04/lieber/marmaypag/Image_Processing/NM_LC_segmentation/test.m",null,"We discussed what would be the appropriate material for us to teach in a course. We will re-use some of our materials from https://github.com/lcolladotor/cshl_rstats_genome_scale_2023.","We went over the proposed response to reviewers, clarified some sentences, and requested co-authors to approve the response to the reviewers.","I pointed her to spot deconvolution scripts used in spatialDLPFC and the high-level overview of the workflow, as well as how we trained the decision tree classifier","I noted that he ran out of memory in his JHPCE job, which crashed R, so I suggested increasing 'mem_free' and 'h_vmem'","https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/tif2mat.m","I referred to examples of JHPCE modules I've created that activate shared conda environments, and explained the key commands used to get a similar module working","I recommended finding and trying to generalize the process a previous coworker followed to produce the list of degradation-associated transcripts, to limit relying on exact transcripts with an exact GENCODE version. I noted that you can convert transcript data to genes (albeit with lost information), but not vice-versa, so in theory either transcript or gene-level data could be supported by converting to genes","https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/mat2tif.m","referred here for clarification https://docs.google.com/presentation/d/1HJgkvlHtiIA5nucuNdHdYhRGAtpPPvASGFoAocqi1-U/edit#slide=id.g1163185b778_1_6","We adapted code from https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R and https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd in order to make the files at https://github.com/LieberInstitute/septum_lateral. See https://github.com/LieberInstitute/septum_lateral/commits/main from https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50 to https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c as well as related commits on the `gh-pages` branch.","Summarized at https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659. This involved explaining why we don't use as a background universe all genes in the annotation in gene ontology / KEGG pathway enrichment analyses. Instead, we use those that pass the expression filters and for which we computed differential expression statistics.","We resolved https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199 at https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&cid=C02RG5VGL6R, which was due to a user incorrectly having a Linux group assigned to files at JHPCE when they are not part of the user group. Uploads through Cyberduck can trigger this. The solution is to run https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example again on a qrsh session.","DropletUtils::emptyDrops on the two LIBD + two example samples has been run. We first clarified some terms and outputs she had recorded. For the most part DropletUtils returns similar number of non-empty cells as CellRanger arc, but didn't confirm that the two methods flag the same cells. I recomended to create a confision matrix between the two methods for each sample to check this.  The main concern  has is the LIBD sample with only  3k cells input, has 9k non-empty droplets returned by emptyDrops. This is quite odd as the CellRanger Arc metrics look quite bad for this sample, but the DropletUtils metrics look fine (there are high UMI values for many droplets). We will want to dig in to this more to understand the differnces between the two methods.","We worked on qsvaR debugging an issue raised by Renee and Louise. We then improved the error messages on qsvaR as noted at https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400.","Using the septum lateral data, I trained Ryan on how to make this type of Globus endpoint, how to set up the right permissions, how to edit and update http://research.libd.org/globus/ (see [this](https://github.com/LieberInstitute/globus/blob/b10e0fb89c40489d23028e8ed3f7812cdea76a6b/index.Rmd#L57) for details), and how to document the Globus information on a manuscript.","Using the lateral septum snRNA-seq data, I trained Heena on how to filter a SingleCellExperiment object to reduce the required memory from 7GB to less than 1GB in order to be able to deploy an iSEE app at shinyapps.io, whose computers have a maximum of 8 GB of RAM. See https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98 and related commits.","We worked with the script where Renee had noticed issues with qsvaR. We adapted it to work on qsvaR local updates through devtools::load_all(). We also resolved https://github.com/LieberInstitute/qsvaR/issues/29 and https://github.com/LieberInstitute/qsvaR/issues/28.","We started with getting github token access for their laptop working. Then started a pull request using usethis for qSVA, for them to add seed capabilities to several functions. We will need to meet several more times to complete the whole help request","they mentioned that they had to login everytime she pushed a commit from the directory in question (but not other repos). I thought we should start by using a github access token to update her login to github from JHPCE. The best way I know how to do this is to use gitcreds::gitcreds_set() in R. At first it gave us an error, which was solved by adding the git modules loaded in the team bashrc to theur basrc file as well as their .gitconfig file. After that we were able to save the token to their JHPCE setup and the github push issue seemed to be solved","I described the 'add_coords' function, how features link to existing coordinates, and that coords must have at least one associated feature to display","I referred her to recent versions of my code (spatialDLPFC and SPG_AD projects) and walked through the high-level steps of each","Reviewed input for tensorqtl (it is counts, not logcounts), and revised what how we filter lowly expressed exon-exon junctions using http://research.libd.org/jaffelab/reference/expression_cutoff.html.","Discussed how to best pseudobulk the data prior to doing this type of analysis. In particular, how to adapt https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148 and related code, as well as plots to make such as boxplots of the min_ncells by cell type (without any filtering) and cutoffs found by `isOutlier()` (different cutoff for each cell type) + PCA plots colored by min_ncells.","Discussed a few different areas of growth, including a change in direction related to our outreach activities.","I showed that the error was essentially a failure to allocate memory and recommended doubling required memory for the failing SPEAQeasy process until success","I described that an entire directory (rather than all of its contents) can be ignored, and should be specified relative to the directory containing the .gitignore","We looked at some old code vs. an adapted version, which was not producing the same output. I recommended splitting the code into ordered sections, and executing the old and new version of each to identify exactly where the error was occurring","I discovered she was using the --small_test argument and not --input, so only the small 3-sample test set was running instead of her actual data","I walked through the meaning of certain information-dense plots I made to show spot-deconvolution results","I recommended filtering rows of the RSE via 'seqnames', which would return genes (or features in general) of the particular chromosomes needed","She asked about any quality-check-related options SPEAQeasy had (I pointed her to '--ercc' for applicable data) and whether trimming defaults were reasonable for her data","I described the general workflow needed to start from raw images and spaceranger outputs, and end up with a Samui-compatible directory that can be dragged into the browser (locally) to view Visium data (including segmented fluorescence channels)","I suggested creating a template configuration file, then looping through her data, using the sed command to write configuration files with the relevant variable replaced from the template","She wanted to add the columns associated with unique rows in one data frame to the other, so I suggested combining the 'unique()' command with 'left_join()'","I suggested using regular expressions in R to get the new filename from a given old filename, then file.copy to perform a safe test, and finally deleting the old files","She was unable to push to GitHub on her laptop. It appeared she didn't have an SSH keypair set up with GitHub, so we set that up to fix the issue","The researcher was concerned about the large number of variables in the reproducibility DE experiment, specifically the PC. I pointed out that the point of the anlaysis was to test the effectiveness of the models (with and without qSVs) so while this might be a valid concern in a sperate DE analysis, we are testing model quality here. I also noticed that the reproducibility anlaysis was not using functions from the qsvaR package, and that as qsvaR is the \"product\" of this whole project its important we use the functions in this anlysis .","I described typical expected alignment rates, the difference between concordant and overall map rate, and details about other SPEAQeasy output metrics","I explained that at the time, I just created the RMD file and added any referenced images to the rstatsclubsource repo, following naming conventions that include the post date and hyphens between words in the blog title. I believe Leo did some manual work to get it fully published, so I referred to the full guide in case my method was incomplete: http://research.libd.org/rstatsclub/2018/03/09/contributing-to-the-libd-rstats-club/","I explained how cell2location is agnostic to what gene identifiers are used, but that gene indentifier type must match between spatial and reference single-cell data. I mentioned the org.Hs.eg.db package.","I described when I make a module for python-based tools, the benefits of conda vs. python virtual environments, and related topics","I suspected improper configuration of email associated with her git commits, which was indeed the problem. Her commits moving forward are associated with her GitHub account, but we couldn't figure out how to retroactively regain credit for commits done with the wrong email","I described 'git pull' to merge in remote changes, and using the 'vi' editor to write a merge message","I discussed with Renee the way I code and showed her some examples and gave her a few coding advices.","Looks like user was trying to create a plot and import data from a .csv file but was getting an error importing the csv\nhttps://community.rstudio.com/t/error-creating-a-windrose/164132","I explained which paths I'd need to get Samui running on our data, and we discussed which features might be useful to show","Project links reviewed: http://research.libd.org/smokingMouse/ and http://research.libd.org/smokingMouse/articles/smokingMouse.html","I described how ACLs are properly set up at dcs05 to inherit downward automatically, so just an 'rsync'/ move to trash is necessary in those transfers","I explained the purpose of Git-LFS, that it's configured for certain repositories, and that we can load a module to use it","Website tools to look for public rna-seq samples & packages to make synthetic replicates [as shown here](https://support.bioconductor.org/p/9150681/#9150698).","I described my use of XQuartz, the 'xhost +local:' command, and the proper ordering of these tools in combination with SSHing to allow plotting to work","We discussed the use of qsvaR. For now the qsvaR only use the Gencode25 annotation. We also commented on the steps needed in case we want to implement the package for the more updated annotation versions in the future.","I described what we use modules for, the general approach to making them for common software types (binaries, python packages), and walked through the process by making a new FastQC module","We went over the code I previoulsy wrote for tensorQTL, and some of the custom functions for preparing the input data. How to input the different datatypes in to python, and run the function, and strategies  for running the different cell types in parralell on the GPU queue","We discussed each step of the pipeline and the outputs. We also had a look at the QC metrics and the code that Cynthia added to the script.","I helped decipher an error message they got in the BioC check in the GitHub Actions involving a local path harcoded in to one of her tests. We also debugged for similar issues by searching her code for other local paths that would cause her code  to break when run remotely","We reviewed the overflow to analyse RNA-Seq and ATAC data with the Seurat and Signac tools. I provided learning resources, such as data file organization, funtions and methos available in the tools. I also provided a complete explanation about QC for preprocessing data integrated.","I helped him set up git on his laptop (username + email, SSH key), showed how to edit R Markdown files using RStudio, and basic version control (git add, commit, and push).","The error message was cryptic, but made more clear in the latest SPEAQeasy: the transcripts FASTA mismatched with the GTF (custom annotation was used) due to a typo in the gene ID in the GTF.","We went through the  https://github.com/HediaTnani/Int_API_R. Then we saw other API examples with authentication (github, ....).","Issue and the solution found here https://support.bioconductor.org/p/9150363/#9150404","user had question about generating rarefaction curves in R. Issue found here https://support.bioconductor.org/p/9150366/#9150386","I shared the tensorQTL example I presented at rstats club. We also went over the different in-puts and formats, including some we can re-use from bulk MDDseq. WE also discusses the differnt setting for tensorQTL and how to adapt sh files to submit to the GPU queue","I initially mistakenly suggested kallisto, but later clarified that using featureCounts with a custom single-gene GTF from the BAM files (produced by SPEAQeasy) would be the proper way to quickly quantify a single gene","See https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099 for more. Led to https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6 and related commits. spatialLIBD version 1.11.10 now supports spaceranger version 2023.0208.0 output files.","We used https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection and also went over the JHPCE and R config files from the team website. We also talked about the DSgs and how it would be beneficial for Kinnary to meet 1 on 1 with all DSgs guides to build familiarity and make it easier to ask for help in the future.","See https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179 for more details. Heena is working with spaceranger output files from 2 different versions, and in particular, version 2023.0208.0 (internal 10x Genomics version) changed their output files structure","Described what we do, the DSgs system, potential for visitors at our team meeting, and research ideas.","We discussed how to merge two datasets in R, making sure they shared the exact set of genes","With Cynthia we saw the differents folders that are part of the template project repository and how to organize your project repository.","The solution involved asking BitSupport to login as a specific user and run https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example. Knowing the exact file paths that triggered the permission issues are important to help check with `ls -lh` and `nfs4_getfacl` under a `qrsh` session.","See the [Bioconductor Slack message](https://community-bioc.slack.com/archives/CR1BLV821/p1677783580326559) I wrote for more details.","End result: https://twitter.com/HeenaDivecha/status/1631360771342098433","See https://github.com/drighelli/SpatialExperiment/issues/135 and the pull request at https://github.com/drighelli/SpatialExperiment/pull/136 that resolves this issue.","I described the need to 'git add' both the old filenames and new filenames, and the use of 'git add --all' to reference deleted files.","I recommended code for matching two data frames up by common genes, and explained that matrices don't support multiple data types (instead a 'data.frame' might be used)","Researcher was having some issues with functions from DeconvoBuddies, namley plot_marker_express. I am going to check that on my end as it seems like the computer wasn't finding the support functions for the main function. They were also having trouble matching the gene names between the sce and bulk rse objects, the ensembl names seemd to have differnt versions, I advided to maybe switch to using the gene symbols to keep things simple.","No public result just yet.","We went over the ratioinal behind psuedobulking and how we can use heatmaps to show marker experssion. Created psuedobulk data for three different clustering methods, and went over the basic process of making heatmaps. Bukola will work on the aestetics of the heatmaps, and cerate one for all thee clustering methods.","related to my meeting with Uma on 2/10. We fixed Boyi's bashrc to allow group write by deafult, fixing the ,git issue and eliminating the need to re-run the permission update script often","I showed how glob expressions can be used to conveniently match many files across potentially several files, and described gitignore-specific rules, such as certain wildcard expression matching files no matter what directory they are in","Researcher was having file permission issues, the group permissions for several files in a shared project were set to read only. This was causing issues with the team using scripts but also in making git commits as some of the idex files in .git were read-only. We updated her bashrc files to contain \"umask u=rwx,g=rwx,o= ## equivalent to umask 007\" to prevent this in the future and ran the premissoin update script.","Researcher hads run muliple clustering methods on the habenula data, I advised to pick the two that were the most simmlar to previous annotations (we were happy with) via rand index, and start examining marker gene expression.","I went over details like naming the OneDrive folder and correctly using CrossReference to refer to that name in figures and references to those figures","I helped port several plots into Illustrator as a figure, describing our normal process of using PDFs and linked files to maintain resolution.","I showed how to filter for individuals with a certain condition, and subset two data frames with the same criteria.","She was unable to clone private repositories due to a problem with her SSH key with GitHub. We solved the problem by setting up a new key pair using the recommended Ed25519 algorithm instead of the deprecated RSA algorithm.","We focused mostly on the options Nextflow provides for managing software dependencies: Lmod modules, conda environments, and docker + singularity containers and the benefits of each potential approach","We discussed how to implement github actions with Nextflow workflow and the use of Conda and/or Docker containers.","We identifed that SPEAQeasy cannot handle runs with only one sample, but that the intention was to use multiple samples (the manifest did not use unique sample names)","The researcher created a github account and I talked them thuogth the basics of git, But we hit a snag with the instalation of git on their laptop, git is built in to xcode in the ventura OS (which her machine is running) but she had some trouble with that instalation too. They're going to try and fix that independtly, and reach out if she needs more help gettign off the ground with git + github","We adapted code from the DLPFC work and https://www.stephaniehicks.com/biocdemo/articles/Demo.html to run GLM-PCA to prep for batch correction","We went over several concepts in Nextflow (e.g. processes and channels) and discussed how to invoke R scripts inside processes.","We (Me + Nick) discussed with Sean the basics of Nextflow and Nextflow DSL2 and how to make processes and channels and workflows. We gave him resources to read  and gave him small examplesto follow. We helped him also with his errors on slack.","We adapted QC code for the habenula data","See [Slack notes](https://jhu-genomics.slack.com/archives/C0498763CUC/p1673451228034439) for further details. They involved https://github.com/LieberInstitute/template_project, https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h, and other specific instructions for setting up a project.","Continuing work with the single cell mouse data, I pointed them towards scater::plotExpression to plot some of the genes her group is intested in observing by cell type. However we ran in to some bugs with running logNormCounts which we think is from an old version of R. I advised to update R and re-install the packages, hopefull that will fix the issue.","We uncovered the source of some unwanted NA values in some genotyping data and eliminated it by removing some code","I helped them build a single cell experiment object with expression and metadata from an publication. Hopefully they can use this to independently explore the gene expression the group is interested in.","We talked about how to compute KL divergence between 2 groups of cell-type proportions on her data using a variation of my code","Reasearcher wanted some background information about the deconvolution process we applied to the degradation data. I explained the mean ratio marker finding, the resulting stats, and how we used them to build our set of marker genes. Then the script for how we ran bisque + the output of bisque .","Researcher wants to test if a gene of intrest to their group is associaited with a ceratain cell type. They are working with bulk mouse data. I suggested that deconvolution may not be able to answer the innital question but can provide depth to their bulk analysis. I suggested that they find a mouse single cell dataset that is relevant to their current data set, then first observe their gene of interest their, see if that answers any questions. Then they could use that data to run deconvioltion with bisque","I described the manual process for resolving a merge conflict with git when git cannot automatically merge two branches","We pinpointed an error message as being due to trying to find the 'colData' of a list containing Summarized Experiments rather than calling 'colData' on the SummarizedExperiment itself","I described how to color ggplot2 plots by a continuous variable, adjust the palette, and other ggplot-related syntax/tricks","We discussed how PCA reduces the dimesions of our data and helps show the variance between samples. I helped strategize how to make PCA plots for the habenula data. Start plotting the first 5 PCs pairwise, color by Dx. Then explore key QC or demographic variables, see if we can indentify any patterns. I was unsure if this should repeat this for exon, jx, tx, as we didnt for MDDseq. But figured gene was a good place to start and if the other features should be explored too, the code should be easily adapatabe.","I gave them a breif overview of the goals and process of deconvolution, including mean ratio marker finding. I linked my deconvolution slides. We then went over the code for running bisque on the qSVA dataset, including details of the input, refrence, and output data.","I recommended some data-preprocessing steps for later downstream manipulation of the data","I gave general advice for how to preprocess some data, including changing data types, transposing matrices, and dropping columns to format things for a collaborator","I described 'facet_wrap' and showed that for some situations, 'plot_grid' with a single output PDF (with width and height scaled appropriately) does well","We generally discussed linear mixed effects models and some common terms used as fixed effects in genomic data","Researcher is trying to run glmPCA on the very large MDD snRNA-seq dataset. They has a run submitted without parallelization that has been runing for 3 days. To try and speed things up she has been trying to use BioCparrallel, via MulticoreParam. But the scheduler keeps killing the job. We checked that -pe local X and MulticoreParam(X) match , so it wasn't that. I recomended to try to run a subset of the data to get a sense of how glmPCA uses memory. But couldn't come up with a fix to make the parallelization work. As its PCA it also can't be run as subsets, so there aren't other workarounds :( I think BioCparallel + sge scheduling would be a good topic for Rstats club as this seems to be a common headache","https://lcolladotor.github.io/bioc_team_ds for my team website, https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing for the LIBD rstats club schedule, https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials could be of interest also, https://github.com/LieberInstitute/template_project for a template for organizing projects, learn about the DSgs https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs, https://rstudio.github.io/reticulate/ for using Python from R","I described how to transpose matrices, change their type, drop columns, and related tricks to solve a data-formatting problem asked by a collaborator","Reseacher walked me through some plots they need for the LS paper: marker genes heatmap and expression violin plots. I pointed them to some code from DLPFC sn that should be helpful (they needs access to this repo). They also needs to get setup with JHPCE on their new laptop, I pointed them to the team laptop setup docs. They are also new to ggplot, so I pointed them to the visulization chapeter of R for data scientist. This will probaly require some follow up meetings","Researcher has added a new color scheme to the heatmaps that makes the differnces between positive and negative more clear, how ever I think the darkest red being close to 0 is misleading, so they're going to fix that. I also recomended to simplify the annotaions on the combined heatmap so its easier to find and interpret patterns.","We reviewed the reasons or correcting counts and why thats important for DE analysis. We went over the math behind TMM, and then how thoes ideas are used in edgeR. I pointed them to a stats quest video with a more detatiled explination of how the edgeR correction works.","Researcher showed me the csv files that he exported from different moduled on HALO. They seem to have the same rows, so copying the needed column in to a differnt csv is easy in excel (where they are working) but I showed that how to check that things are really aligned with some excel functions. We also looked at the data in prism, which I had to quickly learned about. I advised to should show the data in boxplots not as a barplot to better display that data, and went over the checks for running an anova.","I mentioned rtracklayer::liftOver and showed a code example of lifting over genomic coordinates for a SummarizedExperiment","Researcher showed me their dot plots with the correct 0.05 q val filter, they looked good, ready for discussion. One tweak , is they had a combined up + down expresion col for some plots which was redundant and should be removed. They had also made some heatmaps that didn;t show very much difference between case control, the data probaly needs to be normalized by the genes overall expression, but I need to check prevous work to remeber how to best do that","We went throught boxplots Bukola had made for the data, I suggested she add geom_jitter and how to make that work with geom_repel to help show more details about the data. We also discussed direct vs. standard evaluation of variable and how that makes writing funcitons with tidyverse tricky, but how she can use aes_() to do stadard eval with ggplot","I walked through our process for buld data QC, which involves making many exploratory plots of QC metrics and know groups (like batch, or case/control) to understand patterns and identify problematics samples. Then later implementing cutoff values either based on stats (like a 3 MAD cut off) or knowlege of ressonable values. They will spend some time plotting QC values, adapting code from smoking mouse, then we will meet again to interpret these plots.","We went over the dot plots she made for GO analysis from the multiple DE test she ran. The plots look good, but we identified that they needed to use the default q cutoff to get useful results. She will rerun the GO analysis and replot the results.","We adapted my_plotExpression() (a function I wrote) to work for Bukola's data. This required some debugging as subsetting to just the one gene in question altered how some internal dataframes looked. I also gave Bukola some pointers about saving plots using ggplot.","Sang Ho walked me through an analysis they want to do for the AD project, it involves annotating HALO data. We talked thought a strategy, this may be a bit computationally tricky to implement but it seems doable.","We talked about https://github.com/LieberInstitute/template_project, https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/, http://research.libd.org/sgejobs/reference/job_single.html, https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7, https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1, https://happygitwithr.com/, https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.","I described the need for SSH keys between the login and compute nodes at JHPCE, and how an incorrect set-up will manifest as a prompt for your password when using 'qrsh'. I referenced https://jhpce.jhu.edu/knowledge-base/authentication/ssh-key-setup/","We debugged her SSH set-up with JHCPE, finding a syntax mistake, confirming the contents of the '.ssh/config' file, and other details to ultimately get the traditional RStudio + JHPCE set-up working smoothly. We touched a bit on the process for using git from JHPCE but didn't finish fully setting things up","LIBD rstats club schedule https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing. Bioconductor overview video https://youtu.be/c0Ch_sXiGDQ and related notes in case you want to skim them https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing (we typically have this for each video entry). R and RStudio intro https://youtu.be/XTuJ8vGnzBU. Example vignette https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html from https://bioconductor.org/packages/spatialLIBD/. Data Science guidance sessions https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0. JHPCE new user request https://jhpce.jhu.edu/register/user/. Making your personal website with postcards https://youtu.be/9LUCZAx3SlU. Examples include https://lahuuki.github.io/ by Louise, https://jtleek.com/, http://aejaffe.com/.","I described how 'module load conda_R' lines in a submitted shell script override the default version of R that can be loaded by default using the '.bashrc'. In this case this undesired interaction caused a corrupted R package to load and return an error","We adapted someone else's code, slightly changing some awk commands to use cut (which I'm more familiar with) and using a for loop rather than piping a file to a while loop. This allowed the code to work as intended on the data","I pointed to installation guidelines for the cluster to resolve issues about R packages not being installed and a problematic nextflow version.",null,"Related to how we organize files for a publication.","Density is related to the probability density function, see [wiki](https://en.wikipedia.org/wiki/Kernel_density_estimation for) more details. [This link](https://stats.stackexchange.com/questions/4220/can-a-probability-distribution-value-exceeding-1-be-ok) is also useful for understanding the y-axis in more detail. Violin plots are just density plots rotated 90 degrees and mirrored. They are compared against boxplots frequently, which while easier to understand, are not good for non-unimodal data. See [this link](https://mode.com/blog/violin-plot-examples/#:~:text=What%20is%20a%20violin%20plot%3F,the%20density%20of%20each%20variable) for more. I like `plotExplanatoryVariables()` because it's fast to run and gives a general idea of things. See [Figure 1b](https://bioconductor.org/packages/release/bioc/vignettes/variancePartition/inst/doc/variancePartition.pdf) from `variancePartition` for a violin plot of similar data. `variancePartition` can handle mixed effects models (random effects) and in that scenario is more precise, but will also take longer to compute.","I described how to smoothly 'git pull' when updates have been made to both the local and remote repo (adding + committing local changes, running 'git pull', and handling a smooth merge)","I described proper saving and loading of SummarizedExperiment derivatives whose assays are HDF5-backed using the HDF5Array package, and how to convert between HDF5-backed objects and \"ordinary\" in-memory objects","I went over the DE pipeline we use, noting that the model development is the most demanding and non-static part. We talked specifics for their dataset which has some experimental design to consider as its organids and is diffrent from patient case control. I posinted Geo to scripts where we do QC, expression cutoff, and my run_DE script","Heena saw old commits pop up on slacks git notifications when she pushed commits she had locally on her laptop, but they didn't reappear in the git history. I think she did everything right and there might be something strange with the behavior of the slack message (as thats the only place things seemd wrong). We did some detective work between the files and the commits, everything seems good. Heena was able to then pull to the cluster with out conflicts","Had to work over slack again as Daianna was having trouble with zoom. Working with the feedback from the previous meetings, the experimetn has been narrowed down to 5 DE runs, and then comparing some pairs of the outputs. We were unsure if we should include mitoRate as a variable in the adult model as it correlates highly with totalAssignedGene, but decided to include it for now to match the model used in the pup DEA. I also sent her some code I have used to make scatter plots of t-stats to help compare DE results.","Check [EnchancedVolcano](https://bioconductor.org/packages/devel/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html) which shows a quick DE guide and finishes with some Volcano plots, which can be good enough for a grant application. If you want to use limma, you could follow [the SPEAQeasy example](http://research.libd.org/SPEAQeasy-example/de_analysis_speaqeasy.html#16_Modeling) or [our DE bootcamp](http://research.libd.org/SPEAQeasy-example/bootcamp_intro#5_Day_3) a bit. Or dive deeper into [limma's documentation](http://bioconductor.org/packages/release/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html).","We intended to meet on zoom, but Daianna's connection was bad so we used Slack to chat. Daianna has made many plots and visuals about the many DE analysis for smoking mouse. I gave her some feedback about orgaizing her slides so the figures can practically be used inform DE models, right now everything is a bit scattered as shes planning on running 7 analysis so there is a lot to consider. We will need to meet again to finish this.","wrong input was given to the function",null,"should select virtual stack option in bioformats reader when opening and multichannel image in imagej",null,"Daianna showed me her varianceParition plots, and we talked bout how to design the DE anlaysis based on what she's oberved in the mouse data. I advised to keep things simpler (ie not create too many groups) but rather use models to control for vaiables adding variance. We will meet next to talk model design for this work.","Explain how to pick randomn images to train PNN segmentation algorithm","We walked through the code to make a pseudobulked object and plotting the explanatory variables.","Note that `spe$var_colors` works for `spe$var` where the `var` part has to be the same in both variable names. Also `spe$var_colors` should be a named vector such that `unique(names(spe$var_colors))` matches `unique(spe$var)`.","code loops over the different file in the dataset and Marah was clearing it at the end of the loop every time which didnt let her save any data","We worked through a few issues in the cellpose module, which each had a different cause. In one case, the version of torch/CUDA was outdated for the hardware. In another case, user-specific libraries were being referenced instead of the central module's libraries.","Daianna shared her updated QC plots, adn the one sample she planned to drop based on fastCellQC. I advised that she dig deeper in to how that QC finstion works, and what the cutoff values are, as well as exmaming mito vs. ribo precentages, and a two samples that don't seem to group with others of the same phenotype /","NA","The error messages involved a mismatch between sample ID and file name for the 'QualityTrimmed' process, as well as a supposed missing ID in the manifest (which was actually present). Simply resuming the pipeline got rid of both errors.","NA","I explained how to get pheatmap to use the same color scheme for column and row annotations by making the annotaion tables have the same colname, and suppying a color pallet","We discussed a strategy to subset the 102 loci in question, and adapted code to subset the FDR signif pairs to work with the location info.","I described how to configure SPEAQeasy for a particular dataset, and how to manage two separate runs of SPEAQeasy as part of a different git repository (requiring the use of submodules)","NA","Daianna shared the updates she made to the QC plots based on our previous discussion, specifically separating brain and blood samples.","I showed the documentation's description of 'mitoRate' and linked to the code where it's computed: https://github.com/LieberInstitute/SPEAQeasy/blob/bfe4e664281b52c5e239e372393a8aea36084e0d/scripts/create_count_objects.R#L521-L533","She was getting a memory error when running BayesSpace and I ended up advising her to reduce the nrep argument. Also advised her on running fasthplus because it was taking too long. She needed to reduce the proportion of the data being used.","I described the need to manually convert some spatialExperiment slots prior to conversion with zellkonverter, and that some data types are not supported. I pointed to my SpaGCN code and the lines specifically responsible for clustering + writing outputs","Daianna walked me through her progress in the DE analysis. We went over QC plots and discussed experimental design. She will check in with Kristen about how to organize the brain vs. blood samples. Currently she dropped the blood sampeles as they looked very different in several QC metrics. But I advised that they just be handled seperatly as it makes senese they look different as they are different tissue, and the signal is probaly biological rather than technical","I described when 'git pull' can be \"safely\" used, and when 'git reset' might have to be used beforehand",null,"I described the use of 'remotes::install_github' for GitHub-based packages. We encountered a GitHub authentication error that was solved by running the R code 'Sys.unsetenv(\"GITHUB_PAT\")' prior to the normal installation command.","she needed to have the plots as single ggplots instead of a list so I advised her to used vis_clus() instead. Shared code with her that I had used to make some of supplemental figures. She also asked if she need to discard spots before doing batch correction and I told her yes.","We fixed Sanjana's ssh set up and got the qrsh command working. We got git configurated for her locally and on JHPCE, as well as set up tokens. Created a test repo, and made commits from her laptop and JHPCE as an example","checked size of the spe object. checked that only has low res image files. check dim of colData. All sizes checkedout. Believe this to be a package problem.","Maddy asked me how to get the total UMI's for each sample out of her spe object. I wrote a small for loop for her in order to do it.","We went over github basics and motivation behind version control. Quickly discovered that Sanjana's qrsh command is not working, requiring a password. Ended up using the session to troubleshoot that, with no success :( planning on circling back/re-setting up ssh for her. Will meet later about git setup","We checked Sreya's code against the OSCA DE example run. She was worried that there weren't may significant genes, but her results seemed to be in the same range (10s of genes) as the example. We discussed how maybe there were QC or demographic variables that could have been added to the model, but the data she has doesn't have much to work with. We will revisit when I have maybe done more work with sn DE analysis.",null,"Talked about recount2, recount3, snaptron, snapcount, recount.bwtool, recount-brain, recount predictions, megadepth, the recount3 study explorer. I also sent them all the links to them, though you can find many of them at https://jhubiostatistics.shinyapps.io/recount/, https://jhubiostatistics.shinyapps.io/recount-brain/, and http://rna.recount.bio/.","I pointed Nick to a few scripts used in marker finding for the Tran Maynard dataset, using a few functions from DeconvoBuddies. For marker heat maps we went over how I hace built annotation tables, and a few tricks for optimizing pheatmaps for maker plots.","We converted a downloaded dataset from tsv files to a sce object. Went over a strategy for performing DE on single cell data, using OSCA as a reference","We spent the session getting Sanjana set up better to work on JHPCE, setting up ssh keys, cyberduck, RStudio, and config files. Plan to meet again to set up git + start single cell work","This had already been fixed, but Abby had an older version of spatialLIBD installed. We also went over some other steps needed to deploy an `spatialLIBD` app with pseudo-bulked data and modeling results.","Resulted in a fix for [this issue](https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1137544893).","Assessed results of BayesSpace joint clustering and suggested that Alice discard the spots with low library sizes to improve clustering. Offered to create small SPE object for her with just the for samples she will be doing long read sequencing on.","Brainstorming and planning session.","Resolved via email.","created new script to subset spe object by removing some of the images.  updated app.R to load smaller spe object (half the size) and re-ran shiny app locally. Explained how to version control from local computer and fixed problem with version controlling large data file.  Still having same problem of not being able to view shiny app online","Review of statistical models as well as access to data","We both leaned how to use the image function to create a annotation bar for heatmaps based on a custom color pallet","Twitter, LinkedIn, jobRxiv","Tried to troubleshoot adding visium IF images to spe object","Helped setup OneDrive folder for LC manuscript. Explained how to organize figure folders.","Clarified what the data is showing and explained a bit the goals of the LIBD data portal that Geo Pertea is building.",null,"Helped fix error of shiny app not being able to find spe object. Helped sucessfully deploy app.","We went over how to create a new type of refrence (Supp Table/Supp Figure). Also walked though the full process of creating a label and then refrencing it in from the text + considreations for formatting","PhD programs and other options.","We went thourgh some plotting code Sreya had adapted to make a manhattan plot, and went over which aestetics to change to get the desired plot","I described the two JHPCE-based repositories we push to (https://github.com/LieberInstitute/jhpce_mod_source and https://github.com/LieberInstitute/jhpce_module_config), my usual installation approach for python-based GPU-utilizing software, and the contents of the lua-based config enabling a module to load.","See [this Twitter thread](https://twitter.com/lahuuki/status/1521581339903873024) for the result.","Provided guidance on how to share images and how to organize supplementary files. Resulting in this [Figshare](https://doi.org/10.6084/m9.figshare.19729567.v1) zip file.","I clarified the usage of nextflow channels, described when docker integration is likely to be worthwhile, and answered other general questions about processing data with a nextflow-based workflow","NA","I helped come up with a stratagedy to loop over 100+ files for an analysis and exreact a portion of the results in to one R object","NA","Helped explain the different parts of the html output of spaceranger and which fields are important to check for QC","NA","NA","See https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012.","Clarified a few questions about LIBD's SPEAQeasy RSE objects and a problem with some colData() that was missing, that we'll provide early next week. Send links to our deconvolution work with Louise A Huuki-Myers.","If the situation had been simple, that is, that the latest commit was the one that commited the large files, we could have used the solution described at http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/. Instead, we had to use https://rtyley.github.io/bfg-repo-cleaner/.","I helped Heena address an error she was getting when trying to run read10XVisiumWrapper(). We noted an issue with the function and submitted an github issue to spatialLIBD. We got the function to work but only if you provide the path to the gtf file. Met for 1hr.","JHPCE ssh keys (to avoid typing passwords when using qrsh), .Rprofile (both), .bashrc, .zshrc (laptop), iTerm2 (laptop), Textmate (laptop), remote port fowading for rmate, xquartz (laptop) installation and configuration. There are several LIBD rstats club videos on these topics already, but we just did all the setup fast. In particular, the .bashrc configuration was important to resolve a umask setting problem that was interfering with the permissions on the Visium IF AD project.","We installed and configured Cross Refrence, then went over how to use the lable vs. refrence links + pro tips on how to avoid errors","spot size and scaling factor from json file. explained how each metric is associated to different version of the histology images.","Customized VistoSeg code to their histology images",null,"See [this Slack thread](https://neuralplasticity.slack.com/archives/C010NFSQS48/p1649249130880349). Ultimately, this situation involved teaching Tony to share his code.","We want over some plotting basics with ggplot2, and how to use geom_rect to recreate some spatial plots that a halo module produces in R.","https://github.com/LieberInstitute/VistoSeg/issues/11","Able to identify situation where duplicate.correlation should be used and how block data is important to linear modeling. We then looked at how top table can be used to summarize DE outputs.","modified code to work witth her dataset. did graph based clustering and plottiing. Explained benefits of clustering across samples. Advised her to do isoform calling on pseudobulked data for multiple samples. Met for 1hr.","After identifying that 'redstone' was a precompiled binary for Linux, I described how to use 'wget' to download software like this, set execute permissions if necessary, and run at JHPCE.","Finished installing packages. Set up sample_info table. Walked her through the rest of the code.","I recommended some adjustments to a custom GTF and transcripts FASTA to solve an error seen in SPEAQeasy where transcripts and annotation were found to be incompatible.","We applied some test I used for the TREG package to help test more parts of the functions, like checking for errors w/ bad input, warning messages, and qualities of outputs. We discussed possible ways to use testthat on a ggplot, but that might need more thought.","Showed her app.R, deploy.R scripts and places where to edit, showed her how to navigate through the shinyapp portal to looks for logs when deploying the app. EXplained her few erros she ran through when deploying for the 1st time.","We set up a git repo and added code to be adapted to build an spe object from Alice's long read data.","I explained how the first time that you ssh into a new computer (which happens also with git pull/git push), you need to accept that the new host is authentic, which will then edit your ~/.ssh/known_hosts file.","This was a preliminary meeting with Alice to decide where to begin with analyzing her spatial data. We decided to start by generating an spe object with her long read data.","An error occurred in a SPEAQeasy run, which I identified to be due to not providing all the required inputs (specifically the sites at which to perform variant calling)","This is challenging since it involves putting together code from multiple scripts/projects. We summarized the steps to do [on this GitHub issue thread](https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1069493294).","We both verified additional transcript annotation added to a GENCODE GTF looked good for use as input to SPEAQeasy","Louise and I discussed the [Dentate Gyrus, Jaffe at al, 2020](https://doi.org/10.1038/s41593-020-0604-z) paper which used LCM-seq, what deconvolution using Bisque can do, and what role it can play at different analytical stages.","TODO","An error occurred in the counts step, which I discovered was a bug in some filename-matching code","We had some trouble logging in because the WiFi wasn't working without logging in. We solved this by using an ethernet cable at LIBD (which is part of the JHU network). Once we logged in once, then we could use the WiFi.","I described two-factor authentication vs. passwordless login via SSH keys, and gave tips to set up the latter.","The contrast adjusted images on MAC::preview should be saved and uploaded back to JHPCE to run spaceranger. Spaceranger needs the exact same image that was used in the loupe alignment (even if it is changed from raw image on preview)","I described how to add additional transcripts to the relevant FASTA file then use SPEAQeasy's custom annotation option to use the new FASTA with the other default annotation files.","Explained how the image data can be manipulated when claudin5 channel and WFA channel are merged and then the blood vessels are segmented and masked. The correct way is to segment blood vessels from claudin channel and mask the segmented objects in WFA channel.","The indices listed in the tiles's filename is the x,y coordinates of the starting topleft pixel of the given tile. These indices never start with [0,0] as these tiles are picked from the ROI in the phenochart and not the entire imaged area. Once stored in a directory these tiles are not arranged in chronological order.","I described how requesting 'caracol' is similar to requesting 'bluejay', but elaborated on additional complications, like contention for the same GPU and using CUDA_VISIBLE_DEVICES properly, including with array jobs","I described different strateguies for helping others navigate the world of data science. In particular, how weekly coding sessions can be really useful.","We went over the files (data and script) for the deconvolution analysis done for the habenula in BSPIII. We also discussed the theroy/methodology for deconvolution.","Created an R script that reads in the transcripts FASTA file from GENCODE and uses BSgenome from Bioconductor to edit it and save it as a larger .fasta.gz file. Nick Eagles helped us by pointing us to http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation.","Linked to https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13, `/dcs04/lieber/lcolladotor/qSVA_LIBD3080/degradation_experiments/*/bed`, https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855, https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5, https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R, https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R, https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253, https://github.com/CRG-Barcelona/bwtool/wiki#installation, http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html, and http://research.libd.org/qsvaR/. Andrew Jaffe linked `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/degradation_mat_CMC_ribozero.rda`, `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/run_de_analysis_CMC.R`, `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/geneLevel_CMC_qSVsAndMod.rda` and `/users/ajaffe/Lieber/Projects/RNAseq/CMC/`.","Linked to https://github.com/LieberInstitute/qsvaR, http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28. Made public the file `/dcl01/ajaffe/data/lab/qsva_brain/brainseq_phase2_qsv/rdas/degradation_rse_phase2_usingJoint_justFirst.rda` and other ones. See https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5 for details.","Heena was encountering a simple error installing the package. We also spent time going through the vignette to give a conceptual overview of qsva. Her feedback will be helpful in making the vignette more accessible","I referred her to the '--coverage' option with SPEAQeasy, which unfortunately cannot be run alone, which might be useful for an already-processed dataset","Used KJ's code from his rstats club presentation and described how the images shoukd be read using numpy array as uint32 but not floating https://colab.research.google.com/drive/1A4cavWcbgZF1PFQT87s5s_ATyZrN3IxL?usp=sharing","Using BiocMAP http://research.libd.org/BiocMAP/ which Nick is finishing up, you could process the data and have R objects you could load and analyze. So, the data processing would be \"easy\" in a sense. Next, you might want to check the bsseq documentation https://bioconductor.org/packages/bsseq/ if you are interested in finding differentially methylated regions (DMRs) and in any case, because BiocMAP provides that type of R object as noted at http://research.libd.org/BiocMAP/outputs.html. https://jhubiostatistics.shinyapps.io/wgbsExprs/ and the companion paper is the last WGBS data analysis project I did myself with code at https://github.com/LieberInstitute/brain-epigenomics. Kira Perzel-Mandell, who worked with Shizhong, wrote https://github.com/LieberInstitute/wgbs_meqtl_analysis (link to paper there) and did other projects. If you are plotting methylation levels at some locations, then that's very direct. If you are trying to find DMRs, it can take a bit of time to get used to the large objects and the related complications (once you filter the data, it'll be more manageable). If you go into CpH instead of CpG, then memory constraints become harder to deal with.","We discussed how to build and save data with a for loop and with purrr::map. Went over how to define a function in map to do nested loops","Geo was interested int eh workflow I developed for tensorQTL. I went over the requirements and input formats that tensorqtl requirers, and how I had previously made them from our dataset. Also discussed the differnces between the functions and their outputs.","used HALO to show how contrast can be adjusted when changing min max values in the image","NA",null,"Related to JHPCE https://github.com/ntdyjack/fasthplus and http://research.libd.org/sgejobs/",null,"Brainstormed several ideas related to deconvolution of bulk data as well as uses of snRNA-seq. Refered to OSCA https://bioconductor.org/books/release/OSCA/ at time.","We went over how to acess the rowData, and what the rowData inthe MDD project contains. Ended up adding columns to the eqtl risk snp subsets to help with analysis",null,"https://www.nature.com/articles/s41592-021-01358-2",null,null,null,"We tried to discover the source of a seg fault that would occur when she imported tensorqtl in python with the tensorqtl module. Another user and I weren't able to replicate the issue. We later discovered that there were lines in her .bashrc that might've interfered with the module to produce the issue.","I described how to qrsh/sub jobs with the GPU node, including how to automatically set up 'CUDA_VISIBLE_DEVICES' to avoid contention for resources, and how to use the tensorqtl module to ensure GPUs were used when importing pytorch","After ensuring our apparent Rstudio set-ups on our windows machines matched, she still wasn't able to forward X11 while using Rstudio. Regarding a different issue, we discovered that a particular machine required additional lines in the '~/.ssh/config' to properly enable RSA keys.","explained the functions of this pipline as these are not yet documented","I described how it's accessed as a queue, but some manual work is left to the user (checking which GPUs are open and setting 'CUDA_VISIBLE_DEVICES')","command to use `rsync -avzh --progress userid@jhpce-transfer01:/source directory/ /destination directory/`","use rsync, prepend spaces in the paths with `\\` to avoid issues","https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/43106c664883fd5df301e44cca35f2e8f2f289ad","explain how to use IDE's on local machine to edits scripts on JHPCE through cyberduck","Fixed row names to both be ensembl IDs, discussed simplifing cell types in sce, converting sce arrarys to matrix","https://github.com/LieberInstitute/CaPTure/issues/2","I shared some detials of our deconvolution approach, we walked through the SCE to Expression set converson, and discussed some issues that may come up","I described how I normally deduce sample IDs from FASTQ filenames when creating a 'samples.manifest' file for SPEAQeasy, described the difference between '--trim_mode' settings that SPEAQeasy supports, and showed how to check what samples were trimmed.","refining segmentations in matlab by applying size thresholds","Showed code how to scale data based on https://uc-r.github.io/kmeans_clustering and optimise kmeans for the present dataset","Motivated /dcs04/lieber/lcolladotor/_jhpce_org_LIBD001/update_permissions_spatialteam.sh","Showed the steps on one sample HPC image and what outputs to save.","While debugging, its seen that VistoSeg when splitting the raw image missed a part of fiducial frame for capture area D1 of that sample. This happens when the center of the visium slide does not align with the center of the image, which has ti be adjusted while imaging. This image when run through spaceranger provided negative coordinates for the missing fiducial frame part of the image, but MATLAB will not accept negative indices for a image, which was throwing errors when recontructing spot grind in countSpots function.","Showed how to close issues on git with detailed comments of the final results","I described using regular expressions in R to match files in a directory. We encountered a weird case where 'list.files' seemed to return different numbers of matched files when run repeatedly. After ensuring we were in the right directory, the issue went away, but it was unclear what caused the original problem.","made custom Matlab script to batch process her 30 RNAscope images on her local machine","The user directories were cleared during the software updates for LIBD server, emailed bill to create one for me and Heena to use rsync on Neural_Plasiticity. The filenames Heena used for 'source' in rync did not exist at the source directory which was throwing errors.","to pick filename from textfile for a array job","We discussed how to check for duplicate rownames, and stratedgies for variable naming + working with rse objects","We found a R function that helped produce the plot she was looking for","Two main issues were discussed: **1**. After batch correction there is one sample that clusters mostly independently from the other samples. This sample did not stand out in QC (althogh this will be re-examined). There is some evidence that this donor might have been clinacally different from the other donors, and since the batch correction worked well for the other samples, there is the possibility that this is real biologal signal. However since there is only one such sample it would be really hard to confirm this, and the inclusion of this sample would counfound the biologcal question of interest. I advisied to drop this sample post-clustering on the grounds that this donor might be too differnt from the others. **2**. Prelimnary results from DEG by cell type for the three groups find mitocondral genes are the most differntly expressed. They are concerned as MT RNA could be a technical artifact (contamination) this tech isn't the best to explore this. But there is are other findings that cellular metabolism may play a role in this condition. I advisied to keep the DEG results with the MT, add the caveat in the paper that this might not be a perfect experiment for this but it could support real differnces in the cells (plus they already have pretty strict MT QC). Also run a second DEA w/o MT genes to see what other signal exists.","The error came from the names of the clusters just being integers, the names for limma need to be sytactically valid (not just numbers, no spaces, no special characters - basically need to be variable names in R). I advised to simply add a prefix like we did in spatialDLPFC (sp9k1) or even simpler like \"c\" to her cluster names. Maybe we should add a warning to the function about cluster names since this error is pretty hard for users to de-bug and people often have non-sytactucally valid cluster names (especally with cell type names)","Explained some scripts such as /dcs04/lieber/lcolladotor/BrainSEQ_LIBD001/_jhpce_org",null,"Explaination of why an index label M=3 was chosen for refineVNS in section 2.2.","I explained how I get out of the situation where files that are too big get accidentally committed (using 'git log', 'git reset --soft', and other details)","We worked through an error message from the 'VariantsMerge' process, uncovering (arguably) a bug in SPEAQeasy, and I advised how to temporarily circumvent the problem. I also suggested a command for renaming a set of files that initially broke SPEAQeasy's assumptions.",null,"I talked about how Linux does not allow creation of files w/ execution permissions set, regardless of 'umask' settings such as in the '.bashrc'. I also explained how I use branching with git and other tips for managing a repo that others also develop on.","https://github.com/LieberInstitute/dotdotdot/issues/14",null,null,"We downloaded a metadata table from SRA and parsed it through R to programatically create a \"samples.manifest\" file for use in SPEAQeasy, that merged samples that were each split into multiple FASTQ files.",null,null,null,"https://files.slack.com/files-pri/T029RHR07-F024D7ELKC2/image_from_ios.jpg","used back quote instead of single quote when assigning string to variables","We decided to run CellProfiler on a while capture area image of DLPFC sample to look at the segmentations as it was doing good job on the cropped regions of a DLPFC sample","https://github.com/LieberInstitute/dotdotdot/issues/13",null,"SPEAQeasy would appear to run into \"file not found\" errors for just a handful of random samples in a large dataset. I eventually traced the error to a disk mounting issue on a particular compute node, which was resolved through bitsupport.","The H&E image that he was segmenting and the H&E image for which he gets the 'json' and 'csv' are not the same images or at least not the same resolution which is causing the error.","https://github.com/LieberInstitute/dotdotdot/issues/12",null,null,"used the command 'eval'","Did some troubleshooting on nested map functions, when to use ~.x and when to use function() syntax","I mentioned that she encountered a bug that was fixed in a more recent version of SPEAQeasy, but suggested she manually add a line of code temporarily so that she could resume the run before updating","I referred her to a script I had written in the past that contains the required commands and configuration  (with comments) for using the SRA toolkit",null,null,"Went over basic, complex, and nested map() usage and how to apply it to Josh's current work",null,"We were able to update the keys for the cluster, but the ssh key login was still broken",null,null,"We revised key features of Brenda Pardo's Twitter thread https://twitter.com/PardoBree/status/1388253938391175173?s=20 and some features that you would want your Twitter thread to have.","I showed a fairly standard method for concatenating tabular data from multiple files into a single matrix, and saving the resulting R object","We talked about all the details that go into writing an abstract for a conference. These details can take longer to complete than initially anticipated.","We revised how to make a new branch in Git working with https://github.com/lmweber/OSTA-book","We discussed the current DNA genotype pipeline and potential changes others might want.","ibidem","https://github.com/LieberInstitute/dotdotdot/issues/11","From SPEAQeasy, to qSVA, deconvolution, TWAS, etc. It might be best to give a talk about all these topics to whoever is interested at LIBD since we know about all these pieces but others might not.","showed a example of how AWK extracts text lines from a text file","Explain how some scripts in SUFI run and the bash scripts to use in place of python scripts for batch processing","Referred to Vijay's notes",null,null,"I described correct use of the \"--input\" argument, and we also found that many of the input FASTQs were empty, leading to a somewhat vague error message.","In a series of email exchanges we diagnosed subtle errors resulting from bad FASTQ files and unintended options in the main script. Her feedback helped me improve some existing documentation.","I attempted some formatting tricks to eliminate unwanted whitespace but was ultimately unsuccessful. We suspected the LaTeX template required the whitespace to exist. We also tried to fix a bug with the header of a bookdown RMD not rendering as expected. It seemed that re-writing the header (without making any visible changes) fixed the issue, though we weren't sure of the source of the initial problem.","We talked about https://lcolladotor.github.io/bioc_team_ds/ among other topics.","used ggplot",null,null,"We confirmed that an existing VCF was corrupt and found that tabix expected bgzip-compressed VCF input rather than compressed BCF input.",null,"This has changed over time. The most recent setup (as of 2021-08-06) is at https://github.com/lmweber/OSTA-book","I briefly mentioned the `-o` and `-e` arguments, and how several logs will generate in your home directory without those arguments and without `-cwd`, when \"qsub\"ing a job at JHPCE.","We reviewed her code and I gave some pointer on what bugs to look out for with certain errors in ggplot","We talked about the set of videos and resources publicly available to get to learn about R, Bioconductor and these packages. It was a lot of information and I'm sure that we'll have more sessions later on to clarify questions as they arise.","We talked about https://choosealicense.com/, https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository, https://creativecommons.org/choose/.","We discussed the posibility of proposing a class that has a mix of biology lectures and some practical coding sessions. The coding sessions could be based after the CDSB 2021 mini courses series.","used Sys.glob and write.table to write the list to txt file",null,null,"We talked about the public data from Synapse and worked on locating some of it at JHPCE. I explained also how we have used bamtofastq when others don't share their FASTQ files. Basically, we use bamtofastq to reconstruct the original FASTQ files (you might have to append the unmapped FASTQs if they are provided by the original authors) and then run FASTQ-processing tools like our SPEAQeasy pipeline.","cleared up some confusion about filter_snps.R","We met to figure out how to transfer the data for this collaboration, which ended up being quite tricky and involved using an SSH tunnel for rsync. Basically, a command like this one: `rsync -avzh -e 'ssh -o ProxyJump=user@server_somewhere_for_the_tunnel' *_files_of_interest user@final_server:/final_path/`","We discussed the different possible goals and ways to get to them and determined a path forward that relies on the rest of the DSgs system as well as existing LIBD rstats club videos to get started. It'll be supported by a meeting with Leo and others involved in the spatial transcriptomics projects from the R side.","I showed the initial version of the BrainSeq Phase II developmental expression explorer developed by Bill Ulrich and Aaron Goldman. We discussed other possibilities for the collaboration including helping set up SPEAQeasy in their servers.","Pointed her to some oacjages for creating TWAS plots whe was interested in and we walked through some code I used to compare T-stats from different DE models","we talked about how to use the \"--strand\" option, the source of the error that was encountered","we discussed what a bfile is in plink and that made the process much easier.","https://github.com/LieberInstitute/zandiHyde_bipolar_rnaseq/tree/master/dev_twas#bd-twas-workflow","still working on this will touch base again, we are trying to obtain the mds plot for ancestry in the placental data.","the way we had run cluster profiler was generating lists in a way that the plot functions could not work with","N/A","We walked tought script from getRPKM to expression cut to filter rse objects, I also pointed Brenda towards some reasources to get started with ggplot","Used GenomeStudio again to produce PLINK files","We discussed the changes that we should make and the ones we shouldn't based on the feedback received from the practice talk with Keri's group. The feedback was excellent and very useful, however, we discussed which suggestions were maybe complicating the presentation too much and how to best incorporate them.",null,"run clusterprofiler on a set of DE hits",null,"Similar to part 2. The presentation was looking very polished by this meeting!","Helped review the grant application and revise the text.","Further updates on asking for help with this complicated project. Suggested reaching out to other LIBD members who have tried running the software before.","Discussed with Josh what figures we needed to prepare and how to generate them. This also involved base_size = 30 with ggplot2. Otherwise the font was too small to read it on the grant.","Now that a first major draft of the presentation was complete, we revised it and proposed some changes to it. Some of them involved remaking the figures using base_size = 20 or even 30 with ggplot2. For future work, it might be best to save the images with that font size from the beginning instead of having to update them later.","We reviewed the model that Sreya was evaluating for DE. As we as troublshooting the plot functions from the SPEAQeasy bootcamp code. I also advised taht she could create a volcano plot for her results.","We went through the creation of an LIBD JHPCE module, covering the documentation process, lua syntax, and how to test a module before pushing.","We adapted a series of bash scripts requiring parallel functionality for use at JHPCE. This involved correctly setting some path variables and requesting multiple cores through SGE for some scripts.","explained the process of using topmed imputation and later we will get the mds","I described how I typically construct a \"samples.manifest\" file and configure SPEAQeasy for a particular experiment.","We went over some plots we have made for DE previously, (quality control & PC scatter plots, t-stat scatter between models, venn diagrams, and boxplots). Talked about some things to watch out for in her data (small samples size). We are planning to meet again when her data is ready.","We went over the details of the GitHub actions workflow created by biocthis, the parts that you need to run from biocthis::use_bioc_pkg_templates(), and resolving issues prompted by R CMD check + BiocCheck","I gave an overview of the resources and materials we have created as well as the LIBD bootcamps that we have publicly available to help others get started with computational biology. This is likely the first of several meetings.","We spent a significant amount troubleshooting the problem here (nearly two hours?). In the end, using the insert image \"here\" LaTeX syntax resolved the problem.","General advice on how to ask for help from open source projects","Initial planning session for the LIBD seminar. We talked about the different slides we planned to show and the flow of the presentation. We also scheduled 3 practice talks with different groups of people.","We observed an issue where log parsing in SPEAQeasy behaved differently for her and I due to a difference in default python version loaded at JHPCE. I fixed the hisat2/2.2.1 module to ensure consistent python version use.","Overview of the TWAS pipeline","I showed how I typically produce an array job shell script for the situation where I have many samples of data to process the same way (SIMD). We talked about using a text file where each line contains one sample name, and referencing that in the array script.","I showed Sreya how to set up the S-TWAS directory that I adapted in the Zandi-Hyde bipolar project for her own adaptation. I also sent her some useful videos on how to set up GitHub with her JHPCE account and install an SFTP client for Ubuntu (I prefer Filezilla)","We discussed how to handle strandness issues, and how different inferred strand patterns within a group of samples sequenced together can indicate problems with the input data. I described how to use the '--force_strand' option with SPEAQeasy","We went through cloning SPEAQeasy, creating a \"samples.manifest\" file with R, and other relevant details for running SPEAQeasy at JHPCE. I also helped him set up git at JHPCE","We troubleshooted strange errors from the main SPEAQeasy output, and saw that I was able to run the main script (\"run_pipeline_jhpce.sh\") but she encountered errors with an identical script. We suspected permissions issues in her working directory, which she emailed bitsupport about.","I gave Sreya a conceptual introduction to transcriptome imputation and sent her some resources to help her get started on FUSION / TWAS","We got ssh key login working, added config files to JHPCE, went though a \"hello world\" example of using files on JHPCE on RStudio","Although we encountered permissions issues, I showed how to create a \"samples.manifest\" file and configure SPEAQeasy for a particular set of samples.",null,"Henry needed to reset his JHPCE password as he had forgotten it, but we went through RStudio set up and got cyberduck installed, we will set up a meeting next week for ssh keys and configuration","I showed my typical process for creating a \"samples.manifest\" file for a given dataset, and how to submit a pipeline run. We continued to debug some minor issues over slack",null,null,"Created a learning plan involving several team members that is now documented at a channel in JHU Genomics Slack. That way everyone can keep track of what others are teaching the person who asked the question, and we can have a common communication channel among everyone involved. Sessions include learning how to interact with JHPCE through RStudio, how to run SPEAQeasy, key R/Bioconductor packages (limma, SummarizedExperiment), and explore possible visualizations of interest.","I covered various nextflow concepts, as well as my typical approach in creating and running a nextflow project at JHPCE","explaination of nurolucida system, how number of cores and RAM matter for running SUFI",null,"Discussion about features that spatialLIBD could support to make it easier to use for newer data.","Went over basic functions of SummarizedExperiment and SingleCellExperiment, what a Sparse matrix is - how to convert to non-sparse. Steps to take to use getRPKM on SingeCellExperient.","sent him the scripts from andrew, wondering if I should make it into a function","Help with understanding [bookdown](https://bookdown.org/yihui/bookdown/) and ideas on how to organize a documentation website (chapters and information to consider).","Discussed the research project proposal and how we can help.","General advice. Received feedback regarding file backups.","explaination of scale_factors.json, tissuepositions_list.csv from spaceranger and the spotspotcheck package","I identified that https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829 contains both miRNA and RNA-seq data. Using data from the SRA Run selector at https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&o=acc_s%3Aa I was able to filter the data to the RNA-seq samples and subset the input samples.manifest file for SPEAQeasy.","To identify DEGs or eQTLs, the user has to choose what covariates to adjust for. Some studies might require more complicated models than others, as they might be from more than one brain region or present some batch effect. In addition, in newer projects we are considering adjusting for cell type fractions derived from deconvolution. Even if all the covariates were fixed across datasets, in DEGs we have typically adjusted for qSVs which are computed in a dataset-dependent fashion and in essence are PCs derived from degradation-associated genomic regions. Similarly, for eQTLs we adjust for expression PCs. In both we adjust for ancestry PCs. Computing PCs is input-dependent: you won't get the same values or PCs if you change which samples you are using. So, computing DEGs and eQTLs is not as simple as say a map-reduce problem where you compute the same thing for every input sample, then aggregate the per-sample results. We have steps that depend on the whole sample set. Then there's also steps for filtering expression features based on some minimum expression cutoffs. Plus, differences between say polyA and RiboZero data, which don't make sense to process together. For these reasons, I don't think that an on-demand computation pipeline is feasible based on a user selecting a set of input samples. Constructing RNA-seq objects (RangedSummarizedExperiment objects) as well as DNA genotyping VCF files based on a set of input samples is feasible and that's what others are building. After that, I think that it's best to adapt code we have (or even use newer tools) after carefully exploring the data.",null,"I described the need for the \"module use\" command and modifying the .bashrc to easily use LIBD modules","We walked througth the SpatialLIBD code to find where the argument has to be changed","I fetched the SRA files, created a manifest and execution script for SPEAQeasy, but had an issue running \"fastq-dump\". I decided to hand off the request, partially complete.","Failed MacOS installation on Lieber MacBook, something about Java both being and also not being installed","explaination of the functions VNS.m and refineVNS.m","when running VNS.m for several files on same matlab session the data from previous sessions is saved in matlab cache memory automatically. This will cause memory issues with subsequent runs, its important to clear variables after every run.","reffered to package \"effsize\" should be efficient at easily doing this","ran her through an example, explained her the code on github repo (split.m is the main script)","edited the config file for setting up x-quartz","Adapted code and shared it at https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock","He walked me through some code performing qSVA for a recent project, and I offered some tips involving code organization/ readibility.","Created GitHub issues at https://github.com/LieberInstitute/VisiumImgProcessing/issues","fixed the github repo","I showed them the tools I developed in DeconvoBuddies package, and provided links to example code where I had applied these funcitons in previous work","explore loading/saving/exporting options on Qupath","We discussed how Sciwheel can be adapted for different journals, and provided general advice for getting started and formatting a paper for a particular journal.","I explained that SPEAQeasy was used for the recent hg38 processing, and an older version of a similar pipeline was used for the original hg19 processing. We discussed our new method for retrieving gene symbols (\"org.*.eg.db\" packages) and some general code advice.","I went through the setup and example usage of Sciwheel and cross reference, and gave general advice (it's often easiest to start with the \"Methods\" section, etc)","use rsync","Debugging through the input/output file in logs","I explained how to use SPEAQeasy at JHPCE, and we tried an experimental method for downloading SRA data. The method didn't work, and we plan to meet again after I verify a more typical method works.","I hadn't encountered the issue with new local terminals failing to open. I planned to follow up with better specific advice via email, and offer to explain this advice in a future DSGS.","running chmod","Used Amy's rstats blogpost on git","We spent time doing basic Rstudio setup and troubleshooting, and got the data downloaded",null,"We got Heena's MAC registered for passwordless login to JHPCE",null,"We got the ssh keys working for Brenda's github","We went over applying DeconvoBuddies functions to hebenula data","Use of galaxybrain for identifying samples with DNAm, DNA genotype and RNA-seq data","We are going to try an iterative strategy to keep tables smaller",null,"Practiced questions for interviews.","[BioC Support website answer](https://support.bioconductor.org/p/p134069/#p134141)",null,"[BioC Support website answer](https://support.bioconductor.org/p/p133591/#p133927)","Used 'h5ls' command from shell to verify contents of file; eventually found we were looking at the wrong file (error message was cryptic about issue). https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1607034309074800","Need to resolve swap bugs. Plan to create a table of phenotype data (Dx, age, sex, race) for each BrNum to mach genotype data","Went through the setting up passwordless login for jhpce using ssh keys, and coppied the configuration files from https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee","[RStudio Community self-answer](https://community.rstudio.com/t/recipes-package-cannot-create-interaction-term-in-step-interact/89860)","got through the Github vignettes thoroughly",null,"[BioC Support website answer](https://support.bioconductor.org/p/p133110/#p133319)",null,"Started PsychENCODE access request process","[There is a plugin for Genome Studio by Illumina that allows users to output their imputed allele results as PLINK inputs](https://support.illumina.com/downloads/genomestudio-2-0-plug-ins.html)","[BioC Support website answer](https://support.bioconductor.org/p/134569/#p133028)","to use rsync or scp, you need to have user account created on LIBD server (Bill can help create it)",null,"We debugged some code and ultimetly decided that it might be best for Abby to write new code and use the tsne function to cluster and plot the data","I recomended using dcast to do a \"long\" to \"wide\" data transformation","We talked about how the data should be structured before using ggplot (transforming some data and adding catagorical columns)","[I recommended that they reach out to their forum board](https://support.bioconductor.org/p/p132558/#p132872)","[Check Slack helpdesk channel](https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1605123296024100)","[BioC Support website answer](https://support.bioconductor.org/p/134139/#p132829)","[GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/15)","We discussed organizing by analysis (a directory for each one, then prehaps sub- directory for data & plots). Create a readMe for each sub-folder. Use gitignore to keep repo from getting too big.","[BioC Support website answer](https://support.bioconductor.org/p/131600/#p132617)","https://www.mathworks.com/matlabcentral/answers/636365-how-can-i-measure-volume-from-a-3d-image?s_tid=srchtitle","Resulted in recount3 versions 1.1.1 and 1.1.2 on bioc-devel (and 1.0.2 on bioc-release)","[GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/13)","[GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/14)",null,"Explained the `chgrp` and `umask` commands plus `~/.bashrc` file configuration. I also mentioned why we don't use `chmod 777` at JHPCE. Details on [Slack](https://jhu-genomics.slack.com/archives/CR9NYA0BF/p1604086263148400?thread_ts=1603826219.094600&cid=CR9NYA0BF) (private channel).","[BioC Support website answer](https://support.bioconductor.org/p/p132487/#p132521)",null,null,"[BioC Support website answer](https://support.bioconductor.org/p/p132406/#p132459)","[BioC Support website answer](https://support.bioconductor.org/p/p132450/#p132464)","initial issue seemed to do with the spaceranger command arguments; we are continuining to investigate possible module issues or insufficient code in the [blog post](http://research.libd.org/rstatsclub/2020/10/20/using-space-ranger-at-jhpce/)","Needed to use highlight_key and highlight functions to select points between plots. Used rstats example as guide https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14",null,"[BioC Support website answer](https://support.bioconductor.org/p/134532/#134925)","[BioC Support website answer](https://support.bioconductor.org/p/134914/#134915)","[BioC Support website answer](https://support.bioconductor.org/p/131285/#134921)","[BioC Support website answer](https://support.bioconductor.org/p/134876/#134917)","spatstat package https://spatstat.org/releasenotes/spatstat-1.39-0.html, L function https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest, Kmark https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark"]},"columns":[{"id":"date","name":"date","type":"Date","na":"NA","minWidth":125,"style":"function(rowInfo, colInfo) {\nconst rowIndex = rowInfo.index + 1\n}","html":true,"width":120,"align":"right"},{"id":"guide","name":"guide","type":"character","na":"NA","minWidth":125,"style":"function(rowInfo, colInfo) {\nconst rowIndex = rowInfo.index + 1\n}","html":true,"width":100,"align":"left"},{"id":"help_request","name":"help_request","type":"character","na":"NA","minWidth":125,"style":"function(rowInfo, colInfo) {\nconst rowIndex = rowInfo.index + 1\n}","cell":["<span class='gt_from_md'>log2 CPM vs log2 normalized counts<\/span>","<span class='gt_from_md'>Guidance to add spatial registration to shiny app.<\/span>","<span class='gt_from_md'>ways to explore domain specific marker genes compared to other domain genes<\/span>","<span class='gt_from_md'>Resources for processing methylation data<\/span>","<span class='gt_from_md'>git help: accidentally pushing large file<\/span>","<span class='gt_from_md'>NIH K08 grant<\/span>","<span class='gt_from_md'>i have a question about visualization: when i calculate the logcounts matrix of the pseudobulked SRT data, i get some negative values because there are some extremely low counts. under the hood, the function is using edgeR:cpm() by adding an average count of 1 (<a href=\"https://github.com/LieberInstitute/spatialLIBD/blob/35fe720741b0bf44e19475f80b5f3475f347d6b6/R/registration_pseudobulk.R#L163\">https://github.com/LieberInstitute/spatialLIBD/blob/35fe720741b0bf44e19475f80b5f3475f347d6b6/R/registration_pseudobulk.R#L163<\/a>). i think this is fine for DE analysis, but it is hard to interpret when visualizing boxplots of gene expression on the log scale. i tested this out on the DLPFC n=30 SRT data as well, and i also get some negatives when lognormalizing the pseudobulked matrix.\nwhat is the preferred way to visualize this? one option would be to just leave the negatives as is. another option would be to add a slightly larger prior count, like 3, to prevent negatives while preserving the distribution of the data.<\/span>","<span class='gt_from_md'>How to write an NIH R01<\/span>","<span class='gt_from_md'>Learning how to work with a PhD student<\/span>","<span class='gt_from_md'>JHU PhD info<\/span>","<span class='gt_from_md'>Review scripts containing pseudobulk visium data to compute DEG data.<\/span>","<span class='gt_from_md'>Team culture overview<\/span>","<span class='gt_from_md'>formatting data for box plots<\/span>","<span class='gt_from_md'>PhD vs masters career paths<\/span>","<span class='gt_from_md'>DE strategies, pseudobulk process and using pseudoBulkDGE()<\/span>","<span class='gt_from_md'>curl error in SPEAQeasy<\/span>","<span class='gt_from_md'>PCA plots with a batch effect<\/span>","<span class='gt_from_md'>Updates on projector datasets<\/span>","<span class='gt_from_md'>Reviewing code to quantify mRNA expression<\/span>","<span class='gt_from_md'>Feedback previous to presentation<\/span>","<span class='gt_from_md'>Cloud computing trial<\/span>","<span class='gt_from_md'>Donor-level pseudobulk analysis<\/span>","<span class='gt_from_md'>I’m trying to use the spatialDLPFC_snRNAseq dataset from spatialLIBD. I’m not sure how much QC has been done. I’m also wondering what the UMAP was run on.<\/span>","<span class='gt_from_md'>Formatting inputs for <code>spatialLIBD::layer_stat_cor()<\/code><\/span>","<span class='gt_from_md'>PCA plots interpretation<\/span>","<span class='gt_from_md'>registration wrapper debug<\/span>","<span class='gt_from_md'>Spatial registration process<\/span>","<span class='gt_from_md'>SPEAQeasy preprocessing error<\/span>","<span class='gt_from_md'>Project goals discussion<\/span>","<span class='gt_from_md'>Advice for running SPEAQeasy efficiently and estimating memory use<\/span>","<span class='gt_from_md'>How to build a RSE combining megadepth counts and recount3 colData<\/span>","<span class='gt_from_md'>How to speed up SPEAQeasy runs<\/span>","<span class='gt_from_md'>Feedback for her recent presentation in spatial meeting<\/span>","<span class='gt_from_md'>The qvalue and using the pi1 statistic for replication<\/span>","<span class='gt_from_md'>Identification of specific marker genes for a cluster in a published dataset<\/span>","<span class='gt_from_md'>Interpretation of pi1 statistic<\/span>","<span class='gt_from_md'>Replication rate using qvalue<\/span>","<span class='gt_from_md'>job_loop questions and run_all scripts advice<\/span>","<span class='gt_from_md'>Biostats prospective student<\/span>","<span class='gt_from_md'>Check previous steps to DEG process<\/span>","<span class='gt_from_md'>SPEAQeasy preprocessing failure<\/span>","<span class='gt_from_md'>smoking-nicotine-mouse paper reviews<\/span>","<span class='gt_from_md'>Accessing Gandal et al 2018 files, other BrainSEQ re-analyses<\/span>","<span class='gt_from_md'>EB2-NIW<\/span>","<span class='gt_from_md'>JHPCE new PI<\/span>","<span class='gt_from_md'>EB2-NIW<\/span>","<span class='gt_from_md'>discuss about writing methods<\/span>","<span class='gt_from_md'>Issue with several plots in a  PDF file<\/span>","<span class='gt_from_md'>Setting Paylocity goals<\/span>","<span class='gt_from_md'>setting goals for the year- to complete paylocity profile<\/span>","<span class='gt_from_md'>recount2/recount3 base-pair coverage questions<\/span>","<span class='gt_from_md'>Concurrent task limit at JHPCE<\/span>","<span class='gt_from_md'>Shiny error without message<\/span>","<span class='gt_from_md'>Debugging FUSION TWAS<\/span>","<span class='gt_from_md'>SLURM environment variables<\/span>","<span class='gt_from_md'>identifying files in MDDseq project<\/span>","<span class='gt_from_md'>FICTURE Visium HD results<\/span>","<span class='gt_from_md'>I am doing DEG analysis on DLPFC PNN project, I got CAMK2A as the top upregulated DEG in neuropil+ spots. But when I plot the expression of CAMK2A in neuropil+ vs neuropil- spots as bar plot, the bar in neuropil- category is higher which contradicts the DEG result.<\/span>","<span class='gt_from_md'>Git pull fails<\/span>","<span class='gt_from_md'>Help debugging bioc devel (3.21) issues<\/span>","<span class='gt_from_md'>Help updating a GitHub pages website<\/span>","<span class='gt_from_md'>Debugging curl installation<\/span>","<span class='gt_from_md'>Package clashes in conda_R/devel<\/span>","<span class='gt_from_md'>discuss about qtune results, spatial registration script. completed running add clusters and model psuedobulk<\/span>","<span class='gt_from_md'>Running SPEQeasy with mixed-species samples<\/span>","<span class='gt_from_md'>Discuss about next steps spatial registration, Rscripts.<\/span>","<span class='gt_from_md'>Unexpected prompting of credentials when logging into JHPCE<\/span>","<span class='gt_from_md'>read10xVisium error<\/span>","<span class='gt_from_md'>Running BayesSpace, can we go over the preparatory steps<\/span>","<span class='gt_from_md'>Building SPE and plotting with Visium HD<\/span>","<span class='gt_from_md'>Debugging R installation in ‘r_nac’ module<\/span>","<span class='gt_from_md'>Running SPEAQeasy at JHPCE<\/span>","<span class='gt_from_md'>Samui troubleshooting continued<\/span>","<span class='gt_from_md'>Troubleshooting Samui<\/span>","<span class='gt_from_md'>Adding manual annotation<\/span>","<span class='gt_from_md'>debug deploy.R<\/span>","<span class='gt_from_md'>BayesSpace clustering<\/span>","<span class='gt_from_md'>spaceranger metrics<\/span>","<span class='gt_from_md'>Using sciwheel and OneDrive<\/span>","<span class='gt_from_md'>Adding Samui annotations to an SPE object<\/span>","<span class='gt_from_md'>Merging HALO data<\/span>","<span class='gt_from_md'>Using git at JHPCE and ignoring files<\/span>","<span class='gt_from_md'>Letter of support for a grant application<\/span>","<span class='gt_from_md'>Debugging a GitHub Actions workflow error on a BioC 3.20 ExperimentHub package under development<\/span>","<span class='gt_from_md'>Reviewing BioC package contribution process<\/span>","<span class='gt_from_md'>Loading old scikit-learn model<\/span>","<span class='gt_from_md'>spatial registration, Automatic annotation of spatial registration results, spatialLIBD cluster import()/export()<\/span>","<span class='gt_from_md'>visiumStitched revision<\/span>","<span class='gt_from_md'>PRECAST recommendations<\/span>","<span class='gt_from_md'>Team support overview and introduction<\/span>","<span class='gt_from_md'>Paper draft structure<\/span>","<span class='gt_from_md'>Debugging an .Rbuildignore file<\/span>","<span class='gt_from_md'>AtlaxXomics interpretation results<\/span>","<span class='gt_from_md'>SpatialExperiment::rotateObject() with stitched data<\/span>","<span class='gt_from_md'>facing issues with cyberduck - it keeps asking me for the authentication key always<\/span>","<span class='gt_from_md'>Visium Training Steps III and Visium Training, JHPCE commands and configuration<\/span>","<span class='gt_from_md'>Addressing NDA submission errors<\/span>","<span class='gt_from_md'>Project handoff checks and project updates<\/span>","<span class='gt_from_md'>AtlaxXomics interpretation results<\/span>","<span class='gt_from_md'>Visium Training Steps II<\/span>","<span class='gt_from_md'>Cont. PhD application plans<\/span>","<span class='gt_from_md'>Can you please point me to a table (from Maynard et al. 2021, Huuki-Myers et al. 2024, or other?) that lists the necessary marker genes important for delineating each cortical layer (WM, layer 1-6) in clustered 3’ Visium DLPFC datasets?<\/span>","<span class='gt_from_md'>Visium Training Steps<\/span>","<span class='gt_from_md'>smoking-nicotine-mouse paper checks<\/span>","<span class='gt_from_md'>PhD application plans<\/span>","<span class='gt_from_md'>Using modules, slurmjobs, and certain partitions<\/span>","<span class='gt_from_md'>smoking-nicotine-mouse paper checks<\/span>","<span class='gt_from_md'>Visium overview<\/span>","<span class='gt_from_md'>Hb_VTA_projectors updates<\/span>","<span class='gt_from_md'>LIBD internship<\/span>","<span class='gt_from_md'>using GitHub to sync laptop &amp; JHPCE<\/span>","<span class='gt_from_md'>spatialLIBD apps deployment review<\/span>","<span class='gt_from_md'>using GitHub to sync laptop &amp; JHPCE<\/span>","<span class='gt_from_md'>template_project (GitHub, JHPCE)<\/span>","<span class='gt_from_md'>Laptop config: RStudio on laptop + local git clone + work on RStudio terminal<\/span>","<span class='gt_from_md'>Team website self-description<\/span>","<span class='gt_from_md'>Manisha Training- Laptop config: Cyberduck<\/span>","<span class='gt_from_md'>GO / KEGG ORA analysis on cell-type specific DEG results<\/span>","<span class='gt_from_md'>Slack etiquette (questions on channels, avoid DMs, when to tag) and Google Slides weekly update setup<\/span>","<span class='gt_from_md'>Single-cell genomics webinar Latin America<\/span>","<span class='gt_from_md'>LIBD rstats club Google Calendar<\/span>","<span class='gt_from_md'>Team presentations Google Sheet and  How to sign up for a DSgs<\/span>","<span class='gt_from_md'>Getting an integer vector from factor of integers<\/span>","<span class='gt_from_md'>Hb_VTA_projectors updates<\/span>","<span class='gt_from_md'>pseudobulking with Seurat<\/span>","<span class='gt_from_md'>LIBD cloud computing<\/span>","<span class='gt_from_md'>Cont. on team options<\/span>","<span class='gt_from_md'>we find that we have thousands of upregulated DEGs and the volcano plot looks odd. We are following the code in the best practices st book and am not sure if the plot we are getting is just the nature of the data and this brain region, or something about how we are running the DEG testing<\/span>","<span class='gt_from_md'>Clustering vs sub-clustering results comparison<\/span>","<span class='gt_from_md'>Trimming options with SPEAQeasy<\/span>","<span class='gt_from_md'>Adapting scripts to SLURM and using the ‘fusion_twas’ module<\/span>","<span class='gt_from_md'>edgeR 4.0 and clusterProfiler questions<\/span>","<span class='gt_from_md'>Running nnSVG and visualizing PRECAST results<\/span>","<span class='gt_from_md'>Features from JHPCE storage tracker website<\/span>","<span class='gt_from_md'>I have questions regarding the leafcutter pipeline, which is used in the MDD project to produce the clustered files.<\/span>","<span class='gt_from_md'>Random seed usage for PCA/UMAP/tSNE<\/span>","<span class='gt_from_md'>Smoking-nicotine-mouse paper checks<\/span>","<span class='gt_from_md'>Smoking-nicotine-mouse paper checks<\/span>","<span class='gt_from_md'>NDA- submission and authentication<\/span>","<span class='gt_from_md'>Cell type sub-clustering<\/span>","<span class='gt_from_md'>InterJad<\/span>","<span class='gt_from_md'>Review on how to write method sections<\/span>","<span class='gt_from_md'>Continuation on DEG analysis at the cell cluster level<\/span>","<span class='gt_from_md'>Viruses and recount2/3<\/span>","<span class='gt_from_md'>spatialLIBD app results and interpretation for pseudobulk specific bayes space k<\/span>","<span class='gt_from_md'>single cell markers<\/span>","<span class='gt_from_md'>DEG analysis at the cell cluster level<\/span>","<span class='gt_from_md'>LIBD opportunities<\/span>","<span class='gt_from_md'>I am using spatialLIBD::registration_wrapper() for enrichment DE testing. The dimensions of my spe are 29720 x 77553, but the modeling results only represent 12326 genes. I’m not sure what happens to the remaining genes or if this behavior is intentional (because not all the genes have significant adjusted p-values). Why do all the modeling types result in the same number of genes?<\/span>","<span class='gt_from_md'>2024 opportunities<\/span>","<span class='gt_from_md'>SpatialExperiment and other related R objects<\/span>","<span class='gt_from_md'>Team opportunities<\/span>","<span class='gt_from_md'>Spatial-Registration workflow. I am particularly confused with the use of the reference, as it is coming from DLFPC<\/span>","<span class='gt_from_md'>LIBD fundraising<\/span>","<span class='gt_from_md'>GitHub profile review<\/span>","<span class='gt_from_md'>Overview of running PRECAST<\/span>","<span class='gt_from_md'>Overview of using NDA<\/span>","<span class='gt_from_md'>vis_clus issues<\/span>","<span class='gt_from_md'>Shiny issue: reading previous manual annotation<\/span>","<span class='gt_from_md'>Scripts for log normalization and GLM-PCA<\/span>","<span class='gt_from_md'>Review the habenula snRNA-seq data to assess the enrichment of habenula DEGs for fentanyl administration in cell type-specific marker genes<\/span>","<span class='gt_from_md'>Getting SPEAQeasy started<\/span>","<span class='gt_from_md'>Questions about the “McCarroll lab dlPFC web application”<\/span>","<span class='gt_from_md'>10x Genomics reference GTF annotation files<\/span>","<span class='gt_from_md'>Zotero setup<\/span>","<span class='gt_from_md'>qsvaR questions<\/span>","<span class='gt_from_md'>Log normalizing counts in spatial data<\/span>","<span class='gt_from_md'>Resolving merge conflicts<\/span>","<span class='gt_from_md'>Career development<\/span>","<span class='gt_from_md'>Questions about article<\/span>","<span class='gt_from_md'>deconvolution results (cell type proportions) to be used as covariates for differential expression analysis in Brainseq DLPFC, caudate and hippocampus<\/span>","<span class='gt_from_md'>Understanding misalignment in Visium stitching<\/span>","<span class='gt_from_md'>How to check indices used from 10x FASTQ files<\/span>","<span class='gt_from_md'>Deploying and testing Shiny app locally<\/span>","<span class='gt_from_md'>DGE analysis of a data subset questions<\/span>","<span class='gt_from_md'>Basic stuff to run R scripts in the cluster<\/span>","<span class='gt_from_md'>Filtering lowly expressed features prior for replication analyses<\/span>","<span class='gt_from_md'>Updating broken Globus endpoints<\/span>","<span class='gt_from_md'>Trimming error in BiocMAP<\/span>","<span class='gt_from_md'>Making RNAscope plots using HALO csv files<\/span>","<span class='gt_from_md'>Reviewing NDA upload scripts<\/span>","<span class='gt_from_md'>Understanding cleaningY()<\/span>","<span class='gt_from_md'>Help finding odds ratios for PGC3 SNPs to determine risk allele, and general discussion about plot results and aesthetics<\/span>","<span class='gt_from_md'>Module-related JHPCE files<\/span>","<span class='gt_from_md'>Understand how to run a MAGMA analysis for habenula DEGs<\/span>","<span class='gt_from_md'>We want advice as to <em>how best<\/em> to do the deconvolution<\/span>","<span class='gt_from_md'>Query about some LIBD bulk RNA-seq and WGBS datasets<\/span>","<span class='gt_from_md'>Getting BiocMAP running at JHPCE<\/span>","<span class='gt_from_md'>Impact of CellRanger versions over time on single-exon genes embedded inside introns of longer genes<\/span>","<span class='gt_from_md'>Running nextflow based pipeline with singularity<\/span>","<span class='gt_from_md'>SCZD DEG models<\/span>","<span class='gt_from_md'>Resolving merge conflicts<\/span>","<span class='gt_from_md'>Reviewing covariates to use in a DE model for snRNA-seq pseudobulked data by cell type with dreamlet<\/span>","<span class='gt_from_md'>Mean ratio application to HumanPilot data and spatialDLPFC re-analysis<\/span>","<span class='gt_from_md'>Setting up a website similar to the <a href=\"https://research.libd.org/spatialDLPFC/\">spatialDLPFC one<\/a><\/span>","<span class='gt_from_md'>WGCNA network selection<\/span>","<span class='gt_from_md'>Review of SCZD vs NTC DE models<\/span>","<span class='gt_from_md'>Understanding SPEAQeasy and using correct annotation<\/span>","<span class='gt_from_md'>Options for creating links to use in manuscripts<\/span>","<span class='gt_from_md'>Review steps for a new snRNA-seq project<\/span>","<span class='gt_from_md'>Loupe Browser re-analizing and advanced options<\/span>","<span class='gt_from_md'>Running SPEAQeasy with conda<\/span>","<span class='gt_from_md'>Reference letter request<\/span>","<span class='gt_from_md'>Inspect multimome objects with Browse Loupe<\/span>","<span class='gt_from_md'>Support with Config Files<\/span>","<span class='gt_from_md'>Updating ranger_metrics information<\/span>","<span class='gt_from_md'>Avoid using jsonlite when editing spaceranger json files<\/span>","<span class='gt_from_md'>Write a script to manually edit spaceranger json files<\/span>","<span class='gt_from_md'>Debugging spaceranger count<\/span>","<span class='gt_from_md'>feedback on figures<\/span>","<span class='gt_from_md'>Understanding a spaceranger error<\/span>","<span class='gt_from_md'>access bulk RNAseq from samples cataloged in the Lieber-internal bulk rnaseq database<\/span>","<span class='gt_from_md'>debugging samui script for HPC HE data<\/span>","<span class='gt_from_md'>Question about the correct number of reads in sequencing for scRNAseq and scATACseq multiome strategy<\/span>","<span class='gt_from_md'>debugging samui script for HPC HE data<\/span>","<span class='gt_from_md'>LIBD Data Portal for BSP1 and BSP2 DLPFC SCZD vs NTC data<\/span>","<span class='gt_from_md'>Cell Ranger-ARC html report<\/span>","<span class='gt_from_md'>slurm job loops<\/span>","<span class='gt_from_md'>Polishing details on how to ask for help effectively<\/span>","<span class='gt_from_md'>GEX and ATAC clustering<\/span>","<span class='gt_from_md'>More troubleshooting misaligned image<\/span>","<span class='gt_from_md'>Cg_Hb_projector_profiling progress update<\/span>","<span class='gt_from_md'>Overview of use cases for JHPCE disk usage data over time<\/span>","<span class='gt_from_md'>snRNA-seq data analysis steps overview<\/span>","<span class='gt_from_md'>Cloud computing vs JHPCE<\/span>","<span class='gt_from_md'>Troubleshooting misaligned image in Samui<\/span>","<span class='gt_from_md'>Advice prior to a faculty interview<\/span>","<span class='gt_from_md'>Plotting to a multi-page PDF with 3 plots per page<\/span>","<span class='gt_from_md'>Derive code for how to merge ERCCsumLogErr values across sequencing lanes for a given sample<\/span>","<span class='gt_from_md'>Help identifying differences between two RSE objects containing data from the same project<\/span>","<span class='gt_from_md'>I tried to apply “qSV_model_DE_analysis.R” in lithium-exposed samples ( from Dpseq data), but I was confused by some of the code, especially “run_DE.R.”<\/span>","<span class='gt_from_md'>Installing cellprofiler at JHPCE<\/span>","<span class='gt_from_md'>spatialDLPFC and deconvolution-benchmark project details on how snRNA-seq data was used for each of them<\/span>","<span class='gt_from_md'>JHPCE login questions<\/span>","<span class='gt_from_md'>quick big fix. Its the same eQTL code – just one header in a file that’s always missing<\/span>","<span class='gt_from_md'>Grant related discussion<\/span>","<span class='gt_from_md'>Adding notes to Samui<\/span>","<span class='gt_from_md'>Cg_Hb_projector_profiling progress update<\/span>","<span class='gt_from_md'>Better algorithm for dealing with overlapping images in Samui<\/span>","<span class='gt_from_md'>Setting up SSH keypair for passwordless login to JHPCE<\/span>","<span class='gt_from_md'>Vistoseg parameter tuning<\/span>","<span class='gt_from_md'>Feedback on the nascentRNA project<\/span>","<span class='gt_from_md'>Deconvolution<\/span>","<span class='gt_from_md'>How to extract normalized expression values with limma<\/span>","<span class='gt_from_md'>Prospective JHU PhD student<\/span>","<span class='gt_from_md'>Cg_Hb_projector_profiling progress update<\/span>","<span class='gt_from_md'>Cg_Hb_projector_profiling progress update<\/span>","<span class='gt_from_md'>qsvaR bug report resolution<\/span>","<span class='gt_from_md'>Adding phenotype data to coldata<\/span>","<span class='gt_from_md'>Adding packages to a custom R module<\/span>","<span class='gt_from_md'>Rownames not well formatted<\/span>","<span class='gt_from_md'>Cg_Hb_projector_profiling progress update<\/span>","<span class='gt_from_md'>Discuss Seurat, cell type markers, DE<\/span>","<span class='gt_from_md'>Follow-up: removing large files from version control<\/span>","<span class='gt_from_md'>Troubleshooting GitHub authentication issues<\/span>","<span class='gt_from_md'>Removing large files from version control<\/span>","<span class='gt_from_md'>Working with SLURM<\/span>","<span class='gt_from_md'>Career development<\/span>","<span class='gt_from_md'>Running python on cluster<\/span>","<span class='gt_from_md'>Creating a module<\/span>","<span class='gt_from_md'>Permutation analysis<\/span>","<span class='gt_from_md'>making a corrplot with a lot of data points .<\/span>","<span class='gt_from_md'>Review response to qsvaR bug report from last year<\/span>","<span class='gt_from_md'>Follow up on stitching for Samui<\/span>","<span class='gt_from_md'>Maybe divinding them by sample name may help.<\/span>","<span class='gt_from_md'>when I try to rerun R dreamlet pseudobulk on the new JHPCE, I keep getting errors… (provided code example)<\/span>","<span class='gt_from_md'>Reviewing analysis using qsvaR package<\/span>","<span class='gt_from_md'>Reviewing updates to the qsvaR package<\/span>","<span class='gt_from_md'>Creating stitched image in Samui from ImageJ estimates<\/span>","<span class='gt_from_md'>Investigating visual problems in Samui image<\/span>","<span class='gt_from_md'>Help adding phenotype info to colData<\/span>","<span class='gt_from_md'>Investigating R package installation conflicts<\/span>","<span class='gt_from_md'>Investigating error in qSVA funtion call<\/span>","<span class='gt_from_md'>Fix misleading getDegTx error message<\/span>","<span class='gt_from_md'>Performing ‘jaffelab::s’ on large tibble efficiently<\/span>","<span class='gt_from_md'>Follow up on stitching for Samui<\/span>","<span class='gt_from_md'>Creating stitched image in Samui from ImageJ estimates<\/span>","<span class='gt_from_md'>JHPCE R modules<\/span>","<span class='gt_from_md'>Regarding the transcriptome assay from the multiome approach, I have selected canonical gene markers from the different samples; Moving further, I would like to integrate to the analysis external gene-markers shared by S.Page. I am looking the best way to do this.<\/span>","<span class='gt_from_md'>CV writing advice<\/span>","<span class='gt_from_md'>Help with qSVA overview schematic<\/span>","<span class='gt_from_md'>Overview of image stitching process<\/span>","<span class='gt_from_md'>Cg_Hb_projector_profiling progress update<\/span>","<span class='gt_from_md'>Review how to respond to a user’s bug report<\/span>","<span class='gt_from_md'>Job application support<\/span>","<span class='gt_from_md'>I would like to discuss with you the correlation issue related to the cellular_component model of the qsvaR package and get your feedback.<\/span>","<span class='gt_from_md'>Job application support<\/span>","<span class='gt_from_md'>Cg_Hb_projector_profiling progress update<\/span>","<span class='gt_from_md'>Modifying a global-scope variable inside a function<\/span>","<span class='gt_from_md'>Review replication analysis R script<\/span>","<span class='gt_from_md'>Cg_Hb_projector_profiling project overview<\/span>","<span class='gt_from_md'>Review Visium SPG AD blog post<\/span>","<span class='gt_from_md'>Issues linking repo to github<\/span>","<span class='gt_from_md'>Interpreting decision tree and spot-deconvo plots<\/span>","<span class='gt_from_md'>Issues with tensorqtl module<\/span>","<span class='gt_from_md'>Interaction modeling with limma-voom<\/span>","<span class='gt_from_md'>snRNA-seq OSCA-based analysis<\/span>","<span class='gt_from_md'>Review replication analysis R script<\/span>","<span class='gt_from_md'>Debugging a ranger_metrics R script<\/span>","<span class='gt_from_md'>Interpretation of “replication” plots and performance of QSVA models<\/span>","<span class='gt_from_md'>Hi! We would love an opinion on our (now cell-type assigned) single nuclei rna-seq data, i.e. how they look - if anything is off, advice for improvement…<\/span>","<span class='gt_from_md'>Example datasets and getting started with data science<\/span>","<span class='gt_from_md'>Review replication analysis scripts for the qsvaR project<\/span>","<span class='gt_from_md'>Help migrating to SLURM setup<\/span>","<span class='gt_from_md'>snRNA-seq OSCA-based analysis<\/span>","<span class='gt_from_md'>Nextflow issue when running SPEAQeasy<\/span>","<span class='gt_from_md'>Setting up and running SPEAQeasy on a new cluster<\/span>","<span class='gt_from_md'>Installing SPEAQeasy and setting up main script<\/span>","<span class='gt_from_md'>Troubleshooting a SPEAQeasy error<\/span>","<span class='gt_from_md'>qsvaR paper planning<\/span>","<span class='gt_from_md'>Prepare expression bed files for tensorQTL<\/span>","<span class='gt_from_md'>sbatch issue with SPEAQeasy<\/span>","<span class='gt_from_md'>Interpreting and refactoring “concordance” formulas in QSVA results<\/span>","<span class='gt_from_md'>snRNA-seq OSCA-based analysis<\/span>","<span class='gt_from_md'>Prepare covariate input files for tensorQTL<\/span>","<span class='gt_from_md'>Refactoring QSVA code<\/span>","<span class='gt_from_md'>snRNA-seq OSCA-based analysis<\/span>","<span class='gt_from_md'>Interpretation of results plotted in inflection plots of Cellranger-arc data sets (GEX assays). For example, how do you know which genes are representing neurons or glia?<\/span>","<span class='gt_from_md'>Dropping extra factor levels<\/span>","<span class='gt_from_md'>Onboarding questions for a snRNA-seq dataset<\/span>","<span class='gt_from_md'>Confidence metric in spot deconvo model<\/span>","<span class='gt_from_md'>Array jobs over variables with SLURM<\/span>","<span class='gt_from_md'>SGE vs. SLURM R package conflicts<\/span>","<span class='gt_from_md'>help get normalized single cell data from a single cell rse for downstream eQTL<\/span>","<span class='gt_from_md'>snRNA-seq training<\/span>","<span class='gt_from_md'>How to interpret terms on an interaction model<\/span>","<span class='gt_from_md'>Guidance on expression heatmaps<\/span>","<span class='gt_from_md'>Discuss deconvolution for some future work<\/span>","<span class='gt_from_md'>help get normalized single cell data from a single cell rse for downstream eQTL<\/span>","<span class='gt_from_md'>Customizing the home page for spatialLIBD apps<\/span>","<span class='gt_from_md'>Overview of the stitching process for split Visium samples<\/span>","<span class='gt_from_md'>snRNA-seq training<\/span>","<span class='gt_from_md'>Social media ideas<\/span>","<span class='gt_from_md'>SLURM array jobs and X11<\/span>","<span class='gt_from_md'>snRNAseq celltype grouping in DLPFC data<\/span>","<span class='gt_from_md'>Creating documentation sites with bookdown<\/span>","<span class='gt_from_md'>Displaying info about cells, not spots in Samui<\/span>","<span class='gt_from_md'>Interpreting spot deconvolution results<\/span>","<span class='gt_from_md'>I performed the whole analysis as previously established and also made an alternative version based on knowledge of OSCA book for single nuclei (with adaptive thresholds for QC and less tolerance for mito UMIs). Would you help me checking my QC thresholds and seeing if they make sense?<\/span>","<span class='gt_from_md'>snRNA-seq training<\/span>","<span class='gt_from_md'>ranger_metrics<\/span>","<span class='gt_from_md'>Missing genes in Samui<\/span>","<span class='gt_from_md'>My coding set up<\/span>","<span class='gt_from_md'>Recommendations for learning more data science skills<\/span>","<span class='gt_from_md'>Misc spot deconvolution questions<\/span>","<span class='gt_from_md'>Getting started with snRNA-seq<\/span>","<span class='gt_from_md'>recount 3 permissions<\/span>","<span class='gt_from_md'>Using modules at SLURM and configuring bashrc<\/span>","<span class='gt_from_md'>writing tweets efficiently<\/span>","<span class='gt_from_md'>Introduction to what we do as a team and future plans<\/span>","<span class='gt_from_md'>Using Samui from start to finish<\/span>","<span class='gt_from_md'>Running DeconvoBuddies on the Mathy’s data<\/span>","<span class='gt_from_md'>Adding ceil function to splitSlide to split arrays covering fiducials<\/span>","<span class='gt_from_md'>Explain the SPE organization and subsetting an SPE<\/span>","<span class='gt_from_md'>Normalizing spot deconvolution results to answer biological questions of interest<\/span>","<span class='gt_from_md'>memory error with VistoSeg::InFormStitch function<\/span>","<span class='gt_from_md'>How to check if a visium slide data is already entered into REDCap<\/span>","<span class='gt_from_md'>Guidance on reference data for deconvolution and snRNA-seq analysis<\/span>","<span class='gt_from_md'>insert offset in VistoSeg::splitSlide_IF<\/span>","<span class='gt_from_md'>Quantifying spatial promiscuity of genes across clusters<\/span>","<span class='gt_from_md'>Feedback on an R package + GO/KEGG enrichment analyses<\/span>","<span class='gt_from_md'>Feeback on spatial data + code organization<\/span>","<span class='gt_from_md'>Feedback on workshop vignette<\/span>","<span class='gt_from_md'>Evaluate a potential external contractor<\/span>","<span class='gt_from_md'>How to best document what skills I can perform or need to learn<\/span>","<span class='gt_from_md'>Need help debugging iSEE app errors/warnings<\/span>","<span class='gt_from_md'>Practice BioC talk<\/span>","<span class='gt_from_md'>Troubleshooting GHA workflow errors when building workshop<\/span>","<span class='gt_from_md'>REDCap build table error<\/span>","<span class='gt_from_md'>NM pigement segmentation overview for LC samples<\/span>","<span class='gt_from_md'>Overview of image processing Visium and VSPG<\/span>","<span class='gt_from_md'>Input on a course design<\/span>","<span class='gt_from_md'>Feedback on responding to reviewers<\/span>","<span class='gt_from_md'>Spot deconvolution workflow overview<\/span>","<span class='gt_from_md'>Loading large files in R<\/span>","<span class='gt_from_md'>tif2mat conversion<\/span>","<span class='gt_from_md'>Sharing conda environment<\/span>","<span class='gt_from_md'>Performing QSVA with genes or with different annotation<\/span>","<span class='gt_from_md'>mat2tif conversion<\/span>","<span class='gt_from_md'>0% Vs 5% overlap and repeated signal issues in VSPG images<\/span>","<span class='gt_from_md'>How to make a project documentation website<\/span>","<span class='gt_from_md'>clusterProfiler, set.seed, and BiocParallel questions<\/span>","<span class='gt_from_md'>Resolving a permissions issue<\/span>","<span class='gt_from_md'>About empty drops run with GEX cellranger-arc data<\/span>","<span class='gt_from_md'>How to make dynamic error messages<\/span>","<span class='gt_from_md'>How to make a new Globus endpoint for a publication.<\/span>","<span class='gt_from_md'>How to filter an SCE object prior to making an iSEE app<\/span>","<span class='gt_from_md'>How to debug an R package function from a user’s report<\/span>","<span class='gt_from_md'>1/Learn about the roxygen2 syntax for examples. 2/Run devtools::document() like in dev/04_update.R (maybe also auto-style code). 3/Edit the news.md file + bump version on the DESCRIPTION (and change the date). 4/Learn about how to sync her local repo with steps like <a href=\"http://contributions.bioconductor.org/git-version-control.html#new-package-workflow\">http://contributions.bioconductor.org/git-version-control.html#new-package-workflow<\/a> (aka register to <a href=\"https://git.bioconductor.org/BiocCredentials/login/?next=/BiocCredentials/\">https://git.bioconductor.org/BiocCredentials/login/?next=/BiocCredentials/<\/a> like @Daianna González + add the upstream remote + git push to both upstream and origin).<\/span>","<span class='gt_from_md'>I created this github repo: <a href=\"https://github.com/LieberInstitute/neurovasculature-pilot/tree/main\">https://github.com/LieberInstitute/neurovasculature-pilot/tree/main<\/a>. Also, integrated it with slack, but instead of integrating the main branch, I linked the devel branch, and I am unable to see the commits that I make on slack. They are updated on github, though. So, I need help with the integration.<\/span>","<span class='gt_from_md'>Displaying spot coordinates in Samui<\/span>","<span class='gt_from_md'>Using the Samui API to display data locally<\/span>","<span class='gt_from_md'>Options for cell type specific eQTLs<\/span>","<span class='gt_from_md'>Options for cell type specific eQTLs<\/span>","<span class='gt_from_md'>Overview of what we do as a team and outreach options<\/span>","<span class='gt_from_md'>Troubleshooting SPEAQeasy error<\/span>","<span class='gt_from_md'>Ignoring a directory with gitignore<\/span>","<span class='gt_from_md'>Debugging rewritten code<\/span>","<span class='gt_from_md'>Troubleshooting unexpected SPEAQeasy output<\/span>","<span class='gt_from_md'>Interpreting spot-deconvolution plots<\/span>","<span class='gt_from_md'>Subsetting RSE by chromosome<\/span>","<span class='gt_from_md'>SPEAQeasy defaults<\/span>","<span class='gt_from_md'>Using Samui’s API to visualize spatial data<\/span>","<span class='gt_from_md'>Creating many configuration files differing by one variable<\/span>","<span class='gt_from_md'>Merging two dataframes<\/span>","<span class='gt_from_md'>Renaming many files by a pattern<\/span>","<span class='gt_from_md'>Help with SSH keys with GitHub<\/span>","<span class='gt_from_md'>Overfitting in qSVA<\/span>","<span class='gt_from_md'>Interpreting SPEAQeasy metrics<\/span>","<span class='gt_from_md'>Creating a blog post on the rstats website<\/span>","<span class='gt_from_md'>Ensembl IDs vs. gene symbols with cell2location<\/span>","<span class='gt_from_md'>Modules and reproducible software set-up for python<\/span>","<span class='gt_from_md'>Properly tracking git contributions<\/span>","<span class='gt_from_md'>When and how to merge git branches<\/span>","<span class='gt_from_md'>code review<\/span>","<span class='gt_from_md'>user had problem using import() function in R<\/span>","<span class='gt_from_md'>Information needed for Samui<\/span>","<span class='gt_from_md'>Feedback the SmokingMouse Project websites<\/span>","<span class='gt_from_md'>Properly transferring from dcl01 to dcs05,dcs04<\/span>","<span class='gt_from_md'>Git-LFS error messages<\/span>","<span class='gt_from_md'>Alternatives for RNA-seq analysis without replicates<\/span>","<span class='gt_from_md'>Configuring X11 for plotting at JHPCE<\/span>","<span class='gt_from_md'>Using the qsvaR with the new annotation<\/span>","<span class='gt_from_md'>Creating modules at JHPCE<\/span>","<span class='gt_from_md'>tensorQTL code overview<\/span>","<span class='gt_from_md'>seurat + signac<\/span>","<span class='gt_from_md'>Check some issues with GitHub Actions<\/span>","<span class='gt_from_md'>Introduction to CellRanger RNAseq ATAC analysis<\/span>","<span class='gt_from_md'>Using git and RStudio to work with R Markdown files<\/span>","<span class='gt_from_md'>SPEAQeasy error<\/span>","<span class='gt_from_md'>Understanding API authentication<\/span>","<span class='gt_from_md'>the user asked how to change the dot size in the plot_pca( )<\/span>","<span class='gt_from_md'>user had question about generating rarefaction curves in R<\/span>","<span class='gt_from_md'>tensorQTL pipeline overview<\/span>","<span class='gt_from_md'>Quantifying a single gene not in SPEAQeasy’s default reference<\/span>","<span class='gt_from_md'>Updating spatialLIBD::read10xVisiumAnalysis<\/span>","<span class='gt_from_md'>Verify GitHub ssh key setup and team overview<\/span>","<span class='gt_from_md'>Debugging spatialLIBD::read10xVisiumAnalysis<\/span>","<span class='gt_from_md'>Overview on “R/Bioconductor-powered Team Data Science”<\/span>","<span class='gt_from_md'>Finalizing genotyping + expression script<\/span>","<span class='gt_from_md'>Understanding the template project repository<\/span>","<span class='gt_from_md'>Help resolve JHPCE permission issues on the lifespan_hpc_spatial project<\/span>","<span class='gt_from_md'>Debug spaceranger 2023.0208.0 (likely an alpha for 2.0.1) with SpatialExperiment 1.8.0<\/span>","<span class='gt_from_md'>How to write a tweetorial<\/span>","<span class='gt_from_md'>read10xVisium() in version 1.7.1 (and now 1.8.0) creates a dangerous silent error by mismatching tissue positions across multiple samples<\/span>","<span class='gt_from_md'>Version controlling renamed directories<\/span>","<span class='gt_from_md'>Miscellaneous R questions<\/span>","<span class='gt_from_md'>Deconvolu code where the bisque doesnt show any differences<\/span>","<span class='gt_from_md'>How to write a tweetorial<\/span>","<span class='gt_from_md'>Creating marker heatmaps with pseudobulk data<\/span>","<span class='gt_from_md'>Setting up .bashrc file for folder access permission for DLPFC-SCZ project<\/span>","<span class='gt_from_md'>Using a gitignore to not version control specific files<\/span>","<span class='gt_from_md'>I have questions on GitHub and submitting SGE jobs.<\/span>","<span class='gt_from_md'>Moving forward with multiple cluster methods<\/span>","<span class='gt_from_md'>Adding figures to OneDrive and google docs with CrossReference<\/span>","<span class='gt_from_md'>Organizing figure components for Illustrator<\/span>","<span class='gt_from_md'>Lining up gene expression and genotyping data<\/span>","<span class='gt_from_md'>Setting up GitHub SSH keys<\/span>","<span class='gt_from_md'>Managing software dependencies with Nextflow<\/span>","<span class='gt_from_md'>Nextflow with Conda and Github actions<\/span>","<span class='gt_from_md'>SPEAQeasy error<\/span>","<span class='gt_from_md'>GitHub for a beginner<\/span>","<span class='gt_from_md'>snRNA-seq GLM PCA<\/span>","<span class='gt_from_md'>Integrating R scripts with Nextflow<\/span>","<span class='gt_from_md'>Making custom R script work with Netxflow<\/span>","<span class='gt_from_md'>snRNA-seq QC<\/span>","<span class='gt_from_md'>Getting started with snRNA-seq<\/span>","<span class='gt_from_md'>Plotting Gene Expression with SCE<\/span>","<span class='gt_from_md'>Help identify the source of unexpected NA values<\/span>","<span class='gt_from_md'>Help building SCE with downloaded data<\/span>","<span class='gt_from_md'>Adapting spot-deconvolution code to new data<\/span>","<span class='gt_from_md'>Discuss Mean Ratio marker gene finding<\/span>","<span class='gt_from_md'>Deconvolution study with rnaseq bulk data<\/span>","<span class='gt_from_md'>resolving a merge conflict<\/span>","<span class='gt_from_md'>Identifying the source of an error<\/span>","<span class='gt_from_md'>coloring plots by a continuous variable<\/span>","<span class='gt_from_md'>Understanding PCA for Habenula project<\/span>","<span class='gt_from_md'>Review deconvolition script for qSVA<\/span>","<span class='gt_from_md'>wrangling and reformatting expression data<\/span>","<span class='gt_from_md'>preprocessing and reformatting genotype and expression data<\/span>","<span class='gt_from_md'>organizing plots and output files<\/span>","<span class='gt_from_md'>understanding linear mixed-effects models<\/span>","<span class='gt_from_md'>help submitting jobs with R parallelization in jhpce. For instance, I would like to use scater::runPCA with the BPPARAM=BiocParallel::MulticoreParam(8)) parameter<\/span>","<span class='gt_from_md'>Introduction to LIBD and our team<\/span>","<span class='gt_from_md'>formatting genotype data into a matrix for each gene<\/span>","<span class='gt_from_md'>I have this single nucleus RNA sequencing experiment with the mouse LS that Matt analyzed and we have a final single cell experiment, but I don’t know how to do anything with this data set and want to learn how to plot heatmaps and possibly other things for this data set. Can we maybe schedule an hour to start this learning process and then I can schedule future time as needed?<\/span>","<span class='gt_from_md'>Check the DEG heatmaps<\/span>","<span class='gt_from_md'>Review TMM paper<\/span>","<span class='gt_from_md'>csvs that I want to merge together<\/span>","<span class='gt_from_md'>Lifting results between hg19 and hg38<\/span>","<span class='gt_from_md'>GO and KEGG boxplots with the modifications you suggested<\/span>","<span class='gt_from_md'>QC approach for hebenula<\/span>","<span class='gt_from_md'>How to check technical effects for RNAseq data<\/span>","<span class='gt_from_md'>Reviewing GO results and dotplots<\/span>","<span class='gt_from_md'>Working on making a violin plot for just one gene. So by brain region rather than gene type<\/span>","<span class='gt_from_md'>Question on the feasibility of some approach with HALO (for the AD project)<\/span>","<span class='gt_from_md'>Overview on using GitHub for Matlab projects<\/span>","<span class='gt_from_md'>Setting up SSH keys between login and compute nodes<\/span>","<span class='gt_from_md'>Setting up SSH keys for JHPCE<\/span>","<span class='gt_from_md'>Introduction to DSgs, R, Bioconductor, JHPCE<\/span>","<span class='gt_from_md'>Interaction between .bashrc and ‘module load’ in ‘qsub’d scripts<\/span>","<span class='gt_from_md'>Modifying a shell script to segment a VCF by genomic region<\/span>","<span class='gt_from_md'>Setting up SPEAQeasy at JHPCE<\/span>","<span class='gt_from_md'>Code to threshold small sized nuclei from the NaCC visium images<\/span>","<span class='gt_from_md'>How to organize OneDrive files<\/span>","<span class='gt_from_md'>How can I interpret the y-axis from <code>plotExplanatoryVariables()<\/code>?<\/span>","<span class='gt_from_md'>Use of git from multiple local locations<\/span>","<span class='gt_from_md'>Moving and correctly accessing HDF5-backed SummarizedExperiments<\/span>","<span class='gt_from_md'>DE analysis introduction<\/span>","<span class='gt_from_md'>Git push help<\/span>","<span class='gt_from_md'>check the models and the results I have obtained so far from the DEA<\/span>","<span class='gt_from_md'>DE analysis introduction<\/span>","<span class='gt_from_md'>Check the model design of the smoking mouse project<\/span>","<span class='gt_from_md'>VNS error<\/span>","<span class='gt_from_md'>Strategies to pick threshold for cell size in segmentations<\/span>","<span class='gt_from_md'>open multichannel image in imagej<\/span>","<span class='gt_from_md'>code to ectract x and y coordinates of randomnly slected inform tiles for PNN project<\/span>","<span class='gt_from_md'>Review results of Varaince Parition for smoking mouse<\/span>","<span class='gt_from_md'>Pick randomn tiles for the PNN project<\/span>","<span class='gt_from_md'>go over pseudobulk code<\/span>","<span class='gt_from_md'>Help with spatialLIBD colors setup<\/span>","<span class='gt_from_md'>Extracting final output into table<\/span>","<span class='gt_from_md'>CUDA issues with cellpose module<\/span>","<span class='gt_from_md'>QC update for smoking mouse<\/span>","<span class='gt_from_md'>R, RStudio, git, SSH key setup<\/span>","<span class='gt_from_md'>Addressing SPEAQeasy error messages<\/span>","<span class='gt_from_md'>LIBD Data Science<\/span>","<span class='gt_from_md'>help row annotation colors of a pheatmap<\/span>","<span class='gt_from_md'>Subsetting eQTL results with in risk loci<\/span>","<span class='gt_from_md'>Setting up SPEAQeasy<\/span>","<span class='gt_from_md'>Data science coordination<\/span>","<span class='gt_from_md'>QC update for smoking mouse<\/span>","<span class='gt_from_md'>Question about how ‘mitoRate’ is calculated in SPEAQeasy<\/span>","<span class='gt_from_md'>Help troubleshooting error messge when running BayesSpace<\/span>","<span class='gt_from_md'>zellkonverter error and guidance using SpaGCN<\/span>","<span class='gt_from_md'>Review progress on smoking mouse DE project<\/span>","<span class='gt_from_md'>pulling in external changes to a git repository<\/span>","<span class='gt_from_md'>Adjusting thresholds for event detection using CaPTure<\/span>","<span class='gt_from_md'>help installing ‘jaffelab’<\/span>","<span class='gt_from_md'>help with vis_grid_clus()<\/span>","<span class='gt_from_md'>JHPCE Github setup<\/span>","<span class='gt_from_md'>Help troubleshooting shiny app that crashes on gene interactive tab<\/span>","<span class='gt_from_md'>getting total UMI’s for each sample<\/span>","<span class='gt_from_md'>Github setup<\/span>","<span class='gt_from_md'>Revisited the sc analysis and could use some help<\/span>","<span class='gt_from_md'>explain spaceranger output files<\/span>","<span class='gt_from_md'>Tools and resources related to recount2 and recount3<\/span>","<span class='gt_from_md'>I might need a review of how to go about selecting marker genes for the spatialDLPFC data. Also, I’m curious what code you have for making heatmaps showing marker gene expression across cell types.<\/span>","<span class='gt_from_md'>Single cell rna seq analysis<\/span>","<span class='gt_from_md'>I have a mouse single-cell RNAseq data that I am aiming to identify cell types in different clusters.<\/span>","<span class='gt_from_md'>Debug <code>spatialLIBD::sig_genes_extract_all()<\/code>.<\/span>","<span class='gt_from_md'>How to debug an R package code<\/span>","<span class='gt_from_md'>Help with BayesSpace clustering<\/span>","<span class='gt_from_md'>Data science organization.<\/span>","<span class='gt_from_md'>MatLab questions.<\/span>","<span class='gt_from_md'>Subsetting spe object for shiny app<\/span>","<span class='gt_from_md'>Brainomics meeting with collaborators<\/span>","<span class='gt_from_md'>Color bar for heatmap<\/span>","<span class='gt_from_md'>Discussion on how to recruit members<\/span>","<span class='gt_from_md'>VisiumIF DLPFC building spe<\/span>","<span class='gt_from_md'>OneDrive<\/span>","<span class='gt_from_md'>BrainSeq Phase II development browser<\/span>","<span class='gt_from_md'>make modified tissue_positions_list from DLPFC IF samples for running spotsopcheck on mock PNN visium sample<\/span>","<span class='gt_from_md'>Help Deploying Shiny App for DLPFC_IF<\/span>","<span class='gt_from_md'>Cross Reference help<\/span>","<span class='gt_from_md'>Career development<\/span>","<span class='gt_from_md'>Need some help to make a plot<\/span>","<span class='gt_from_md'>Guide to creating a module at JHPCE<\/span>","<span class='gt_from_md'>How to make a tweetorial for a pre-print<\/span>","<span class='gt_from_md'>File organization for a paper submission<\/span>","<span class='gt_from_md'>creating a nextflow pipeline and supporting docker<\/span>","<span class='gt_from_md'>In promptu meeting<\/span>","<span class='gt_from_md'>Help with purrr loops for WGCNA funtions<\/span>","<span class='gt_from_md'>Grant proposal feedback<\/span>","<span class='gt_from_md'>help with html output of spaceranger<\/span>","<span class='gt_from_md'>CV feedback<\/span>","<span class='gt_from_md'>Cover letter feedback<\/span>","<span class='gt_from_md'>Debugged code from <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/33\">https://github.com/LieberInstitute/spatialLIBD/issues/33<\/a><\/span>","<span class='gt_from_md'>Brainomics meeting with collaborators<\/span>","<span class='gt_from_md'>How to delete large files from GitHub history<\/span>","<span class='gt_from_md'>help with read10xVisiumWrapper()<\/span>","<span class='gt_from_md'>macOS and JHPCE laptop set<\/span>","<span class='gt_from_md'>writing papers in google docs and setting up links for figures<\/span>","<span class='gt_from_md'>Hey Maddy, I need something called “spot size”  on the 12 published DLPFC samples<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/VistoSeg/issues/12\">https://github.com/LieberInstitute/VistoSeg/issues/12<\/a><\/span>","<span class='gt_from_md'>applying size threshold to vistoseg outputs to get rid of smaller regions that are not nuclei<\/span>","<span class='gt_from_md'>Debugging JHPCE configuration files<\/span>","<span class='gt_from_md'>I’d like to consult with Louise about how to make a spatial plot in R using a data set from the HALO FISH-IF module.<\/span>","<span class='gt_from_md'>directed the person to the VistoSeg website for steps to run VistoSeg<\/span>","<span class='gt_from_md'>using toptable and duplicate correlation<\/span>","<span class='gt_from_md'>help with clustering portion of th build_spe<\/span>","<span class='gt_from_md'>Using the “redstone” software<\/span>","<span class='gt_from_md'>help with build_spe<\/span>","<span class='gt_from_md'>Input files for custom annotation with SPEAQeasy<\/span>","<span class='gt_from_md'>Advice on how to increase code coverate with testthat<\/span>","<span class='gt_from_md'>Help run trial shiny app deployment<\/span>","<span class='gt_from_md'>set up git repo for long read project<\/span>","<span class='gt_from_md'>The authenticity of host ‘github.com’<\/span>","<span class='gt_from_md'>Assistance with analyzing spatial data<\/span>","<span class='gt_from_md'>Follow-up on custom annotation with SPEAQeasy<\/span>","<span class='gt_from_md'>Plan how to make spatial registration R functions<\/span>","<span class='gt_from_md'>Adding lines to a GTF for use in SPEAQeasy with custom annotation<\/span>","<span class='gt_from_md'>Clarifying what role deconvolution can play on LCM-seq<\/span>","<span class='gt_from_md'>TODO<\/span>","<span class='gt_from_md'>Error message running SPEAQeasy<\/span>","<span class='gt_from_md'>How to log in to JHU-controlled laptop<\/span>","<span class='gt_from_md'>Setting up passwordless login<\/span>","<span class='gt_from_md'>VisiumIF loupe alignment<\/span>","<span class='gt_from_md'>Using custom annotation with SPEAQeasy<\/span>","<span class='gt_from_md'>Masking of blood vessels from PNN channel in PNN project<\/span>","<span class='gt_from_md'>How are InForm tiles arranged<\/span>","<span class='gt_from_md'>Using the caracol queue<\/span>","<span class='gt_from_md'>How to guide others learning DS<\/span>","<span class='gt_from_md'>Understand the covariates used in scRNA-seq cell type analysis for a project we are doing with striatal organoids<\/span>","<span class='gt_from_md'>“Megadepth usage to add a new annotation to hg38 file”<\/span>","<span class='gt_from_md'>How to build a cov_rse object using qSVA v1<\/span>","<span class='gt_from_md'>Debradation matrix for LIBD riboZ<\/span>","<span class='gt_from_md'>Work through qsvaR vignette together<\/span>","<span class='gt_from_md'>Generating bigWigs with SPEAQeasy<\/span>","<span class='gt_from_md'>Data type change of image data when loading into Python<\/span>","<span class='gt_from_md'>WGBS data analysis overview<\/span>","<span class='gt_from_md'>Help building for loops to analyze WGCNA ouput data<\/span>","<span class='gt_from_md'>I would like to learn how you ran tensorQTL on the MDD data, maybe with using the GPU nodes for it.<\/span>","<span class='gt_from_md'>contrast adjustment, intensity normalization, channel/gene description in PNN images<\/span>","<span class='gt_from_md'>Ideas for possible future analyses<\/span>","<span class='gt_from_md'>rename files on JHPCE through command line<\/span>","<span class='gt_from_md'>JHPCE array jobs and fasthplus<\/span>","<span class='gt_from_md'>Provide comparison table for different metrics obtained from image meta data in MATLAB, for Visium IF workflow<\/span>","<span class='gt_from_md'>snRNA-seq and deconvolution overview<\/span>","<span class='gt_from_md'>Access to the gene info in the rowData of the Summarized Experiment objects to annotate eqtl results<\/span>","<span class='gt_from_md'>Explain Tangram, scRNAseq count matrix, spe object<\/span>","<span class='gt_from_md'>Explain the concept of Squidpy paper<\/span>","<span class='gt_from_md'>tiling and metadata changes in PNN project<\/span>","<span class='gt_from_md'>explaination on Image metadata, bit depth<\/span>","<span class='gt_from_md'>Need for rotation of visium array images and its spatial coordinnnate in spe object<\/span>","<span class='gt_from_md'>Debugging seg fault with tensorqtl module<\/span>","<span class='gt_from_md'>Using tensorqtl with a GPU<\/span>","<span class='gt_from_md'>X11 and SSH issues<\/span>","<span class='gt_from_md'>VistoSegIF pipeline<\/span>","<span class='gt_from_md'>accessing the ‘gpu’ node at JHPCE correctly<\/span>","<span class='gt_from_md'>Transfer data from JHPCE to Neural_Plasticity<\/span>","<span class='gt_from_md'>Transfer data Neural_Plasticity to JHPCE<\/span>","<span class='gt_from_md'>JHPCE directory organization, repo linking, making commits<\/span>","<span class='gt_from_md'>opening python, R, MATLAB console interactive session on JHPCE<\/span>","<span class='gt_from_md'>Troubleshooting ExprssionSet conversion<\/span>","<span class='gt_from_md'>glitch in CAPTure tutorial about input data types to a function<\/span>","<span class='gt_from_md'>Help using BisqueRNA, getting snRNA-seq into correct input format, chosing refrence vs. marker based deconvolution<\/span>","<span class='gt_from_md'>creating a ‘samples.manifest’ and choosing trimming settings<\/span>","<span class='gt_from_md'>Removing smaller objects from VistoSeg segmentations<\/span>","<span class='gt_from_md'>applying kmeans to CA3 mouse RNAscope data to obtain gene expression levels<\/span>","<span class='gt_from_md'>JHPCE file permissions and ACLs<\/span>","<span class='gt_from_md'>running loupe browser for HPC images<\/span>","<span class='gt_from_md'>countSpots from VistoSeg throwing errors for one DLPFC sample from round4<\/span>","<span class='gt_from_md'>Resolve issues with comments<\/span>","<span class='gt_from_md'>using ‘list.files’ in R<\/span>","<span class='gt_from_md'>batch processing - dotdotdot, on local machine<\/span>","<span class='gt_from_md'>error in transferring Visuim HPC data to JHPCE<\/span>","<span class='gt_from_md'>using awk<\/span>","<span class='gt_from_md'>Debugging help with rse_tx object<\/span>","<span class='gt_from_md'>qc general scripts, rnaseq scripts general, ggplot histogram with breaks<\/span>","<span class='gt_from_md'>We would love an opinion on our (now cell-type assigned) single nuclei rna-seq data, i.e. how they look - if anything is off, advice for improvement…<\/span>","<span class='gt_from_md'>I am trying to compute DEGs from thefunction registration_stats_enrichment…\nError in limma::makeContrasts(contrasts = z, levels = registration_model) :\nThe levels must by syntactically valid names in R<\/span>","<span class='gt_from_md'>Transfering files between JHPCE disks<\/span>","<span class='gt_from_md'>running spotspotcheck for SFN<\/span>","<span class='gt_from_md'>doubts on VistoSeg<\/span>","<span class='gt_from_md'>help with git<\/span>","<span class='gt_from_md'>debugging SPEAQeasy errors<\/span>","<span class='gt_from_md'>Presentation preparation feedback<\/span>","<span class='gt_from_md'>help with ‘umask’ and git<\/span>","<span class='gt_from_md'>issues with regioprops function in dotdotdot<\/span>","<span class='gt_from_md'>Fiducial frame enhancement for Visium IF images using preview on MAC<\/span>","<span class='gt_from_md'>running Loupe browser for mannual alignment<\/span>","<span class='gt_from_md'>merging FASTQ files automatically via SPEAQeasy<\/span>","<span class='gt_from_md'>Creating cyberduck bookmarks for JHPCE paths<\/span>","<span class='gt_from_md'>writing shell script for countNuclei function in VistoSeg<\/span>","<span class='gt_from_md'>Explaining Github issues, history and version control<\/span>","<span class='gt_from_md'>Explaining SPE (Spatial Experiment) SCE (Single Cell Experiment) objects<\/span>","<span class='gt_from_md'>syntax error in shell script<\/span>","<span class='gt_from_md'>CellProfiler H&amp;E segmentations<\/span>","<span class='gt_from_md'>Trouble with implementing RNAscope function for humans<\/span>","<span class='gt_from_md'>Discussion about rerunning FUSION/TWAS with new GWAS with extra attention paid to the number of samples for each brain region<\/span>","<span class='gt_from_md'>identifying SPEAQeasy error<\/span>","<span class='gt_from_md'>Error running VistoSeg::spospotcheck (Error in spotspotcheck&gt;start (line 60))<\/span>","<span class='gt_from_md'>Error while running rnascope_mouse - line 159<\/span>","<span class='gt_from_md'>parameter tuning for dotdotdot to split the nuclei clusters<\/span>","<span class='gt_from_md'>improve lipofuscin masking in dotdotdot<\/span>","<span class='gt_from_md'>dynamically name structures in matlab based on user inputs<\/span>","<span class='gt_from_md'>using map to cbind a list of six to its corresponding list of lists. The second is a list of 6, each a list of 3.<\/span>","<span class='gt_from_md'>resuming SPEAQeasy after an issue in the counts step<\/span>","<span class='gt_from_md'>downloading FASTQ files from SRA<\/span>","<span class='gt_from_md'>Explain how to merge 2 data tables in R<\/span>","<span class='gt_from_md'>Explain public keys and the authorized_keys file on JHPCE, removed duplicate public keys for the local machine to resolve ssh key login<\/span>","<span class='gt_from_md'>Using purrr and map to loop over datasets and apply qsva<\/span>","<span class='gt_from_md'>resolving issues with multiple keys in the authorized_keys file<\/span>","<span class='gt_from_md'>Help with JHPCE login issues<\/span>","<span class='gt_from_md'>explanation of the methods and algorithm of a paper we are reviewing together<\/span>","<span class='gt_from_md'>steps to resolve image segmentation of LC heated samples round 2<\/span>","<span class='gt_from_md'>How to write a Twitter thread for a paper<\/span>","<span class='gt_from_md'>Forming a data frame from data in multiple files<\/span>","<span class='gt_from_md'>Refining an abstract submission<\/span>","<span class='gt_from_md'>Git branches<\/span>","<span class='gt_from_md'>DNA genotyping overview<\/span>","<span class='gt_from_md'>DNA genotyping overview<\/span>","<span class='gt_from_md'>Optimizing DAPI segmentation in dotdotdot<\/span>","<span class='gt_from_md'>Overview of the RNA-seq universe<\/span>","<span class='gt_from_md'>Explain how awk works<\/span>","<span class='gt_from_md'>Issues with SUFI<\/span>","<span class='gt_from_md'>Resolve errors produced by dotdotdot on SUFI output<\/span>","<span class='gt_from_md'>R Code to create list of files to be processed through VisiumLIBD<\/span>","<span class='gt_from_md'>Setup aliases in bashrc<\/span>","<span class='gt_from_md'>debugging SPEAQeasy errors<\/span>","<span class='gt_from_md'>getting SPEAQeasy running<\/span>","<span class='gt_from_md'>formatting a LaTeX document and bookdown website<\/span>","<span class='gt_from_md'>Setting up a team at LIBD<\/span>","<span class='gt_from_md'>recreating spatstat plots with more features<\/span>","<span class='gt_from_md'>Helping understand errors from VisiumLIBD<\/span>","<span class='gt_from_md'>Helping understand errors from VisiumLIBD<\/span>","<span class='gt_from_md'>Help indexing a possibly corrupt VCF file<\/span>","<span class='gt_from_md'>Running Genome Studio on n = 200, generating PLINK outputs<\/span>","<span class='gt_from_md'>How to set up a GitHub actions-powered website<\/span>","<span class='gt_from_md'>Specifying output logs on SGE<\/span>","<span class='gt_from_md'>Debugging ggplot code<\/span>","<span class='gt_from_md'>Planning session for getting up to speed with SpatialExperiment, spatialLIBD, here and ggplot2<\/span>","<span class='gt_from_md'>How to choose and add a licence to a GitHub repo<\/span>","<span class='gt_from_md'>Designing an MS-level class<\/span>","<span class='gt_from_md'>making a list of filenames in a specific folder and subfolder in R<\/span>","<span class='gt_from_md'>Rectified issues for batch running emma’s RNAscope data<\/span>","<span class='gt_from_md'>create shell script for batch running images on VisiumLIBD<\/span>","<span class='gt_from_md'>Locating public data at JHPCE + bamtofastq workflow<\/span>","<span class='gt_from_md'>TWAS questions<\/span>","<span class='gt_from_md'>How to transfer recount3 data<\/span>","<span class='gt_from_md'>How to get started with R<\/span>","<span class='gt_from_md'>Brainstorming session on collaboration<\/span>","<span class='gt_from_md'>Manhattan plot making and tstatistic<\/span>","<span class='gt_from_md'>addressing SPEAQeasy error<\/span>","<span class='gt_from_md'>got mds score for the vcf she presented using plink<\/span>","<span class='gt_from_md'>FUSION / TWAS Walkthrough<\/span>","<span class='gt_from_md'>mds for vcfs<\/span>","<span class='gt_from_md'>checking plots and data structure in clusterprofiler<\/span>","<span class='gt_from_md'>Havent got my mds yet so making the meeting for Friday<\/span>","<span class='gt_from_md'>questions about expression_cut and pca plots<\/span>","<span class='gt_from_md'>Using GenomeStudio<\/span>","<span class='gt_from_md'>How to prepare for a LIBD seminar part 4<\/span>","<span class='gt_from_md'>troubleshoot SUFI output and figured it was not compatible with dotdotdot<\/span>","<span class='gt_from_md'>gene ontology for genesets<\/span>","<span class='gt_from_md'>Differential Expression help (continuation of 5/20)<\/span>","<span class='gt_from_md'>How to prepare for a LIBD seminar part 3<\/span>","<span class='gt_from_md'>Grant application review<\/span>","<span class='gt_from_md'>PredictDB weights<\/span>","<span class='gt_from_md'>Help preparing a figure for a grant application<\/span>","<span class='gt_from_md'>How to prepare for a LIBD seminar part 2<\/span>","<span class='gt_from_md'>Making plots for DE studies<\/span>","<span class='gt_from_md'>Making a module<\/span>","<span class='gt_from_md'>Configuring a parallel series of bash scripts<\/span>","<span class='gt_from_md'>questions about processing genotype data<\/span>","<span class='gt_from_md'>Setting up a SPEAQeasy run<\/span>","<span class='gt_from_md'>Making plots for DE studies<\/span>","<span class='gt_from_md'>Setting up covr using GitHub actions in an R package<\/span>","<span class='gt_from_md'>How to get started with computational biology<\/span>","<span class='gt_from_md'>How to insert a figure using the Oxford Bioinfomratics Overleaf LaTeX template<\/span>","<span class='gt_from_md'>PredictDB weights<\/span>","<span class='gt_from_md'>How to prepare for a LIBD seminar<\/span>","<span class='gt_from_md'>Running SPEAQeasy- follow up<\/span>","<span class='gt_from_md'>TWAS prep<\/span>","<span class='gt_from_md'>Constructing an array job from an existing sequential shell script<\/span>","<span class='gt_from_md'>Setting up FUSION / TWAS for practice<\/span>","<span class='gt_from_md'>Running SPEAQeasy- follow up<\/span>","<span class='gt_from_md'>Running SPEAQeasy<\/span>","<span class='gt_from_md'>Running SPEAQeasy- follow up<\/span>","<span class='gt_from_md'>Running FUSION / TWAS<\/span>","<span class='gt_from_md'>JHPCE set up on mac<\/span>","<span class='gt_from_md'>Running SPEAQeasy<\/span>","<span class='gt_from_md'>running and trouble shooting VisiumIF image splitting code<\/span>","<span class='gt_from_md'>JHPCE set up on mac<\/span>","<span class='gt_from_md'>Running the old RNA-seq pipeline on some SRA data<\/span>","<span class='gt_from_md'>Resolving issues with running dotdotdot on 20X RNAscope images<\/span>","<span class='gt_from_md'>runnning Hiplex registration<\/span>","<span class='gt_from_md'>General guidance on how to learn how to do differential expression analyses<\/span>","<span class='gt_from_md'>Nextflow overview for work with imputation pipeline<\/span>","<span class='gt_from_md'>creating logs for issues from running SUFI (spectral unmixing of fluorescent biological images)<\/span>","<span class='gt_from_md'>steps to troubleshoot issues with stitched multi tile RNAscope images<\/span>","<span class='gt_from_md'>spatialLIBD support<\/span>","<span class='gt_from_md'>Need help understanding the sce - single cell object<\/span>","<span class='gt_from_md'>help doing a power analysis for RNA-seq data<\/span>","<span class='gt_from_md'>Creating a documentation website using bookdown<\/span>","<span class='gt_from_md'>Clarification of LIBD support<\/span>","<span class='gt_from_md'>General advice<\/span>","<span class='gt_from_md'>running nucleiCount for spatial DLPFC samples<\/span>","<span class='gt_from_md'>Download SRA data and run SPEAQeasy<\/span>","<span class='gt_from_md'>Explain the feasibility of an on-demand end user interface for DEG and eQTLs<\/span>","<span class='gt_from_md'>extracting p-values from linear mixed effect model for Calcium imaging data<\/span>","<span class='gt_from_md'>Using an LIBD module<\/span>","<span class='gt_from_md'>Change point size in vis_clust_grid<\/span>","<span class='gt_from_md'>Download SRA data and run SPEAQeasy<\/span>","<span class='gt_from_md'>Attempted SPEAQeasy installation on MacOS<\/span>","<span class='gt_from_md'>visium image segmentation<\/span>","<span class='gt_from_md'>troubleshooting memory usage for VNS function from VisiumLIBD<\/span>","<span class='gt_from_md'>get effect size for calcium imaging<\/span>","<span class='gt_from_md'>split histology images into respective capture areas<\/span>","<span class='gt_from_md'>x-quartz installation<\/span>","<span class='gt_from_md'>BSP2 eQTLs with a subset of SNPs<\/span>","<span class='gt_from_md'>qSVA explanation and code review<\/span>","<span class='gt_from_md'>How to write a small methods paper<\/span>","<span class='gt_from_md'>resolve figure generation error in human rnascope script from dotdotdot<\/span>","<span class='gt_from_md'>Maker finding for cell types in single cell data<\/span>","<span class='gt_from_md'>image analysis on QuPath<\/span>","<span class='gt_from_md'>Advice for writing papers<\/span>","<span class='gt_from_md'>Clarification about how recent CMC data was processed with SPEAQeasy<\/span>","<span class='gt_from_md'>Our group’s set-up and approach for writing papers with google docs, and managing citations<\/span>","<span class='gt_from_md'>data transfer from LIBD server to JHPCE<\/span>","<span class='gt_from_md'>Submitting/debugging bash job for dotdotdot on cluster<\/span>","<span class='gt_from_md'>I need help in: downloading data from SRA to JHPCE, running SPEAQeasy to create rse object (one session will likely not be enough).<\/span>","<span class='gt_from_md'>I need help with setting up Mobaexterm. Unable to open new terminals and folder permission issues<\/span>","<span class='gt_from_md'>explaination of Owner/Group/Other permissions for files on JHPCe<\/span>","<span class='gt_from_md'>download repository from github to cluster and run dotdotdot on cluster<\/span>","<span class='gt_from_md'>Visualizing RPKM valus for genes in Humna and Primarte data<\/span>","<span class='gt_from_md'>BSP2 LIBD eQTLs request<\/span>","<span class='gt_from_md'>JHPCE ssh keypair set up<\/span>","<span class='gt_from_md'>dotdotdot parameter tuning for Ema’s rat RNAscope data<\/span>","<span class='gt_from_md'>Github ssh key set up<\/span>","<span class='gt_from_md'>Work on habenula deconvolution and debug the deconvolution package<\/span>","<span class='gt_from_md'>Help determining what data is available and what can be used for an analysis<\/span>","<span class='gt_from_md'>group_by bug for get_mean_ratio<\/span>","<span class='gt_from_md'>Guidance on how to download data from a study in GEO<\/span>","<span class='gt_from_md'>Career development<\/span>","<span class='gt_from_md'>biocthis macOS GHA bug<\/span>","<span class='gt_from_md'>Questions related to accessing results from the DG paper by Andrew Jaffe<\/span>","<span class='gt_from_md'>Correlation values from labeledHeatmap<\/span>","<span class='gt_from_md'>Help with importing HDF5 file<\/span>","<span class='gt_from_md'>Resolving issues in phenotype data between genotype, expression, and LIBDpheno<\/span>","<span class='gt_from_md'>jhpce set up on mac<\/span>","<span class='gt_from_md'>Inline formula throwing an error for interaction term<\/span>","<span class='gt_from_md'>run dotdotdot pipeline local computer and cluster<\/span>","<span class='gt_from_md'>how to open matlab (GUI and command line) on JHPCE<\/span>","<span class='gt_from_md'>Matching Variants Between Two VCF Objects<\/span>","<span class='gt_from_md'>how to connect to LIBD servers on windows and mac<\/span>","<span class='gt_from_md'>Accesing CMC RNA-seq data<\/span>","<span class='gt_from_md'>Ran Genome Studio, installed PLINK input plugin<\/span>","<span class='gt_from_md'>ChipQC ‘names’ attribute [9] must be the same length as the vector [2]’<\/span>","<span class='gt_from_md'>data transfer from LIBD server to JHPCE<\/span>","<span class='gt_from_md'>download matlab on local computer through JHU software portal<\/span>","<span class='gt_from_md'>Help clustering sce data<\/span>","<span class='gt_from_md'>Using reshape package to transform table for plotting<\/span>","<span class='gt_from_md'>Help adapting ggplot script for TWAS Z-score<\/span>","<span class='gt_from_md'>RDavid Error<\/span>","<span class='gt_from_md'>Differences between recount2’s “bp_length” and SPEAQeasy’s “Length”<\/span>","<span class='gt_from_md'>How does one modify a single value of a field in a seqinfo object?<\/span>","<span class='gt_from_md'>GitHub actions workflow issue with macOS on R-devel<\/span>","<span class='gt_from_md'>LC repo organization<\/span>","<span class='gt_from_md'>Extracting Sequences from Fasta Files with Rsamtools<\/span>","<span class='gt_from_md'>3D volume extraction in Matlab<\/span>","<span class='gt_from_md'>Fixed two bugs on recount3 related to locate_url() for the BigWig files.<\/span>","<span class='gt_from_md'>Fixed a bug on biocthis for release branches<\/span>","<span class='gt_from_md'>Issue with knitcitations<\/span>","<span class='gt_from_md'>Help with SpaceRanger<\/span>","<span class='gt_from_md'>Issue with JHPCE file permissions<\/span>","<span class='gt_from_md'>recount3 GTEx data access<\/span>","<span class='gt_from_md'>customize dotdotdot to his data<\/span>","<span class='gt_from_md'>Learning Lexical Scoping Rules<\/span>","<span class='gt_from_md'>Changing Row Names of MultiAssayExperiment Assay<\/span>","<span class='gt_from_md'>exceRpt pipeline counts<\/span>","<span class='gt_from_md'>issue running spaceranger<\/span>","<span class='gt_from_md'>Plotly help if possible? I’m trying to export an html file that allows the user to look at each snp/gene on the plot, and I am almost successful. Leo recommended that I talk to you on this matter.<\/span>","<span class='gt_from_md'>Help running speakeasy on MARCC and JHPCE please<\/span>","<span class='gt_from_md'>Question: Subset center regions of a set of different length sequences<\/span>","<span class='gt_from_md'>Ballgown stattest<\/span>","<span class='gt_from_md'>Question: Cannot Install gmapR Bioconductor Package<\/span>","<span class='gt_from_md'>copy number variation analysis (CNV)<\/span>","<span class='gt_from_md'>Help with spatstat<\/span>"],"html":true,"width":200,"align":"left"},{"id":"public_notes","name":"public_notes","type":"character","na":"NA","minWidth":125,"style":"function(rowInfo, colInfo) {\nconst rowIndex = rowInfo.index + 1\n}","cell":["<span class='gt_from_md'>We worked on <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/106\">https://github.com/LieberInstitute/spatialLIBD/issues/106<\/a> and eventually resolved it thanks to answers from Gordon Smyth and James W. MacDonald at <a href=\"https://support.bioconductor.org/p/9161754\">https://support.bioconductor.org/p/9161754<\/a>. See the GitHub issue thread for more details about the differences between <code>edgeR::cpm(log = TRUE)<\/code> and <code>scuttle::logNormCounts()<\/code>. The documentation at <code>spatialLIBD::registration_pseudobulk()<\/code> has been expanded to further explain this difference.<\/span>","<span class='gt_from_md'>spatialLIBD::app_server seems to be breaking for layer_stat_cor_plot when retriving colors, this looked more like an issues with the spatialLIBD code than the users application. Will investagate further<\/span>","<span class='gt_from_md'>Researcher is Intrested in definind what makes a spatial domain unique from the others. I recomended to review the top enrichemnt and parwise genes from the modeling results. Running MeanRatio may also be informative. I also recomended reviewing litature to see if there is any other work that may help caracterize this domain.<\/span>","<span class='gt_from_md'>I pointed to the public rstats google sheet in case there were helpful sessions, then recommended BiocMAP as our pipeline for methylation data<\/span>","<span class='gt_from_md'>Since the commit with the large file was recent, I recommended using <code>git log<\/code> to identify when the commit occurred, then use <code>git reset --soft<\/code> to undo commits (without modifying files) after that point<\/span>","<span class='gt_from_md'>We discussed how I can help on a K08 grant application, the support structure my team provides via the DSgs system, and I shared a draft support letter from another grant application.<\/span>","<span class='gt_from_md'>The negative values are probably 0s in the counts matrix, I agreed that the negative values in vaisualizations are tricky to expain, it would be more intutaive if they were also zeros. I am not sure if changeing the prior.count to &gt; 1 would help. I suggested plotting the pseudobulk counts as a sanity check and exploring the count values that end up as negtive in logcounts. Researcher may leave an issue on spatialLIBD about this.<\/span>","<span class='gt_from_md'>We talked about the process of writing an NIH R01 grant, how to organize some grant related files, options for how to collaborate and the related budget implications.<\/span>","<span class='gt_from_md'>We talked about how to train a JHU PhD student, how to organize learning tasks, and updates on a mouse snRNA-seq project.<\/span>","<span class='gt_from_md'>We talked about applying to JHU PhD programs at BSPH, how to best communicate via email with busy faculty (short emails asking specific questions), and other aspects of studying in Baltimore.<\/span>","<span class='gt_from_md'>We debugged code for pseudobulkDGE, we found several silent errors that were leading to empty output, but not errors. After correcting some of the variables we got closer to expected output, but still need to further debug. I think the function could be improved, prehaps we’ll create a github issue<\/span>","<span class='gt_from_md'>We talked about LIBD, the team history, team culture, how we are organized, how we train people, and other related topics.<\/span>","<span class='gt_from_md'>For a specific set of boxplots, I described how grouping and summarization should work in order to produce the expected number of points at each x value. I also emphasized keeping data in long format for plotting<\/span>","<span class='gt_from_md'>We discussed some career options and what math background classes would be useful to take before applying to a PhD in biostatistics.<\/span>","<span class='gt_from_md'>We discussed the DE analysis steps, focusing on pseudobulking. I explained the overall workflow I am using, including the use of pseudoBulkDGE(), the covariates to consider, and how to set coefficient in the design matrix for analysis.<\/span>","<span class='gt_from_md'>Since we couldn’t directly access the SPEAQeasy configuration file (it was in another user’s home directory), I recommended reinstalling SPEAQeasy in a new location. The error was indeed due to being unable to read configuration variables<\/span>","<span class='gt_from_md'>We talked about results from <code>multiBatchPCA()<\/code>, and ultimately preferredd the results from computing PCs in each batch separately, then we interpreted the plots to evaluate whether biological or technical signal dominated PC1 and PC2 within each batch.<\/span>","<span class='gt_from_md'>We talked about new data being generated, the questions we have of interest, and the overall timeline.<\/span>","<span class='gt_from_md'>Feedback on how to document code for working with HALO/RNAScope data. From example scripts I think it would be best to build an R package. I provided some examples of what that looks like. They are also thinking a shiny element would be good, that adds additional, but managable, complexity.<\/span>","<span class='gt_from_md'>I provided feedback on both technical and aesthetic aspects of the presentation. On the technical side, I pointed out areas where additional context might help fill potential gaps. Aesthetically, I suggested  enhance clarity, such as adding axis labels to plots, maintaining consistent terminology across slides, and including descriptive titles or annotations to make visual elements more self-explanatory.<\/span>","<span class='gt_from_md'>We discussed ideas for how to proceed with a trial.<\/span>","<span class='gt_from_md'>We went over how to use <code>pseudoBulkDGE()<\/code> for performing a DGE analysis at the donor level, and what covariates we would need to compute.<\/span>","<span class='gt_from_md'>We reviewed the key details for the spatialDLPFC snRNA-seq data. Key colums are cellType_layer, and cellType_broad_hc, also dropping sce$cellType_hc == “Ambiguous”. I noted our documentation for fetch_data() could benifit from more details about this dataset.<\/span>","<span class='gt_from_md'>I described how one key input needs a dataframe and the other needs a list of dataframes. I showed how to loop over various reference datasets with <code>lapply<\/code><\/span>","<span class='gt_from_md'>I provided a brief explaination of PCA used to visualize variance patterns such as clustering by cell type, condition, or batch. I highlighed diferent situation, for example, points close together represent similar profiles, while separation along PC1 or PC2 often indicates major sources of variation; also explained that  coloring by metadata (like sample ID or treatment) helps identify biological or technical drivers, that could be rare populations or quality issues.<\/span>","<span class='gt_from_md'>we worked on filling the rowData of a sce object after converting from seurat, as the gene ids are key for running layer_stat_cor later in the spatial registration process<\/span>","<span class='gt_from_md'>I described the general concept of spatial registration, pointed to <code>spatialLIBD::registration_wrapper()<\/code> as a key function, then helped prepare input data and parameters for running that function, starting with a Seurat object. This seemed to involve manually adding <code>rowData()<\/code> columns to the converted SingleCellExperiment.<\/span>","<span class='gt_from_md'>They noted the exit code of 137, which I agreed (often) indicated a job being killed due to exceeding  memory limits. I suggested doubling or greatly increasing the memory for the <code>PreprocessInputs<\/code> process in <code>conf/jhpce.config<\/code>.<\/span>","<span class='gt_from_md'>We refreshed our memory on where the project stands and what would be some of the next steps that need to be performed.<\/span>","<span class='gt_from_md'>I noted that as a Nextflow pipeline, parallelism is built in to the design of SPEAQeasy; processes will execute as soon as they have input data ready, and many SLURM jobs execute at once. I noted that most processes shouldn’t need memory modifications for large datasets, since they usually operate on one sample at a time, but I mentioned the <code>CountObjects<\/code> process as a possible exception and noted it could be edited in <code>conf/jhpce.config<\/code>.<\/span>","<span class='gt_from_md'>I showed how you can obtain the sample information (colData) from an RSE gene-level <code>recount3<\/code> object, then by sample ID, match with the colData you have on a small RSE made manually, and replace the colData info. The small RSE has counts computed with <code>megadepth<\/code>. This could end up being a <code>recount3<\/code> or <code>megadepth<\/code> helper function. In any case, having the full <code>recount3<\/code> colData enables then the use of functions such as <code>transform_counts()<\/code> to convert the <code>recount3<\/code> / <code>megadepth<\/code> base-pair level raw counts into read counts for downstream analyses.<\/span>","<span class='gt_from_md'>I mentioned that one option involves contacting your HPC admins (JHPCE BitSupport for us) and asking for an increased allocation of computing cores. That way you can run more jobs in parallel. Though ultimately it’s best to meet with Nick Eagles for a DSgs as Nick is the author and maintainer of SPEAQeasy.<\/span>","<span class='gt_from_md'>I explained the Harmony plots before and after batch correction, highlighting differences between the Harmony pure plot and Harmony UMAP plot. We also discuss quick tests to assess whether low statistical power was affecting DEG identification. I suggested applying a power function (as in WGCNA) and duplicating samples to evaluate dataset limitations, while also discussing the constraints of these approaches.<\/span>","<span class='gt_from_md'>We reviewed the concepts in a blog post about the meaning of different p-value distributions in multiple-testing scenarios, and I explained how p-value distributions connect to the pi1 statistic as used to estimate replication rate<\/span>","<span class='gt_from_md'>We discussed the links at <a href=\"https://www.one-tab.com/page/oKFdo7_aRT-3ffoVddJ0sQ\">https://www.one-tab.com/page/oKFdo7_aRT-3ffoVddJ0sQ<\/a>. I suggested using <code>DeconvoBuddies::get_mean_ratio()<\/code> on the public data available from GEO. Then checking if that cluster should be merged with another one, and if so, then repeating the Mean Ratio analysis.<\/span>","<span class='gt_from_md'>I looked over some code and clarified that the pi1 statistic from the qvalue package can be used to estimate replication (the proportion of DE genes from one dataset expected to be DE in another)<\/span>","<span class='gt_from_md'>We reviewed code for using <code>qvalue::qvalue()<\/code> to assesss replication rate of a set of DEGs using p-values from a replication dataset. This used code written by Gabriel Hoffman.<\/span>","<span class='gt_from_md'>I described the meaning of the syntax used to redirect output to a custom log name as internally created by <code>slurmjobs::job_loop()<\/code>. I described the <code>-f<\/code> parameter to <code>rm<\/code> for removing output files in a <code>run_all.sh<\/code> script without failing if they don’t exist<\/span>","<span class='gt_from_md'>Discussed our work, life in Baltimore, and other aspects related to choosing a lab to work for during a PhD.<\/span>","<span class='gt_from_md'>We reviewed the steps leading up to DEG identification, as she wanted to confirm the correctness of her process. I shared a couple of tests I typically run to verify range and summarize various metrics, enabling her to check if her target thresholds are present in the matrix data.<\/span>","<span class='gt_from_md'>I investigated the source of the issue and recommended checking for typo in samples.manifest, which ended up being the issue<\/span>","<span class='gt_from_md'>We discussed a strategy for responding to reviewers.<\/span>","<span class='gt_from_md'>I showed how to access the no qSV results from the <code>outGene0<\/code> object listed at <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/casectrl_DLPFC.R#L43\">https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/casectrl_DLPFC.R#L43<\/a> that is part of <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/casectrl_DLPFC.R#L59\">https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/casectrl_DLPFC.R#L59<\/a>. I also suggested that they could use the BSP1 data aligned with HISAT2 as part of the BSP2 paper by re-runnign the code at <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase1_qsv/casectrl_DLPFC_allFeatures.R#L43\">https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase1_qsv/casectrl_DLPFC_allFeatures.R#L43<\/a> and basically not adding the qSVs if they don’t want them. I also showed them the BSP3 code from the habenulaPilot paper at <a href=\"https://github.com/LieberInstitute/Habenula_Pilot/blob/99b5d8b0d5486b475d8b2e21920e9ddb1b71ffc4/code/15_compare_vs_BrainSeq/01_Hb_vs_BSP2_BSP3_DG.R#L32\">https://github.com/LieberInstitute/Habenula_Pilot/blob/99b5d8b0d5486b475d8b2e21920e9ddb1b71ffc4/code/15_compare_vs_BrainSeq/01_Hb_vs_BSP2_BSP3_DG.R#L32<\/a>.<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>We got him setup at JHU Genomics Collective’s Slack workspace, talked about DSgs system, LIBD rstats club, and then also JHPCE new PI form. We contacted Francesca and John for more details needed for the new PI form.<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>Demoed how to use sciwheel and cross ref in google docs for witing paper<\/span>","<span class='gt_from_md'>We found that par does not interact well with patchwork plot objects. To arrange a list of patchwork objects in to 4 rows use Reduce(”/”, plot_list)<\/span>","<span class='gt_from_md'>I mentioned specific goals I made and the types/ categories of goals that could be appropriate (specific progress towards a manuscript, specific presention-related goals, etc)<\/span>","<span class='gt_from_md'>We discussed strategies set qunatiative goals, such as number of presentations or breaking large projects in to individual, completable steps<\/span>","<span class='gt_from_md'>Check that the input coordinates are in hg38 and not in hg19 positions. Use <code>disjoin()<\/code> on the exons of a given gene to find the disjoint exon pieces, quantify those, then add them up to get the gene expression. Do this instead of quantifying start to end (aka introns might be noisy).<\/span>","<span class='gt_from_md'>I noted the 100 CPU and 1TB RAM limit per user, though noted the number of array tasks for a user can be quite high<\/span>","<span class='gt_from_md'>I guessed that the issue was a memory limit and we confirmed by loading the object in R and using <code>gc()<\/code> (<code>lobstr::obj_size()<\/code> would’ve been more precise). I recommended using HDF5-backed assays<\/span>","<span class='gt_from_md'>See links from today’s session <a href=\"https://www.one-tab.com/page/UkWtmL03STGtP2Y1Gojdig\">here<\/a>. To debug the issue at hand, I advised manually running <a href=\"https://github.com/LieberInstitute/fusion_twas/blob/jhpce/FUSION.compute_weights.R\">https://github.com/LieberInstitute/fusion_twas/blob/jhpce/FUSION.compute_weights.R<\/a> for one input gene set of files.<\/span>","<span class='gt_from_md'>I corrected a few environment variables in a partially working script to sbatch and noted the ‘#SBATCH –array=’ syntax for job arrays<\/span>","<span class='gt_from_md'>We revieded the project history and  orgainzation to help find needed scripts<\/span>","<span class='gt_from_md'>I shared FICTURE results for the LC dataset and we discussed<\/span>","<span class='gt_from_md'>Tools like scater::ploExpression or DeconvoBuddies::plotGeneExpress() will help plot gene expressio vs. catagorey. jafflab::cleaningY() will regress out varaibles controled for in DEG and provide the best representation of what the “model” sees. We could consider adding the “clean” expression as an assay durring the “modeling” pipeline from spatialLIBD.<\/span>","<span class='gt_from_md'>debugging git pull with conflicts<\/span>","<span class='gt_from_md'>See <a href=\"https://community-bioc.slack.com/archives/CLUJWDQF4/p1733847223678279?thread_ts=1733845668.397749&amp;cid=CLUJWDQF4\">https://community-bioc.slack.com/archives/CLUJWDQF4/p1733847223678279?thread_ts=1733845668.397749&amp;cid=CLUJWDQF4<\/a> for more information. Basically, the <code>bib<\/code> object was not well made and this caused a “subscript out of bounds” error on R devel (4.5) and bioc devel (3.21) that wasn’t an error before.<\/span>","<span class='gt_from_md'>I helped update <a href=\"https://github.com/LieberInstitute/spatial_hpc/tree/gh-pages\">https://github.com/LieberInstitute/spatial_hpc/tree/gh-pages<\/a>. Note that the html file has a base64 representation of the image, so commit 1 is to update the image, commit 2 is to re-render the html file (once the new image image is available). Also, the image wasn’t been displayed at <a href=\"https://github.com/LieberInstitute/spatial_hpc\">https://github.com/LieberInstitute/spatial_hpc<\/a> since it was too big. So I resized it so the max dimension would be 1,200 pixels (in this case, the width).<\/span>","<span class='gt_from_md'>Ultimately this person needs to update their macOS computer. They had avoided doing so for too long. See also <a href=\"https://github.com/jeroen/curl/issues/358\">https://github.com/jeroen/curl/issues/358<\/a> for more details from someone with a nearly identical issue.<\/span>","<span class='gt_from_md'>She ran into a large amount of package installation and loading failures while using the conda_R/devel module. I noted that while the R version periodically updates for this module, the user directory for packages stays the same. leading to conflicts. I noted deleting the user-specific directories was one failproof but extreme option<\/span>","<span class='gt_from_md'>Review results from q-tune, looks like it might not align with the desired resolution for the data. Review output from modelining analysis.<\/span>","<span class='gt_from_md'>I suggested using ‘–strand_mode “declare”’ to cirumvent possible issues with poor transcriptome pseudoalignment, and noted HISAT2 can be configured to penalize alignment mismatches more aggressively with the ‘–mp’ parameter<\/span>","<span class='gt_from_md'>Set up dataset for spatial registration, review code, plan for how to interpret output<\/span>","<span class='gt_from_md'>I noted that somehow the ~/.ssh directory had overly open permissions, leading to issues with git and SSHing into the cluster. We ran ‘chmod -R 700 ~/.ssh’, then later discovered the home directory as a whole needed a similar permissions fix<\/span>","<span class='gt_from_md'>The error hinted at a clash of gene names, which I traced back to having multiple samples using different spaceranger reference files<\/span>","<span class='gt_from_md'>We discussed the quality control, feature seelction, and batch corrections steps needed before clustering visium data<\/span>","<span class='gt_from_md'>I mentioned tricks I used to build a SpatialExperiment for Visium HD data and hacks for using ‘vis_gene’ and similar<\/span>","<span class='gt_from_md'>I suggested a ‘conda install’ command that might fix a missing system dependency seen when trying to install a partcular R package. I also noted that python dependencies could be installed in the same module (conda environment) if desired<\/span>","<span class='gt_from_md'>I described the installation process for SPEAQeasy and typical command-line parameters one might typically change<\/span>","<span class='gt_from_md'>We discovered one source of odd behavior stemming from not fully deleting the directory for a given Samui sample before reconstructing it<\/span>","<span class='gt_from_md'>I walked through Python code to check when building a Samui sample to properly align coordinates and the image, while showing gene expression<\/span>","<span class='gt_from_md'>I explained to him from scratch how to make a manual annotation using his own SpatialLIBD application. I explained to him how to select slides, select spots with lasso or one to one, and how to import a csv file for further analysis. I also explained to him the conceptual use of the annotation, how he can change the names of the labels and share the file with other people to work collaboratively.<\/span>","<span class='gt_from_md'>We comapred the app.R and deploy.R scripts to working ones, corrected some issues with the here function. This did not succesfully deploy the app (local version works). Likewise we matched the settings to the ERC spp settings, still failed to run. We may need a second set of eyes to help finetune the shiny.io settings or diagnose the issue here.<\/span>","<span class='gt_from_md'>We discussed the steps needed to prepare for spatial clustering: quality control, normalization, feature selection, demension reduction, batch correction THEN clustering w/ BayesSpace. We went over the clustering code from LFF ERC<\/span>","<span class='gt_from_md'>I explained to him where he can find the results summary report generated for Spaceranger. We focused mainly on the CSV file as it is more detailed. I explained to him the specific metrics he was interested in and where he can find on the 10x website the technical notes for 10x Genomics products for further questions.<\/span>","<span class='gt_from_md'>I described how we put any component of each manuscript figure in a directory to be stored on OneDrive (often consisting of individual plots in PDF form), and described how Sciwheel integrates with a google document<\/span>","<span class='gt_from_md'>I referenced <a href=\"https://github.com/LieberInstitute/spatialdACC/blob/main/code/09_samui/09_find_unannotated.R\">https://github.com/LieberInstitute/spatialdACC/blob/main/code/09_samui/09_find_unannotated.R<\/a> and described that the ‘id’ column in the Samui CSV refers to the index of the ‘coords.csv’ file for the Samui sample.<\/span>","<span class='gt_from_md'>Discussed tips for working with HALO data, and how to visualize<\/span>","<span class='gt_from_md'>I described how .gitignore files work and mentioned the best practice of putting them in the directory where files to ignore are present, not at the top level<\/span>","<span class='gt_from_md'>We discussed their grant and how my letter of support should be framed. I sent the letter at the end of the day.<\/span>","<span class='gt_from_md'><ol>\n<li>Check with <code>lobstr::obj_size()<\/code> how big your objects are. They might not be compatible with <a href=\"https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners\">memory limits provided by free GHA workers<\/a> (GitHub Actions). 2. <code>biocthis<\/code> ‘s GHA workflow tries to emulate the Biocondutor SPB (Single Package Builder). People can submit packages to BioC without having tested them on GHA, though then they might run into R CMD build / check / BiocCheck issues for the first time there and have a slower peer review process. 3. Consider using <code>rm()<\/code> once you are done with the first large object. 4. In general, it’s best to break long code chunks into smaller ones (and name them). That way, you can get more specific locations of the errors. Hence why we added that <code>stopifnot()<\/code> from my PR to formally check that the snapshot date is valid. 5. For debugging, it’s useful to have a local environment where you can reproduce in detail the issues. I didn’t have one ready before the DSgs for BioC 3.20, but if we meet again, I could be ready.<\/li>\n<\/ol>\n<\/span>","<span class='gt_from_md'>We went over the process of synching a GitHub package with the Bioconductor git server, what is the Bioconductor Single Package Builder (SBP), when we can tell that the SBP has been configured (and is thus ready to access new versions via bumps in the DESCRIPTION), and how biocthis’s GHA simulates part of what the SBP does but how they still have some differences. Ultimately in this case, GHA doesn’t have a timeout limit (15 min) that SBP does have, which is why SBP is erroring out while GHA is ok. We also ran into an issue with the development version of pkgdown.<\/span>","<span class='gt_from_md'>I noted that you must in general load scikit-learn models in using the same scikit-learn version they were trained in, and mentioned the cellpose 2.2 module in particular for an old model I trained.<\/span>","<span class='gt_from_md'>Reviewed the spatial registration pipeline: pseudobulking, modeling, correlation, and annotation<\/span>","<span class='gt_from_md'>We went over the response to the reviewers (from the journal) and clarified how to address some of the comments.<\/span>","<span class='gt_from_md'>I pointed to <a href=\"https://github.com/LieberInstitute/visiumStitched_brain/blob/devel/code/03_stitching/07_precast_unstitched.R\">https://github.com/LieberInstitute/visiumStitched_brain/blob/devel/code/03_stitching/07_precast_unstitched.R<\/a> and mentioned that some evidence from stitching analysis (in particular, <a href=\"https://github.com/LieberInstitute/visiumStitched_brain/blob/devel/plots/03_stitching/agreement_at_overlaps.pdf\">https://github.com/LieberInstitute/visiumStitched_brain/blob/devel/plots/03_stitching/agreement_at_overlaps.pdf<\/a>) suggested SVGs were slightly favorable to HVGs as input<\/span>","<span class='gt_from_md'>I provided an overview of what the team does, how we can help the OP, added them to Slack, the LIBD rstats club, and other intro first steps.<\/span>","<span class='gt_from_md'>We reviewed <a href=\"https://lcolladotor.github.io/bioc_team_ds/writing-papers.html\">https://lcolladotor.github.io/bioc_team_ds/writing-papers.html<\/a> and started a Google Doc, SciWheel, LIBD OneDrive for this paper.<\/span>","<span class='gt_from_md'>I helped debug an issue with a new R package. Basically, some files were not being included in the .Rbuildignore that should have been.<\/span>","<span class='gt_from_md'>I explained in more detail the FRiP score and TSS quality control metrics for AtlasXomics sequenced data for the pilot samples provided by Lieber for Hb and DLPFC. We compared the recommended thresholds with those obtained in the pilot samples. I emphasized that the FRiP should be greater than 0.3, but values ​​greater than 0.2 are also acceptable.<\/span>","<span class='gt_from_md'>She encountered an open bug with this function that I described. I noted a workaround was to rotate in increments (e.g. 90 degrees 3 times instead of 270 degress once).<\/span>","<span class='gt_from_md'>Trouble shot cyberduck so login works automatically with ssh keys<\/span>","<span class='gt_from_md'>Using as an example our <a href=\"https://libd.shinyapps.io/Habenula_Visium_raw/\">https://libd.shinyapps.io/Habenula_Visium_raw/<\/a>  app, I conitnued explaining her how to explore data from discrete variables, for example the 10x graph-clusters and PCs. I also explained her how explore a particular cluster of interest and a glimse of how to check the gene expresion using multiple features (multi-gene function).<\/span>","<span class='gt_from_md'>I recommended using a GUID-mapping table I generated in the spatialDLPFC dataset, since the issue was related to subject-GUID mapping.<\/span>","<span class='gt_from_md'>We discussed the general setup we will move ahead with now that we have to wait a few months before new data is generated to complete this study. We brainstormed options moving forward on how to best collaborate.<\/span>","<span class='gt_from_md'>I helped her understand the results of the AtlasXomics pilot review for the samples provided by Lieber for Hb and DLPFC. I explained the basic concepts and quality controls expected according to the ENCODE standard, however not all of them apply to this type of data. I also pointed out that the sequenced samples were bulk ATAC and could be another metric to consider outside my field of expertise.<\/span>","<span class='gt_from_md'>I explained her the main layout and variables included on the spatialLIBD shiny app, Using as an example our <a href=\"https://libd.shinyapps.io/Habenula_Visium_raw/\">https://libd.shinyapps.io/Habenula_Visium_raw/<\/a>   app I showed her many examples using continuos and discrete variables. As well I provided her a glimse of the 10x graph-clustering  wrapped in the app.<\/span>","<span class='gt_from_md'>Ibidem<\/span>","<span class='gt_from_md'>Pointed to the different ways we have to access modeling data from the fiffernt DLPFC studies<\/span>","<span class='gt_from_md'>I explained her main steps to build a spatialLIBD shiny app with raw data. I quickly provided her an overview of spaceranger, and then I explained her how to build a basic spe object and the settings of the ShinnyApps on the cloud. We use as example  our <a href=\"https://libd.shinyapps.io/Habenula_Visium_raw/\">https://libd.shinyapps.io/Habenula_Visium_raw/<\/a>   app.<\/span>","<span class='gt_from_md'>We reviewed responses from the OP to recent comments by a co-author of this paper.<\/span>","<span class='gt_from_md'>We reviewed some information about applying for PhD programs: how to decide, application strategy in general, information that is best to provide for those writing letters, and other related info.<\/span>","<span class='gt_from_md'>I demonstrated to use of modules for command-line software, R, and python. I pointed to slurmjobs vignettes and the ‘job_single()’ function in particular. I noted katun or caracol are reserved partitions that can be accessed with ‘-p katun’ when using srun or sbatch.<\/span>","<span class='gt_from_md'>We reviewed recent changes made by OP to the paper draft and ultimately accepted / resolved current open comments.<\/span>","<span class='gt_from_md'>I explained her relevant points when working with visium samples. We focus on the general concepts and main workflow. I also show her resources to deep into this subjects, such as presentations and the use of the shinny up to reproduce some results.<\/span>","<span class='gt_from_md'>We discussed recent results with our colleagues and made a list of pending items to clarify.<\/span>","<span class='gt_from_md'>We set up the logistics for performing a remote LIBD internship with my team by re-using materials from the CDSB 2021 snRNA-seq OSCA-based workshop materials and videos.<\/span>","<span class='gt_from_md'>Setting up and testing github access on laptop and JHPCE, continued<\/span>","<span class='gt_from_md'>We reviewed how to deploy spatialLIBD shiny apps on shinyapps.io and why lines such as <code>library(&quot;markdown&quot;)<\/code> are important to keep in the <code>app.R<\/code> R script file.<\/span>","<span class='gt_from_md'>Setting up and testing github access on laptop and JHPCE<\/span>","<span class='gt_from_md'>I showed him the template we currently use. I showed him many examples of how the various directories are expected to be related.<\/span>","<span class='gt_from_md'>I help her to navigate in the resources to setup an appropiate configuration for her laptop. We installed some programs.<\/span>","<span class='gt_from_md'>I showed her the team repository so she could update her profile. I helped her clone the repository and showed her the sections she needed to update. I presented her with different examples so she could create her own profile.<\/span>","<span class='gt_from_md'>We setup cyberduck and ssh keys for the new laptop<\/span>","<span class='gt_from_md'>We reviewed how the universe of background genes (for GO/KEGG ORA) is different for each cluster when using <code>pseudoBulkDGE()<\/code> due to the <code>filterByExpr()<\/code> internal step.<\/span>","<span class='gt_from_md'>I showed her how to use Slack to have efficient communication, I also showed her examples of what kind of assumptions can be made in the different channels. I help her to organize her threads. I also show her my Google Slides weekly update setup, and ways to improve the current report.<\/span>","<span class='gt_from_md'>We went over the preparation for my invited speaker presentation for this webinar series.<\/span>","<span class='gt_from_md'>I explained the purpose of the RStat Club, showed her where the calendar is, how to schedule her participations in the shared Excel sheet, and how to send an email to invite other subscribers, as well as how to receive these notifications in the Google Slack thread.<\/span>","<span class='gt_from_md'>I explained the purpose of the team presentations, showed her where the calendar is, how to schedule her participations in the shared Excel sheet, and how to follow the notifications in the Google Slack thread. I also explained her how to schedule a Dsgs calendar for ask help.<\/span>","<span class='gt_from_md'>I recommended the code ‘as.integer(levels(clusters)[as.integer(clusters)])’ to address the particular use case, for a factor of integers ‘clusters’<\/span>","<span class='gt_from_md'>Discussed recent updates.<\/span>","<span class='gt_from_md'>She encountered a cryptic error when pseudobulking on particular data. I recommended trying the code with simpler data to identify whether the data or code was a problem.<\/span>","<span class='gt_from_md'>We discussed some future options for LIBD cloud computing to make sure that we were on the same page.<\/span>","<span class='gt_from_md'>Further discussed options for working with the team.<\/span>","NA","<span class='gt_from_md'>We interpreted and made annotations on the results comparing the clustering vs sub-clustering analyses.<\/span>","<span class='gt_from_md'>I explained that often samples aren’t trimmed by default, as the adapter content metric from FastQC must fail to trigger trimming. I mentioned the ‘–trim_mode force’ option to force trimming<\/span>","<span class='gt_from_md'>I described several common sbatch options and their syntax (and SGE equivalent). We ultimately discovered that the ‘fusion_twas’ module was never rebuilt, so I did that afterward<\/span>","<span class='gt_from_md'>We briefly mentioned the edgeR 4.0 paper at <a href=\"https://doi.org/10.1101/2024.01.21.576131\">https://doi.org/10.1101/2024.01.21.576131<\/a> and then mostly spent time reviewing the documentation for <code>clusterProfiler<\/code> at <a href=\"https://yulab-smu.top/biomedical-knowledge-mining-book/clusterprofiler-comparecluster.html\">https://yulab-smu.top/biomedical-knowledge-mining-book/clusterprofiler-comparecluster.html<\/a>.<\/span>","<span class='gt_from_md'>I found a basic example script where I ran nnSVG and described the array job I use to run it for each sample, followed by an ordinary job to aggregate results by averaging gene ranks<\/span>","<span class='gt_from_md'>We discussed the preliminary version of the website for visualizing storage use over time at JHPCE.<\/span>","<span class='gt_from_md'>We walked thorught the code for leafcutter and the output for each stage<\/span>","<span class='gt_from_md'>We reviewed when to use <code>set.seed()<\/code> when using <code>runPCA()<\/code>, <code>runUMAP()<\/code>, <code>runTSNE()<\/code> and other related analysis steps.<\/span>","<span class='gt_from_md'>Ibidem.<\/span>","<span class='gt_from_md'>We discussed feedback from collaborators and how to best address it.<\/span>","<span class='gt_from_md'>I showed the specific scripts I used to perform the upload, described the process for resetting your password to obtain one usable for the CLI, and the python library for authenticating with NDA<\/span>","<span class='gt_from_md'>We discussed why a sub-clustering analysis would be of interest for this dataset.<\/span>","<span class='gt_from_md'>We talked about data science education and the InterJad platform.<\/span>","<span class='gt_from_md'>We reviewed <a href=\"https://lcolladotor.github.io/bioc_team_ds/writing-papers.html\">https://lcolladotor.github.io/bioc_team_ds/writing-papers.html<\/a> and other guidelines for writing method sections of papers.<\/span>","<span class='gt_from_md'>We reviewed <a href=\"https://rdrr.io/bioc/scran/man/pseudoBulkDGE.html\">https://rdrr.io/bioc/scran/man/pseudoBulkDGE.html<\/a> and then also talked about how to perform ORA (Over Representation Analysis) on GO / KEGG. We also reviewed how to make volcano plots with <a href=\"https://www.bioconductor.org/packages/release/bioc/vignettes/ideal/inst/doc/ideal-usersguide.html#75_plot_volcano\">https://www.bioconductor.org/packages/release/bioc/vignettes/ideal/inst/doc/ideal-usersguide.html#75_plot_volcano<\/a> and <a href=\"https://bioconductor.org/packages/release/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html#plot-the-most-basic-volcano-plot\">https://bioconductor.org/packages/release/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html#plot-the-most-basic-volcano-plot<\/a>.<\/span>","<span class='gt_from_md'>We discussed options for how to use <code>derfinder<\/code>, <code>recount2<\/code>, <code>recount3<\/code>, <code>megadepth<\/code> and other related packages and tools (like <code>recount.bwtool<\/code>) to quantify expression from BigWig files from the recount2/3 projects. There is a need for a <code>megadepth<\/code> function that is similar to <code>recount.bwtool::coverage_matrix()<\/code> that works via the web given the current limitations with <code>rtracklayer<\/code>.<\/span>","<span class='gt_from_md'>We de-bugged some issues with launching the app, and made some observations about the clustering at different ks, utimatley the optimal k will be based on the biological question at hand<\/span>","<span class='gt_from_md'>We discueed MeanRatio and 1vAll methods for finding marker genes<\/span>","<span class='gt_from_md'>We reviewed material from OSCA on <code>pseudoBulkDGE()<\/code> documented at <a href=\"https://bioconductor.org/books/3.19/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis\">https://bioconductor.org/books/3.19/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis<\/a> for performing DEG analysis within each cell type cluster.<\/span>","<span class='gt_from_md'>We discussed opportunities within my team and LIBD in general.<\/span>","<span class='gt_from_md'>registation wrapper filters the pseudobulked object,dropping lowly expressed genes. Often resutlsing in ~12k genes (comapred to ~30lk genes in the original object). This is expected behaviour.<\/span>","<span class='gt_from_md'>We discussed options to extend our collaboration in 2024.<\/span>","<span class='gt_from_md'>We reviewed the properties of SpatialExperiment objects and other type of R objects (like a <code>GRangesList<\/code>) we used in Huuki-Myers et al, Science, 2024. The goal was to promote interoperability with planned work by the OP. We also went over <a href=\"https://research.libd.org/DeconvoBuddies/reference/findMarkers_1vAll.html\">https://research.libd.org/DeconvoBuddies/reference/findMarkers_1vAll.html<\/a>, <a href=\"https://research.libd.org/spatialLIBD/reference/registration_pseudobulk.html\">https://research.libd.org/spatialLIBD/reference/registration_pseudobulk.html<\/a>, <a href=\"https://bioconductor.org/books/3.19/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis\">https://bioconductor.org/books/3.19/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis<\/a>, and <a href=\"https://research.libd.org/spatialLIBD/reference/registration_wrapper.html\">https://research.libd.org/spatialLIBD/reference/registration_wrapper.html<\/a>.<\/span>","<span class='gt_from_md'>We discussed opportunities within my team and options for collaborating moving forward.<\/span>","<span class='gt_from_md'>We discussced refernce vs. querery in spatial registation and the required modeing to prep the data for the registration anlaysis.<\/span>","<span class='gt_from_md'>I provided an overview of what my team does and the type of projects we work on. We brainstormed a bit about fundraising.<\/span>","<span class='gt_from_md'>We discussed their GitHub profile and other evidence of coding skills.<\/span>","<span class='gt_from_md'>I discussed possible choices of input genes, the fact that raw counts are used, and pointed to some example code where I had run PRECAST<\/span>","<span class='gt_from_md'>I reviewed the general process and location of code I used for the spatialDLPFC project for uploading to NDA<\/span>","<span class='gt_from_md'>Demoed issues with ‘vis_clus’ when plotting logical variables, which we ultimately determined to be a spatialLIBD bug (wrote a GH issue as a result)<\/span>","<span class='gt_from_md'>I showed the source code inducing the error message (suggesting a mismatch in keys between previous annotation and the current SPE object), and we increasingly tested simpler annotation examples until something worked. Ultimately, we think the editor that was used in manually modifying the annotation CSV may have been the culprit<\/span>","<span class='gt_from_md'>I pointed to an initial version of my code which had a simpler workflow than the reference code she was modeling hers after, and gave a brief explanation of the purpose of GLM-PCA<\/span>","<span class='gt_from_md'>Identified the correct data and organization for this analysis<\/span>","<span class='gt_from_md'>I helped with building the samples.manifest file and addressed a permissions issue on annotation files that was halting the first SPEAQeasy run<\/span>","<span class='gt_from_md'>We went over some documentation and features of <a href=\"https://dlpfc.mccarrolllab.org/app/dlpfc\">https://dlpfc.mccarrolllab.org/app/dlpfc<\/a>.<\/span>","<span class='gt_from_md'>We revisited <a href=\"https://kb.10xgenomics.com/hc/en-us/articles/115003616023-Files-and-criteria-used-to-generate-10x-references\">documentation by 10x Genomics on how the reference GTFs were made<\/a> such as the <a href=\"https://www.10xgenomics.com/support/software/cell-ranger/downloads/cr-ref-build-steps#mouse-ref-2020-a\">mouse 2020 ref<\/a>.<\/span>","<span class='gt_from_md'>We went over how to configure Zotero <a href=\"https://www.zotero.org/\">https://www.zotero.org/<\/a> and Zotfile <a href=\"https://github.com/jlegewie/zotfile\">https://github.com/jlegewie/zotfile<\/a> for organizing PDF files of papers in your laptop. I keep the PDFs on my Dropbox and use Zotfile for automatically naming the PDFs when I download them via the <a href=\"https://chromewebstore.google.com/detail/zotero-connector/ekhagklcjbdpajgpjgmbionohlpdbjgc?pli=1\">Zotero Google Chrome plugin<\/a>.<\/span>","<span class='gt_from_md'>We discussed some questions about the application of qsvaR to the habenulaPilot bulk RNA-seq dataset.<\/span>","<span class='gt_from_md'>I showed the code I’ve used for log normalizing spatial data counts, and linked to OSCA’s section on log normalization via deconvolution, after which the code was based<\/span>","<span class='gt_from_md'>I showed the syntax git uses when a merge conflict occurs, and how to resolve conflicted files<\/span>","<span class='gt_from_md'>We checked their LinkedIn, website, and CV. I provided a list of changes to improve all 3.<\/span>","<span class='gt_from_md'>We discussed the spatialDLPFC paper, how we integrated the sn and visium datasets<\/span>","<span class='gt_from_md'>We have old results for the brainseq DLPFC, but i described the method we chose from deconvolution benchmark to get most accurate deconvolution results<\/span>","<span class='gt_from_md'>She showed that capture area positions from images stitched with ImageJ disagreed with those stitched in the python-and-Samui-based workflow. I went through suspicious areas of code without success. Later, she identified that it was an issue related to differences in scalefactors and image resolutions, and differences in underlying assumptions between the stitching steps<\/span>","<span class='gt_from_md'>We used commands such as <code>for i in *Cg*/*.fastq.gz; do echo <link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.jsdelivr.net/npm/katex@0.16.9/dist/katex.min.css\" data-external=\"1\">\n<span class=\"katex\"><span class=\"katex-mathml\"><math xmlns=\"http://www.w3.org/1998/Math/MathML\"><semantics><mrow><mi>i<\/mi><mo separator=\"true\">;<\/mo><mi>z<\/mi><mi>c<\/mi><mi>a<\/mi><mi>t<\/mi><\/mrow><annotation encoding=\"application/x-tex\">i; zcat <\/annotation><\/semantics><\/math><\/span><span class=\"katex-html\" aria-hidden=\"true\"><span class=\"base\"><span class=\"strut\" style=\"height:0.854em;vertical-align:-0.1944em;\"><\/span><span class=\"mord mathnormal\">i<\/span><span class=\"mpunct\">;<\/span><span class=\"mspace\" style=\"margin-right:0.1667em;\"><\/span><span class=\"mord mathnormal\">zc<\/span><span class=\"mord mathnormal\">a<\/span><span class=\"mord mathnormal\">t<\/span><\/span><\/span><\/span>i | head; done<\/code> to show the first few lines of 10x Genomics FASTQ files (Illumina sequencing), then checked the indexes vs those that our colleagues used for based on their wet lab notes.<\/span>","<span class='gt_from_md'>I went through the scripts involved in locally deploying a Shiny app, as well as which files to download from JHPCE to interactively test<\/span>","<span class='gt_from_md'>We discussed a few things including <code>jaffelab::expression_cutoff()<\/code> logic and briefly mentioned <code>edgeR::filterByExpr()<\/code> as a more conservative alternative. In any case, when working on a sample subset I do recommend starting with the raw data and filtering out lowly-expressed genes (features) as which genes are expressed in a subset can be very different from the full dataset. We talked about ranges of logFC and t-statistics seen in DGE results from different projects, which can change depending on the biological covariate of interest. We also discussed our current strategy for selecting covariates in a DGE analysis when working on a small(er) dataset.<\/span>","<span class='gt_from_md'>We discussed tools to manage files on JHPCE, and stradagies to get code to the cluster (either edit remote code, or use git)<\/span>","<span class='gt_from_md'>We discussed that on a discovery dataset, we apply an FDR filter, and thus filtering out lowly-expressed features (genes) can affect the results. It doesn’t impact the target dataset at all since in the target we are using nominal p-value (aka not FDR-adjusted p-values) and sign(logFC).<\/span>","<span class='gt_from_md'>We went over how to update globus <a href=\"https://research.libd.org/globus/\">https://research.libd.org/globus/<\/a> by updating <a href=\"https://github.com/LieberInstitute/globus/blob/master/index.Rmd\">https://github.com/LieberInstitute/globus/blob/master/index.Rmd<\/a>, then clicking the “knit” button. We also went over how to create Globus endpoints (collections that you share), how to set the right permissions, add other administrators, and get the link for sharing which is what we need for the index.Rmd script.<\/span>","<span class='gt_from_md'>I noted the error about termination by the external system, which suggested to me enforcement of time or memory limits. Looking in the working directory for the failing Trimming process, we saw a message about OOM, so I suggested bumping memory up for Trimming in the config file<\/span>","<span class='gt_from_md'>I showed the three kinds of plots we made for the habenula RNAScope data, geom_hex/stat_summary_hex for continuious variables across the tissue, geom_rect for specific cells, and geom_rect + geom_point for highlighting rare cell types.<\/span>","<span class='gt_from_md'>I jogged my memory of the NDA upload process to provide guidance about creating an NDA study. I then described the order, purpose, and inputs to some scripts for actually uploading the data<\/span>","<span class='gt_from_md'>I described the overall idea: fitting a linear model and dropping specific terms, keeping just the residuals. I recommended setting <code>P = 2<\/code> for case-control analyses where the diagnosis term is first, in order to explore if dropping the effect of other terms, such as batch, could clean up the signal<\/span>","<span class='gt_from_md'>We double chekecked the stats for eQTL results, refined results and plots accordingly<\/span>","<span class='gt_from_md'>He noted that an ldpaths and Makeconf file were created with bad permissions sometimes when working with the r_nac module at JHPCE. We identified that switching from one conda-based module to r_nac triggered the issue, and I suggested tricks for changing permissions on the files to prevent the issue from recurring<\/span>","<span class='gt_from_md'>I walked Daianna through the 3 steps of MAGMA, and the needed input from GWAS data and gene sets. There are many detials to consider, especally in preparing this data. Daianna will start with preparing the GWAS data and we will revisit the next steps.<\/span>","<span class='gt_from_md'>We talked thorough their proposed analysis. THery Decided Bisque may be the test boption I pointed them to my tuttorial for Bisque Bisque <a href=\"https://research.libd.org/DeconvoBuddies/articles/Deconvolution_with_Bisque.htm\">https://research.libd.org/DeconvoBuddies/articles/Deconvolution_with_Bisque.htm<\/a><\/span>","<span class='gt_from_md'>We discussed the nuances of some of LIBD’s internal datasets, questions that need to be asked, whom to ask them to, and potential reasons for including/not including myself or others as Co-Is on a grant application.<\/span>","<span class='gt_from_md'>He noted some configuration was out of date, and I pushed updated configuration (related to the transition to SLURM). I also recommended using Arioc 1.51, since the 1.43 module was failing to install<\/span>","<span class='gt_from_md'>We discussed how it could be useful to compare CellRanger v3 with the pre-mRNA modified GTF, v6 without the include-introns flag, v6 with the include-introns flag, Alevin, and STARsolo. Alevin and STARsolo might have similar include-introns flags to CellRanger by 10x Genomics. We discussed how this could potentially affect the class of single exon genes that are embededded within introns of longer genes. Violin plots by cell type across the different computational pipelines and alignment options could be useful to learn more how this affects gene quantification of this special gene class.<\/span>","<span class='gt_from_md'>I found a config file for the particular software that supported singularity, and noted which module to use at JHPCE (also warning that it may require a good amount of memory on the first run)<\/span>","<span class='gt_from_md'>We discussed in detail the DEG models for the BSP1, BSP2 (combined regions, one region at a time), BSP3, Habenula, and CMC Phase 1 datasets. All should be age &gt; 17. BSP2 overall should adjust for brain region (a large source of biological variation) and also drop the 43 HPC RiboZeroGold samples. BSP2 HPC only should also drop those same 43 RiboZeroGold samples. Habenula should adjust for the tot.Hb and tot.Thal estimated fractions: here’s the <a href=\"https://github.com/LieberInstitute/Habenula_Pilot/blob/ca6dba5f629a8ec580297ab6d56b36b4f0cb66da/code/10_DEA/04_DEA.R#L124-L126\">published model<\/a>. CMC Phase 1 should adjust for BrainBank. Here’s the BSP1 hg38 (part of the BSP2 paper) <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase1_qsv/casectrl_DLPFC_allFeatures.R#L40\">model<\/a>.<\/span>","<span class='gt_from_md'>I outlined that staged but not committed changes prevent merging, and how to resolve merge conflicts in files modified in different ways from two different sources<\/span>","<span class='gt_from_md'>They taught me that <code>dreamlet<\/code> can produce <code>mito_percent<\/code> and <code>ribo_percent<\/code> for each cell, and thus you can summarize them with the mean across all cells of a given cell type for a given donor. This expands the options for covariates in a DEG model. We discussed why it could be interesting to run a model with and without adjusting for <code>mean(mito_percent)<\/code> given that in spatial data (such as the data from <code>HumanPilot<\/code>) <code>mito_percent<\/code> is related to the spatial location of WM and L1 (richer in oligodendrocytes and other glia cells, low presence of neurons). We also went over a part of the <a href=\"https://doi.org/10.7554/eLife.84628.3\">LC eLife paper<\/a> and revised some steps from <a href=\"https://bioconductor.org/books/3.18/OSCA.multisample/multi-sample-comparisons.html#performing-the-de-analysis\">OSCA on multi-sample DE<\/a>. An equivalent of <a href=\"https://research.libd.org/SPEAQeasy/outputs.html#quality-metrics\"><code>totalAssignedGene<\/code> produced by <code>SPEAQeasy<\/code><\/a> could be available at the donor (individual) level, but maybe it wouldn’t be that useful as a covariate in these cell-type specific analyses.<\/span>","<span class='gt_from_md'>Re-organized code to run with a shell script from slurm_jobs. We solved a bug with args index, and talked about adding catches for unexpected inputs. Future goal is to use map to replace for loops when handling list of data objects<\/span>","<span class='gt_from_md'>After making a new <code>git clone<\/code>, we created the <code>gh-pages<\/code> branch with commands from <a href=\"https://jiafulow.github.io/blog/2020/07/09/create-gh-pages-branch-in-existing-repo/\">https://jiafulow.github.io/blog/2020/07/09/create-gh-pages-branch-in-existing-repo/<\/a>. For more on GitHub pages check <a href=\"https://pages.github.com/\">this page<\/a>. We created a favicon using <a href=\"https://favicon.io/favicon-converter/\">https://favicon.io/favicon-converter/<\/a>. We created a <a href=\"https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd\"><code>README.Rmd<\/code> file re-using some code from the <code>spatialDLPFC<\/code> one<\/a>, and also adapted the <a href=\"https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R\"><code>update_website.R<\/code> script<\/a>. We also did a few other manual setup steps like creating the <code>img<\/code> directory and the <code>.nojekyll<\/code> file on the <code>gh-pages<\/code> branch, plus version controlling them. I might write a blog post about this full process in detail.<\/span>","<span class='gt_from_md'>I explained the difference between signed and unsigned networks, as well as the statistical values that describe an eigengene module. I also explained the network representation in terms of a gene expression data frame versus a differential gene expression data frame. I also explained some functions to validate the genetic significance and module membership of eigengenes under this methodological approach for rna-seq data.<\/span>","<span class='gt_from_md'>I explained why <code>age &gt; 17<\/code> is a must for SCZD vs NTC DGE analyses: we have no SCZD cases with <code>age &lt;= 17<\/code>, so if you don’t filter the NTC donors with <code>age &lt;=17<\/code>, you end up basically comparing older (SCZD) vs younger (NTC) donors, which is not what you want to do. We also talked about how in BSP2 there are some HPC (HIPPO) samples from 2 RiboZero protocols (Gold and HMR), how the Gold HPC samples cluster with the DLPFC ones on PC1 vs PC2, and thus why we ended up dropping the 43 Gold HPC samples in previous work (see <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/make_qSVs.R#L62-L69\">https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/make_qSVs.R#L62-L69<\/a>). This information is reflected by <code>analysis_sczd_casecontrol_hippo<\/code> at <a href=\"https://s3.us-east-2.amazonaws.com/libd-brainseq2/methprop_pd.Rdata\">https://s3.us-east-2.amazonaws.com/libd-brainseq2/methprop_pd.Rdata<\/a> listed at <a href=\"https://eqtl.brainseq.org/phase2/\">https://eqtl.brainseq.org/phase2/<\/a>. Finally, I emphasized that self-reported <code>race<\/code> is not a trustworthy variable and we instead rely on <code>snpPC<\/code>s.<\/span>","<span class='gt_from_md'>I outlined the process of running SPEAQeasy as a whole, and showed how to find which annotation settings were used on a particular project<\/span>","<span class='gt_from_md'>I explained why using something like <a href=\"https://research.libd.org/spatialDLPFC/#interactive-websites\">https://research.libd.org/spatialDLPFC/#interactive-websites<\/a> or <a href=\"https://research.libd.org/globus/\">https://research.libd.org/globus/<\/a> give us control over links that we might need to update later on, instead of including other less stable links on manuscripts (domains we don’t control, tools that might change their URL syntax, etc). I provided links to how to make each option such as <a href=\"https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R\">https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R<\/a> and <a href=\"https://github.com/LieberInstitute/globus/blob/98dc479ad2cde0a047cf7a46769a85c32969b7b1/index.Rmd#L8-L37\">https://github.com/LieberInstitute/globus/blob/98dc479ad2cde0a047cf7a46769a85c32969b7b1/index.Rmd#L8-L37<\/a>.<\/span>","<span class='gt_from_md'>We went over the data that we have for this project, worked on making a soft link to the output of cellranger that was generated in a different repository (and thus JHPCE location) to enable the use of <code>here::here()<\/code> in future scripts, and discussed potential issues we might observe in this new dataset.<\/span>","<span class='gt_from_md'>In the session, we explored reanalyzing cloupe objects with the Loupe browser for scRNAseq assays. This involved using multiple projections like UMAP and t-SNE, splitting clusters, and filtering data based on UMI count and mitochondrial percentage. While there was interest in analyzing integrated datasets, the Loupe browser currently only supports exploring one Cloupe object at a time.<\/span>","<span class='gt_from_md'>I suggested some temporary workarounds to get SPEAQeasy to recognize a conda environment with the necessary dependencies, but noted that we didn’t officially support using conda for this purpose<\/span>","<span class='gt_from_md'>We discussed the specifics of the <a href=\"https://kavlijhu.org/funding/4\">KavliNDI Distinguished Graduate Fellowship<\/a> and revised information that I can use while writing the reference letter (project title, assays proposed, that this involved new software, how long we’ve worked together, education history of the applicant, etc).<\/span>","<span class='gt_from_md'>I explained where is located the cloupe output object generated by cell ranger-arc. Then, I explained how to use the tool to explore gene expression and atac assays by separate and also together. I also explained how to inspect clustering and re-label clusters and sub clusters; plus how to check DEG and look at their gene distributions. Finally, I explained how to re-run an analysis and use the most classic formats to save the analysis performed.<\/span>","<span class='gt_from_md'>I explained how to prepare the configuration files for working R and JHPCE.  I showed all the section that need to completed plus the configuration to access JHPCE, with links pointing specifically to Windows system. I explained the purpose of each section in the chapter 17 of <a href=\"https://lcolladotor.github.io/bioc_team_ds/config-files.html\">https://lcolladotor.github.io/bioc_team_ds/config-files.html<\/a>. I also provided the updated link (<a href=\"https://jhpce.jhu.edu/access/ssh/\">https://jhpce.jhu.edu/access/ssh/<\/a>)  for the JHPCE configuration due the past link is broken because of the change to the new web version. Finally, I provided the most recent video made by Leo  <a href=\"https://youtu.be/qtn3eTwFlZI\">https://youtu.be/qtn3eTwFlZI<\/a>  (minute 17:50) to do this specific task for windows OS.<\/span>","<span class='gt_from_md'>We added the <code>LFF_spatial_ERC<\/code> and <code>LFF_spatial_LC<\/code> datasets to <a href=\"https://github.com/lieberinstitute/ranger_metrics\">https://github.com/lieberinstitute/ranger_metrics<\/a>. This involved debugging the code a bit to handle updates on the Excel sheet files describing samples prior to being sent for sequencing.<\/span>","<span class='gt_from_md'>We found out that <code>spaceranger count<\/code> did not like the JSON files we created with <code>jsonlite::write_json()<\/code> so instead, we edited them manually using <code>gsub()<\/code> and <code>read/writeLines()<\/code> as documented <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/commit/bbf810ad39012ba6ee4e0403063bf83bf844d007\">here<\/a>. Note that we had to manuallly revert the JSON files to their original state as documented <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/commit/79ed5479d502cd41d51f3aa8ee1467bd8100a08a\">here<\/a>.<\/span>","<span class='gt_from_md'>We wrote a script to manually update the json files and add the Visium Slide ID as well as the capture area. This will enable us to not have to use <code>--unknown-slide=visium-1<\/code> <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/blob/6e85c05ce9f531c555569e616f8d5c351b427b4f/code/01_spaceranger/spaceranger_SLURM.sh#L63\">https://github.com/LieberInstitute/LFF_spatial_ERC/blob/6e85c05ce9f531c555569e616f8d5c351b427b4f/code/01_spaceranger/spaceranger_SLURM.sh#L63<\/a> that 10x Genomics support had recommended using. Doing so led to an error at <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/blob/b5b92a8bb9744ee424ec4a0642ae692423a91d56/code/01_spaceranger/logs/spaceranger_17.txt#L22\">https://github.com/LieberInstitute/LFF_spatial_ERC/blob/b5b92a8bb9744ee424ec4a0642ae692423a91d56/code/01_spaceranger/logs/spaceranger_17.txt#L22<\/a> which stated that <code>--slide<\/code> cannot be used as the same time as <code>--unknown-slide<\/code>. This is a workaround that was previously manually implemented at <a href=\"https://github.com/LieberInstitute/Habenula_Visium\">https://github.com/LieberInstitute/Habenula_Visium<\/a>. Since we have lots of samples, we wrote a script to avoid introducing copy-paste errors.<\/span>","<span class='gt_from_md'>The issue on this <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/blob/e913ff28fffbb17335d3844d312ec6486a7b182f/code/01_spaceranger/spaceranger_SLURM.sh#L55\">script<\/a> and related ones is that there is a missing <code>\\ <\/code> (note the space) before the <code>#<\/code> comment symbol. Thus the chained command is breaking into two, and the second half is not being recognized as options to the first half.<\/span>","<span class='gt_from_md'>We discussed tools for spacing and aligning objects in BioRender. I suggested maximizing font sizes and reducing white space<\/span>","<span class='gt_from_md'>We worked on the following error message from <code>spaceranger count<\/code>: <em>ERROR: We detected a mixture of different R1 lengths ([26-28]), which breaks assumptions in how UMIs are tabulated and corrected. To process these data, you will need to truncate to the shortest observed R1 length by providing 26 to the –r1-length argument if are running count/vdj, or via the r1-length parameter in each of the following tables of your multi config CSV if you are running multi: [gene-expression]<\/em>. We inspected the soft links for the FASTQ files and used <code>zcat fastq.gz | head<\/code> to inspect the first few lines of FASTQ files. We confirmed that the <code>R1<\/code> FASTQs were indeed of varying lenghts and will attempt a new <code>spaceranger count<\/code> run using <code>--r1-length = 26<\/code> though it’s likely that we’ll next get an error with the <code>R2<\/code> FASTQ files as we observed that those files also have varying lengths.<\/span>","<span class='gt_from_md'>Able to access the data throught the LIBD dataportal, I also pointed to the combined LIBD rse_gene in brain_swap (which I think might be the source of the portal data…would need to check)<\/span>","<span class='gt_from_md'>Updated script by commenting out unwanted lines and providing correct paths to the input files<\/span>","<span class='gt_from_md'>I explained what the key metrics are for each assay in a Multiome experiment from 10x scRNAseq and scATACseq data sets. I showed several HTML reports of data sets processed with cell ranger-arc to illustrate different scenarios targeting different numbers of reads for each assay, to help you understand the relevance of quality beyond the number of reads in sequencing. I showed how with a dataset with a moderate number of good quality reads you can perform a good analysis, while having many reads with compromised quality it is possible to have more challenges during the analysis. Finally, I explained the minimum recommended number and that it would be a good idea to search the literature for technical data from other experiments similar to his as a reference.<\/span>","<span class='gt_from_md'>Cross checked the script and rectified filename/samplename processing<\/span>","<span class='gt_from_md'>I explained that BSP1 is polyA, aligned vs hg19, older annotation, aligned with TopHat2 (an older aligner). BSP2 is RiboZeroGold for DLPFC and RiboZeroHMR for HIPPO (with 43 RiboZeroGold samples that were dropped from the DE analysis), aligned vs hg38, against Gencode v25 (from July 2016), aligned with HISAT2. I also explained the naming structure of a Supplementary Table for BSP1’s paper where <code>adj<\/code>, <code>qSVA<\/code>, and <code>PCA<\/code> are labels specifying whether it was a model <code>adj<\/code>usting for covariates, whether it adjusts for qSVs using <code>qSVA<\/code> (the better model to use), or adjusting for gene-level expression PCs (<code>PCA<\/code>). These names are related to the <a href=\"https://www.pnas.org/doi/full/10.1073/pnas.1617384114\"><code>qSVA<\/code> 2017 paper<\/a> Figure 2. I also explained how in the BSP2 paper we did re-align the BSP1 data against hg38, though there is no Supplementary Table showing which of the 243 DLPFC DEGs replicated in BSP1 (though the data does exist since it was used to make Figure S28 and S29). We would need another DSgs or two to actually locate the data and construct an CSV file or table that others can use if they are interested in replication of BSP2 results using BSP1 and/or CommonMind Consortium (CMC) Phase 1 data (that was aligned vs hg19 with TopHat2). I also used <a href=\"https://speakerdeck.com/lcolladotor/libd-ds-tldr?slide=56\">this slide<\/a> to explain all the pieces that go into a DEG model and why the visualizations provided by the LIBD Data Portal (boxplots for SCZD vs NTC) are just a visual guide using normalized gene expression data and not the full answer (as the LIBD Data Portal doesn’t use the full details we include in a DEG model).<\/span>","<span class='gt_from_md'>I explained in detail the metrics reported by cellranger arc and helped you download a multiome public sample to run the pipeline.<\/span>","<span class='gt_from_md'>Re-organized code to run with a shell script from slurm_jobs. We solved a bug with args index, and talked about adding catches for unexpected inputs. Future goal is to use map to replace for loops when handling list of data objects<\/span>","<span class='gt_from_md'>We disscussed specific details on how to ask for help. For example, it’s ok to ask those around you in person but you should also ask on the <code>#libd_helpdesk<\/code> channel (and update the channel if someone else gave you an answer). That is, don’t assume that only those around you (in person) know the answer. We went over details on how to preview GitHub messages (to verify that they are correctly formatted), why GitHub issues are useful to the authors, what information is needed for others to help you answer a question (R session info, <code>reprex::reprex()<\/code> output, link to code on a public GitHub repo; private is ok when asking internally for help). We also discussed the overall idea of why learning how to make an R package will be useful to then improve one’s tooltik for debugging R packages made by others.<\/span>","<span class='gt_from_md'>I explained the previous steps to perform clustering with GEX and ATAC 10x multiome data using Seurat and Signac.<\/span>","<span class='gt_from_md'>We interactively ran the python code for reading in ImageJ transforms to rule out an issue where the wrong data was read in. Ultimately, we couldn’t find issues with the data, but the Samui full-res images still seemed misaligned relative to the low-res images<\/span>","<span class='gt_from_md'>We went over how to update a GitHub remote following <a href=\"https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories\">information from the documentation<\/a>, finishing stylistic design touches on an <code>iSEE<\/code> app, and setting up a new repository for the next project.<\/span>","<span class='gt_from_md'>We discussed details of the use cases I have in mind for how to search and visualize the JHPCE disk usage data over time and across different JHPCE paths. This information could be useful in forecasting data growth for individual PIs or projects, identifying who has exceeded their storage allocation, identifying static projects that could potentially can be moved off JHPCE / deleted / or moved to non-production disk space.<\/span>","<span class='gt_from_md'>We discussed the general overview of snRNA-seq data analysis steps and how the code at <a href=\"https://github.com/LieberInstitute/Habenula_Pilot\"><code>Habenula_Pilot<\/code><\/a> could be a good starting place. I also recommended signing up for a Data Science guidance session with Louise as she is quite familiar with the code in that repository prior to diving into it.<\/span>","<span class='gt_from_md'>I explained details about how data is organized at JHPCE, data sharing limitations, and several suggestions we have been working on that could be useful to implement at a larger scale across LIBD teams.<\/span>","<span class='gt_from_md'>I pointed to suspicious areas of code where dataset-specific regexs were originally used, though the regexs looked good for his data as well. We decided to interactively walk line by line at a later date<\/span>","<span class='gt_from_md'>Discussed collaborations, personalities, and other details of the work environment with some of our collaborators with whom this person was going to interview for a job with.<\/span>","<span class='gt_from_md'>I suggested building a list() where each element was a 3-piece plot (combined with a function like cowplot::plot_grid), then using print(plot_list)<\/span>","<span class='gt_from_md'>See <a href=\"https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1710454284251579\">https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1710454284251579<\/a> for the full code details. We might add this to solution to <a href=\"https://github.com/LieberInstitute/jaffelab/blob/devel/R/merge_rse_metrics.R\">https://github.com/LieberInstitute/jaffelab/blob/devel/R/merge_rse_metrics.R<\/a>, although it’s only an approximation as you need to go back to the raw ERCC output files (typically generated with <code>kallisto<\/code>) to actually compute the ERCCsumLogErr accurately across different sequencing lanes for a given sample.<\/span>","<span class='gt_from_md'>We have RSE objects from the LIBD Data Portal (or those that helped build that portal) as well as RSE objects made prior to the LIBD Data Portal. They have some differences, mostly on the <code>colData()<\/code> slot. Note how <a href=\"https://github.com/LieberInstitute/jaffelab/blob/21e65745a5b0852fc13ebdb5e5d23c6f1f66010b/R/merge_rse_metrics.R#L83-L97\">https://github.com/LieberInstitute/jaffelab/blob/21e65745a5b0852fc13ebdb5e5d23c6f1f66010b/R/merge_rse_metrics.R#L83-L97<\/a> and related code merges <code>colData()<\/code> information for samples that were sequenced in more than one lane and whose FASTQ files were not merged prior to aligning with <code>SPEAQeasy<\/code>.<\/span>","<span class='gt_from_md'>I walked throught the run_DE function I wrote for MDDseq (a wrapper for the limma-voom) pipline. I pointed out what parts of the MDDseq code that could adapted and simplified for this use case (its less complicated than the many DE runs we completed for MDDseq)<\/span>","<span class='gt_from_md'>I suggested using conda after pip failed (missing system dependencies). Ultimately, my troubleshooting was enough to just build a module for everyone<\/span>","<span class='gt_from_md'>I explained details about the spatialDLPFC and deconvolution-benchmark papers. I also went over how we addressed feedback from the editors for the spatialDLPFC paper.<\/span>","<span class='gt_from_md'>Explained how to access authorized keys and exit JHPCE<\/span>","<span class='gt_from_md'>Having some conectivity issues with JHPCE, so we didn’t get a full bug fix in I suggested some de-buggin strategies  [Update 3/15] We met again breifly to confirm the functions were working as intended. Initial issue was caused by un-intentinally overwriting the function<\/span>","<span class='gt_from_md'>We discussed how to address feedback from a recent grant application, how our teams can get to know each other better, cleared up misunderstandings related to my team’s role as well as the DSgs system, and decided next steps to respond to the grant reviewers.<\/span>","<span class='gt_from_md'>I described how a markdown of notes could be added using the Samui python API, though this is not how I had done it in a previous project ( I was unable to trace the original method I used)<\/span>","<span class='gt_from_md'>We discussed more details about <code>iSEE<\/code> and shinyapps.io deployment details, such as memory settings and other configuration needed to actually customize and share an <code>iSEE<\/code> app. We also asked our collaborator for the list of genes they want to have plotted outside of <code>iSEE<\/code>.<\/span>","<span class='gt_from_md'>I helped implement an algorithm taking the minimum pixel intensity in case of overlapping images to display an H&amp;E image in Samui<\/span>","<span class='gt_from_md'>I diagnosed several potential issues, ultimately missing one detail that was the culprit: permissions were too open on some .ssh files on her laptop<\/span>","<span class='gt_from_md'>Explained what parts of VNS and refineVNS can be tuned to get appropriate segmentations for Nac project<\/span>","<span class='gt_from_md'>I sent a long and detailed email explaining different components of the LIBD BrainSEQ projects, background information on several covariates we typically have used in DEG models such as RIN and age at time of death, as well as other ones we have not used such as postmortem interval (PMI). I also explained the DSgs system and how it could be used by our JHU collaborators.<\/span>","<span class='gt_from_md'>Giovanna and Jisu are working with some bulk RNA-seq data as a some pilot data in a new brain region prior to preforming saptail. They want to use deconvolution to help QC the samples, selecting for good disections. I walked them through our current recomndations for deconolition to help with this goal.<\/span>","<span class='gt_from_md'>We went over internal code from the limma package using <a href=\"https://code.bioconductor.org/\">https://code.bioconductor.org/<\/a>, particularly for the <code>voom()<\/code> function. Doing so, we discussed that’s actually returned in the <code>voom()$E<\/code> slot. We also went over how to model a ratio using a linear regression model.<\/span>","<span class='gt_from_md'>We went topics such as live in Baltimore, research at JHU, overview of the different JHU campi, how to compare PhD programs across universities, and other questions this future PhD student had.<\/span>","<span class='gt_from_md'>Discussed feedback from collaborators and detailed next steps to wrap up this project.<\/span>","<span class='gt_from_md'>Went over recent updated plots and results, as well as slides for a presentation with our collaborators for 2024-02-21.<\/span>","<span class='gt_from_md'>We discussed the updates implemented in qsvaR to handle input with ENSEMBL transcript IDs instead of GENCODE ones, updates to automatically detect if the input is at the gene level instead of the transcript level, how to check for correlation between qSVs and covariates, among other related issues.<\/span>","<span class='gt_from_md'>I described my typical strategy of using left_join by sample ID, as well as ensuring rownames don’t get lost from conversion to a tibble (thus dropping colnames from the SummarizedExperiment unintentionally)<\/span>","<span class='gt_from_md'>I noted that the r_nac module was built similarly to conda_R, where the user doesn’t have to do anything special to install more packages. We ultimately traced the issue to permissions problems on files created as part of the proper use of the module by other users, likely due to problematic ‘umask’ settings<\/span>","<span class='gt_from_md'>Sreya went through an error issue in qsvaR due to rownames not well formatted. The error message was misleading. This problem has been fixed in the latest updates of the package. I advised Sreya to get the latest version of the package.<\/span>","<span class='gt_from_md'>Discussed updates to violing plots and how to organize result files prior to sharing them with our collaborators.<\/span>","<span class='gt_from_md'>I showed him to the 2 approaches available in Seurat to perform DGE, including the FindMarkers() function to find the genes that are different between control and treatments. I also show him code to perform quality controls in the main covariates related with these data-type. I shared with him consulted resources and links with public-code for his reference.<\/span>","<span class='gt_from_md'>I ultimately recommended ‘git reset –soft’ to the commit ID before the recent mistaken addition of the large files<\/span>","<span class='gt_from_md'>I advised using the SSH address (not HTTPS) for the ‘git remote’, ensuring the ‘git config’ email matches the primary GitHub email, and using the up-to-date SSH key (not RSA, which is no longer supported/ recommended)<\/span>","<span class='gt_from_md'>We explored options: BFG cleaner, git rebase and git reset –soft and discussed which was best for her situation<\/span>","<span class='gt_from_md'>I described the use of ‘srun’ (including with X11) vs. ‘sbatch’, and generally recommended ‘sbatch’ where possible to document how analyses were run/ ensure reproducibility<\/span>","<span class='gt_from_md'>Caught up and talked about career development next steps.<\/span>","<span class='gt_from_md'>I mostly focused on my setup using VS code with its remote SSH extension, and custom keyboard shortcuts for running python code interactively<\/span>","<span class='gt_from_md'>I mostly confirmed the general process he had started for a particular module made from software packages as a precompiled binary<\/span>","<span class='gt_from_md'>Went over the details of how to compute a permutation analysis with R.<\/span>","<span class='gt_from_md'>Sreya wants to compare the correlation of 5 transcipts with 2k degradation transcripts. She had the data in the right format but needed help appllying the cor function to the table using group_by. Corrrplot was not the correct function for what she wanted to achive here.<\/span>","<span class='gt_from_md'>Clarified updated automatic checks and information for users who attempt to use qsvaR with gene-level data instead of transcript-level data.<\/span>","<span class='gt_from_md'>I described how code might change when using IF images rather than H&amp;E, for which the code was originally developed<\/span>","<span class='gt_from_md'>Michael and Peter are working on a BPD vs. MDD DE analysis. They were getting more DEGs than were reported in an early version of the MDDseq project. I could not find the code for the data they were refrencing, with out the code it will be impossible to diagnosis what is differnt in the two analysis. I recomended they use the more up to date data from MDDseq, and pointed them to the correct data &amp; scripts.<\/span>","<span class='gt_from_md'>Tricia and Fernando are re-running serval analysis they previously had working on the sge cluster, with the update to slurm they are now having issues. They are also having differnt errors in the same scrips + data. This could be a multi-facet issue, but firstly sounds like a R package version issue. I advisded the first step should be to have the same set-up, work in the same version of R and the same version (most updated) packages. See what errors persist and go from there. I also recomended Frenando start using git repos and version control with his team to help keep projects organized and consistent.<\/span>","<span class='gt_from_md'>During the meeting Sreya showed me her progress on the analysis she’s doing taking the 2976 txs from the cell-component model and she wanted to compute the correlation between those and selected transcripts she’s interested in. She thinks those transcripts are involved in the degradation process. We couldn’t find those transcripts as part of the 2976 txs from the cell-component model.<\/span>","<span class='gt_from_md'>We checked udpates to the <code>getDegTx()<\/code> and <code>DEqual()<\/code> functions, as well as discussed the response to a bug report from last year.<\/span>","<span class='gt_from_md'>I described how to adapt a script I wrote for NAc to her dataset, where the goal was simply to display a stitched-together image in Samui<\/span>","<span class='gt_from_md'>I traced the issue to the use of 8-bit integers, where an overflow occured during normal addition of pixel values (the solution was to use floats while computing the mean, and then convert back to uint8)<\/span>","<span class='gt_from_md'>I described how I typically use dplyr::left_join, as well as how to add back missing rownames to the colData (or colnames of the SummarizedExperiment)<\/span>","<span class='gt_from_md'>I described the conflict between SGE and SLURM R packages, as well as identified lock files preventing re-installation of problematic packages<\/span>","<span class='gt_from_md'>We traced the offending line of R code in the qsvar package and I described how unit testing could identify problematic function arguments and give more informative error messages<\/span>","<span class='gt_from_md'>I met with Sreya because she’s having an issue with the getDegTx function from the qsvaR package. I tried to solve the issue but I wasn’t successful so I asked Sreya to give me more time and access to the rse object to have a more deeper look to understand the issue. After using her data I realized that the error message was misleading  and improved the function and unit tests.<\/span>","<span class='gt_from_md'>I noted that the function could be applied before the data was converted to long format, where it was impractically large to work with<\/span>","<span class='gt_from_md'>We identified issues causing incorrect coloring in the Samui outputs<\/span>","<span class='gt_from_md'>I described how to adapt a script I wrote for NAc to his dataset, where the goal was simply to display a stitched-together image in Samui<\/span>","<span class='gt_from_md'>I explained the different modules such as <code>module load conda_R/4.3.x<\/code> versus <code>conda_R/4.3<\/code>, how each is tied to a different version of Bioconductor. I also linked to <a href=\"https://youtu.be/D6t2GR7xaTE\">https://youtu.be/D6t2GR7xaTE<\/a> and <a href=\"https://docs.google.com/document/d/1DD9MUI2e3jp5S8C3PWRulZ4jwbXo94D3UTCQNJl_-Qo/edit?usp=sharing\">https://docs.google.com/document/d/1DD9MUI2e3jp5S8C3PWRulZ4jwbXo94D3UTCQNJl_-Qo/edit?usp=sharing<\/a> from the LIBD rstats club.<\/span>","<span class='gt_from_md'>Researchers are starting the annotation/cluster integration of the multiome data. The data is currently clusted indiviudally by sample, sn, ATAC only, and combined sn+ATAC. findMarker has be used to calc marker stats for each cluster in each sample (there is a ton of data to digest). I am unfamilar with the sample independent cluster, consered marker gene integration from seurat. But I gave advice to check the current clusters for high expression of any of the known marker genes to see if we’re picking up the expected biological signal, and do some prelimary annotations as a sanity check. Also to keep following the workflow and see if the conserved marker genes match with any of the know markers. Also to start recoring annotations for the clusters in a csv file for easy editing by hand as well as easy reading in to R<\/span>","<span class='gt_from_md'>Provided feedback on CV writing for data science job applications.<\/span>","<span class='gt_from_md'>I advised on how to adress some issues with the qSVA overview plot, both minor issues like aliginig and resising objects with biorender tools, and major rearngment of the table and DE application part.<\/span>","<span class='gt_from_md'>I described the workflow for stitching Visium capture areas with Samui, and various Samui API functions like ‘add_chunked_feature’ that are helpful<\/span>","<span class='gt_from_md'>We discussed the <a href=\"https://doi.org/10.1186/s13059-019-1861-6\">GLM-PCA paper<\/a> and decided to use GLM-PCA instead of logcounts by <code>quickCluster()<\/code> + <code>calculateSumFactors()<\/code> as described at <a href=\"https://comunidadbioinfo.github.io/cdsb2021_scRNAseq/normalizaci%C3%B3n-de-datos.html#normalizaci%C3%B3n-por-decircunvoluci%C3%B3n-deconvolution\">the CDSB 2021 snRNA-seq workshop<\/a>. We also discussed the development version of <code>Harmony<\/code>, in particular with the <code>lambda = NULL<\/code> setting.<\/span>","<span class='gt_from_md'>We went over how to build a reprex for a user’s bug report, how to simplify the process of debugging, and the actual results from this particular bug report. We also discussed how to relay this information back to the person who initially asked the question.<\/span>","<span class='gt_from_md'>I provided background information on a job ad from a colleague at Boston’s Children Hospital that I had shared. I also provided suggestions for preparing their GitHub account, CV, and other information useful for job interviewers.<\/span>","<span class='gt_from_md'>Adressing feedback on some unexpected behavior from qSVAR package. The qSV values show no significant correlation with other phenotype metrics (they have in the past), while this is not impossible it is strange. There are many factors that could have lead to this issue. We talked about some debugging strategies, starting with re-running the workflow on familar data to see if we can re-create the issue or detirmine if it is unique the external dataset.<\/span>","<span class='gt_from_md'>I provided background information on a job ad from a colleague at FHCRC that I had shared.<\/span>","<span class='gt_from_md'>I gave feedback on scripts, how to organize them, the order of some analytical steps, which gene annotation files to use, and things like that. We don’t have results on the QC steps to share just yet.<\/span>","<span class='gt_from_md'>I showed a workaround for making sure a variable has global scope before editing it inside a function, though I noted it’s better not to use a function in her particular case<\/span>","<span class='gt_from_md'>Continued reviewing qsvaR replication analysis code.<\/span>","<span class='gt_from_md'>Detailed analysis steps for <a href=\"https://github.com/LieberInstitute/Cg_Hb_projector_profiling\">this project<\/a> using snRNA-seq data and analyzing it following OSCA-based steps from the CDSB 2021 snRNA-seq workshop.<\/span>","<span class='gt_from_md'>Reviewed draft blog post on the Visium SPG AD project and published it at <a href=\"https://research.libd.org/rstatsclub/2023/11/01/our-first-adventure-with-visium-spatial-proteogenomics/\">https://research.libd.org/rstatsclub/2023/11/01/our-first-adventure-with-visium-spatial-proteogenomics/<\/a>.<\/span>","<span class='gt_from_md'>I identified two separate issues: the repository name and using HTTPS remote instead of SSH<\/span>","<span class='gt_from_md'>I walked through how a decision tree works and can be interpreted, then described how correlation and RMSE are computed in a couple spatialDLPFC spot deconvo plots<\/span>","<span class='gt_from_md'>He noted that the wrong version of python was being used, and we eventually traced this to user-specific conda settings interfering with the module<\/span>","<span class='gt_from_md'>Explained how interaction models work with limma-voom.<\/span>","<span class='gt_from_md'>Ibidem to 2023-09-29 session.<\/span>","<span class='gt_from_md'>Reviewed <a href=\"https://github.com/LieberInstitute/QSVA_2020/blob/914e3b58f7fb97c204654493c9f9dcb2e48d112b/code/06_Replication/02_DE_and_replication_plot.R#L220-L313\">this script<\/a> for the replication analysis performed as part of the qsvaR project.<\/span>","<span class='gt_from_md'>Paired programming session to debug <code>/dcs04/lieber/lcolladotor/with10x_LIBD001/ranger_metrics/code/spaceranger/03_merged_metrics/02_explore_metrics.R<\/code>. Further explained the original intent of the code in this script and steps for updating it.<\/span>","<span class='gt_from_md'>We interpreted the intuitive meaning of a formula from some old code, which attempted to quantify the similarity of a set of significant genes between datasets. I also helped interpret the meaning of a plot based on this metric<\/span>","<span class='gt_from_md'>This data has one cluster that is unqique to one donor, post batch correction. This donor also has some unique clinical background as well, I advised to drop the cluster as their batch corection seemed to be correct, and it possible that this is a rare biologically real cluster but there is not enough data to confirm.<\/span>","<span class='gt_from_md'>I recommended ‘spatialLIBD::fetch_data’ for easy retrieval of public data to get started with spatial computational analysis. I also recommended pushing for an account at JHPCE, where computational resources are more plentiful, and getting used to submitting jobs, accessing software, etc is valuable to learn<\/span>","<span class='gt_from_md'>Reviewed some scripts related to replication analyses. I explained the different thresholds used on the discovery as well as the target datasets, as well as the different levels for assessing replication.<\/span>","<span class='gt_from_md'>I walked through the new login commands for SLURM + updating his bashrc file so they can work with the slurm cluser<\/span>","<span class='gt_from_md'>Ibidem to 2023-09-29 session.<\/span>","<span class='gt_from_md'>I noted that conda_R seemed to be changing the version of java loaded, which affected how the nextflow module worked. I recommended unloading conda_R and avoiding the loading of conda_R via the .bashrc<\/span>","<span class='gt_from_md'>I described the general process of setting up SPEAQeasy and gave recommendations for his cluster: using singularity and SLURM-based installation/ set-up<\/span>","<span class='gt_from_md'>I mostly covered documentation but also the appropriate options for his particular data<\/span>","<span class='gt_from_md'>I investigated a “failure to submit process to scheduler” error using ‘sacct’ and similar commands, ultimately discovering the issue was related to SLURM configuration. There was an email chain with bithelp to follow up<\/span>","<span class='gt_from_md'>We went over the changes needed to take the current qsvaR paper draft by Josh Stolz to a production-level result (aka for pre-printing / publishing it).<\/span>","<span class='gt_from_md'>Adapted mode code for tensorQTL prep, had to add genomics ranges to the sce object, and deal with bugs around rse_to_bed function<\/span>","<span class='gt_from_md'>I followed up with bitsupport, as the issue seemed to involve some failed DNS query internal to SLURM, rather than SPEAQeasy<\/span>","<span class='gt_from_md'>We walked through old code which aimed to quantify the similarity between significant DE results, interpreting formulas from the code and constructing more intuitive formulas<\/span>","<span class='gt_from_md'>Ibidem to 2023-09-29 session.<\/span>","<span class='gt_from_md'>Worked through addapting code for tensorQTL prep for the MDD single cell data - had to deal with bugs relating to cell type names<\/span>","<span class='gt_from_md'>I gave general advice on re-writing code– keeping separate scripts, chunking code into segments and testing equality of outputs between old and new code after each chunk, and other strategies<\/span>","<span class='gt_from_md'>We continued working with the OSCA-based CDSB 2021 intro to snRNA-seq workshop to further understand snRNA-seq data analysis steps, packages, and how they all relate to each other. The person who requested this DSgs was working on the workshop contents and had questions that we reviewed weekly (or so). Once they completed this OSCA-based training, we moved to analyzing an actual unpublished snRNA-seq dataset.<\/span>","<span class='gt_from_md'>We looked at the rank vs. total UMI curves relevant for emptyDrops for the atac-seq samples. We have discussed these samples before. We went over how the emptyDrops function works again and what we can guesss is happening in the curve (neurons vs. glia vs. empty droplets).  There is concerned beause for the 42-4 sample the wetbench target number of nuc was 7k, emptyDrops retrurns 9k. I thought that that sounded in the right ball park<\/span>","<span class='gt_from_md'>I described the ‘droplevels()’ function, designed to remove unused factor levels<\/span>","<span class='gt_from_md'>We went over the background for this project (lateral septum) and an overview of the DSgs system, GitHub setup, and other onboarding material.<\/span>","<span class='gt_from_md'>I discussed how we selected training data for the spot-deconvolution image model– using logistic regression to compute a confidence metric across the original training examples, that we then binned into 4 quartiles to re-select new examples to label in a less biased way<\/span>","<span class='gt_from_md'>I mentioned slurmjobs::job_loop and also described how to use it in conjunction with getopt in R for looping over several variables<\/span>","<span class='gt_from_md'>She had an issue installing a particular R package, which we pinpointed to which modules were loaded (including via bashrc). I also described the general problem of SLURM and SGE versions of JHPCE sharing a library path, which could result in subtle problems. I recommended deleting SGE packages (better would’ve been backing up!) and reinstalling on SLURM<\/span>","<span class='gt_from_md'>I helped modify my code for creating input files to work witht the new pseudobulk data. We got the gene PCs done, he needs to get the genotype files together for this data before we can move to the next steps. Planning to meet more next week to continue this process.<\/span>","<span class='gt_from_md'>Resolved some questions related to <a href=\"https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq\">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq<\/a> and made two pull requests to fix typos.<\/span>","<span class='gt_from_md'>We went over the math derivations for interpreting beta1, beta2, and beta3 on an interaction model where X1 is a continuous variable, X2 is a discrete variable with 2 options and we model <code>Y ~ X1 * X2<\/code>. We also used <code>ExploreModelMatrix<\/code>.<\/span>","<span class='gt_from_md'>I explained the benefits of centering and scaling and referred to both the short video on this topic <a href=\"https://youtu.be/c8ffeFVUzk8\">https://youtu.be/c8ffeFVUzk8<\/a> as well as the material we developed for the CSHL 2023 course <a href=\"https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html\">https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html<\/a>.<\/span>","<span class='gt_from_md'>They were interested in running deconvolution on the bulk MDDseq with the new large MDD snRNA-seq dataset. The MDD snRNA-seq dataset is only sACC, we concluded that only haveing one region might make this analysis a bit awkward for the Amyg data. I them to the current deconvolution results and how we applied those results in downstream analysis.<\/span>","<span class='gt_from_md'>I walked through using registration_pseudobulk() on the large MDD single cell dataset and went over how the function works. We are planning to meet agian to go over running PCA to prep for eQTL analysis.<\/span>","<span class='gt_from_md'>I noted the ‘www’ directory (or the directory passed to the ‘docs_path’ parameter) controls the home page, and we typically use a relative soft link to the main repository README for the bulk of the content.<\/span>","<span class='gt_from_md'>I described the general workflow, involving ImageJ, Samui, VisiumStitcher, and custom R + python code needed to eventually produce SpatialExperiment objetcs with one donor per sample. I focused on the high level of what the Samui piece looked like/ how much work is involved<\/span>","<span class='gt_from_md'>Continued reviewing material from <a href=\"https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq\">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq<\/a><\/span>","<span class='gt_from_md'>Discussed a few ideas about LIBD and social media<\/span>","<span class='gt_from_md'>I showed code examples of writing array jobs with SLURM (and how it compares to SGE), then diagnosed an X11 forwarding issue where the ‘-x11’ srun option was insufficient. Adding X11-related configuration in the SSH config fixed the issue (i.e. ForwardX11 and ForwardX11Trusted lines)<\/span>","<span class='gt_from_md'>I discussed diagnosing marker quality for the spatial hypocampus project. They are on the right track, and have created violin + hockystick plots for the different iterations of the dataset. I advised they elimate a noisy cluster and re-compute markers.<\/span>","<span class='gt_from_md'>I described the pieces that go into a bookdown-powered documentation site, showing RMD code examples and the GHA workflow that deploys the site<\/span>","<span class='gt_from_md'>I noted that Samui supports many types of features, and the spatial coordinates of cells can be supplied instead of spot coordinates to display features related to cells<\/span>","<span class='gt_from_md'>I described the meaning of cell2location outputs and how to compare outputs between software tools. I also described where plots came from in R<\/span>","<span class='gt_from_md'>Used MAD cutoffs on total UMIs, genes detected, mito %, and a metric we haven’t worked with yet: genes per UMI. They evaluated the data all together (not by sample) and chose #Mads by what seemed reasonable. The values  used look good, but perhaps a bit strict. We looked at the distributions, which are slightly different between samples, I advised to use scater::isOutlier to see what the cutoffs would be by sample. One step lacking is doublet detection, I suggested to apply scDblFinder::computeDoubletDensity. The dataset already has clusteres and is annotated  (lots of work) I suggested that a great double check is to calcluate the median value of the QC metrics for each final cluster and see if any are much higher than the others as a good double check that his initial QC process worked well.<\/span>","<span class='gt_from_md'>Answered questions related to material at <a href=\"https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq\">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq<\/a><\/span>","<span class='gt_from_md'>We talked about the first scripts in <a href=\"https://github.com/LieberInstitute/ranger_metrics\">https://github.com/LieberInstitute/ranger_metrics<\/a><\/span>","<span class='gt_from_md'>He noted that not all measured genes were displayed in the Visium_SPG_AD project. I found in the code why I subsetted and made a GH issue to fix it<\/span>","<span class='gt_from_md'>I shared my favored set up for getting computational work done: working directly on JHPCE via the SSH extension for VSCode, using system-wide JHPCE modules for consistent software versioning, and X11 for interactive plotting (with a note I tend to just create and view PDFs for complex plots)<\/span>","<span class='gt_from_md'>My major recommendation was learning version control with git, though I also mentioned learning SGE/SLURM also is a valuable skill for larger-scale coding projects. I helped him create a git repository for his current project (including account set up) and described the basics of git add, commit, and push<\/span>","<span class='gt_from_md'>I described how the various scripts connect in spatialDLPFC for my work with deconvolution, to help share and re-use them for her project<\/span>","<span class='gt_from_md'>We talked about material at <a href=\"https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq\">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq<\/a><\/span>","<span class='gt_from_md'>I explained the status of the recount-pump transfer, where files were, and noted that chris or JHPCE admins would have to change permissions to allow her access to some files<\/span>","<span class='gt_from_md'>I noted that modules at SGE vs. SLURM are totally disjoint, and that the HOSTNAME trick originally used in the team recommended bashrc didn’t (at the time, though this has recently changed) work anymore, and the SLURMD_NODENAME variable can instead be used to infer where the user is<\/span>","<span class='gt_from_md'>I gave some pointers about writing tweets. Mostly keep it simple, give people a call to action “check out this link!”, add something eye catching. Also to spend sometime reading tweets and pay attention to what you like that other people tweet? What caught your interest? We also talked some stradgedy about tweeting around confrences, calling peoples attention to your talk/making it easy for people to find you to get in touch<\/span>","<span class='gt_from_md'>We talked about how I became an Investigator at LIBD, the overall type of work we do as a team, some of our main collaborations, and overall plans for applying for funding in the future. In particular, I highlighted how I want to train team members on how to write an NIH biosketch, though not all are going to be “key personnel”. I sent a link to the DSgs, team meeting schedule, rstats club schedule, and our <a href=\"http://lcolladotor.github.io/talk/teamtldr2022/\">http://lcolladotor.github.io/talk/teamtldr2022/<\/a> summary from April 20, 2022.<\/span>","<span class='gt_from_md'>I did a complete tutorial of using the Samui API to create Samui-importable directories, including how to preprocess image files and how to host results publicly<\/span>","<span class='gt_from_md'>We talked thorough the thigns to think about for the Mathy’s dataset (this dataset has case + control paitents and found DEGs w/in cell types). I advised to start simple with the broad cell tytpes and not consider Dx at first. Check if the DEG they idetified are in our marker set. I think this will need some discussion with the project team if Dx is a concern for spot deconvolution.<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/5e84f5ce32e6723b1a1eea20dc47ee320d55ca10\">https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/5e84f5ce32e6723b1a1eea20dc47ee320d55ca10<\/a><\/span>","NA","<span class='gt_from_md'>He noted that just taking the most plentiful cell type in any cluster was leading to too many clusters with excitatory layer-5 cells. I mentioned that the raw outputs of spot deconvolution tools are fairly unreliable, but they still carry useful information when normalized. For his example, I mentioned dividing by the total number of cells for each cell type when making comparisons between cell types<\/span>","<span class='gt_from_md'>Suggested deleting all old files as the code might be appending data instead of creating new files and also increasing the h_fsize, as the error relates disk space<\/span>","<span class='gt_from_md'>guided him to the search engine on REDCap<\/span>","<span class='gt_from_md'>For deconvolution, we talked about finding an appropriate reference dataset and we talked about an expert we can contact to ask which dataset(s) might be the most relevant. For snRNA-seq data analysis, I mentioned OSCA (Orchestrating Single Cell Analysis with Bioconductor) and how Louise could help guide this person on our code based off OSCA through a series of DSgs sessions.<\/span>","<span class='gt_from_md'>removing the top 2500 pixels before splitting the image corrects the offset<\/span>","<span class='gt_from_md'>I mentioned (information) entropy as a metric that would determine how well a gene is confined to a particular cluster (using average counts of a gene across all spots in each cluster, then normalizing the sum to add to 1), but cautioned that I wasn’t familiar with favored metrics within the field<\/span>","<span class='gt_from_md'>We transferred <a href=\"https://github.com/LieberInstitute/qsvaRPkgDemo\">https://github.com/LieberInstitute/qsvaRPkgDemo<\/a> to the LieberInstitute account. I asked to include some citations for sources (like the image source) and some other small edits. For GO / KEGG enrichment with clusterProfiler I asked to not filter results (pvalueCutoff / qvalueCutoff parameters), so we can see more results prior to choosing which ones to plot.<\/span>","<span class='gt_from_md'>We took a look at the changes in the clustering in the spatial data before and after cropping out a area of questionalble quality. To me it looks like the QC params are too strict leaving spots floating in space, which I’m guessing is contributing to weird BayesSpace results (but this is a guess). We also talked code organization strtagedy - mostly keep it simple and consistent<\/span>","<span class='gt_from_md'>I helped improve the formatting &amp; content for a BioC qSVA workshop, main feedback ins to focus on and highlight the qsvaR steps in the workflow (rather than SummarizedExperiment overvew &amp; DEG analysis)<\/span>","<span class='gt_from_md'>We discussed the current status of the team, the project, what skills are required, boundaries in terms of time and who would be involved (and who wouldn’t). We ended the session with me relaying a list of resources to check and read more carefully. Then in about 2 weeks we’ll meet again to determine the next steps.<\/span>","<span class='gt_from_md'>We made a Google sheet with columns such as <code>study_type<\/code>, <code>skill_name<\/code>, <code>learning_priority<\/code>, etc. I also worked on it after the DSgs session to help complete the current information. See <a href=\"https://neuralplasticity.slack.com/archives/C04Q8MX99FY/p1690397027111449\">Slack<\/a> for more details.<\/span>","<span class='gt_from_md'>We resolved this issue by making sure that the variable names for which they wanted to specify the correct colors were actually part of the colData(SE). See <a href=\"https://jhu-genomics.slack.com/archives/C01KSLPCGAC/p1689858182290369\">Slack<\/a> for the actual code links. I also asked them to make sure to use soft links / full paths instead of using setwd() for files outside the project + to keep all related files at JHPCE instead of on a laptop, to make it easier to rule out any sources of differences (aka for reproducibility purposes).<\/span>","<span class='gt_from_md'>I helped review a practice BioC short talk, Mostly the talk needs to give more slides to discussing the results &amp; include more actual plots not just icons (this is much improved this presentation for the 7/26 meeting :) )<\/span>","<span class='gt_from_md'>I helped discover the source and solutions to errors and warnings (mostly from BiocCheck and R CMD check), and found how to specify a single author in in-text citations<\/span>","NA","<span class='gt_from_md'>reffered to this script /dcs04/lieber/marmaypag/Image_Processing/NM_LC_segmentation/test.m<\/span>","NA","<span class='gt_from_md'>We discussed what would be the appropriate material for us to teach in a course. We will re-use some of our materials from <a href=\"https://github.com/lcolladotor/cshl_rstats_genome_scale_2023\">https://github.com/lcolladotor/cshl_rstats_genome_scale_2023<\/a>.<\/span>","<span class='gt_from_md'>We went over the proposed response to reviewers, clarified some sentences, and requested co-authors to approve the response to the reviewers.<\/span>","<span class='gt_from_md'>I pointed her to spot deconvolution scripts used in spatialDLPFC and the high-level overview of the workflow, as well as how we trained the decision tree classifier<\/span>","<span class='gt_from_md'>I noted that he ran out of memory in his JHPCE job, which crashed R, so I suggested increasing ‘mem_free’ and ‘h_vmem’<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/tif2mat.m\">https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/tif2mat.m<\/a><\/span>","<span class='gt_from_md'>I referred to examples of JHPCE modules I’ve created that activate shared conda environments, and explained the key commands used to get a similar module working<\/span>","<span class='gt_from_md'>I recommended finding and trying to generalize the process a previous coworker followed to produce the list of degradation-associated transcripts, to limit relying on exact transcripts with an exact GENCODE version. I noted that you can convert transcript data to genes (albeit with lost information), but not vice-versa, so in theory either transcript or gene-level data could be supported by converting to genes<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/mat2tif.m\">https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/mat2tif.m<\/a><\/span>","<span class='gt_from_md'>referred here for clarification <a href=\"https://docs.google.com/presentation/d/1HJgkvlHtiIA5nucuNdHdYhRGAtpPPvASGFoAocqi1-U/edit#slide=id.g1163185b778_1_6\">https://docs.google.com/presentation/d/1HJgkvlHtiIA5nucuNdHdYhRGAtpPPvASGFoAocqi1-U/edit#slide=id.g1163185b778_1_6<\/a><\/span>","<span class='gt_from_md'>We adapted code from <a href=\"https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R\">https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R<\/a> and <a href=\"https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd\">https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd<\/a> in order to make the files at <a href=\"https://github.com/LieberInstitute/septum_lateral\">https://github.com/LieberInstitute/septum_lateral<\/a>. See <a href=\"https://github.com/LieberInstitute/septum_lateral/commits/main\">https://github.com/LieberInstitute/septum_lateral/commits/main<\/a> from <a href=\"https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50\">https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50<\/a> to <a href=\"https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c\">https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c<\/a> as well as related commits on the <code>gh-pages<\/code> branch.<\/span>","<span class='gt_from_md'>Summarized at <a href=\"https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659\">https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659<\/a>. This involved explaining why we don’t use as a background universe all genes in the annotation in gene ontology / KEGG pathway enrichment analyses. Instead, we use those that pass the expression filters and for which we computed differential expression statistics.<\/span>","<span class='gt_from_md'>We resolved <a href=\"https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199\">https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199<\/a> at <a href=\"https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&amp;cid=C02RG5VGL6R\">https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&amp;cid=C02RG5VGL6R<\/a>, which was due to a user incorrectly having a Linux group assigned to files at JHPCE when they are not part of the user group. Uploads through Cyberduck can trigger this. The solution is to run <a href=\"https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example\">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example<\/a> again on a qrsh session.<\/span>","<span class='gt_from_md'>DropletUtils::emptyDrops on the two LIBD + two example samples has been run. We first clarified some terms and outputs she had recorded. For the most part DropletUtils returns similar number of non-empty cells as CellRanger arc, but didn’t confirm that the two methods flag the same cells. I recomended to create a confision matrix between the two methods for each sample to check this.  The main concern  has is the LIBD sample with only  3k cells input, has 9k non-empty droplets returned by emptyDrops. This is quite odd as the CellRanger Arc metrics look quite bad for this sample, but the DropletUtils metrics look fine (there are high UMI values for many droplets). We will want to dig in to this more to understand the differnces between the two methods.<\/span>","<span class='gt_from_md'>We worked on qsvaR debugging an issue raised by Renee and Louise. We then improved the error messages on qsvaR as noted at <a href=\"https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400\">https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400<\/a>.<\/span>","<span class='gt_from_md'>Using the septum lateral data, I trained Ryan on how to make this type of Globus endpoint, how to set up the right permissions, how to edit and update <a href=\"http://research.libd.org/globus/\">http://research.libd.org/globus/<\/a> (see <a href=\"https://github.com/LieberInstitute/globus/blob/b10e0fb89c40489d23028e8ed3f7812cdea76a6b/index.Rmd#L57\">this<\/a> for details), and how to document the Globus information on a manuscript.<\/span>","<span class='gt_from_md'>Using the lateral septum snRNA-seq data, I trained Heena on how to filter a SingleCellExperiment object to reduce the required memory from 7GB to less than 1GB in order to be able to deploy an iSEE app at shinyapps.io, whose computers have a maximum of 8 GB of RAM. See <a href=\"https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98\">https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98<\/a> and related commits.<\/span>","<span class='gt_from_md'>We worked with the script where Renee had noticed issues with qsvaR. We adapted it to work on qsvaR local updates through devtools::load_all(). We also resolved <a href=\"https://github.com/LieberInstitute/qsvaR/issues/29\">https://github.com/LieberInstitute/qsvaR/issues/29<\/a> and <a href=\"https://github.com/LieberInstitute/qsvaR/issues/28\">https://github.com/LieberInstitute/qsvaR/issues/28<\/a>.<\/span>","<span class='gt_from_md'>We started with getting github token access for their laptop working. Then started a pull request using usethis for qSVA, for them to add seed capabilities to several functions. We will need to meet several more times to complete the whole help request<\/span>","<span class='gt_from_md'>they mentioned that they had to login everytime she pushed a commit from the directory in question (but not other repos). I thought we should start by using a github access token to update her login to github from JHPCE. The best way I know how to do this is to use gitcreds::gitcreds_set() in R. At first it gave us an error, which was solved by adding the git modules loaded in the team bashrc to theur basrc file as well as their .gitconfig file. After that we were able to save the token to their JHPCE setup and the github push issue seemed to be solved<\/span>","<span class='gt_from_md'>I described the ‘add_coords’ function, how features link to existing coordinates, and that coords must have at least one associated feature to display<\/span>","<span class='gt_from_md'>I referred her to recent versions of my code (spatialDLPFC and SPG_AD projects) and walked through the high-level steps of each<\/span>","<span class='gt_from_md'>Reviewed input for tensorqtl (it is counts, not logcounts), and revised what how we filter lowly expressed exon-exon junctions using <a href=\"http://research.libd.org/jaffelab/reference/expression_cutoff.html\">http://research.libd.org/jaffelab/reference/expression_cutoff.html<\/a>.<\/span>","<span class='gt_from_md'>Discussed how to best pseudobulk the data prior to doing this type of analysis. In particular, how to adapt <a href=\"https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148\">https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148<\/a> and related code, as well as plots to make such as boxplots of the min_ncells by cell type (without any filtering) and cutoffs found by <code>isOutlier()<\/code> (different cutoff for each cell type) + PCA plots colored by min_ncells.<\/span>","<span class='gt_from_md'>Discussed a few different areas of growth, including a change in direction related to our outreach activities.<\/span>","<span class='gt_from_md'>I showed that the error was essentially a failure to allocate memory and recommended doubling required memory for the failing SPEAQeasy process until success<\/span>","<span class='gt_from_md'>I described that an entire directory (rather than all of its contents) can be ignored, and should be specified relative to the directory containing the .gitignore<\/span>","<span class='gt_from_md'>We looked at some old code vs. an adapted version, which was not producing the same output. I recommended splitting the code into ordered sections, and executing the old and new version of each to identify exactly where the error was occurring<\/span>","<span class='gt_from_md'>I discovered she was using the –small_test argument and not –input, so only the small 3-sample test set was running instead of her actual data<\/span>","<span class='gt_from_md'>I walked through the meaning of certain information-dense plots I made to show spot-deconvolution results<\/span>","<span class='gt_from_md'>I recommended filtering rows of the RSE via ‘seqnames’, which would return genes (or features in general) of the particular chromosomes needed<\/span>","<span class='gt_from_md'>She asked about any quality-check-related options SPEAQeasy had (I pointed her to ‘–ercc’ for applicable data) and whether trimming defaults were reasonable for her data<\/span>","<span class='gt_from_md'>I described the general workflow needed to start from raw images and spaceranger outputs, and end up with a Samui-compatible directory that can be dragged into the browser (locally) to view Visium data (including segmented fluorescence channels)<\/span>","<span class='gt_from_md'>I suggested creating a template configuration file, then looping through her data, using the sed command to write configuration files with the relevant variable replaced from the template<\/span>","<span class='gt_from_md'>She wanted to add the columns associated with unique rows in one data frame to the other, so I suggested combining the ‘unique()’ command with ‘left_join()’<\/span>","<span class='gt_from_md'>I suggested using regular expressions in R to get the new filename from a given old filename, then file.copy to perform a safe test, and finally deleting the old files<\/span>","<span class='gt_from_md'>She was unable to push to GitHub on her laptop. It appeared she didn’t have an SSH keypair set up with GitHub, so we set that up to fix the issue<\/span>","<span class='gt_from_md'>The researcher was concerned about the large number of variables in the reproducibility DE experiment, specifically the PC. I pointed out that the point of the anlaysis was to test the effectiveness of the models (with and without qSVs) so while this might be a valid concern in a sperate DE analysis, we are testing model quality here. I also noticed that the reproducibility anlaysis was not using functions from the qsvaR package, and that as qsvaR is the “product” of this whole project its important we use the functions in this anlysis .<\/span>","<span class='gt_from_md'>I described typical expected alignment rates, the difference between concordant and overall map rate, and details about other SPEAQeasy output metrics<\/span>","<span class='gt_from_md'>I explained that at the time, I just created the RMD file and added any referenced images to the rstatsclubsource repo, following naming conventions that include the post date and hyphens between words in the blog title. I believe Leo did some manual work to get it fully published, so I referred to the full guide in case my method was incomplete: <a href=\"http://research.libd.org/rstatsclub/2018/03/09/contributing-to-the-libd-rstats-club/\">http://research.libd.org/rstatsclub/2018/03/09/contributing-to-the-libd-rstats-club/<\/a><\/span>","<span class='gt_from_md'>I explained how cell2location is agnostic to what gene identifiers are used, but that gene indentifier type must match between spatial and reference single-cell data. I mentioned the org.Hs.eg.db package.<\/span>","<span class='gt_from_md'>I described when I make a module for python-based tools, the benefits of conda vs. python virtual environments, and related topics<\/span>","<span class='gt_from_md'>I suspected improper configuration of email associated with her git commits, which was indeed the problem. Her commits moving forward are associated with her GitHub account, but we couldn’t figure out how to retroactively regain credit for commits done with the wrong email<\/span>","<span class='gt_from_md'>I described ‘git pull’ to merge in remote changes, and using the ‘vi’ editor to write a merge message<\/span>","<span class='gt_from_md'>I discussed with Renee the way I code and showed her some examples and gave her a few coding advices.<\/span>","<span class='gt_from_md'>Looks like user was trying to create a plot and import data from a .csv file but was getting an error importing the csv\n<a href=\"https://community.rstudio.com/t/error-creating-a-windrose/164132\">https://community.rstudio.com/t/error-creating-a-windrose/164132<\/a><\/span>","<span class='gt_from_md'>I explained which paths I’d need to get Samui running on our data, and we discussed which features might be useful to show<\/span>","<span class='gt_from_md'>Project links reviewed: <a href=\"http://research.libd.org/smokingMouse/\">http://research.libd.org/smokingMouse/<\/a> and <a href=\"http://research.libd.org/smokingMouse/articles/smokingMouse.html\">http://research.libd.org/smokingMouse/articles/smokingMouse.html<\/a><\/span>","<span class='gt_from_md'>I described how ACLs are properly set up at dcs05 to inherit downward automatically, so just an ‘rsync’/ move to trash is necessary in those transfers<\/span>","<span class='gt_from_md'>I explained the purpose of Git-LFS, that it’s configured for certain repositories, and that we can load a module to use it<\/span>","<span class='gt_from_md'>Website tools to look for public rna-seq samples &amp; packages to make synthetic replicates <a href=\"https://support.bioconductor.org/p/9150681/#9150698\">as shown here<\/a>.<\/span>","<span class='gt_from_md'>I described my use of XQuartz, the ‘xhost +local:’ command, and the proper ordering of these tools in combination with SSHing to allow plotting to work<\/span>","<span class='gt_from_md'>We discussed the use of qsvaR. For now the qsvaR only use the Gencode25 annotation. We also commented on the steps needed in case we want to implement the package for the more updated annotation versions in the future.<\/span>","<span class='gt_from_md'>I described what we use modules for, the general approach to making them for common software types (binaries, python packages), and walked through the process by making a new FastQC module<\/span>","<span class='gt_from_md'>We went over the code I previoulsy wrote for tensorQTL, and some of the custom functions for preparing the input data. How to input the different datatypes in to python, and run the function, and strategies  for running the different cell types in parralell on the GPU queue<\/span>","<span class='gt_from_md'>We discussed each step of the pipeline and the outputs. We also had a look at the QC metrics and the code that Cynthia added to the script.<\/span>","<span class='gt_from_md'>I helped decipher an error message they got in the BioC check in the GitHub Actions involving a local path harcoded in to one of her tests. We also debugged for similar issues by searching her code for other local paths that would cause her code  to break when run remotely<\/span>","<span class='gt_from_md'>We reviewed the overflow to analyse RNA-Seq and ATAC data with the Seurat and Signac tools. I provided learning resources, such as data file organization, funtions and methos available in the tools. I also provided a complete explanation about QC for preprocessing data integrated.<\/span>","<span class='gt_from_md'>I helped him set up git on his laptop (username + email, SSH key), showed how to edit R Markdown files using RStudio, and basic version control (git add, commit, and push).<\/span>","<span class='gt_from_md'>The error message was cryptic, but made more clear in the latest SPEAQeasy: the transcripts FASTA mismatched with the GTF (custom annotation was used) due to a typo in the gene ID in the GTF.<\/span>","<span class='gt_from_md'>We went through the  <a href=\"https://github.com/HediaTnani/Int_API_R\">https://github.com/HediaTnani/Int_API_R<\/a>. Then we saw other API examples with authentication (github, ….).<\/span>","<span class='gt_from_md'>Issue and the solution found here <a href=\"https://support.bioconductor.org/p/9150363/#9150404\">https://support.bioconductor.org/p/9150363/#9150404<\/a><\/span>","<span class='gt_from_md'>user had question about generating rarefaction curves in R. Issue found here <a href=\"https://support.bioconductor.org/p/9150366/#9150386\">https://support.bioconductor.org/p/9150366/#9150386<\/a><\/span>","<span class='gt_from_md'>I shared the tensorQTL example I presented at rstats club. We also went over the different in-puts and formats, including some we can re-use from bulk MDDseq. WE also discusses the differnt setting for tensorQTL and how to adapt sh files to submit to the GPU queue<\/span>","<span class='gt_from_md'>I initially mistakenly suggested kallisto, but later clarified that using featureCounts with a custom single-gene GTF from the BAM files (produced by SPEAQeasy) would be the proper way to quickly quantify a single gene<\/span>","<span class='gt_from_md'>See <a href=\"https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099\">https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099<\/a> for more. Led to <a href=\"https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6\">https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6<\/a> and related commits. spatialLIBD version 1.11.10 now supports spaceranger version 2023.0208.0 output files.<\/span>","<span class='gt_from_md'>We used <a href=\"https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection\">https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection<\/a> and also went over the JHPCE and R config files from the team website. We also talked about the DSgs and how it would be beneficial for Kinnary to meet 1 on 1 with all DSgs guides to build familiarity and make it easier to ask for help in the future.<\/span>","<span class='gt_from_md'>See <a href=\"https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179\">https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179<\/a> for more details. Heena is working with spaceranger output files from 2 different versions, and in particular, version 2023.0208.0 (internal 10x Genomics version) changed their output files structure<\/span>","<span class='gt_from_md'>Described what we do, the DSgs system, potential for visitors at our team meeting, and research ideas.<\/span>","<span class='gt_from_md'>We discussed how to merge two datasets in R, making sure they shared the exact set of genes<\/span>","<span class='gt_from_md'>With Cynthia we saw the differents folders that are part of the template project repository and how to organize your project repository.<\/span>","<span class='gt_from_md'>The solution involved asking BitSupport to login as a specific user and run <a href=\"https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example\">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example<\/a>. Knowing the exact file paths that triggered the permission issues are important to help check with <code>ls -lh<\/code> and <code>nfs4_getfacl<\/code> under a <code>qrsh<\/code> session.<\/span>","<span class='gt_from_md'>See the <a href=\"https://community-bioc.slack.com/archives/CR1BLV821/p1677783580326559\">Bioconductor Slack message<\/a> I wrote for more details.<\/span>","<span class='gt_from_md'>End result: <a href=\"https://twitter.com/HeenaDivecha/status/1631360771342098433\">https://twitter.com/HeenaDivecha/status/1631360771342098433<\/a><\/span>","<span class='gt_from_md'>See <a href=\"https://github.com/drighelli/SpatialExperiment/issues/135\">https://github.com/drighelli/SpatialExperiment/issues/135<\/a> and the pull request at <a href=\"https://github.com/drighelli/SpatialExperiment/pull/136\">https://github.com/drighelli/SpatialExperiment/pull/136<\/a> that resolves this issue.<\/span>","<span class='gt_from_md'>I described the need to ‘git add’ both the old filenames and new filenames, and the use of ‘git add –all’ to reference deleted files.<\/span>","<span class='gt_from_md'>I recommended code for matching two data frames up by common genes, and explained that matrices don’t support multiple data types (instead a ‘data.frame’ might be used)<\/span>","<span class='gt_from_md'>Researcher was having some issues with functions from DeconvoBuddies, namley plot_marker_express. I am going to check that on my end as it seems like the computer wasn’t finding the support functions for the main function. They were also having trouble matching the gene names between the sce and bulk rse objects, the ensembl names seemd to have differnt versions, I advided to maybe switch to using the gene symbols to keep things simple.<\/span>","<span class='gt_from_md'>No public result just yet.<\/span>","<span class='gt_from_md'>We went over the ratioinal behind psuedobulking and how we can use heatmaps to show marker experssion. Created psuedobulk data for three different clustering methods, and went over the basic process of making heatmaps. Bukola will work on the aestetics of the heatmaps, and cerate one for all thee clustering methods.<\/span>","<span class='gt_from_md'>related to my meeting with Uma on 2/10. We fixed Boyi’s bashrc to allow group write by deafult, fixing the ,git issue and eliminating the need to re-run the permission update script often<\/span>","<span class='gt_from_md'>I showed how glob expressions can be used to conveniently match many files across potentially several files, and described gitignore-specific rules, such as certain wildcard expression matching files no matter what directory they are in<\/span>","<span class='gt_from_md'>Researcher was having file permission issues, the group permissions for several files in a shared project were set to read only. This was causing issues with the team using scripts but also in making git commits as some of the idex files in .git were read-only. We updated her bashrc files to contain “umask u=rwx,g=rwx,o= ## equivalent to umask 007” to prevent this in the future and ran the premissoin update script.<\/span>","<span class='gt_from_md'>Researcher hads run muliple clustering methods on the habenula data, I advised to pick the two that were the most simmlar to previous annotations (we were happy with) via rand index, and start examining marker gene expression.<\/span>","<span class='gt_from_md'>I went over details like naming the OneDrive folder and correctly using CrossReference to refer to that name in figures and references to those figures<\/span>","<span class='gt_from_md'>I helped port several plots into Illustrator as a figure, describing our normal process of using PDFs and linked files to maintain resolution.<\/span>","<span class='gt_from_md'>I showed how to filter for individuals with a certain condition, and subset two data frames with the same criteria.<\/span>","<span class='gt_from_md'>She was unable to clone private repositories due to a problem with her SSH key with GitHub. We solved the problem by setting up a new key pair using the recommended Ed25519 algorithm instead of the deprecated RSA algorithm.<\/span>","<span class='gt_from_md'>We focused mostly on the options Nextflow provides for managing software dependencies: Lmod modules, conda environments, and docker + singularity containers and the benefits of each potential approach<\/span>","<span class='gt_from_md'>We discussed how to implement github actions with Nextflow workflow and the use of Conda and/or Docker containers.<\/span>","<span class='gt_from_md'>We identifed that SPEAQeasy cannot handle runs with only one sample, but that the intention was to use multiple samples (the manifest did not use unique sample names)<\/span>","<span class='gt_from_md'>The researcher created a github account and I talked them thuogth the basics of git, But we hit a snag with the instalation of git on their laptop, git is built in to xcode in the ventura OS (which her machine is running) but she had some trouble with that instalation too. They’re going to try and fix that independtly, and reach out if she needs more help gettign off the ground with git + github<\/span>","<span class='gt_from_md'>We adapted code from the DLPFC work and <a href=\"https://www.stephaniehicks.com/biocdemo/articles/Demo.html\">https://www.stephaniehicks.com/biocdemo/articles/Demo.html<\/a> to run GLM-PCA to prep for batch correction<\/span>","<span class='gt_from_md'>We went over several concepts in Nextflow (e.g. processes and channels) and discussed how to invoke R scripts inside processes.<\/span>","<span class='gt_from_md'>We (Me + Nick) discussed with Sean the basics of Nextflow and Nextflow DSL2 and how to make processes and channels and workflows. We gave him resources to read  and gave him small examplesto follow. We helped him also with his errors on slack.<\/span>","<span class='gt_from_md'>We adapted QC code for the habenula data<\/span>","<span class='gt_from_md'>See <a href=\"https://jhu-genomics.slack.com/archives/C0498763CUC/p1673451228034439\">Slack notes<\/a> for further details. They involved <a href=\"https://github.com/LieberInstitute/template_project\">https://github.com/LieberInstitute/template_project<\/a>, <a href=\"https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h\">https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h<\/a>, and other specific instructions for setting up a project.<\/span>","<span class='gt_from_md'>Continuing work with the single cell mouse data, I pointed them towards scater::plotExpression to plot some of the genes her group is intested in observing by cell type. However we ran in to some bugs with running logNormCounts which we think is from an old version of R. I advised to update R and re-install the packages, hopefull that will fix the issue.<\/span>","<span class='gt_from_md'>We uncovered the source of some unwanted NA values in some genotyping data and eliminated it by removing some code<\/span>","<span class='gt_from_md'>I helped them build a single cell experiment object with expression and metadata from an publication. Hopefully they can use this to independently explore the gene expression the group is interested in.<\/span>","<span class='gt_from_md'>We talked about how to compute KL divergence between 2 groups of cell-type proportions on her data using a variation of my code<\/span>","<span class='gt_from_md'>Reasearcher wanted some background information about the deconvolution process we applied to the degradation data. I explained the mean ratio marker finding, the resulting stats, and how we used them to build our set of marker genes. Then the script for how we ran bisque + the output of bisque .<\/span>","<span class='gt_from_md'>Researcher wants to test if a gene of intrest to their group is associaited with a ceratain cell type. They are working with bulk mouse data. I suggested that deconvolution may not be able to answer the innital question but can provide depth to their bulk analysis. I suggested that they find a mouse single cell dataset that is relevant to their current data set, then first observe their gene of interest their, see if that answers any questions. Then they could use that data to run deconvioltion with bisque<\/span>","<span class='gt_from_md'>I described the manual process for resolving a merge conflict with git when git cannot automatically merge two branches<\/span>","<span class='gt_from_md'>We pinpointed an error message as being due to trying to find the ‘colData’ of a list containing Summarized Experiments rather than calling ‘colData’ on the SummarizedExperiment itself<\/span>","<span class='gt_from_md'>I described how to color ggplot2 plots by a continuous variable, adjust the palette, and other ggplot-related syntax/tricks<\/span>","<span class='gt_from_md'>We discussed how PCA reduces the dimesions of our data and helps show the variance between samples. I helped strategize how to make PCA plots for the habenula data. Start plotting the first 5 PCs pairwise, color by Dx. Then explore key QC or demographic variables, see if we can indentify any patterns. I was unsure if this should repeat this for exon, jx, tx, as we didnt for MDDseq. But figured gene was a good place to start and if the other features should be explored too, the code should be easily adapatabe.<\/span>","<span class='gt_from_md'>I gave them a breif overview of the goals and process of deconvolution, including mean ratio marker finding. I linked my deconvolution slides. We then went over the code for running bisque on the qSVA dataset, including details of the input, refrence, and output data.<\/span>","<span class='gt_from_md'>I recommended some data-preprocessing steps for later downstream manipulation of the data<\/span>","<span class='gt_from_md'>I gave general advice for how to preprocess some data, including changing data types, transposing matrices, and dropping columns to format things for a collaborator<\/span>","<span class='gt_from_md'>I described ‘facet_wrap’ and showed that for some situations, ‘plot_grid’ with a single output PDF (with width and height scaled appropriately) does well<\/span>","<span class='gt_from_md'>We generally discussed linear mixed effects models and some common terms used as fixed effects in genomic data<\/span>","<span class='gt_from_md'>Researcher is trying to run glmPCA on the very large MDD snRNA-seq dataset. They has a run submitted without parallelization that has been runing for 3 days. To try and speed things up she has been trying to use BioCparrallel, via MulticoreParam. But the scheduler keeps killing the job. We checked that -pe local X and MulticoreParam(X) match , so it wasn’t that. I recomended to try to run a subset of the data to get a sense of how glmPCA uses memory. But couldn’t come up with a fix to make the parallelization work. As its PCA it also can’t be run as subsets, so there aren’t other workarounds :( I think BioCparallel + sge scheduling would be a good topic for Rstats club as this seems to be a common headache<\/span>","<span class='gt_from_md'><a href=\"https://lcolladotor.github.io/bioc_team_ds\">https://lcolladotor.github.io/bioc_team_ds<\/a> for my team website, <a href=\"https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing\">https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing<\/a> for the LIBD rstats club schedule, <a href=\"https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials\">https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials<\/a> could be of interest also, <a href=\"https://github.com/LieberInstitute/template_project\">https://github.com/LieberInstitute/template_project<\/a> for a template for organizing projects, learn about the DSgs <a href=\"https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs\">https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs<\/a>, <a href=\"https://rstudio.github.io/reticulate/\">https://rstudio.github.io/reticulate/<\/a> for using Python from R<\/span>","<span class='gt_from_md'>I described how to transpose matrices, change their type, drop columns, and related tricks to solve a data-formatting problem asked by a collaborator<\/span>","<span class='gt_from_md'>Reseacher walked me through some plots they need for the LS paper: marker genes heatmap and expression violin plots. I pointed them to some code from DLPFC sn that should be helpful (they needs access to this repo). They also needs to get setup with JHPCE on their new laptop, I pointed them to the team laptop setup docs. They are also new to ggplot, so I pointed them to the visulization chapeter of R for data scientist. This will probaly require some follow up meetings<\/span>","<span class='gt_from_md'>Researcher has added a new color scheme to the heatmaps that makes the differnces between positive and negative more clear, how ever I think the darkest red being close to 0 is misleading, so they’re going to fix that. I also recomended to simplify the annotaions on the combined heatmap so its easier to find and interpret patterns.<\/span>","<span class='gt_from_md'>We reviewed the reasons or correcting counts and why thats important for DE analysis. We went over the math behind TMM, and then how thoes ideas are used in edgeR. I pointed them to a stats quest video with a more detatiled explination of how the edgeR correction works.<\/span>","<span class='gt_from_md'>Researcher showed me the csv files that he exported from different moduled on HALO. They seem to have the same rows, so copying the needed column in to a differnt csv is easy in excel (where they are working) but I showed that how to check that things are really aligned with some excel functions. We also looked at the data in prism, which I had to quickly learned about. I advised to should show the data in boxplots not as a barplot to better display that data, and went over the checks for running an anova.<\/span>","<span class='gt_from_md'>I mentioned rtracklayer::liftOver and showed a code example of lifting over genomic coordinates for a SummarizedExperiment<\/span>","<span class='gt_from_md'>Researcher showed me their dot plots with the correct 0.05 q val filter, they looked good, ready for discussion. One tweak , is they had a combined up + down expresion col for some plots which was redundant and should be removed. They had also made some heatmaps that didn;t show very much difference between case control, the data probaly needs to be normalized by the genes overall expression, but I need to check prevous work to remeber how to best do that<\/span>","<span class='gt_from_md'>We went throught boxplots Bukola had made for the data, I suggested she add geom_jitter and how to make that work with geom_repel to help show more details about the data. We also discussed direct vs. standard evaluation of variable and how that makes writing funcitons with tidyverse tricky, but how she can use aes_() to do stadard eval with ggplot<\/span>","<span class='gt_from_md'>I walked through our process for buld data QC, which involves making many exploratory plots of QC metrics and know groups (like batch, or case/control) to understand patterns and identify problematics samples. Then later implementing cutoff values either based on stats (like a 3 MAD cut off) or knowlege of ressonable values. They will spend some time plotting QC values, adapting code from smoking mouse, then we will meet again to interpret these plots.<\/span>","<span class='gt_from_md'>We went over the dot plots she made for GO analysis from the multiple DE test she ran. The plots look good, but we identified that they needed to use the default q cutoff to get useful results. She will rerun the GO analysis and replot the results.<\/span>","<span class='gt_from_md'>We adapted my_plotExpression() (a function I wrote) to work for Bukola’s data. This required some debugging as subsetting to just the one gene in question altered how some internal dataframes looked. I also gave Bukola some pointers about saving plots using ggplot.<\/span>","<span class='gt_from_md'>Sang Ho walked me through an analysis they want to do for the AD project, it involves annotating HALO data. We talked thought a strategy, this may be a bit computationally tricky to implement but it seems doable.<\/span>","<span class='gt_from_md'>We talked about <a href=\"https://github.com/LieberInstitute/template_project\">https://github.com/LieberInstitute/template_project<\/a>, <a href=\"https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/\">https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/<\/a>, <a href=\"http://research.libd.org/sgejobs/reference/job_single.html\">http://research.libd.org/sgejobs/reference/job_single.html<\/a>, <a href=\"https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7\">https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7<\/a>, <a href=\"https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1\">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1<\/a>, <a href=\"https://happygitwithr.com/\">https://happygitwithr.com/<\/a>, <a href=\"https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent\">https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent<\/a>.<\/span>","<span class='gt_from_md'>I described the need for SSH keys between the login and compute nodes at JHPCE, and how an incorrect set-up will manifest as a prompt for your password when using ‘qrsh’. I referenced <a href=\"https://jhpce.jhu.edu/knowledge-base/authentication/ssh-key-setup/\">https://jhpce.jhu.edu/knowledge-base/authentication/ssh-key-setup/<\/a><\/span>","<span class='gt_from_md'>We debugged her SSH set-up with JHCPE, finding a syntax mistake, confirming the contents of the ‘.ssh/config’ file, and other details to ultimately get the traditional RStudio + JHPCE set-up working smoothly. We touched a bit on the process for using git from JHPCE but didn’t finish fully setting things up<\/span>","<span class='gt_from_md'>LIBD rstats club schedule <a href=\"https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing\">https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing<\/a>. Bioconductor overview video <a href=\"https://youtu.be/c0Ch_sXiGDQ\">https://youtu.be/c0Ch_sXiGDQ<\/a> and related notes in case you want to skim them <a href=\"https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing\">https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing<\/a> (we typically have this for each video entry). R and RStudio intro <a href=\"https://youtu.be/XTuJ8vGnzBU\">https://youtu.be/XTuJ8vGnzBU<\/a>. Example vignette <a href=\"https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html\">https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html<\/a> from <a href=\"https://bioconductor.org/packages/spatialLIBD/\">https://bioconductor.org/packages/spatialLIBD/<\/a>. Data Science guidance sessions <a href=\"https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0\">https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0<\/a>. JHPCE new user request <a href=\"https://jhpce.jhu.edu/register/user/\">https://jhpce.jhu.edu/register/user/<\/a>. Making your personal website with postcards <a href=\"https://youtu.be/9LUCZAx3SlU\">https://youtu.be/9LUCZAx3SlU<\/a>. Examples include <a href=\"https://lahuuki.github.io/\">https://lahuuki.github.io/<\/a> by Louise, <a href=\"https://jtleek.com/\">https://jtleek.com/<\/a>, <a href=\"http://aejaffe.com/\">http://aejaffe.com/<\/a>.<\/span>","<span class='gt_from_md'>I described how ‘module load conda_R’ lines in a submitted shell script override the default version of R that can be loaded by default using the ‘.bashrc’. In this case this undesired interaction caused a corrupted R package to load and return an error<\/span>","<span class='gt_from_md'>We adapted someone else’s code, slightly changing some awk commands to use cut (which I’m more familiar with) and using a for loop rather than piping a file to a while loop. This allowed the code to work as intended on the data<\/span>","<span class='gt_from_md'>I pointed to installation guidelines for the cluster to resolve issues about R packages not being installed and a problematic nextflow version.<\/span>","NA","<span class='gt_from_md'>Related to how we organize files for a publication.<\/span>","<span class='gt_from_md'>Density is related to the probability density function, see [wiki](<a href=\"https://en.wikipedia.org/wiki/Kernel_density_estimation\">https://en.wikipedia.org/wiki/Kernel_density_estimation<\/a> for) more details. <a href=\"https://stats.stackexchange.com/questions/4220/can-a-probability-distribution-value-exceeding-1-be-ok\">This link<\/a> is also useful for understanding the y-axis in more detail. Violin plots are just density plots rotated 90 degrees and mirrored. They are compared against boxplots frequently, which while easier to understand, are not good for non-unimodal data. See <a href=\"https://mode.com/blog/violin-plot-examples/#:~:text=What%20is%20a%20violin%20plot%3F,the%20density%20of%20each%20variable\">this link<\/a> for more. I like <code>plotExplanatoryVariables()<\/code> because it’s fast to run and gives a general idea of things. See <a href=\"https://bioconductor.org/packages/release/bioc/vignettes/variancePartition/inst/doc/variancePartition.pdf\">Figure 1b<\/a> from <code>variancePartition<\/code> for a violin plot of similar data. <code>variancePartition<\/code> can handle mixed effects models (random effects) and in that scenario is more precise, but will also take longer to compute.<\/span>","<span class='gt_from_md'>I described how to smoothly ‘git pull’ when updates have been made to both the local and remote repo (adding + committing local changes, running ‘git pull’, and handling a smooth merge)<\/span>","<span class='gt_from_md'>I described proper saving and loading of SummarizedExperiment derivatives whose assays are HDF5-backed using the HDF5Array package, and how to convert between HDF5-backed objects and “ordinary” in-memory objects<\/span>","<span class='gt_from_md'>I went over the DE pipeline we use, noting that the model development is the most demanding and non-static part. We talked specifics for their dataset which has some experimental design to consider as its organids and is diffrent from patient case control. I posinted Geo to scripts where we do QC, expression cutoff, and my run_DE script<\/span>","<span class='gt_from_md'>Heena saw old commits pop up on slacks git notifications when she pushed commits she had locally on her laptop, but they didn’t reappear in the git history. I think she did everything right and there might be something strange with the behavior of the slack message (as thats the only place things seemd wrong). We did some detective work between the files and the commits, everything seems good. Heena was able to then pull to the cluster with out conflicts<\/span>","<span class='gt_from_md'>Had to work over slack again as Daianna was having trouble with zoom. Working with the feedback from the previous meetings, the experimetn has been narrowed down to 5 DE runs, and then comparing some pairs of the outputs. We were unsure if we should include mitoRate as a variable in the adult model as it correlates highly with totalAssignedGene, but decided to include it for now to match the model used in the pup DEA. I also sent her some code I have used to make scatter plots of t-stats to help compare DE results.<\/span>","<span class='gt_from_md'>Check <a href=\"https://bioconductor.org/packages/devel/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html\">EnchancedVolcano<\/a> which shows a quick DE guide and finishes with some Volcano plots, which can be good enough for a grant application. If you want to use limma, you could follow <a href=\"http://research.libd.org/SPEAQeasy-example/de_analysis_speaqeasy.html#16_Modeling\">the SPEAQeasy example<\/a> or <a href=\"http://research.libd.org/SPEAQeasy-example/bootcamp_intro#5_Day_3\">our DE bootcamp<\/a> a bit. Or dive deeper into <a href=\"http://bioconductor.org/packages/release/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html\">limma’s documentation<\/a>.<\/span>","<span class='gt_from_md'>We intended to meet on zoom, but Daianna’s connection was bad so we used Slack to chat. Daianna has made many plots and visuals about the many DE analysis for smoking mouse. I gave her some feedback about orgaizing her slides so the figures can practically be used inform DE models, right now everything is a bit scattered as shes planning on running 7 analysis so there is a lot to consider. We will need to meet again to finish this.<\/span>","<span class='gt_from_md'>wrong input was given to the function<\/span>","NA","<span class='gt_from_md'>should select virtual stack option in bioformats reader when opening and multichannel image in imagej<\/span>","NA","<span class='gt_from_md'>Daianna showed me her varianceParition plots, and we talked bout how to design the DE anlaysis based on what she’s oberved in the mouse data. I advised to keep things simpler (ie not create too many groups) but rather use models to control for vaiables adding variance. We will meet next to talk model design for this work.<\/span>","<span class='gt_from_md'>Explain how to pick randomn images to train PNN segmentation algorithm<\/span>","<span class='gt_from_md'>We walked through the code to make a pseudobulked object and plotting the explanatory variables.<\/span>","<span class='gt_from_md'>Note that <code>spe<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.jsdelivr.net/npm/katex@0.16.9/dist/katex.min.css\" data-external=\"1\">\n<span class=\"katex\"><span class=\"katex-mathml\"><math xmlns=\"http://www.w3.org/1998/Math/MathML\"><semantics><mrow><mi>v<\/mi><mi>a<\/mi><msub><mi>r<\/mi><mi>c<\/mi><\/msub><mi>o<\/mi><mi>l<\/mi><mi>o<\/mi><mi>r<\/mi><mi>s<\/mi><mi mathvariant=\"normal\">‘<\/mi><mi>w<\/mi><mi>o<\/mi><mi>r<\/mi><mi>k<\/mi><mi>s<\/mi><mi>f<\/mi><mi>o<\/mi><mi>r<\/mi><mi mathvariant=\"normal\">‘<\/mi><mi>s<\/mi><mi>p<\/mi><mi>e<\/mi><\/mrow><annotation encoding=\"application/x-tex\">var_colors` works for `spe<\/annotation><\/semantics><\/math><\/span><span class=\"katex-html\" aria-hidden=\"true\"><span class=\"base\"><span class=\"strut\" style=\"height:0.8889em;vertical-align:-0.1944em;\"><\/span><span class=\"mord mathnormal\" style=\"margin-right:0.03588em;\">v<\/span><span class=\"mord mathnormal\">a<\/span><span class=\"mord\"><span class=\"mord mathnormal\" style=\"margin-right:0.02778em;\">r<\/span><span class=\"msupsub\"><span class=\"vlist-t vlist-t2\"><span class=\"vlist-r\"><span class=\"vlist\" style=\"height:0.1514em;\"><span style=\"top:-2.55em;margin-left:-0.0278em;margin-right:0.05em;\"><span class=\"pstrut\" style=\"height:2.7em;\"><\/span><span class=\"sizing reset-size6 size3 mtight\"><span class=\"mord mathnormal mtight\">c<\/span><\/span><\/span><\/span><span class=\"vlist-s\">​<\/span><\/span><span class=\"vlist-r\"><span class=\"vlist\" style=\"height:0.15em;\"><span><\/span><\/span><\/span><\/span><\/span><\/span><span class=\"mord mathnormal\">o<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.01968em;\">l<\/span><span class=\"mord mathnormal\">ors<\/span><span class=\"mord\">‘<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.02691em;\">w<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.02778em;\">or<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.03148em;\">k<\/span><span class=\"mord mathnormal\">s<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.10764em;\">f<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.02778em;\">or<\/span><span class=\"mord\">‘<\/span><span class=\"mord mathnormal\">s<\/span><span class=\"mord mathnormal\">p<\/span><span class=\"mord mathnormal\">e<\/span><\/span><\/span><\/span>var<\/code> where the <code>var<\/code> part has to be the same in both variable names. Also <code>spe<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.jsdelivr.net/npm/katex@0.16.9/dist/katex.min.css\" data-external=\"1\">\n<span class=\"katex\"><span class=\"katex-mathml\"><math xmlns=\"http://www.w3.org/1998/Math/MathML\"><semantics><mrow><mi>v<\/mi><mi>a<\/mi><msub><mi>r<\/mi><mi>c<\/mi><\/msub><mi>o<\/mi><mi>l<\/mi><mi>o<\/mi><mi>r<\/mi><mi>s<\/mi><mi mathvariant=\"normal\">‘<\/mi><mi>s<\/mi><mi>h<\/mi><mi>o<\/mi><mi>u<\/mi><mi>l<\/mi><mi>d<\/mi><mi>b<\/mi><mi>e<\/mi><mi>a<\/mi><mi>n<\/mi><mi>a<\/mi><mi>m<\/mi><mi>e<\/mi><mi>d<\/mi><mi>v<\/mi><mi>e<\/mi><mi>c<\/mi><mi>t<\/mi><mi>o<\/mi><mi>r<\/mi><mi>s<\/mi><mi>u<\/mi><mi>c<\/mi><mi>h<\/mi><mi>t<\/mi><mi>h<\/mi><mi>a<\/mi><mi>t<\/mi><mi mathvariant=\"normal\">‘<\/mi><mi>u<\/mi><mi>n<\/mi><mi>i<\/mi><mi>q<\/mi><mi>u<\/mi><mi>e<\/mi><mo stretchy=\"false\">(<\/mo><mi>n<\/mi><mi>a<\/mi><mi>m<\/mi><mi>e<\/mi><mi>s<\/mi><mo stretchy=\"false\">(<\/mo><mi>s<\/mi><mi>p<\/mi><mi>e<\/mi><\/mrow><annotation encoding=\"application/x-tex\">var_colors` should be a named vector such that `unique(names(spe<\/annotation><\/semantics><\/math><\/span><span class=\"katex-html\" aria-hidden=\"true\"><span class=\"base\"><span class=\"strut\" style=\"height:1em;vertical-align:-0.25em;\"><\/span><span class=\"mord mathnormal\" style=\"margin-right:0.03588em;\">v<\/span><span class=\"mord mathnormal\">a<\/span><span class=\"mord\"><span class=\"mord mathnormal\" style=\"margin-right:0.02778em;\">r<\/span><span class=\"msupsub\"><span class=\"vlist-t vlist-t2\"><span class=\"vlist-r\"><span class=\"vlist\" style=\"height:0.1514em;\"><span style=\"top:-2.55em;margin-left:-0.0278em;margin-right:0.05em;\"><span class=\"pstrut\" style=\"height:2.7em;\"><\/span><span class=\"sizing reset-size6 size3 mtight\"><span class=\"mord mathnormal mtight\">c<\/span><\/span><\/span><\/span><span class=\"vlist-s\">​<\/span><\/span><span class=\"vlist-r\"><span class=\"vlist\" style=\"height:0.15em;\"><span><\/span><\/span><\/span><\/span><\/span><\/span><span class=\"mord mathnormal\">o<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.01968em;\">l<\/span><span class=\"mord mathnormal\">ors<\/span><span class=\"mord\">‘<\/span><span class=\"mord mathnormal\">s<\/span><span class=\"mord mathnormal\">h<\/span><span class=\"mord mathnormal\">o<\/span><span class=\"mord mathnormal\">u<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.01968em;\">l<\/span><span class=\"mord mathnormal\">d<\/span><span class=\"mord mathnormal\">b<\/span><span class=\"mord mathnormal\">e<\/span><span class=\"mord mathnormal\">anam<\/span><span class=\"mord mathnormal\">e<\/span><span class=\"mord mathnormal\">d<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.03588em;\">v<\/span><span class=\"mord mathnormal\">ec<\/span><span class=\"mord mathnormal\">t<\/span><span class=\"mord mathnormal\">ors<\/span><span class=\"mord mathnormal\">u<\/span><span class=\"mord mathnormal\">c<\/span><span class=\"mord mathnormal\">h<\/span><span class=\"mord mathnormal\">t<\/span><span class=\"mord mathnormal\">ha<\/span><span class=\"mord mathnormal\">t<\/span><span class=\"mord\">‘<\/span><span class=\"mord mathnormal\">u<\/span><span class=\"mord mathnormal\">ni<\/span><span class=\"mord mathnormal\" style=\"margin-right:0.03588em;\">q<\/span><span class=\"mord mathnormal\">u<\/span><span class=\"mord mathnormal\">e<\/span><span class=\"mopen\">(<\/span><span class=\"mord mathnormal\">nam<\/span><span class=\"mord mathnormal\">es<\/span><span class=\"mopen\">(<\/span><span class=\"mord mathnormal\">s<\/span><span class=\"mord mathnormal\">p<\/span><span class=\"mord mathnormal\">e<\/span><\/span><\/span><\/span>var_colors))<\/code> matches <code>unique(spe$var)<\/code>.<\/span>","<span class='gt_from_md'>code loops over the different file in the dataset and Marah was clearing it at the end of the loop every time which didnt let her save any data<\/span>","<span class='gt_from_md'>We worked through a few issues in the cellpose module, which each had a different cause. In one case, the version of torch/CUDA was outdated for the hardware. In another case, user-specific libraries were being referenced instead of the central module’s libraries.<\/span>","<span class='gt_from_md'>Daianna shared her updated QC plots, adn the one sample she planned to drop based on fastCellQC. I advised that she dig deeper in to how that QC finstion works, and what the cutoff values are, as well as exmaming mito vs. ribo precentages, and a two samples that don’t seem to group with others of the same phenotype /<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>The error messages involved a mismatch between sample ID and file name for the ‘QualityTrimmed’ process, as well as a supposed missing ID in the manifest (which was actually present). Simply resuming the pipeline got rid of both errors.<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>I explained how to get pheatmap to use the same color scheme for column and row annotations by making the annotaion tables have the same colname, and suppying a color pallet<\/span>","<span class='gt_from_md'>We discussed a strategy to subset the 102 loci in question, and adapted code to subset the FDR signif pairs to work with the location info.<\/span>","<span class='gt_from_md'>I described how to configure SPEAQeasy for a particular dataset, and how to manage two separate runs of SPEAQeasy as part of a different git repository (requiring the use of submodules)<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>Daianna shared the updates she made to the QC plots based on our previous discussion, specifically separating brain and blood samples.<\/span>","<span class='gt_from_md'>I showed the documentation’s description of ‘mitoRate’ and linked to the code where it’s computed: <a href=\"https://github.com/LieberInstitute/SPEAQeasy/blob/bfe4e664281b52c5e239e372393a8aea36084e0d/scripts/create_count_objects.R#L521-L533\">https://github.com/LieberInstitute/SPEAQeasy/blob/bfe4e664281b52c5e239e372393a8aea36084e0d/scripts/create_count_objects.R#L521-L533<\/a><\/span>","<span class='gt_from_md'>She was getting a memory error when running BayesSpace and I ended up advising her to reduce the nrep argument. Also advised her on running fasthplus because it was taking too long. She needed to reduce the proportion of the data being used.<\/span>","<span class='gt_from_md'>I described the need to manually convert some spatialExperiment slots prior to conversion with zellkonverter, and that some data types are not supported. I pointed to my SpaGCN code and the lines specifically responsible for clustering + writing outputs<\/span>","<span class='gt_from_md'>Daianna walked me through her progress in the DE analysis. We went over QC plots and discussed experimental design. She will check in with Kristen about how to organize the brain vs. blood samples. Currently she dropped the blood sampeles as they looked very different in several QC metrics. But I advised that they just be handled seperatly as it makes senese they look different as they are different tissue, and the signal is probaly biological rather than technical<\/span>","<span class='gt_from_md'>I described when ‘git pull’ can be “safely” used, and when ‘git reset’ might have to be used beforehand<\/span>","NA","<span class='gt_from_md'>I described the use of ‘remotes::install_github’ for GitHub-based packages. We encountered a GitHub authentication error that was solved by running the R code ‘Sys.unsetenv(“GITHUB_PAT”)’ prior to the normal installation command.<\/span>","<span class='gt_from_md'>she needed to have the plots as single ggplots instead of a list so I advised her to used vis_clus() instead. Shared code with her that I had used to make some of supplemental figures. She also asked if she need to discard spots before doing batch correction and I told her yes.<\/span>","<span class='gt_from_md'>We fixed Sanjana’s ssh set up and got the qrsh command working. We got git configurated for her locally and on JHPCE, as well as set up tokens. Created a test repo, and made commits from her laptop and JHPCE as an example<\/span>","<span class='gt_from_md'>checked size of the spe object. checked that only has low res image files. check dim of colData. All sizes checkedout. Believe this to be a package problem.<\/span>","<span class='gt_from_md'>Maddy asked me how to get the total UMI’s for each sample out of her spe object. I wrote a small for loop for her in order to do it.<\/span>","<span class='gt_from_md'>We went over github basics and motivation behind version control. Quickly discovered that Sanjana’s qrsh command is not working, requiring a password. Ended up using the session to troubleshoot that, with no success :( planning on circling back/re-setting up ssh for her. Will meet later about git setup<\/span>","<span class='gt_from_md'>We checked Sreya’s code against the OSCA DE example run. She was worried that there weren’t may significant genes, but her results seemed to be in the same range (10s of genes) as the example. We discussed how maybe there were QC or demographic variables that could have been added to the model, but the data she has doesn’t have much to work with. We will revisit when I have maybe done more work with sn DE analysis.<\/span>","NA","<span class='gt_from_md'>Talked about recount2, recount3, snaptron, snapcount, recount.bwtool, recount-brain, recount predictions, megadepth, the recount3 study explorer. I also sent them all the links to them, though you can find many of them at <a href=\"https://jhubiostatistics.shinyapps.io/recount/\">https://jhubiostatistics.shinyapps.io/recount/<\/a>, <a href=\"https://jhubiostatistics.shinyapps.io/recount-brain/\">https://jhubiostatistics.shinyapps.io/recount-brain/<\/a>, and <a href=\"http://rna.recount.bio/\">http://rna.recount.bio/<\/a>.<\/span>","<span class='gt_from_md'>I pointed Nick to a few scripts used in marker finding for the Tran Maynard dataset, using a few functions from DeconvoBuddies. For marker heat maps we went over how I hace built annotation tables, and a few tricks for optimizing pheatmaps for maker plots.<\/span>","<span class='gt_from_md'>We converted a downloaded dataset from tsv files to a sce object. Went over a strategy for performing DE on single cell data, using OSCA as a reference<\/span>","<span class='gt_from_md'>We spent the session getting Sanjana set up better to work on JHPCE, setting up ssh keys, cyberduck, RStudio, and config files. Plan to meet again to set up git + start single cell work<\/span>","<span class='gt_from_md'>This had already been fixed, but Abby had an older version of spatialLIBD installed. We also went over some other steps needed to deploy an <code>spatialLIBD<\/code> app with pseudo-bulked data and modeling results.<\/span>","<span class='gt_from_md'>Resulted in a fix for <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1137544893\">this issue<\/a>.<\/span>","<span class='gt_from_md'>Assessed results of BayesSpace joint clustering and suggested that Alice discard the spots with low library sizes to improve clustering. Offered to create small SPE object for her with just the for samples she will be doing long read sequencing on.<\/span>","<span class='gt_from_md'>Brainstorming and planning session.<\/span>","<span class='gt_from_md'>Resolved via email.<\/span>","<span class='gt_from_md'>created new script to subset spe object by removing some of the images.  updated app.R to load smaller spe object (half the size) and re-ran shiny app locally. Explained how to version control from local computer and fixed problem with version controlling large data file.  Still having same problem of not being able to view shiny app online<\/span>","<span class='gt_from_md'>Review of statistical models as well as access to data<\/span>","<span class='gt_from_md'>We both leaned how to use the image function to create a annotation bar for heatmaps based on a custom color pallet<\/span>","<span class='gt_from_md'>Twitter, LinkedIn, jobRxiv<\/span>","<span class='gt_from_md'>Tried to troubleshoot adding visium IF images to spe object<\/span>","<span class='gt_from_md'>Helped setup OneDrive folder for LC manuscript. Explained how to organize figure folders.<\/span>","<span class='gt_from_md'>Clarified what the data is showing and explained a bit the goals of the LIBD data portal that Geo Pertea is building.<\/span>","NA","<span class='gt_from_md'>Helped fix error of shiny app not being able to find spe object. Helped sucessfully deploy app.<\/span>","<span class='gt_from_md'>We went over how to create a new type of refrence (Supp Table/Supp Figure). Also walked though the full process of creating a label and then refrencing it in from the text + considreations for formatting<\/span>","<span class='gt_from_md'>PhD programs and other options.<\/span>","<span class='gt_from_md'>We went thourgh some plotting code Sreya had adapted to make a manhattan plot, and went over which aestetics to change to get the desired plot<\/span>","<span class='gt_from_md'>I described the two JHPCE-based repositories we push to (<a href=\"https://github.com/LieberInstitute/jhpce_mod_source\">https://github.com/LieberInstitute/jhpce_mod_source<\/a> and <a href=\"https://github.com/LieberInstitute/jhpce_module_config\">https://github.com/LieberInstitute/jhpce_module_config<\/a>), my usual installation approach for python-based GPU-utilizing software, and the contents of the lua-based config enabling a module to load.<\/span>","<span class='gt_from_md'>See <a href=\"https://twitter.com/lahuuki/status/1521581339903873024\">this Twitter thread<\/a> for the result.<\/span>","<span class='gt_from_md'>Provided guidance on how to share images and how to organize supplementary files. Resulting in this <a href=\"https://doi.org/10.6084/m9.figshare.19729567.v1\">Figshare<\/a> zip file.<\/span>","<span class='gt_from_md'>I clarified the usage of nextflow channels, described when docker integration is likely to be worthwhile, and answered other general questions about processing data with a nextflow-based workflow<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>I helped come up with a stratagedy to loop over 100+ files for an analysis and exreact a portion of the results in to one R object<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>Helped explain the different parts of the html output of spaceranger and which fields are important to check for QC<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>NA<\/span>","<span class='gt_from_md'>See <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012\">https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012<\/a>.<\/span>","<span class='gt_from_md'>Clarified a few questions about LIBD’s SPEAQeasy RSE objects and a problem with some colData() that was missing, that we’ll provide early next week. Send links to our deconvolution work with Louise A Huuki-Myers.<\/span>","<span class='gt_from_md'>If the situation had been simple, that is, that the latest commit was the one that commited the large files, we could have used the solution described at <a href=\"http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/\">http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/<\/a>. Instead, we had to use <a href=\"https://rtyley.github.io/bfg-repo-cleaner/\">https://rtyley.github.io/bfg-repo-cleaner/<\/a>.<\/span>","<span class='gt_from_md'>I helped Heena address an error she was getting when trying to run read10XVisiumWrapper(). We noted an issue with the function and submitted an github issue to spatialLIBD. We got the function to work but only if you provide the path to the gtf file. Met for 1hr.<\/span>","<span class='gt_from_md'>JHPCE ssh keys (to avoid typing passwords when using qrsh), .Rprofile (both), .bashrc, .zshrc (laptop), iTerm2 (laptop), Textmate (laptop), remote port fowading for rmate, xquartz (laptop) installation and configuration. There are several LIBD rstats club videos on these topics already, but we just did all the setup fast. In particular, the .bashrc configuration was important to resolve a umask setting problem that was interfering with the permissions on the Visium IF AD project.<\/span>","<span class='gt_from_md'>We installed and configured Cross Refrence, then went over how to use the lable vs. refrence links + pro tips on how to avoid errors<\/span>","<span class='gt_from_md'>spot size and scaling factor from json file. explained how each metric is associated to different version of the histology images.<\/span>","<span class='gt_from_md'>Customized VistoSeg code to their histology images<\/span>","NA","<span class='gt_from_md'>See <a href=\"https://neuralplasticity.slack.com/archives/C010NFSQS48/p1649249130880349\">this Slack thread<\/a>. Ultimately, this situation involved teaching Tony to share his code.<\/span>","<span class='gt_from_md'>We want over some plotting basics with ggplot2, and how to use geom_rect to recreate some spatial plots that a halo module produces in R.<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/VistoSeg/issues/11\">https://github.com/LieberInstitute/VistoSeg/issues/11<\/a><\/span>","<span class='gt_from_md'>Able to identify situation where duplicate.correlation should be used and how block data is important to linear modeling. We then looked at how top table can be used to summarize DE outputs.<\/span>","<span class='gt_from_md'>modified code to work witth her dataset. did graph based clustering and plottiing. Explained benefits of clustering across samples. Advised her to do isoform calling on pseudobulked data for multiple samples. Met for 1hr.<\/span>","<span class='gt_from_md'>After identifying that ‘redstone’ was a precompiled binary for Linux, I described how to use ‘wget’ to download software like this, set execute permissions if necessary, and run at JHPCE.<\/span>","<span class='gt_from_md'>Finished installing packages. Set up sample_info table. Walked her through the rest of the code.<\/span>","<span class='gt_from_md'>I recommended some adjustments to a custom GTF and transcripts FASTA to solve an error seen in SPEAQeasy where transcripts and annotation were found to be incompatible.<\/span>","<span class='gt_from_md'>We applied some test I used for the TREG package to help test more parts of the functions, like checking for errors w/ bad input, warning messages, and qualities of outputs. We discussed possible ways to use testthat on a ggplot, but that might need more thought.<\/span>","<span class='gt_from_md'>Showed her app.R, deploy.R scripts and places where to edit, showed her how to navigate through the shinyapp portal to looks for logs when deploying the app. EXplained her few erros she ran through when deploying for the 1st time.<\/span>","<span class='gt_from_md'>We set up a git repo and added code to be adapted to build an spe object from Alice’s long read data.<\/span>","<span class='gt_from_md'>I explained how the first time that you ssh into a new computer (which happens also with git pull/git push), you need to accept that the new host is authentic, which will then edit your ~/.ssh/known_hosts file.<\/span>","<span class='gt_from_md'>This was a preliminary meeting with Alice to decide where to begin with analyzing her spatial data. We decided to start by generating an spe object with her long read data.<\/span>","<span class='gt_from_md'>An error occurred in a SPEAQeasy run, which I identified to be due to not providing all the required inputs (specifically the sites at which to perform variant calling)<\/span>","<span class='gt_from_md'>This is challenging since it involves putting together code from multiple scripts/projects. We summarized the steps to do <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1069493294\">on this GitHub issue thread<\/a>.<\/span>","<span class='gt_from_md'>We both verified additional transcript annotation added to a GENCODE GTF looked good for use as input to SPEAQeasy<\/span>","<span class='gt_from_md'>Louise and I discussed the <a href=\"https://doi.org/10.1038/s41593-020-0604-z\">Dentate Gyrus, Jaffe at al, 2020<\/a> paper which used LCM-seq, what deconvolution using Bisque can do, and what role it can play at different analytical stages.<\/span>","<span class='gt_from_md'>TODO<\/span>","<span class='gt_from_md'>An error occurred in the counts step, which I discovered was a bug in some filename-matching code<\/span>","<span class='gt_from_md'>We had some trouble logging in because the WiFi wasn’t working without logging in. We solved this by using an ethernet cable at LIBD (which is part of the JHU network). Once we logged in once, then we could use the WiFi.<\/span>","<span class='gt_from_md'>I described two-factor authentication vs. passwordless login via SSH keys, and gave tips to set up the latter.<\/span>","<span class='gt_from_md'>The contrast adjusted images on MAC::preview should be saved and uploaded back to JHPCE to run spaceranger. Spaceranger needs the exact same image that was used in the loupe alignment (even if it is changed from raw image on preview)<\/span>","<span class='gt_from_md'>I described how to add additional transcripts to the relevant FASTA file then use SPEAQeasy’s custom annotation option to use the new FASTA with the other default annotation files.<\/span>","<span class='gt_from_md'>Explained how the image data can be manipulated when claudin5 channel and WFA channel are merged and then the blood vessels are segmented and masked. The correct way is to segment blood vessels from claudin channel and mask the segmented objects in WFA channel.<\/span>","<span class='gt_from_md'>The indices listed in the tiles’s filename is the x,y coordinates of the starting topleft pixel of the given tile. These indices never start with [0,0] as these tiles are picked from the ROI in the phenochart and not the entire imaged area. Once stored in a directory these tiles are not arranged in chronological order.<\/span>","<span class='gt_from_md'>I described how requesting ‘caracol’ is similar to requesting ‘bluejay’, but elaborated on additional complications, like contention for the same GPU and using CUDA_VISIBLE_DEVICES properly, including with array jobs<\/span>","<span class='gt_from_md'>I described different strateguies for helping others navigate the world of data science. In particular, how weekly coding sessions can be really useful.<\/span>","<span class='gt_from_md'>We went over the files (data and script) for the deconvolution analysis done for the habenula in BSPIII. We also discussed the theroy/methodology for deconvolution.<\/span>","<span class='gt_from_md'>Created an R script that reads in the transcripts FASTA file from GENCODE and uses BSgenome from Bioconductor to edit it and save it as a larger .fasta.gz file. Nick Eagles helped us by pointing us to <a href=\"http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation\">http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation<\/a>.<\/span>","<span class='gt_from_md'>Linked to <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13<\/a>, <code>/dcs04/lieber/lcolladotor/qSVA_LIBD3080/degradation_experiments/*/bed<\/code>, <a href=\"https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855\">https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855<\/a>, <a href=\"https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh\">https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5<\/a>, <a href=\"https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R\">https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R<\/a>, <a href=\"https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R\">https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R<\/a>, <a href=\"https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253\">https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253<\/a>, <a href=\"https://github.com/CRG-Barcelona/bwtool/wiki#installation\">https://github.com/CRG-Barcelona/bwtool/wiki#installation<\/a>, <a href=\"http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html\">http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html<\/a>, and <a href=\"http://research.libd.org/qsvaR/\">http://research.libd.org/qsvaR/<\/a>. Andrew Jaffe linked <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/degradation_mat_CMC_ribozero.rda<\/code>, <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/run_de_analysis_CMC.R<\/code>, <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/geneLevel_CMC_qSVsAndMod.rda<\/code> and <code>/users/ajaffe/Lieber/Projects/RNAseq/CMC/<\/code>.<\/span>","<span class='gt_from_md'>Linked to <a href=\"https://github.com/LieberInstitute/qsvaR\">https://github.com/LieberInstitute/qsvaR<\/a>, <a href=\"http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0\">http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28<\/a>. Made public the file <code>/dcl01/ajaffe/data/lab/qsva_brain/brainseq_phase2_qsv/rdas/degradation_rse_phase2_usingJoint_justFirst.rda<\/code> and other ones. See <a href=\"https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5\">https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5<\/a> for details.<\/span>","<span class='gt_from_md'>Heena was encountering a simple error installing the package. We also spent time going through the vignette to give a conceptual overview of qsva. Her feedback will be helpful in making the vignette more accessible<\/span>","<span class='gt_from_md'>I referred her to the ‘–coverage’ option with SPEAQeasy, which unfortunately cannot be run alone, which might be useful for an already-processed dataset<\/span>","<span class='gt_from_md'>Used KJ’s code from his rstats club presentation and described how the images shoukd be read using numpy array as uint32 but not floating <a href=\"https://colab.research.google.com/drive/1A4cavWcbgZF1PFQT87s5s_ATyZrN3IxL?usp=sharing\">https://colab.research.google.com/drive/1A4cavWcbgZF1PFQT87s5s_ATyZrN3IxL?usp=sharing<\/a><\/span>","<span class='gt_from_md'>Using BiocMAP <a href=\"http://research.libd.org/BiocMAP/\">http://research.libd.org/BiocMAP/<\/a> which Nick is finishing up, you could process the data and have R objects you could load and analyze. So, the data processing would be “easy” in a sense. Next, you might want to check the bsseq documentation <a href=\"https://bioconductor.org/packages/bsseq/\">https://bioconductor.org/packages/bsseq/<\/a> if you are interested in finding differentially methylated regions (DMRs) and in any case, because BiocMAP provides that type of R object as noted at <a href=\"http://research.libd.org/BiocMAP/outputs.html\">http://research.libd.org/BiocMAP/outputs.html<\/a>. <a href=\"https://jhubiostatistics.shinyapps.io/wgbsExprs/\">https://jhubiostatistics.shinyapps.io/wgbsExprs/<\/a> and the companion paper is the last WGBS data analysis project I did myself with code at <a href=\"https://github.com/LieberInstitute/brain-epigenomics\">https://github.com/LieberInstitute/brain-epigenomics<\/a>. Kira Perzel-Mandell, who worked with Shizhong, wrote <a href=\"https://github.com/LieberInstitute/wgbs_meqtl_analysis\">https://github.com/LieberInstitute/wgbs_meqtl_analysis<\/a> (link to paper there) and did other projects. If you are plotting methylation levels at some locations, then that’s very direct. If you are trying to find DMRs, it can take a bit of time to get used to the large objects and the related complications (once you filter the data, it’ll be more manageable). If you go into CpH instead of CpG, then memory constraints become harder to deal with.<\/span>","<span class='gt_from_md'>We discussed how to build and save data with a for loop and with purrr::map. Went over how to define a function in map to do nested loops<\/span>","<span class='gt_from_md'>Geo was interested int eh workflow I developed for tensorQTL. I went over the requirements and input formats that tensorqtl requirers, and how I had previously made them from our dataset. Also discussed the differnces between the functions and their outputs.<\/span>","<span class='gt_from_md'>used HALO to show how contrast can be adjusted when changing min max values in the image<\/span>","<span class='gt_from_md'>NA<\/span>","NA","<span class='gt_from_md'>Related to JHPCE <a href=\"https://github.com/ntdyjack/fasthplus\">https://github.com/ntdyjack/fasthplus<\/a> and <a href=\"http://research.libd.org/sgejobs/\">http://research.libd.org/sgejobs/<\/a><\/span>","NA","<span class='gt_from_md'>Brainstormed several ideas related to deconvolution of bulk data as well as uses of snRNA-seq. Refered to OSCA <a href=\"https://bioconductor.org/books/release/OSCA/\">https://bioconductor.org/books/release/OSCA/<\/a> at time.<\/span>","<span class='gt_from_md'>We went over how to acess the rowData, and what the rowData inthe MDD project contains. Ended up adding columns to the eqtl risk snp subsets to help with analysis<\/span>","NA","<span class='gt_from_md'><a href=\"https://www.nature.com/articles/s41592-021-01358-2\">https://www.nature.com/articles/s41592-021-01358-2<\/a><\/span>","NA","NA","NA","<span class='gt_from_md'>We tried to discover the source of a seg fault that would occur when she imported tensorqtl in python with the tensorqtl module. Another user and I weren’t able to replicate the issue. We later discovered that there were lines in her .bashrc that might’ve interfered with the module to produce the issue.<\/span>","<span class='gt_from_md'>I described how to qrsh/sub jobs with the GPU node, including how to automatically set up ‘CUDA_VISIBLE_DEVICES’ to avoid contention for resources, and how to use the tensorqtl module to ensure GPUs were used when importing pytorch<\/span>","<span class='gt_from_md'>After ensuring our apparent Rstudio set-ups on our windows machines matched, she still wasn’t able to forward X11 while using Rstudio. Regarding a different issue, we discovered that a particular machine required additional lines in the ‘~/.ssh/config’ to properly enable RSA keys.<\/span>","<span class='gt_from_md'>explained the functions of this pipline as these are not yet documented<\/span>","<span class='gt_from_md'>I described how it’s accessed as a queue, but some manual work is left to the user (checking which GPUs are open and setting ‘CUDA_VISIBLE_DEVICES’)<\/span>","<span class='gt_from_md'>command to use <code>rsync -avzh --progress userid@jhpce-transfer01:/source directory/ /destination directory/<\/code><\/span>","<span class='gt_from_md'>use rsync, prepend spaces in the paths with <code>\\<\/code> to avoid issues<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/43106c664883fd5df301e44cca35f2e8f2f289ad\">https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/43106c664883fd5df301e44cca35f2e8f2f289ad<\/a><\/span>","<span class='gt_from_md'>explain how to use IDE’s on local machine to edits scripts on JHPCE through cyberduck<\/span>","<span class='gt_from_md'>Fixed row names to both be ensembl IDs, discussed simplifing cell types in sce, converting sce arrarys to matrix<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/CaPTure/issues/2\">https://github.com/LieberInstitute/CaPTure/issues/2<\/a><\/span>","<span class='gt_from_md'>I shared some detials of our deconvolution approach, we walked through the SCE to Expression set converson, and discussed some issues that may come up<\/span>","<span class='gt_from_md'>I described how I normally deduce sample IDs from FASTQ filenames when creating a ‘samples.manifest’ file for SPEAQeasy, described the difference between ‘–trim_mode’ settings that SPEAQeasy supports, and showed how to check what samples were trimmed.<\/span>","<span class='gt_from_md'>refining segmentations in matlab by applying size thresholds<\/span>","<span class='gt_from_md'>Showed code how to scale data based on <a href=\"https://uc-r.github.io/kmeans_clustering\">https://uc-r.github.io/kmeans_clustering<\/a> and optimise kmeans for the present dataset<\/span>","<span class='gt_from_md'>Motivated /dcs04/lieber/lcolladotor/_jhpce_org_LIBD001/update_permissions_spatialteam.sh<\/span>","<span class='gt_from_md'>Showed the steps on one sample HPC image and what outputs to save.<\/span>","<span class='gt_from_md'>While debugging, its seen that VistoSeg when splitting the raw image missed a part of fiducial frame for capture area D1 of that sample. This happens when the center of the visium slide does not align with the center of the image, which has ti be adjusted while imaging. This image when run through spaceranger provided negative coordinates for the missing fiducial frame part of the image, but MATLAB will not accept negative indices for a image, which was throwing errors when recontructing spot grind in countSpots function.<\/span>","<span class='gt_from_md'>Showed how to close issues on git with detailed comments of the final results<\/span>","<span class='gt_from_md'>I described using regular expressions in R to match files in a directory. We encountered a weird case where ‘list.files’ seemed to return different numbers of matched files when run repeatedly. After ensuring we were in the right directory, the issue went away, but it was unclear what caused the original problem.<\/span>","<span class='gt_from_md'>made custom Matlab script to batch process her 30 RNAscope images on her local machine<\/span>","<span class='gt_from_md'>The user directories were cleared during the software updates for LIBD server, emailed bill to create one for me and Heena to use rsync on Neural_Plasiticity. The filenames Heena used for ‘source’ in rync did not exist at the source directory which was throwing errors.<\/span>","<span class='gt_from_md'>to pick filename from textfile for a array job<\/span>","<span class='gt_from_md'>We discussed how to check for duplicate rownames, and stratedgies for variable naming + working with rse objects<\/span>","<span class='gt_from_md'>We found a R function that helped produce the plot she was looking for<\/span>","<span class='gt_from_md'>Two main issues were discussed: <strong>1<\/strong>. After batch correction there is one sample that clusters mostly independently from the other samples. This sample did not stand out in QC (althogh this will be re-examined). There is some evidence that this donor might have been clinacally different from the other donors, and since the batch correction worked well for the other samples, there is the possibility that this is real biologal signal. However since there is only one such sample it would be really hard to confirm this, and the inclusion of this sample would counfound the biologcal question of interest. I advisied to drop this sample post-clustering on the grounds that this donor might be too differnt from the others. <strong>2<\/strong>. Prelimnary results from DEG by cell type for the three groups find mitocondral genes are the most differntly expressed. They are concerned as MT RNA could be a technical artifact (contamination) this tech isn’t the best to explore this. But there is are other findings that cellular metabolism may play a role in this condition. I advisied to keep the DEG results with the MT, add the caveat in the paper that this might not be a perfect experiment for this but it could support real differnces in the cells (plus they already have pretty strict MT QC). Also run a second DEA w/o MT genes to see what other signal exists.<\/span>","<span class='gt_from_md'>The error came from the names of the clusters just being integers, the names for limma need to be sytactically valid (not just numbers, no spaces, no special characters - basically need to be variable names in R). I advised to simply add a prefix like we did in spatialDLPFC (sp9k1) or even simpler like “c” to her cluster names. Maybe we should add a warning to the function about cluster names since this error is pretty hard for users to de-bug and people often have non-sytactucally valid cluster names (especally with cell type names)<\/span>","<span class='gt_from_md'>Explained some scripts such as /dcs04/lieber/lcolladotor/BrainSEQ_LIBD001/_jhpce_org<\/span>","NA","<span class='gt_from_md'>Explaination of why an index label M=3 was chosen for refineVNS in section 2.2.<\/span>","<span class='gt_from_md'>I explained how I get out of the situation where files that are too big get accidentally committed (using ‘git log’, ‘git reset –soft’, and other details)<\/span>","<span class='gt_from_md'>We worked through an error message from the ‘VariantsMerge’ process, uncovering (arguably) a bug in SPEAQeasy, and I advised how to temporarily circumvent the problem. I also suggested a command for renaming a set of files that initially broke SPEAQeasy’s assumptions.<\/span>","NA","<span class='gt_from_md'>I talked about how Linux does not allow creation of files w/ execution permissions set, regardless of ‘umask’ settings such as in the ‘.bashrc’. I also explained how I use branching with git and other tips for managing a repo that others also develop on.<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/dotdotdot/issues/14\">https://github.com/LieberInstitute/dotdotdot/issues/14<\/a><\/span>","NA","NA","<span class='gt_from_md'>We downloaded a metadata table from SRA and parsed it through R to programatically create a “samples.manifest” file for use in SPEAQeasy, that merged samples that were each split into multiple FASTQ files.<\/span>","NA","NA","NA","<span class='gt_from_md'><a href=\"https://files.slack.com/files-pri/T029RHR07-F024D7ELKC2/image_from_ios.jpg\">https://files.slack.com/files-pri/T029RHR07-F024D7ELKC2/image_from_ios.jpg<\/a><\/span>","<span class='gt_from_md'>used back quote instead of single quote when assigning string to variables<\/span>","<span class='gt_from_md'>We decided to run CellProfiler on a while capture area image of DLPFC sample to look at the segmentations as it was doing good job on the cropped regions of a DLPFC sample<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/dotdotdot/issues/13\">https://github.com/LieberInstitute/dotdotdot/issues/13<\/a><\/span>","NA","<span class='gt_from_md'>SPEAQeasy would appear to run into “file not found” errors for just a handful of random samples in a large dataset. I eventually traced the error to a disk mounting issue on a particular compute node, which was resolved through bitsupport.<\/span>","<span class='gt_from_md'>The H&amp;E image that he was segmenting and the H&amp;E image for which he gets the ‘json’ and ‘csv’ are not the same images or at least not the same resolution which is causing the error.<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/dotdotdot/issues/12\">https://github.com/LieberInstitute/dotdotdot/issues/12<\/a><\/span>","NA","NA","<span class='gt_from_md'>used the command ‘eval’<\/span>","<span class='gt_from_md'>Did some troubleshooting on nested map functions, when to use ~.x and when to use function() syntax<\/span>","<span class='gt_from_md'>I mentioned that she encountered a bug that was fixed in a more recent version of SPEAQeasy, but suggested she manually add a line of code temporarily so that she could resume the run before updating<\/span>","<span class='gt_from_md'>I referred her to a script I had written in the past that contains the required commands and configuration  (with comments) for using the SRA toolkit<\/span>","NA","NA","<span class='gt_from_md'>Went over basic, complex, and nested map() usage and how to apply it to Josh’s current work<\/span>","NA","<span class='gt_from_md'>We were able to update the keys for the cluster, but the ssh key login was still broken<\/span>","NA","NA","<span class='gt_from_md'>We revised key features of Brenda Pardo’s Twitter thread <a href=\"https://twitter.com/PardoBree/status/1388253938391175173?s=20\">https://twitter.com/PardoBree/status/1388253938391175173?s=20<\/a> and some features that you would want your Twitter thread to have.<\/span>","<span class='gt_from_md'>I showed a fairly standard method for concatenating tabular data from multiple files into a single matrix, and saving the resulting R object<\/span>","<span class='gt_from_md'>We talked about all the details that go into writing an abstract for a conference. These details can take longer to complete than initially anticipated.<\/span>","<span class='gt_from_md'>We revised how to make a new branch in Git working with <a href=\"https://github.com/lmweber/OSTA-book\">https://github.com/lmweber/OSTA-book<\/a><\/span>","<span class='gt_from_md'>We discussed the current DNA genotype pipeline and potential changes others might want.<\/span>","<span class='gt_from_md'>ibidem<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/dotdotdot/issues/11\">https://github.com/LieberInstitute/dotdotdot/issues/11<\/a><\/span>","<span class='gt_from_md'>From SPEAQeasy, to qSVA, deconvolution, TWAS, etc. It might be best to give a talk about all these topics to whoever is interested at LIBD since we know about all these pieces but others might not.<\/span>","<span class='gt_from_md'>showed a example of how AWK extracts text lines from a text file<\/span>","<span class='gt_from_md'>Explain how some scripts in SUFI run and the bash scripts to use in place of python scripts for batch processing<\/span>","<span class='gt_from_md'>Referred to Vijay’s notes<\/span>","NA","NA","<span class='gt_from_md'>I described correct use of the “–input” argument, and we also found that many of the input FASTQs were empty, leading to a somewhat vague error message.<\/span>","<span class='gt_from_md'>In a series of email exchanges we diagnosed subtle errors resulting from bad FASTQ files and unintended options in the main script. Her feedback helped me improve some existing documentation.<\/span>","<span class='gt_from_md'>I attempted some formatting tricks to eliminate unwanted whitespace but was ultimately unsuccessful. We suspected the LaTeX template required the whitespace to exist. We also tried to fix a bug with the header of a bookdown RMD not rendering as expected. It seemed that re-writing the header (without making any visible changes) fixed the issue, though we weren’t sure of the source of the initial problem.<\/span>","<span class='gt_from_md'>We talked about <a href=\"https://lcolladotor.github.io/bioc_team_ds/\">https://lcolladotor.github.io/bioc_team_ds/<\/a> among other topics.<\/span>","<span class='gt_from_md'>used ggplot<\/span>","NA","NA","<span class='gt_from_md'>We confirmed that an existing VCF was corrupt and found that tabix expected bgzip-compressed VCF input rather than compressed BCF input.<\/span>","NA","<span class='gt_from_md'>This has changed over time. The most recent setup (as of 2021-08-06) is at <a href=\"https://github.com/lmweber/OSTA-book\">https://github.com/lmweber/OSTA-book<\/a><\/span>","<span class='gt_from_md'>I briefly mentioned the <code>-o<\/code> and <code>-e<\/code> arguments, and how several logs will generate in your home directory without those arguments and without <code>-cwd<\/code>, when “qsub”ing a job at JHPCE.<\/span>","<span class='gt_from_md'>We reviewed her code and I gave some pointer on what bugs to look out for with certain errors in ggplot<\/span>","<span class='gt_from_md'>We talked about the set of videos and resources publicly available to get to learn about R, Bioconductor and these packages. It was a lot of information and I’m sure that we’ll have more sessions later on to clarify questions as they arise.<\/span>","<span class='gt_from_md'>We talked about <a href=\"https://choosealicense.com/\">https://choosealicense.com/<\/a>, <a href=\"https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository\">https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository<\/a>, <a href=\"https://creativecommons.org/choose/\">https://creativecommons.org/choose/<\/a>.<\/span>","<span class='gt_from_md'>We discussed the posibility of proposing a class that has a mix of biology lectures and some practical coding sessions. The coding sessions could be based after the CDSB 2021 mini courses series.<\/span>","<span class='gt_from_md'>used Sys.glob and write.table to write the list to txt file<\/span>","NA","NA","<span class='gt_from_md'>We talked about the public data from Synapse and worked on locating some of it at JHPCE. I explained also how we have used bamtofastq when others don’t share their FASTQ files. Basically, we use bamtofastq to reconstruct the original FASTQ files (you might have to append the unmapped FASTQs if they are provided by the original authors) and then run FASTQ-processing tools like our SPEAQeasy pipeline.<\/span>","<span class='gt_from_md'>cleared up some confusion about filter_snps.R<\/span>","<span class='gt_from_md'>We met to figure out how to transfer the data for this collaboration, which ended up being quite tricky and involved using an SSH tunnel for rsync. Basically, a command like this one: <code>rsync -avzh -e 'ssh -o ProxyJump=user@server_somewhere_for_the_tunnel' *_files_of_interest user@final_server:/final_path/<\/code><\/span>","<span class='gt_from_md'>We discussed the different possible goals and ways to get to them and determined a path forward that relies on the rest of the DSgs system as well as existing LIBD rstats club videos to get started. It’ll be supported by a meeting with Leo and others involved in the spatial transcriptomics projects from the R side.<\/span>","<span class='gt_from_md'>I showed the initial version of the BrainSeq Phase II developmental expression explorer developed by Bill Ulrich and Aaron Goldman. We discussed other possibilities for the collaboration including helping set up SPEAQeasy in their servers.<\/span>","<span class='gt_from_md'>Pointed her to some oacjages for creating TWAS plots whe was interested in and we walked through some code I used to compare T-stats from different DE models<\/span>","<span class='gt_from_md'>we talked about how to use the “–strand” option, the source of the error that was encountered<\/span>","<span class='gt_from_md'>we discussed what a bfile is in plink and that made the process much easier.<\/span>","<span class='gt_from_md'><a href=\"https://github.com/LieberInstitute/zandiHyde_bipolar_rnaseq/tree/master/dev_twas#bd-twas-workflow\">https://github.com/LieberInstitute/zandiHyde_bipolar_rnaseq/tree/master/dev_twas#bd-twas-workflow<\/a><\/span>","<span class='gt_from_md'>still working on this will touch base again, we are trying to obtain the mds plot for ancestry in the placental data.<\/span>","<span class='gt_from_md'>the way we had run cluster profiler was generating lists in a way that the plot functions could not work with<\/span>","<span class='gt_from_md'>N/A<\/span>","<span class='gt_from_md'>We walked tought script from getRPKM to expression cut to filter rse objects, I also pointed Brenda towards some reasources to get started with ggplot<\/span>","<span class='gt_from_md'>Used GenomeStudio again to produce PLINK files<\/span>","<span class='gt_from_md'>We discussed the changes that we should make and the ones we shouldn’t based on the feedback received from the practice talk with Keri’s group. The feedback was excellent and very useful, however, we discussed which suggestions were maybe complicating the presentation too much and how to best incorporate them.<\/span>","NA","<span class='gt_from_md'>run clusterprofiler on a set of DE hits<\/span>","NA","<span class='gt_from_md'>Similar to part 2. The presentation was looking very polished by this meeting!<\/span>","<span class='gt_from_md'>Helped review the grant application and revise the text.<\/span>","<span class='gt_from_md'>Further updates on asking for help with this complicated project. Suggested reaching out to other LIBD members who have tried running the software before.<\/span>","<span class='gt_from_md'>Discussed with Josh what figures we needed to prepare and how to generate them. This also involved base_size = 30 with ggplot2. Otherwise the font was too small to read it on the grant.<\/span>","<span class='gt_from_md'>Now that a first major draft of the presentation was complete, we revised it and proposed some changes to it. Some of them involved remaking the figures using base_size = 20 or even 30 with ggplot2. For future work, it might be best to save the images with that font size from the beginning instead of having to update them later.<\/span>","<span class='gt_from_md'>We reviewed the model that Sreya was evaluating for DE. As we as troublshooting the plot functions from the SPEAQeasy bootcamp code. I also advised taht she could create a volcano plot for her results.<\/span>","<span class='gt_from_md'>We went through the creation of an LIBD JHPCE module, covering the documentation process, lua syntax, and how to test a module before pushing.<\/span>","<span class='gt_from_md'>We adapted a series of bash scripts requiring parallel functionality for use at JHPCE. This involved correctly setting some path variables and requesting multiple cores through SGE for some scripts.<\/span>","<span class='gt_from_md'>explained the process of using topmed imputation and later we will get the mds<\/span>","<span class='gt_from_md'>I described how I typically construct a “samples.manifest” file and configure SPEAQeasy for a particular experiment.<\/span>","<span class='gt_from_md'>We went over some plots we have made for DE previously, (quality control &amp; PC scatter plots, t-stat scatter between models, venn diagrams, and boxplots). Talked about some things to watch out for in her data (small samples size). We are planning to meet again when her data is ready.<\/span>","<span class='gt_from_md'>We went over the details of the GitHub actions workflow created by biocthis, the parts that you need to run from biocthis::use_bioc_pkg_templates(), and resolving issues prompted by R CMD check + BiocCheck<\/span>","<span class='gt_from_md'>I gave an overview of the resources and materials we have created as well as the LIBD bootcamps that we have publicly available to help others get started with computational biology. This is likely the first of several meetings.<\/span>","<span class='gt_from_md'>We spent a significant amount troubleshooting the problem here (nearly two hours?). In the end, using the insert image “here” LaTeX syntax resolved the problem.<\/span>","<span class='gt_from_md'>General advice on how to ask for help from open source projects<\/span>","<span class='gt_from_md'>Initial planning session for the LIBD seminar. We talked about the different slides we planned to show and the flow of the presentation. We also scheduled 3 practice talks with different groups of people.<\/span>","<span class='gt_from_md'>We observed an issue where log parsing in SPEAQeasy behaved differently for her and I due to a difference in default python version loaded at JHPCE. I fixed the hisat2/2.2.1 module to ensure consistent python version use.<\/span>","<span class='gt_from_md'>Overview of the TWAS pipeline<\/span>","<span class='gt_from_md'>I showed how I typically produce an array job shell script for the situation where I have many samples of data to process the same way (SIMD). We talked about using a text file where each line contains one sample name, and referencing that in the array script.<\/span>","<span class='gt_from_md'>I showed Sreya how to set up the S-TWAS directory that I adapted in the Zandi-Hyde bipolar project for her own adaptation. I also sent her some useful videos on how to set up GitHub with her JHPCE account and install an SFTP client for Ubuntu (I prefer Filezilla)<\/span>","<span class='gt_from_md'>We discussed how to handle strandness issues, and how different inferred strand patterns within a group of samples sequenced together can indicate problems with the input data. I described how to use the ‘–force_strand’ option with SPEAQeasy<\/span>","<span class='gt_from_md'>We went through cloning SPEAQeasy, creating a “samples.manifest” file with R, and other relevant details for running SPEAQeasy at JHPCE. I also helped him set up git at JHPCE<\/span>","<span class='gt_from_md'>We troubleshooted strange errors from the main SPEAQeasy output, and saw that I was able to run the main script (“run_pipeline_jhpce.sh”) but she encountered errors with an identical script. We suspected permissions issues in her working directory, which she emailed bitsupport about.<\/span>","<span class='gt_from_md'>I gave Sreya a conceptual introduction to transcriptome imputation and sent her some resources to help her get started on FUSION / TWAS<\/span>","<span class='gt_from_md'>We got ssh key login working, added config files to JHPCE, went though a “hello world” example of using files on JHPCE on RStudio<\/span>","<span class='gt_from_md'>Although we encountered permissions issues, I showed how to create a “samples.manifest” file and configure SPEAQeasy for a particular set of samples.<\/span>","NA","<span class='gt_from_md'>Henry needed to reset his JHPCE password as he had forgotten it, but we went through RStudio set up and got cyberduck installed, we will set up a meeting next week for ssh keys and configuration<\/span>","<span class='gt_from_md'>I showed my typical process for creating a “samples.manifest” file for a given dataset, and how to submit a pipeline run. We continued to debug some minor issues over slack<\/span>","NA","NA","<span class='gt_from_md'>Created a learning plan involving several team members that is now documented at a channel in JHU Genomics Slack. That way everyone can keep track of what others are teaching the person who asked the question, and we can have a common communication channel among everyone involved. Sessions include learning how to interact with JHPCE through RStudio, how to run SPEAQeasy, key R/Bioconductor packages (limma, SummarizedExperiment), and explore possible visualizations of interest.<\/span>","<span class='gt_from_md'>I covered various nextflow concepts, as well as my typical approach in creating and running a nextflow project at JHPCE<\/span>","<span class='gt_from_md'>explaination of nurolucida system, how number of cores and RAM matter for running SUFI<\/span>","NA","<span class='gt_from_md'>Discussion about features that spatialLIBD could support to make it easier to use for newer data.<\/span>","<span class='gt_from_md'>Went over basic functions of SummarizedExperiment and SingleCellExperiment, what a Sparse matrix is - how to convert to non-sparse. Steps to take to use getRPKM on SingeCellExperient.<\/span>","<span class='gt_from_md'>sent him the scripts from andrew, wondering if I should make it into a function<\/span>","<span class='gt_from_md'>Help with understanding <a href=\"https://bookdown.org/yihui/bookdown/\">bookdown<\/a> and ideas on how to organize a documentation website (chapters and information to consider).<\/span>","<span class='gt_from_md'>Discussed the research project proposal and how we can help.<\/span>","<span class='gt_from_md'>General advice. Received feedback regarding file backups.<\/span>","<span class='gt_from_md'>explaination of scale_factors.json, tissuepositions_list.csv from spaceranger and the spotspotcheck package<\/span>","<span class='gt_from_md'>I identified that <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829\">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829<\/a> contains both miRNA and RNA-seq data. Using data from the SRA Run selector at <a href=\"https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&amp;o=acc_s%3Aa\">https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&amp;o=acc_s%3Aa<\/a> I was able to filter the data to the RNA-seq samples and subset the input samples.manifest file for SPEAQeasy.<\/span>","<span class='gt_from_md'>To identify DEGs or eQTLs, the user has to choose what covariates to adjust for. Some studies might require more complicated models than others, as they might be from more than one brain region or present some batch effect. In addition, in newer projects we are considering adjusting for cell type fractions derived from deconvolution. Even if all the covariates were fixed across datasets, in DEGs we have typically adjusted for qSVs which are computed in a dataset-dependent fashion and in essence are PCs derived from degradation-associated genomic regions. Similarly, for eQTLs we adjust for expression PCs. In both we adjust for ancestry PCs. Computing PCs is input-dependent: you won’t get the same values or PCs if you change which samples you are using. So, computing DEGs and eQTLs is not as simple as say a map-reduce problem where you compute the same thing for every input sample, then aggregate the per-sample results. We have steps that depend on the whole sample set. Then there’s also steps for filtering expression features based on some minimum expression cutoffs. Plus, differences between say polyA and RiboZero data, which don’t make sense to process together. For these reasons, I don’t think that an on-demand computation pipeline is feasible based on a user selecting a set of input samples. Constructing RNA-seq objects (RangedSummarizedExperiment objects) as well as DNA genotyping VCF files based on a set of input samples is feasible and that’s what others are building. After that, I think that it’s best to adapt code we have (or even use newer tools) after carefully exploring the data.<\/span>","NA","<span class='gt_from_md'>I described the need for the “module use” command and modifying the .bashrc to easily use LIBD modules<\/span>","<span class='gt_from_md'>We walked througth the SpatialLIBD code to find where the argument has to be changed<\/span>","<span class='gt_from_md'>I fetched the SRA files, created a manifest and execution script for SPEAQeasy, but had an issue running “fastq-dump”. I decided to hand off the request, partially complete.<\/span>","<span class='gt_from_md'>Failed MacOS installation on Lieber MacBook, something about Java both being and also not being installed<\/span>","<span class='gt_from_md'>explaination of the functions VNS.m and refineVNS.m<\/span>","<span class='gt_from_md'>when running VNS.m for several files on same matlab session the data from previous sessions is saved in matlab cache memory automatically. This will cause memory issues with subsequent runs, its important to clear variables after every run.<\/span>","<span class='gt_from_md'>reffered to package “effsize” should be efficient at easily doing this<\/span>","<span class='gt_from_md'>ran her through an example, explained her the code on github repo (split.m is the main script)<\/span>","<span class='gt_from_md'>edited the config file for setting up x-quartz<\/span>","<span class='gt_from_md'>Adapted code and shared it at <a href=\"https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock\">https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock<\/a><\/span>","<span class='gt_from_md'>He walked me through some code performing qSVA for a recent project, and I offered some tips involving code organization/ readibility.<\/span>","<span class='gt_from_md'>Created GitHub issues at <a href=\"https://github.com/LieberInstitute/VisiumImgProcessing/issues\">https://github.com/LieberInstitute/VisiumImgProcessing/issues<\/a><\/span>","<span class='gt_from_md'>fixed the github repo<\/span>","<span class='gt_from_md'>I showed them the tools I developed in DeconvoBuddies package, and provided links to example code where I had applied these funcitons in previous work<\/span>","<span class='gt_from_md'>explore loading/saving/exporting options on Qupath<\/span>","<span class='gt_from_md'>We discussed how Sciwheel can be adapted for different journals, and provided general advice for getting started and formatting a paper for a particular journal.<\/span>","<span class='gt_from_md'>I explained that SPEAQeasy was used for the recent hg38 processing, and an older version of a similar pipeline was used for the original hg19 processing. We discussed our new method for retrieving gene symbols (“org.*.eg.db” packages) and some general code advice.<\/span>","<span class='gt_from_md'>I went through the setup and example usage of Sciwheel and cross reference, and gave general advice (it’s often easiest to start with the “Methods” section, etc)<\/span>","<span class='gt_from_md'>use rsync<\/span>","<span class='gt_from_md'>Debugging through the input/output file in logs<\/span>","<span class='gt_from_md'>I explained how to use SPEAQeasy at JHPCE, and we tried an experimental method for downloading SRA data. The method didn’t work, and we plan to meet again after I verify a more typical method works.<\/span>","<span class='gt_from_md'>I hadn’t encountered the issue with new local terminals failing to open. I planned to follow up with better specific advice via email, and offer to explain this advice in a future DSGS.<\/span>","<span class='gt_from_md'>running chmod<\/span>","<span class='gt_from_md'>Used Amy’s rstats blogpost on git<\/span>","<span class='gt_from_md'>We spent time doing basic Rstudio setup and troubleshooting, and got the data downloaded<\/span>","NA","<span class='gt_from_md'>We got Heena’s MAC registered for passwordless login to JHPCE<\/span>","NA","<span class='gt_from_md'>We got the ssh keys working for Brenda’s github<\/span>","<span class='gt_from_md'>We went over applying DeconvoBuddies functions to hebenula data<\/span>","<span class='gt_from_md'>Use of galaxybrain for identifying samples with DNAm, DNA genotype and RNA-seq data<\/span>","<span class='gt_from_md'>We are going to try an iterative strategy to keep tables smaller<\/span>","NA","<span class='gt_from_md'>Practiced questions for interviews.<\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/p134069/#p134141\">BioC Support website answer<\/a><\/span>","NA","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/p133591/#p133927\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'>Used ‘h5ls’ command from shell to verify contents of file; eventually found we were looking at the wrong file (error message was cryptic about issue). <a href=\"https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1607034309074800\">https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1607034309074800<\/a><\/span>","<span class='gt_from_md'>Need to resolve swap bugs. Plan to create a table of phenotype data (Dx, age, sex, race) for each BrNum to mach genotype data<\/span>","<span class='gt_from_md'>Went through the setting up passwordless login for jhpce using ssh keys, and coppied the configuration files from <a href=\"https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee\">https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee<\/a><\/span>","<span class='gt_from_md'><a href=\"https://community.rstudio.com/t/recipes-package-cannot-create-interaction-term-in-step-interact/89860\">RStudio Community self-answer<\/a><\/span>","<span class='gt_from_md'>got through the Github vignettes thoroughly<\/span>","NA","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/p133110/#p133319\">BioC Support website answer<\/a><\/span>","NA","<span class='gt_from_md'>Started PsychENCODE access request process<\/span>","<span class='gt_from_md'><a href=\"https://support.illumina.com/downloads/genomestudio-2-0-plug-ins.html\">There is a plugin for Genome Studio by Illumina that allows users to output their imputed allele results as PLINK inputs<\/a><\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/134569/#p133028\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'>to use rsync or scp, you need to have user account created on LIBD server (Bill can help create it)<\/span>","NA","<span class='gt_from_md'>We debugged some code and ultimetly decided that it might be best for Abby to write new code and use the tsne function to cluster and plot the data<\/span>","<span class='gt_from_md'>I recomended using dcast to do a “long” to “wide” data transformation<\/span>","<span class='gt_from_md'>We talked about how the data should be structured before using ggplot (transforming some data and adding catagorical columns)<\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/p132558/#p132872\">I recommended that they reach out to their forum board<\/a><\/span>","<span class='gt_from_md'><a href=\"https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1605123296024100\">Check Slack helpdesk channel<\/a><\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/134139/#p132829\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'><a href=\"https://github.com/lcolladotor/biocthis/issues/15\">GitHub issue at biocthis<\/a><\/span>","<span class='gt_from_md'>We discussed organizing by analysis (a directory for each one, then prehaps sub- directory for data &amp; plots). Create a readMe for each sub-folder. Use gitignore to keep repo from getting too big.<\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/131600/#p132617\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'><a href=\"https://www.mathworks.com/matlabcentral/answers/636365-how-can-i-measure-volume-from-a-3d-image?s_tid=srchtitle\">https://www.mathworks.com/matlabcentral/answers/636365-how-can-i-measure-volume-from-a-3d-image?s_tid=srchtitle<\/a><\/span>","<span class='gt_from_md'>Resulted in recount3 versions 1.1.1 and 1.1.2 on bioc-devel (and 1.0.2 on bioc-release)<\/span>","<span class='gt_from_md'><a href=\"https://github.com/lcolladotor/biocthis/issues/13\">GitHub issue at biocthis<\/a><\/span>","<span class='gt_from_md'><a href=\"https://github.com/lcolladotor/biocthis/issues/14\">GitHub issue at biocthis<\/a><\/span>","NA","<span class='gt_from_md'>Explained the <code>chgrp<\/code> and <code>umask<\/code> commands plus <code>~/.bashrc<\/code> file configuration. I also mentioned why we don’t use <code>chmod 777<\/code> at JHPCE. Details on <a href=\"https://jhu-genomics.slack.com/archives/CR9NYA0BF/p1604086263148400?thread_ts=1603826219.094600&amp;cid=CR9NYA0BF\">Slack<\/a> (private channel).<\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/p132487/#p132521\">BioC Support website answer<\/a><\/span>","NA","NA","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/p132406/#p132459\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/p132450/#p132464\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'>initial issue seemed to do with the spaceranger command arguments; we are continuining to investigate possible module issues or insufficient code in the <a href=\"http://research.libd.org/rstatsclub/2020/10/20/using-space-ranger-at-jhpce/\">blog post<\/a><\/span>","<span class='gt_from_md'>Needed to use highlight_key and highlight functions to select points between plots. Used rstats example as guide <a href=\"https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14\">https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14<\/a><\/span>","NA","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/134532/#134925\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/134914/#134915\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/131285/#134921\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'><a href=\"https://support.bioconductor.org/p/134876/#134917\">BioC Support website answer<\/a><\/span>","<span class='gt_from_md'>spatstat package <a href=\"https://spatstat.org/releasenotes/spatstat-1.39-0.html\">https://spatstat.org/releasenotes/spatstat-1.39-0.html<\/a>, L function <a href=\"https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest\">https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest<\/a>, Kmark <a href=\"https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark\">https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark<\/a><\/span>"],"html":true,"width":680,"align":"left"}],"resizable":true,"filterable":true,"searchable":true,"defaultPageSize":10,"showPageSizeOptions":false,"pageSizeOptions":[10,25,50,100],"paginationType":"numbers","showPagination":true,"showPageInfo":true,"minRows":1,"width":"1200px","height":"auto","theme":{"color":"#333333","backgroundColor":"#FFFFFF","stripedColor":"rgba(128,128,128,0.05)","style":{"font-family":"system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif","fontSize":"16px"},"tableStyle":{"borderTopStyle":"solid","borderTopWidth":"2px","borderTopColor":"#D3D3D3"},"headerStyle":{"fontWeight":"normal","backgroundColor":"transparent","borderBottomStyle":"solid","borderBottomWidth":"2px","borderBottomColor":"#D3D3D3"},"groupHeaderStyle":{"fontWeight":"normal","backgroundColor":"transparent","borderBottomStyle":"solid","borderBottomWidth":"2px","borderBottomColor":"#D3D3D3"},"cellStyle":{"fontWeight":"normal"}},"elementId":"egkyysggow","dataKey":"8b0269e3dcfc57c3410e8cf24d1247ef"},"children":[]},"class":"reactR_markup"},"evals":["tag.attribs.columns.0.style","tag.attribs.columns.1.style","tag.attribs.columns.2.style","tag.attribs.columns.3.style"],"jsHooks":[]}</script>
</div>
</div>
<div class="container">
    <footer class="site-footer">

      <center>
<p class="powered-by">
© 2011-2023. All thoughts and opinions here are my own. The icon was designed by <a href="https://www.linkedin.com/in/mauricio-guzman-6529b551/">Mauricio Guzmán</a> and is inspired by <a href="https://en.wikipedia.org/wiki/Huichol">Huichol culture</a>; it represents my community building interests.
</p>

<p class="powered-by copyright-license-text">
This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">CC BY NC SA 4.0</a>
</p>

<p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">
      <img src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" alt="CC icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/by.svg" alt="CC by icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/nc.svg" alt="CC NC icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/sa.svg" alt="CC SA icon" width="32px">
    </a>
</p>

<div id="clustrmaps-widget"></div><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=300&t=m&d=Zo-deincaJGsqhqjIWsujtdoNOMJsnnZLefKmxDKVv8&co=124b73&cmo=dd53ff&cmn=4ec724'></script>

<p class="powered-by">
Published with <a href="https://bookdown.org/" target="_blank" rel="noopener">Bookdown</a>
</p>

</center>



<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-3CBVR6GQ07"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-3CBVR6GQ07');
</script>

<!-- Go to www.addthis.com/dashboard to customize your tools -->
<script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4ef23e545d41b87c"></script>

<script type="text/javascript">
var addthis_config = {
	data_ga_property: 'G-3CBVR6GQ07',
	data_ga_social : true,
	data_track_addressbar: true
};
var addthis_share = {
	// ... other options
	url_transforms : {
		shorten: {
			twitter: 'bitly'
		}
	},
	shorteners : {
		bitly : {}
	}
}
</script>

</footer>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="r-session-information.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
