<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Full logs | Data Science guidance session logs</title>
  <meta name="description" content="Full logs | Data Science guidance session logs" />
  <meta name="generator" content="bookdown 0.39 and GitBook 2.6.7" />

  <meta property="og:title" content="Full logs | Data Science guidance session logs" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="lcolladotor/dsgs_logs" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Full logs | Data Science guidance session logs" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="icon_32.png" type="image/x-icon" />
<link rel="prev" href="index.html"/>
<link rel="next" href="r-session-information.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.4/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>
<script src="libs/core-js-2.5.3/shim.min.js"></script>
<script src="libs/react-18.2.0/react.min.js"></script>
<script src="libs/react-18.2.0/react-dom.min.js"></script>
<script src="libs/reactwidget-1.0.0/react-tools.js"></script>
<link href="libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="libs/reactable-binding-0.4.4/reactable.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#dsgs-guides"><i class="fa fa-check"></i>DSgs-guides</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#graphs"><i class="fa fa-check"></i>Graphs</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="full-logs.html"><a href="full-logs.html"><i class="fa fa-check"></i>Full logs</a></li>
<li class="chapter" data-level="" data-path="r-session-information.html"><a href="r-session-information.html"><i class="fa fa-check"></i>R session information</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data Science guidance session logs</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="full-logs" class="section level1 unnumbered hasAnchor">
<h1>Full logs<a href="full-logs.html#full-logs" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Here you an explore the full logs. The name of the person asking the question(s) is not displayed for privacy.</p>
<p>The idea is that this information could be useful for people who have similar questions, as well as for DSgs guides who want to re-use material from previous answers when helping someone.</p>
<div id="vzyonjxzbt" class=".gt_table" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#vzyonjxzbt table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#vzyonjxzbt thead, #vzyonjxzbt tbody, #vzyonjxzbt tfoot, #vzyonjxzbt tr, #vzyonjxzbt td, #vzyonjxzbt th {
  border-style: none;
}

#vzyonjxzbt p {
  margin: 0;
  padding: 0;
}

#vzyonjxzbt .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 1200px;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#vzyonjxzbt .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#vzyonjxzbt .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#vzyonjxzbt .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#vzyonjxzbt .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#vzyonjxzbt .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vzyonjxzbt .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#vzyonjxzbt .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#vzyonjxzbt .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#vzyonjxzbt .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#vzyonjxzbt .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#vzyonjxzbt .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#vzyonjxzbt .gt_spanner_row {
  border-bottom-style: hidden;
}

#vzyonjxzbt .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#vzyonjxzbt .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#vzyonjxzbt .gt_from_md > :first-child {
  margin-top: 0;
}

#vzyonjxzbt .gt_from_md > :last-child {
  margin-bottom: 0;
}

#vzyonjxzbt .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#vzyonjxzbt .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#vzyonjxzbt .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#vzyonjxzbt .gt_row_group_first td {
  border-top-width: 2px;
}

#vzyonjxzbt .gt_row_group_first th {
  border-top-width: 2px;
}

#vzyonjxzbt .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#vzyonjxzbt .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#vzyonjxzbt .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#vzyonjxzbt .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vzyonjxzbt .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#vzyonjxzbt .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#vzyonjxzbt .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#vzyonjxzbt .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#vzyonjxzbt .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#vzyonjxzbt .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#vzyonjxzbt .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#vzyonjxzbt .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#vzyonjxzbt .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#vzyonjxzbt .gt_left {
  text-align: left;
}

#vzyonjxzbt .gt_center {
  text-align: center;
}

#vzyonjxzbt .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#vzyonjxzbt .gt_font_normal {
  font-weight: normal;
}

#vzyonjxzbt .gt_font_bold {
  font-weight: bold;
}

#vzyonjxzbt .gt_font_italic {
  font-style: italic;
}

#vzyonjxzbt .gt_super {
  font-size: 65%;
}

#vzyonjxzbt .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#vzyonjxzbt .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#vzyonjxzbt .gt_indent_1 {
  text-indent: 5px;
}

#vzyonjxzbt .gt_indent_2 {
  text-indent: 10px;
}

#vzyonjxzbt .gt_indent_3 {
  text-indent: 15px;
}

#vzyonjxzbt .gt_indent_4 {
  text-indent: 20px;
}

#vzyonjxzbt .gt_indent_5 {
  text-indent: 25px;
}
</style>
<div id="vzyonjxzbt" class="reactable html-widget" style="width:1200px;height:auto;"></div>
<script type="application/json" data-for="vzyonjxzbt">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"date":["2024-04-16","2024-04-15","2024-04-11","2024-04-08","2024-04-05","2024-04-05","2024-04-05","2024-04-03","2024-04-02","2024-04-01","2024-03-29","2024-03-27","2024-03-27","2024-03-27","2024-03-25","2024-03-25","2024-03-25","2024-03-23","2024-03-22","2024-03-22","2024-03-20","2024-03-20","2024-03-20","2024-03-19","2024-03-19","2024-03-19","2024-03-15","2024-03-15","2024-03-14","2024-03-14","2024-03-14","2024-03-13","2024-03-12","2024-03-11","2024-03-11","2024-03-07","2024-03-07","2024-03-05","2024-03-05","2024-03-04","2024-03-01","2024-03-01","2024-02-28","2024-02-28","2024-02-26","2024-02-23","2024-02-16","2024-02-16","2024-02-16","2024-02-12","2024-02-09","2024-02-08","2024-02-07","2024-02-05","2024-02-02","2024-02-01","2024-01-30","2024-01-29","2024-01-26","2024-01-26","2024-01-25","2024-01-25","2024-01-24","2024-01-24","2024-01-22","2024-01-22","2024-01-18","2024-01-11","2024-01-08","2024-01-05","2023-12-12","2023-12-11","2023-12-08","2023-12-08","2023-12-08","2023-12-08","2023-12-05","2023-12-01","2023-11-30","2023-11-29","2023-11-29","2023-11-21","2023-11-16","2023-11-16","2023-11-15","2023-11-15","2023-11-08","2023-11-08","2023-11-08","2023-11-03","2023-11-01","2023-11-01","2023-11-01","2023-10-30","2023-10-30","2023-10-27","2023-10-27","2023-10-26","2023-10-26","2023-10-25","2023-10-25","2023-10-24","2023-10-20","2023-10-19","2023-10-18","2023-10-17","2023-10-16","2023-10-13","2023-10-13","2023-10-12","2023-10-12","2023-10-11","2023-10-11","2023-10-06","2023-10-05","2023-10-04","2023-09-29","2023-09-29","2023-09-28","2023-09-27","2023-09-27","2023-09-26","2023-09-25","2023-09-22","2023-09-22","2023-09-22","2023-09-21","2023-09-21","2023-09-21","2023-09-19","2023-09-19","2023-09-14","2023-09-13","2023-09-12","2023-09-06","2023-08-28","2023-08-23","2023-08-22","2023-08-21","2023-08-21","2023-08-18","2023-08-17","2023-08-16","2023-08-16","2023-08-14","2023-08-14","2023-08-11","2023-08-11","2023-08-10","2023-08-10","2023-08-09","2023-08-09","2023-08-09","2023-08-09","2023-08-07","2023-08-03","2023-08-02","2023-07-27","2023-07-27","2023-07-27","2023-07-27","2023-07-26","2023-07-26","2023-07-26","2023-07-26","2023-07-26","2023-07-25","2023-07-25","2023-07-24","2023-07-21","2023-07-20","2023-07-20","2023-07-20","2023-07-19","2023-07-18","2023-07-17","2023-07-17","2023-07-12","2023-07-07","2023-07-01","2023-06-29","2023-06-28","2023-06-28","2023-06-28","2023-06-27","2023-06-23","2023-06-22","2023-06-21","2023-06-15","2023-06-14","2023-06-09","2023-06-08","2023-06-08","2023-06-08","2023-06-08","2023-06-07","2023-06-02","2023-06-02","2023-06-01","2023-05-26","2023-05-25","2023-05-25","2023-05-24","2023-05-18","2023-05-18","2023-05-17","2023-05-16","2023-05-09","2023-05-02","2023-04-25","2023-04-21","2023-04-19","2023-04-17","2023-04-14","2023-04-12","2023-04-11","2023-04-10","2023-04-10","2023-04-07","2023-04-06","2023-04-06","2023-04-04","2023-03-31","2023-03-28","2023-03-24","2023-03-23","2023-03-23","2023-03-22","2023-03-22","2023-03-22","2023-03-21","2023-03-21","2023-03-21","2023-03-20","2023-03-15","2023-03-13","2023-03-13","2023-03-10","2023-03-09","2023-03-08","2023-03-02","2023-03-02","2023-03-02","2023-03-02","2023-03-02","2023-03-01","2023-03-01","2023-03-01","2023-02-28","2023-02-21","2023-02-17","2023-02-15","2023-02-10","2023-02-09","2023-02-06","2023-02-03","2023-02-01","2023-01-25","2023-01-25","2023-01-25","2023-01-20","2023-01-19","2023-01-18","2023-01-18","2023-01-18","2023-01-13","2023-01-11","2023-01-11","2023-01-10","2023-01-04","2023-01-03","2022-12-14","2022-12-14","2022-12-06","2022-12-05","2022-12-01","2022-11-30","2022-11-30","2022-11-30","2022-11-23","2022-11-16","2022-11-15","2022-11-03","2022-11-01","2022-10-28","2022-10-27","2022-10-26","2022-10-17","2022-10-14","2022-10-13","2022-10-11","2022-10-10","2022-10-05","2022-10-05","2022-09-30","2022-09-30","2022-09-29","2022-09-26","2022-09-23","2022-09-23","2022-09-16","2022-09-16","2022-09-12","2022-09-09","2022-09-06","2022-08-24","2022-08-24","2022-08-22","2022-08-22","2022-08-18","2022-08-17","2022-08-17","2022-08-10","2022-08-09","2022-08-09","2022-08-09","2022-08-08","2022-08-03","2022-07-28","2022-07-20","2022-07-19","2022-07-19","2022-07-18","2022-07-15","2022-07-15","2022-07-13","2022-07-12","2022-07-07","2022-07-07","2022-07-06","2022-07-06","2022-07-05","2022-07-01","2022-06-29","2022-06-29","2022-06-28","2022-06-23","2022-06-23","2022-06-21","2022-06-21","2022-06-17","2022-06-15","2022-06-15","2022-06-15","2022-06-15","2022-06-10","2022-06-03","2022-06-03","2022-06-02","2022-06-01","2022-05-27","2022-05-27","2022-05-27","2022-05-27","2022-05-25","2022-05-20","2022-05-20","2022-05-18","2022-05-18","2022-05-17","2022-05-17","2022-05-17","2022-05-17","2022-05-10","2022-05-09","2022-05-06","2022-05-05","2022-05-05","2022-05-03","2022-04-29","2022-04-29","2022-04-28","2022-04-27","2022-04-27","2022-04-27","2022-04-21","2022-04-20","2022-04-15","2022-04-15","2022-04-08","2022-04-08","2022-04-08","2022-04-07","2022-04-07","2022-04-07","2022-04-06","2022-04-06","2022-04-06","2022-04-05","2022-04-04","2022-04-04","2022-04-01","2022-03-27","2022-03-25","2022-03-24","2022-03-22","2022-03-18","2022-03-18","2022-03-17","2022-03-17","2022-03-16","2022-03-16","2022-03-15","2022-03-15","2022-03-14","2022-03-11","2022-03-09","2022-03-09","2022-03-09","2022-03-09","2022-03-09","2022-03-04","2022-03-04","2022-03-04","2022-03-04","2022-03-03","2022-03-03","2022-03-02","2022-03-02","2022-02-25","2022-02-24","2022-02-24","2022-02-24","2022-02-23","2022-02-18","2022-02-17","2022-02-16","2022-02-16","2022-02-15","2022-02-15","2022-02-15","2022-02-14","2022-02-11","2022-02-11","2022-02-10","2022-02-07","2022-02-04","2022-02-04","2022-01-25","2022-01-21","2022-01-21","2022-01-21","2022-01-21","2022-01-21","2022-01-20","2022-01-20","2022-01-19","2021-12-15","2021-12-09","2021-12-08","2021-12-06","2021-11-18","2021-11-18","2021-11-18","2021-11-17","2021-11-16","2021-11-15","2021-11-04","2021-11-03","2021-10-29","2021-10-25","2021-10-23","2021-10-20","2021-10-14","2021-10-07","2021-10-06","2021-10-04","2021-09-27","2021-09-22","2021-09-20","2021-09-14","2021-09-14","2021-09-10","2021-09-09","2021-09-09","2021-09-09","2021-09-09","2021-09-09","2021-09-08","2021-09-07","2021-09-03","2021-09-03","2021-09-02","2021-09-02","2021-08-30","2021-08-30","2021-08-30","2021-08-26","2021-08-26","2021-08-25","2021-08-23","2021-08-23","2021-08-23","2021-08-21","2021-08-20","2021-08-18","2021-08-11","2021-08-06","2021-08-06","2021-08-03","2021-07-30","2021-07-28","2021-07-28","2021-07-20","2021-07-20","2021-07-15","2021-07-15","2021-07-14","2021-07-14","2021-07-14","2021-07-14","2021-07-14","2021-07-13","2021-07-13","2021-07-09","2021-07-09","2021-07-09","2021-07-02","2021-07-01","2021-07-01","2021-06-30","2021-06-30","2021-06-29","2021-06-29","2021-06-29","2021-06-24","2021-06-24","2021-06-24","2021-06-21","2021-06-21","2021-06-17","2021-06-17","2021-06-17","2021-06-17","2021-06-16","2021-06-16","2021-06-15","2021-06-10","2021-06-05","2021-06-04","2021-06-03","2021-06-02","2021-06-02","2021-06-01","2021-06-01","2021-05-28","2021-05-26","2021-05-24","2021-05-21","2021-05-20","2021-05-20","2021-05-20","2021-05-19","2021-05-19","2021-05-14","2021-05-13","2021-05-12","2021-05-10","2021-05-07","2021-05-07","2021-05-06","2021-05-06","2021-05-03","2021-05-03","2021-04-28","2021-04-28","2021-04-26","2021-04-22","2021-04-22","2021-04-20","2021-04-19","2021-04-19","2021-04-15","2021-04-15","2021-04-12","2021-04-09","2021-04-09","2021-04-08","2021-04-07","2021-04-01","2021-03-30","2021-03-29","2021-03-26","2021-03-26","2021-03-25","2021-03-25","2021-03-25","2021-03-24","2021-03-24","2021-03-24","2021-03-23","2021-03-22","2021-03-19","2021-03-18","2021-03-18","2021-03-17","2021-03-17","2021-03-16","2021-03-12","2021-03-12","2021-03-11","2021-03-10","2021-03-10","2021-03-06","2021-03-05","2021-03-05","2021-03-03","2021-03-02","2021-03-02","2021-02-26","2021-02-26","2021-02-25","2021-02-23","2021-02-19","2021-02-12","2021-02-09","2021-02-05","2021-02-05","2021-02-05","2021-02-04","2021-02-04","2021-01-29","2021-01-27","2021-01-26","2021-01-14","2021-01-11","2021-01-06","2020-12-09","2020-12-08","2020-12-07","2020-12-03","2020-12-03","2020-12-03","2020-12-02","2020-12-01","2020-11-25","2020-11-23","2020-11-20","2020-11-19","2020-11-19","2020-11-18","2020-11-13","2020-11-12","2020-11-12","2020-11-11","2020-11-11","2020-11-11","2020-11-05","2020-11-04","2020-11-04","2020-11-03","2020-11-03","2020-11-03","2020-11-02","2020-10-30","2020-10-30","2020-10-28","2020-10-28","2020-10-28","2020-10-28","2020-10-28","2020-10-27","2020-10-26","2020-10-21","2020-10-21","2020-10-21","2020-10-21","2020-10-20"],"guide":["Leo","Leo","Cynthia","Leo","Cynthia","Cynthia","Leo","Leo","Leo","Leo","Leo","Louise","Maddy","Cynthia","Maddy","Leo","Cynthia","Louise","Leo","Cynthia","Nick","Leo","Leo","Leo","Leo","Nick","Leo","Nick","Leo","Leo","Louise","Nick","Leo","Maddy","Louise","Leo","Nick","Leo","Nick","Nick","Maddy","Leo","Louise","Leo","Leo","Leo","Leo","Leo","Nick","Nick","Hedia","Leo","Cynthia","Nick","Nick","Nick","Nick","Leo","Nick","Nick","Leo","Louise","Leo","Nick","Louise","Louise","Hedia","Leo","Nick","Nick","Nick","Nick","Nick","Hedia","Nick","Nick","Nick","Leo","Louise","Leo","Louise","Nick","Leo","Leo","Leo","Louise","Leo","Leo","Nick","Leo","Leo","Leo","Nick","Nick","Nick","Leo","Leo","Leo","Leo","Nick","Louise","Nick","Leo","Louise","Leo","Nick","Nick","Nick","Nick","Leo","Louise","Nick","Nick","Leo","Louise","Nick","Leo","Louise","Nick","Leo","Nick","Nick","Nick","Louise","Leo","Leo","Leo","Louise","Louise","Nick","Nick","Leo","Leo","Nick","Louise","Nick","Nick","Nick","Louise","Leo","Leo","Nick","Nick","Nick","Nick","Leo","Nick","Nick","Louise","Leo","Nick","Louise","Maddy","Maddy","Nick","Maddy","Maddy","Leo","Maddy","Nick","Leo","Louise","Louise","Leo","Leo","Leo","Louise","Nick","Maddy","Maddy","Maddy","Leo","Leo","Nick","Nick","Maddy","Nick","Nick","Maddy","Maddy","Leo","Leo","Leo","Louise","Leo","Leo","Leo","Leo","Louise","Louise","Nick","Nick","Louise","Leo","Leo","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Louise","Nick","Nick","Nick","Nick","Nick","Nick","Hedia","Ryan","Nick","Cynthia","Nick","Nick","Cynthia","Nick","Hedia","Nick","Louise","Hedia","Louise","Cynthia","Nick","Nick","Hedia","Hedia","Ryan","Louise","Nick","Leo","Leo","Leo","Leo","Nick","Hedia","Leo","Leo","Leo","Leo","Nick","Nick","Louise","Leo","Louise","Louise","Nick","Louise","Louise","Nick","Nick","Nick","Nick","Nick","Hedia","Nick","Louise","Louise","Nick","Hedia","Louise","Leo","Louise","Nick","Louise","Nick","Louise","Louise","Nick","Nick","Nick","Louise","Louise","Nick","Nick","Nick","Nick","Louise","Leo","Nick","Louise","Louise","Louise","Louise","Nick","Louise","Louise","Louise","Louise","Louise","Louise","Leo","Nick","Nick","Leo","Nick","Nick","Nick","Maddy","Leo","Leo","Nick","Nick","Louise","Louise","Louise","Leo","Louise","Maddy","Maddy","Maddy","Maddy","Louise","Maddy","Abby","Leo","Maddy","Nick","Louise","Leo","Nick","Leo","Louise","Louise","Nick","Leo","Louise","Nick","Abby","Nick","Louise","Nick","Maddy","Nick","Abby","Louise","Abby","Abby","Louise","Louise","Maddy","Leo","Louise","Louise","Louise","Leo","Leo","Abby","Leo","Leo","Abby","Leo","Louise","Leo","Abby","Abby","Leo","Maddy","Abby","Louise","Leo","Louise","Nick","Leo","Leo","Nick","Leo","Louise","Leo","Abby","Leo","Leo","Leo","Leo","Leo","Abby","Leo","Louise","Maddy","Maddy","Maddy","Leo","Louise","Maddy","Josh","Abby","Nick","Abby","Nick","Louise","Maddy","Abby","Leo","Abby","Nick","Leo","Nick","Leo","Louise","Nick","Leo","Nick","Maddy","Nick","Maddy","Maddy","Nick","Leo","Louise","Leo","Leo","Leo","Josh","Nick","Maddy","Leo","Louise","Louise","Maddy","Leo","Maddy","Leo","Maddy","Leo","Louise","Maddy","Maddy","Maddy","Maddy","Maddy","Nick","Nick","Nick","Maddy","Nick","Maddy","Maddy","Maddy","Maddy","Louise","Maddy","Louise","Nick","Maddy","Maddy","Leo","Maddy","Maddy","Maddy","Nick","Maddy","Maddy","Maddy","Louise","Louise","Louise","Louise","Leo","Maddy","Maddy","Nick","Nick","Leo","Nick","Maddy","Maddy","Maddy","Nick","Maddy","Maddy","Maddy","Maddy","Maddy","Maddy","Maddy","Arta","Nick","Maddy","Maddy","Maddy","Maddy","Maddy","Louise","Nick","Nick","Maddy","Maddy","Louise","Maddy","Louise","Maddy","Maddy","Leo","Nick","Leo","Leo","Leo","Josh","Maddy","Leo","Maddy","Maddy","Maddy","Maddy","Maddy","Nick","Nick","Nick","Leo","Maddy","Maddy","Maddy","Nick","Arta","Leo","Nick","Louise","Leo","Leo","Leo","Maddy","Maddy","Maddy","Leo","Arta","Leo","Leo","Leo","Louise","Nick","Josh","Arta","Josh","Josh","Arta","Louise","Arta","Leo","Maddy","Josh","Louise","Leo","Leo","Leo","Leo","Leo","Louise","Nick","Nick","Josh","Nick","Louise","Leo","Leo","Leo","Leo","Leo","Nick","Leo","Nick","Arta","Nick","Nick","Nick","Arta","Louise","Nick","Maddy","Louise","Nick","Maddy","Maddy","Leo","Nick","Maddy","Maddy","Leo","Louise","Josh","Leo","Leo","Leo","Maddy","Leo","Leo","Maddy","Nick","Louise","Nick","Arta","Maddy","Maddy","Josh","Maddy","Maddy","Leo","Nick","Leo","Maddy","Louise","Maddy","Nick","Nick","Nick","Maddy","Maddy","Nick","Nick","Maddy","Maddy","Louise","Leo","Maddy","Maddy","Louise","Louise","Leo","Louise","Leo","Leo","Leo","Nick","Nick","Louise","Louise","Arta","Maddy","Maddy","Nick","Maddy","Leo","Arta","Nick","Maddy","Maddy","Louise","Louise","Louise","Arta","Leo","Nick","Leo","Louise","Nick","Maddy","Leo","Leo","Leo","Nick","Leo","Leo","Maddy","Josh","Nick","Leo","Nick","Louise","Nick","Louise","Leo","Nick","Arta","Louise"],"help_request":["Review of SCZD vs NTC DE models","Options for creating links to use in manuscripts","Loupe Browser re-analizing and advanced options","Reference letter request","Inspect multimome objects with Browse Loupe","Support with Config Files","Updating ranger_metrics information","Avoid using jsonlite when editing spaceranger json files","Write a script to manually edit spaceranger json files","Debugging spaceranger count","Understanding a spaceranger error","access bulk RNAseq from samples cataloged in the Lieber-internal bulk rnaseq database","debugging samui script for HPC HE data","Question about the correct number of reads in sequencing for scRNAseq and scATACseq multiome strategy","debugging samui script for HPC HE data","LIBD Data Portal for BSP1 and BSP2 DLPFC SCZD vs NTC data","Cell Ranger-ARC html report","slurm job loops","Polishing details on how to ask for help effectively","GEX and ATAC clustering","More troubleshooting misaligned image","Cg_Hb_projector_profiling progress update","Overview of use cases for JHPCE disk usage data over time","snRNA-seq data analysis steps overview","Cloud computing vs JHPCE","Troubleshooting misaligned image in Samui","Advice prior to a faculty interview","Plotting to a multi-page PDF with 3 plots per page","Derive code for how to merge ERCCsumLogErr values across sequencing lanes for a given sample","Help identifying differences between two RSE objects containing data from the same project","I tried to apply \"qSV_model_DE_analysis.R\" in lithium-exposed samples ( from Dpseq data), but I was confused by some of the code, especially \"run_DE.R.\"","Installing cellprofiler at JHPCE","spatialDLPFC and deconvolution-benchmark project details on how snRNA-seq data was used for each of them","JHPCE login questions","quick big fix. Its the same eQTL code -- just one header in a file that's always missing","Grant related discussion","Adding notes to Samui","Cg_Hb_projector_profiling progress update","Better algorithm for dealing with overlapping images in Samui","Setting up SSH keypair for passwordless login to JHPCE","Vistoseg parameter tuning","Feedback on the nascentRNA project","Deconvolution","How to extract normalized expression values with limma","Prospective JHU PhD student","Cg_Hb_projector_profiling progress update","Cg_Hb_projector_profiling progress update","qsvaR bug report resolution","Adding phenotype data to coldata","Adding packages to a custom R module","Rownames not well formatted","Cg_Hb_projector_profiling progress update","Discuss Seurat, cell type markers, DE","Follow-up: removing large files from version control","Troubleshooting GitHub authentication issues","Removing large files from version control","Working with SLURM","Career development","Running python on cluster","Creating a module","Permutation analysis","making a corrplot with a lot of data points .","Review response to qsvaR bug report from last year","Follow up on stitching for Samui","Maybe divinding them by sample name may help.","when I try to rerun R dreamlet pseudobulk on the new JHPCE, I keep getting errors... (provided code example)","Reviewing analysis using qsvaR package","Reviewing updates to the qsvaR package","Creating stitched image in Samui from ImageJ estimates","Investigating visual problems in Samui image","Help adding phenotype info to colData","Investigating R package installation conflicts","Investigating error in qSVA funtion call","Fix misleading getDegTx error message","Performing 'jaffelab::s' on large tibble efficiently","Follow up on stitching for Samui","Creating stitched image in Samui from ImageJ estimates","JHPCE R modules","Regarding the transcriptome assay from the multiome approach, I have selected canonical gene markers from the different samples; Moving further, I would like to integrate to the analysis external gene-markers shared by S.Page. I am looking the best way to do this.","CV writing advice","Help with qSVA overview schematic","Overview of image stitching process","Cg_Hb_projector_profiling progress update","Review how to respond to a user's bug report","Job application support","I would like to discuss with you the correlation issue related to the cellular_component model of the qsvaR package and get your feedback.","Job application support","Cg_Hb_projector_profiling progress update","Modifying a global-scope variable inside a function","Review replication analysis R script","Cg_Hb_projector_profiling project overview","Review Visium SPG AD blog post","Issues linking repo to github","Interpreting decision tree and spot-deconvo plots","Issues with tensorqtl module","Interaction modeling with limma-voom","snRNA-seq OSCA-based analysis","Review replication analysis R script","Debugging a ranger_metrics R script","Interpretation of \"replication\" plots and performance of QSVA models","Hi! We would love an opinion on our (now cell-type assigned) single nuclei rna-seq data, i.e. how they look - if anything is off, advice for improvement...","Example datasets and getting started with data science","Review replication analysis scripts for the qsvaR project","Help migrating to SLURM setup","snRNA-seq OSCA-based analysis","Nextflow issue when running SPEAQeasy","Setting up and running SPEAQeasy on a new cluster","Installing SPEAQeasy and setting up main script","Troubleshooting a SPEAQeasy error","qsvaR paper planning","Prepare expression bed files for tensorQTL","sbatch issue with SPEAQeasy","Interpreting and refactoring \"concordance\" formulas in QSVA results","snRNA-seq OSCA-based analysis","Prepare covariate input files for tensorQTL","Refactoring QSVA code","snRNA-seq OSCA-based analysis","Interpretation of results plotted in inflection plots of Cellranger-arc data sets (GEX assays). For example, how do you know which genes are representing neurons or glia?","Dropping extra factor levels","Onboarding questions for a snRNA-seq dataset","Confidence metric in spot deconvo model","Array jobs over variables with SLURM","SGE vs. SLURM R package conflicts","help get normalized single cell data from a single cell rse for downstream eQTL","snRNA-seq training","How to interpret terms on an interaction model","Guidance on expression heatmaps","Discuss deconvolution for some future work","help get normalized single cell data from a single cell rse for downstream eQTL","Customizing the home page for spatialLIBD apps","Overview of the stitching process for split Visium samples","snRNA-seq training","Social media ideas","SLURM array jobs and X11","snRNAseq celltype grouping in DLPFC data","Creating documentation sites with bookdown","Displaying info about cells, not spots in Samui","Interpreting spot deconvolution results","I performed the whole analysis as previously established and also made an alternative version based on knowledge of OSCA book for single nuclei (with adaptive thresholds for QC and less tolerance for mito UMIs). Would you help me checking my QC thresholds and seeing if they make sense?","snRNA-seq training","ranger_metrics","Missing genes in Samui","My coding set up","Recommendations for learning more data science skills","Misc spot deconvolution questions","Getting started with snRNA-seq","recount 3 permissions","Using modules at SLURM and configuring bashrc","writing tweets efficiently","Introduction to what we do as a team and future plans","Using Samui from start to finish","Running DeconvoBuddies on the Mathy's data","Adding ceil function to splitSlide to split arrays covering fiducials","Explain the SPE organization and subsetting an SPE","Normalizing spot deconvolution results to answer biological questions of interest","memory error with VistoSeg::InFormStitch function","How to check if a visium slide data is already entered into REDCap","Guidance on reference data for deconvolution and snRNA-seq analysis","insert offset in VistoSeg::splitSlide_IF","Quantifying spatial promiscuity of genes across clusters","Feedback on an R package + GO/KEGG enrichment analyses","Feeback on spatial data + code organization","Feedback on workshop vignette","Evaluate a potential external contractor","How to best document what skills I can perform or need to learn","Need help debugging iSEE app errors/warnings","Practice BioC talk","Troubleshooting GHA workflow errors when building workshop","REDCap build table error","NM pigement segmentation overview for LC samples","Overview of image processing Visium and VSPG","Input on a course design","Feedback on responding to reviewers","Spot deconvolution workflow overview","Loading large files in R","tif2mat conversion","Sharing conda environment","Performing QSVA with genes or with different annotation","mat2tif conversion","0% Vs 5% overlap and repeated signal issues in VSPG images","How to make a project documentation website","clusterProfiler, set.seed, and BiocParallel questions","Resolving a permissions issue","About empty drops run with GEX cellranger-arc data","How to make dynamic error messages","How to make a new Globus endpoint for a publication.","How to filter an SCE object prior to making an iSEE app","How to debug an R package function from a user's report","1/Learn about the roxygen2 syntax for examples. 2/Run devtools::document() like in dev/04_update.R (maybe also auto-style code). 3/Edit the news.md file + bump version on the DESCRIPTION (and change the date). 4/Learn about how to sync her local repo with steps like http://contributions.bioconductor.org/git-version-control.html#new-package-workflow (aka register to https://git.bioconductor.org/BiocCredentials/login/?next=/BiocCredentials/ like @Daianna Gonz√°lez + add the upstream remote + git push to both upstream and origin).","I created this github repo: https://github.com/LieberInstitute/neurovasculature-pilot/tree/main. Also, integrated it with slack, but instead of integrating the main branch, I linked the devel branch, and I am unable to see the commits that I make on slack. They are updated on github, though. So, I need help with the integration.","Displaying spot coordinates in Samui","Using the Samui API to display data locally","Options for cell type specific eQTLs","Options for cell type specific eQTLs","Overview of what we do as a team and outreach options","Troubleshooting SPEAQeasy error","Ignoring a directory with gitignore","Debugging rewritten code","Troubleshooting unexpected SPEAQeasy output","Interpreting spot-deconvolution plots","Subsetting RSE by chromosome","SPEAQeasy defaults","Using Samui's API to visualize spatial data","Creating many configuration files differing by one variable","Merging two dataframes","Renaming many files by a pattern","Help with SSH keys with GitHub","Overfitting in qSVA","Interpreting SPEAQeasy metrics","Creating a blog post on the rstats website","Ensembl IDs vs. gene symbols with cell2location","Modules and reproducible software set-up for python","Properly tracking git contributions","When and how to merge git branches","code review","user had problem using import() function in R","Information needed for Samui","Feedback the SmokingMouse Project websites","Properly transferring from dcl01 to dcs05,dcs04","Git-LFS error messages","Alternatives for RNA-seq analysis without replicates","Configuring X11 for plotting at JHPCE","Using the qsvaR with the new annotation","Creating modules at JHPCE","tensorQTL code overview","seurat + signac","Check some issues with GitHub Actions","Introduction to CellRanger RNAseq ATAC analysis","Using git and RStudio to work with R Markdown files","SPEAQeasy error","Understanding API authentication","the user asked how to change the dot size in the plot_pca( )","user had question about generating rarefaction curves in R","tensorQTL pipeline overview","Quantifying a single gene not in SPEAQeasy's default reference","Updating spatialLIBD::read10xVisiumAnalysis","Verify GitHub ssh key setup and team overview","Debugging spatialLIBD::read10xVisiumAnalysis","Overview on \"R/Bioconductor-powered Team Data Science\"","Finalizing genotyping + expression script","Understanding the template project repository","Help resolve JHPCE permission issues on the lifespan_hpc_spatial project","Debug spaceranger 2023.0208.0 (likely an alpha for 2.0.1) with SpatialExperiment 1.8.0","How to write a tweetorial","read10xVisium() in version 1.7.1 (and now 1.8.0) creates a dangerous silent error by mismatching tissue positions across multiple samples","Version controlling renamed directories","Miscellaneous R questions","Deconvolu code where the bisque doesnt show any differences","How to write a tweetorial","Creating marker heatmaps with pseudobulk data","Setting up .bashrc file for folder access permission for DLPFC-SCZ project","Using a gitignore to not version control specific files","I have questions on GitHub and submitting SGE jobs.","Moving forward with multiple cluster methods","Adding figures to OneDrive and google docs with CrossReference","Organizing figure components for Illustrator","Lining up gene expression and genotyping data","Setting up GitHub SSH keys","Managing software dependencies with Nextflow","Nextflow with Conda and Github actions","SPEAQeasy error","GitHub for a beginner","snRNA-seq GLM PCA","Integrating R scripts with Nextflow","Making custom R script work with Netxflow","snRNA-seq QC","Getting started with snRNA-seq","Plotting Gene Expression with SCE","Help identify the source of unexpected NA values","Help building SCE with downloaded data","Adapting spot-deconvolution code to new data","Discuss Mean Ratio marker gene finding","Deconvolution study with rnaseq bulk data","resolving a merge conflict","Identifying the source of an error","coloring plots by a continuous variable","Understanding PCA for Habenula project","Review deconvolition script for qSVA","wrangling and reformatting expression data","preprocessing and reformatting genotype and expression data","organizing plots and output files","understanding linear mixed-effects models","help submitting jobs with R parallelization in jhpce. For instance, I would like to use scater::runPCA with the BPPARAM=BiocParallel::MulticoreParam(8)) parameter","Introduction to LIBD and our team","formatting genotype data into a matrix for each gene","I have this single nucleus RNA sequencing experiment with the mouse LS that Matt analyzed and we have a final single cell experiment, but I don't know how to do anything with this data set and want to learn how to plot heatmaps and possibly other things for this data set. Can we maybe schedule an hour to start this learning process and then I can schedule future time as needed?","Check the DEG heatmaps","Review TMM paper","csvs that I want to merge together","Lifting results between hg19 and hg38","GO and KEGG boxplots with the modifications you suggested","QC approach for hebenula","How to check technical effects for RNAseq data","Reviewing GO results and dotplots","Working on making a violin plot for just one gene. So by brain region rather than gene type","Question on the feasibility of some approach with HALO (for the AD project)","Overview on using GitHub for Matlab projects","Setting up SSH keys between login and compute nodes","Setting up SSH keys for JHPCE","Introduction to DSgs, R, Bioconductor, JHPCE","Interaction between .bashrc and 'module load' in 'qsub'd scripts","Modifying a shell script to segment a VCF by genomic region","Setting up SPEAQeasy at JHPCE","Code to threshold small sized nuclei from the NaCC visium images","How to organize OneDrive files","How can I interpret the y-axis from `plotExplanatoryVariables()`?","Use of git from multiple local locations","Moving and correctly accessing HDF5-backed SummarizedExperiments","DE analysis introduction","Git push help","check the models and the results I have obtained so far from the DEA","DE analysis introduction","Check the model design of the smoking mouse project","VNS error","Strategies to pick threshold for cell size in segmentations","open multichannel image in imagej","code to ectract x and y coordinates of randomnly slected inform tiles for PNN project","Review results of Varaince Parition for smoking mouse","Pick randomn tiles for the PNN project","go over pseudobulk code","Help with spatialLIBD colors setup","Extracting final output into table","CUDA issues with cellpose module","QC update for smoking mouse","R, RStudio, git, SSH key setup","Addressing SPEAQeasy error messages","LIBD Data Science","help row annotation colors of a pheatmap","Subsetting eQTL results with in risk loci","Setting up SPEAQeasy","Data science coordination","QC update for smoking mouse","Question about how 'mitoRate' is calculated in SPEAQeasy","Help troubleshooting error messge when running BayesSpace","zellkonverter error and guidance using SpaGCN","Review progress on smoking mouse DE project","pulling in external changes to a git repository","Adjusting thresholds for event detection using CaPTure","help installing 'jaffelab'","help with vis_grid_clus()","JHPCE Github setup","Help troubleshooting shiny app that crashes on gene interactive tab","getting total UMI's for each sample","Github setup","Revisited the sc analysis and could use some help","explain spaceranger output files","Tools and resources related to recount2 and recount3","I might need a review of how to go about selecting marker genes for the spatialDLPFC data. Also, I'm curious what code you have for making heatmaps showing marker gene expression across cell types.","Single cell rna seq analysis","I have a mouse single-cell RNAseq data that I am aiming to identify cell types in different clusters.","Debug `spatialLIBD::sig_genes_extract_all()`.","How to debug an R package code","Help with BayesSpace clustering","Data science organization.","MatLab questions.","Subsetting spe object for shiny app","Brainomics meeting with collaborators","Color bar for heatmap","Discussion on how to recruit members","VisiumIF DLPFC building spe","OneDrive","BrainSeq Phase II development browser","make modified tissue_positions_list from DLPFC IF samples for running spotsopcheck on mock PNN visium sample","Help Deploying Shiny App for DLPFC_IF","Cross Reference help","Career development","Need some help to make a plot","Guide to creating a module at JHPCE","How to make a tweetorial for a pre-print","File organization for a paper submission","creating a nextflow pipeline and supporting docker","In promptu meeting","Help with purrr loops for WGCNA funtions","Grant proposal feedback","help with html output of spaceranger","CV feedback","Cover letter feedback","Debugged code from https://github.com/LieberInstitute/spatialLIBD/issues/33","Brainomics meeting with collaborators","How to delete large files from GitHub history","help with read10xVisiumWrapper()","macOS and JHPCE laptop set","writing papers in google docs and setting up links for figures","Hey Maddy, I need something called \"spot size\"  on the 12 published DLPFC samples","https://github.com/LieberInstitute/VistoSeg/issues/12","applying size threshold to vistoseg outputs to get rid of smaller regions that are not nuclei","Debugging JHPCE configuration files","I'd like to consult with Louise about how to make a spatial plot in R using a data set from the HALO FISH-IF module.","directed the person to the VistoSeg website for steps to run VistoSeg","using toptable and duplicate correlation","help with clustering portion of th build_spe","Using the \"redstone\" software","help with build_spe","Input files for custom annotation with SPEAQeasy","Advice on how to increase code coverate with testthat","Help run trial shiny app deployment","set up git repo for long read project","The authenticity of host 'github.com'","Assistance with analyzing spatial data","Follow-up on custom annotation with SPEAQeasy","Plan how to make spatial registration R functions","Adding lines to a GTF for use in SPEAQeasy with custom annotation","Clarifying what role deconvolution can play on LCM-seq","TODO","Error message running SPEAQeasy","How to log in to JHU-controlled laptop","Setting up passwordless login","VisiumIF loupe alignment","Using custom annotation with SPEAQeasy","Masking of blood vessels from PNN channel in PNN project","How are InForm tiles arranged","Using the caracol queue","How to guide others learning DS","Understand the covariates used in scRNA-seq cell type analysis for a project we are doing with striatal organoids","\"Megadepth usage to add a new annotation to hg38 file\"","How to build a cov_rse object using qSVA v1","Debradation matrix for LIBD riboZ","Work through qsvaR vignette together","Generating bigWigs with SPEAQeasy","Data type change of image data when loading into Python","WGBS data analysis overview","Help building for loops to analyze WGCNA ouput data","I would like to learn how you ran tensorQTL on the MDD data, maybe with using the GPU nodes for it.","contrast adjustment, intensity normalization, channel/gene description in PNN images","Ideas for possible future analyses","rename files on JHPCE through command line","JHPCE array jobs and fasthplus","Provide comparison table for different metrics obtained from image meta data in MATLAB, for Visium IF workflow","snRNA-seq and deconvolution overview","Access to the gene info in the rowData of the Summarized Experiment objects to annotate eqtl results","Explain Tangram, scRNAseq count matrix, spe object","Explain the concept of Squidpy paper","tiling and metadata changes in PNN project","explaination on Image metadata, bit depth","Need for rotation of visium array images and its spatial coordinnnate in spe object","Debugging seg fault with tensorqtl module","Using tensorqtl with a GPU","X11 and SSH issues","VistoSegIF pipeline","accessing the 'gpu' node at JHPCE correctly","Transfer data from JHPCE to Neural_Plasticity","Transfer data Neural_Plasticity to JHPCE","JHPCE directory organization, repo linking, making commits","opening python, R, MATLAB console interactive session on JHPCE","Troubleshooting ExprssionSet conversion","glitch in CAPTure tutorial about input data types to a function","Help using BisqueRNA, getting snRNA-seq into correct input format, chosing refrence vs. marker based deconvolution","creating a 'samples.manifest' and choosing trimming settings","Removing smaller objects from VistoSeg segmentations","applying kmeans to CA3 mouse RNAscope data to obtain gene expression levels","JHPCE file permissions and ACLs","running loupe browser for HPC images","countSpots from VistoSeg throwing errors for one DLPFC sample from round4","Resolve issues with comments","using 'list.files' in R","batch processing - dotdotdot, on local machine","error in transferring Visuim HPC data to JHPCE","using awk","Debugging help with rse_tx object","qc general scripts, rnaseq scripts general, ggplot histogram with breaks","We would love an opinion on our (now cell-type assigned) single nuclei rna-seq data, i.e. how they look - if anything is off, advice for improvement...","I am trying to compute DEGs from thefunction registration_stats_enrichment... \nError in limma::makeContrasts(contrasts = z, levels = registration_model) :\nThe levels must by syntactically valid names in R","Transfering files between JHPCE disks","running spotspotcheck for SFN","doubts on VistoSeg","help with git","debugging SPEAQeasy errors","Presentation preparation feedback","help with 'umask' and git","issues with regioprops function in dotdotdot","Fiducial frame enhancement for Visium IF images using preview on MAC","running Loupe browser for mannual alignment","merging FASTQ files automatically via SPEAQeasy","Creating cyberduck bookmarks for JHPCE paths","writing shell script for countNuclei function in VistoSeg","Explaining Github issues, history and version control","Explaining SPE (Spatial Experiment) SCE (Single Cell Experiment) objects","syntax error in shell script","CellProfiler H&E segmentations","Trouble with implementing RNAscope function for humans","Discussion about rerunning FUSION/TWAS with new GWAS with extra attention paid to the number of samples for each brain region","identifying SPEAQeasy error","Error running VistoSeg::spospotcheck (Error in spotspotcheck>start (line 60))","Error while running rnascope_mouse - line 159","parameter tuning for dotdotdot to split the nuclei clusters","improve lipofuscin masking in dotdotdot","dynamically name structures in matlab based on user inputs","using map to cbind a list of six to its corresponding list of lists. The second is a list of 6, each a list of 3.","resuming SPEAQeasy after an issue in the counts step","downloading FASTQ files from SRA","Explain how to merge 2 data tables in R","Explain public keys and the authorized_keys file on JHPCE, removed duplicate public keys for the local machine to resolve ssh key login","Using purrr and map to loop over datasets and apply qsva","resolving issues with multiple keys in the authorized_keys file","Help with JHPCE login issues","explanation of the methods and algorithm of a paper we are reviewing together","steps to resolve image segmentation of LC heated samples round 2","How to write a Twitter thread for a paper","Forming a data frame from data in multiple files","Refining an abstract submission","Git branches","DNA genotyping overview","DNA genotyping overview","Optimizing DAPI segmentation in dotdotdot","Overview of the RNA-seq universe","Explain how awk works","Issues with SUFI","Resolve errors produced by dotdotdot on SUFI output","R Code to create list of files to be processed through VisiumLIBD","Setup aliases in bashrc","debugging SPEAQeasy errors","getting SPEAQeasy running","formatting a LaTeX document and bookdown website","Setting up a team at LIBD","recreating spatstat plots with more features","Helping understand errors from VisiumLIBD","Helping understand errors from VisiumLIBD","Help indexing a possibly corrupt VCF file","Running Genome Studio on n = 200, generating PLINK outputs","How to set up a GitHub actions-powered website","Specifying output logs on SGE","Debugging ggplot code","Planning session for getting up to speed with SpatialExperiment, spatialLIBD, here and ggplot2","How to choose and add a licence to a GitHub repo","Designing an MS-level class","making a list of filenames in a specific folder and subfolder in R","Rectified issues for batch running emma's RNAscope data","create shell script for batch running images on VisiumLIBD","Locating public data at JHPCE + bamtofastq workflow","TWAS questions","How to transfer recount3 data","How to get started with R","Brainstorming session on collaboration","Manhattan plot making and tstatistic","addressing SPEAQeasy error","got mds score for the vcf she presented using plink","FUSION / TWAS Walkthrough","mds for vcfs","checking plots and data structure in clusterprofiler","Havent got my mds yet so making the meeting for Friday","questions about expression_cut and pca plots","Using GenomeStudio","How to prepare for a LIBD seminar part 4","troubleshoot SUFI output and figured it was not compatible with dotdotdot","gene ontology for genesets","Differential Expression help (continuation of 5/20)","How to prepare for a LIBD seminar part 3","Grant application review","PredictDB weights","Help preparing a figure for a grant application","How to prepare for a LIBD seminar part 2","Making plots for DE studies","Making a module","Configuring a parallel series of bash scripts","questions about processing genotype data","Setting up a SPEAQeasy run","Making plots for DE studies","Setting up covr using GitHub actions in an R package","How to get started with computational biology","How to insert a figure using the Oxford Bioinfomratics Overleaf LaTeX template","PredictDB weights","How to prepare for a LIBD seminar","Running SPEAQeasy- follow up","TWAS prep","Constructing an array job from an existing sequential shell script","Setting up FUSION / TWAS for practice","Running SPEAQeasy- follow up","Running SPEAQeasy","Running SPEAQeasy- follow up","Running FUSION / TWAS","JHPCE set up on mac","Running SPEAQeasy","running and trouble shooting VisiumIF image splitting code","JHPCE set up on mac","Running the old RNA-seq pipeline on some SRA data","Resolving issues with running dotdotdot on 20X RNAscope images","runnning Hiplex registration","General guidance on how to learn how to do differential expression analyses","Nextflow overview for work with imputation pipeline","creating logs for issues from running SUFI (spectral unmixing of fluorescent biological images)","steps to troubleshoot issues with stitched multi tile RNAscope images","spatialLIBD support","Need help understanding the sce - single cell object","help doing a power analysis for RNA-seq data","Creating a documentation website using bookdown","Clarification of LIBD support","General advice","running nucleiCount for spatial DLPFC samples","Download SRA data and run SPEAQeasy","Explain the feasibility of an on-demand end user interface for DEG and eQTLs","extracting p-values from linear mixed effect model for Calcium imaging data","Using an LIBD module","Change point size in vis_clust_grid","Download SRA data and run SPEAQeasy","Attempted SPEAQeasy installation on MacOS","visium image segmentation","troubleshooting memory usage for VNS function from VisiumLIBD","get effect size for calcium imaging","split histology images into respective capture areas","x-quartz installation","BSP2 eQTLs with a subset of SNPs","qSVA explanation and code review","How to write a small methods paper","resolve figure generation error in human rnascope script from dotdotdot","Maker finding for cell types in single cell data","image analysis on QuPath","Advice for writing papers","Clarification about how recent CMC data was processed with SPEAQeasy","Our group's set-up and approach for writing papers with google docs, and managing citations","data transfer from LIBD server to JHPCE","Submitting/debugging bash job for dotdotdot on cluster","I need help in: downloading data from SRA to JHPCE, running SPEAQeasy to create rse object (one session will likely not be enough).","I need help with setting up Mobaexterm. Unable to open new terminals and folder permission issues","explaination of Owner/Group/Other permissions for files on JHPCe","download repository from github to cluster and run dotdotdot on cluster","Visualizing RPKM valus for genes in Humna and Primarte data","BSP2 LIBD eQTLs request","JHPCE ssh keypair set up","dotdotdot parameter tuning for Ema's rat RNAscope data","Github ssh key set up","Work on habenula deconvolution and debug the deconvolution package","Help determining what data is available and what can be used for an analysis","group_by bug for get_mean_ratio","Guidance on how to download data from a study in GEO","biocthis macOS GHA bug","Questions related to accessing results from the DG paper by Andrew Jaffe","Correlation values from labeledHeatmap","Help with importing HDF5 file","Resolving issues in phenotype data between genotype, expression, and LIBDpheno","jhpce set up on mac","Inline formula throwing an error for interaction term","run dotdotdot pipeline local computer and cluster","how to open matlab (GUI and command line) on JHPCE","Matching Variants Between Two VCF Objects","how to connect to LIBD servers on windows and mac","Accesing CMC RNA-seq data","Ran Genome Studio, installed PLINK input plugin","ChipQC 'names' attribute [9] must be the same length as the vector [2]'","data transfer from LIBD server to JHPCE","download matlab on local computer through JHU software portal","Help clustering sce data","Using reshape package to transform table for plotting","Help adapting ggplot script for TWAS Z-score","RDavid Error","Differences between recount2's \"bp_length\" and SPEAQeasy's \"Length\"","How does one modify a single value of a field in a seqinfo object?","GitHub actions workflow issue with macOS on R-devel","LC repo organization","Extracting Sequences from Fasta Files with Rsamtools","3D volume extraction in Matlab","Fixed two bugs on recount3 related to locate_url() for the BigWig files.","Fixed a bug on biocthis for release branches","Issue with knitcitations","Help with SpaceRanger","Issue with JHPCE file permissions","recount3 GTEx data access","customize dotdotdot to his data","Learning Lexical Scoping Rules","Changing Row Names of MultiAssayExperiment Assay","exceRpt pipeline counts","issue running spaceranger","Plotly help if possible? I'm trying to export an html file that allows the user to look at each snp/gene on the plot, and I am almost successful. Leo recommended that I talk to you on this matter.","Help running speakeasy on MARCC and JHPCE please","Question: Subset center regions of a set of different length sequences","Ballgown stattest","Question: Cannot Install gmapR Bioconductor Package","copy number variation analysis (CNV)","Help with spatstat"],"public_notes":["I explained why `age > 17` is a must for SCZD vs NTC DGE analyses: we have no SCZD cases with `age <= 17`, so if you don't filter the NTC donors with `age <=17`, you end up basically comparing older (SCZD) vs younger (NTC) donors, which is not what you want to do. We also talked about how in BSP2 there are some HPC (HIPPO) samples from 2 RiboZero protocols (Gold and HMR), how the Gold HPC samples cluster with the DLPFC ones on PC1 vs PC2, and thus why we ended up dropping the 43 Gold HPC samples in previous work (see https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/make_qSVs.R#L62-L69). This information is reflected by `analysis_sczd_casecontrol_hippo` at https://s3.us-east-2.amazonaws.com/libd-brainseq2/methprop_pd.Rdata listed at https://eqtl.brainseq.org/phase2/. Finally, I emphasized that self-reported `race` is not a trustworthy variable and we instead rely on `snpPC`s.","I explained why using something like https://research.libd.org/spatialDLPFC/#interactive-websites or https://research.libd.org/globus/ give us control over links that we might need to update later on, instead of including other less stable links on manuscripts (domains we don't control, tools that might change their URL syntax, etc). I provided links to how to make each option such as https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R and https://github.com/LieberInstitute/globus/blob/98dc479ad2cde0a047cf7a46769a85c32969b7b1/index.Rmd#L8-L37.","In the session, we explored reanalyzing cloupe objects with the Loupe browser for scRNAseq assays. This involved using multiple projections like UMAP and t-SNE, splitting clusters, and filtering data based on UMI count and mitochondrial percentage. While there was interest in analyzing integrated datasets, the Loupe browser currently only supports exploring one Cloupe object at a time.","We discussed the specifics of the [KavliNDI Distinguished Graduate Fellowship](https://kavlijhu.org/funding/4) and revised information that I can use while writing the reference letter (project title, assays proposed, that this involved new software, how long we've worked together, education history of the applicant, etc).","I explained where is located the cloupe output object generated by cell ranger-arc. Then, I explained how to use the tool to explore gene expression and atac assays by separate and also together. I also explained how to inspect clustering and re-label clusters and sub clusters; plus how to check DEG and look at their gene distributions. Finally, I explained how to re-run an analysis and use the most classic formats to save the analysis performed.","I explained how to prepare the configuration files for working R and JHPCE.  I showed all the section that need to completed plus the configuration to access JHPCE, with links pointing specifically to Windows system. I explained the purpose of each section in the chapter 17 of https://lcolladotor.github.io/bioc_team_ds/config-files.html. I also provided the updated link (https://jhpce.jhu.edu/access/ssh/)  for the JHPCE configuration due the past link is broken because of the change to the new web version. \nFinally, I provided the most recent video made by Leo  https://youtu.be/qtn3eTwFlZI  (minuto 17:50) to do this specific task for windows OS.","We added the `LFF_spatial_ERC` and `LFF_spatial_LC` datasets to https://github.com/lieberinstitute/ranger_metrics. This involved debugging the code a bit to handle updates on the Excel sheet files describing samples prior to being sent for sequencing.","We found out that `spaceranger count` did not like the JSON files we created with `jsonlite::write_json()` so instead, we edited them manually using `gsub()` and `read/writeLines()` as documented [here](https://github.com/LieberInstitute/LFF_spatial_ERC/commit/bbf810ad39012ba6ee4e0403063bf83bf844d007). Note that we had to manuallly revert the JSON files to their original state as documented [here](https://github.com/LieberInstitute/LFF_spatial_ERC/commit/79ed5479d502cd41d51f3aa8ee1467bd8100a08a).","We wrote a script to manually update the json files and add the Visium Slide ID as well as the capture area. This will enable us to not have to use `--unknown-slide=visium-1` https://github.com/LieberInstitute/LFF_spatial_ERC/blob/6e85c05ce9f531c555569e616f8d5c351b427b4f/code/01_spaceranger/spaceranger_SLURM.sh#L63 that 10x Genomics support had recommended using. Doing so led to an error at https://github.com/LieberInstitute/LFF_spatial_ERC/blob/b5b92a8bb9744ee424ec4a0642ae692423a91d56/code/01_spaceranger/logs/spaceranger_17.txt#L22 which stated that `--slide` cannot be used as the same time as `--unknown-slide`. This is a workaround that was previously manually implemented at https://github.com/LieberInstitute/Habenula_Visium. Since we have lots of samples, we wrote a script to avoid introducing copy-paste errors.","The issue on this [script](https://github.com/LieberInstitute/LFF_spatial_ERC/blob/e913ff28fffbb17335d3844d312ec6486a7b182f/code/01_spaceranger/spaceranger_SLURM.sh#L55) and related ones is that there is a missing `\\ ` (note the space) before the `#` comment symbol. Thus the chained command is breaking into two, and the second half is not being recognized as options to the first half.","We worked on the following error message from `spaceranger count`: _ERROR: We detected a mixture of different R1 lengths ([26-28]), which breaks assumptions in how UMIs are tabulated and corrected. To process these data, you will need to truncate to the shortest observed R1 length by providing 26 to the --r1-length argument if are running count/vdj, or via the r1-length parameter in each of the following tables of your multi config CSV if you are running multi: [gene-expression]_. We inspected the soft links for the FASTQ files and used `zcat fastq.gz | head` to inspect the first few lines of FASTQ files. We confirmed that the `R1` FASTQs were indeed of varying lenghts and will attempt a new `spaceranger count` run using `--r1-length = 26` though it's likely that we'll next get an error with the `R2` FASTQ files as we observed that those files also have varying lengths.","Bernie was able to access the data throught the LIBD dataportal, I also pointed him to the combined LIBD rse_gene in brain_swap (which I think might be the source of the portal data...would need to check)","Updated script by commenting out unwanted lines and providing correct paths to the input files","I explained what the key metrics are for each assay in a Multiome experiment from 10x scRNAseq and scATACseq data sets. I showed several HTML reports of data sets processed with cell ranger-arc to illustrate different scenarios targeting different numbers of reads for each assay, to help you understand the relevance of quality beyond the number of reads in sequencing. I showed how with a dataset with a moderate number of good quality reads you can perform a good analysis, while having many reads with compromised quality it is possible to have more challenges during the analysis. Finally, I explained the minimum recommended number and that it would be a good idea to search the literature for technical data from other experiments similar to his as a reference.","Cross checked the script and rectified filename/samplename processing","I explained that BSP1 is polyA, aligned vs hg19, older annotation, aligned with TopHat2 (an older aligner). BSP2 is RiboZeroGold for DLPFC and RiboZeroHMR for HIPPO (with 43 RiboZeroGold samples that were dropped from the DE analysis), aligned vs hg38, against Gencode v25 (from July 2016), aligned with HISAT2. I also explained the naming structure of a Supplementary Table for BSP1's paper where `adj`, `qSVA`, and `PCA` are labels specifying whether it was a model `adj`usting for covariates, whether it adjusts for qSVs using `qSVA` (the better model to use), or adjusting for gene-level expression PCs (`PCA`). These names are related to the [`qSVA` 2017 paper](https://www.pnas.org/doi/full/10.1073/pnas.1617384114) Figure 2. I also explained how in the BSP2 paper we did re-align the BSP1 data against hg38, though there is no Supplementary Table showing which of the 243 DLPFC DEGs replicated in BSP1 (though the data does exist since it was used to make Figure S28 and S29). We would need another DSgs or two to actually locate the data and construct an CSV file or table that others can use if they are interested in replication of BSP2 results using BSP1 and/or CommonMind Consortium (CMC) Phase 1 data (that was aligned vs hg19 with TopHat2). I also used [this slide](https://speakerdeck.com/lcolladotor/libd-ds-tldr?slide=56) to explain all the pieces that go into a DEG model and why the visualizations provided by the LIBD Data Portal (boxplots for SCZD vs NTC) are just a visual guide using normalized gene expression data and not the full answer (as the LIBD Data Portal doesn't use the full details we include in a DEG model).","I explained in detail the metrics reported by cellranger arc and helped you download a multiome public sample to run the pipeline.","Cynthia re-organized her code to run with a shell script from slurm_jobs. WE solved a bug with args index, and talked about adding catches for unexpected inputs. Future goal is to use map to replace for loops when handling list of data objects","We disscussed specific details on how to ask for help. For example, it's ok to ask those around you in person but you should also ask on the `#libd_helpdesk` channel (and update the channel if someone else gave you an answer). That is, don't assume that only those around you (in person) know the answer. We went over details on how to preview GitHub messages (to verify that they are correctly formatted), why GitHub issues are useful to the authors, what information is needed for others to help you answer a question (R session info, `reprex::reprex()` output, link to code on a public GitHub repo; private is ok when asking internally for help). We also discussed the overall idea of why learning how to make an R package will be useful to then improve one's tooltik for debugging R packages made by others.","I explained the previous steps to perform clustering with GEX and ATAC 10x multiome data using Seurat and Signac.","We interactively ran the python code for reading in ImageJ transforms to rule out an issue where the wrong data was read in. Ultimately, we couldn't find issues with the data, but the Samui full-res images still seemed misaligned relative to the low-res images","We went over how to update a GitHub remote following [information from the documentation](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories), finishing stylistic design touches on an `iSEE` app, and setting up a new repository for the next project.","We discussed details of the use cases I have in mind for how to search and visualize the JHPCE disk usage data over time and across different JHPCE paths. This information could be useful in forecasting data growth for individual PIs or projects, identifying who has exceeded their storage allocation, identifying static projects that could potentially can be moved off JHPCE / deleted / or moved to non-production disk space.","We discussed the general overview of snRNA-seq data analysis steps and how the code at [`Habenula_Pilot`](https://github.com/LieberInstitute/Habenula_Pilot) could be a good starting place. I also recommended signing up for a Data Science guidance session with Louise as she is quite familiar with the code in that repository prior to diving into it.","I explained details about how data is organized at JHPCE, data sharing limitations, and several suggestions we have been working on that could be useful to implement at a larger scale across LIBD teams.","I pointed to suspicious areas of code where dataset-specific regexs were originally used, though the regexs looked good for his data as well. We decided to interactively walk line by line at a later date","Discussed collaborations, personalities, and other details of the work environment with some of our collaborators with whom this person was going to interview for a job with.","I suggested building a list() where each element was a 3-piece plot (combined with a function like cowplot::plot_grid), then using print(plot_list)","See https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1710454284251579 for the full code details. We might add this to solution to https://github.com/LieberInstitute/jaffelab/blob/devel/R/merge_rse_metrics.R, although it's only an approximation as you need to go back to the raw ERCC output files (typically generated with `kallisto`) to actually compute the ERCCsumLogErr accurately across different sequencing lanes for a given sample.","We have RSE objects from the LIBD Data Portal (or those that helped build that portal) as well as RSE objects made prior to the LIBD Data Portal. They have some differences, mostly on the `colData()` slot. Note how https://github.com/LieberInstitute/jaffelab/blob/21e65745a5b0852fc13ebdb5e5d23c6f1f66010b/R/merge_rse_metrics.R#L83-L97 and related code merges `colData()` information for samples that were sequenced in more than one lane and whose FASTQ files were not merged prior to aligning with `SPEAQeasy`.","I walked throught the run_DE function I wrote for MDDseq (a wrapper for the limma-voom) pipline. I pointed out what parts of the MDDseq code he could adapt and simplify for his use case (its less complicated than the many DE runs we completed for MDDseq)","I suggested using conda after pip failed (missing system dependencies). Ultimately, my troubleshooting was enough to just build a module for everyone","I explained details about the spatialDLPFC and deconvolution-benchmark papers. I also went over how we addressed feedback from the editors for the spatialDLPFC paper.","Explained how to access authorized keys and exit JHPCE","Fernando was having some conectivity issues with JHPCE, so we didn't get a full bug fix in I suggested some de-buggin strategies  [Update 3/15] We met again breifly to confirm the functions were working as intended. Initial issue was caused by un-intentinally overwriting the function","We discussed how to address feedback from a recent grant application, how our teams can get to know each other better, cleared up misunderstandings related to my team's role as well as the DSgs system, and decided next steps to respond to the grant reviewers.","I described how a markdown of notes could be added using the Samui python API, though this is not how I had done it in a previous project ( I was unable to trace the original method I used)","We discussed more details about `iSEE` and shinyapps.io deployment details, such as memory settings and other configuration needed to actually customize and share an `iSEE` app. We also asked our collaborator for the list of genes they want to have plotted outside of `iSEE`.","I helped implement an algorithm taking the minimum pixel intensity in case of overlapping images to display an H&E image in Samui","I diagnosed several potential issues, ultimately missing one detail that was the culprit: permissions were too open on some .ssh files on her laptop","Explained what parts of VNS and refineVNS can be tuned to get appropriate segmentations for Nac project","I sent a long and detailed email explaining different components of the LIBD BrainSEQ projects, background information on several covariates we typically have used in DEG models such as RIN and age at time of death, as well as other ones we have not used such as postmortem interval (PMI). I also explained the DSgs system and how it could be used by our JHU collaborators.","Giovanna and Jisu are working with some bulk RNA-seq data as a some pilot data in a new brain region prior to preforming saptail. They want to use deconvolution to help QC the samples, selecting for good disections. I walked them through our current recomndations for deconolition to help with this goal.","We went over internal code from the limma package using https://code.bioconductor.org/, particularly for the `voom()` function. Doing so, we discussed that's actually returned in the `voom()$E` slot. We also went over how to model a ratio using a linear regression model.","We went topics such as live in Baltimore, research at JHU, overview of the different JHU campi, how to compare PhD programs across universities, and other questions this future PhD student had.","Discussed feedback from collaborators and detailed next steps to wrap up this project.","Went over recent updated plots and results, as well as slides for a presentation with our collaborators for 2024-02-21.","We discussed the updates implemented in qsvaR to handle input with ENSEMBL transcript IDs instead of GENCODE ones, updates to automatically detect if the input is at the gene level instead of the transcript level, how to check for correlation between qSVs and covariates, among other related issues.","I described my typical strategy of using left_join by sample ID, as well as ensuring rownames don't get lost from conversion to a tibble (thus dropping colnames from the SummarizedExperiment unintentionally)","I noted that the r_nac module was built similarly to conda_R, where the user doesn't have to do anything special to install more packages. We ultimately traced the issue to permissions problems on files created as part of the proper use of the module by other users, likely due to problematic 'umask' settings","Sreya went through an error issue in qsvaR due to rownames not well formatted. The error message was misleading. This problem has been fixed in the latest updates of the package. I advised Sreya to get the latest version of the package.","Discussed updates to violing plots and how to organize result files prior to sharing them with our collaborators.","I showed him to the 2 approaches available in Seurat to perform DGE, including the FindMarkers()¬†function to find the genes that are different between control and treatments. I also show him code to perform quality controls in the main covariates related with these data-type. I shared with him consulted resources and links with public-code for his reference.","I ultimately recommended 'git reset --soft' to the commit ID before the recent mistaken addition of the large files","I advised using the SSH address (not HTTPS) for the 'git remote', ensuring the 'git config' email matches the primary GitHub email, and using the up-to-date SSH key (not RSA, which is no longer supported/ recommended)","We explored options: BFG cleaner, git rebase and git reset --soft and discussed which was best for her situation","I described the use of 'srun' (including with X11) vs. 'sbatch', and generally recommended 'sbatch' where possible to document how analyses were run/ ensure reproducibility","Caught up and talked about career development next steps.","I mostly focused on my setup using VS code with its remote SSH extension, and custom keyboard shortcuts for running python code interactively","I mostly confirmed the general process he had started for a particular module made from software packages as a precompiled binary","Went over the details of how to compute a permutation analysis with R.","Sreya wants to compare the correlation of 5 transcipts with 2k degradation transcripts. She had the data in the right format but needed help appllying the cor function to the table using group_by. Corrrplot was not the correct function for what she wanted to achive here.","Clarified updated automatic checks and information for users who attempt to use qsvaR with gene-level data instead of transcript-level data.","I described how code might change when using IF images rather than H&E, for which the code was originally developed","Michael and Peter are working on a BPD vs. MDD DE analysis. They were getting more DEGs than were reported in an early version of the MDDseq project. I could not find the code for the data they were refrencing, with out the code it will be impossible to diagnosis what is differnt in the two analysis. I recomended they use the more up to date data from MDDseq, and pointed them to the correct data & scripts.","Tricia and Fernando are re-running serval analysis they previously had working on the sge cluster, with the update to slurm they are now having issues. They are also having differnt errors in the same scrips + data. This could be a multi-facet issue, but firstly sounds like a R package version issue. I advisded the first step should be to have the same set-up, work in the same version of R and the same version (most updated) packages. See what errors persist and go from there. I also recomended Frenando start using git repos and version control with his team to help keep projects organized and consistent.","During the meeting Sreya showed me her progress on the analysis she's doing taking the 2976 txs from the cell-component model and she wanted to compute the correlation between those and selected transcripts she's interested in. She thinks those transcripts are involved in the degradation process. We couldn't find those transcripts as part of the 2976 txs from the cell-component model.","We checked udpates to the `getDegTx()` and `DEqual()` functions, as well as discussed the response to a bug report from last year.","I described how to adapt a script I wrote for NAc to her dataset, where the goal was simply to display a stitched-together image in Samui","I traced the issue to the use of 8-bit integers, where an overflow occured during normal addition of pixel values (the solution was to use floats while computing the mean, and then convert back to uint8)","I described how I typically use dplyr::left_join, as well as how to add back missing rownames to the colData (or colnames of the SummarizedExperiment)","I described the conflict between SGE and SLURM R packages, as well as identified lock files preventing re-installation of problematic packages","We traced the offending line of R code in the qsvar package and I described how unit testing could identify problematic function arguments and give more informative error messages","I met with Sreya because she's having an issue with the getDegTx function from the qsvaR package. I tried to solve the issue but I wasn't successful so I asked Sreya to give me more time and access to the rse object to have a more deeper look to understand the issue. After using her data I realized that the error message was misleading  and improved the function and unit tests.","I noted that the function could be applied before the data was converted to long format, where it was impractically large to work with","We identified issues causing incorrect coloring in the Samui outputs","I described how to adapt a script I wrote for NAc to his dataset, where the goal was simply to display a stitched-together image in Samui","I explained the different modules such as `module load conda_R/4.3.x` versus `conda_R/4.3`, how each is tied to a different version of Bioconductor. I also linked to https://youtu.be/D6t2GR7xaTE and https://docs.google.com/document/d/1DD9MUI2e3jp5S8C3PWRulZ4jwbXo94D3UTCQNJl_-Qo/edit?usp=sharing from the LIBD rstats club.","Cynthia and Hedia are starting the annotation/cluster integration of the multiome data. The data is currently clusted indiviudally by sample, sn, ATAC only, and combined sn+ATAC. findMarker has be used to calc marker stats for each cluster in each sample (there is a ton of data to digest). I am unfamilar with the sample independendt cluster, consered marker gene integration from seurat. But I gave advice to check the current clusters for hihg expression of any of the known marker genes to see if we're picking up the expected biological signal, and do some prelimary annotations as a sanity check. Also to keep following the workflow and see if the conserved marker genes match with any of the know markers. Also to start recoring annotations for the clusters in a csv file for easy editing by hand as well as easy reading in to R","Provided feedback on CV writing for data science job applications.","I advised Hedia on how to adress some issues with the qSVA overview plot, both minor issues like aliginig and resising objects with biorender tools, and major rearngment of the table and DE application part.","I described the workflow for stitching Visium capture areas with Samui, and various Samui API functions like 'add_chunked_feature' that are helpful","We discussed the [GLM-PCA paper](https://doi.org/10.1186/s13059-019-1861-6) and decided to use GLM-PCA instead of logcounts by `quickCluster()` + `calculateSumFactors()` as described at [the CDSB 2021 snRNA-seq workshop](https://comunidadbioinfo.github.io/cdsb2021_scRNAseq/normalizaci%C3%B3n-de-datos.html#normalizaci%C3%B3n-por-decircunvoluci%C3%B3n-deconvolution). We also discussed the development version of `Harmony`, in particular with the `lambda = NULL` setting.","We went over how to build a reprex for a user's bug report, how to simplify the process of debugging, and the actual results from this particular bug report. We also discussed how to relay this information back to the person who initially asked the question.","I provided background information on a job ad from a colleague at Boston's Children Hospital that I had shared. I also provided suggestions for preparing their GitHub account, CV, and other information useful for job interviewers.","Hedia has gotten some feedback on some unexpected behavior from qSVAR package. The qSV values show no significant correlation with other phenotype metrics (they have in the past), while this is not impossible it is strange. There are many factors that could have lead to this issue. Hedia and I talked about some debugging strategies, starting with re-running the workflow on familar data to see if we can re-create the issue or detirmine if it is unique to KJ's dataset.","I provided background information on a job ad from a colleague at FHCRC that I had shared.","I gave feedback on scripts, how to organize them, the order of some analytical steps, which gene annotation files to use, and things like that. We don‚Äôt have results on the QC steps to share just yet.","I showed a workaround for making sure a variable has global scope before editing it inside a function, though I noted it's better not to use a function in her particular case","Continued reviewing qsvaR replication analysis code.","Detailed analysis steps for [this project](https://github.com/LieberInstitute/Cg_Hb_projector_profiling) using snRNA-seq data and analyzing it following OSCA-based steps from the CDSB 2021 snRNA-seq workshop.","Reviewed draft blog post on the Visium SPG AD project and published it at https://research.libd.org/rstatsclub/2023/11/01/our-first-adventure-with-visium-spatial-proteogenomics/.","I identified two separate issues: the repository name and using HTTPS remote instead of SSH","I walked through how a decision tree works and can be interpreted, then described how correlation and RMSE are computed in a couple spatialDLPFC spot deconvo plots","He noted that the wrong version of python was being used, and we eventually traced this to user-specific conda settings interfering with the module","Explained how interaction models work with limma-voom.","Ibidem to 2023-09-29 session.","Reviewed [this script](https://github.com/LieberInstitute/QSVA_2020/blob/914e3b58f7fb97c204654493c9f9dcb2e48d112b/code/06_Replication/02_DE_and_replication_plot.R#L220-L313) for the replication analysis performed as part of the qsvaR project.","Paired programming session to debug `/dcs04/lieber/lcolladotor/with10x_LIBD001/ranger_metrics/code/spaceranger/03_merged_metrics/02_explore_metrics.R`. Further explained the original intent of the code in this script and steps for updating it.","We interpreted the intuitive meaning of a formula from some old code, which attempted to quantify the similarity of a set of significant genes between datasets. I also helped interpret the meaning of a plot based on this metric","Giovanna's data has one cluster that is unqique to one donor, post batch correction. This donor also has some unique clinical background as well, I advised to drop the cluster as their batch corection seemed to be correct, and it possible that this is a rare biologically real cluster but there is not enough data to confirm.","I recommended 'spatialLIBD::fetch_data' for easy retrieval of public data to get started with spatial computational analysis. I also recommended pushing for an account at JHPCE, where computational resources are more plentiful, and getting used to submitting jobs, accessing software, etc is valuable to learn","Reviewed some scripts related to replication analyses. I explained the different thresholds used on the discovery as well as the target datasets, as well as the different levels for assessing replication.","I walked Lionel through the new login commands for SLURM + updating his bashrc file so he can work with the slurm cluser","Ibidem to 2023-09-29 session.","I noted that conda_R seemed to be changing the version of java loaded, which affected how the nextflow module worked. I recommended unloading conda_R and avoiding the loading of conda_R via the .bashrc","I described the general process of setting up SPEAQeasy and gave recommendations for his cluster: using singularity and SLURM-based installation/ set-up","I mostly covered documentation but also the appropriate options for his particular data","I investigated a \"failure to submit process to scheduler\" error using 'sacct' and similar commands, ultimately discovering the issue was related to SLURM configuration. There was an email chain with bithelp to follow up","We went over the changes needed to take the current qsvaR paper draft by Josh Stolz to a production-level result (aka for pre-printing / publishing it).","Adapted mode code for tensorQTL prep, had to add genomics ranges to the sce object, and deal with bugs around rse_to_bed function","I followed up with bitsupport, as the issue seemed to involve some failed DNS query internal to SLURM, rather than SPEAQeasy","We walked through old code which aimed to quantify the similarity between significant DE results, interpreting formulas from the code and constructing more intuitive formulas","Ibidem to 2023-09-29 session.","Worked through addapting code for tensorQTL prep for the MDD single cell data - had to deal with bugs relating to cell type names","I gave general advice on re-writing code-- keeping separate scripts, chunking code into segments and testing equality of outputs between old and new code after each chunk, and other strategies","We continued working with the OSCA-based CDSB 2021 intro to snRNA-seq workshop to further understand snRNA-seq data analysis steps, packages, and how they all relate to each other. The person who requested this DSgs was working on the workshop contents and had questions that we reviewed weekly (or so). Once they completed this OSCA-based training, we moved to analyzing an actual unpublished snRNA-seq dataset.","We looked at the rank vs. total UMI curves relevant for emptyDrops for the atac-seq samples. Cynthia and I have discussed these samples before. We went over how the emptyDrops function works again and what we can guesss is happenin in the curve (neurons vs. glia vs. empty droplets).  She is concerned beause for the 42-4 sample the wetbench target number of nuc was 7k, emptyDrops retrurns 9k. I thought that that sounded in the right ball park","I described the 'droplevels()' function, designed to remove unused factor levels","We went over the background for this project (lateral septum) and an overview of the DSgs system, GitHub setup, and other onboarding material.","I discussed how we selected training data for the spot-deconvolution image model-- using logistic regression to compute a confidence metric across the original training examples, that we then binned into 4 quartiles to re-select new examples to label in a less biased way","I mentioned slurmjobs::job_loop and also described how to use it in conjunction with getopt in R for looping over several variables","She had an issue installing a particular R package, which we pinpointed to which modules were loaded (including via bashrc). I also described the general problem of SLURM and SGE versions of JHPCE sharing a library path, which could result in subtle problems. I recommended deleting SGE packages (better would've been backing up!) and reinstalling on SLURM","I helped Fernando modify my code for creating input files to work witht the new pseudobulk data. Wegot the gene PCs done, he needs to get the genotype files together for this data before we can move to the next steps. Planning to meet more next week to continue this process.","Resolved some questions related to https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq and made two pull requests to fix typos.","We went over the math derivations for interpreting beta1, beta2, and beta3 on an interaction model where X1 is a continuous variable, X2 is a discrete variable with 2 options and we model `Y ~ X1 * X2`. We also used `ExploreModelMatrix`.","I explained the benefits of centering and scaling and referred to both the short video on this topic https://youtu.be/c8ffeFVUzk8 as well as the material we developed for the CSHL 2023 course https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html.","Peter and Reza were interested in running deconvolution on the bulk MDDseq with the new large MDD snRNA-seq dataset. The MDD snRNA-seq dataset is only sACC, we concluded that only haveing one region might make this analysis a bit awkward for the Amyg data. I them to the current deconvolution results and how we applied thoes results in downstream analysis.","I walked Fernando through using registration_pseudobulk() on the large MDD single cell dataset and went over how the function works. We are planning to meet agian to go over running PCA to prep for eQTL analysis.","I noted the 'www' directory (or the directory passed to the 'docs_path' parameter) controls the home page, and we typically use a relative soft link to the main repository README for the bulk of the content.","I described the general workflow, involving ImageJ, Samui, VisiumStitcher, and custom R + python code needed to eventually produce SpatialExperiment objetcs with one donor per sample. I focused on the high level of what the Samui piece looked like/ how much work is involved","Continued reviewing material from https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq","Discussed a few ideas about LIBD and social media","I showed code examples of writing array jobs with SLURM (and how it compares to SGE), then diagnosed an X11 forwarding issue where the '-x11' srun option was insufficient. Adding X11-related configuration in the SSH config fixed the issue (i.e. ForwardX11 and ForwardX11Trusted lines)","I discussed diagnosing marker quality with Maddy + Stephanie Page for the spatial hypocampus project. They are on the right track, and have created violin + hockystick plots for the different iterations of the dataset. I advised they elimate a noisy cluster and re-compute markers.","I described the pieces that go into a bookdown-powered documentation site, showing RMD code examples and the GHA workflow that deploys the site","I noted that Samui supports many types of features, and the spatial coordinates of cells can be supplied instead of spot coordinates to display features related to cells","I described the meaning of cell2location outputs and how to compare outputs between software tools. I also described where plots came from in R","Alan has used MAD cutoffs on total UMIs, genes detected, mito %, and a metric we haven't worked with yet: genes per UMI. He evaluated the data all together (not by sample) and chose #Mads by what seemed reasonable. The values he has used look good, but perhaps a bit strict. We looked at the distributions, which are slightly different between samples, I advised he use scater::isOutlier to see what the cutoffs would be by sample. One step he is lacking is doublet detection, I suggested he apply scDblFinder::computeDoubletDensity. Alan already has clusteres and annoted this data (lots of work) I suggested that a great double check is to calcluate the median value of the QC metrics for each final cluster and see if any are much higher than the others as a good double check that his initial QC process worked well.","Answered questions related to material at https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq","We talked about the first scripts in https://github.com/LieberInstitute/ranger_metrics","He noted that not all measured genes were displayed in the Visium_SPG_AD project. I found in the code why I subsetted and made a GH issue to fix it","I shared my favored set up for getting computational work done: working directly on JHPCE via the SSH extension for VSCode, using system-wide JHPCE modules for consistent software versioning, and X11 for interactive plotting (with a note I tend to just create and view PDFs for complex plots)","My major recommendation was learning version control with git, though I also mentioned learning SGE/SLURM also is a valuable skill for larger-scale coding projects. I helped him create a git repository for his current project (including account set up) and described the basics of git add, commit, and push","I described how the various scripts connect in spatialDLPFC for my work with deconvolution, to help share and re-use them for her project","We talked about material at https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq","I explained the status of the recount-pump transfer, where files were, and noted that chris or JHPCE admins would have to change permissions to allow her access to some files","I noted that modules at SGE vs. SLURM are totally disjoint, and that the HOSTNAME trick originally used in the team recommended bashrc didn't (at the time, though this has recently changed) work anymore, and the SLURMD_NODENAME variable can instead be used to infer where the user is","I gave some pointers about writing tweets. Mostly keep it simple, give people a call to action \"check out this link!\", add something eye catching. Also to spend sometime reading tweets and pay attention to what you like that other people tweet? What caught your interest? We also talked some stradgedy about tweeting around confrences, calling peoples attention to your talk/making it easy for people to find you to get in touch","We talked about how I became an Investigator at LIBD, the overall type of work we do as a team, some of our main collaborations, and overall plans for applying for funding in the future. In particular, I highlighted how I want to train team members on how to write an NIH biosketch, though not all are going to be \"key personnel\". I sent a link to the DSgs, team meeting schedule, rstats club schedule, and our http://lcolladotor.github.io/talk/teamtldr2022/ summary from April 20, 2022.","I did a complete tutorial of using the Samui API to create Samui-importable directories, including how to preprocess image files and how to host results publicly","We talked thorough the thigns to think about for the Mathy's dataset (this dataset has case + control paitents and found DEGs w/in cell types). I advised to start simple with the broad cell tytpes and not consider Dx at first. Check if the DEG they idetified are in our marker set. I think this will need some discussion with the project team if Dx is a concern for spot deconvolution.","https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/5e84f5ce32e6723b1a1eea20dc47ee320d55ca10",null,"He noted that just taking the most plentiful cell type in any cluster was leading to too many clusters with excitatory layer-5 cells. I mentioned that the raw outputs of spot deconvolution tools are fairly unreliable, but they still carry useful information when normalized. For his example, I mentioned dividing by the total number of cells for each cell type when making comparisons between cell types","Suggested deleting all old files as the code might be appending data instead of creating new files and also increasing the h_fsize, as the error relates disk space","guided him to the search engine on REDCap","For deconvolution, we talked about finding an appropriate reference dataset and we talked about an expert we can contact to ask which dataset(s) might be the most relevant. For snRNA-seq data analysis, I mentioned OSCA (Orchestrating Single Cell Analysis with Bioconductor) and how Louise could help guide this person on our code based off OSCA through a series of DSgs sessions.","removing the top 2500 pixels before splitting the image corrects the offset","I mentioned (information) entropy as a metric that would determine how well a gene is confined to a particular cluster (using average counts of a gene across all spots in each cluster, then normalizing the sum to add to 1), but cautioned that I wasn't familiar with favored metrics within the field","We transferred https://github.com/LieberInstitute/qsvaRPkgDemo to the LieberInstitute account. I asked to include some citations for sources (like the image source) and some other small edits. For GO / KEGG enrichment with clusterProfiler I asked to not filter results (pvalueCutoff / qvalueCutoff parameters), so we can see more results prior to choosing which ones to plot.","We ttok a look at the changes in the clustring in the spatial data before and after cropping out a area of questionalble quality. To me it looks like the QC params are too strict leaving spots floating in space, which I'm guessing is contributing to weird BayesSpace results (but this is a guess). We also talked code organization strtagedy - mostly keep it simple and consistent","I helped Hedia improve the formatting & content for her BioC qSVA workshop, main feedback ins to focus on and highlight the qsvaR steps in the workflow (rather than SummarizedExperiment overvew & DEG analysis)","We discussed the current status of the team, the project, what skills are required, boundaries in terms of time and who would be involved (and who wouldn't). We ended the session with me relaying a list of resources to check and read more carefully. Then in about 2 weeks we'll meet again to determine the next steps.","We made a Google sheet with columns such as `study_type`, `skill_name`, `learning_priority`, etc. I also worked on it after the DSgs session to help complete the current information. See [Slack](https://neuralplasticity.slack.com/archives/C04Q8MX99FY/p1690397027111449) for more details.","We resolved this issue by making sure that the variable names for which they wanted to specify the correct colors were actually part of the colData(SE). See [Slack](https://jhu-genomics.slack.com/archives/C01KSLPCGAC/p1689858182290369) for the actual code links. I also asked them to make sure to use soft links / full paths instead of using setwd() for files outside the project + to keep all related files at JHPCE instead of on a laptop, to make it easier to rule out any sources of differences (aka for reproducibility purposes).","I helped Daianna practice her BioC short talk, Mostly she needs to give more slides to discussing her results & include more actual plots not just icons (she much improved this presentation for the 7/26 meeting :) )","I helped discover the source and solutions to errors and warnings (mostly from BiocCheck and R CMD check), and found how to specify a single author in in-text citations",null,"reffered to this script /dcs04/lieber/marmaypag/Image_Processing/NM_LC_segmentation/test.m",null,"We discussed what would be the appropriate material for us to teach in a course. We will re-use some of our materials from https://github.com/lcolladotor/cshl_rstats_genome_scale_2023.","We went over the proposed response to reviewers, clarified some sentences, and requested co-authors to approve the response to the reviewers.","I pointed her to spot deconvolution scripts used in spatialDLPFC and the high-level overview of the workflow, as well as how we trained the decision tree classifier","I noted that he ran out of memory in his JHPCE job, which crashed R, so I suggested increasing 'mem_free' and 'h_vmem'","https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/tif2mat.m","I referred to examples of JHPCE modules I've created that activate shared conda environments, and explained the key commands used to get a similar module working","I recommended finding and trying to generalize the process a previous coworker followed to produce the list of degradation-associated transcripts, to limit relying on exact transcripts with an exact GENCODE version. I noted that you can convert transcript data to genes (albeit with lost information), but not vice-versa, so in theory either transcript or gene-level data could be supported by converting to genes","https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/mat2tif.m","referred here for clarification https://docs.google.com/presentation/d/1HJgkvlHtiIA5nucuNdHdYhRGAtpPPvASGFoAocqi1-U/edit#slide=id.g1163185b778_1_6","We adapted code from https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R and https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd in order to make the files at https://github.com/LieberInstitute/septum_lateral. See https://github.com/LieberInstitute/septum_lateral/commits/main from https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50 to https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c as well as related commits on the `gh-pages` branch.","Summarized at https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659. This involved explaining why we don't use as a background universe all genes in the annotation in gene ontology / KEGG pathway enrichment analyses. Instead, we use those that pass the expression filters and for which we computed differential expression statistics.","We resolved https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199 at https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&cid=C02RG5VGL6R, which was due to a user incorrectly having a Linux group assigned to files at JHPCE when they are not part of the user group. Uploads through Cyberduck can trigger this. The solution is to run https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example again on a qrsh session.","Cynthia has run DropletUtils::emptyDrops on the two LIBD + two example samples. We first clarified some terms and outputs she had recorded. For the most part DropletUtils returns similar number of non-empty cells as CellRanger arc, but Cynthia didn't confirm that the two methods flag the same cells. I asked her to creat a confision matrix between the two methods for each sample to check this.  The main concern Cynthia has was the LIBD sample with only  3k cells input, has 9k non-empty droplets returned by emptyDrops. This is quite odd as the CellRanger Arc metrics look quite bad for this sample, but the DropletUtils metrics look fine (there are high UMI values for many droplets). We will want to dig in to this more to understand the differnces between the two methods.","We worked on qsvaR debugging an issue raised by Renee and Louise. We then improved the error messages on qsvaR as noted at https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400.","Using the septum lateral data, I trained Ryan on how to make this type of Globus endpoint, how to set up the right permissions, how to edit and update http://research.libd.org/globus/ (see [this](https://github.com/LieberInstitute/globus/blob/b10e0fb89c40489d23028e8ed3f7812cdea76a6b/index.Rmd#L57) for details), and how to document the Globus information on a manuscript.","Using the lateral septum snRNA-seq data, I trained Heena on how to filter a SingleCellExperiment object to reduce the required memory from 7GB to less than 1GB in order to be able to deploy an iSEE app at shinyapps.io, whose computers have a maximum of 8 GB of RAM. See https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98 and related commits.","We worked with the script where Renee had noticed issues with qsvaR. We adapted it to work on qsvaR local updates through devtools::load_all(). We also resolved https://github.com/LieberInstitute/qsvaR/issues/29 and https://github.com/LieberInstitute/qsvaR/issues/28.","We started with getting github token access for Hedia's laptop working. Then started a pull request using usethis for qSVA, for Hedia to add seed capabilities to several functions. We will need to meet several more times to complete the whole help request","Uma mentioned that she had to login everytime she pushed a commit from the directory in question (but not other repos). I thought we should start by using a github access token to update her login to github from JHPCE. The best way I know how to do this is to use gitcreds::gitcreds_set() in R. At first it gave us an error, which was solved by adding the git modules loaded in the team bashrc to Uma's basrc file as well as her .gitconfig file. After that we were able to save the token to her JHPCe setup and the github push issue seemed to be solved","I described the 'add_coords' function, how features link to existing coordinates, and that coords must have at least one associated feature to display","I referred her to recent versions of my code (spatialDLPFC and SPG_AD projects) and walked through the high-level steps of each","Reviewed input for tensorqtl (it is counts, not logcounts), and revised what how we filter lowly expressed exon-exon junctions using http://research.libd.org/jaffelab/reference/expression_cutoff.html.","Discussed how to best pseudobulk the data prior to doing this type of analysis. In particular, how to adapt https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148 and related code, as well as plots to make such as boxplots of the min_ncells by cell type (without any filtering) and cutoffs found by `isOutlier()` (different cutoff for each cell type) + PCA plots colored by min_ncells.","Discussed a few different areas of growth, including a change in direction related to our outreach activities.","I showed that the error was essentially a failure to allocate memory and recommended doubling required memory for the failing SPEAQeasy process until success","I described that an entire directory (rather than all of its contents) can be ignored, and should be specified relative to the directory containing the .gitignore","We looked at some old code vs. an adapted version, which was not producing the same output. I recommended splitting the code into ordered sections, and executing the old and new version of each to identify exactly where the error was occurring","I discovered she was using the --small_test argument and not --input, so only the small 3-sample test set was running instead of her actual data","I walked through the meaning of certain information-dense plots I made to show spot-deconvolution results","I recommended filtering rows of the RSE via 'seqnames', which would return genes (or features in general) of the particular chromosomes needed","She asked about any quality-check-related options SPEAQeasy had (I pointed her to '--ercc' for applicable data) and whether trimming defaults were reasonable for her data","I described the general workflow needed to start from raw images and spaceranger outputs, and end up with a Samui-compatible directory that can be dragged into the browser (locally) to view Visium data (including segmented fluorescence channels)","I suggested creating a template configuration file, then looping through her data, using the sed command to write configuration files with the relevant variable replaced from the template","She wanted to add the columns associated with unique rows in one data frame to the other, so I suggested combining the 'unique()' command with 'left_join()'","I suggested using regular expressions in R to get the new filename from a given old filename, then file.copy to perform a safe test, and finally deleting the old files","She was unable to push to GitHub on her laptop. It appeared she didn't have an SSH keypair set up with GitHub, so we set that up to fix the issue","Hedia was concerned about the large number of variables in the reproducibility DE experiment, specifically the PC. I pointed out that the point of the anlaysis was to test the effectiveness of the models (with and without qSVs) so while this might be a valid concern in a sperate DE analysis, we are testing model quality here. I also noticed that the reproducibility anlaysis was not using functions from the qsvaR package, and that as qsvaR is the \"product\" of this whole project its important we use the functions in this anlysis .","I described typical expected alignment rates, the difference between concordant and overall map rate, and details about other SPEAQeasy output metrics","I explained that at the time, I just created the RMD file and added any referenced images to the rstatsclubsource repo, following naming conventions that include the post date and hyphens between words in the blog title. I believe Leo did some manual work to get it fully published, so I referred to the full guide in case my method was incomplete: http://research.libd.org/rstatsclub/2018/03/09/contributing-to-the-libd-rstats-club/","I explained how cell2location is agnostic to what gene identifiers are used, but that gene indentifier type must match between spatial and reference single-cell data. I mentioned the org.Hs.eg.db package.","I described when I make a module for python-based tools, the benefits of conda vs. python virtual environments, and related topics","I suspected improper configuration of email associated with her git commits, which was indeed the problem. Her commits moving forward are associated with her GitHub account, but we couldn't figure out how to retroactively regain credit for commits done with the wrong email","I described 'git pull' to merge in remote changes, and using the 'vi' editor to write a merge message","I discussed with Renee the way I code and showed her some examples and gave her a few coding advices.","Looks like user was trying to create a plot and import data from a .csv file but was getting an error importing the csv\nhttps://community.rstudio.com/t/error-creating-a-windrose/164132","I explained which paths I'd need to get Samui running on our data, and we discussed which features might be useful to show","Project links reviewed: http://research.libd.org/smokingMouse/ and http://research.libd.org/smokingMouse/articles/smokingMouse.html","I described how ACLs are properly set up at dcs05 to inherit downward automatically, so just an 'rsync'/ move to trash is necessary in those transfers","I explained the purpose of Git-LFS, that it's configured for certain repositories, and that we can load a module to use it","Website tools to look for public rna-seq samples & packages to make synthetic replicates [as shown here](https://support.bioconductor.org/p/9150681/#9150698).","I described my use of XQuartz, the 'xhost +local:' command, and the proper ordering of these tools in combination with SSHing to allow plotting to work","We discussed the use of qsvaR. For now the qsvaR only use the Gencode25 annotation. We also commented on the steps needed in case we want to implement the package for the more updated annotation versions in the future.","I described what we use modules for, the general approach to making them for common software types (binaries, python packages), and walked through the process by making a new FastQC module","Trish and I went over the code I previoulsy wrote for tensorQTL, and some of the custom functions for preparing the input data. How to input the different datatypes in to python, and run the function, and strategies  for running the different cell types in parralell on the GPU queue","We discussed each step of the pipeline and the outputs. We also had a look at the QC metrics and the code that Cynthia added to the script.","I helped Daianna decipher an error message she got in the BioC check in the GitHub Actions involving a local path harcoded in to one of her tests. We also debugged for similar issues by searching her code for other local paths that would cause her code  to break when run remotely","We reviewed the overflow to analyse RNA-Seq and ATAC data with the Seurat and Signac tools. I provided learning resources, such as data file organization, funtions and methos available in the tools. I also provided a complete explanation about QC for preprocessing data integrated.","I helped him set up git on his laptop (username + email, SSH key), showed how to edit R Markdown files using RStudio, and basic version control (git add, commit, and push).","The error message was cryptic, but made more clear in the latest SPEAQeasy: the transcripts FASTA mismatched with the GTF (custom annotation was used) due to a typo in the gene ID in the GTF.","We went through the  https://github.com/HediaTnani/Int_API_R. Then we saw other API examples with authentication (github, ....).","Issue and the solution found here https://support.bioconductor.org/p/9150363/#9150404","user had question about generating rarefaction curves in R. Issue found here https://support.bioconductor.org/p/9150366/#9150386","I shared the tensorQTL example I presented at rstats club. We also went over the different in-puts and formats, including some we can re-use from bulk MDDseq. WE also discusses the differnt setting for tensorQTL and how to adapt sh files to submit to the GPU queue","I initially mistakenly suggested kallisto, but later clarified that using featureCounts with a custom single-gene GTF from the BAM files (produced by SPEAQeasy) would be the proper way to quickly quantify a single gene","See https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099 for more. Led to https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6 and related commits. spatialLIBD version 1.11.10 now supports spaceranger version 2023.0208.0 output files.","We used https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection and also went over the JHPCE and R config files from the team website. We also talked about the DSgs and how it would be beneficial for Kinnary to meet 1 on 1 with all DSgs guides to build familiarity and make it easier to ask for help in the future.","See https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179 for more details. Heena is working with spaceranger output files from 2 different versions, and in particular, version 2023.0208.0 (internal 10x Genomics version) changed their output files structure","Described what we do, the DSgs system, potential for visitors at our team meeting, and research ideas.","We discussed how to merge two datasets in R, making sure they shared the exact set of genes","With Cynthia we saw the differents folders that are part of the template project repository and how to organize your project repository.","The solution involved asking BitSupport to login as a specific user and run https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example. Knowing the exact file paths that triggered the permission issues are important to help check with `ls -lh` and `nfs4_getfacl` under a `qrsh` session.","See the [Bioconductor Slack message](https://community-bioc.slack.com/archives/CR1BLV821/p1677783580326559) I wrote for more details.","End result: https://twitter.com/HeenaDivecha/status/1631360771342098433","See https://github.com/drighelli/SpatialExperiment/issues/135 and the pull request at https://github.com/drighelli/SpatialExperiment/pull/136 that resolves this issue.","I described the need to 'git add' both the old filenames and new filenames, and the use of 'git add --all' to reference deleted files.","I recommended code for matching two data frames up by common genes, and explained that matrices don't support multiple data types (instead a 'data.frame' might be used)","Sreya was having some issues with functions from DeconvoBuddies, namley plot_marker_express. I am going to check that on my end as it seems like her computer wasn't finding the support functions for the main function. She was also having trouble matching the gene names between the sce and bulk rse objects, the ensembl names seemd to have differnt versions, I advided she maybe switch to using the gene symbols to keep things simple.","No public result just yet.","We went over the ratioinal behind psuedobulking and how we can use heatmaps to show marker experssion. Created psuedobulk data for three different clustering methods, and went over the basic process of making heatmaps. Bukola will work on the aestetics of the heatmaps, and cerate one for all thee clustering methods.","related to my meeting with Uma on 2/10. We fixed Boyi's bashrc to allow group write by deafult, fixing the ,git issue and eliminating the need to re-run the permission update script often","I showed how glob expressions can be used to conveniently match many files across potentially several files, and described gitignore-specific rules, such as certain wildcard expression matching files no matter what directory they are in","Uma was having file permission issues, the group permissions for several files in a shared project were set to read only. This was causing issues with the team using scripts but also in making git commits as some of the idex files in .git were read-only. We updated her bashrc files to contain \"umask u=rwx,g=rwx,o= ## equivalent to umask 007\" to prevent this in the future and ran the premissoin update script.","Bukola hads run muliple clustering methods on the habenula data, I advised she pick the two that were the most simmlar to ERiks previous annotations (we were happy with) via rand index, and start examining marker gene expression.","I went over details like naming the OneDrive folder and correctly using CrossReference to refer to that name in figures and references to those figures","I helped port several plots into Illustrator as a figure, describing our normal process of using PDFs and linked files to maintain resolution.","I showed how to filter for individuals with a certain condition, and subset two data frames with the same criteria.","She was unable to clone private repositories due to a problem with her SSH key with GitHub. We solved the problem by setting up a new key pair using the recommended Ed25519 algorithm instead of the deprecated RSA algorithm.","We focused mostly on the options Nextflow provides for managing software dependencies: Lmod modules, conda environments, and docker + singularity containers and the benefits of each potential approach","We discussed how to implement github actions with Nextflow workflow and the use of Conda and/or Docker containers.","We identifed that SPEAQeasy cannot handle runs with only one sample, but that the intention was to use multiple samples (the manifest did not use unique sample names)","Hope created a github account and I talked her thuogth the basics of git, But we hit a snag with the instalation of git on her laptop, git is built in to xcode in the ventura OS (which her machine is running) but she had some trouble with that instalation too. Shes going to try and fix that independtly, and reasch out if she needs more help gettign off hte ground with git + github","We adapted code from the DLPFC work and https://www.stephaniehicks.com/biocdemo/articles/Demo.html to run GLM-PCA to prep for batch correction","We went over several concepts in Nextflow (e.g. processes and channels) and discussed how to invoke R scripts inside processes.","We (Me + Nick) discussed with Sean the basics of Nextflow and Nextflow DSL2 and how to make processes and channels and workflows. We gave him resources to read  and gave him small examplesto follow. We helped him also with his errors on slack.","Bukola and I adapted QC code for the habenula data","See [Slack notes](https://jhu-genomics.slack.com/archives/C0498763CUC/p1673451228034439) for further details. They involved https://github.com/LieberInstitute/template_project, https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h, and other specific instructions for setting up a project.","Continuing Sreyas work with the single cell mouse data, I pointed her towards scater::plotExpression to plot some of the genes her group is intested in observing by cell type. However we ran in to some bugs with running logNormCounts which we think is from an old version of R. I advised that she update R and re-install the packages, hopefull that will fix the issue.","We uncovered the source of some unwanted NA values in some genotyping data and eliminated it by removing some code","I helped Sreya build a single cell experiment object with expression and metadata csvs she had from an publication. Hopefully she can use this to independently explore the gene expression her group is interested in.","We talked about how to compute KL divergence between 2 groups of cell-type proportions on her data using a variation of my code","Hedia wanted some background information about the deconvolution process we applied to the degradation data. I explained the mean ratio marker finding, the resulting stats, and how we ued them to build our set of marker genes. Then the script for how we ran bisque + the output of bisque .","Sreya wants to test if a gene of intrest to her group is associaited with a ceratain cell type. They are working with bulk mouse data. I suggested that deconvolution may not be able to answer theire innital question but can provide depth to their bulk analysis. I suggested that they find a mouse single cell dataset that is relevant to their current data set, then first observe their gene of interest their, see if that answers any questions. Then they could use that data to run deconvioltion with bisuqe (I suggested a follow up meeting if she needs guidance with this)","I described the manual process for resolving a merge conflict with git when git cannot automatically merge two branches","We pinpointed an error message as being due to trying to find the 'colData' of a list containing Summarized Experiments rather than calling 'colData' on the SummarizedExperiment itself","I described how to color ggplot2 plots by a continuous variable, adjust the palette, and other ggplot-related syntax/tricks","We discussed how PCA reduces the dimesions of our data and helps show the variance between samples. I helped Bukola strategize how to make PCA plots for the habenula data. Start plotting the first 5 PCs pairwise, color by Dx. Then explore key QC or demographic variables, see if we can indentify any patterns. I was unsure if she should repeat this for exon, jx, tx, as we didnt for MDDseq. But figured gene was a good place to start and if the other features should be explored too, the code should be easily adapatabe.","I gave Hedia a breif overview of the goals and process of deconvolution, including mean ratio marker finding. I linked my deconvolution slides. We then went over the code for running bisque on the qSVA dataset, including details of the input, refrence, and output data.","I recommended some data-preprocessing steps for later downstream manipulation of the data","I gave general advice for how to preprocess some data, including changing data types, transposing matrices, and dropping columns to format things for a collaborator","I described 'facet_wrap' and showed that for some situations, 'plot_grid' with a single output PDF (with width and height scaled appropriately) does well","We generally discussed linear mixed effects models and some common terms used as fixed effects in genomic data","Tricia is trying to run glmPCA on the very large MDD snRNA-seq dataset. She has a run submitted without parallelization that has been runing for 3 days. To try and speed things up she has been trying to use BioCparrallel, via MulticoreParam. But the scheduler keeps killing her job. We checked that -pe local X and MulticoreParam(X) match , so it wasn't that. I recomended she try to run a subset of the data to get a sense of how glmPCA uses memory. But couldn't come up with a fix to make the parallelization work. As its PCA it also can't be run as subsets, so there aren't other workarounds :( I think BioCparallel + sge scheduling would be a good topic for Rstats club as this seems to be a common headache","https://lcolladotor.github.io/bioc_team_ds for my team website, https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing for the LIBD rstats club schedule, https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials could be of interest also, https://github.com/LieberInstitute/template_project for a template for organizing projects, learn about the DSgs https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs, https://rstudio.github.io/reticulate/ for using Python from R","I described how to transpose matrices, change their type, drop columns, and related tricks to solve a data-formatting problem asked by a collaborator","Lionel walked me through some plots they need for the LS paper: marker genes heatmap and expression violin plots. I pointed him to some code from DLPFC sn that should be helpful (he needs access to this repo). He also needs to get setup with JHPCE on his new laptop, I pointed him to the team laptop setup docs. He also is new to ggplot, so I pointed him to the visulization chapeter of R for data scientist. This will probaly require some follow up meetings","Daianna has added a new color scheme to the heatmaps that makes the differnces between positive and negative more clear, how ever I think the darkest red being close to 0 is misleading, so shes going to fix that. I also recomended she simplify the annotaions on the combined heatmap so its easier to find and interpret patterns.","We reviewed the reasons or correcting counts and why thats important for DE analysis. We went over the math behind TMM, and then how thoes ideas are used in edgeR. I pointed Bukola to a stats quest video with a more detatiled explination of how the edgeR correction works.","Sang Ho showed me the csv files that he exported from different moduled on HALO. They seem to have the same rows, so copying the needed column in to a differnt csv is easy in excel (where Sang Ho is working) but I showed him how to check that things are really aligned with some excel functions. We also looked at the data in prism, which I had to quickly learned about. I advised that he should show the data in boxplots not as a barplot to better display that data, and went over the checks for running an anova.","I mentioned rtracklayer::liftOver and showed a code example of lifting over genomic coordinates for a SummarizedExperiment","Daianna showed me her dot plots with the correct 0.05 q val filter, they looked good, ready for discussion. One tweak , is hshe had a combined up + down expresion col for some plots which was redundant and should be removed. She had also made some heatmaps that didn;t show very much difference between case control, htey probaly need to be normalized by the genes overall expression, but I need to check prevous work to remeber how to best do that","We went throught boxplots Bukola had made for the data, I suggested she add geom_jitter and how to make that work with geom_repel to help show more details about the data. We also discussed direct vs. standard evaluation of variable and how that makes writing funcitons with tidyverse tricky, but how she can use aes_() to do stadard eval with ggplot","I walked Bukola through our process for buld data QC, which involves making many exploratory plots of QC metrics and know groups (like batch, or case/control) to understand patterns and identify problematics samples. Then later implementing cutoff values either based on stats (like a 3 MAD cut off) or knowlege of ressonable values. Bukola will spend some time plotting QC values, adapting code from smoking mouse, then we will meet again to interpret these plots.","Daianna and I went over the dot plots she made for GO analysis from the multiple DE test she ran. Her plots look good, but we identified that she needed to use the default q cutoff to get useful results. She will rerun the GO analysis and replot the results.","We adapted my_plotExpression() (a function I wrote) to work for Bukola's data. This required some debugging as subsetting to just the one gene in question altered how some internal dataframes looked. I also gave Bukola some pointers about saving plots using ggplot.","Sang Ho walked me through an analysis they want to do for the AD project, it involves annotating HALO data. We talked thought a strategy, this may be a bit computationally tricky to implement but it seems doable.","We talked about https://github.com/LieberInstitute/template_project, https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/, http://research.libd.org/sgejobs/reference/job_single.html, https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7, https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1, https://happygitwithr.com/, https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.","I described the need for SSH keys between the login and compute nodes at JHPCE, and how an incorrect set-up will manifest as a prompt for your password when using 'qrsh'. I referenced https://jhpce.jhu.edu/knowledge-base/authentication/ssh-key-setup/","We debugged her SSH set-up with JHCPE, finding a syntax mistake, confirming the contents of the '.ssh/config' file, and other details to ultimately get the traditional RStudio + JHPCE set-up working smoothly. We touched a bit on the process for using git from JHPCE but didn't finish fully setting things up","LIBD rstats club schedule https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing. Bioconductor overview video https://youtu.be/c0Ch_sXiGDQ and related notes in case you want to skim them https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing (we typically have this for each video entry). R and RStudio intro https://youtu.be/XTuJ8vGnzBU. Example vignette https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html from https://bioconductor.org/packages/spatialLIBD/. Data Science guidance sessions https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0. JHPCE new user request https://jhpce.jhu.edu/register/user/. Making your personal website with postcards https://youtu.be/9LUCZAx3SlU. Examples include https://lahuuki.github.io/ by Louise, https://jtleek.com/, http://aejaffe.com/.","I described how 'module load conda_R' lines in a submitted shell script override the default version of R that can be loaded by default using the '.bashrc'. In this case this undesired interaction caused a corrupted R package to load and return an error","We adapted someone else's code, slightly changing some awk commands to use cut (which I'm more familiar with) and using a for loop rather than piping a file to a while loop. This allowed the code to work as intended on the data","I pointed to installation guidelines for the cluster to resolve issues about R packages not being installed and a problematic nextflow version.",null,"Related to how we organize files for a publication.","Density is related to the probability density function, see [wiki](https://en.wikipedia.org/wiki/Kernel_density_estimation for) more details. [This link](https://stats.stackexchange.com/questions/4220/can-a-probability-distribution-value-exceeding-1-be-ok) is also useful for understanding the y-axis in more detail. Violin plots are just density plots rotated 90 degrees and mirrored. They are compared against boxplots frequently, which while easier to understand, are not good for non-unimodal data. See [this link](https://mode.com/blog/violin-plot-examples/#:~:text=What%20is%20a%20violin%20plot%3F,the%20density%20of%20each%20variable) for more. I like `plotExplanatoryVariables()` because it's fast to run and gives a general idea of things. See [Figure 1b](https://bioconductor.org/packages/release/bioc/vignettes/variancePartition/inst/doc/variancePartition.pdf) from `variancePartition` for a violin plot of similar data. `variancePartition` can handle mixed effects models (random effects) and in that scenario is more precise, but will also take longer to compute.","I described how to smoothly 'git pull' when updates have been made to both the local and remote repo (adding + committing local changes, running 'git pull', and handling a smooth merge)","I described proper saving and loading of SummarizedExperiment derivatives whose assays are HDF5-backed using the HDF5Array package, and how to convert between HDF5-backed objects and \"ordinary\" in-memory objects","I went over the DE pipeline we use, noting that the model development is the most demanding and non-static part. We talked specifics for their dataset which has some experimental design to consider as its organids and is diffrent from patient case control. I posinted Geo to scripts where we do QC, expression cutoff, and my run_DE script","Heena saw old commits pop up on slacks git notifications when she pushed commits she had locally on her laptop, but they didn't reappear in the git history. I think she did everything right and there might be something strange with the behavior of the slack message (as thats the only place things seemd wrong). We did some detective work between the files and the commits, everything seems good. Heena was able to then pull to the cluster with out conflicts","Had to work over slack again as Daianna was having trouble with zoom. Working with the feedback from the previous meetings, the experimetn has been narrowed down to 5 DE runs, and then comparing some pairs of the outputs. We were unsure if we should include mitoRate as a variable in the adult model as it correlates highly with totalAssignedGene, but decided to include it for now to match the model used in the pup DEA. I also sent her some code I have used to make scatter plots of t-stats to help compare DE results.","Check [EnchancedVolcano](https://bioconductor.org/packages/devel/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html) which shows a quick DE guide and finishes with some Volcano plots, which can be good enough for a grant application. If you want to use limma, you could follow [the SPEAQeasy example](http://research.libd.org/SPEAQeasy-example/de_analysis_speaqeasy.html#16_Modeling) or [our DE bootcamp](http://research.libd.org/SPEAQeasy-example/bootcamp_intro#5_Day_3) a bit. Or dive deeper into [limma's documentation](http://bioconductor.org/packages/release/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html).","We intended to meet on zoom, but Daianna's connection was bad so we used Slack to chat. Daianna has made many plots and visuals about the many DE analysis for smoking mouse. I gave her some feedback about orgaizing her slides so the figures can practically be used inform DE models, right now everything is a bit scattered as shes planning on running 7 analysis so there is a lot to consider. We will need to meet again to finish this.","wrong input was given to the function",null,"should select virtual stack option in bioformats reader when opening and multichannel image in imagej",null,"Daianna showed me her varianceParition plots, and we talked bout how to design the DE anlaysis based on what she's oberved in the mouse data. I advised to keep things simpler (ie not create too many groups) but rather use models to control for vaiables adding variance. We will meet next to talk model design for this work.","Explain how to pick randomn images to train PNN segmentation algorithm","We walked through the code to make a pseudobulked object and plotting the explanatory variables.","Note that `spe$var_colors` works for `spe$var` where the `var` part has to be the same in both variable names. Also `spe$var_colors` should be a named vector such that `unique(names(spe$var_colors))` matches `unique(spe$var)`.","code loops over the different file in the dataset and Marah was clearing it at the end of the loop every time which didnt let her save any data","We worked through a few issues in the cellpose module, which each had a different cause. In one case, the version of torch/CUDA was outdated for the hardware. In another case, user-specific libraries were being referenced instead of the central module's libraries.","Daianna shared her updated QC plots, adn the one sample she planned to drop based on fastCellQC. I advised that she dig deeper in to how that QC finstion works, and what the cutoff values are, as well as exmaming mito vs. ribo precentages, and a two samples that don't seem to group with others of the same phenotype /","NA","The error messages involved a mismatch between sample ID and file name for the 'QualityTrimmed' process, as well as a supposed missing ID in the manifest (which was actually present). Simply resuming the pipeline got rid of both errors.","NA","I explained how to get pheatmap to use the same color scheme for column and row annotations by making the annotaion tables have the same colname, and suppying a color pallet","We discussed a strategy to subset the 102 loci in question, and adapted code to subset the FDR signif pairs to work with the location info.","I described how to configure SPEAQeasy for a particular dataset, and how to manage two separate runs of SPEAQeasy as part of a different git repository (requiring the use of submodules)","NA","Daianna shared the updates she made to the QC plots based on our previous discussion, specifically separating brain and blood samples.","I showed the documentation's description of 'mitoRate' and linked to the code where it's computed: https://github.com/LieberInstitute/SPEAQeasy/blob/bfe4e664281b52c5e239e372393a8aea36084e0d/scripts/create_count_objects.R#L521-L533","She was getting a memory error when running BayesSpace and I ended up advising her to reduce the nrep argument. Also advised her on running fasthplus because it was taking too long. She needed to reduce the proportion of the data being used.","I described the need to manually convert some spatialExperiment slots prior to conversion with zellkonverter, and that some data types are not supported. I pointed to my SpaGCN code and the lines specifically responsible for clustering + writing outputs","Daianna walked me through her progress in the DE analysis. We went over QC plots and discussed experimental design. She will check in with Kristen about how to organize the brain vs. blood samples. Currently she dropped the blood sampeles as they looked very different in several QC metrics. But I advised that they just be handled seperatly as it makes senese they look different as they are different tissue, and the signal is probaly biological rather than technical","I described when 'git pull' can be \"safely\" used, and when 'git reset' might have to be used beforehand",null,"I described the use of 'remotes::install_github' for GitHub-based packages. We encountered a GitHub authentication error that was solved by running the R code 'Sys.unsetenv(\"GITHUB_PAT\")' prior to the normal installation command.","she needed to have the plots as single ggplots instead of a list so I advised her to used vis_clus() instead. Shared code with her that I had used to make some of supplemental figures. She also asked if she need to discard spots before doing batch correction and I told her yes.","We fixed Sanjana's ssh set up and got the qrsh command working. We got git configurated for her locally and on JHPCE, as well as set up tokens. Created a test repo, and made commits from her laptop and JHPCE as an example","checked size of the spe object. checked that only has low res image files. check dim of colData. All sizes checkedout. Believe this to be a package problem.","Maddy asked me how to get the total UMI's for each sample out of her spe object. I wrote a small for loop for her in order to do it.","We went over github basics and motivation behind version control. Quickly discovered that Sanjana's qrsh command is not working, requiring a password. Ended up using the session to troubleshoot that, with no success :( planning on circling back/re-setting up ssh for her. Will meet later about git setup","We checked Sreya's code against the OSCA DE example run. She was worried that there weren't may significant genes, but her results seemed to be in the same range (10s of genes) as the example. We discussed how maybe there were QC or demographic variables that could have been added to the model, but the data she has doesn't have much to work with. We will revisit when I have maybe done more work with sn DE analysis.",null,"Talked about recount2, recount3, snaptron, snapcount, recount.bwtool, recount-brain, recount predictions, megadepth, the recount3 study explorer. I also sent them all the links to them, though you can find many of them at https://jhubiostatistics.shinyapps.io/recount/, https://jhubiostatistics.shinyapps.io/recount-brain/, and http://rna.recount.bio/.","I pointed Nick to a few scripts used in marker finding for the Tran Maynard dataset, using a few functions from DeconvoBuddies. For marker heat maps we went over how I hace built annotation tables, and a few tricks for optimizing pheatmaps for maker plots.","We converted a downloaded dataset from tsv files to a sce object. Went over a strategy for performing DE on single cell data, using OSCA as a reference","We spent the session getting Sanjana set up better to work on JHPCE, setting up ssh keys, cyberduck, RStudio, and config files. Plan to meet again to set up git + start single cell work","This had already been fixed, but Abby had an older version of spatialLIBD installed. We also went over some other steps needed to deploy an `spatialLIBD` app with pseudo-bulked data and modeling results.","Resulted in a fix for [this issue](https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1137544893).","Assessed results of BayesSpace joint clustering and suggested that Alice discard the spots with low library sizes to improve clustering. Offered to create small SPE object for her with just the for samples she will be doing long read sequencing on.","Brainstorming and planning session.","Resolved via email.","created new script to subset spe object by removing some of the images.  updated app.R to load smaller spe object (half the size) and re-ran shiny app locally. Explained how to version control from local computer and fixed problem with version controlling large data file.  Still having same problem of not being able to view shiny app online","Review of statistical models as well as access to data","We both leaned how to use the image function to create a annotation bar for heatmaps based on a custom color pallet","Twitter, LinkedIn, jobRxiv","Tried to troubleshoot adding visium IF images to spe object","Helped setup OneDrive folder for LC manuscript. Explained how to organize figure folders.","Clarified what the data is showing and explained a bit the goals of the LIBD data portal that Geo Pertea is building.",null,"Helped fix error of shiny app not being able to find spe object. Helped sucessfully deploy app.","We went over how to create a new type of refrence (Supp Table/Supp Figure). Also walked though the full process of creating a label and then refrencing it in from the text + considreations for formatting","PhD programs and other options.","We went thourgh some plotting code Sreya had adapted to make a manhattan plot, and went over which aestetics to change to get the desired plot","I described the two JHPCE-based repositories we push to (https://github.com/LieberInstitute/jhpce_mod_source and https://github.com/LieberInstitute/jhpce_module_config), my usual installation approach for python-based GPU-utilizing software, and the contents of the lua-based config enabling a module to load.","See [this Twitter thread](https://twitter.com/lahuuki/status/1521581339903873024) for the result.","Provided guidance on how to share images and how to organize supplementary files. Resulting in this [Figshare](https://doi.org/10.6084/m9.figshare.19729567.v1) zip file.","I clarified the usage of nextflow channels, described when docker integration is likely to be worthwhile, and answered other general questions about processing data with a nextflow-based workflow","NA","I helped come up with a stratagedy to loop over 100+ files for an analysis and exreact a portion of the results in to one R object","NA","Helped explain the different parts of the html output of spaceranger and which fields are important to check for QC","NA","NA","See https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012.","Clarified a few questions about LIBD's SPEAQeasy RSE objects and a problem with some colData() that was missing, that we'll provide early next week. Send links to our deconvolution work with Louise A Huuki-Myers.","If the situation had been simple, that is, that the latest commit was the one that commited the large files, we could have used the solution described at http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/. Instead, we had to use https://rtyley.github.io/bfg-repo-cleaner/.","I helped Heena address an error she was getting when trying to run read10XVisiumWrapper(). We noted an issue with the function and submitted an github issue to spatialLIBD. We got the function to work but only if you provide the path to the gtf file. Met for 1hr.","JHPCE ssh keys (to avoid typing passwords when using qrsh), .Rprofile (both), .bashrc, .zshrc (laptop), iTerm2 (laptop), Textmate (laptop), remote port fowading for rmate, xquartz (laptop) installation and configuration. There are several LIBD rstats club videos on these topics already, but we just did all the setup fast. In particular, the .bashrc configuration was important to resolve a umask setting problem that was interfering with the permissions on the Visium IF AD project.","We installed and configured Cross Refrence, then went over how to use the lable vs. refrence links + pro tips on how to avoid errors","spot size and scaling factor from json file. explained how each metric is associated to different version of the histology images.","Customized VistoSeg code to their histology images",null,"See [this Slack thread](https://neuralplasticity.slack.com/archives/C010NFSQS48/p1649249130880349). Ultimately, this situation involved teaching Tony to share his code.","We want over some plotting basics with ggplot2, and how to use geom_rect to recreate some spatial plots that a halo module produces in R.","https://github.com/LieberInstitute/VistoSeg/issues/11","Able to identify situation where duplicate.correlation should be used and how block data is important to linear modeling. We then looked at how top table can be used to summarize DE outputs.","modified code to work witth her dataset. did graph based clustering and plottiing. Explained benefits of clustering across samples. Advised her to do isoform calling on pseudobulked data for multiple samples. Met for 1hr.","After identifying that 'redstone' was a precompiled binary for Linux, I described how to use 'wget' to download software like this, set execute permissions if necessary, and run at JHPCE.","Finished installing packages. Set up sample_info table. Walked her through the rest of the code.","I recommended some adjustments to a custom GTF and transcripts FASTA to solve an error seen in SPEAQeasy where transcripts and annotation were found to be incompatible.","We applied some test I used for the TREG package to help test more parts of the functions, like checking for errors w/ bad input, warning messages, and qualities of outputs. We discussed possible ways to use testthat on a ggplot, but that might need more thought.","Showed her app.R, deploy.R scripts and places where to edit, showed her how to navigate through the shinyapp portal to looks for logs when deploying the app. EXplained her few erros she ran through when deploying for the 1st time.","We set up a git repo and added code to be adapted to build an spe object from Alice's long read data.","I explained how the first time that you ssh into a new computer (which happens also with git pull/git push), you need to accept that the new host is authentic, which will then edit your ~/.ssh/known_hosts file.","This was a preliminary meeting with Alice to decide where to begin with analyzing her spatial data. We decided to start by generating an spe object with her long read data.","An error occurred in a SPEAQeasy run, which I identified to be due to not providing all the required inputs (specifically the sites at which to perform variant calling)","This is challenging since it involves putting together code from multiple scripts/projects. We summarized the steps to do [on this GitHub issue thread](https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1069493294).","We both verified additional transcript annotation added to a GENCODE GTF looked good for use as input to SPEAQeasy","Louise and I discussed the [Dentate Gyrus, Jaffe at al, 2020](https://doi.org/10.1038/s41593-020-0604-z) paper which used LCM-seq, what deconvolution using Bisque can do, and what role it can play at different analytical stages.","TODO","An error occurred in the counts step, which I discovered was a bug in some filename-matching code","We had some trouble logging in because the WiFi wasn't working without logging in. We solved this by using an ethernet cable at LIBD (which is part of the JHU network). Once we logged in once, then we could use the WiFi.","I described two-factor authentication vs. passwordless login via SSH keys, and gave tips to set up the latter.","The contrast adjusted images on MAC::preview should be saved and uploaded back to JHPCE to run spaceranger. Spaceranger needs the exact same image that was used in the loupe alignment (even if it is changed from raw image on preview)","I described how to add additional transcripts to the relevant FASTA file then use SPEAQeasy's custom annotation option to use the new FASTA with the other default annotation files.","Explained how the image data can be manipulated when claudin5 channel and WFA channel are merged and then the blood vessels are segmented and masked. The correct way is to segment blood vessels from claudin channel and mask the segmented objects in WFA channel.","The indices listed in the tiles's filename is the x,y coordinates of the starting topleft pixel of the given tile. These indices never start with [0,0] as these tiles are picked from the ROI in the phenochart and not the entire imaged area. Once stored in a directory these tiles are not arranged in chronological order.","I described how requesting 'caracol' is similar to requesting 'bluejay', but elaborated on additional complications, like contention for the same GPU and using CUDA_VISIBLE_DEVICES properly, including with array jobs","I described different strateguies for helping others navigate the world of data science. In particular, how weekly coding sessions can be really useful.","We went over the files (data and script) for the deconvolution analysis done for the habenula in BSPIII. We also discussed the theroy/methodology for deconvolution.","Created an R script that reads in the transcripts FASTA file from GENCODE and uses BSgenome from Bioconductor to edit it and save it as a larger .fasta.gz file. Nick Eagles helped us by pointing us to http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation.","Linked to https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13, `/dcs04/lieber/lcolladotor/qSVA_LIBD3080/degradation_experiments/*/bed`, https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855, https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5, https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R, https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R, https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253, https://github.com/CRG-Barcelona/bwtool/wiki#installation, http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html, and http://research.libd.org/qsvaR/. Andrew Jaffe linked `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/degradation_mat_CMC_ribozero.rda`, `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/run_de_analysis_CMC.R`, `/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/geneLevel_CMC_qSVsAndMod.rda` and `/users/ajaffe/Lieber/Projects/RNAseq/CMC/`.","Linked to https://github.com/LieberInstitute/qsvaR, http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R, https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28. Made public the file `/dcl01/ajaffe/data/lab/qsva_brain/brainseq_phase2_qsv/rdas/degradation_rse_phase2_usingJoint_justFirst.rda` and other ones. See https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5 for details.","Heena was encountering a simple error installing the package. We also spent time going through the vignette to give a conceptual overview of qsva. Her feedback will be helpful in making the vignette more accessible","I referred her to the '--coverage' option with SPEAQeasy, which unfortunately cannot be run alone, which might be useful for an already-processed dataset","Used KJ's code from his rstats club presentation and described how the images shoukd be read using numpy array as uint32 but not floating https://colab.research.google.com/drive/1A4cavWcbgZF1PFQT87s5s_ATyZrN3IxL?usp=sharing","Using BiocMAP http://research.libd.org/BiocMAP/ which Nick is finishing up, you could process the data and have R objects you could load and analyze. So, the data processing would be \"easy\" in a sense. Next, you might want to check the bsseq documentation https://bioconductor.org/packages/bsseq/ if you are interested in finding differentially methylated regions (DMRs) and in any case, because BiocMAP provides that type of R object as noted at http://research.libd.org/BiocMAP/outputs.html. https://jhubiostatistics.shinyapps.io/wgbsExprs/ and the companion paper is the last WGBS data analysis project I did myself with code at https://github.com/LieberInstitute/brain-epigenomics. Kira Perzel-Mandell, who worked with Shizhong, wrote https://github.com/LieberInstitute/wgbs_meqtl_analysis (link to paper there) and did other projects. If you are plotting methylation levels at some locations, then that's very direct. If you are trying to find DMRs, it can take a bit of time to get used to the large objects and the related complications (once you filter the data, it'll be more manageable). If you go into CpH instead of CpG, then memory constraints become harder to deal with.","We discussed how to build and save data with a for loop and with purrr::map. Went over how to define a function in map to do nested loops","Geo was interested int eh workflow I developed for tensorQTL. I went over the requirements and input formats that tensorqtl requirers, and how I had previously made them from our dataset. Also discussed the differnces between the functions and their outputs.","used HALO to show how contrast can be adjusted when changing min max values in the image","NA",null,"Related to JHPCE https://github.com/ntdyjack/fasthplus and http://research.libd.org/sgejobs/",null,"Brainstormed several ideas related to deconvolution of bulk data as well as uses of snRNA-seq. Refered to OSCA https://bioconductor.org/books/release/OSCA/ at time.","We went over how to acess the rowData, and what the rowData inthe MDD project contains. Ended up adding columns to the eqtl risk snp subsets to help with analysis",null,"https://www.nature.com/articles/s41592-021-01358-2",null,null,null,"We tried to discover the source of a seg fault that would occur when she imported tensorqtl in python with the tensorqtl module. Another user and I weren't able to replicate the issue. We later discovered that there were lines in her .bashrc that might've interfered with the module to produce the issue.","I described how to qrsh/sub jobs with the GPU node, including how to automatically set up 'CUDA_VISIBLE_DEVICES' to avoid contention for resources, and how to use the tensorqtl module to ensure GPUs were used when importing pytorch","After ensuring our apparent Rstudio set-ups on our windows machines matched, she still wasn't able to forward X11 while using Rstudio. Regarding a different issue, we discovered that a particular machine required additional lines in the '~/.ssh/config' to properly enable RSA keys.","explained the functions of this pipline as these are not yet documented","I described how it's accessed as a queue, but some manual work is left to the user (checking which GPUs are open and setting 'CUDA_VISIBLE_DEVICES')","command to use `rsync -avzh --progress userid@jhpce-transfer01:/source directory/ /destination directory/`","use rsync, prepend spaces in the paths with `\\` to avoid issues","https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/43106c664883fd5df301e44cca35f2e8f2f289ad","explain how to use IDE's on local machine to edits scripts on JHPCE through cyberduck","Fixed row names to both be ensembl IDs, discussed simplifing cell types in sce, converting sce arrarys to matrix","https://github.com/LieberInstitute/CaPTure/issues/2","I shared some detials of our deconvolution approach, we walked through the SCE to Expression set converson, and discussed some issues that may come up","I described how I normally deduce sample IDs from FASTQ filenames when creating a 'samples.manifest' file for SPEAQeasy, described the difference between '--trim_mode' settings that SPEAQeasy supports, and showed how to check what samples were trimmed.","refining segmentations in matlab by applying size thresholds","Showed code how to scale data based on https://uc-r.github.io/kmeans_clustering and optimise kmeans for the present dataset","Motivated /dcs04/lieber/lcolladotor/_jhpce_org_LIBD001/update_permissions_spatialteam.sh","Showed the steps on one sample HPC image and what outputs to save.","While debugging, its seen that VistoSeg when splitting the raw image missed a part of fiducial frame for capture area D1 of that sample. This happens when the center of the visium slide does not align with the center of the image, which has ti be adjusted while imaging. This image when run through spaceranger provided negative coordinates for the missing fiducial frame part of the image, but MATLAB will not accept negative indices for a image, which was throwing errors when recontructing spot grind in countSpots function.","Showed how to close issues on git with detailed comments of the final results","I described using regular expressions in R to match files in a directory. We encountered a weird case where 'list.files' seemed to return different numbers of matched files when run repeatedly. After ensuring we were in the right directory, the issue went away, but it was unclear what caused the original problem.","made custom Matlab script to batch process her 30 RNAscope images on her local machine","The user directories were cleared during the software updates for LIBD server, emailed bill to create one for me and Heena to use rsync on Neural_Plasiticity. The filenames Heena used for 'source' in rync did not exist at the source directory which was throwing errors.","to pick filename from textfile for a array job","We discussed how to check for duplicate rownames, and stratedgies for variable naming + working with rse objects","We found a R function that helped produce the plot she was looking for","Two main issues were discussed: **1**. After batch correction there is one sample that clusters mostly independently from the other samples. This sample did not stand out in QC (althogh this will be re-examined). There is some evidence that this donor might have been clinacally different from the other donors, and since the batch correction worked well for the other samples, there is the possibility that this is real biologal signal. However since there is only one such sample it would be really hard to confirm this, and the inclusion of this sample would counfound the biologcal question of interest. I advisied to drop this sample post-clustering on the grounds that this donor might be too differnt from the others. **2**. Prelimnary results from DEG by cell type for the three groups find mitocondral genes are the most differntly expressed. They are concerned as MT RNA could be a technical artifact (contamination) this tech isn't the best to explore this. But there is are other findings that cellular metabolism may play a role in this condition. I advisied to keep the DEG results with the MT, add the caveat in the paper that this might not be a perfect experiment for this but it could support real differnces in the cells (plus they already have pretty strict MT QC). Also run a second DEA w/o MT genes to see what other signal exists.","The error came from the names of the clusters just being integers, the names for limma need to be sytactically valid (not just numbers, no spaces, no special characters - basically need to be variable names in R). I advised to simply add a prefix like we did in spatialDLPFC (sp9k1) or even simpler like \"c\" to her cluster names. Maybe we should add a warning to the function about cluster names since this error is pretty hard for users to de-bug and people often have non-sytactucally valid cluster names (especally with cell type names)","Explained some scripts such as /dcs04/lieber/lcolladotor/BrainSEQ_LIBD001/_jhpce_org",null,"Explaination of why an index label M=3 was chosen for refineVNS in section 2.2.","I explained how I get out of the situation where files that are too big get accidentally committed (using 'git log', 'git reset --soft', and other details)","We worked through an error message from the 'VariantsMerge' process, uncovering (arguably) a bug in SPEAQeasy, and I advised how to temporarily circumvent the problem. I also suggested a command for renaming a set of files that initially broke SPEAQeasy's assumptions.",null,"I talked about how Linux does not allow creation of files w/ execution permissions set, regardless of 'umask' settings such as in the '.bashrc'. I also explained how I use branching with git and other tips for managing a repo that others also develop on.","https://github.com/LieberInstitute/dotdotdot/issues/14",null,null,"We downloaded a metadata table from SRA and parsed it through R to programatically create a \"samples.manifest\" file for use in SPEAQeasy, that merged samples that were each split into multiple FASTQ files.",null,null,null,"https://files.slack.com/files-pri/T029RHR07-F024D7ELKC2/image_from_ios.jpg","used back quote instead of single quote when assigning string to variables","We decided to run CellProfiler on a while capture area image of DLPFC sample to look at the segmentations as it was doing good job on the cropped regions of a DLPFC sample","https://github.com/LieberInstitute/dotdotdot/issues/13",null,"SPEAQeasy would appear to run into \"file not found\" errors for just a handful of random samples in a large dataset. I eventually traced the error to a disk mounting issue on a particular compute node, which was resolved through bitsupport.","The H&E image that he was segmenting and the H&E image for which he gets the 'json' and 'csv' are not the same images or at least not the same resolution which is causing the error.","https://github.com/LieberInstitute/dotdotdot/issues/12",null,null,"used the command 'eval'","Did some troubleshooting on nested map functions, when to use ~.x and when to use function() syntax","I mentioned that she encountered a bug that was fixed in a more recent version of SPEAQeasy, but suggested she manually add a line of code temporarily so that she could resume the run before updating","I referred her to a script I had written in the past that contains the required commands and configuration  (with comments) for using the SRA toolkit",null,null,"Went over basic, complex, and nested map() usage and how to apply it to Josh's current work",null,"We were able to update the keys for the cluster, but the ssh key login was still broken",null,null,"We revised key features of Brenda Pardo's Twitter thread https://twitter.com/PardoBree/status/1388253938391175173?s=20 and some features that you would want your Twitter thread to have.","I showed a fairly standard method for concatenating tabular data from multiple files into a single matrix, and saving the resulting R object","We talked about all the details that go into writing an abstract for a conference. These details can take longer to complete than initially anticipated.","We revised how to make a new branch in Git working with https://github.com/lmweber/OSTA-book","We discussed the current DNA genotype pipeline and potential changes others might want.","ibidem","https://github.com/LieberInstitute/dotdotdot/issues/11","From SPEAQeasy, to qSVA, deconvolution, TWAS, etc. It might be best to give a talk about all these topics to whoever is interested at LIBD since we know about all these pieces but others might not.","showed a example of how AWK extracts text lines from a text file","Explain how some scripts in SUFI run and the bash scripts to use in place of python scripts for batch processing","Referred to Vijay's notes",null,null,"I described correct use of the \"--input\" argument, and we also found that many of the input FASTQs were empty, leading to a somewhat vague error message.","In a series of email exchanges we diagnosed subtle errors resulting from bad FASTQ files and unintended options in the main script. Her feedback helped me improve some existing documentation.","I attempted some formatting tricks to eliminate unwanted whitespace but was ultimately unsuccessful. We suspected the LaTeX template required the whitespace to exist. We also tried to fix a bug with the header of a bookdown RMD not rendering as expected. It seemed that re-writing the header (without making any visible changes) fixed the issue, though we weren't sure of the source of the initial problem.","We talked about https://lcolladotor.github.io/bioc_team_ds/ among other topics.","used ggplot",null,null,"We confirmed that an existing VCF was corrupt and found that tabix expected bgzip-compressed VCF input rather than compressed BCF input.",null,"This has changed over time. The most recent setup (as of 2021-08-06) is at https://github.com/lmweber/OSTA-book","I briefly mentioned the \"#$ -o\" and \"#$ -e\" arguments, and how several logs will generate in your home directory without those arguments and without \"#$ -cwd\", when \"qsub\"ing a job at JHPCE.","We reviewed her code and I gave some pointer on what bugs to look out for with certain errors in ggplot","We talked about the set of videos and resources publicly available to get to learn about R, Bioconductor and these packages. It was a lot of information and I'm sure that we'll have more sessions later on to clarify questions as they arise.","We talked about https://choosealicense.com/, https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository, https://creativecommons.org/choose/.","We discussed the posibility of proposing a class that has a mix of biology lectures and some practical coding sessions. The coding sessions could be based after the CDSB 2021 mini courses series.","used Sys.glob and write.table to write the list to txt file",null,null,"We talked about the public data from Synapse and worked on locating some of it at JHPCE. I explained also how we have used bamtofastq when others don't share their FASTQ files. Basically, we use bamtofastq to reconstruct the original FASTQ files (you might have to append the unmapped FASTQs if they are provided by the original authors) and then run FASTQ-processing tools like our SPEAQeasy pipeline.","cleared up some confusion about filter_snps.R","We met to figure out how to transfer the data for this collaboration, which ended up being quite tricky and involved using an SSH tunnel for rsync. Basically, a command like this one: `rsync -avzh -e 'ssh -o ProxyJump=user@server_somewhere_for_the_tunnel' *_files_of_interest user@final_server:/final_path/`","We discussed the different possible goals and ways to get to them and determined a path forward that relies on the rest of the DSgs system as well as existing LIBD rstats club videos to get started. It'll be supported by a meeting with Leo and others involved in the spatial transcriptomics projects from the R side.","I showed the initial version of the BrainSeq Phase II developmental expression explorer developed by Bill Ulrich and Aaron Goldman. We discussed other possibilities for the collaboration including helping set up SPEAQeasy in their servers.","Pointed her to some oacjages for creating TWAS plots whe was interested in and we walked through some code I used to compare T-stats from different DE models","we talked about how to use the \"--strand\" option, the source of the error that was encountered","we discussed what a bfile is in plink and that made the process much easier.","https://github.com/LieberInstitute/zandiHyde_bipolar_rnaseq/tree/master/dev_twas#bd-twas-workflow","still working on this will touch base again, we are trying to obtain the mds plot for ancestry in the placental data.","the way we had run cluster profiler was generating lists in a way that the plot functions could not work with","N/A","We walked tought script from getRPKM to expression cut to filter rse objects, I also pointed Brenda towards some reasources to get started with ggplot","Used GenomeStudio again to produce PLINK files","We discussed the changes that we should make and the ones we shouldn't based on the feedback received from the practice talk with Keri's group. The feedback was excellent and very useful, however, we discussed which suggestions were maybe complicating the presentation too much and how to best incorporate them.",null,"run clusterprofiler on a set of DE hits",null,"Similar to part 2. The presentation was looking very polished by this meeting!","Helped review the grant application and revise the text.","Further updates on asking for help with this complicated project. Suggested reaching out to other LIBD members who have tried running the software before.","Discussed with Josh what figures we needed to prepare and how to generate them. This also involved base_size = 30 with ggplot2. Otherwise the font was too small to read it on the grant.","Now that a first major draft of the presentation was complete, we revised it and proposed some changes to it. Some of them involved remaking the figures using base_size = 20 or even 30 with ggplot2. For future work, it might be best to save the images with that font size from the beginning instead of having to update them later.","We reviewed the model that Sreya was evaluating for DE. As we as troublshooting the plot functions from the SPEAQeasy bootcamp code. I also advised taht she could create a volcano plot for her results.","We went through the creation of an LIBD JHPCE module, covering the documentation process, lua syntax, and how to test a module before pushing.","We adapted a series of bash scripts requiring parallel functionality for use at JHPCE. This involved correctly setting some path variables and requesting multiple cores through SGE for some scripts.","explained the process of using topmed imputation and later we will get the mds","I described how I typically construct a \"samples.manifest\" file and configure SPEAQeasy for a particular experiment.","We went over some plots we have made for DE previously, (quality control & PC scatter plots, t-stat scatter between models, venn diagrams, and boxplots). Talked about some things to watch out for in her data (small samples size). We are planning to meet again when her data is ready.","We went over the details of the GitHub actions workflow created by biocthis, the parts that you need to run from biocthis::use_bioc_pkg_templates(), and resolving issues prompted by R CMD check + BiocCheck","I gave an overview of the resources and materials we have created as well as the LIBD bootcamps that we have publicly available to help others get started with computational biology. This is likely the first of several meetings.","We spent a significant amount troubleshooting the problem here (nearly two hours?). In the end, using the insert image \"here\" LaTeX syntax resolved the problem.","General advice on how to ask for help from open source projects","Initial planning session for the LIBD seminar. We talked about the different slides we planned to show and the flow of the presentation. We also scheduled 3 practice talks with different groups of people.","We observed an issue where log parsing in SPEAQeasy behaved differently for her and I due to a difference in default python version loaded at JHPCE. I fixed the hisat2/2.2.1 module to ensure consistent python version use.","Overview of the TWAS pipeline","I showed how I typically produce an array job shell script for the situation where I have many samples of data to process the same way (SIMD). We talked about using a text file where each line contains one sample name, and referencing that in the array script.","I showed Sreya how to set up the S-TWAS directory that I adapted in the Zandi-Hyde bipolar project for her own adaptation. I also sent her some useful videos on how to set up GitHub with her JHPCE account and install an SFTP client for Ubuntu (I prefer Filezilla)","We discussed how to handle strandness issues, and how different inferred strand patterns within a group of samples sequenced together can indicate problems with the input data. I described how to use the '--force_strand' option with SPEAQeasy","We went through cloning SPEAQeasy, creating a \"samples.manifest\" file with R, and other relevant details for running SPEAQeasy at JHPCE. I also helped him set up git at JHPCE","We troubleshooted strange errors from the main SPEAQeasy output, and saw that I was able to run the main script (\"run_pipeline_jhpce.sh\") but she encountered errors with an identical script. We suspected permissions issues in her working directory, which she emailed bitsupport about.","I gave Sreya a conceptual introduction to transcriptome imputation and sent her some resources to help her get started on FUSION / TWAS","We got ssh key login working, added config files to JHPCE, went though a \"hello world\" example of using files on JHPCE on RStudio","Although we encountered permissions issues, I showed how to create a \"samples.manifest\" file and configure SPEAQeasy for a particular set of samples.",null,"Henry needed to reset his JHPCE password as he had forgotten it, but we went through RStudio set up and got cyberduck installed, we will set up a meeting next week for ssh keys and configuration","I showed my typical process for creating a \"samples.manifest\" file for a given dataset, and how to submit a pipeline run. We continued to debug some minor issues over slack",null,null,"Created a learning plan involving several team members that is now documented at a channel in JHU Genomics Slack. That way everyone can keep track of what others are teaching the person who asked the question, and we can have a common communication channel among everyone involved. Sessions include learning how to interact with JHPCE through RStudio, how to run SPEAQeasy, key R/Bioconductor packages (limma, SummarizedExperiment), and explore possible visualizations of interest.","I covered various nextflow concepts, as well as my typical approach in creating and running a nextflow project at JHPCE","explaination of nurolucida system, how number of cores and RAM matter for running SUFI",null,"Discussion about features that spatialLIBD could support to make it easier to use for newer data.","Went over basic functions of SummarizedExperiment and SingleCellExperiment, what a Sparse matrix is - how to convert to non-sparse. Steps to take to use getRPKM on SingeCellExperient.","sent him the scripts from andrew, wondering if I should make it into a function","Help with understanding [bookdown](https://bookdown.org/yihui/bookdown/) and ideas on how to organize a documentation website (chapters and information to consider).","Discussed the research project proposal and how we can help.","General advice. Received feedback regarding file backups.","explaination of scale_factors.json, tissuepositions_list.csv from spaceranger and the spotspotcheck package","I identified that https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829 contains both miRNA and RNA-seq data. Using data from the SRA Run selector at https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&o=acc_s%3Aa I was able to filter the data to the RNA-seq samples and subset the input samples.manifest file for SPEAQeasy.","To identify DEGs or eQTLs, the user has to choose what covariates to adjust for. Some studies might require more complicated models than others, as they might be from more than one brain region or present some batch effect. In addition, in newer projects we are considering adjusting for cell type fractions derived from deconvolution. Even if all the covariates were fixed across datasets, in DEGs we have typically adjusted for qSVs which are computed in a dataset-dependent fashion and in essence are PCs derived from degradation-associated genomic regions. Similarly, for eQTLs we adjust for expression PCs. In both we adjust for ancestry PCs. Computing PCs is input-dependent: you won't get the same values or PCs if you change which samples you are using. So, computing DEGs and eQTLs is not as simple as say a map-reduce problem where you compute the same thing for every input sample, then aggregate the per-sample results. We have steps that depend on the whole sample set. Then there's also steps for filtering expression features based on some minimum expression cutoffs. Plus, differences between say polyA and RiboZero data, which don't make sense to process together. For these reasons, I don't think that an on-demand computation pipeline is feasible based on a user selecting a set of input samples. Constructing RNA-seq objects (RangedSummarizedExperiment objects) as well as DNA genotyping VCF files based on a set of input samples is feasible and that's what others are building. After that, I think that it's best to adapt code we have (or even use newer tools) after carefully exploring the data.",null,"I described the need for the \"module use\" command and modifying the .bashrc to easily use LIBD modules","We walked througth the SpatialLIBD code to find where the argument has to be changed","I fetched the SRA files, created a manifest and execution script for SPEAQeasy, but had an issue running \"fastq-dump\". I decided to hand off the request, partially complete.","Failed MacOS installation on Lieber MacBook, something about Java both being and also not being installed","explaination of the functions VNS.m and refineVNS.m","when running VNS.m for several files on same matlab session the data from previous sessions is saved in matlab cache memory automatically. This will cause memory issues with subsequent runs, its important to clear variables after every run.","reffered to package \"effsize\" should be efficient at easily doing this","ran her through an example, explained her the code on github repo (split.m is the main script)","edited the config file for setting up x-quartz","Adapted code and shared it at https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock","He walked me through some code performing qSVA for a recent project, and I offered some tips involving code organization/ readibility.","Created GitHub issues at https://github.com/LieberInstitute/VisiumImgProcessing/issues","fixed the github repo","I showed them the tools I developed in DeconvoBuddies package, and provided links to example code where I had applied these funcitons in previous work","explore loading/saving/exporting options on Qupath","We discussed how Sciwheel can be adapted for different journals, and provided general advice for getting started and formatting a paper for a particular journal.","I explained that SPEAQeasy was used for the recent hg38 processing, and an older version of a similar pipeline was used for the original hg19 processing. We discussed our new method for retrieving gene symbols (\"org.*.eg.db\" packages) and some general code advice.","I went through the setup and example usage of Sciwheel and cross reference, and gave general advice (it's often easiest to start with the \"Methods\" section, etc)","use rsync","Debugging through the input/output file in logs","I explained how to use SPEAQeasy at JHPCE, and we tried an experimental method for downloading SRA data. The method didn't work, and we plan to meet again after I verify a more typical method works.","I hadn't encountered the issue with new local terminals failing to open. I planned to follow up with better specific advice via email, and offer to explain this advice in a future DSGS.","running chmod","Used Amy's rstats blogpost on git","We spent time doing basic Rstudio setup and troubleshooting, and got the data downloaded",null,"We got Heena's MAC registered for passwordless login to JHPCE",null,"We got the ssh keys working for Brenda's github","We went over applying DeconvoBuddies functions to hebenula data","Use of galaxybrain for identifying samples with DNAm, DNA genotype and RNA-seq data","We are going to try an iterative strategy to keep tables smaller",null,"[BioC Support website answer](https://support.bioconductor.org/p/p134069/#p134141)",null,"[BioC Support website answer](https://support.bioconductor.org/p/p133591/#p133927)","Used 'h5ls' command from shell to verify contents of file; eventually found we were looking at the wrong file (error message was cryptic about issue). https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1607034309074800","Need to resolve swap bugs. Plan to create a table of phenotype data (Dx, age, sex, race) for each BrNum to mach genotype data","Went through the setting up passwordless login for jhpce using ssh keys, and coppied the configuration files from https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee","[RStudio Community self-answer](https://community.rstudio.com/t/recipes-package-cannot-create-interaction-term-in-step-interact/89860)","got through the Github vignettes thoroughly",null,"[BioC Support website answer](https://support.bioconductor.org/p/p133110/#p133319)",null,"Started PsychENCODE access request process","[There is a plugin for Genome Studio by Illumina that allows users to output their imputed allele results as PLINK inputs](https://support.illumina.com/downloads/genomestudio-2-0-plug-ins.html)","[BioC Support website answer](https://support.bioconductor.org/p/134569/#p133028)","to use rsync or scp, you need to have user account created on LIBD server (Bill can help create it)",null,"We debugged some code and ultimetly decided that it might be best for Abby to write new code and use the tsne function to cluster and plot the data","I recomended using dcast to do a \"long\" to \"wide\" data transformation","We talked about how the data should be structured before using ggplot (transforming some data and adding catagorical columns)","[I recommended that they reach out to their forum board](https://support.bioconductor.org/p/p132558/#p132872)","[Check Slack helpdesk channel](https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1605123296024100)","[BioC Support website answer](https://support.bioconductor.org/p/134139/#p132829)","[GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/15)","We discussed organizing by analysis (a directory for each one, then prehaps sub- directory for data & plots). Create a readMe for each sub-folder. Use gitignore to keep repo from getting too big.","[BioC Support website answer](https://support.bioconductor.org/p/131600/#p132617)","https://www.mathworks.com/matlabcentral/answers/636365-how-can-i-measure-volume-from-a-3d-image?s_tid=srchtitle","Resulted in recount3 versions 1.1.1 and 1.1.2 on bioc-devel (and 1.0.2 on bioc-release)","[GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/13)","[GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/14)",null,"Explained the `chgrp` and `umask` commands plus `~/.bashrc` file configuration. I also mentioned why we don't use `chmod 777` at JHPCE. Details on [Slack](https://jhu-genomics.slack.com/archives/CR9NYA0BF/p1604086263148400?thread_ts=1603826219.094600&cid=CR9NYA0BF) (private channel).","[BioC Support website answer](https://support.bioconductor.org/p/p132487/#p132521)",null,null,"[BioC Support website answer](https://support.bioconductor.org/p/p132406/#p132459)","[BioC Support website answer](https://support.bioconductor.org/p/p132450/#p132464)","initial issue seemed to do with the spaceranger command arguments; we are continuining to investigate possible module issues or insufficient code in the [blog post](http://research.libd.org/rstatsclub/2020/10/20/using-space-ranger-at-jhpce/)","Needed to use highlight_key and highlight functions to select points between plots. Used rstats example as guide https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14",null,"[BioC Support website answer](https://support.bioconductor.org/p/134532/#134925)","[BioC Support website answer](https://support.bioconductor.org/p/134914/#134915)","[BioC Support website answer](https://support.bioconductor.org/p/131285/#134921)","[BioC Support website answer](https://support.bioconductor.org/p/134876/#134917)","spatstat package https://spatstat.org/releasenotes/spatstat-1.39-0.html, L function https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest, Kmark https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark"]},"columns":[{"id":"date","name":"date","type":"Date","style":"function(rowInfo, colInfo) {\nconst rowIndex = rowInfo.index + 1\n}","cell":["2024-04-16","2024-04-15","2024-04-11","2024-04-08","2024-04-05","2024-04-05","2024-04-05","2024-04-03","2024-04-02","2024-04-01","2024-03-29","2024-03-27","2024-03-27","2024-03-27","2024-03-25","2024-03-25","2024-03-25","2024-03-23","2024-03-22","2024-03-22","2024-03-20","2024-03-20","2024-03-20","2024-03-19","2024-03-19","2024-03-19","2024-03-15","2024-03-15","2024-03-14","2024-03-14","2024-03-14","2024-03-13","2024-03-12","2024-03-11","2024-03-11","2024-03-07","2024-03-07","2024-03-05","2024-03-05","2024-03-04","2024-03-01","2024-03-01","2024-02-28","2024-02-28","2024-02-26","2024-02-23","2024-02-16","2024-02-16","2024-02-16","2024-02-12","2024-02-09","2024-02-08","2024-02-07","2024-02-05","2024-02-02","2024-02-01","2024-01-30","2024-01-29","2024-01-26","2024-01-26","2024-01-25","2024-01-25","2024-01-24","2024-01-24","2024-01-22","2024-01-22","2024-01-18","2024-01-11","2024-01-08","2024-01-05","2023-12-12","2023-12-11","2023-12-08","2023-12-08","2023-12-08","2023-12-08","2023-12-05","2023-12-01","2023-11-30","2023-11-29","2023-11-29","2023-11-21","2023-11-16","2023-11-16","2023-11-15","2023-11-15","2023-11-08","2023-11-08","2023-11-08","2023-11-03","2023-11-01","2023-11-01","2023-11-01","2023-10-30","2023-10-30","2023-10-27","2023-10-27","2023-10-26","2023-10-26","2023-10-25","2023-10-25","2023-10-24","2023-10-20","2023-10-19","2023-10-18","2023-10-17","2023-10-16","2023-10-13","2023-10-13","2023-10-12","2023-10-12","2023-10-11","2023-10-11","2023-10-06","2023-10-05","2023-10-04","2023-09-29","2023-09-29","2023-09-28","2023-09-27","2023-09-27","2023-09-26","2023-09-25","2023-09-22","2023-09-22","2023-09-22","2023-09-21","2023-09-21","2023-09-21","2023-09-19","2023-09-19","2023-09-14","2023-09-13","2023-09-12","2023-09-06","2023-08-28","2023-08-23","2023-08-22","2023-08-21","2023-08-21","2023-08-18","2023-08-17","2023-08-16","2023-08-16","2023-08-14","2023-08-14","2023-08-11","2023-08-11","2023-08-10","2023-08-10","2023-08-09","2023-08-09","2023-08-09","2023-08-09","2023-08-07","2023-08-03","2023-08-02","2023-07-27","2023-07-27","2023-07-27","2023-07-27","2023-07-26","2023-07-26","2023-07-26","2023-07-26","2023-07-26","2023-07-25","2023-07-25","2023-07-24","2023-07-21","2023-07-20","2023-07-20","2023-07-20","2023-07-19","2023-07-18","2023-07-17","2023-07-17","2023-07-12","2023-07-07","2023-07-01","2023-06-29","2023-06-28","2023-06-28","2023-06-28","2023-06-27","2023-06-23","2023-06-22","2023-06-21","2023-06-15","2023-06-14","2023-06-09","2023-06-08","2023-06-08","2023-06-08","2023-06-08","2023-06-07","2023-06-02","2023-06-02","2023-06-01","2023-05-26","2023-05-25","2023-05-25","2023-05-24","2023-05-18","2023-05-18","2023-05-17","2023-05-16","2023-05-09","2023-05-02","2023-04-25","2023-04-21","2023-04-19","2023-04-17","2023-04-14","2023-04-12","2023-04-11","2023-04-10","2023-04-10","2023-04-07","2023-04-06","2023-04-06","2023-04-04","2023-03-31","2023-03-28","2023-03-24","2023-03-23","2023-03-23","2023-03-22","2023-03-22","2023-03-22","2023-03-21","2023-03-21","2023-03-21","2023-03-20","2023-03-15","2023-03-13","2023-03-13","2023-03-10","2023-03-09","2023-03-08","2023-03-02","2023-03-02","2023-03-02","2023-03-02","2023-03-02","2023-03-01","2023-03-01","2023-03-01","2023-02-28","2023-02-21","2023-02-17","2023-02-15","2023-02-10","2023-02-09","2023-02-06","2023-02-03","2023-02-01","2023-01-25","2023-01-25","2023-01-25","2023-01-20","2023-01-19","2023-01-18","2023-01-18","2023-01-18","2023-01-13","2023-01-11","2023-01-11","2023-01-10","2023-01-04","2023-01-03","2022-12-14","2022-12-14","2022-12-06","2022-12-05","2022-12-01","2022-11-30","2022-11-30","2022-11-30","2022-11-23","2022-11-16","2022-11-15","2022-11-03","2022-11-01","2022-10-28","2022-10-27","2022-10-26","2022-10-17","2022-10-14","2022-10-13","2022-10-11","2022-10-10","2022-10-05","2022-10-05","2022-09-30","2022-09-30","2022-09-29","2022-09-26","2022-09-23","2022-09-23","2022-09-16","2022-09-16","2022-09-12","2022-09-09","2022-09-06","2022-08-24","2022-08-24","2022-08-22","2022-08-22","2022-08-18","2022-08-17","2022-08-17","2022-08-10","2022-08-09","2022-08-09","2022-08-09","2022-08-08","2022-08-03","2022-07-28","2022-07-20","2022-07-19","2022-07-19","2022-07-18","2022-07-15","2022-07-15","2022-07-13","2022-07-12","2022-07-07","2022-07-07","2022-07-06","2022-07-06","2022-07-05","2022-07-01","2022-06-29","2022-06-29","2022-06-28","2022-06-23","2022-06-23","2022-06-21","2022-06-21","2022-06-17","2022-06-15","2022-06-15","2022-06-15","2022-06-15","2022-06-10","2022-06-03","2022-06-03","2022-06-02","2022-06-01","2022-05-27","2022-05-27","2022-05-27","2022-05-27","2022-05-25","2022-05-20","2022-05-20","2022-05-18","2022-05-18","2022-05-17","2022-05-17","2022-05-17","2022-05-17","2022-05-10","2022-05-09","2022-05-06","2022-05-05","2022-05-05","2022-05-03","2022-04-29","2022-04-29","2022-04-28","2022-04-27","2022-04-27","2022-04-27","2022-04-21","2022-04-20","2022-04-15","2022-04-15","2022-04-08","2022-04-08","2022-04-08","2022-04-07","2022-04-07","2022-04-07","2022-04-06","2022-04-06","2022-04-06","2022-04-05","2022-04-04","2022-04-04","2022-04-01","2022-03-27","2022-03-25","2022-03-24","2022-03-22","2022-03-18","2022-03-18","2022-03-17","2022-03-17","2022-03-16","2022-03-16","2022-03-15","2022-03-15","2022-03-14","2022-03-11","2022-03-09","2022-03-09","2022-03-09","2022-03-09","2022-03-09","2022-03-04","2022-03-04","2022-03-04","2022-03-04","2022-03-03","2022-03-03","2022-03-02","2022-03-02","2022-02-25","2022-02-24","2022-02-24","2022-02-24","2022-02-23","2022-02-18","2022-02-17","2022-02-16","2022-02-16","2022-02-15","2022-02-15","2022-02-15","2022-02-14","2022-02-11","2022-02-11","2022-02-10","2022-02-07","2022-02-04","2022-02-04","2022-01-25","2022-01-21","2022-01-21","2022-01-21","2022-01-21","2022-01-21","2022-01-20","2022-01-20","2022-01-19","2021-12-15","2021-12-09","2021-12-08","2021-12-06","2021-11-18","2021-11-18","2021-11-18","2021-11-17","2021-11-16","2021-11-15","2021-11-04","2021-11-03","2021-10-29","2021-10-25","2021-10-23","2021-10-20","2021-10-14","2021-10-07","2021-10-06","2021-10-04","2021-09-27","2021-09-22","2021-09-20","2021-09-14","2021-09-14","2021-09-10","2021-09-09","2021-09-09","2021-09-09","2021-09-09","2021-09-09","2021-09-08","2021-09-07","2021-09-03","2021-09-03","2021-09-02","2021-09-02","2021-08-30","2021-08-30","2021-08-30","2021-08-26","2021-08-26","2021-08-25","2021-08-23","2021-08-23","2021-08-23","2021-08-21","2021-08-20","2021-08-18","2021-08-11","2021-08-06","2021-08-06","2021-08-03","2021-07-30","2021-07-28","2021-07-28","2021-07-20","2021-07-20","2021-07-15","2021-07-15","2021-07-14","2021-07-14","2021-07-14","2021-07-14","2021-07-14","2021-07-13","2021-07-13","2021-07-09","2021-07-09","2021-07-09","2021-07-02","2021-07-01","2021-07-01","2021-06-30","2021-06-30","2021-06-29","2021-06-29","2021-06-29","2021-06-24","2021-06-24","2021-06-24","2021-06-21","2021-06-21","2021-06-17","2021-06-17","2021-06-17","2021-06-17","2021-06-16","2021-06-16","2021-06-15","2021-06-10","2021-06-05","2021-06-04","2021-06-03","2021-06-02","2021-06-02","2021-06-01","2021-06-01","2021-05-28","2021-05-26","2021-05-24","2021-05-21","2021-05-20","2021-05-20","2021-05-20","2021-05-19","2021-05-19","2021-05-14","2021-05-13","2021-05-12","2021-05-10","2021-05-07","2021-05-07","2021-05-06","2021-05-06","2021-05-03","2021-05-03","2021-04-28","2021-04-28","2021-04-26","2021-04-22","2021-04-22","2021-04-20","2021-04-19","2021-04-19","2021-04-15","2021-04-15","2021-04-12","2021-04-09","2021-04-09","2021-04-08","2021-04-07","2021-04-01","2021-03-30","2021-03-29","2021-03-26","2021-03-26","2021-03-25","2021-03-25","2021-03-25","2021-03-24","2021-03-24","2021-03-24","2021-03-23","2021-03-22","2021-03-19","2021-03-18","2021-03-18","2021-03-17","2021-03-17","2021-03-16","2021-03-12","2021-03-12","2021-03-11","2021-03-10","2021-03-10","2021-03-06","2021-03-05","2021-03-05","2021-03-03","2021-03-02","2021-03-02","2021-02-26","2021-02-26","2021-02-25","2021-02-23","2021-02-19","2021-02-12","2021-02-09","2021-02-05","2021-02-05","2021-02-05","2021-02-04","2021-02-04","2021-01-29","2021-01-27","2021-01-26","2021-01-14","2021-01-11","2021-01-06","2020-12-09","2020-12-08","2020-12-07","2020-12-03","2020-12-03","2020-12-03","2020-12-02","2020-12-01","2020-11-25","2020-11-23","2020-11-20","2020-11-19","2020-11-19","2020-11-18","2020-11-13","2020-11-12","2020-11-12","2020-11-11","2020-11-11","2020-11-11","2020-11-05","2020-11-04","2020-11-04","2020-11-03","2020-11-03","2020-11-03","2020-11-02","2020-10-30","2020-10-30","2020-10-28","2020-10-28","2020-10-28","2020-10-28","2020-10-28","2020-10-27","2020-10-26","2020-10-21","2020-10-21","2020-10-21","2020-10-21","2020-10-20"],"html":true,"width":120,"align":"right","headerStyle":{"font-weight":"normal"}},{"id":"guide","name":"guide","type":"character","style":"function(rowInfo, colInfo) {\nconst rowIndex = rowInfo.index + 1\n}","cell":["Leo","Leo","Cynthia","Leo","Cynthia","Cynthia","Leo","Leo","Leo","Leo","Leo","Louise","Maddy","Cynthia","Maddy","Leo","Cynthia","Louise","Leo","Cynthia","Nick","Leo","Leo","Leo","Leo","Nick","Leo","Nick","Leo","Leo","Louise","Nick","Leo","Maddy","Louise","Leo","Nick","Leo","Nick","Nick","Maddy","Leo","Louise","Leo","Leo","Leo","Leo","Leo","Nick","Nick","Hedia","Leo","Cynthia","Nick","Nick","Nick","Nick","Leo","Nick","Nick","Leo","Louise","Leo","Nick","Louise","Louise","Hedia","Leo","Nick","Nick","Nick","Nick","Nick","Hedia","Nick","Nick","Nick","Leo","Louise","Leo","Louise","Nick","Leo","Leo","Leo","Louise","Leo","Leo","Nick","Leo","Leo","Leo","Nick","Nick","Nick","Leo","Leo","Leo","Leo","Nick","Louise","Nick","Leo","Louise","Leo","Nick","Nick","Nick","Nick","Leo","Louise","Nick","Nick","Leo","Louise","Nick","Leo","Louise","Nick","Leo","Nick","Nick","Nick","Louise","Leo","Leo","Leo","Louise","Louise","Nick","Nick","Leo","Leo","Nick","Louise","Nick","Nick","Nick","Louise","Leo","Leo","Nick","Nick","Nick","Nick","Leo","Nick","Nick","Louise","Leo","Nick","Louise","Maddy","Maddy","Nick","Maddy","Maddy","Leo","Maddy","Nick","Leo","Louise","Louise","Leo","Leo","Leo","Louise","Nick","Maddy","Maddy","Maddy","Leo","Leo","Nick","Nick","Maddy","Nick","Nick","Maddy","Maddy","Leo","Leo","Leo","Louise","Leo","Leo","Leo","Leo","Louise","Louise","Nick","Nick","Louise","Leo","Leo","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Nick","Louise","Nick","Nick","Nick","Nick","Nick","Nick","Hedia","Ryan","Nick","Cynthia","Nick","Nick","Cynthia","Nick","Hedia","Nick","Louise","Hedia","Louise","Cynthia","Nick","Nick","Hedia","Hedia","Ryan","Louise","Nick","Leo","Leo","Leo","Leo","Nick","Hedia","Leo","Leo","Leo","Leo","Nick","Nick","Louise","Leo","Louise","Louise","Nick","Louise","Louise","Nick","Nick","Nick","Nick","Nick","Hedia","Nick","Louise","Louise","Nick","Hedia","Louise","Leo","Louise","Nick","Louise","Nick","Louise","Louise","Nick","Nick","Nick","Louise","Louise","Nick","Nick","Nick","Nick","Louise","Leo","Nick","Louise","Louise","Louise","Louise","Nick","Louise","Louise","Louise","Louise","Louise","Louise","Leo","Nick","Nick","Leo","Nick","Nick","Nick","Maddy","Leo","Leo","Nick","Nick","Louise","Louise","Louise","Leo","Louise","Maddy","Maddy","Maddy","Maddy","Louise","Maddy","Abby","Leo","Maddy","Nick","Louise","Leo","Nick","Leo","Louise","Louise","Nick","Leo","Louise","Nick","Abby","Nick","Louise","Nick","Maddy","Nick","Abby","Louise","Abby","Abby","Louise","Louise","Maddy","Leo","Louise","Louise","Louise","Leo","Leo","Abby","Leo","Leo","Abby","Leo","Louise","Leo","Abby","Abby","Leo","Maddy","Abby","Louise","Leo","Louise","Nick","Leo","Leo","Nick","Leo","Louise","Leo","Abby","Leo","Leo","Leo","Leo","Leo","Abby","Leo","Louise","Maddy","Maddy","Maddy","Leo","Louise","Maddy","Josh","Abby","Nick","Abby","Nick","Louise","Maddy","Abby","Leo","Abby","Nick","Leo","Nick","Leo","Louise","Nick","Leo","Nick","Maddy","Nick","Maddy","Maddy","Nick","Leo","Louise","Leo","Leo","Leo","Josh","Nick","Maddy","Leo","Louise","Louise","Maddy","Leo","Maddy","Leo","Maddy","Leo","Louise","Maddy","Maddy","Maddy","Maddy","Maddy","Nick","Nick","Nick","Maddy","Nick","Maddy","Maddy","Maddy","Maddy","Louise","Maddy","Louise","Nick","Maddy","Maddy","Leo","Maddy","Maddy","Maddy","Nick","Maddy","Maddy","Maddy","Louise","Louise","Louise","Louise","Leo","Maddy","Maddy","Nick","Nick","Leo","Nick","Maddy","Maddy","Maddy","Nick","Maddy","Maddy","Maddy","Maddy","Maddy","Maddy","Maddy","Arta","Nick","Maddy","Maddy","Maddy","Maddy","Maddy","Louise","Nick","Nick","Maddy","Maddy","Louise","Maddy","Louise","Maddy","Maddy","Leo","Nick","Leo","Leo","Leo","Josh","Maddy","Leo","Maddy","Maddy","Maddy","Maddy","Maddy","Nick","Nick","Nick","Leo","Maddy","Maddy","Maddy","Nick","Arta","Leo","Nick","Louise","Leo","Leo","Leo","Maddy","Maddy","Maddy","Leo","Arta","Leo","Leo","Leo","Louise","Nick","Josh","Arta","Josh","Josh","Arta","Louise","Arta","Leo","Maddy","Josh","Louise","Leo","Leo","Leo","Leo","Leo","Louise","Nick","Nick","Josh","Nick","Louise","Leo","Leo","Leo","Leo","Leo","Nick","Leo","Nick","Arta","Nick","Nick","Nick","Arta","Louise","Nick","Maddy","Louise","Nick","Maddy","Maddy","Leo","Nick","Maddy","Maddy","Leo","Louise","Josh","Leo","Leo","Leo","Maddy","Leo","Leo","Maddy","Nick","Louise","Nick","Arta","Maddy","Maddy","Josh","Maddy","Maddy","Leo","Nick","Leo","Maddy","Louise","Maddy","Nick","Nick","Nick","Maddy","Maddy","Nick","Nick","Maddy","Maddy","Louise","Leo","Maddy","Maddy","Louise","Louise","Leo","Louise","Leo","Leo","Leo","Nick","Nick","Louise","Louise","Arta","Maddy","Maddy","Nick","Maddy","Leo","Arta","Nick","Maddy","Maddy","Louise","Louise","Louise","Arta","Leo","Nick","Leo","Louise","Nick","Maddy","Leo","Leo","Leo","Nick","Leo","Leo","Maddy","Josh","Nick","Leo","Nick","Louise","Nick","Louise","Leo","Nick","Arta","Louise"],"html":true,"width":100,"align":"left","headerStyle":{"font-weight":"normal"}},{"id":"help_request","name":"help_request","type":"character","style":"function(rowInfo, colInfo) {\nconst rowIndex = rowInfo.index + 1\n}","cell":["<div class='gt_from_md'><p>Review of SCZD vs NTC DE models<\/p>\n<\/div>","<div class='gt_from_md'><p>Options for creating links to use in manuscripts<\/p>\n<\/div>","<div class='gt_from_md'><p>Loupe Browser re-analizing and advanced options<\/p>\n<\/div>","<div class='gt_from_md'><p>Reference letter request<\/p>\n<\/div>","<div class='gt_from_md'><p>Inspect multimome objects with Browse Loupe<\/p>\n<\/div>","<div class='gt_from_md'><p>Support with Config Files<\/p>\n<\/div>","<div class='gt_from_md'><p>Updating ranger_metrics information<\/p>\n<\/div>","<div class='gt_from_md'><p>Avoid using jsonlite when editing spaceranger json files<\/p>\n<\/div>","<div class='gt_from_md'><p>Write a script to manually edit spaceranger json files<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging spaceranger count<\/p>\n<\/div>","<div class='gt_from_md'><p>Understanding a spaceranger error<\/p>\n<\/div>","<div class='gt_from_md'><p>access bulk RNAseq from samples cataloged in the Lieber-internal bulk rnaseq database<\/p>\n<\/div>","<div class='gt_from_md'><p>debugging samui script for HPC HE data<\/p>\n<\/div>","<div class='gt_from_md'><p>Question about the correct number of reads in sequencing for scRNAseq and scATACseq multiome strategy<\/p>\n<\/div>","<div class='gt_from_md'><p>debugging samui script for HPC HE data<\/p>\n<\/div>","<div class='gt_from_md'><p>LIBD Data Portal for BSP1 and BSP2 DLPFC SCZD vs NTC data<\/p>\n<\/div>","<div class='gt_from_md'><p>Cell Ranger-ARC html report<\/p>\n<\/div>","<div class='gt_from_md'><p>slurm job loops<\/p>\n<\/div>","<div class='gt_from_md'><p>Polishing details on how to ask for help effectively<\/p>\n<\/div>","<div class='gt_from_md'><p>GEX and ATAC clustering<\/p>\n<\/div>","<div class='gt_from_md'><p>More troubleshooting misaligned image<\/p>\n<\/div>","<div class='gt_from_md'><p>Cg_Hb_projector_profiling progress update<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview of use cases for JHPCE disk usage data over time<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq data analysis steps overview<\/p>\n<\/div>","<div class='gt_from_md'><p>Cloud computing vs JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Troubleshooting misaligned image in Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>Advice prior to a faculty interview<\/p>\n<\/div>","<div class='gt_from_md'><p>Plotting to a multi-page PDF with 3 plots per page<\/p>\n<\/div>","<div class='gt_from_md'><p>Derive code for how to merge ERCCsumLogErr values across sequencing lanes for a given sample<\/p>\n<\/div>","<div class='gt_from_md'><p>Help identifying differences between two RSE objects containing data from the same project<\/p>\n<\/div>","<div class='gt_from_md'><p>I tried to apply ‚ÄúqSV_model_DE_analysis.R‚Äù in lithium-exposed samples ( from Dpseq data), but I was confused by some of the code, especially ‚Äúrun_DE.R.‚Äù<\/p>\n<\/div>","<div class='gt_from_md'><p>Installing cellprofiler at JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>spatialDLPFC and deconvolution-benchmark project details on how snRNA-seq data was used for each of them<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE login questions<\/p>\n<\/div>","<div class='gt_from_md'><p>quick big fix. Its the same eQTL code ‚Äì just one header in a file that‚Äôs always missing<\/p>\n<\/div>","<div class='gt_from_md'><p>Grant related discussion<\/p>\n<\/div>","<div class='gt_from_md'><p>Adding notes to Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>Cg_Hb_projector_profiling progress update<\/p>\n<\/div>","<div class='gt_from_md'><p>Better algorithm for dealing with overlapping images in Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up SSH keypair for passwordless login to JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Vistoseg parameter tuning<\/p>\n<\/div>","<div class='gt_from_md'><p>Feedback on the nascentRNA project<\/p>\n<\/div>","<div class='gt_from_md'><p>Deconvolution<\/p>\n<\/div>","<div class='gt_from_md'><p>How to extract normalized expression values with limma<\/p>\n<\/div>","<div class='gt_from_md'><p>Prospective JHU PhD student<\/p>\n<\/div>","<div class='gt_from_md'><p>Cg_Hb_projector_profiling progress update<\/p>\n<\/div>","<div class='gt_from_md'><p>Cg_Hb_projector_profiling progress update<\/p>\n<\/div>","<div class='gt_from_md'><p>qsvaR bug report resolution<\/p>\n<\/div>","<div class='gt_from_md'><p>Adding phenotype data to coldata<\/p>\n<\/div>","<div class='gt_from_md'><p>Adding packages to a custom R module<\/p>\n<\/div>","<div class='gt_from_md'><p>Rownames not well formatted<\/p>\n<\/div>","<div class='gt_from_md'><p>Cg_Hb_projector_profiling progress update<\/p>\n<\/div>","<div class='gt_from_md'><p>Discuss Seurat, cell type markers, DE<\/p>\n<\/div>","<div class='gt_from_md'><p>Follow-up: removing large files from version control<\/p>\n<\/div>","<div class='gt_from_md'><p>Troubleshooting GitHub authentication issues<\/p>\n<\/div>","<div class='gt_from_md'><p>Removing large files from version control<\/p>\n<\/div>","<div class='gt_from_md'><p>Working with SLURM<\/p>\n<\/div>","<div class='gt_from_md'><p>Career development<\/p>\n<\/div>","<div class='gt_from_md'><p>Running python on cluster<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating a module<\/p>\n<\/div>","<div class='gt_from_md'><p>Permutation analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>making a corrplot with a lot of data points .<\/p>\n<\/div>","<div class='gt_from_md'><p>Review response to qsvaR bug report from last year<\/p>\n<\/div>","<div class='gt_from_md'><p>Follow up on stitching for Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>Maybe divinding them by sample name may help.<\/p>\n<\/div>","<div class='gt_from_md'><p>when I try to rerun R dreamlet pseudobulk on the new JHPCE, I keep getting errors‚Ä¶ (provided code example)<\/p>\n<\/div>","<div class='gt_from_md'><p>Reviewing analysis using qsvaR package<\/p>\n<\/div>","<div class='gt_from_md'><p>Reviewing updates to the qsvaR package<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating stitched image in Samui from ImageJ estimates<\/p>\n<\/div>","<div class='gt_from_md'><p>Investigating visual problems in Samui image<\/p>\n<\/div>","<div class='gt_from_md'><p>Help adding phenotype info to colData<\/p>\n<\/div>","<div class='gt_from_md'><p>Investigating R package installation conflicts<\/p>\n<\/div>","<div class='gt_from_md'><p>Investigating error in qSVA funtion call<\/p>\n<\/div>","<div class='gt_from_md'><p>Fix misleading getDegTx error message<\/p>\n<\/div>","<div class='gt_from_md'><p>Performing ‚Äòjaffelab::s‚Äô on large tibble efficiently<\/p>\n<\/div>","<div class='gt_from_md'><p>Follow up on stitching for Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating stitched image in Samui from ImageJ estimates<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE R modules<\/p>\n<\/div>","<div class='gt_from_md'><p>Regarding the transcriptome assay from the multiome approach, I have selected canonical gene markers from the different samples; Moving further, I would like to integrate to the analysis external gene-markers shared by S.Page. I am looking the best way to do this.<\/p>\n<\/div>","<div class='gt_from_md'><p>CV writing advice<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with qSVA overview schematic<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview of image stitching process<\/p>\n<\/div>","<div class='gt_from_md'><p>Cg_Hb_projector_profiling progress update<\/p>\n<\/div>","<div class='gt_from_md'><p>Review how to respond to a user‚Äôs bug report<\/p>\n<\/div>","<div class='gt_from_md'><p>Job application support<\/p>\n<\/div>","<div class='gt_from_md'><p>I would like to discuss with you the correlation issue related to the cellular_component model of the qsvaR package and get your feedback.<\/p>\n<\/div>","<div class='gt_from_md'><p>Job application support<\/p>\n<\/div>","<div class='gt_from_md'><p>Cg_Hb_projector_profiling progress update<\/p>\n<\/div>","<div class='gt_from_md'><p>Modifying a global-scope variable inside a function<\/p>\n<\/div>","<div class='gt_from_md'><p>Review replication analysis R script<\/p>\n<\/div>","<div class='gt_from_md'><p>Cg_Hb_projector_profiling project overview<\/p>\n<\/div>","<div class='gt_from_md'><p>Review Visium SPG AD blog post<\/p>\n<\/div>","<div class='gt_from_md'><p>Issues linking repo to github<\/p>\n<\/div>","<div class='gt_from_md'><p>Interpreting decision tree and spot-deconvo plots<\/p>\n<\/div>","<div class='gt_from_md'><p>Issues with tensorqtl module<\/p>\n<\/div>","<div class='gt_from_md'><p>Interaction modeling with limma-voom<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq OSCA-based analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>Review replication analysis R script<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging a ranger_metrics R script<\/p>\n<\/div>","<div class='gt_from_md'><p>Interpretation of ‚Äúreplication‚Äù plots and performance of QSVA models<\/p>\n<\/div>","<div class='gt_from_md'><p>Hi! We would love an opinion on our (now cell-type assigned) single nuclei rna-seq data, i.e. how they look - if anything is off, advice for improvement‚Ä¶<\/p>\n<\/div>","<div class='gt_from_md'><p>Example datasets and getting started with data science<\/p>\n<\/div>","<div class='gt_from_md'><p>Review replication analysis scripts for the qsvaR project<\/p>\n<\/div>","<div class='gt_from_md'><p>Help migrating to SLURM setup<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq OSCA-based analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>Nextflow issue when running SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up and running SPEAQeasy on a new cluster<\/p>\n<\/div>","<div class='gt_from_md'><p>Installing SPEAQeasy and setting up main script<\/p>\n<\/div>","<div class='gt_from_md'><p>Troubleshooting a SPEAQeasy error<\/p>\n<\/div>","<div class='gt_from_md'><p>qsvaR paper planning<\/p>\n<\/div>","<div class='gt_from_md'><p>Prepare expression bed files for tensorQTL<\/p>\n<\/div>","<div class='gt_from_md'><p>sbatch issue with SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Interpreting and refactoring ‚Äúconcordance‚Äù formulas in QSVA results<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq OSCA-based analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>Prepare covariate input files for tensorQTL<\/p>\n<\/div>","<div class='gt_from_md'><p>Refactoring QSVA code<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq OSCA-based analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>Interpretation of results plotted in inflection plots of Cellranger-arc data sets (GEX assays). For example, how do you know which genes are representing neurons or glia?<\/p>\n<\/div>","<div class='gt_from_md'><p>Dropping extra factor levels<\/p>\n<\/div>","<div class='gt_from_md'><p>Onboarding questions for a snRNA-seq dataset<\/p>\n<\/div>","<div class='gt_from_md'><p>Confidence metric in spot deconvo model<\/p>\n<\/div>","<div class='gt_from_md'><p>Array jobs over variables with SLURM<\/p>\n<\/div>","<div class='gt_from_md'><p>SGE vs. SLURM R package conflicts<\/p>\n<\/div>","<div class='gt_from_md'><p>help get normalized single cell data from a single cell rse for downstream eQTL<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq training<\/p>\n<\/div>","<div class='gt_from_md'><p>How to interpret terms on an interaction model<\/p>\n<\/div>","<div class='gt_from_md'><p>Guidance on expression heatmaps<\/p>\n<\/div>","<div class='gt_from_md'><p>Discuss deconvolution for some future work<\/p>\n<\/div>","<div class='gt_from_md'><p>help get normalized single cell data from a single cell rse for downstream eQTL<\/p>\n<\/div>","<div class='gt_from_md'><p>Customizing the home page for spatialLIBD apps<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview of the stitching process for split Visium samples<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq training<\/p>\n<\/div>","<div class='gt_from_md'><p>Social media ideas<\/p>\n<\/div>","<div class='gt_from_md'><p>SLURM array jobs and X11<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNAseq celltype grouping in DLPFC data<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating documentation sites with bookdown<\/p>\n<\/div>","<div class='gt_from_md'><p>Displaying info about cells, not spots in Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>Interpreting spot deconvolution results<\/p>\n<\/div>","<div class='gt_from_md'><p>I performed the whole analysis as previously established and also made an alternative version based on knowledge of OSCA book for single nuclei (with adaptive thresholds for QC and less tolerance for mito UMIs). Would you help me checking my QC thresholds and seeing if they make sense?<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq training<\/p>\n<\/div>","<div class='gt_from_md'><p>ranger_metrics<\/p>\n<\/div>","<div class='gt_from_md'><p>Missing genes in Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>My coding set up<\/p>\n<\/div>","<div class='gt_from_md'><p>Recommendations for learning more data science skills<\/p>\n<\/div>","<div class='gt_from_md'><p>Misc spot deconvolution questions<\/p>\n<\/div>","<div class='gt_from_md'><p>Getting started with snRNA-seq<\/p>\n<\/div>","<div class='gt_from_md'><p>recount 3 permissions<\/p>\n<\/div>","<div class='gt_from_md'><p>Using modules at SLURM and configuring bashrc<\/p>\n<\/div>","<div class='gt_from_md'><p>writing tweets efficiently<\/p>\n<\/div>","<div class='gt_from_md'><p>Introduction to what we do as a team and future plans<\/p>\n<\/div>","<div class='gt_from_md'><p>Using Samui from start to finish<\/p>\n<\/div>","<div class='gt_from_md'><p>Running DeconvoBuddies on the Mathy‚Äôs data<\/p>\n<\/div>","<div class='gt_from_md'><p>Adding ceil function to splitSlide to split arrays covering fiducials<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain the SPE organization and subsetting an SPE<\/p>\n<\/div>","<div class='gt_from_md'><p>Normalizing spot deconvolution results to answer biological questions of interest<\/p>\n<\/div>","<div class='gt_from_md'><p>memory error with VistoSeg::InFormStitch function<\/p>\n<\/div>","<div class='gt_from_md'><p>How to check if a visium slide data is already entered into REDCap<\/p>\n<\/div>","<div class='gt_from_md'><p>Guidance on reference data for deconvolution and snRNA-seq analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>insert offset in VistoSeg::splitSlide_IF<\/p>\n<\/div>","<div class='gt_from_md'><p>Quantifying spatial promiscuity of genes across clusters<\/p>\n<\/div>","<div class='gt_from_md'><p>Feedback on an R package + GO/KEGG enrichment analyses<\/p>\n<\/div>","<div class='gt_from_md'><p>Feeback on spatial data + code organization<\/p>\n<\/div>","<div class='gt_from_md'><p>Feedback on workshop vignette<\/p>\n<\/div>","<div class='gt_from_md'><p>Evaluate a potential external contractor<\/p>\n<\/div>","<div class='gt_from_md'><p>How to best document what skills I can perform or need to learn<\/p>\n<\/div>","<div class='gt_from_md'><p>Need help debugging iSEE app errors/warnings<\/p>\n<\/div>","<div class='gt_from_md'><p>Practice BioC talk<\/p>\n<\/div>","<div class='gt_from_md'><p>Troubleshooting GHA workflow errors when building workshop<\/p>\n<\/div>","<div class='gt_from_md'><p>REDCap build table error<\/p>\n<\/div>","<div class='gt_from_md'><p>NM pigement segmentation overview for LC samples<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview of image processing Visium and VSPG<\/p>\n<\/div>","<div class='gt_from_md'><p>Input on a course design<\/p>\n<\/div>","<div class='gt_from_md'><p>Feedback on responding to reviewers<\/p>\n<\/div>","<div class='gt_from_md'><p>Spot deconvolution workflow overview<\/p>\n<\/div>","<div class='gt_from_md'><p>Loading large files in R<\/p>\n<\/div>","<div class='gt_from_md'><p>tif2mat conversion<\/p>\n<\/div>","<div class='gt_from_md'><p>Sharing conda environment<\/p>\n<\/div>","<div class='gt_from_md'><p>Performing QSVA with genes or with different annotation<\/p>\n<\/div>","<div class='gt_from_md'><p>mat2tif conversion<\/p>\n<\/div>","<div class='gt_from_md'><p>0% Vs 5% overlap and repeated signal issues in VSPG images<\/p>\n<\/div>","<div class='gt_from_md'><p>How to make a project documentation website<\/p>\n<\/div>","<div class='gt_from_md'><p>clusterProfiler, set.seed, and BiocParallel questions<\/p>\n<\/div>","<div class='gt_from_md'><p>Resolving a permissions issue<\/p>\n<\/div>","<div class='gt_from_md'><p>About empty drops run with GEX cellranger-arc data<\/p>\n<\/div>","<div class='gt_from_md'><p>How to make dynamic error messages<\/p>\n<\/div>","<div class='gt_from_md'><p>How to make a new Globus endpoint for a publication.<\/p>\n<\/div>","<div class='gt_from_md'><p>How to filter an SCE object prior to making an iSEE app<\/p>\n<\/div>","<div class='gt_from_md'><p>How to debug an R package function from a user‚Äôs report<\/p>\n<\/div>","<div class='gt_from_md'><p>1/Learn about the roxygen2 syntax for examples. 2/Run devtools::document() like in dev/04_update.R (maybe also auto-style code). 3/Edit the news.md file + bump version on the DESCRIPTION (and change the date). 4/Learn about how to sync her local repo with steps like <a href=\"http://contributions.bioconductor.org/git-version-control.html#new-package-workflow\">http://contributions.bioconductor.org/git-version-control.html#new-package-workflow<\/a> (aka register to <a href=\"https://git.bioconductor.org/BiocCredentials/login/?next=/BiocCredentials/\">https://git.bioconductor.org/BiocCredentials/login/?next=/BiocCredentials/<\/a> like @Daianna Gonz√°lez + add the upstream remote + git push to both upstream and origin).<\/p>\n<\/div>","<div class='gt_from_md'><p>I created this github repo: <a href=\"https://github.com/LieberInstitute/neurovasculature-pilot/tree/main\">https://github.com/LieberInstitute/neurovasculature-pilot/tree/main<\/a>. Also, integrated it with slack, but instead of integrating the main branch, I linked the devel branch, and I am unable to see the commits that I make on slack. They are updated on github, though. So, I need help with the integration.<\/p>\n<\/div>","<div class='gt_from_md'><p>Displaying spot coordinates in Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>Using the Samui API to display data locally<\/p>\n<\/div>","<div class='gt_from_md'><p>Options for cell type specific eQTLs<\/p>\n<\/div>","<div class='gt_from_md'><p>Options for cell type specific eQTLs<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview of what we do as a team and outreach options<\/p>\n<\/div>","<div class='gt_from_md'><p>Troubleshooting SPEAQeasy error<\/p>\n<\/div>","<div class='gt_from_md'><p>Ignoring a directory with gitignore<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging rewritten code<\/p>\n<\/div>","<div class='gt_from_md'><p>Troubleshooting unexpected SPEAQeasy output<\/p>\n<\/div>","<div class='gt_from_md'><p>Interpreting spot-deconvolution plots<\/p>\n<\/div>","<div class='gt_from_md'><p>Subsetting RSE by chromosome<\/p>\n<\/div>","<div class='gt_from_md'><p>SPEAQeasy defaults<\/p>\n<\/div>","<div class='gt_from_md'><p>Using Samui‚Äôs API to visualize spatial data<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating many configuration files differing by one variable<\/p>\n<\/div>","<div class='gt_from_md'><p>Merging two dataframes<\/p>\n<\/div>","<div class='gt_from_md'><p>Renaming many files by a pattern<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with SSH keys with GitHub<\/p>\n<\/div>","<div class='gt_from_md'><p>Overfitting in qSVA<\/p>\n<\/div>","<div class='gt_from_md'><p>Interpreting SPEAQeasy metrics<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating a blog post on the rstats website<\/p>\n<\/div>","<div class='gt_from_md'><p>Ensembl IDs vs. gene symbols with cell2location<\/p>\n<\/div>","<div class='gt_from_md'><p>Modules and reproducible software set-up for python<\/p>\n<\/div>","<div class='gt_from_md'><p>Properly tracking git contributions<\/p>\n<\/div>","<div class='gt_from_md'><p>When and how to merge git branches<\/p>\n<\/div>","<div class='gt_from_md'><p>code review<\/p>\n<\/div>","<div class='gt_from_md'><p>user had problem using import() function in R<\/p>\n<\/div>","<div class='gt_from_md'><p>Information needed for Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>Feedback the SmokingMouse Project websites<\/p>\n<\/div>","<div class='gt_from_md'><p>Properly transferring from dcl01 to dcs05,dcs04<\/p>\n<\/div>","<div class='gt_from_md'><p>Git-LFS error messages<\/p>\n<\/div>","<div class='gt_from_md'><p>Alternatives for RNA-seq analysis without replicates<\/p>\n<\/div>","<div class='gt_from_md'><p>Configuring X11 for plotting at JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Using the qsvaR with the new annotation<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating modules at JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>tensorQTL code overview<\/p>\n<\/div>","<div class='gt_from_md'><p>seurat + signac<\/p>\n<\/div>","<div class='gt_from_md'><p>Check some issues with GitHub Actions<\/p>\n<\/div>","<div class='gt_from_md'><p>Introduction to CellRanger RNAseq ATAC analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>Using git and RStudio to work with R Markdown files<\/p>\n<\/div>","<div class='gt_from_md'><p>SPEAQeasy error<\/p>\n<\/div>","<div class='gt_from_md'><p>Understanding API authentication<\/p>\n<\/div>","<div class='gt_from_md'><p>the user asked how to change the dot size in the plot_pca( )<\/p>\n<\/div>","<div class='gt_from_md'><p>user had question about generating rarefaction curves in R<\/p>\n<\/div>","<div class='gt_from_md'><p>tensorQTL pipeline overview<\/p>\n<\/div>","<div class='gt_from_md'><p>Quantifying a single gene not in SPEAQeasy‚Äôs default reference<\/p>\n<\/div>","<div class='gt_from_md'><p>Updating spatialLIBD::read10xVisiumAnalysis<\/p>\n<\/div>","<div class='gt_from_md'><p>Verify GitHub ssh key setup and team overview<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging spatialLIBD::read10xVisiumAnalysis<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview on ‚ÄúR/Bioconductor-powered Team Data Science‚Äù<\/p>\n<\/div>","<div class='gt_from_md'><p>Finalizing genotyping + expression script<\/p>\n<\/div>","<div class='gt_from_md'><p>Understanding the template project repository<\/p>\n<\/div>","<div class='gt_from_md'><p>Help resolve JHPCE permission issues on the lifespan_hpc_spatial project<\/p>\n<\/div>","<div class='gt_from_md'><p>Debug spaceranger 2023.0208.0 (likely an alpha for 2.0.1) with SpatialExperiment 1.8.0<\/p>\n<\/div>","<div class='gt_from_md'><p>How to write a tweetorial<\/p>\n<\/div>","<div class='gt_from_md'><p>read10xVisium() in version 1.7.1 (and now 1.8.0) creates a dangerous silent error by mismatching tissue positions across multiple samples<\/p>\n<\/div>","<div class='gt_from_md'><p>Version controlling renamed directories<\/p>\n<\/div>","<div class='gt_from_md'><p>Miscellaneous R questions<\/p>\n<\/div>","<div class='gt_from_md'><p>Deconvolu code where the bisque doesnt show any differences<\/p>\n<\/div>","<div class='gt_from_md'><p>How to write a tweetorial<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating marker heatmaps with pseudobulk data<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up .bashrc file for folder access permission for DLPFC-SCZ project<\/p>\n<\/div>","<div class='gt_from_md'><p>Using a gitignore to not version control specific files<\/p>\n<\/div>","<div class='gt_from_md'><p>I have questions on GitHub and submitting SGE jobs.<\/p>\n<\/div>","<div class='gt_from_md'><p>Moving forward with multiple cluster methods<\/p>\n<\/div>","<div class='gt_from_md'><p>Adding figures to OneDrive and google docs with CrossReference<\/p>\n<\/div>","<div class='gt_from_md'><p>Organizing figure components for Illustrator<\/p>\n<\/div>","<div class='gt_from_md'><p>Lining up gene expression and genotyping data<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up GitHub SSH keys<\/p>\n<\/div>","<div class='gt_from_md'><p>Managing software dependencies with Nextflow<\/p>\n<\/div>","<div class='gt_from_md'><p>Nextflow with Conda and Github actions<\/p>\n<\/div>","<div class='gt_from_md'><p>SPEAQeasy error<\/p>\n<\/div>","<div class='gt_from_md'><p>GitHub for a beginner<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq GLM PCA<\/p>\n<\/div>","<div class='gt_from_md'><p>Integrating R scripts with Nextflow<\/p>\n<\/div>","<div class='gt_from_md'><p>Making custom R script work with Netxflow<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq QC<\/p>\n<\/div>","<div class='gt_from_md'><p>Getting started with snRNA-seq<\/p>\n<\/div>","<div class='gt_from_md'><p>Plotting Gene Expression with SCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Help identify the source of unexpected NA values<\/p>\n<\/div>","<div class='gt_from_md'><p>Help building SCE with downloaded data<\/p>\n<\/div>","<div class='gt_from_md'><p>Adapting spot-deconvolution code to new data<\/p>\n<\/div>","<div class='gt_from_md'><p>Discuss Mean Ratio marker gene finding<\/p>\n<\/div>","<div class='gt_from_md'><p>Deconvolution study with rnaseq bulk data<\/p>\n<\/div>","<div class='gt_from_md'><p>resolving a merge conflict<\/p>\n<\/div>","<div class='gt_from_md'><p>Identifying the source of an error<\/p>\n<\/div>","<div class='gt_from_md'><p>coloring plots by a continuous variable<\/p>\n<\/div>","<div class='gt_from_md'><p>Understanding PCA for Habenula project<\/p>\n<\/div>","<div class='gt_from_md'><p>Review deconvolition script for qSVA<\/p>\n<\/div>","<div class='gt_from_md'><p>wrangling and reformatting expression data<\/p>\n<\/div>","<div class='gt_from_md'><p>preprocessing and reformatting genotype and expression data<\/p>\n<\/div>","<div class='gt_from_md'><p>organizing plots and output files<\/p>\n<\/div>","<div class='gt_from_md'><p>understanding linear mixed-effects models<\/p>\n<\/div>","<div class='gt_from_md'><p>help submitting jobs with R parallelization in jhpce. For instance, I would like to use scater::runPCA with the BPPARAM=BiocParallel::MulticoreParam(8)) parameter<\/p>\n<\/div>","<div class='gt_from_md'><p>Introduction to LIBD and our team<\/p>\n<\/div>","<div class='gt_from_md'><p>formatting genotype data into a matrix for each gene<\/p>\n<\/div>","<div class='gt_from_md'><p>I have this single nucleus RNA sequencing experiment with the mouse LS that Matt analyzed and we have a final single cell experiment, but I don‚Äôt know how to do anything with this data set and want to learn how to plot heatmaps and possibly other things for this data set. Can we maybe schedule an hour to start this learning process and then I can schedule future time as needed?<\/p>\n<\/div>","<div class='gt_from_md'><p>Check the DEG heatmaps<\/p>\n<\/div>","<div class='gt_from_md'><p>Review TMM paper<\/p>\n<\/div>","<div class='gt_from_md'><p>csvs that I want to merge together<\/p>\n<\/div>","<div class='gt_from_md'><p>Lifting results between hg19 and hg38<\/p>\n<\/div>","<div class='gt_from_md'><p>GO and KEGG boxplots with the modifications you suggested<\/p>\n<\/div>","<div class='gt_from_md'><p>QC approach for hebenula<\/p>\n<\/div>","<div class='gt_from_md'><p>How to check technical effects for RNAseq data<\/p>\n<\/div>","<div class='gt_from_md'><p>Reviewing GO results and dotplots<\/p>\n<\/div>","<div class='gt_from_md'><p>Working on making a violin plot for just one gene. So by brain region rather than gene type<\/p>\n<\/div>","<div class='gt_from_md'><p>Question on the feasibility of some approach with HALO (for the AD project)<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview on using GitHub for Matlab projects<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up SSH keys between login and compute nodes<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up SSH keys for JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Introduction to DSgs, R, Bioconductor, JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Interaction between .bashrc and ‚Äòmodule load‚Äô in ‚Äòqsub‚Äôd scripts<\/p>\n<\/div>","<div class='gt_from_md'><p>Modifying a shell script to segment a VCF by genomic region<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up SPEAQeasy at JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Code to threshold small sized nuclei from the NaCC visium images<\/p>\n<\/div>","<div class='gt_from_md'><p>How to organize OneDrive files<\/p>\n<\/div>","<div class='gt_from_md'><p>How can I interpret the y-axis from <code>plotExplanatoryVariables()<\/code>?<\/p>\n<\/div>","<div class='gt_from_md'><p>Use of git from multiple local locations<\/p>\n<\/div>","<div class='gt_from_md'><p>Moving and correctly accessing HDF5-backed SummarizedExperiments<\/p>\n<\/div>","<div class='gt_from_md'><p>DE analysis introduction<\/p>\n<\/div>","<div class='gt_from_md'><p>Git push help<\/p>\n<\/div>","<div class='gt_from_md'><p>check the models and the results I have obtained so far from the DEA<\/p>\n<\/div>","<div class='gt_from_md'><p>DE analysis introduction<\/p>\n<\/div>","<div class='gt_from_md'><p>Check the model design of the smoking mouse project<\/p>\n<\/div>","<div class='gt_from_md'><p>VNS error<\/p>\n<\/div>","<div class='gt_from_md'><p>Strategies to pick threshold for cell size in segmentations<\/p>\n<\/div>","<div class='gt_from_md'><p>open multichannel image in imagej<\/p>\n<\/div>","<div class='gt_from_md'><p>code to ectract x and y coordinates of randomnly slected inform tiles for PNN project<\/p>\n<\/div>","<div class='gt_from_md'><p>Review results of Varaince Parition for smoking mouse<\/p>\n<\/div>","<div class='gt_from_md'><p>Pick randomn tiles for the PNN project<\/p>\n<\/div>","<div class='gt_from_md'><p>go over pseudobulk code<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with spatialLIBD colors setup<\/p>\n<\/div>","<div class='gt_from_md'><p>Extracting final output into table<\/p>\n<\/div>","<div class='gt_from_md'><p>CUDA issues with cellpose module<\/p>\n<\/div>","<div class='gt_from_md'><p>QC update for smoking mouse<\/p>\n<\/div>","<div class='gt_from_md'><p>R, RStudio, git, SSH key setup<\/p>\n<\/div>","<div class='gt_from_md'><p>Addressing SPEAQeasy error messages<\/p>\n<\/div>","<div class='gt_from_md'><p>LIBD Data Science<\/p>\n<\/div>","<div class='gt_from_md'><p>help row annotation colors of a pheatmap<\/p>\n<\/div>","<div class='gt_from_md'><p>Subsetting eQTL results with in risk loci<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Data science coordination<\/p>\n<\/div>","<div class='gt_from_md'><p>QC update for smoking mouse<\/p>\n<\/div>","<div class='gt_from_md'><p>Question about how ‚ÄòmitoRate‚Äô is calculated in SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Help troubleshooting error messge when running BayesSpace<\/p>\n<\/div>","<div class='gt_from_md'><p>zellkonverter error and guidance using SpaGCN<\/p>\n<\/div>","<div class='gt_from_md'><p>Review progress on smoking mouse DE project<\/p>\n<\/div>","<div class='gt_from_md'><p>pulling in external changes to a git repository<\/p>\n<\/div>","<div class='gt_from_md'><p>Adjusting thresholds for event detection using CaPTure<\/p>\n<\/div>","<div class='gt_from_md'><p>help installing ‚Äòjaffelab‚Äô<\/p>\n<\/div>","<div class='gt_from_md'><p>help with vis_grid_clus()<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE Github setup<\/p>\n<\/div>","<div class='gt_from_md'><p>Help troubleshooting shiny app that crashes on gene interactive tab<\/p>\n<\/div>","<div class='gt_from_md'><p>getting total UMI‚Äôs for each sample<\/p>\n<\/div>","<div class='gt_from_md'><p>Github setup<\/p>\n<\/div>","<div class='gt_from_md'><p>Revisited the sc analysis and could use some help<\/p>\n<\/div>","<div class='gt_from_md'><p>explain spaceranger output files<\/p>\n<\/div>","<div class='gt_from_md'><p>Tools and resources related to recount2 and recount3<\/p>\n<\/div>","<div class='gt_from_md'><p>I might need a review of how to go about selecting marker genes for the spatialDLPFC data. Also, I‚Äôm curious what code you have for making heatmaps showing marker gene expression across cell types.<\/p>\n<\/div>","<div class='gt_from_md'><p>Single cell rna seq analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>I have a mouse single-cell RNAseq data that I am aiming to identify cell types in different clusters.<\/p>\n<\/div>","<div class='gt_from_md'><p>Debug <code>spatialLIBD::sig_genes_extract_all()<\/code>.<\/p>\n<\/div>","<div class='gt_from_md'><p>How to debug an R package code<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with BayesSpace clustering<\/p>\n<\/div>","<div class='gt_from_md'><p>Data science organization.<\/p>\n<\/div>","<div class='gt_from_md'><p>MatLab questions.<\/p>\n<\/div>","<div class='gt_from_md'><p>Subsetting spe object for shiny app<\/p>\n<\/div>","<div class='gt_from_md'><p>Brainomics meeting with collaborators<\/p>\n<\/div>","<div class='gt_from_md'><p>Color bar for heatmap<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussion on how to recruit members<\/p>\n<\/div>","<div class='gt_from_md'><p>VisiumIF DLPFC building spe<\/p>\n<\/div>","<div class='gt_from_md'><p>OneDrive<\/p>\n<\/div>","<div class='gt_from_md'><p>BrainSeq Phase II development browser<\/p>\n<\/div>","<div class='gt_from_md'><p>make modified tissue_positions_list from DLPFC IF samples for running spotsopcheck on mock PNN visium sample<\/p>\n<\/div>","<div class='gt_from_md'><p>Help Deploying Shiny App for DLPFC_IF<\/p>\n<\/div>","<div class='gt_from_md'><p>Cross Reference help<\/p>\n<\/div>","<div class='gt_from_md'><p>Career development<\/p>\n<\/div>","<div class='gt_from_md'><p>Need some help to make a plot<\/p>\n<\/div>","<div class='gt_from_md'><p>Guide to creating a module at JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>How to make a tweetorial for a pre-print<\/p>\n<\/div>","<div class='gt_from_md'><p>File organization for a paper submission<\/p>\n<\/div>","<div class='gt_from_md'><p>creating a nextflow pipeline and supporting docker<\/p>\n<\/div>","<div class='gt_from_md'><p>In promptu meeting<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with purrr loops for WGCNA funtions<\/p>\n<\/div>","<div class='gt_from_md'><p>Grant proposal feedback<\/p>\n<\/div>","<div class='gt_from_md'><p>help with html output of spaceranger<\/p>\n<\/div>","<div class='gt_from_md'><p>CV feedback<\/p>\n<\/div>","<div class='gt_from_md'><p>Cover letter feedback<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugged code from <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/33\">https://github.com/LieberInstitute/spatialLIBD/issues/33<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>Brainomics meeting with collaborators<\/p>\n<\/div>","<div class='gt_from_md'><p>How to delete large files from GitHub history<\/p>\n<\/div>","<div class='gt_from_md'><p>help with read10xVisiumWrapper()<\/p>\n<\/div>","<div class='gt_from_md'><p>macOS and JHPCE laptop set<\/p>\n<\/div>","<div class='gt_from_md'><p>writing papers in google docs and setting up links for figures<\/p>\n<\/div>","<div class='gt_from_md'><p>Hey Maddy, I need something called ‚Äúspot size‚Äù  on the 12 published DLPFC samples<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/VistoSeg/issues/12\">https://github.com/LieberInstitute/VistoSeg/issues/12<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>applying size threshold to vistoseg outputs to get rid of smaller regions that are not nuclei<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging JHPCE configuration files<\/p>\n<\/div>","<div class='gt_from_md'><p>I‚Äôd like to consult with Louise about how to make a spatial plot in R using a data set from the HALO FISH-IF module.<\/p>\n<\/div>","<div class='gt_from_md'><p>directed the person to the VistoSeg website for steps to run VistoSeg<\/p>\n<\/div>","<div class='gt_from_md'><p>using toptable and duplicate correlation<\/p>\n<\/div>","<div class='gt_from_md'><p>help with clustering portion of th build_spe<\/p>\n<\/div>","<div class='gt_from_md'><p>Using the ‚Äúredstone‚Äù software<\/p>\n<\/div>","<div class='gt_from_md'><p>help with build_spe<\/p>\n<\/div>","<div class='gt_from_md'><p>Input files for custom annotation with SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Advice on how to increase code coverate with testthat<\/p>\n<\/div>","<div class='gt_from_md'><p>Help run trial shiny app deployment<\/p>\n<\/div>","<div class='gt_from_md'><p>set up git repo for long read project<\/p>\n<\/div>","<div class='gt_from_md'><p>The authenticity of host ‚Äògithub.com‚Äô<\/p>\n<\/div>","<div class='gt_from_md'><p>Assistance with analyzing spatial data<\/p>\n<\/div>","<div class='gt_from_md'><p>Follow-up on custom annotation with SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Plan how to make spatial registration R functions<\/p>\n<\/div>","<div class='gt_from_md'><p>Adding lines to a GTF for use in SPEAQeasy with custom annotation<\/p>\n<\/div>","<div class='gt_from_md'><p>Clarifying what role deconvolution can play on LCM-seq<\/p>\n<\/div>","<div class='gt_from_md'><p>TODO<\/p>\n<\/div>","<div class='gt_from_md'><p>Error message running SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>How to log in to JHU-controlled laptop<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up passwordless login<\/p>\n<\/div>","<div class='gt_from_md'><p>VisiumIF loupe alignment<\/p>\n<\/div>","<div class='gt_from_md'><p>Using custom annotation with SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Masking of blood vessels from PNN channel in PNN project<\/p>\n<\/div>","<div class='gt_from_md'><p>How are InForm tiles arranged<\/p>\n<\/div>","<div class='gt_from_md'><p>Using the caracol queue<\/p>\n<\/div>","<div class='gt_from_md'><p>How to guide others learning DS<\/p>\n<\/div>","<div class='gt_from_md'><p>Understand the covariates used in scRNA-seq cell type analysis for a project we are doing with striatal organoids<\/p>\n<\/div>","<div class='gt_from_md'><p>‚ÄúMegadepth usage to add a new annotation to hg38 file‚Äù<\/p>\n<\/div>","<div class='gt_from_md'><p>How to build a cov_rse object using qSVA v1<\/p>\n<\/div>","<div class='gt_from_md'><p>Debradation matrix for LIBD riboZ<\/p>\n<\/div>","<div class='gt_from_md'><p>Work through qsvaR vignette together<\/p>\n<\/div>","<div class='gt_from_md'><p>Generating bigWigs with SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Data type change of image data when loading into Python<\/p>\n<\/div>","<div class='gt_from_md'><p>WGBS data analysis overview<\/p>\n<\/div>","<div class='gt_from_md'><p>Help building for loops to analyze WGCNA ouput data<\/p>\n<\/div>","<div class='gt_from_md'><p>I would like to learn how you ran tensorQTL on the MDD data, maybe with using the GPU nodes for it.<\/p>\n<\/div>","<div class='gt_from_md'><p>contrast adjustment, intensity normalization, channel/gene description in PNN images<\/p>\n<\/div>","<div class='gt_from_md'><p>Ideas for possible future analyses<\/p>\n<\/div>","<div class='gt_from_md'><p>rename files on JHPCE through command line<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE array jobs and fasthplus<\/p>\n<\/div>","<div class='gt_from_md'><p>Provide comparison table for different metrics obtained from image meta data in MATLAB, for Visium IF workflow<\/p>\n<\/div>","<div class='gt_from_md'><p>snRNA-seq and deconvolution overview<\/p>\n<\/div>","<div class='gt_from_md'><p>Access to the gene info in the rowData of the Summarized Experiment objects to annotate eqtl results<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain Tangram, scRNAseq count matrix, spe object<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain the concept of Squidpy paper<\/p>\n<\/div>","<div class='gt_from_md'><p>tiling and metadata changes in PNN project<\/p>\n<\/div>","<div class='gt_from_md'><p>explaination on Image metadata, bit depth<\/p>\n<\/div>","<div class='gt_from_md'><p>Need for rotation of visium array images and its spatial coordinnnate in spe object<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging seg fault with tensorqtl module<\/p>\n<\/div>","<div class='gt_from_md'><p>Using tensorqtl with a GPU<\/p>\n<\/div>","<div class='gt_from_md'><p>X11 and SSH issues<\/p>\n<\/div>","<div class='gt_from_md'><p>VistoSegIF pipeline<\/p>\n<\/div>","<div class='gt_from_md'><p>accessing the ‚Äògpu‚Äô node at JHPCE correctly<\/p>\n<\/div>","<div class='gt_from_md'><p>Transfer data from JHPCE to Neural_Plasticity<\/p>\n<\/div>","<div class='gt_from_md'><p>Transfer data Neural_Plasticity to JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE directory organization, repo linking, making commits<\/p>\n<\/div>","<div class='gt_from_md'><p>opening python, R, MATLAB console interactive session on JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Troubleshooting ExprssionSet conversion<\/p>\n<\/div>","<div class='gt_from_md'><p>glitch in CAPTure tutorial about input data types to a function<\/p>\n<\/div>","<div class='gt_from_md'><p>Help using BisqueRNA, getting snRNA-seq into correct input format, chosing refrence vs. marker based deconvolution<\/p>\n<\/div>","<div class='gt_from_md'><p>creating a ‚Äòsamples.manifest‚Äô and choosing trimming settings<\/p>\n<\/div>","<div class='gt_from_md'><p>Removing smaller objects from VistoSeg segmentations<\/p>\n<\/div>","<div class='gt_from_md'><p>applying kmeans to CA3 mouse RNAscope data to obtain gene expression levels<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE file permissions and ACLs<\/p>\n<\/div>","<div class='gt_from_md'><p>running loupe browser for HPC images<\/p>\n<\/div>","<div class='gt_from_md'><p>countSpots from VistoSeg throwing errors for one DLPFC sample from round4<\/p>\n<\/div>","<div class='gt_from_md'><p>Resolve issues with comments<\/p>\n<\/div>","<div class='gt_from_md'><p>using ‚Äòlist.files‚Äô in R<\/p>\n<\/div>","<div class='gt_from_md'><p>batch processing - dotdotdot, on local machine<\/p>\n<\/div>","<div class='gt_from_md'><p>error in transferring Visuim HPC data to JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>using awk<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging help with rse_tx object<\/p>\n<\/div>","<div class='gt_from_md'><p>qc general scripts, rnaseq scripts general, ggplot histogram with breaks<\/p>\n<\/div>","<div class='gt_from_md'><p>We would love an opinion on our (now cell-type assigned) single nuclei rna-seq data, i.e. how they look - if anything is off, advice for improvement‚Ä¶<\/p>\n<\/div>","<div class='gt_from_md'><p>I am trying to compute DEGs from thefunction registration_stats_enrichment‚Ä¶\nError in limma::makeContrasts(contrasts = z, levels = registration_model) :\nThe levels must by syntactically valid names in R<\/p>\n<\/div>","<div class='gt_from_md'><p>Transfering files between JHPCE disks<\/p>\n<\/div>","<div class='gt_from_md'><p>running spotspotcheck for SFN<\/p>\n<\/div>","<div class='gt_from_md'><p>doubts on VistoSeg<\/p>\n<\/div>","<div class='gt_from_md'><p>help with git<\/p>\n<\/div>","<div class='gt_from_md'><p>debugging SPEAQeasy errors<\/p>\n<\/div>","<div class='gt_from_md'><p>Presentation preparation feedback<\/p>\n<\/div>","<div class='gt_from_md'><p>help with ‚Äòumask‚Äô and git<\/p>\n<\/div>","<div class='gt_from_md'><p>issues with regioprops function in dotdotdot<\/p>\n<\/div>","<div class='gt_from_md'><p>Fiducial frame enhancement for Visium IF images using preview on MAC<\/p>\n<\/div>","<div class='gt_from_md'><p>running Loupe browser for mannual alignment<\/p>\n<\/div>","<div class='gt_from_md'><p>merging FASTQ files automatically via SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating cyberduck bookmarks for JHPCE paths<\/p>\n<\/div>","<div class='gt_from_md'><p>writing shell script for countNuclei function in VistoSeg<\/p>\n<\/div>","<div class='gt_from_md'><p>Explaining Github issues, history and version control<\/p>\n<\/div>","<div class='gt_from_md'><p>Explaining SPE (Spatial Experiment) SCE (Single Cell Experiment) objects<\/p>\n<\/div>","<div class='gt_from_md'><p>syntax error in shell script<\/p>\n<\/div>","<div class='gt_from_md'><p>CellProfiler H&amp;E segmentations<\/p>\n<\/div>","<div class='gt_from_md'><p>Trouble with implementing RNAscope function for humans<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussion about rerunning FUSION/TWAS with new GWAS with extra attention paid to the number of samples for each brain region<\/p>\n<\/div>","<div class='gt_from_md'><p>identifying SPEAQeasy error<\/p>\n<\/div>","<div class='gt_from_md'><p>Error running VistoSeg::spospotcheck (Error in spotspotcheck&gt;start (line 60))<\/p>\n<\/div>","<div class='gt_from_md'><p>Error while running rnascope_mouse - line 159<\/p>\n<\/div>","<div class='gt_from_md'><p>parameter tuning for dotdotdot to split the nuclei clusters<\/p>\n<\/div>","<div class='gt_from_md'><p>improve lipofuscin masking in dotdotdot<\/p>\n<\/div>","<div class='gt_from_md'><p>dynamically name structures in matlab based on user inputs<\/p>\n<\/div>","<div class='gt_from_md'><p>using map to cbind a list of six to its corresponding list of lists. The second is a list of 6, each a list of 3.<\/p>\n<\/div>","<div class='gt_from_md'><p>resuming SPEAQeasy after an issue in the counts step<\/p>\n<\/div>","<div class='gt_from_md'><p>downloading FASTQ files from SRA<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain how to merge 2 data tables in R<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain public keys and the authorized_keys file on JHPCE, removed duplicate public keys for the local machine to resolve ssh key login<\/p>\n<\/div>","<div class='gt_from_md'><p>Using purrr and map to loop over datasets and apply qsva<\/p>\n<\/div>","<div class='gt_from_md'><p>resolving issues with multiple keys in the authorized_keys file<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with JHPCE login issues<\/p>\n<\/div>","<div class='gt_from_md'><p>explanation of the methods and algorithm of a paper we are reviewing together<\/p>\n<\/div>","<div class='gt_from_md'><p>steps to resolve image segmentation of LC heated samples round 2<\/p>\n<\/div>","<div class='gt_from_md'><p>How to write a Twitter thread for a paper<\/p>\n<\/div>","<div class='gt_from_md'><p>Forming a data frame from data in multiple files<\/p>\n<\/div>","<div class='gt_from_md'><p>Refining an abstract submission<\/p>\n<\/div>","<div class='gt_from_md'><p>Git branches<\/p>\n<\/div>","<div class='gt_from_md'><p>DNA genotyping overview<\/p>\n<\/div>","<div class='gt_from_md'><p>DNA genotyping overview<\/p>\n<\/div>","<div class='gt_from_md'><p>Optimizing DAPI segmentation in dotdotdot<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview of the RNA-seq universe<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain how awk works<\/p>\n<\/div>","<div class='gt_from_md'><p>Issues with SUFI<\/p>\n<\/div>","<div class='gt_from_md'><p>Resolve errors produced by dotdotdot on SUFI output<\/p>\n<\/div>","<div class='gt_from_md'><p>R Code to create list of files to be processed through VisiumLIBD<\/p>\n<\/div>","<div class='gt_from_md'><p>Setup aliases in bashrc<\/p>\n<\/div>","<div class='gt_from_md'><p>debugging SPEAQeasy errors<\/p>\n<\/div>","<div class='gt_from_md'><p>getting SPEAQeasy running<\/p>\n<\/div>","<div class='gt_from_md'><p>formatting a LaTeX document and bookdown website<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up a team at LIBD<\/p>\n<\/div>","<div class='gt_from_md'><p>recreating spatstat plots with more features<\/p>\n<\/div>","<div class='gt_from_md'><p>Helping understand errors from VisiumLIBD<\/p>\n<\/div>","<div class='gt_from_md'><p>Helping understand errors from VisiumLIBD<\/p>\n<\/div>","<div class='gt_from_md'><p>Help indexing a possibly corrupt VCF file<\/p>\n<\/div>","<div class='gt_from_md'><p>Running Genome Studio on n = 200, generating PLINK outputs<\/p>\n<\/div>","<div class='gt_from_md'><p>How to set up a GitHub actions-powered website<\/p>\n<\/div>","<div class='gt_from_md'><p>Specifying output logs on SGE<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging ggplot code<\/p>\n<\/div>","<div class='gt_from_md'><p>Planning session for getting up to speed with SpatialExperiment, spatialLIBD, here and ggplot2<\/p>\n<\/div>","<div class='gt_from_md'><p>How to choose and add a licence to a GitHub repo<\/p>\n<\/div>","<div class='gt_from_md'><p>Designing an MS-level class<\/p>\n<\/div>","<div class='gt_from_md'><p>making a list of filenames in a specific folder and subfolder in R<\/p>\n<\/div>","<div class='gt_from_md'><p>Rectified issues for batch running emma‚Äôs RNAscope data<\/p>\n<\/div>","<div class='gt_from_md'><p>create shell script for batch running images on VisiumLIBD<\/p>\n<\/div>","<div class='gt_from_md'><p>Locating public data at JHPCE + bamtofastq workflow<\/p>\n<\/div>","<div class='gt_from_md'><p>TWAS questions<\/p>\n<\/div>","<div class='gt_from_md'><p>How to transfer recount3 data<\/p>\n<\/div>","<div class='gt_from_md'><p>How to get started with R<\/p>\n<\/div>","<div class='gt_from_md'><p>Brainstorming session on collaboration<\/p>\n<\/div>","<div class='gt_from_md'><p>Manhattan plot making and tstatistic<\/p>\n<\/div>","<div class='gt_from_md'><p>addressing SPEAQeasy error<\/p>\n<\/div>","<div class='gt_from_md'><p>got mds score for the vcf she presented using plink<\/p>\n<\/div>","<div class='gt_from_md'><p>FUSION / TWAS Walkthrough<\/p>\n<\/div>","<div class='gt_from_md'><p>mds for vcfs<\/p>\n<\/div>","<div class='gt_from_md'><p>checking plots and data structure in clusterprofiler<\/p>\n<\/div>","<div class='gt_from_md'><p>Havent got my mds yet so making the meeting for Friday<\/p>\n<\/div>","<div class='gt_from_md'><p>questions about expression_cut and pca plots<\/p>\n<\/div>","<div class='gt_from_md'><p>Using GenomeStudio<\/p>\n<\/div>","<div class='gt_from_md'><p>How to prepare for a LIBD seminar part 4<\/p>\n<\/div>","<div class='gt_from_md'><p>troubleshoot SUFI output and figured it was not compatible with dotdotdot<\/p>\n<\/div>","<div class='gt_from_md'><p>gene ontology for genesets<\/p>\n<\/div>","<div class='gt_from_md'><p>Differential Expression help (continuation of 5/20)<\/p>\n<\/div>","<div class='gt_from_md'><p>How to prepare for a LIBD seminar part 3<\/p>\n<\/div>","<div class='gt_from_md'><p>Grant application review<\/p>\n<\/div>","<div class='gt_from_md'><p>PredictDB weights<\/p>\n<\/div>","<div class='gt_from_md'><p>Help preparing a figure for a grant application<\/p>\n<\/div>","<div class='gt_from_md'><p>How to prepare for a LIBD seminar part 2<\/p>\n<\/div>","<div class='gt_from_md'><p>Making plots for DE studies<\/p>\n<\/div>","<div class='gt_from_md'><p>Making a module<\/p>\n<\/div>","<div class='gt_from_md'><p>Configuring a parallel series of bash scripts<\/p>\n<\/div>","<div class='gt_from_md'><p>questions about processing genotype data<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up a SPEAQeasy run<\/p>\n<\/div>","<div class='gt_from_md'><p>Making plots for DE studies<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up covr using GitHub actions in an R package<\/p>\n<\/div>","<div class='gt_from_md'><p>How to get started with computational biology<\/p>\n<\/div>","<div class='gt_from_md'><p>How to insert a figure using the Oxford Bioinfomratics Overleaf LaTeX template<\/p>\n<\/div>","<div class='gt_from_md'><p>PredictDB weights<\/p>\n<\/div>","<div class='gt_from_md'><p>How to prepare for a LIBD seminar<\/p>\n<\/div>","<div class='gt_from_md'><p>Running SPEAQeasy- follow up<\/p>\n<\/div>","<div class='gt_from_md'><p>TWAS prep<\/p>\n<\/div>","<div class='gt_from_md'><p>Constructing an array job from an existing sequential shell script<\/p>\n<\/div>","<div class='gt_from_md'><p>Setting up FUSION / TWAS for practice<\/p>\n<\/div>","<div class='gt_from_md'><p>Running SPEAQeasy- follow up<\/p>\n<\/div>","<div class='gt_from_md'><p>Running SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Running SPEAQeasy- follow up<\/p>\n<\/div>","<div class='gt_from_md'><p>Running FUSION / TWAS<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE set up on mac<\/p>\n<\/div>","<div class='gt_from_md'><p>Running SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>running and trouble shooting VisiumIF image splitting code<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE set up on mac<\/p>\n<\/div>","<div class='gt_from_md'><p>Running the old RNA-seq pipeline on some SRA data<\/p>\n<\/div>","<div class='gt_from_md'><p>Resolving issues with running dotdotdot on 20X RNAscope images<\/p>\n<\/div>","<div class='gt_from_md'><p>runnning Hiplex registration<\/p>\n<\/div>","<div class='gt_from_md'><p>General guidance on how to learn how to do differential expression analyses<\/p>\n<\/div>","<div class='gt_from_md'><p>Nextflow overview for work with imputation pipeline<\/p>\n<\/div>","<div class='gt_from_md'><p>creating logs for issues from running SUFI (spectral unmixing of fluorescent biological images)<\/p>\n<\/div>","<div class='gt_from_md'><p>steps to troubleshoot issues with stitched multi tile RNAscope images<\/p>\n<\/div>","<div class='gt_from_md'><p>spatialLIBD support<\/p>\n<\/div>","<div class='gt_from_md'><p>Need help understanding the sce - single cell object<\/p>\n<\/div>","<div class='gt_from_md'><p>help doing a power analysis for RNA-seq data<\/p>\n<\/div>","<div class='gt_from_md'><p>Creating a documentation website using bookdown<\/p>\n<\/div>","<div class='gt_from_md'><p>Clarification of LIBD support<\/p>\n<\/div>","<div class='gt_from_md'><p>General advice<\/p>\n<\/div>","<div class='gt_from_md'><p>running nucleiCount for spatial DLPFC samples<\/p>\n<\/div>","<div class='gt_from_md'><p>Download SRA data and run SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain the feasibility of an on-demand end user interface for DEG and eQTLs<\/p>\n<\/div>","<div class='gt_from_md'><p>extracting p-values from linear mixed effect model for Calcium imaging data<\/p>\n<\/div>","<div class='gt_from_md'><p>Using an LIBD module<\/p>\n<\/div>","<div class='gt_from_md'><p>Change point size in vis_clust_grid<\/p>\n<\/div>","<div class='gt_from_md'><p>Download SRA data and run SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Attempted SPEAQeasy installation on MacOS<\/p>\n<\/div>","<div class='gt_from_md'><p>visium image segmentation<\/p>\n<\/div>","<div class='gt_from_md'><p>troubleshooting memory usage for VNS function from VisiumLIBD<\/p>\n<\/div>","<div class='gt_from_md'><p>get effect size for calcium imaging<\/p>\n<\/div>","<div class='gt_from_md'><p>split histology images into respective capture areas<\/p>\n<\/div>","<div class='gt_from_md'><p>x-quartz installation<\/p>\n<\/div>","<div class='gt_from_md'><p>BSP2 eQTLs with a subset of SNPs<\/p>\n<\/div>","<div class='gt_from_md'><p>qSVA explanation and code review<\/p>\n<\/div>","<div class='gt_from_md'><p>How to write a small methods paper<\/p>\n<\/div>","<div class='gt_from_md'><p>resolve figure generation error in human rnascope script from dotdotdot<\/p>\n<\/div>","<div class='gt_from_md'><p>Maker finding for cell types in single cell data<\/p>\n<\/div>","<div class='gt_from_md'><p>image analysis on QuPath<\/p>\n<\/div>","<div class='gt_from_md'><p>Advice for writing papers<\/p>\n<\/div>","<div class='gt_from_md'><p>Clarification about how recent CMC data was processed with SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Our group‚Äôs set-up and approach for writing papers with google docs, and managing citations<\/p>\n<\/div>","<div class='gt_from_md'><p>data transfer from LIBD server to JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Submitting/debugging bash job for dotdotdot on cluster<\/p>\n<\/div>","<div class='gt_from_md'><p>I need help in: downloading data from SRA to JHPCE, running SPEAQeasy to create rse object (one session will likely not be enough).<\/p>\n<\/div>","<div class='gt_from_md'><p>I need help with setting up Mobaexterm. Unable to open new terminals and folder permission issues<\/p>\n<\/div>","<div class='gt_from_md'><p>explaination of Owner/Group/Other permissions for files on JHPCe<\/p>\n<\/div>","<div class='gt_from_md'><p>download repository from github to cluster and run dotdotdot on cluster<\/p>\n<\/div>","<div class='gt_from_md'><p>Visualizing RPKM valus for genes in Humna and Primarte data<\/p>\n<\/div>","<div class='gt_from_md'><p>BSP2 LIBD eQTLs request<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE ssh keypair set up<\/p>\n<\/div>","<div class='gt_from_md'><p>dotdotdot parameter tuning for Ema‚Äôs rat RNAscope data<\/p>\n<\/div>","<div class='gt_from_md'><p>Github ssh key set up<\/p>\n<\/div>","<div class='gt_from_md'><p>Work on habenula deconvolution and debug the deconvolution package<\/p>\n<\/div>","<div class='gt_from_md'><p>Help determining what data is available and what can be used for an analysis<\/p>\n<\/div>","<div class='gt_from_md'><p>group_by bug for get_mean_ratio<\/p>\n<\/div>","<div class='gt_from_md'><p>Guidance on how to download data from a study in GEO<\/p>\n<\/div>","<div class='gt_from_md'><p>biocthis macOS GHA bug<\/p>\n<\/div>","<div class='gt_from_md'><p>Questions related to accessing results from the DG paper by Andrew Jaffe<\/p>\n<\/div>","<div class='gt_from_md'><p>Correlation values from labeledHeatmap<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with importing HDF5 file<\/p>\n<\/div>","<div class='gt_from_md'><p>Resolving issues in phenotype data between genotype, expression, and LIBDpheno<\/p>\n<\/div>","<div class='gt_from_md'><p>jhpce set up on mac<\/p>\n<\/div>","<div class='gt_from_md'><p>Inline formula throwing an error for interaction term<\/p>\n<\/div>","<div class='gt_from_md'><p>run dotdotdot pipeline local computer and cluster<\/p>\n<\/div>","<div class='gt_from_md'><p>how to open matlab (GUI and command line) on JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Matching Variants Between Two VCF Objects<\/p>\n<\/div>","<div class='gt_from_md'><p>how to connect to LIBD servers on windows and mac<\/p>\n<\/div>","<div class='gt_from_md'><p>Accesing CMC RNA-seq data<\/p>\n<\/div>","<div class='gt_from_md'><p>Ran Genome Studio, installed PLINK input plugin<\/p>\n<\/div>","<div class='gt_from_md'><p>ChipQC ‚Äònames‚Äô attribute [9] must be the same length as the vector [2]‚Äô<\/p>\n<\/div>","<div class='gt_from_md'><p>data transfer from LIBD server to JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>download matlab on local computer through JHU software portal<\/p>\n<\/div>","<div class='gt_from_md'><p>Help clustering sce data<\/p>\n<\/div>","<div class='gt_from_md'><p>Using reshape package to transform table for plotting<\/p>\n<\/div>","<div class='gt_from_md'><p>Help adapting ggplot script for TWAS Z-score<\/p>\n<\/div>","<div class='gt_from_md'><p>RDavid Error<\/p>\n<\/div>","<div class='gt_from_md'><p>Differences between recount2‚Äôs ‚Äúbp_length‚Äù and SPEAQeasy‚Äôs ‚ÄúLength‚Äù<\/p>\n<\/div>","<div class='gt_from_md'><p>How does one modify a single value of a field in a seqinfo object?<\/p>\n<\/div>","<div class='gt_from_md'><p>GitHub actions workflow issue with macOS on R-devel<\/p>\n<\/div>","<div class='gt_from_md'><p>LC repo organization<\/p>\n<\/div>","<div class='gt_from_md'><p>Extracting Sequences from Fasta Files with Rsamtools<\/p>\n<\/div>","<div class='gt_from_md'><p>3D volume extraction in Matlab<\/p>\n<\/div>","<div class='gt_from_md'><p>Fixed two bugs on recount3 related to locate_url() for the BigWig files.<\/p>\n<\/div>","<div class='gt_from_md'><p>Fixed a bug on biocthis for release branches<\/p>\n<\/div>","<div class='gt_from_md'><p>Issue with knitcitations<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with SpaceRanger<\/p>\n<\/div>","<div class='gt_from_md'><p>Issue with JHPCE file permissions<\/p>\n<\/div>","<div class='gt_from_md'><p>recount3 GTEx data access<\/p>\n<\/div>","<div class='gt_from_md'><p>customize dotdotdot to his data<\/p>\n<\/div>","<div class='gt_from_md'><p>Learning Lexical Scoping Rules<\/p>\n<\/div>","<div class='gt_from_md'><p>Changing Row Names of MultiAssayExperiment Assay<\/p>\n<\/div>","<div class='gt_from_md'><p>exceRpt pipeline counts<\/p>\n<\/div>","<div class='gt_from_md'><p>issue running spaceranger<\/p>\n<\/div>","<div class='gt_from_md'><p>Plotly help if possible? I‚Äôm trying to export an html file that allows the user to look at each snp/gene on the plot, and I am almost successful. Leo recommended that I talk to you on this matter.<\/p>\n<\/div>","<div class='gt_from_md'><p>Help running speakeasy on MARCC and JHPCE please<\/p>\n<\/div>","<div class='gt_from_md'><p>Question: Subset center regions of a set of different length sequences<\/p>\n<\/div>","<div class='gt_from_md'><p>Ballgown stattest<\/p>\n<\/div>","<div class='gt_from_md'><p>Question: Cannot Install gmapR Bioconductor Package<\/p>\n<\/div>","<div class='gt_from_md'><p>copy number variation analysis (CNV)<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with spatstat<\/p>\n<\/div>"],"html":true,"width":200,"align":"left","headerStyle":{"font-weight":"normal"}},{"id":"public_notes","name":"public_notes","type":"character","style":"function(rowInfo, colInfo) {\nconst rowIndex = rowInfo.index + 1\n}","cell":["<div class='gt_from_md'><p>I explained why <code>age &gt; 17<\/code> is a must for SCZD vs NTC DGE analyses: we have no SCZD cases with <code>age &lt;= 17<\/code>, so if you don‚Äôt filter the NTC donors with <code>age &lt;=17<\/code>, you end up basically comparing older (SCZD) vs younger (NTC) donors, which is not what you want to do. We also talked about how in BSP2 there are some HPC (HIPPO) samples from 2 RiboZero protocols (Gold and HMR), how the Gold HPC samples cluster with the DLPFC ones on PC1 vs PC2, and thus why we ended up dropping the 43 Gold HPC samples in previous work (see <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/make_qSVs.R#L62-L69\">https://github.com/LieberInstitute/qsva_brain/blob/cba8fa5e886be9242dcbaae7854d1eed0f69cef5/brainseq_phase2_qsv/make_qSVs.R#L62-L69<\/a>). This information is reflected by <code>analysis_sczd_casecontrol_hippo<\/code> at <a href=\"https://s3.us-east-2.amazonaws.com/libd-brainseq2/methprop_pd.Rdata\">https://s3.us-east-2.amazonaws.com/libd-brainseq2/methprop_pd.Rdata<\/a> listed at <a href=\"https://eqtl.brainseq.org/phase2/\">https://eqtl.brainseq.org/phase2/<\/a>. Finally, I emphasized that self-reported <code>race<\/code> is not a trustworthy variable and we instead rely on <code>snpPC<\/code>s.<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained why using something like <a href=\"https://research.libd.org/spatialDLPFC/#interactive-websites\">https://research.libd.org/spatialDLPFC/#interactive-websites<\/a> or <a href=\"https://research.libd.org/globus/\">https://research.libd.org/globus/<\/a> give us control over links that we might need to update later on, instead of including other less stable links on manuscripts (domains we don‚Äôt control, tools that might change their URL syntax, etc). I provided links to how to make each option such as <a href=\"https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R\">https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R<\/a> and <a href=\"https://github.com/LieberInstitute/globus/blob/98dc479ad2cde0a047cf7a46769a85c32969b7b1/index.Rmd#L8-L37\">https://github.com/LieberInstitute/globus/blob/98dc479ad2cde0a047cf7a46769a85c32969b7b1/index.Rmd#L8-L37<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>In the session, we explored reanalyzing cloupe objects with the Loupe browser for scRNAseq assays. This involved using multiple projections like UMAP and t-SNE, splitting clusters, and filtering data based on UMI count and mitochondrial percentage. While there was interest in analyzing integrated datasets, the Loupe browser currently only supports exploring one Cloupe object at a time.<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the specifics of the <a href=\"https://kavlijhu.org/funding/4\">KavliNDI Distinguished Graduate Fellowship<\/a> and revised information that I can use while writing the reference letter (project title, assays proposed, that this involved new software, how long we‚Äôve worked together, education history of the applicant, etc).<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained where is located the cloupe output object generated by cell ranger-arc. Then, I explained how to use the tool to explore gene expression and atac assays by separate and also together. I also explained how to inspect clustering and re-label clusters and sub clusters; plus how to check DEG and look at their gene distributions. Finally, I explained how to re-run an analysis and use the most classic formats to save the analysis performed.<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained how to prepare the configuration files for working R and JHPCE.  I showed all the section that need to completed plus the configuration to access JHPCE, with links pointing specifically to Windows system. I explained the purpose of each section in the chapter 17 of <a href=\"https://lcolladotor.github.io/bioc_team_ds/config-files.html\">https://lcolladotor.github.io/bioc_team_ds/config-files.html<\/a>. I also provided the updated link (<a href=\"https://jhpce.jhu.edu/access/ssh/\">https://jhpce.jhu.edu/access/ssh/<\/a>)  for the JHPCE configuration due the past link is broken because of the change to the new web version.\nFinally, I provided the most recent video made by Leo  <a href=\"https://youtu.be/qtn3eTwFlZI\">https://youtu.be/qtn3eTwFlZI<\/a>  (minuto 17:50) to do this specific task for windows OS.<\/p>\n<\/div>","<div class='gt_from_md'><p>We added the <code>LFF_spatial_ERC<\/code> and <code>LFF_spatial_LC<\/code> datasets to <a href=\"https://github.com/lieberinstitute/ranger_metrics\">https://github.com/lieberinstitute/ranger_metrics<\/a>. This involved debugging the code a bit to handle updates on the Excel sheet files describing samples prior to being sent for sequencing.<\/p>\n<\/div>","<div class='gt_from_md'><p>We found out that <code>spaceranger count<\/code> did not like the JSON files we created with <code>jsonlite::write_json()<\/code> so instead, we edited them manually using <code>gsub()<\/code> and <code>read/writeLines()<\/code> as documented <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/commit/bbf810ad39012ba6ee4e0403063bf83bf844d007\">here<\/a>. Note that we had to manuallly revert the JSON files to their original state as documented <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/commit/79ed5479d502cd41d51f3aa8ee1467bd8100a08a\">here<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>We wrote a script to manually update the json files and add the Visium Slide ID as well as the capture area. This will enable us to not have to use <code>--unknown-slide=visium-1<\/code> <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/blob/6e85c05ce9f531c555569e616f8d5c351b427b4f/code/01_spaceranger/spaceranger_SLURM.sh#L63\">https://github.com/LieberInstitute/LFF_spatial_ERC/blob/6e85c05ce9f531c555569e616f8d5c351b427b4f/code/01_spaceranger/spaceranger_SLURM.sh#L63<\/a> that 10x Genomics support had recommended using. Doing so led to an error at <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/blob/b5b92a8bb9744ee424ec4a0642ae692423a91d56/code/01_spaceranger/logs/spaceranger_17.txt#L22\">https://github.com/LieberInstitute/LFF_spatial_ERC/blob/b5b92a8bb9744ee424ec4a0642ae692423a91d56/code/01_spaceranger/logs/spaceranger_17.txt#L22<\/a> which stated that <code>--slide<\/code> cannot be used as the same time as <code>--unknown-slide<\/code>. This is a workaround that was previously manually implemented at <a href=\"https://github.com/LieberInstitute/Habenula_Visium\">https://github.com/LieberInstitute/Habenula_Visium<\/a>. Since we have lots of samples, we wrote a script to avoid introducing copy-paste errors.<\/p>\n<\/div>","<div class='gt_from_md'><p>The issue on this <a href=\"https://github.com/LieberInstitute/LFF_spatial_ERC/blob/e913ff28fffbb17335d3844d312ec6486a7b182f/code/01_spaceranger/spaceranger_SLURM.sh#L55\">script<\/a> and related ones is that there is a missing <code>\\ <\/code> (note the space) before the <code>#<\/code> comment symbol. Thus the chained command is breaking into two, and the second half is not being recognized as options to the first half.<\/p>\n<\/div>","<div class='gt_from_md'><p>We worked on the following error message from <code>spaceranger count<\/code>: <em>ERROR: We detected a mixture of different R1 lengths ([26-28]), which breaks assumptions in how UMIs are tabulated and corrected. To process these data, you will need to truncate to the shortest observed R1 length by providing 26 to the ‚Äìr1-length argument if are running count/vdj, or via the r1-length parameter in each of the following tables of your multi config CSV if you are running multi: [gene-expression]<\/em>. We inspected the soft links for the FASTQ files and used <code>zcat fastq.gz | head<\/code> to inspect the first few lines of FASTQ files. We confirmed that the <code>R1<\/code> FASTQs were indeed of varying lenghts and will attempt a new <code>spaceranger count<\/code> run using <code>--r1-length = 26<\/code> though it‚Äôs likely that we‚Äôll next get an error with the <code>R2<\/code> FASTQ files as we observed that those files also have varying lengths.<\/p>\n<\/div>","<div class='gt_from_md'><p>Bernie was able to access the data throught the LIBD dataportal, I also pointed him to the combined LIBD rse_gene in brain_swap (which I think might be the source of the portal data‚Ä¶would need to check)<\/p>\n<\/div>","<div class='gt_from_md'><p>Updated script by commenting out unwanted lines and providing correct paths to the input files<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained what the key metrics are for each assay in a Multiome experiment from 10x scRNAseq and scATACseq data sets. I showed several HTML reports of data sets processed with cell ranger-arc to illustrate different scenarios targeting different numbers of reads for each assay, to help you understand the relevance of quality beyond the number of reads in sequencing. I showed how with a dataset with a moderate number of good quality reads you can perform a good analysis, while having many reads with compromised quality it is possible to have more challenges during the analysis. Finally, I explained the minimum recommended number and that it would be a good idea to search the literature for technical data from other experiments similar to his as a reference.<\/p>\n<\/div>","<div class='gt_from_md'><p>Cross checked the script and rectified filename/samplename processing<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained that BSP1 is polyA, aligned vs hg19, older annotation, aligned with TopHat2 (an older aligner). BSP2 is RiboZeroGold for DLPFC and RiboZeroHMR for HIPPO (with 43 RiboZeroGold samples that were dropped from the DE analysis), aligned vs hg38, against Gencode v25 (from July 2016), aligned with HISAT2. I also explained the naming structure of a Supplementary Table for BSP1‚Äôs paper where <code>adj<\/code>, <code>qSVA<\/code>, and <code>PCA<\/code> are labels specifying whether it was a model <code>adj<\/code>usting for covariates, whether it adjusts for qSVs using <code>qSVA<\/code> (the better model to use), or adjusting for gene-level expression PCs (<code>PCA<\/code>). These names are related to the <a href=\"https://www.pnas.org/doi/full/10.1073/pnas.1617384114\"><code>qSVA<\/code> 2017 paper<\/a> Figure 2. I also explained how in the BSP2 paper we did re-align the BSP1 data against hg38, though there is no Supplementary Table showing which of the 243 DLPFC DEGs replicated in BSP1 (though the data does exist since it was used to make Figure S28 and S29). We would need another DSgs or two to actually locate the data and construct an CSV file or table that others can use if they are interested in replication of BSP2 results using BSP1 and/or CommonMind Consortium (CMC) Phase 1 data (that was aligned vs hg19 with TopHat2). I also used <a href=\"https://speakerdeck.com/lcolladotor/libd-ds-tldr?slide=56\">this slide<\/a> to explain all the pieces that go into a DEG model and why the visualizations provided by the LIBD Data Portal (boxplots for SCZD vs NTC) are just a visual guide using normalized gene expression data and not the full answer (as the LIBD Data Portal doesn‚Äôt use the full details we include in a DEG model).<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained in detail the metrics reported by cellranger arc and helped you download a multiome public sample to run the pipeline.<\/p>\n<\/div>","<div class='gt_from_md'><p>Cynthia re-organized her code to run with a shell script from slurm_jobs. WE solved a bug with args index, and talked about adding catches for unexpected inputs. Future goal is to use map to replace for loops when handling list of data objects<\/p>\n<\/div>","<div class='gt_from_md'><p>We disscussed specific details on how to ask for help. For example, it‚Äôs ok to ask those around you in person but you should also ask on the <code>#libd_helpdesk<\/code> channel (and update the channel if someone else gave you an answer). That is, don‚Äôt assume that only those around you (in person) know the answer. We went over details on how to preview GitHub messages (to verify that they are correctly formatted), why GitHub issues are useful to the authors, what information is needed for others to help you answer a question (R session info, <code>reprex::reprex()<\/code> output, link to code on a public GitHub repo; private is ok when asking internally for help). We also discussed the overall idea of why learning how to make an R package will be useful to then improve one‚Äôs tooltik for debugging R packages made by others.<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained the previous steps to perform clustering with GEX and ATAC 10x multiome data using Seurat and Signac.<\/p>\n<\/div>","<div class='gt_from_md'><p>We interactively ran the python code for reading in ImageJ transforms to rule out an issue where the wrong data was read in. Ultimately, we couldn‚Äôt find issues with the data, but the Samui full-res images still seemed misaligned relative to the low-res images<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over how to update a GitHub remote following <a href=\"https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories\">information from the documentation<\/a>, finishing stylistic design touches on an <code>iSEE<\/code> app, and setting up a new repository for the next project.<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed details of the use cases I have in mind for how to search and visualize the JHPCE disk usage data over time and across different JHPCE paths. This information could be useful in forecasting data growth for individual PIs or projects, identifying who has exceeded their storage allocation, identifying static projects that could potentially can be moved off JHPCE / deleted / or moved to non-production disk space.<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the general overview of snRNA-seq data analysis steps and how the code at <a href=\"https://github.com/LieberInstitute/Habenula_Pilot\"><code>Habenula_Pilot<\/code><\/a> could be a good starting place. I also recommended signing up for a Data Science guidance session with Louise as she is quite familiar with the code in that repository prior to diving into it.<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained details about how data is organized at JHPCE, data sharing limitations, and several suggestions we have been working on that could be useful to implement at a larger scale across LIBD teams.<\/p>\n<\/div>","<div class='gt_from_md'><p>I pointed to suspicious areas of code where dataset-specific regexs were originally used, though the regexs looked good for his data as well. We decided to interactively walk line by line at a later date<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussed collaborations, personalities, and other details of the work environment with some of our collaborators with whom this person was going to interview for a job with.<\/p>\n<\/div>","<div class='gt_from_md'><p>I suggested building a list() where each element was a 3-piece plot (combined with a function like cowplot::plot_grid), then using print(plot_list)<\/p>\n<\/div>","<div class='gt_from_md'><p>See <a href=\"https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1710454284251579\">https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1710454284251579<\/a> for the full code details. We might add this to solution to <a href=\"https://github.com/LieberInstitute/jaffelab/blob/devel/R/merge_rse_metrics.R\">https://github.com/LieberInstitute/jaffelab/blob/devel/R/merge_rse_metrics.R<\/a>, although it‚Äôs only an approximation as you need to go back to the raw ERCC output files (typically generated with <code>kallisto<\/code>) to actually compute the ERCCsumLogErr accurately across different sequencing lanes for a given sample.<\/p>\n<\/div>","<div class='gt_from_md'><p>We have RSE objects from the LIBD Data Portal (or those that helped build that portal) as well as RSE objects made prior to the LIBD Data Portal. They have some differences, mostly on the <code>colData()<\/code> slot. Note how <a href=\"https://github.com/LieberInstitute/jaffelab/blob/21e65745a5b0852fc13ebdb5e5d23c6f1f66010b/R/merge_rse_metrics.R#L83-L97\">https://github.com/LieberInstitute/jaffelab/blob/21e65745a5b0852fc13ebdb5e5d23c6f1f66010b/R/merge_rse_metrics.R#L83-L97<\/a> and related code merges <code>colData()<\/code> information for samples that were sequenced in more than one lane and whose FASTQ files were not merged prior to aligning with <code>SPEAQeasy<\/code>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I walked throught the run_DE function I wrote for MDDseq (a wrapper for the limma-voom) pipline. I pointed out what parts of the MDDseq code he could adapt and simplify for his use case (its less complicated than the many DE runs we completed for MDDseq)<\/p>\n<\/div>","<div class='gt_from_md'><p>I suggested using conda after pip failed (missing system dependencies). Ultimately, my troubleshooting was enough to just build a module for everyone<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained details about the spatialDLPFC and deconvolution-benchmark papers. I also went over how we addressed feedback from the editors for the spatialDLPFC paper.<\/p>\n<\/div>","<div class='gt_from_md'><p>Explained how to access authorized keys and exit JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>Fernando was having some conectivity issues with JHPCE, so we didn‚Äôt get a full bug fix in I suggested some de-buggin strategies  [Update 3/15] We met again breifly to confirm the functions were working as intended. Initial issue was caused by un-intentinally overwriting the function<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed how to address feedback from a recent grant application, how our teams can get to know each other better, cleared up misunderstandings related to my team‚Äôs role as well as the DSgs system, and decided next steps to respond to the grant reviewers.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how a markdown of notes could be added using the Samui python API, though this is not how I had done it in a previous project ( I was unable to trace the original method I used)<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed more details about <code>iSEE<\/code> and shinyapps.io deployment details, such as memory settings and other configuration needed to actually customize and share an <code>iSEE<\/code> app. We also asked our collaborator for the list of genes they want to have plotted outside of <code>iSEE<\/code>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped implement an algorithm taking the minimum pixel intensity in case of overlapping images to display an H&amp;E image in Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>I diagnosed several potential issues, ultimately missing one detail that was the culprit: permissions were too open on some .ssh files on her laptop<\/p>\n<\/div>","<div class='gt_from_md'><p>Explained what parts of VNS and refineVNS can be tuned to get appropriate segmentations for Nac project<\/p>\n<\/div>","<div class='gt_from_md'><p>I sent a long and detailed email explaining different components of the LIBD BrainSEQ projects, background information on several covariates we typically have used in DEG models such as RIN and age at time of death, as well as other ones we have not used such as postmortem interval (PMI). I also explained the DSgs system and how it could be used by our JHU collaborators.<\/p>\n<\/div>","<div class='gt_from_md'><p>Giovanna and Jisu are working with some bulk RNA-seq data as a some pilot data in a new brain region prior to preforming saptail. They want to use deconvolution to help QC the samples, selecting for good disections. I walked them through our current recomndations for deconolition to help with this goal.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over internal code from the limma package using <a href=\"https://code.bioconductor.org/\">https://code.bioconductor.org/<\/a>, particularly for the <code>voom()<\/code> function. Doing so, we discussed that‚Äôs actually returned in the <code>voom()$E<\/code> slot. We also went over how to model a ratio using a linear regression model.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went topics such as live in Baltimore, research at JHU, overview of the different JHU campi, how to compare PhD programs across universities, and other questions this future PhD student had.<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussed feedback from collaborators and detailed next steps to wrap up this project.<\/p>\n<\/div>","<div class='gt_from_md'><p>Went over recent updated plots and results, as well as slides for a presentation with our collaborators for 2024-02-21.<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the updates implemented in qsvaR to handle input with ENSEMBL transcript IDs instead of GENCODE ones, updates to automatically detect if the input is at the gene level instead of the transcript level, how to check for correlation between qSVs and covariates, among other related issues.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described my typical strategy of using left_join by sample ID, as well as ensuring rownames don‚Äôt get lost from conversion to a tibble (thus dropping colnames from the SummarizedExperiment unintentionally)<\/p>\n<\/div>","<div class='gt_from_md'><p>I noted that the r_nac module was built similarly to conda_R, where the user doesn‚Äôt have to do anything special to install more packages. We ultimately traced the issue to permissions problems on files created as part of the proper use of the module by other users, likely due to problematic ‚Äòumask‚Äô settings<\/p>\n<\/div>","<div class='gt_from_md'><p>Sreya went through an error issue in qsvaR due to rownames not well formatted. The error message was misleading. This problem has been fixed in the latest updates of the package. I advised Sreya to get the latest version of the package.<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussed updates to violing plots and how to organize result files prior to sharing them with our collaborators.<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed him to the 2 approaches available in Seurat to perform DGE, including the FindMarkers()¬†function to find the genes that are different between control and treatments. I also show him code to perform quality controls in the main covariates related with these data-type. I shared with him consulted resources and links with public-code for his reference.<\/p>\n<\/div>","<div class='gt_from_md'><p>I ultimately recommended ‚Äògit reset ‚Äìsoft‚Äô to the commit ID before the recent mistaken addition of the large files<\/p>\n<\/div>","<div class='gt_from_md'><p>I advised using the SSH address (not HTTPS) for the ‚Äògit remote‚Äô, ensuring the ‚Äògit config‚Äô email matches the primary GitHub email, and using the up-to-date SSH key (not RSA, which is no longer supported/ recommended)<\/p>\n<\/div>","<div class='gt_from_md'><p>We explored options: BFG cleaner, git rebase and git reset ‚Äìsoft and discussed which was best for her situation<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the use of ‚Äòsrun‚Äô (including with X11) vs. ‚Äòsbatch‚Äô, and generally recommended ‚Äòsbatch‚Äô where possible to document how analyses were run/ ensure reproducibility<\/p>\n<\/div>","<div class='gt_from_md'><p>Caught up and talked about career development next steps.<\/p>\n<\/div>","<div class='gt_from_md'><p>I mostly focused on my setup using VS code with its remote SSH extension, and custom keyboard shortcuts for running python code interactively<\/p>\n<\/div>","<div class='gt_from_md'><p>I mostly confirmed the general process he had started for a particular module made from software packages as a precompiled binary<\/p>\n<\/div>","<div class='gt_from_md'><p>Went over the details of how to compute a permutation analysis with R.<\/p>\n<\/div>","<div class='gt_from_md'><p>Sreya wants to compare the correlation of 5 transcipts with 2k degradation transcripts. She had the data in the right format but needed help appllying the cor function to the table using group_by. Corrrplot was not the correct function for what she wanted to achive here.<\/p>\n<\/div>","<div class='gt_from_md'><p>Clarified updated automatic checks and information for users who attempt to use qsvaR with gene-level data instead of transcript-level data.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how code might change when using IF images rather than H&amp;E, for which the code was originally developed<\/p>\n<\/div>","<div class='gt_from_md'><p>Michael and Peter are working on a BPD vs. MDD DE analysis. They were getting more DEGs than were reported in an early version of the MDDseq project. I could not find the code for the data they were refrencing, with out the code it will be impossible to diagnosis what is differnt in the two analysis. I recomended they use the more up to date data from MDDseq, and pointed them to the correct data &amp; scripts.<\/p>\n<\/div>","<div class='gt_from_md'><p>Tricia and Fernando are re-running serval analysis they previously had working on the sge cluster, with the update to slurm they are now having issues. They are also having differnt errors in the same scrips + data. This could be a multi-facet issue, but firstly sounds like a R package version issue. I advisded the first step should be to have the same set-up, work in the same version of R and the same version (most updated) packages. See what errors persist and go from there. I also recomended Frenando start using git repos and version control with his team to help keep projects organized and consistent.<\/p>\n<\/div>","<div class='gt_from_md'><p>During the meeting Sreya showed me her progress on the analysis she‚Äôs doing taking the 2976 txs from the cell-component model and she wanted to compute the correlation between those and selected transcripts she‚Äôs interested in. She thinks those transcripts are involved in the degradation process. We couldn‚Äôt find those transcripts as part of the 2976 txs from the cell-component model.<\/p>\n<\/div>","<div class='gt_from_md'><p>We checked udpates to the <code>getDegTx()<\/code> and <code>DEqual()<\/code> functions, as well as discussed the response to a bug report from last year.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how to adapt a script I wrote for NAc to her dataset, where the goal was simply to display a stitched-together image in Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>I traced the issue to the use of 8-bit integers, where an overflow occured during normal addition of pixel values (the solution was to use floats while computing the mean, and then convert back to uint8)<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how I typically use dplyr::left_join, as well as how to add back missing rownames to the colData (or colnames of the SummarizedExperiment)<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the conflict between SGE and SLURM R packages, as well as identified lock files preventing re-installation of problematic packages<\/p>\n<\/div>","<div class='gt_from_md'><p>We traced the offending line of R code in the qsvar package and I described how unit testing could identify problematic function arguments and give more informative error messages<\/p>\n<\/div>","<div class='gt_from_md'><p>I met with Sreya because she‚Äôs having an issue with the getDegTx function from the qsvaR package. I tried to solve the issue but I wasn‚Äôt successful so I asked Sreya to give me more time and access to the rse object to have a more deeper look to understand the issue. After using her data I realized that the error message was misleading  and improved the function and unit tests.<\/p>\n<\/div>","<div class='gt_from_md'><p>I noted that the function could be applied before the data was converted to long format, where it was impractically large to work with<\/p>\n<\/div>","<div class='gt_from_md'><p>We identified issues causing incorrect coloring in the Samui outputs<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how to adapt a script I wrote for NAc to his dataset, where the goal was simply to display a stitched-together image in Samui<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained the different modules such as <code>module load conda_R/4.3.x<\/code> versus <code>conda_R/4.3<\/code>, how each is tied to a different version of Bioconductor. I also linked to <a href=\"https://youtu.be/D6t2GR7xaTE\">https://youtu.be/D6t2GR7xaTE<\/a> and <a href=\"https://docs.google.com/document/d/1DD9MUI2e3jp5S8C3PWRulZ4jwbXo94D3UTCQNJl_-Qo/edit?usp=sharing\">https://docs.google.com/document/d/1DD9MUI2e3jp5S8C3PWRulZ4jwbXo94D3UTCQNJl_-Qo/edit?usp=sharing<\/a> from the LIBD rstats club.<\/p>\n<\/div>","<div class='gt_from_md'><p>Cynthia and Hedia are starting the annotation/cluster integration of the multiome data. The data is currently clusted indiviudally by sample, sn, ATAC only, and combined sn+ATAC. findMarker has be used to calc marker stats for each cluster in each sample (there is a ton of data to digest). I am unfamilar with the sample independendt cluster, consered marker gene integration from seurat. But I gave advice to check the current clusters for hihg expression of any of the known marker genes to see if we‚Äôre picking up the expected biological signal, and do some prelimary annotations as a sanity check. Also to keep following the workflow and see if the conserved marker genes match with any of the know markers. Also to start recoring annotations for the clusters in a csv file for easy editing by hand as well as easy reading in to R<\/p>\n<\/div>","<div class='gt_from_md'><p>Provided feedback on CV writing for data science job applications.<\/p>\n<\/div>","<div class='gt_from_md'><p>I advised Hedia on how to adress some issues with the qSVA overview plot, both minor issues like aliginig and resising objects with biorender tools, and major rearngment of the table and DE application part.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the workflow for stitching Visium capture areas with Samui, and various Samui API functions like ‚Äòadd_chunked_feature‚Äô that are helpful<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the <a href=\"https://doi.org/10.1186/s13059-019-1861-6\">GLM-PCA paper<\/a> and decided to use GLM-PCA instead of logcounts by <code>quickCluster()<\/code> + <code>calculateSumFactors()<\/code> as described at <a href=\"https://comunidadbioinfo.github.io/cdsb2021_scRNAseq/normalizaci%C3%B3n-de-datos.html#normalizaci%C3%B3n-por-decircunvoluci%C3%B3n-deconvolution\">the CDSB 2021 snRNA-seq workshop<\/a>. We also discussed the development version of <code>Harmony<\/code>, in particular with the <code>lambda = NULL<\/code> setting.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over how to build a reprex for a user‚Äôs bug report, how to simplify the process of debugging, and the actual results from this particular bug report. We also discussed how to relay this information back to the person who initially asked the question.<\/p>\n<\/div>","<div class='gt_from_md'><p>I provided background information on a job ad from a colleague at Boston‚Äôs Children Hospital that I had shared. I also provided suggestions for preparing their GitHub account, CV, and other information useful for job interviewers.<\/p>\n<\/div>","<div class='gt_from_md'><p>Hedia has gotten some feedback on some unexpected behavior from qSVAR package. The qSV values show no significant correlation with other phenotype metrics (they have in the past), while this is not impossible it is strange. There are many factors that could have lead to this issue. Hedia and I talked about some debugging strategies, starting with re-running the workflow on familar data to see if we can re-create the issue or detirmine if it is unique to KJ‚Äôs dataset.<\/p>\n<\/div>","<div class='gt_from_md'><p>I provided background information on a job ad from a colleague at FHCRC that I had shared.<\/p>\n<\/div>","<div class='gt_from_md'><p>I gave feedback on scripts, how to organize them, the order of some analytical steps, which gene annotation files to use, and things like that. We don‚Äôt have results on the QC steps to share just yet.<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed a workaround for making sure a variable has global scope before editing it inside a function, though I noted it‚Äôs better not to use a function in her particular case<\/p>\n<\/div>","<div class='gt_from_md'><p>Continued reviewing qsvaR replication analysis code.<\/p>\n<\/div>","<div class='gt_from_md'><p>Detailed analysis steps for <a href=\"https://github.com/LieberInstitute/Cg_Hb_projector_profiling\">this project<\/a> using snRNA-seq data and analyzing it following OSCA-based steps from the CDSB 2021 snRNA-seq workshop.<\/p>\n<\/div>","<div class='gt_from_md'><p>Reviewed draft blog post on the Visium SPG AD project and published it at <a href=\"https://research.libd.org/rstatsclub/2023/11/01/our-first-adventure-with-visium-spatial-proteogenomics/\">https://research.libd.org/rstatsclub/2023/11/01/our-first-adventure-with-visium-spatial-proteogenomics/<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I identified two separate issues: the repository name and using HTTPS remote instead of SSH<\/p>\n<\/div>","<div class='gt_from_md'><p>I walked through how a decision tree works and can be interpreted, then described how correlation and RMSE are computed in a couple spatialDLPFC spot deconvo plots<\/p>\n<\/div>","<div class='gt_from_md'><p>He noted that the wrong version of python was being used, and we eventually traced this to user-specific conda settings interfering with the module<\/p>\n<\/div>","<div class='gt_from_md'><p>Explained how interaction models work with limma-voom.<\/p>\n<\/div>","<div class='gt_from_md'><p>Ibidem to 2023-09-29 session.<\/p>\n<\/div>","<div class='gt_from_md'><p>Reviewed <a href=\"https://github.com/LieberInstitute/QSVA_2020/blob/914e3b58f7fb97c204654493c9f9dcb2e48d112b/code/06_Replication/02_DE_and_replication_plot.R#L220-L313\">this script<\/a> for the replication analysis performed as part of the qsvaR project.<\/p>\n<\/div>","<div class='gt_from_md'><p>Paired programming session to debug <code>/dcs04/lieber/lcolladotor/with10x_LIBD001/ranger_metrics/code/spaceranger/03_merged_metrics/02_explore_metrics.R<\/code>. Further explained the original intent of the code in this script and steps for updating it.<\/p>\n<\/div>","<div class='gt_from_md'><p>We interpreted the intuitive meaning of a formula from some old code, which attempted to quantify the similarity of a set of significant genes between datasets. I also helped interpret the meaning of a plot based on this metric<\/p>\n<\/div>","<div class='gt_from_md'><p>Giovanna‚Äôs data has one cluster that is unqique to one donor, post batch correction. This donor also has some unique clinical background as well, I advised to drop the cluster as their batch corection seemed to be correct, and it possible that this is a rare biologically real cluster but there is not enough data to confirm.<\/p>\n<\/div>","<div class='gt_from_md'><p>I recommended ‚ÄòspatialLIBD::fetch_data‚Äô for easy retrieval of public data to get started with spatial computational analysis. I also recommended pushing for an account at JHPCE, where computational resources are more plentiful, and getting used to submitting jobs, accessing software, etc is valuable to learn<\/p>\n<\/div>","<div class='gt_from_md'><p>Reviewed some scripts related to replication analyses. I explained the different thresholds used on the discovery as well as the target datasets, as well as the different levels for assessing replication.<\/p>\n<\/div>","<div class='gt_from_md'><p>I walked Lionel through the new login commands for SLURM + updating his bashrc file so he can work with the slurm cluser<\/p>\n<\/div>","<div class='gt_from_md'><p>Ibidem to 2023-09-29 session.<\/p>\n<\/div>","<div class='gt_from_md'><p>I noted that conda_R seemed to be changing the version of java loaded, which affected how the nextflow module worked. I recommended unloading conda_R and avoiding the loading of conda_R via the .bashrc<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the general process of setting up SPEAQeasy and gave recommendations for his cluster: using singularity and SLURM-based installation/ set-up<\/p>\n<\/div>","<div class='gt_from_md'><p>I mostly covered documentation but also the appropriate options for his particular data<\/p>\n<\/div>","<div class='gt_from_md'><p>I investigated a ‚Äúfailure to submit process to scheduler‚Äù error using ‚Äòsacct‚Äô and similar commands, ultimately discovering the issue was related to SLURM configuration. There was an email chain with bithelp to follow up<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over the changes needed to take the current qsvaR paper draft by Josh Stolz to a production-level result (aka for pre-printing / publishing it).<\/p>\n<\/div>","<div class='gt_from_md'><p>Adapted mode code for tensorQTL prep, had to add genomics ranges to the sce object, and deal with bugs around rse_to_bed function<\/p>\n<\/div>","<div class='gt_from_md'><p>I followed up with bitsupport, as the issue seemed to involve some failed DNS query internal to SLURM, rather than SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>We walked through old code which aimed to quantify the similarity between significant DE results, interpreting formulas from the code and constructing more intuitive formulas<\/p>\n<\/div>","<div class='gt_from_md'><p>Ibidem to 2023-09-29 session.<\/p>\n<\/div>","<div class='gt_from_md'><p>Worked through addapting code for tensorQTL prep for the MDD single cell data - had to deal with bugs relating to cell type names<\/p>\n<\/div>","<div class='gt_from_md'><p>I gave general advice on re-writing code‚Äì keeping separate scripts, chunking code into segments and testing equality of outputs between old and new code after each chunk, and other strategies<\/p>\n<\/div>","<div class='gt_from_md'><p>We continued working with the OSCA-based CDSB 2021 intro to snRNA-seq workshop to further understand snRNA-seq data analysis steps, packages, and how they all relate to each other. The person who requested this DSgs was working on the workshop contents and had questions that we reviewed weekly (or so). Once they completed this OSCA-based training, we moved to analyzing an actual unpublished snRNA-seq dataset.<\/p>\n<\/div>","<div class='gt_from_md'><p>We looked at the rank vs. total UMI curves relevant for emptyDrops for the atac-seq samples. Cynthia and I have discussed these samples before. We went over how the emptyDrops function works again and what we can guesss is happenin in the curve (neurons vs. glia vs. empty droplets).  She is concerned beause for the 42-4 sample the wetbench target number of nuc was 7k, emptyDrops retrurns 9k. I thought that that sounded in the right ball park<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the ‚Äòdroplevels()‚Äô function, designed to remove unused factor levels<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over the background for this project (lateral septum) and an overview of the DSgs system, GitHub setup, and other onboarding material.<\/p>\n<\/div>","<div class='gt_from_md'><p>I discussed how we selected training data for the spot-deconvolution image model‚Äì using logistic regression to compute a confidence metric across the original training examples, that we then binned into 4 quartiles to re-select new examples to label in a less biased way<\/p>\n<\/div>","<div class='gt_from_md'><p>I mentioned slurmjobs::job_loop and also described how to use it in conjunction with getopt in R for looping over several variables<\/p>\n<\/div>","<div class='gt_from_md'><p>She had an issue installing a particular R package, which we pinpointed to which modules were loaded (including via bashrc). I also described the general problem of SLURM and SGE versions of JHPCE sharing a library path, which could result in subtle problems. I recommended deleting SGE packages (better would‚Äôve been backing up!) and reinstalling on SLURM<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped Fernando modify my code for creating input files to work witht the new pseudobulk data. Wegot the gene PCs done, he needs to get the genotype files together for this data before we can move to the next steps. Planning to meet more next week to continue this process.<\/p>\n<\/div>","<div class='gt_from_md'><p>Resolved some questions related to <a href=\"https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq\">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq<\/a> and made two pull requests to fix typos.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over the math derivations for interpreting beta1, beta2, and beta3 on an interaction model where X1 is a continuous variable, X2 is a discrete variable with 2 options and we model <code>Y ~ X1 * X2<\/code>. We also used <code>ExploreModelMatrix<\/code>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained the benefits of centering and scaling and referred to both the short video on this topic <a href=\"https://youtu.be/c8ffeFVUzk8\">https://youtu.be/c8ffeFVUzk8<\/a> as well as the material we developed for the CSHL 2023 course <a href=\"https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html\">https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>Peter and Reza were interested in running deconvolution on the bulk MDDseq with the new large MDD snRNA-seq dataset. The MDD snRNA-seq dataset is only sACC, we concluded that only haveing one region might make this analysis a bit awkward for the Amyg data. I them to the current deconvolution results and how we applied thoes results in downstream analysis.<\/p>\n<\/div>","<div class='gt_from_md'><p>I walked Fernando through using registration_pseudobulk() on the large MDD single cell dataset and went over how the function works. We are planning to meet agian to go over running PCA to prep for eQTL analysis.<\/p>\n<\/div>","<div class='gt_from_md'><p>I noted the ‚Äòwww‚Äô directory (or the directory passed to the ‚Äòdocs_path‚Äô parameter) controls the home page, and we typically use a relative soft link to the main repository README for the bulk of the content.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the general workflow, involving ImageJ, Samui, VisiumStitcher, and custom R + python code needed to eventually produce SpatialExperiment objetcs with one donor per sample. I focused on the high level of what the Samui piece looked like/ how much work is involved<\/p>\n<\/div>","<div class='gt_from_md'><p>Continued reviewing material from <a href=\"https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq\">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>Discussed a few ideas about LIBD and social media<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed code examples of writing array jobs with SLURM (and how it compares to SGE), then diagnosed an X11 forwarding issue where the ‚Äò-x11‚Äô srun option was insufficient. Adding X11-related configuration in the SSH config fixed the issue (i.e. ForwardX11 and ForwardX11Trusted lines)<\/p>\n<\/div>","<div class='gt_from_md'><p>I discussed diagnosing marker quality with Maddy + Stephanie Page for the spatial hypocampus project. They are on the right track, and have created violin + hockystick plots for the different iterations of the dataset. I advised they elimate a noisy cluster and re-compute markers.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the pieces that go into a bookdown-powered documentation site, showing RMD code examples and the GHA workflow that deploys the site<\/p>\n<\/div>","<div class='gt_from_md'><p>I noted that Samui supports many types of features, and the spatial coordinates of cells can be supplied instead of spot coordinates to display features related to cells<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the meaning of cell2location outputs and how to compare outputs between software tools. I also described where plots came from in R<\/p>\n<\/div>","<div class='gt_from_md'><p>Alan has used MAD cutoffs on total UMIs, genes detected, mito %, and a metric we haven‚Äôt worked with yet: genes per UMI. He evaluated the data all together (not by sample) and chose #Mads by what seemed reasonable. The values he has used look good, but perhaps a bit strict. We looked at the distributions, which are slightly different between samples, I advised he use scater::isOutlier to see what the cutoffs would be by sample. One step he is lacking is doublet detection, I suggested he apply scDblFinder::computeDoubletDensity. Alan already has clusteres and annoted this data (lots of work) I suggested that a great double check is to calcluate the median value of the QC metrics for each final cluster and see if any are much higher than the others as a good double check that his initial QC process worked well.<\/p>\n<\/div>","<div class='gt_from_md'><p>Answered questions related to material at <a href=\"https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq\">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about the first scripts in <a href=\"https://github.com/LieberInstitute/ranger_metrics\">https://github.com/LieberInstitute/ranger_metrics<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>He noted that not all measured genes were displayed in the Visium_SPG_AD project. I found in the code why I subsetted and made a GH issue to fix it<\/p>\n<\/div>","<div class='gt_from_md'><p>I shared my favored set up for getting computational work done: working directly on JHPCE via the SSH extension for VSCode, using system-wide JHPCE modules for consistent software versioning, and X11 for interactive plotting (with a note I tend to just create and view PDFs for complex plots)<\/p>\n<\/div>","<div class='gt_from_md'><p>My major recommendation was learning version control with git, though I also mentioned learning SGE/SLURM also is a valuable skill for larger-scale coding projects. I helped him create a git repository for his current project (including account set up) and described the basics of git add, commit, and push<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how the various scripts connect in spatialDLPFC for my work with deconvolution, to help share and re-use them for her project<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about material at <a href=\"https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq\">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>I explained the status of the recount-pump transfer, where files were, and noted that chris or JHPCE admins would have to change permissions to allow her access to some files<\/p>\n<\/div>","<div class='gt_from_md'><p>I noted that modules at SGE vs. SLURM are totally disjoint, and that the HOSTNAME trick originally used in the team recommended bashrc didn‚Äôt (at the time, though this has recently changed) work anymore, and the SLURMD_NODENAME variable can instead be used to infer where the user is<\/p>\n<\/div>","<div class='gt_from_md'><p>I gave some pointers about writing tweets. Mostly keep it simple, give people a call to action ‚Äúcheck out this link!‚Äù, add something eye catching. Also to spend sometime reading tweets and pay attention to what you like that other people tweet? What caught your interest? We also talked some stradgedy about tweeting around confrences, calling peoples attention to your talk/making it easy for people to find you to get in touch<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about how I became an Investigator at LIBD, the overall type of work we do as a team, some of our main collaborations, and overall plans for applying for funding in the future. In particular, I highlighted how I want to train team members on how to write an NIH biosketch, though not all are going to be ‚Äúkey personnel‚Äù. I sent a link to the DSgs, team meeting schedule, rstats club schedule, and our <a href=\"http://lcolladotor.github.io/talk/teamtldr2022/\">http://lcolladotor.github.io/talk/teamtldr2022/<\/a> summary from April 20, 2022.<\/p>\n<\/div>","<div class='gt_from_md'><p>I did a complete tutorial of using the Samui API to create Samui-importable directories, including how to preprocess image files and how to host results publicly<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked thorough the thigns to think about for the Mathy‚Äôs dataset (this dataset has case + control paitents and found DEGs w/in cell types). I advised to start simple with the broad cell tytpes and not consider Dx at first. Check if the DEG they idetified are in our marker set. I think this will need some discussion with the project team if Dx is a concern for spot deconvolution.<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/5e84f5ce32e6723b1a1eea20dc47ee320d55ca10\">https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/5e84f5ce32e6723b1a1eea20dc47ee320d55ca10<\/a><\/p>\n<\/div>","NA","<div class='gt_from_md'><p>He noted that just taking the most plentiful cell type in any cluster was leading to too many clusters with excitatory layer-5 cells. I mentioned that the raw outputs of spot deconvolution tools are fairly unreliable, but they still carry useful information when normalized. For his example, I mentioned dividing by the total number of cells for each cell type when making comparisons between cell types<\/p>\n<\/div>","<div class='gt_from_md'><p>Suggested deleting all old files as the code might be appending data instead of creating new files and also increasing the h_fsize, as the error relates disk space<\/p>\n<\/div>","<div class='gt_from_md'><p>guided him to the search engine on REDCap<\/p>\n<\/div>","<div class='gt_from_md'><p>For deconvolution, we talked about finding an appropriate reference dataset and we talked about an expert we can contact to ask which dataset(s) might be the most relevant. For snRNA-seq data analysis, I mentioned OSCA (Orchestrating Single Cell Analysis with Bioconductor) and how Louise could help guide this person on our code based off OSCA through a series of DSgs sessions.<\/p>\n<\/div>","<div class='gt_from_md'><p>removing the top 2500 pixels before splitting the image corrects the offset<\/p>\n<\/div>","<div class='gt_from_md'><p>I mentioned (information) entropy as a metric that would determine how well a gene is confined to a particular cluster (using average counts of a gene across all spots in each cluster, then normalizing the sum to add to 1), but cautioned that I wasn‚Äôt familiar with favored metrics within the field<\/p>\n<\/div>","<div class='gt_from_md'><p>We transferred <a href=\"https://github.com/LieberInstitute/qsvaRPkgDemo\">https://github.com/LieberInstitute/qsvaRPkgDemo<\/a> to the LieberInstitute account. I asked to include some citations for sources (like the image source) and some other small edits. For GO / KEGG enrichment with clusterProfiler I asked to not filter results (pvalueCutoff / qvalueCutoff parameters), so we can see more results prior to choosing which ones to plot.<\/p>\n<\/div>","<div class='gt_from_md'><p>We ttok a look at the changes in the clustring in the spatial data before and after cropping out a area of questionalble quality. To me it looks like the QC params are too strict leaving spots floating in space, which I‚Äôm guessing is contributing to weird BayesSpace results (but this is a guess). We also talked code organization strtagedy - mostly keep it simple and consistent<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped Hedia improve the formatting &amp; content for her BioC qSVA workshop, main feedback ins to focus on and highlight the qsvaR steps in the workflow (rather than SummarizedExperiment overvew &amp; DEG analysis)<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the current status of the team, the project, what skills are required, boundaries in terms of time and who would be involved (and who wouldn‚Äôt). We ended the session with me relaying a list of resources to check and read more carefully. Then in about 2 weeks we‚Äôll meet again to determine the next steps.<\/p>\n<\/div>","<div class='gt_from_md'><p>We made a Google sheet with columns such as <code>study_type<\/code>, <code>skill_name<\/code>, <code>learning_priority<\/code>, etc. I also worked on it after the DSgs session to help complete the current information. See <a href=\"https://neuralplasticity.slack.com/archives/C04Q8MX99FY/p1690397027111449\">Slack<\/a> for more details.<\/p>\n<\/div>","<div class='gt_from_md'><p>We resolved this issue by making sure that the variable names for which they wanted to specify the correct colors were actually part of the colData(SE). See <a href=\"https://jhu-genomics.slack.com/archives/C01KSLPCGAC/p1689858182290369\">Slack<\/a> for the actual code links. I also asked them to make sure to use soft links / full paths instead of using setwd() for files outside the project + to keep all related files at JHPCE instead of on a laptop, to make it easier to rule out any sources of differences (aka for reproducibility purposes).<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped Daianna practice her BioC short talk, Mostly she needs to give more slides to discussing her results &amp; include more actual plots not just icons (she much improved this presentation for the 7/26 meeting :) )<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped discover the source and solutions to errors and warnings (mostly from BiocCheck and R CMD check), and found how to specify a single author in in-text citations<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>reffered to this script /dcs04/lieber/marmaypag/Image_Processing/NM_LC_segmentation/test.m<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>We discussed what would be the appropriate material for us to teach in a course. We will re-use some of our materials from <a href=\"https://github.com/lcolladotor/cshl_rstats_genome_scale_2023\">https://github.com/lcolladotor/cshl_rstats_genome_scale_2023<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over the proposed response to reviewers, clarified some sentences, and requested co-authors to approve the response to the reviewers.<\/p>\n<\/div>","<div class='gt_from_md'><p>I pointed her to spot deconvolution scripts used in spatialDLPFC and the high-level overview of the workflow, as well as how we trained the decision tree classifier<\/p>\n<\/div>","<div class='gt_from_md'><p>I noted that he ran out of memory in his JHPCE job, which crashed R, so I suggested increasing ‚Äòmem_free‚Äô and ‚Äòh_vmem‚Äô<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/tif2mat.m\">https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/tif2mat.m<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>I referred to examples of JHPCE modules I‚Äôve created that activate shared conda environments, and explained the key commands used to get a similar module working<\/p>\n<\/div>","<div class='gt_from_md'><p>I recommended finding and trying to generalize the process a previous coworker followed to produce the list of degradation-associated transcripts, to limit relying on exact transcripts with an exact GENCODE version. I noted that you can convert transcript data to genes (albeit with lost information), but not vice-versa, so in theory either transcript or gene-level data could be supported by converting to genes<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/mat2tif.m\">https://github.com/LieberInstitute/spatialDLPFC_SCZ/blob/main/code/vistoseg/lib_updated/mat2tif.m<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>referred here for clarification <a href=\"https://docs.google.com/presentation/d/1HJgkvlHtiIA5nucuNdHdYhRGAtpPPvASGFoAocqi1-U/edit#slide=id.g1163185b778_1_6\">https://docs.google.com/presentation/d/1HJgkvlHtiIA5nucuNdHdYhRGAtpPPvASGFoAocqi1-U/edit#slide=id.g1163185b778_1_6<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>We adapted code from <a href=\"https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R\">https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R<\/a> and <a href=\"https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd\">https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd<\/a> in order to make the files at <a href=\"https://github.com/LieberInstitute/septum_lateral\">https://github.com/LieberInstitute/septum_lateral<\/a>. See <a href=\"https://github.com/LieberInstitute/septum_lateral/commits/main\">https://github.com/LieberInstitute/septum_lateral/commits/main<\/a> from <a href=\"https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50\">https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50<\/a> to <a href=\"https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c\">https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c<\/a> as well as related commits on the <code>gh-pages<\/code> branch.<\/p>\n<\/div>","<div class='gt_from_md'><p>Summarized at <a href=\"https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659\">https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659<\/a>. This involved explaining why we don‚Äôt use as a background universe all genes in the annotation in gene ontology / KEGG pathway enrichment analyses. Instead, we use those that pass the expression filters and for which we computed differential expression statistics.<\/p>\n<\/div>","<div class='gt_from_md'><p>We resolved <a href=\"https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199\">https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199<\/a> at <a href=\"https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&amp;cid=C02RG5VGL6R\">https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&amp;cid=C02RG5VGL6R<\/a>, which was due to a user incorrectly having a Linux group assigned to files at JHPCE when they are not part of the user group. Uploads through Cyberduck can trigger this. The solution is to run <a href=\"https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example\">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example<\/a> again on a qrsh session.<\/p>\n<\/div>","<div class='gt_from_md'><p>Cynthia has run DropletUtils::emptyDrops on the two LIBD + two example samples. We first clarified some terms and outputs she had recorded. For the most part DropletUtils returns similar number of non-empty cells as CellRanger arc, but Cynthia didn‚Äôt confirm that the two methods flag the same cells. I asked her to creat a confision matrix between the two methods for each sample to check this.  The main concern Cynthia has was the LIBD sample with only  3k cells input, has 9k non-empty droplets returned by emptyDrops. This is quite odd as the CellRanger Arc metrics look quite bad for this sample, but the DropletUtils metrics look fine (there are high UMI values for many droplets). We will want to dig in to this more to understand the differnces between the two methods.<\/p>\n<\/div>","<div class='gt_from_md'><p>We worked on qsvaR debugging an issue raised by Renee and Louise. We then improved the error messages on qsvaR as noted at <a href=\"https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400\">https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>Using the septum lateral data, I trained Ryan on how to make this type of Globus endpoint, how to set up the right permissions, how to edit and update <a href=\"http://research.libd.org/globus/\">http://research.libd.org/globus/<\/a> (see <a href=\"https://github.com/LieberInstitute/globus/blob/b10e0fb89c40489d23028e8ed3f7812cdea76a6b/index.Rmd#L57\">this<\/a> for details), and how to document the Globus information on a manuscript.<\/p>\n<\/div>","<div class='gt_from_md'><p>Using the lateral septum snRNA-seq data, I trained Heena on how to filter a SingleCellExperiment object to reduce the required memory from 7GB to less than 1GB in order to be able to deploy an iSEE app at shinyapps.io, whose computers have a maximum of 8 GB of RAM. See <a href=\"https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98\">https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98<\/a> and related commits.<\/p>\n<\/div>","<div class='gt_from_md'><p>We worked with the script where Renee had noticed issues with qsvaR. We adapted it to work on qsvaR local updates through devtools::load_all(). We also resolved <a href=\"https://github.com/LieberInstitute/qsvaR/issues/29\">https://github.com/LieberInstitute/qsvaR/issues/29<\/a> and <a href=\"https://github.com/LieberInstitute/qsvaR/issues/28\">https://github.com/LieberInstitute/qsvaR/issues/28<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>We started with getting github token access for Hedia‚Äôs laptop working. Then started a pull request using usethis for qSVA, for Hedia to add seed capabilities to several functions. We will need to meet several more times to complete the whole help request<\/p>\n<\/div>","<div class='gt_from_md'><p>Uma mentioned that she had to login everytime she pushed a commit from the directory in question (but not other repos). I thought we should start by using a github access token to update her login to github from JHPCE. The best way I know how to do this is to use gitcreds::gitcreds_set() in R. At first it gave us an error, which was solved by adding the git modules loaded in the team bashrc to Uma‚Äôs basrc file as well as her .gitconfig file. After that we were able to save the token to her JHPCe setup and the github push issue seemed to be solved<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the ‚Äòadd_coords‚Äô function, how features link to existing coordinates, and that coords must have at least one associated feature to display<\/p>\n<\/div>","<div class='gt_from_md'><p>I referred her to recent versions of my code (spatialDLPFC and SPG_AD projects) and walked through the high-level steps of each<\/p>\n<\/div>","<div class='gt_from_md'><p>Reviewed input for tensorqtl (it is counts, not logcounts), and revised what how we filter lowly expressed exon-exon junctions using <a href=\"http://research.libd.org/jaffelab/reference/expression_cutoff.html\">http://research.libd.org/jaffelab/reference/expression_cutoff.html<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussed how to best pseudobulk the data prior to doing this type of analysis. In particular, how to adapt <a href=\"https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148\">https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148<\/a> and related code, as well as plots to make such as boxplots of the min_ncells by cell type (without any filtering) and cutoffs found by <code>isOutlier()<\/code> (different cutoff for each cell type) + PCA plots colored by min_ncells.<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussed a few different areas of growth, including a change in direction related to our outreach activities.<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed that the error was essentially a failure to allocate memory and recommended doubling required memory for the failing SPEAQeasy process until success<\/p>\n<\/div>","<div class='gt_from_md'><p>I described that an entire directory (rather than all of its contents) can be ignored, and should be specified relative to the directory containing the .gitignore<\/p>\n<\/div>","<div class='gt_from_md'><p>We looked at some old code vs. an adapted version, which was not producing the same output. I recommended splitting the code into ordered sections, and executing the old and new version of each to identify exactly where the error was occurring<\/p>\n<\/div>","<div class='gt_from_md'><p>I discovered she was using the ‚Äìsmall_test argument and not ‚Äìinput, so only the small 3-sample test set was running instead of her actual data<\/p>\n<\/div>","<div class='gt_from_md'><p>I walked through the meaning of certain information-dense plots I made to show spot-deconvolution results<\/p>\n<\/div>","<div class='gt_from_md'><p>I recommended filtering rows of the RSE via ‚Äòseqnames‚Äô, which would return genes (or features in general) of the particular chromosomes needed<\/p>\n<\/div>","<div class='gt_from_md'><p>She asked about any quality-check-related options SPEAQeasy had (I pointed her to ‚Äò‚Äìercc‚Äô for applicable data) and whether trimming defaults were reasonable for her data<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the general workflow needed to start from raw images and spaceranger outputs, and end up with a Samui-compatible directory that can be dragged into the browser (locally) to view Visium data (including segmented fluorescence channels)<\/p>\n<\/div>","<div class='gt_from_md'><p>I suggested creating a template configuration file, then looping through her data, using the sed command to write configuration files with the relevant variable replaced from the template<\/p>\n<\/div>","<div class='gt_from_md'><p>She wanted to add the columns associated with unique rows in one data frame to the other, so I suggested combining the ‚Äòunique()‚Äô command with ‚Äòleft_join()‚Äô<\/p>\n<\/div>","<div class='gt_from_md'><p>I suggested using regular expressions in R to get the new filename from a given old filename, then file.copy to perform a safe test, and finally deleting the old files<\/p>\n<\/div>","<div class='gt_from_md'><p>She was unable to push to GitHub on her laptop. It appeared she didn‚Äôt have an SSH keypair set up with GitHub, so we set that up to fix the issue<\/p>\n<\/div>","<div class='gt_from_md'><p>Hedia was concerned about the large number of variables in the reproducibility DE experiment, specifically the PC. I pointed out that the point of the anlaysis was to test the effectiveness of the models (with and without qSVs) so while this might be a valid concern in a sperate DE analysis, we are testing model quality here. I also noticed that the reproducibility anlaysis was not using functions from the qsvaR package, and that as qsvaR is the ‚Äúproduct‚Äù of this whole project its important we use the functions in this anlysis .<\/p>\n<\/div>","<div class='gt_from_md'><p>I described typical expected alignment rates, the difference between concordant and overall map rate, and details about other SPEAQeasy output metrics<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained that at the time, I just created the RMD file and added any referenced images to the rstatsclubsource repo, following naming conventions that include the post date and hyphens between words in the blog title. I believe Leo did some manual work to get it fully published, so I referred to the full guide in case my method was incomplete: <a href=\"http://research.libd.org/rstatsclub/2018/03/09/contributing-to-the-libd-rstats-club/\">http://research.libd.org/rstatsclub/2018/03/09/contributing-to-the-libd-rstats-club/<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>I explained how cell2location is agnostic to what gene identifiers are used, but that gene indentifier type must match between spatial and reference single-cell data. I mentioned the org.Hs.eg.db package.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described when I make a module for python-based tools, the benefits of conda vs. python virtual environments, and related topics<\/p>\n<\/div>","<div class='gt_from_md'><p>I suspected improper configuration of email associated with her git commits, which was indeed the problem. Her commits moving forward are associated with her GitHub account, but we couldn‚Äôt figure out how to retroactively regain credit for commits done with the wrong email<\/p>\n<\/div>","<div class='gt_from_md'><p>I described ‚Äògit pull‚Äô to merge in remote changes, and using the ‚Äòvi‚Äô editor to write a merge message<\/p>\n<\/div>","<div class='gt_from_md'><p>I discussed with Renee the way I code and showed her some examples and gave her a few coding advices.<\/p>\n<\/div>","<div class='gt_from_md'><p>Looks like user was trying to create a plot and import data from a .csv file but was getting an error importing the csv\n<a href=\"https://community.rstudio.com/t/error-creating-a-windrose/164132\">https://community.rstudio.com/t/error-creating-a-windrose/164132<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>I explained which paths I‚Äôd need to get Samui running on our data, and we discussed which features might be useful to show<\/p>\n<\/div>","<div class='gt_from_md'><p>Project links reviewed: <a href=\"http://research.libd.org/smokingMouse/\">http://research.libd.org/smokingMouse/<\/a> and <a href=\"http://research.libd.org/smokingMouse/articles/smokingMouse.html\">http://research.libd.org/smokingMouse/articles/smokingMouse.html<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>I described how ACLs are properly set up at dcs05 to inherit downward automatically, so just an ‚Äòrsync‚Äô/ move to trash is necessary in those transfers<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained the purpose of Git-LFS, that it‚Äôs configured for certain repositories, and that we can load a module to use it<\/p>\n<\/div>","<div class='gt_from_md'><p>Website tools to look for public rna-seq samples &amp; packages to make synthetic replicates <a href=\"https://support.bioconductor.org/p/9150681/#9150698\">as shown here<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described my use of XQuartz, the ‚Äòxhost +local:‚Äô command, and the proper ordering of these tools in combination with SSHing to allow plotting to work<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the use of qsvaR. For now the qsvaR only use the Gencode25 annotation. We also commented on the steps needed in case we want to implement the package for the more updated annotation versions in the future.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described what we use modules for, the general approach to making them for common software types (binaries, python packages), and walked through the process by making a new FastQC module<\/p>\n<\/div>","<div class='gt_from_md'><p>Trish and I went over the code I previoulsy wrote for tensorQTL, and some of the custom functions for preparing the input data. How to input the different datatypes in to python, and run the function, and strategies  for running the different cell types in parralell on the GPU queue<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed each step of the pipeline and the outputs. We also had a look at the QC metrics and the code that Cynthia added to the script.<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped Daianna decipher an error message she got in the BioC check in the GitHub Actions involving a local path harcoded in to one of her tests. We also debugged for similar issues by searching her code for other local paths that would cause her code  to break when run remotely<\/p>\n<\/div>","<div class='gt_from_md'><p>We reviewed the overflow to analyse RNA-Seq and ATAC data with the Seurat and Signac tools. I provided learning resources, such as data file organization, funtions and methos available in the tools. I also provided a complete explanation about QC for preprocessing data integrated.<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped him set up git on his laptop (username + email, SSH key), showed how to edit R Markdown files using RStudio, and basic version control (git add, commit, and push).<\/p>\n<\/div>","<div class='gt_from_md'><p>The error message was cryptic, but made more clear in the latest SPEAQeasy: the transcripts FASTA mismatched with the GTF (custom annotation was used) due to a typo in the gene ID in the GTF.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went through the  <a href=\"https://github.com/HediaTnani/Int_API_R\">https://github.com/HediaTnani/Int_API_R<\/a>. Then we saw other API examples with authentication (github, ‚Ä¶.).<\/p>\n<\/div>","<div class='gt_from_md'><p>Issue and the solution found here <a href=\"https://support.bioconductor.org/p/9150363/#9150404\">https://support.bioconductor.org/p/9150363/#9150404<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>user had question about generating rarefaction curves in R. Issue found here <a href=\"https://support.bioconductor.org/p/9150366/#9150386\">https://support.bioconductor.org/p/9150366/#9150386<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>I shared the tensorQTL example I presented at rstats club. We also went over the different in-puts and formats, including some we can re-use from bulk MDDseq. WE also discusses the differnt setting for tensorQTL and how to adapt sh files to submit to the GPU queue<\/p>\n<\/div>","<div class='gt_from_md'><p>I initially mistakenly suggested kallisto, but later clarified that using featureCounts with a custom single-gene GTF from the BAM files (produced by SPEAQeasy) would be the proper way to quickly quantify a single gene<\/p>\n<\/div>","<div class='gt_from_md'><p>See <a href=\"https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099\">https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099<\/a> for more. Led to <a href=\"https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6\">https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6<\/a> and related commits. spatialLIBD version 1.11.10 now supports spaceranger version 2023.0208.0 output files.<\/p>\n<\/div>","<div class='gt_from_md'><p>We used <a href=\"https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection\">https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection<\/a> and also went over the JHPCE and R config files from the team website. We also talked about the DSgs and how it would be beneficial for Kinnary to meet 1 on 1 with all DSgs guides to build familiarity and make it easier to ask for help in the future.<\/p>\n<\/div>","<div class='gt_from_md'><p>See <a href=\"https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179\">https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179<\/a> for more details. Heena is working with spaceranger output files from 2 different versions, and in particular, version 2023.0208.0 (internal 10x Genomics version) changed their output files structure<\/p>\n<\/div>","<div class='gt_from_md'><p>Described what we do, the DSgs system, potential for visitors at our team meeting, and research ideas.<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed how to merge two datasets in R, making sure they shared the exact set of genes<\/p>\n<\/div>","<div class='gt_from_md'><p>With Cynthia we saw the differents folders that are part of the template project repository and how to organize your project repository.<\/p>\n<\/div>","<div class='gt_from_md'><p>The solution involved asking BitSupport to login as a specific user and run <a href=\"https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example\">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example<\/a>. Knowing the exact file paths that triggered the permission issues are important to help check with <code>ls -lh<\/code> and <code>nfs4_getfacl<\/code> under a <code>qrsh<\/code> session.<\/p>\n<\/div>","<div class='gt_from_md'><p>See the <a href=\"https://community-bioc.slack.com/archives/CR1BLV821/p1677783580326559\">Bioconductor Slack message<\/a> I wrote for more details.<\/p>\n<\/div>","<div class='gt_from_md'><p>End result: <a href=\"https://twitter.com/HeenaDivecha/status/1631360771342098433\">https://twitter.com/HeenaDivecha/status/1631360771342098433<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>See <a href=\"https://github.com/drighelli/SpatialExperiment/issues/135\">https://github.com/drighelli/SpatialExperiment/issues/135<\/a> and the pull request at <a href=\"https://github.com/drighelli/SpatialExperiment/pull/136\">https://github.com/drighelli/SpatialExperiment/pull/136<\/a> that resolves this issue.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the need to ‚Äògit add‚Äô both the old filenames and new filenames, and the use of ‚Äògit add ‚Äìall‚Äô to reference deleted files.<\/p>\n<\/div>","<div class='gt_from_md'><p>I recommended code for matching two data frames up by common genes, and explained that matrices don‚Äôt support multiple data types (instead a ‚Äòdata.frame‚Äô might be used)<\/p>\n<\/div>","<div class='gt_from_md'><p>Sreya was having some issues with functions from DeconvoBuddies, namley plot_marker_express. I am going to check that on my end as it seems like her computer wasn‚Äôt finding the support functions for the main function. She was also having trouble matching the gene names between the sce and bulk rse objects, the ensembl names seemd to have differnt versions, I advided she maybe switch to using the gene symbols to keep things simple.<\/p>\n<\/div>","<div class='gt_from_md'><p>No public result just yet.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over the ratioinal behind psuedobulking and how we can use heatmaps to show marker experssion. Created psuedobulk data for three different clustering methods, and went over the basic process of making heatmaps. Bukola will work on the aestetics of the heatmaps, and cerate one for all thee clustering methods.<\/p>\n<\/div>","<div class='gt_from_md'><p>related to my meeting with Uma on 2/10. We fixed Boyi‚Äôs bashrc to allow group write by deafult, fixing the ,git issue and eliminating the need to re-run the permission update script often<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed how glob expressions can be used to conveniently match many files across potentially several files, and described gitignore-specific rules, such as certain wildcard expression matching files no matter what directory they are in<\/p>\n<\/div>","<div class='gt_from_md'><p>Uma was having file permission issues, the group permissions for several files in a shared project were set to read only. This was causing issues with the team using scripts but also in making git commits as some of the idex files in .git were read-only. We updated her bashrc files to contain ‚Äúumask u=rwx,g=rwx,o= ## equivalent to umask 007‚Äù to prevent this in the future and ran the premissoin update script.<\/p>\n<\/div>","<div class='gt_from_md'><p>Bukola hads run muliple clustering methods on the habenula data, I advised she pick the two that were the most simmlar to ERiks previous annotations (we were happy with) via rand index, and start examining marker gene expression.<\/p>\n<\/div>","<div class='gt_from_md'><p>I went over details like naming the OneDrive folder and correctly using CrossReference to refer to that name in figures and references to those figures<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped port several plots into Illustrator as a figure, describing our normal process of using PDFs and linked files to maintain resolution.<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed how to filter for individuals with a certain condition, and subset two data frames with the same criteria.<\/p>\n<\/div>","<div class='gt_from_md'><p>She was unable to clone private repositories due to a problem with her SSH key with GitHub. We solved the problem by setting up a new key pair using the recommended Ed25519 algorithm instead of the deprecated RSA algorithm.<\/p>\n<\/div>","<div class='gt_from_md'><p>We focused mostly on the options Nextflow provides for managing software dependencies: Lmod modules, conda environments, and docker + singularity containers and the benefits of each potential approach<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed how to implement github actions with Nextflow workflow and the use of Conda and/or Docker containers.<\/p>\n<\/div>","<div class='gt_from_md'><p>We identifed that SPEAQeasy cannot handle runs with only one sample, but that the intention was to use multiple samples (the manifest did not use unique sample names)<\/p>\n<\/div>","<div class='gt_from_md'><p>Hope created a github account and I talked her thuogth the basics of git, But we hit a snag with the instalation of git on her laptop, git is built in to xcode in the ventura OS (which her machine is running) but she had some trouble with that instalation too. Shes going to try and fix that independtly, and reasch out if she needs more help gettign off hte ground with git + github<\/p>\n<\/div>","<div class='gt_from_md'><p>We adapted code from the DLPFC work and <a href=\"https://www.stephaniehicks.com/biocdemo/articles/Demo.html\">https://www.stephaniehicks.com/biocdemo/articles/Demo.html<\/a> to run GLM-PCA to prep for batch correction<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over several concepts in Nextflow (e.g. processes and channels) and discussed how to invoke R scripts inside processes.<\/p>\n<\/div>","<div class='gt_from_md'><p>We (Me + Nick) discussed with Sean the basics of Nextflow and Nextflow DSL2 and how to make processes and channels and workflows. We gave him resources to read  and gave him small examplesto follow. We helped him also with his errors on slack.<\/p>\n<\/div>","<div class='gt_from_md'><p>Bukola and I adapted QC code for the habenula data<\/p>\n<\/div>","<div class='gt_from_md'><p>See <a href=\"https://jhu-genomics.slack.com/archives/C0498763CUC/p1673451228034439\">Slack notes<\/a> for further details. They involved <a href=\"https://github.com/LieberInstitute/template_project\">https://github.com/LieberInstitute/template_project<\/a>, <a href=\"https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h\">https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h<\/a>, and other specific instructions for setting up a project.<\/p>\n<\/div>","<div class='gt_from_md'><p>Continuing Sreyas work with the single cell mouse data, I pointed her towards scater::plotExpression to plot some of the genes her group is intested in observing by cell type. However we ran in to some bugs with running logNormCounts which we think is from an old version of R. I advised that she update R and re-install the packages, hopefull that will fix the issue.<\/p>\n<\/div>","<div class='gt_from_md'><p>We uncovered the source of some unwanted NA values in some genotyping data and eliminated it by removing some code<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped Sreya build a single cell experiment object with expression and metadata csvs she had from an publication. Hopefully she can use this to independently explore the gene expression her group is interested in.<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about how to compute KL divergence between 2 groups of cell-type proportions on her data using a variation of my code<\/p>\n<\/div>","<div class='gt_from_md'><p>Hedia wanted some background information about the deconvolution process we applied to the degradation data. I explained the mean ratio marker finding, the resulting stats, and how we ued them to build our set of marker genes. Then the script for how we ran bisque + the output of bisque .<\/p>\n<\/div>","<div class='gt_from_md'><p>Sreya wants to test if a gene of intrest to her group is associaited with a ceratain cell type. They are working with bulk mouse data. I suggested that deconvolution may not be able to answer theire innital question but can provide depth to their bulk analysis. I suggested that they find a mouse single cell dataset that is relevant to their current data set, then first observe their gene of interest their, see if that answers any questions. Then they could use that data to run deconvioltion with bisuqe (I suggested a follow up meeting if she needs guidance with this)<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the manual process for resolving a merge conflict with git when git cannot automatically merge two branches<\/p>\n<\/div>","<div class='gt_from_md'><p>We pinpointed an error message as being due to trying to find the ‚ÄòcolData‚Äô of a list containing Summarized Experiments rather than calling ‚ÄòcolData‚Äô on the SummarizedExperiment itself<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how to color ggplot2 plots by a continuous variable, adjust the palette, and other ggplot-related syntax/tricks<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed how PCA reduces the dimesions of our data and helps show the variance between samples. I helped Bukola strategize how to make PCA plots for the habenula data. Start plotting the first 5 PCs pairwise, color by Dx. Then explore key QC or demographic variables, see if we can indentify any patterns. I was unsure if she should repeat this for exon, jx, tx, as we didnt for MDDseq. But figured gene was a good place to start and if the other features should be explored too, the code should be easily adapatabe.<\/p>\n<\/div>","<div class='gt_from_md'><p>I gave Hedia a breif overview of the goals and process of deconvolution, including mean ratio marker finding. I linked my deconvolution slides. We then went over the code for running bisque on the qSVA dataset, including details of the input, refrence, and output data.<\/p>\n<\/div>","<div class='gt_from_md'><p>I recommended some data-preprocessing steps for later downstream manipulation of the data<\/p>\n<\/div>","<div class='gt_from_md'><p>I gave general advice for how to preprocess some data, including changing data types, transposing matrices, and dropping columns to format things for a collaborator<\/p>\n<\/div>","<div class='gt_from_md'><p>I described ‚Äòfacet_wrap‚Äô and showed that for some situations, ‚Äòplot_grid‚Äô with a single output PDF (with width and height scaled appropriately) does well<\/p>\n<\/div>","<div class='gt_from_md'><p>We generally discussed linear mixed effects models and some common terms used as fixed effects in genomic data<\/p>\n<\/div>","<div class='gt_from_md'><p>Tricia is trying to run glmPCA on the very large MDD snRNA-seq dataset. She has a run submitted without parallelization that has been runing for 3 days. To try and speed things up she has been trying to use BioCparrallel, via MulticoreParam. But the scheduler keeps killing her job. We checked that -pe local X and MulticoreParam(X) match , so it wasn‚Äôt that. I recomended she try to run a subset of the data to get a sense of how glmPCA uses memory. But couldn‚Äôt come up with a fix to make the parallelization work. As its PCA it also can‚Äôt be run as subsets, so there aren‚Äôt other workarounds :( I think BioCparallel + sge scheduling would be a good topic for Rstats club as this seems to be a common headache<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://lcolladotor.github.io/bioc_team_ds\">https://lcolladotor.github.io/bioc_team_ds<\/a> for my team website, <a href=\"https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing\">https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing<\/a> for the LIBD rstats club schedule, <a href=\"https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials\">https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials<\/a> could be of interest also, <a href=\"https://github.com/LieberInstitute/template_project\">https://github.com/LieberInstitute/template_project<\/a> for a template for organizing projects, learn about the DSgs <a href=\"https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs\">https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs<\/a>, <a href=\"https://rstudio.github.io/reticulate/\">https://rstudio.github.io/reticulate/<\/a> for using Python from R<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how to transpose matrices, change their type, drop columns, and related tricks to solve a data-formatting problem asked by a collaborator<\/p>\n<\/div>","<div class='gt_from_md'><p>Lionel walked me through some plots they need for the LS paper: marker genes heatmap and expression violin plots. I pointed him to some code from DLPFC sn that should be helpful (he needs access to this repo). He also needs to get setup with JHPCE on his new laptop, I pointed him to the team laptop setup docs. He also is new to ggplot, so I pointed him to the visulization chapeter of R for data scientist. This will probaly require some follow up meetings<\/p>\n<\/div>","<div class='gt_from_md'><p>Daianna has added a new color scheme to the heatmaps that makes the differnces between positive and negative more clear, how ever I think the darkest red being close to 0 is misleading, so shes going to fix that. I also recomended she simplify the annotaions on the combined heatmap so its easier to find and interpret patterns.<\/p>\n<\/div>","<div class='gt_from_md'><p>We reviewed the reasons or correcting counts and why thats important for DE analysis. We went over the math behind TMM, and then how thoes ideas are used in edgeR. I pointed Bukola to a stats quest video with a more detatiled explination of how the edgeR correction works.<\/p>\n<\/div>","<div class='gt_from_md'><p>Sang Ho showed me the csv files that he exported from different moduled on HALO. They seem to have the same rows, so copying the needed column in to a differnt csv is easy in excel (where Sang Ho is working) but I showed him how to check that things are really aligned with some excel functions. We also looked at the data in prism, which I had to quickly learned about. I advised that he should show the data in boxplots not as a barplot to better display that data, and went over the checks for running an anova.<\/p>\n<\/div>","<div class='gt_from_md'><p>I mentioned rtracklayer::liftOver and showed a code example of lifting over genomic coordinates for a SummarizedExperiment<\/p>\n<\/div>","<div class='gt_from_md'><p>Daianna showed me her dot plots with the correct 0.05 q val filter, they looked good, ready for discussion. One tweak , is hshe had a combined up + down expresion col for some plots which was redundant and should be removed. She had also made some heatmaps that didn;t show very much difference between case control, htey probaly need to be normalized by the genes overall expression, but I need to check prevous work to remeber how to best do that<\/p>\n<\/div>","<div class='gt_from_md'><p>We went throught boxplots Bukola had made for the data, I suggested she add geom_jitter and how to make that work with geom_repel to help show more details about the data. We also discussed direct vs. standard evaluation of variable and how that makes writing funcitons with tidyverse tricky, but how she can use aes_() to do stadard eval with ggplot<\/p>\n<\/div>","<div class='gt_from_md'><p>I walked Bukola through our process for buld data QC, which involves making many exploratory plots of QC metrics and know groups (like batch, or case/control) to understand patterns and identify problematics samples. Then later implementing cutoff values either based on stats (like a 3 MAD cut off) or knowlege of ressonable values. Bukola will spend some time plotting QC values, adapting code from smoking mouse, then we will meet again to interpret these plots.<\/p>\n<\/div>","<div class='gt_from_md'><p>Daianna and I went over the dot plots she made for GO analysis from the multiple DE test she ran. Her plots look good, but we identified that she needed to use the default q cutoff to get useful results. She will rerun the GO analysis and replot the results.<\/p>\n<\/div>","<div class='gt_from_md'><p>We adapted my_plotExpression() (a function I wrote) to work for Bukola‚Äôs data. This required some debugging as subsetting to just the one gene in question altered how some internal dataframes looked. I also gave Bukola some pointers about saving plots using ggplot.<\/p>\n<\/div>","<div class='gt_from_md'><p>Sang Ho walked me through an analysis they want to do for the AD project, it involves annotating HALO data. We talked thought a strategy, this may be a bit computationally tricky to implement but it seems doable.<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about <a href=\"https://github.com/LieberInstitute/template_project\">https://github.com/LieberInstitute/template_project<\/a>, <a href=\"https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/\">https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/<\/a>, <a href=\"http://research.libd.org/sgejobs/reference/job_single.html\">http://research.libd.org/sgejobs/reference/job_single.html<\/a>, <a href=\"https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7\">https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7<\/a>, <a href=\"https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1\">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1<\/a>, <a href=\"https://happygitwithr.com/\">https://happygitwithr.com/<\/a>, <a href=\"https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent\">https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the need for SSH keys between the login and compute nodes at JHPCE, and how an incorrect set-up will manifest as a prompt for your password when using ‚Äòqrsh‚Äô. I referenced <a href=\"https://jhpce.jhu.edu/knowledge-base/authentication/ssh-key-setup/\">https://jhpce.jhu.edu/knowledge-base/authentication/ssh-key-setup/<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>We debugged her SSH set-up with JHCPE, finding a syntax mistake, confirming the contents of the ‚Äò.ssh/config‚Äô file, and other details to ultimately get the traditional RStudio + JHPCE set-up working smoothly. We touched a bit on the process for using git from JHPCE but didn‚Äôt finish fully setting things up<\/p>\n<\/div>","<div class='gt_from_md'><p>LIBD rstats club schedule <a href=\"https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing\">https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing<\/a>. Bioconductor overview video <a href=\"https://youtu.be/c0Ch_sXiGDQ\">https://youtu.be/c0Ch_sXiGDQ<\/a> and related notes in case you want to skim them <a href=\"https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing\">https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing<\/a> (we typically have this for each video entry). R and RStudio intro <a href=\"https://youtu.be/XTuJ8vGnzBU\">https://youtu.be/XTuJ8vGnzBU<\/a>. Example vignette <a href=\"https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html\">https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html<\/a> from <a href=\"https://bioconductor.org/packages/spatialLIBD/\">https://bioconductor.org/packages/spatialLIBD/<\/a>. Data Science guidance sessions <a href=\"https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0\">https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0<\/a>. JHPCE new user request <a href=\"https://jhpce.jhu.edu/register/user/\">https://jhpce.jhu.edu/register/user/<\/a>. Making your personal website with postcards <a href=\"https://youtu.be/9LUCZAx3SlU\">https://youtu.be/9LUCZAx3SlU<\/a>. Examples include <a href=\"https://lahuuki.github.io/\">https://lahuuki.github.io/<\/a> by Louise, <a href=\"https://jtleek.com/\">https://jtleek.com/<\/a>, <a href=\"http://aejaffe.com/\">http://aejaffe.com/<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how ‚Äòmodule load conda_R‚Äô lines in a submitted shell script override the default version of R that can be loaded by default using the ‚Äò.bashrc‚Äô. In this case this undesired interaction caused a corrupted R package to load and return an error<\/p>\n<\/div>","<div class='gt_from_md'><p>We adapted someone else‚Äôs code, slightly changing some awk commands to use cut (which I‚Äôm more familiar with) and using a for loop rather than piping a file to a while loop. This allowed the code to work as intended on the data<\/p>\n<\/div>","<div class='gt_from_md'><p>I pointed to installation guidelines for the cluster to resolve issues about R packages not being installed and a problematic nextflow version.<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Related to how we organize files for a publication.<\/p>\n<\/div>","<div class='gt_from_md'><p>Density is related to the probability density function, see [wiki](<a href=\"https://en.wikipedia.org/wiki/Kernel_density_estimation\">https://en.wikipedia.org/wiki/Kernel_density_estimation<\/a> for) more details. <a href=\"https://stats.stackexchange.com/questions/4220/can-a-probability-distribution-value-exceeding-1-be-ok\">This link<\/a> is also useful for understanding the y-axis in more detail. Violin plots are just density plots rotated 90 degrees and mirrored. They are compared against boxplots frequently, which while easier to understand, are not good for non-unimodal data. See <a href=\"https://mode.com/blog/violin-plot-examples/#:~:text=What%20is%20a%20violin%20plot%3F,the%20density%20of%20each%20variable\">this link<\/a> for more. I like <code>plotExplanatoryVariables()<\/code> because it‚Äôs fast to run and gives a general idea of things. See <a href=\"https://bioconductor.org/packages/release/bioc/vignettes/variancePartition/inst/doc/variancePartition.pdf\">Figure 1b<\/a> from <code>variancePartition<\/code> for a violin plot of similar data. <code>variancePartition<\/code> can handle mixed effects models (random effects) and in that scenario is more precise, but will also take longer to compute.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how to smoothly ‚Äògit pull‚Äô when updates have been made to both the local and remote repo (adding + committing local changes, running ‚Äògit pull‚Äô, and handling a smooth merge)<\/p>\n<\/div>","<div class='gt_from_md'><p>I described proper saving and loading of SummarizedExperiment derivatives whose assays are HDF5-backed using the HDF5Array package, and how to convert between HDF5-backed objects and ‚Äúordinary‚Äù in-memory objects<\/p>\n<\/div>","<div class='gt_from_md'><p>I went over the DE pipeline we use, noting that the model development is the most demanding and non-static part. We talked specifics for their dataset which has some experimental design to consider as its organids and is diffrent from patient case control. I posinted Geo to scripts where we do QC, expression cutoff, and my run_DE script<\/p>\n<\/div>","<div class='gt_from_md'><p>Heena saw old commits pop up on slacks git notifications when she pushed commits she had locally on her laptop, but they didn‚Äôt reappear in the git history. I think she did everything right and there might be something strange with the behavior of the slack message (as thats the only place things seemd wrong). We did some detective work between the files and the commits, everything seems good. Heena was able to then pull to the cluster with out conflicts<\/p>\n<\/div>","<div class='gt_from_md'><p>Had to work over slack again as Daianna was having trouble with zoom. Working with the feedback from the previous meetings, the experimetn has been narrowed down to 5 DE runs, and then comparing some pairs of the outputs. We were unsure if we should include mitoRate as a variable in the adult model as it correlates highly with totalAssignedGene, but decided to include it for now to match the model used in the pup DEA. I also sent her some code I have used to make scatter plots of t-stats to help compare DE results.<\/p>\n<\/div>","<div class='gt_from_md'><p>Check <a href=\"https://bioconductor.org/packages/devel/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html\">EnchancedVolcano<\/a> which shows a quick DE guide and finishes with some Volcano plots, which can be good enough for a grant application. If you want to use limma, you could follow <a href=\"http://research.libd.org/SPEAQeasy-example/de_analysis_speaqeasy.html#16_Modeling\">the SPEAQeasy example<\/a> or <a href=\"http://research.libd.org/SPEAQeasy-example/bootcamp_intro#5_Day_3\">our DE bootcamp<\/a> a bit. Or dive deeper into <a href=\"http://bioconductor.org/packages/release/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html\">limma‚Äôs documentation<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>We intended to meet on zoom, but Daianna‚Äôs connection was bad so we used Slack to chat. Daianna has made many plots and visuals about the many DE analysis for smoking mouse. I gave her some feedback about orgaizing her slides so the figures can practically be used inform DE models, right now everything is a bit scattered as shes planning on running 7 analysis so there is a lot to consider. We will need to meet again to finish this.<\/p>\n<\/div>","<div class='gt_from_md'><p>wrong input was given to the function<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>should select virtual stack option in bioformats reader when opening and multichannel image in imagej<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Daianna showed me her varianceParition plots, and we talked bout how to design the DE anlaysis based on what she‚Äôs oberved in the mouse data. I advised to keep things simpler (ie not create too many groups) but rather use models to control for vaiables adding variance. We will meet next to talk model design for this work.<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain how to pick randomn images to train PNN segmentation algorithm<\/p>\n<\/div>","<div class='gt_from_md'><p>We walked through the code to make a pseudobulked object and plotting the explanatory variables.<\/p>\n<\/div>","<div class='gt_from_md'><p>Note that <code>spe$var_colors<\/code> works for <code>spe$var<\/code> where the <code>var<\/code> part has to be the same in both variable names. Also <code>spe$var_colors<\/code> should be a named vector such that <code>unique(names(spe$var_colors))<\/code> matches <code>unique(spe$var)<\/code>.<\/p>\n<\/div>","<div class='gt_from_md'><p>code loops over the different file in the dataset and Marah was clearing it at the end of the loop every time which didnt let her save any data<\/p>\n<\/div>","<div class='gt_from_md'><p>We worked through a few issues in the cellpose module, which each had a different cause. In one case, the version of torch/CUDA was outdated for the hardware. In another case, user-specific libraries were being referenced instead of the central module‚Äôs libraries.<\/p>\n<\/div>","<div class='gt_from_md'><p>Daianna shared her updated QC plots, adn the one sample she planned to drop based on fastCellQC. I advised that she dig deeper in to how that QC finstion works, and what the cutoff values are, as well as exmaming mito vs. ribo precentages, and a two samples that don‚Äôt seem to group with others of the same phenotype /<\/p>\n<\/div>","<div class='gt_from_md'><p>NA<\/p>\n<\/div>","<div class='gt_from_md'><p>The error messages involved a mismatch between sample ID and file name for the ‚ÄòQualityTrimmed‚Äô process, as well as a supposed missing ID in the manifest (which was actually present). Simply resuming the pipeline got rid of both errors.<\/p>\n<\/div>","<div class='gt_from_md'><p>NA<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained how to get pheatmap to use the same color scheme for column and row annotations by making the annotaion tables have the same colname, and suppying a color pallet<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed a strategy to subset the 102 loci in question, and adapted code to subset the FDR signif pairs to work with the location info.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how to configure SPEAQeasy for a particular dataset, and how to manage two separate runs of SPEAQeasy as part of a different git repository (requiring the use of submodules)<\/p>\n<\/div>","<div class='gt_from_md'><p>NA<\/p>\n<\/div>","<div class='gt_from_md'><p>Daianna shared the updates she made to the QC plots based on our previous discussion, specifically separating brain and blood samples.<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed the documentation‚Äôs description of ‚ÄòmitoRate‚Äô and linked to the code where it‚Äôs computed: <a href=\"https://github.com/LieberInstitute/SPEAQeasy/blob/bfe4e664281b52c5e239e372393a8aea36084e0d/scripts/create_count_objects.R#L521-L533\">https://github.com/LieberInstitute/SPEAQeasy/blob/bfe4e664281b52c5e239e372393a8aea36084e0d/scripts/create_count_objects.R#L521-L533<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>She was getting a memory error when running BayesSpace and I ended up advising her to reduce the nrep argument. Also advised her on running fasthplus because it was taking too long. She needed to reduce the proportion of the data being used.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the need to manually convert some spatialExperiment slots prior to conversion with zellkonverter, and that some data types are not supported. I pointed to my SpaGCN code and the lines specifically responsible for clustering + writing outputs<\/p>\n<\/div>","<div class='gt_from_md'><p>Daianna walked me through her progress in the DE analysis. We went over QC plots and discussed experimental design. She will check in with Kristen about how to organize the brain vs. blood samples. Currently she dropped the blood sampeles as they looked very different in several QC metrics. But I advised that they just be handled seperatly as it makes senese they look different as they are different tissue, and the signal is probaly biological rather than technical<\/p>\n<\/div>","<div class='gt_from_md'><p>I described when ‚Äògit pull‚Äô can be ‚Äúsafely‚Äù used, and when ‚Äògit reset‚Äô might have to be used beforehand<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>I described the use of ‚Äòremotes::install_github‚Äô for GitHub-based packages. We encountered a GitHub authentication error that was solved by running the R code ‚ÄòSys.unsetenv(‚ÄúGITHUB_PAT‚Äù)‚Äô prior to the normal installation command.<\/p>\n<\/div>","<div class='gt_from_md'><p>she needed to have the plots as single ggplots instead of a list so I advised her to used vis_clus() instead. Shared code with her that I had used to make some of supplemental figures. She also asked if she need to discard spots before doing batch correction and I told her yes.<\/p>\n<\/div>","<div class='gt_from_md'><p>We fixed Sanjana‚Äôs ssh set up and got the qrsh command working. We got git configurated for her locally and on JHPCE, as well as set up tokens. Created a test repo, and made commits from her laptop and JHPCE as an example<\/p>\n<\/div>","<div class='gt_from_md'><p>checked size of the spe object. checked that only has low res image files. check dim of colData. All sizes checkedout. Believe this to be a package problem.<\/p>\n<\/div>","<div class='gt_from_md'><p>Maddy asked me how to get the total UMI‚Äôs for each sample out of her spe object. I wrote a small for loop for her in order to do it.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over github basics and motivation behind version control. Quickly discovered that Sanjana‚Äôs qrsh command is not working, requiring a password. Ended up using the session to troubleshoot that, with no success :( planning on circling back/re-setting up ssh for her. Will meet later about git setup<\/p>\n<\/div>","<div class='gt_from_md'><p>We checked Sreya‚Äôs code against the OSCA DE example run. She was worried that there weren‚Äôt may significant genes, but her results seemed to be in the same range (10s of genes) as the example. We discussed how maybe there were QC or demographic variables that could have been added to the model, but the data she has doesn‚Äôt have much to work with. We will revisit when I have maybe done more work with sn DE analysis.<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Talked about recount2, recount3, snaptron, snapcount, recount.bwtool, recount-brain, recount predictions, megadepth, the recount3 study explorer. I also sent them all the links to them, though you can find many of them at <a href=\"https://jhubiostatistics.shinyapps.io/recount/\">https://jhubiostatistics.shinyapps.io/recount/<\/a>, <a href=\"https://jhubiostatistics.shinyapps.io/recount-brain/\">https://jhubiostatistics.shinyapps.io/recount-brain/<\/a>, and <a href=\"http://rna.recount.bio/\">http://rna.recount.bio/<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I pointed Nick to a few scripts used in marker finding for the Tran Maynard dataset, using a few functions from DeconvoBuddies. For marker heat maps we went over how I hace built annotation tables, and a few tricks for optimizing pheatmaps for maker plots.<\/p>\n<\/div>","<div class='gt_from_md'><p>We converted a downloaded dataset from tsv files to a sce object. Went over a strategy for performing DE on single cell data, using OSCA as a reference<\/p>\n<\/div>","<div class='gt_from_md'><p>We spent the session getting Sanjana set up better to work on JHPCE, setting up ssh keys, cyberduck, RStudio, and config files. Plan to meet again to set up git + start single cell work<\/p>\n<\/div>","<div class='gt_from_md'><p>This had already been fixed, but Abby had an older version of spatialLIBD installed. We also went over some other steps needed to deploy an <code>spatialLIBD<\/code> app with pseudo-bulked data and modeling results.<\/p>\n<\/div>","<div class='gt_from_md'><p>Resulted in a fix for <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1137544893\">this issue<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>Assessed results of BayesSpace joint clustering and suggested that Alice discard the spots with low library sizes to improve clustering. Offered to create small SPE object for her with just the for samples she will be doing long read sequencing on.<\/p>\n<\/div>","<div class='gt_from_md'><p>Brainstorming and planning session.<\/p>\n<\/div>","<div class='gt_from_md'><p>Resolved via email.<\/p>\n<\/div>","<div class='gt_from_md'><p>created new script to subset spe object by removing some of the images.  updated app.R to load smaller spe object (half the size) and re-ran shiny app locally. Explained how to version control from local computer and fixed problem with version controlling large data file.  Still having same problem of not being able to view shiny app online<\/p>\n<\/div>","<div class='gt_from_md'><p>Review of statistical models as well as access to data<\/p>\n<\/div>","<div class='gt_from_md'><p>We both leaned how to use the image function to create a annotation bar for heatmaps based on a custom color pallet<\/p>\n<\/div>","<div class='gt_from_md'><p>Twitter, LinkedIn, jobRxiv<\/p>\n<\/div>","<div class='gt_from_md'><p>Tried to troubleshoot adding visium IF images to spe object<\/p>\n<\/div>","<div class='gt_from_md'><p>Helped setup OneDrive folder for LC manuscript. Explained how to organize figure folders.<\/p>\n<\/div>","<div class='gt_from_md'><p>Clarified what the data is showing and explained a bit the goals of the LIBD data portal that Geo Pertea is building.<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Helped fix error of shiny app not being able to find spe object. Helped sucessfully deploy app.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over how to create a new type of refrence (Supp Table/Supp Figure). Also walked though the full process of creating a label and then refrencing it in from the text + considreations for formatting<\/p>\n<\/div>","<div class='gt_from_md'><p>PhD programs and other options.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went thourgh some plotting code Sreya had adapted to make a manhattan plot, and went over which aestetics to change to get the desired plot<\/p>\n<\/div>","<div class='gt_from_md'><p>I described the two JHPCE-based repositories we push to (<a href=\"https://github.com/LieberInstitute/jhpce_mod_source\">https://github.com/LieberInstitute/jhpce_mod_source<\/a> and <a href=\"https://github.com/LieberInstitute/jhpce_module_config\">https://github.com/LieberInstitute/jhpce_module_config<\/a>), my usual installation approach for python-based GPU-utilizing software, and the contents of the lua-based config enabling a module to load.<\/p>\n<\/div>","<div class='gt_from_md'><p>See <a href=\"https://twitter.com/lahuuki/status/1521581339903873024\">this Twitter thread<\/a> for the result.<\/p>\n<\/div>","<div class='gt_from_md'><p>Provided guidance on how to share images and how to organize supplementary files. Resulting in this <a href=\"https://doi.org/10.6084/m9.figshare.19729567.v1\">Figshare<\/a> zip file.<\/p>\n<\/div>","<div class='gt_from_md'><p>I clarified the usage of nextflow channels, described when docker integration is likely to be worthwhile, and answered other general questions about processing data with a nextflow-based workflow<\/p>\n<\/div>","<div class='gt_from_md'><p>NA<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped come up with a stratagedy to loop over 100+ files for an analysis and exreact a portion of the results in to one R object<\/p>\n<\/div>","<div class='gt_from_md'><p>NA<\/p>\n<\/div>","<div class='gt_from_md'><p>Helped explain the different parts of the html output of spaceranger and which fields are important to check for QC<\/p>\n<\/div>","<div class='gt_from_md'><p>NA<\/p>\n<\/div>","<div class='gt_from_md'><p>NA<\/p>\n<\/div>","<div class='gt_from_md'><p>See <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012\">https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>Clarified a few questions about LIBD‚Äôs SPEAQeasy RSE objects and a problem with some colData() that was missing, that we‚Äôll provide early next week. Send links to our deconvolution work with Louise A Huuki-Myers.<\/p>\n<\/div>","<div class='gt_from_md'><p>If the situation had been simple, that is, that the latest commit was the one that commited the large files, we could have used the solution described at <a href=\"http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/\">http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/<\/a>. Instead, we had to use <a href=\"https://rtyley.github.io/bfg-repo-cleaner/\">https://rtyley.github.io/bfg-repo-cleaner/<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>I helped Heena address an error she was getting when trying to run read10XVisiumWrapper(). We noted an issue with the function and submitted an github issue to spatialLIBD. We got the function to work but only if you provide the path to the gtf file. Met for 1hr.<\/p>\n<\/div>","<div class='gt_from_md'><p>JHPCE ssh keys (to avoid typing passwords when using qrsh), .Rprofile (both), .bashrc, .zshrc (laptop), iTerm2 (laptop), Textmate (laptop), remote port fowading for rmate, xquartz (laptop) installation and configuration. There are several LIBD rstats club videos on these topics already, but we just did all the setup fast. In particular, the .bashrc configuration was important to resolve a umask setting problem that was interfering with the permissions on the Visium IF AD project.<\/p>\n<\/div>","<div class='gt_from_md'><p>We installed and configured Cross Refrence, then went over how to use the lable vs. refrence links + pro tips on how to avoid errors<\/p>\n<\/div>","<div class='gt_from_md'><p>spot size and scaling factor from json file. explained how each metric is associated to different version of the histology images.<\/p>\n<\/div>","<div class='gt_from_md'><p>Customized VistoSeg code to their histology images<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>See <a href=\"https://neuralplasticity.slack.com/archives/C010NFSQS48/p1649249130880349\">this Slack thread<\/a>. Ultimately, this situation involved teaching Tony to share his code.<\/p>\n<\/div>","<div class='gt_from_md'><p>We want over some plotting basics with ggplot2, and how to use geom_rect to recreate some spatial plots that a halo module produces in R.<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/VistoSeg/issues/11\">https://github.com/LieberInstitute/VistoSeg/issues/11<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>Able to identify situation where duplicate.correlation should be used and how block data is important to linear modeling. We then looked at how top table can be used to summarize DE outputs.<\/p>\n<\/div>","<div class='gt_from_md'><p>modified code to work witth her dataset. did graph based clustering and plottiing. Explained benefits of clustering across samples. Advised her to do isoform calling on pseudobulked data for multiple samples. Met for 1hr.<\/p>\n<\/div>","<div class='gt_from_md'><p>After identifying that ‚Äòredstone‚Äô was a precompiled binary for Linux, I described how to use ‚Äòwget‚Äô to download software like this, set execute permissions if necessary, and run at JHPCE.<\/p>\n<\/div>","<div class='gt_from_md'><p>Finished installing packages. Set up sample_info table. Walked her through the rest of the code.<\/p>\n<\/div>","<div class='gt_from_md'><p>I recommended some adjustments to a custom GTF and transcripts FASTA to solve an error seen in SPEAQeasy where transcripts and annotation were found to be incompatible.<\/p>\n<\/div>","<div class='gt_from_md'><p>We applied some test I used for the TREG package to help test more parts of the functions, like checking for errors w/ bad input, warning messages, and qualities of outputs. We discussed possible ways to use testthat on a ggplot, but that might need more thought.<\/p>\n<\/div>","<div class='gt_from_md'><p>Showed her app.R, deploy.R scripts and places where to edit, showed her how to navigate through the shinyapp portal to looks for logs when deploying the app. EXplained her few erros she ran through when deploying for the 1st time.<\/p>\n<\/div>","<div class='gt_from_md'><p>We set up a git repo and added code to be adapted to build an spe object from Alice‚Äôs long read data.<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained how the first time that you ssh into a new computer (which happens also with git pull/git push), you need to accept that the new host is authentic, which will then edit your ~/.ssh/known_hosts file.<\/p>\n<\/div>","<div class='gt_from_md'><p>This was a preliminary meeting with Alice to decide where to begin with analyzing her spatial data. We decided to start by generating an spe object with her long read data.<\/p>\n<\/div>","<div class='gt_from_md'><p>An error occurred in a SPEAQeasy run, which I identified to be due to not providing all the required inputs (specifically the sites at which to perform variant calling)<\/p>\n<\/div>","<div class='gt_from_md'><p>This is challenging since it involves putting together code from multiple scripts/projects. We summarized the steps to do <a href=\"https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1069493294\">on this GitHub issue thread<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>We both verified additional transcript annotation added to a GENCODE GTF looked good for use as input to SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>Louise and I discussed the <a href=\"https://doi.org/10.1038/s41593-020-0604-z\">Dentate Gyrus, Jaffe at al, 2020<\/a> paper which used LCM-seq, what deconvolution using Bisque can do, and what role it can play at different analytical stages.<\/p>\n<\/div>","<div class='gt_from_md'><p>TODO<\/p>\n<\/div>","<div class='gt_from_md'><p>An error occurred in the counts step, which I discovered was a bug in some filename-matching code<\/p>\n<\/div>","<div class='gt_from_md'><p>We had some trouble logging in because the WiFi wasn‚Äôt working without logging in. We solved this by using an ethernet cable at LIBD (which is part of the JHU network). Once we logged in once, then we could use the WiFi.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described two-factor authentication vs. passwordless login via SSH keys, and gave tips to set up the latter.<\/p>\n<\/div>","<div class='gt_from_md'><p>The contrast adjusted images on MAC::preview should be saved and uploaded back to JHPCE to run spaceranger. Spaceranger needs the exact same image that was used in the loupe alignment (even if it is changed from raw image on preview)<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how to add additional transcripts to the relevant FASTA file then use SPEAQeasy‚Äôs custom annotation option to use the new FASTA with the other default annotation files.<\/p>\n<\/div>","<div class='gt_from_md'><p>Explained how the image data can be manipulated when claudin5 channel and WFA channel are merged and then the blood vessels are segmented and masked. The correct way is to segment blood vessels from claudin channel and mask the segmented objects in WFA channel.<\/p>\n<\/div>","<div class='gt_from_md'><p>The indices listed in the tiles‚Äôs filename is the x,y coordinates of the starting topleft pixel of the given tile. These indices never start with [0,0] as these tiles are picked from the ROI in the phenochart and not the entire imaged area. Once stored in a directory these tiles are not arranged in chronological order.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how requesting ‚Äòcaracol‚Äô is similar to requesting ‚Äòbluejay‚Äô, but elaborated on additional complications, like contention for the same GPU and using CUDA_VISIBLE_DEVICES properly, including with array jobs<\/p>\n<\/div>","<div class='gt_from_md'><p>I described different strateguies for helping others navigate the world of data science. In particular, how weekly coding sessions can be really useful.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over the files (data and script) for the deconvolution analysis done for the habenula in BSPIII. We also discussed the theroy/methodology for deconvolution.<\/p>\n<\/div>","<div class='gt_from_md'><p>Created an R script that reads in the transcripts FASTA file from GENCODE and uses BSgenome from Bioconductor to edit it and save it as a larger .fasta.gz file. Nick Eagles helped us by pointing us to <a href=\"http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation\">http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>Linked to <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13<\/a>, <code>/dcs04/lieber/lcolladotor/qSVA_LIBD3080/degradation_experiments/*/bed<\/code>, <a href=\"https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855\">https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855<\/a>, <a href=\"https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh\">https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5<\/a>, <a href=\"https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R\">https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R<\/a>, <a href=\"https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R\">https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R<\/a>, <a href=\"https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253\">https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253<\/a>, <a href=\"https://github.com/CRG-Barcelona/bwtool/wiki#installation\">https://github.com/CRG-Barcelona/bwtool/wiki#installation<\/a>, <a href=\"http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html\">http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html<\/a>, and <a href=\"http://research.libd.org/qsvaR/\">http://research.libd.org/qsvaR/<\/a>. Andrew Jaffe linked <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/degradation_mat_CMC_ribozero.rda<\/code>, <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/run_de_analysis_CMC.R<\/code>, <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/geneLevel_CMC_qSVsAndMod.rda<\/code> and <code>/users/ajaffe/Lieber/Projects/RNAseq/CMC/<\/code>.<\/p>\n<\/div>","<div class='gt_from_md'><p>Linked to <a href=\"https://github.com/LieberInstitute/qsvaR\">https://github.com/LieberInstitute/qsvaR<\/a>, <a href=\"http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0\">http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R<\/a>, <a href=\"https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28\">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28<\/a>. Made public the file <code>/dcl01/ajaffe/data/lab/qsva_brain/brainseq_phase2_qsv/rdas/degradation_rse_phase2_usingJoint_justFirst.rda<\/code> and other ones. See <a href=\"https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5\">https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5<\/a> for details.<\/p>\n<\/div>","<div class='gt_from_md'><p>Heena was encountering a simple error installing the package. We also spent time going through the vignette to give a conceptual overview of qsva. Her feedback will be helpful in making the vignette more accessible<\/p>\n<\/div>","<div class='gt_from_md'><p>I referred her to the ‚Äò‚Äìcoverage‚Äô option with SPEAQeasy, which unfortunately cannot be run alone, which might be useful for an already-processed dataset<\/p>\n<\/div>","<div class='gt_from_md'><p>Used KJ‚Äôs code from his rstats club presentation and described how the images shoukd be read using numpy array as uint32 but not floating <a href=\"https://colab.research.google.com/drive/1A4cavWcbgZF1PFQT87s5s_ATyZrN3IxL?usp=sharing\">https://colab.research.google.com/drive/1A4cavWcbgZF1PFQT87s5s_ATyZrN3IxL?usp=sharing<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>Using BiocMAP <a href=\"http://research.libd.org/BiocMAP/\">http://research.libd.org/BiocMAP/<\/a> which Nick is finishing up, you could process the data and have R objects you could load and analyze. So, the data processing would be ‚Äúeasy‚Äù in a sense. Next, you might want to check the bsseq documentation <a href=\"https://bioconductor.org/packages/bsseq/\">https://bioconductor.org/packages/bsseq/<\/a> if you are interested in finding differentially methylated regions (DMRs) and in any case, because BiocMAP provides that type of R object as noted at <a href=\"http://research.libd.org/BiocMAP/outputs.html\">http://research.libd.org/BiocMAP/outputs.html<\/a>. <a href=\"https://jhubiostatistics.shinyapps.io/wgbsExprs/\">https://jhubiostatistics.shinyapps.io/wgbsExprs/<\/a> and the companion paper is the last WGBS data analysis project I did myself with code at <a href=\"https://github.com/LieberInstitute/brain-epigenomics\">https://github.com/LieberInstitute/brain-epigenomics<\/a>. Kira Perzel-Mandell, who worked with Shizhong, wrote <a href=\"https://github.com/LieberInstitute/wgbs_meqtl_analysis\">https://github.com/LieberInstitute/wgbs_meqtl_analysis<\/a> (link to paper there) and did other projects. If you are plotting methylation levels at some locations, then that‚Äôs very direct. If you are trying to find DMRs, it can take a bit of time to get used to the large objects and the related complications (once you filter the data, it‚Äôll be more manageable). If you go into CpH instead of CpG, then memory constraints become harder to deal with.<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed how to build and save data with a for loop and with purrr::map. Went over how to define a function in map to do nested loops<\/p>\n<\/div>","<div class='gt_from_md'><p>Geo was interested int eh workflow I developed for tensorQTL. I went over the requirements and input formats that tensorqtl requirers, and how I had previously made them from our dataset. Also discussed the differnces between the functions and their outputs.<\/p>\n<\/div>","<div class='gt_from_md'><p>used HALO to show how contrast can be adjusted when changing min max values in the image<\/p>\n<\/div>","<div class='gt_from_md'><p>NA<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Related to JHPCE <a href=\"https://github.com/ntdyjack/fasthplus\">https://github.com/ntdyjack/fasthplus<\/a> and <a href=\"http://research.libd.org/sgejobs/\">http://research.libd.org/sgejobs/<\/a><\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Brainstormed several ideas related to deconvolution of bulk data as well as uses of snRNA-seq. Refered to OSCA <a href=\"https://bioconductor.org/books/release/OSCA/\">https://bioconductor.org/books/release/OSCA/<\/a> at time.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over how to acess the rowData, and what the rowData inthe MDD project contains. Ended up adding columns to the eqtl risk snp subsets to help with analysis<\/p>\n<\/div>","NA","<div class='gt_from_md'><p><a href=\"https://www.nature.com/articles/s41592-021-01358-2\">https://www.nature.com/articles/s41592-021-01358-2<\/a><\/p>\n<\/div>","NA","NA","NA","<div class='gt_from_md'><p>We tried to discover the source of a seg fault that would occur when she imported tensorqtl in python with the tensorqtl module. Another user and I weren‚Äôt able to replicate the issue. We later discovered that there were lines in her .bashrc that might‚Äôve interfered with the module to produce the issue.<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how to qrsh/sub jobs with the GPU node, including how to automatically set up ‚ÄòCUDA_VISIBLE_DEVICES‚Äô to avoid contention for resources, and how to use the tensorqtl module to ensure GPUs were used when importing pytorch<\/p>\n<\/div>","<div class='gt_from_md'><p>After ensuring our apparent Rstudio set-ups on our windows machines matched, she still wasn‚Äôt able to forward X11 while using Rstudio. Regarding a different issue, we discovered that a particular machine required additional lines in the ‚Äò~/.ssh/config‚Äô to properly enable RSA keys.<\/p>\n<\/div>","<div class='gt_from_md'><p>explained the functions of this pipline as these are not yet documented<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how it‚Äôs accessed as a queue, but some manual work is left to the user (checking which GPUs are open and setting ‚ÄòCUDA_VISIBLE_DEVICES‚Äô)<\/p>\n<\/div>","<div class='gt_from_md'><p>command to use <code>rsync -avzh --progress userid@jhpce-transfer01:/source directory/ /destination directory/<\/code><\/p>\n<\/div>","<div class='gt_from_md'><p>use rsync, prepend spaces in the paths with <code>\\<\/code> to avoid issues<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/43106c664883fd5df301e44cca35f2e8f2f289ad\">https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/43106c664883fd5df301e44cca35f2e8f2f289ad<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>explain how to use IDE‚Äôs on local machine to edits scripts on JHPCE through cyberduck<\/p>\n<\/div>","<div class='gt_from_md'><p>Fixed row names to both be ensembl IDs, discussed simplifing cell types in sce, converting sce arrarys to matrix<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/CaPTure/issues/2\">https://github.com/LieberInstitute/CaPTure/issues/2<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>I shared some detials of our deconvolution approach, we walked through the SCE to Expression set converson, and discussed some issues that may come up<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how I normally deduce sample IDs from FASTQ filenames when creating a ‚Äòsamples.manifest‚Äô file for SPEAQeasy, described the difference between ‚Äò‚Äìtrim_mode‚Äô settings that SPEAQeasy supports, and showed how to check what samples were trimmed.<\/p>\n<\/div>","<div class='gt_from_md'><p>refining segmentations in matlab by applying size thresholds<\/p>\n<\/div>","<div class='gt_from_md'><p>Showed code how to scale data based on <a href=\"https://uc-r.github.io/kmeans_clustering\">https://uc-r.github.io/kmeans_clustering<\/a> and optimise kmeans for the present dataset<\/p>\n<\/div>","<div class='gt_from_md'><p>Motivated /dcs04/lieber/lcolladotor/_jhpce_org_LIBD001/update_permissions_spatialteam.sh<\/p>\n<\/div>","<div class='gt_from_md'><p>Showed the steps on one sample HPC image and what outputs to save.<\/p>\n<\/div>","<div class='gt_from_md'><p>While debugging, its seen that VistoSeg when splitting the raw image missed a part of fiducial frame for capture area D1 of that sample. This happens when the center of the visium slide does not align with the center of the image, which has ti be adjusted while imaging. This image when run through spaceranger provided negative coordinates for the missing fiducial frame part of the image, but MATLAB will not accept negative indices for a image, which was throwing errors when recontructing spot grind in countSpots function.<\/p>\n<\/div>","<div class='gt_from_md'><p>Showed how to close issues on git with detailed comments of the final results<\/p>\n<\/div>","<div class='gt_from_md'><p>I described using regular expressions in R to match files in a directory. We encountered a weird case where ‚Äòlist.files‚Äô seemed to return different numbers of matched files when run repeatedly. After ensuring we were in the right directory, the issue went away, but it was unclear what caused the original problem.<\/p>\n<\/div>","<div class='gt_from_md'><p>made custom Matlab script to batch process her 30 RNAscope images on her local machine<\/p>\n<\/div>","<div class='gt_from_md'><p>The user directories were cleared during the software updates for LIBD server, emailed bill to create one for me and Heena to use rsync on Neural_Plasiticity. The filenames Heena used for ‚Äòsource‚Äô in rync did not exist at the source directory which was throwing errors.<\/p>\n<\/div>","<div class='gt_from_md'><p>to pick filename from textfile for a array job<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed how to check for duplicate rownames, and stratedgies for variable naming + working with rse objects<\/p>\n<\/div>","<div class='gt_from_md'><p>We found a R function that helped produce the plot she was looking for<\/p>\n<\/div>","<div class='gt_from_md'><p>Two main issues were discussed: <strong>1<\/strong>. After batch correction there is one sample that clusters mostly independently from the other samples. This sample did not stand out in QC (althogh this will be re-examined). There is some evidence that this donor might have been clinacally different from the other donors, and since the batch correction worked well for the other samples, there is the possibility that this is real biologal signal. However since there is only one such sample it would be really hard to confirm this, and the inclusion of this sample would counfound the biologcal question of interest. I advisied to drop this sample post-clustering on the grounds that this donor might be too differnt from the others. <strong>2<\/strong>. Prelimnary results from DEG by cell type for the three groups find mitocondral genes are the most differntly expressed. They are concerned as MT RNA could be a technical artifact (contamination) this tech isn‚Äôt the best to explore this. But there is are other findings that cellular metabolism may play a role in this condition. I advisied to keep the DEG results with the MT, add the caveat in the paper that this might not be a perfect experiment for this but it could support real differnces in the cells (plus they already have pretty strict MT QC). Also run a second DEA w/o MT genes to see what other signal exists.<\/p>\n<\/div>","<div class='gt_from_md'><p>The error came from the names of the clusters just being integers, the names for limma need to be sytactically valid (not just numbers, no spaces, no special characters - basically need to be variable names in R). I advised to simply add a prefix like we did in spatialDLPFC (sp9k1) or even simpler like ‚Äúc‚Äù to her cluster names. Maybe we should add a warning to the function about cluster names since this error is pretty hard for users to de-bug and people often have non-sytactucally valid cluster names (especally with cell type names)<\/p>\n<\/div>","<div class='gt_from_md'><p>Explained some scripts such as /dcs04/lieber/lcolladotor/BrainSEQ_LIBD001/_jhpce_org<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Explaination of why an index label M=3 was chosen for refineVNS in section 2.2.<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained how I get out of the situation where files that are too big get accidentally committed (using ‚Äògit log‚Äô, ‚Äògit reset ‚Äìsoft‚Äô, and other details)<\/p>\n<\/div>","<div class='gt_from_md'><p>We worked through an error message from the ‚ÄòVariantsMerge‚Äô process, uncovering (arguably) a bug in SPEAQeasy, and I advised how to temporarily circumvent the problem. I also suggested a command for renaming a set of files that initially broke SPEAQeasy‚Äôs assumptions.<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>I talked about how Linux does not allow creation of files w/ execution permissions set, regardless of ‚Äòumask‚Äô settings such as in the ‚Äò.bashrc‚Äô. I also explained how I use branching with git and other tips for managing a repo that others also develop on.<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/dotdotdot/issues/14\">https://github.com/LieberInstitute/dotdotdot/issues/14<\/a><\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p>We downloaded a metadata table from SRA and parsed it through R to programatically create a ‚Äúsamples.manifest‚Äù file for use in SPEAQeasy, that merged samples that were each split into multiple FASTQ files.<\/p>\n<\/div>","NA","NA","NA","<div class='gt_from_md'><p><a href=\"https://files.slack.com/files-pri/T029RHR07-F024D7ELKC2/image_from_ios.jpg\">https://files.slack.com/files-pri/T029RHR07-F024D7ELKC2/image_from_ios.jpg<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>used back quote instead of single quote when assigning string to variables<\/p>\n<\/div>","<div class='gt_from_md'><p>We decided to run CellProfiler on a while capture area image of DLPFC sample to look at the segmentations as it was doing good job on the cropped regions of a DLPFC sample<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/dotdotdot/issues/13\">https://github.com/LieberInstitute/dotdotdot/issues/13<\/a><\/p>\n<\/div>","NA","<div class='gt_from_md'><p>SPEAQeasy would appear to run into ‚Äúfile not found‚Äù errors for just a handful of random samples in a large dataset. I eventually traced the error to a disk mounting issue on a particular compute node, which was resolved through bitsupport.<\/p>\n<\/div>","<div class='gt_from_md'><p>The H&amp;E image that he was segmenting and the H&amp;E image for which he gets the ‚Äòjson‚Äô and ‚Äòcsv‚Äô are not the same images or at least not the same resolution which is causing the error.<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/dotdotdot/issues/12\">https://github.com/LieberInstitute/dotdotdot/issues/12<\/a><\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p>used the command ‚Äòeval‚Äô<\/p>\n<\/div>","<div class='gt_from_md'><p>Did some troubleshooting on nested map functions, when to use ~.x and when to use function() syntax<\/p>\n<\/div>","<div class='gt_from_md'><p>I mentioned that she encountered a bug that was fixed in a more recent version of SPEAQeasy, but suggested she manually add a line of code temporarily so that she could resume the run before updating<\/p>\n<\/div>","<div class='gt_from_md'><p>I referred her to a script I had written in the past that contains the required commands and configuration  (with comments) for using the SRA toolkit<\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p>Went over basic, complex, and nested map() usage and how to apply it to Josh‚Äôs current work<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>We were able to update the keys for the cluster, but the ssh key login was still broken<\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p>We revised key features of Brenda Pardo‚Äôs Twitter thread <a href=\"https://twitter.com/PardoBree/status/1388253938391175173?s=20\">https://twitter.com/PardoBree/status/1388253938391175173?s=20<\/a> and some features that you would want your Twitter thread to have.<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed a fairly standard method for concatenating tabular data from multiple files into a single matrix, and saving the resulting R object<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about all the details that go into writing an abstract for a conference. These details can take longer to complete than initially anticipated.<\/p>\n<\/div>","<div class='gt_from_md'><p>We revised how to make a new branch in Git working with <a href=\"https://github.com/lmweber/OSTA-book\">https://github.com/lmweber/OSTA-book<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the current DNA genotype pipeline and potential changes others might want.<\/p>\n<\/div>","<div class='gt_from_md'><p>ibidem<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/dotdotdot/issues/11\">https://github.com/LieberInstitute/dotdotdot/issues/11<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>From SPEAQeasy, to qSVA, deconvolution, TWAS, etc. It might be best to give a talk about all these topics to whoever is interested at LIBD since we know about all these pieces but others might not.<\/p>\n<\/div>","<div class='gt_from_md'><p>showed a example of how AWK extracts text lines from a text file<\/p>\n<\/div>","<div class='gt_from_md'><p>Explain how some scripts in SUFI run and the bash scripts to use in place of python scripts for batch processing<\/p>\n<\/div>","<div class='gt_from_md'><p>Referred to Vijay‚Äôs notes<\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p>I described correct use of the ‚Äú‚Äìinput‚Äù argument, and we also found that many of the input FASTQs were empty, leading to a somewhat vague error message.<\/p>\n<\/div>","<div class='gt_from_md'><p>In a series of email exchanges we diagnosed subtle errors resulting from bad FASTQ files and unintended options in the main script. Her feedback helped me improve some existing documentation.<\/p>\n<\/div>","<div class='gt_from_md'><p>I attempted some formatting tricks to eliminate unwanted whitespace but was ultimately unsuccessful. We suspected the LaTeX template required the whitespace to exist. We also tried to fix a bug with the header of a bookdown RMD not rendering as expected. It seemed that re-writing the header (without making any visible changes) fixed the issue, though we weren‚Äôt sure of the source of the initial problem.<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about <a href=\"https://lcolladotor.github.io/bioc_team_ds/\">https://lcolladotor.github.io/bioc_team_ds/<\/a> among other topics.<\/p>\n<\/div>","<div class='gt_from_md'><p>used ggplot<\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p>We confirmed that an existing VCF was corrupt and found that tabix expected bgzip-compressed VCF input rather than compressed BCF input.<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>This has changed over time. The most recent setup (as of 2021-08-06) is at <a href=\"https://github.com/lmweber/OSTA-book\">https://github.com/lmweber/OSTA-book<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>I briefly mentioned the ‚Äú#$ -o‚Äù and ‚Äú#$ -e‚Äù arguments, and how several logs will generate in your home directory without those arguments and without ‚Äú#$ -cwd‚Äù, when ‚Äúqsub‚Äùing a job at JHPCE.<\/p>\n<\/div>","<div class='gt_from_md'><p>We reviewed her code and I gave some pointer on what bugs to look out for with certain errors in ggplot<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about the set of videos and resources publicly available to get to learn about R, Bioconductor and these packages. It was a lot of information and I‚Äôm sure that we‚Äôll have more sessions later on to clarify questions as they arise.<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about <a href=\"https://choosealicense.com/\">https://choosealicense.com/<\/a>, <a href=\"https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository\">https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository<\/a>, <a href=\"https://creativecommons.org/choose/\">https://creativecommons.org/choose/<\/a>.<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the posibility of proposing a class that has a mix of biology lectures and some practical coding sessions. The coding sessions could be based after the CDSB 2021 mini courses series.<\/p>\n<\/div>","<div class='gt_from_md'><p>used Sys.glob and write.table to write the list to txt file<\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p>We talked about the public data from Synapse and worked on locating some of it at JHPCE. I explained also how we have used bamtofastq when others don‚Äôt share their FASTQ files. Basically, we use bamtofastq to reconstruct the original FASTQ files (you might have to append the unmapped FASTQs if they are provided by the original authors) and then run FASTQ-processing tools like our SPEAQeasy pipeline.<\/p>\n<\/div>","<div class='gt_from_md'><p>cleared up some confusion about filter_snps.R<\/p>\n<\/div>","<div class='gt_from_md'><p>We met to figure out how to transfer the data for this collaboration, which ended up being quite tricky and involved using an SSH tunnel for rsync. Basically, a command like this one: <code>rsync -avzh -e 'ssh -o ProxyJump=user@server_somewhere_for_the_tunnel' *_files_of_interest user@final_server:/final_path/<\/code><\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the different possible goals and ways to get to them and determined a path forward that relies on the rest of the DSgs system as well as existing LIBD rstats club videos to get started. It‚Äôll be supported by a meeting with Leo and others involved in the spatial transcriptomics projects from the R side.<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed the initial version of the BrainSeq Phase II developmental expression explorer developed by Bill Ulrich and Aaron Goldman. We discussed other possibilities for the collaboration including helping set up SPEAQeasy in their servers.<\/p>\n<\/div>","<div class='gt_from_md'><p>Pointed her to some oacjages for creating TWAS plots whe was interested in and we walked through some code I used to compare T-stats from different DE models<\/p>\n<\/div>","<div class='gt_from_md'><p>we talked about how to use the ‚Äú‚Äìstrand‚Äù option, the source of the error that was encountered<\/p>\n<\/div>","<div class='gt_from_md'><p>we discussed what a bfile is in plink and that made the process much easier.<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/LieberInstitute/zandiHyde_bipolar_rnaseq/tree/master/dev_twas#bd-twas-workflow\">https://github.com/LieberInstitute/zandiHyde_bipolar_rnaseq/tree/master/dev_twas#bd-twas-workflow<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>still working on this will touch base again, we are trying to obtain the mds plot for ancestry in the placental data.<\/p>\n<\/div>","<div class='gt_from_md'><p>the way we had run cluster profiler was generating lists in a way that the plot functions could not work with<\/p>\n<\/div>","<div class='gt_from_md'><p>N/A<\/p>\n<\/div>","<div class='gt_from_md'><p>We walked tought script from getRPKM to expression cut to filter rse objects, I also pointed Brenda towards some reasources to get started with ggplot<\/p>\n<\/div>","<div class='gt_from_md'><p>Used GenomeStudio again to produce PLINK files<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed the changes that we should make and the ones we shouldn‚Äôt based on the feedback received from the practice talk with Keri‚Äôs group. The feedback was excellent and very useful, however, we discussed which suggestions were maybe complicating the presentation too much and how to best incorporate them.<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>run clusterprofiler on a set of DE hits<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Similar to part 2. The presentation was looking very polished by this meeting!<\/p>\n<\/div>","<div class='gt_from_md'><p>Helped review the grant application and revise the text.<\/p>\n<\/div>","<div class='gt_from_md'><p>Further updates on asking for help with this complicated project. Suggested reaching out to other LIBD members who have tried running the software before.<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussed with Josh what figures we needed to prepare and how to generate them. This also involved base_size = 30 with ggplot2. Otherwise the font was too small to read it on the grant.<\/p>\n<\/div>","<div class='gt_from_md'><p>Now that a first major draft of the presentation was complete, we revised it and proposed some changes to it. Some of them involved remaking the figures using base_size = 20 or even 30 with ggplot2. For future work, it might be best to save the images with that font size from the beginning instead of having to update them later.<\/p>\n<\/div>","<div class='gt_from_md'><p>We reviewed the model that Sreya was evaluating for DE. As we as troublshooting the plot functions from the SPEAQeasy bootcamp code. I also advised taht she could create a volcano plot for her results.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went through the creation of an LIBD JHPCE module, covering the documentation process, lua syntax, and how to test a module before pushing.<\/p>\n<\/div>","<div class='gt_from_md'><p>We adapted a series of bash scripts requiring parallel functionality for use at JHPCE. This involved correctly setting some path variables and requesting multiple cores through SGE for some scripts.<\/p>\n<\/div>","<div class='gt_from_md'><p>explained the process of using topmed imputation and later we will get the mds<\/p>\n<\/div>","<div class='gt_from_md'><p>I described how I typically construct a ‚Äúsamples.manifest‚Äù file and configure SPEAQeasy for a particular experiment.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over some plots we have made for DE previously, (quality control &amp; PC scatter plots, t-stat scatter between models, venn diagrams, and boxplots). Talked about some things to watch out for in her data (small samples size). We are planning to meet again when her data is ready.<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over the details of the GitHub actions workflow created by biocthis, the parts that you need to run from biocthis::use_bioc_pkg_templates(), and resolving issues prompted by R CMD check + BiocCheck<\/p>\n<\/div>","<div class='gt_from_md'><p>I gave an overview of the resources and materials we have created as well as the LIBD bootcamps that we have publicly available to help others get started with computational biology. This is likely the first of several meetings.<\/p>\n<\/div>","<div class='gt_from_md'><p>We spent a significant amount troubleshooting the problem here (nearly two hours?). In the end, using the insert image ‚Äúhere‚Äù LaTeX syntax resolved the problem.<\/p>\n<\/div>","<div class='gt_from_md'><p>General advice on how to ask for help from open source projects<\/p>\n<\/div>","<div class='gt_from_md'><p>Initial planning session for the LIBD seminar. We talked about the different slides we planned to show and the flow of the presentation. We also scheduled 3 practice talks with different groups of people.<\/p>\n<\/div>","<div class='gt_from_md'><p>We observed an issue where log parsing in SPEAQeasy behaved differently for her and I due to a difference in default python version loaded at JHPCE. I fixed the hisat2/2.2.1 module to ensure consistent python version use.<\/p>\n<\/div>","<div class='gt_from_md'><p>Overview of the TWAS pipeline<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed how I typically produce an array job shell script for the situation where I have many samples of data to process the same way (SIMD). We talked about using a text file where each line contains one sample name, and referencing that in the array script.<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed Sreya how to set up the S-TWAS directory that I adapted in the Zandi-Hyde bipolar project for her own adaptation. I also sent her some useful videos on how to set up GitHub with her JHPCE account and install an SFTP client for Ubuntu (I prefer Filezilla)<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed how to handle strandness issues, and how different inferred strand patterns within a group of samples sequenced together can indicate problems with the input data. I described how to use the ‚Äò‚Äìforce_strand‚Äô option with SPEAQeasy<\/p>\n<\/div>","<div class='gt_from_md'><p>We went through cloning SPEAQeasy, creating a ‚Äúsamples.manifest‚Äù file with R, and other relevant details for running SPEAQeasy at JHPCE. I also helped him set up git at JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>We troubleshooted strange errors from the main SPEAQeasy output, and saw that I was able to run the main script (‚Äúrun_pipeline_jhpce.sh‚Äù) but she encountered errors with an identical script. We suspected permissions issues in her working directory, which she emailed bitsupport about.<\/p>\n<\/div>","<div class='gt_from_md'><p>I gave Sreya a conceptual introduction to transcriptome imputation and sent her some resources to help her get started on FUSION / TWAS<\/p>\n<\/div>","<div class='gt_from_md'><p>We got ssh key login working, added config files to JHPCE, went though a ‚Äúhello world‚Äù example of using files on JHPCE on RStudio<\/p>\n<\/div>","<div class='gt_from_md'><p>Although we encountered permissions issues, I showed how to create a ‚Äúsamples.manifest‚Äù file and configure SPEAQeasy for a particular set of samples.<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Henry needed to reset his JHPCE password as he had forgotten it, but we went through RStudio set up and got cyberduck installed, we will set up a meeting next week for ssh keys and configuration<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed my typical process for creating a ‚Äúsamples.manifest‚Äù file for a given dataset, and how to submit a pipeline run. We continued to debug some minor issues over slack<\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p>Created a learning plan involving several team members that is now documented at a channel in JHU Genomics Slack. That way everyone can keep track of what others are teaching the person who asked the question, and we can have a common communication channel among everyone involved. Sessions include learning how to interact with JHPCE through RStudio, how to run SPEAQeasy, key R/Bioconductor packages (limma, SummarizedExperiment), and explore possible visualizations of interest.<\/p>\n<\/div>","<div class='gt_from_md'><p>I covered various nextflow concepts, as well as my typical approach in creating and running a nextflow project at JHPCE<\/p>\n<\/div>","<div class='gt_from_md'><p>explaination of nurolucida system, how number of cores and RAM matter for running SUFI<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Discussion about features that spatialLIBD could support to make it easier to use for newer data.<\/p>\n<\/div>","<div class='gt_from_md'><p>Went over basic functions of SummarizedExperiment and SingleCellExperiment, what a Sparse matrix is - how to convert to non-sparse. Steps to take to use getRPKM on SingeCellExperient.<\/p>\n<\/div>","<div class='gt_from_md'><p>sent him the scripts from andrew, wondering if I should make it into a function<\/p>\n<\/div>","<div class='gt_from_md'><p>Help with understanding <a href=\"https://bookdown.org/yihui/bookdown/\">bookdown<\/a> and ideas on how to organize a documentation website (chapters and information to consider).<\/p>\n<\/div>","<div class='gt_from_md'><p>Discussed the research project proposal and how we can help.<\/p>\n<\/div>","<div class='gt_from_md'><p>General advice. Received feedback regarding file backups.<\/p>\n<\/div>","<div class='gt_from_md'><p>explaination of scale_factors.json, tissuepositions_list.csv from spaceranger and the spotspotcheck package<\/p>\n<\/div>","<div class='gt_from_md'><p>I identified that <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829\">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829<\/a> contains both miRNA and RNA-seq data. Using data from the SRA Run selector at <a href=\"https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&amp;o=acc_s%3Aa\">https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&amp;o=acc_s%3Aa<\/a> I was able to filter the data to the RNA-seq samples and subset the input samples.manifest file for SPEAQeasy.<\/p>\n<\/div>","<div class='gt_from_md'><p>To identify DEGs or eQTLs, the user has to choose what covariates to adjust for. Some studies might require more complicated models than others, as they might be from more than one brain region or present some batch effect. In addition, in newer projects we are considering adjusting for cell type fractions derived from deconvolution. Even if all the covariates were fixed across datasets, in DEGs we have typically adjusted for qSVs which are computed in a dataset-dependent fashion and in essence are PCs derived from degradation-associated genomic regions. Similarly, for eQTLs we adjust for expression PCs. In both we adjust for ancestry PCs. Computing PCs is input-dependent: you won‚Äôt get the same values or PCs if you change which samples you are using. So, computing DEGs and eQTLs is not as simple as say a map-reduce problem where you compute the same thing for every input sample, then aggregate the per-sample results. We have steps that depend on the whole sample set. Then there‚Äôs also steps for filtering expression features based on some minimum expression cutoffs. Plus, differences between say polyA and RiboZero data, which don‚Äôt make sense to process together. For these reasons, I don‚Äôt think that an on-demand computation pipeline is feasible based on a user selecting a set of input samples. Constructing RNA-seq objects (RangedSummarizedExperiment objects) as well as DNA genotyping VCF files based on a set of input samples is feasible and that‚Äôs what others are building. After that, I think that it‚Äôs best to adapt code we have (or even use newer tools) after carefully exploring the data.<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>I described the need for the ‚Äúmodule use‚Äù command and modifying the .bashrc to easily use LIBD modules<\/p>\n<\/div>","<div class='gt_from_md'><p>We walked througth the SpatialLIBD code to find where the argument has to be changed<\/p>\n<\/div>","<div class='gt_from_md'><p>I fetched the SRA files, created a manifest and execution script for SPEAQeasy, but had an issue running ‚Äúfastq-dump‚Äù. I decided to hand off the request, partially complete.<\/p>\n<\/div>","<div class='gt_from_md'><p>Failed MacOS installation on Lieber MacBook, something about Java both being and also not being installed<\/p>\n<\/div>","<div class='gt_from_md'><p>explaination of the functions VNS.m and refineVNS.m<\/p>\n<\/div>","<div class='gt_from_md'><p>when running VNS.m for several files on same matlab session the data from previous sessions is saved in matlab cache memory automatically. This will cause memory issues with subsequent runs, its important to clear variables after every run.<\/p>\n<\/div>","<div class='gt_from_md'><p>reffered to package ‚Äúeffsize‚Äù should be efficient at easily doing this<\/p>\n<\/div>","<div class='gt_from_md'><p>ran her through an example, explained her the code on github repo (split.m is the main script)<\/p>\n<\/div>","<div class='gt_from_md'><p>edited the config file for setting up x-quartz<\/p>\n<\/div>","<div class='gt_from_md'><p>Adapted code and shared it at <a href=\"https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock\">https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>He walked me through some code performing qSVA for a recent project, and I offered some tips involving code organization/ readibility.<\/p>\n<\/div>","<div class='gt_from_md'><p>Created GitHub issues at <a href=\"https://github.com/LieberInstitute/VisiumImgProcessing/issues\">https://github.com/LieberInstitute/VisiumImgProcessing/issues<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>fixed the github repo<\/p>\n<\/div>","<div class='gt_from_md'><p>I showed them the tools I developed in DeconvoBuddies package, and provided links to example code where I had applied these funcitons in previous work<\/p>\n<\/div>","<div class='gt_from_md'><p>explore loading/saving/exporting options on Qupath<\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed how Sciwheel can be adapted for different journals, and provided general advice for getting started and formatting a paper for a particular journal.<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained that SPEAQeasy was used for the recent hg38 processing, and an older version of a similar pipeline was used for the original hg19 processing. We discussed our new method for retrieving gene symbols (‚Äúorg.*.eg.db‚Äù packages) and some general code advice.<\/p>\n<\/div>","<div class='gt_from_md'><p>I went through the setup and example usage of Sciwheel and cross reference, and gave general advice (it‚Äôs often easiest to start with the ‚ÄúMethods‚Äù section, etc)<\/p>\n<\/div>","<div class='gt_from_md'><p>use rsync<\/p>\n<\/div>","<div class='gt_from_md'><p>Debugging through the input/output file in logs<\/p>\n<\/div>","<div class='gt_from_md'><p>I explained how to use SPEAQeasy at JHPCE, and we tried an experimental method for downloading SRA data. The method didn‚Äôt work, and we plan to meet again after I verify a more typical method works.<\/p>\n<\/div>","<div class='gt_from_md'><p>I hadn‚Äôt encountered the issue with new local terminals failing to open. I planned to follow up with better specific advice via email, and offer to explain this advice in a future DSGS.<\/p>\n<\/div>","<div class='gt_from_md'><p>running chmod<\/p>\n<\/div>","<div class='gt_from_md'><p>Used Amy‚Äôs rstats blogpost on git<\/p>\n<\/div>","<div class='gt_from_md'><p>We spent time doing basic Rstudio setup and troubleshooting, and got the data downloaded<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>We got Heena‚Äôs MAC registered for passwordless login to JHPCE<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>We got the ssh keys working for Brenda‚Äôs github<\/p>\n<\/div>","<div class='gt_from_md'><p>We went over applying DeconvoBuddies functions to hebenula data<\/p>\n<\/div>","<div class='gt_from_md'><p>Use of galaxybrain for identifying samples with DNAm, DNA genotype and RNA-seq data<\/p>\n<\/div>","<div class='gt_from_md'><p>We are going to try an iterative strategy to keep tables smaller<\/p>\n<\/div>","NA","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/p134069/#p134141\">BioC Support website answer<\/a><\/p>\n<\/div>","NA","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/p133591/#p133927\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>Used ‚Äòh5ls‚Äô command from shell to verify contents of file; eventually found we were looking at the wrong file (error message was cryptic about issue). <a href=\"https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1607034309074800\">https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1607034309074800<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>Need to resolve swap bugs. Plan to create a table of phenotype data (Dx, age, sex, race) for each BrNum to mach genotype data<\/p>\n<\/div>","<div class='gt_from_md'><p>Went through the setting up passwordless login for jhpce using ssh keys, and coppied the configuration files from <a href=\"https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee\">https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://community.rstudio.com/t/recipes-package-cannot-create-interaction-term-in-step-interact/89860\">RStudio Community self-answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>got through the Github vignettes thoroughly<\/p>\n<\/div>","NA","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/p133110/#p133319\">BioC Support website answer<\/a><\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Started PsychENCODE access request process<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.illumina.com/downloads/genomestudio-2-0-plug-ins.html\">There is a plugin for Genome Studio by Illumina that allows users to output their imputed allele results as PLINK inputs<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/134569/#p133028\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>to use rsync or scp, you need to have user account created on LIBD server (Bill can help create it)<\/p>\n<\/div>","NA","<div class='gt_from_md'><p>We debugged some code and ultimetly decided that it might be best for Abby to write new code and use the tsne function to cluster and plot the data<\/p>\n<\/div>","<div class='gt_from_md'><p>I recomended using dcast to do a ‚Äúlong‚Äù to ‚Äúwide‚Äù data transformation<\/p>\n<\/div>","<div class='gt_from_md'><p>We talked about how the data should be structured before using ggplot (transforming some data and adding catagorical columns)<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/p132558/#p132872\">I recommended that they reach out to their forum board<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1605123296024100\">Check Slack helpdesk channel<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/134139/#p132829\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/lcolladotor/biocthis/issues/15\">GitHub issue at biocthis<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>We discussed organizing by analysis (a directory for each one, then prehaps sub- directory for data &amp; plots). Create a readMe for each sub-folder. Use gitignore to keep repo from getting too big.<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/131600/#p132617\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://www.mathworks.com/matlabcentral/answers/636365-how-can-i-measure-volume-from-a-3d-image?s_tid=srchtitle\">https://www.mathworks.com/matlabcentral/answers/636365-how-can-i-measure-volume-from-a-3d-image?s_tid=srchtitle<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>Resulted in recount3 versions 1.1.1 and 1.1.2 on bioc-devel (and 1.0.2 on bioc-release)<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/lcolladotor/biocthis/issues/13\">GitHub issue at biocthis<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://github.com/lcolladotor/biocthis/issues/14\">GitHub issue at biocthis<\/a><\/p>\n<\/div>","NA","<div class='gt_from_md'><p>Explained the <code>chgrp<\/code> and <code>umask<\/code> commands plus <code>~/.bashrc<\/code> file configuration. I also mentioned why we don‚Äôt use <code>chmod 777<\/code> at JHPCE. Details on <a href=\"https://jhu-genomics.slack.com/archives/CR9NYA0BF/p1604086263148400?thread_ts=1603826219.094600&amp;cid=CR9NYA0BF\">Slack<\/a> (private channel).<\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/p132487/#p132521\">BioC Support website answer<\/a><\/p>\n<\/div>","NA","NA","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/p132406/#p132459\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/p132450/#p132464\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>initial issue seemed to do with the spaceranger command arguments; we are continuining to investigate possible module issues or insufficient code in the <a href=\"http://research.libd.org/rstatsclub/2020/10/20/using-space-ranger-at-jhpce/\">blog post<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>Needed to use highlight_key and highlight functions to select points between plots. Used rstats example as guide <a href=\"https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14\">https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14<\/a><\/p>\n<\/div>","NA","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/134532/#134925\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/134914/#134915\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/131285/#134921\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p><a href=\"https://support.bioconductor.org/p/134876/#134917\">BioC Support website answer<\/a><\/p>\n<\/div>","<div class='gt_from_md'><p>spatstat package <a href=\"https://spatstat.org/releasenotes/spatstat-1.39-0.html\">https://spatstat.org/releasenotes/spatstat-1.39-0.html<\/a>, L function <a href=\"https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest\">https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest<\/a>, Kmark <a href=\"https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark\">https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark<\/a><\/p>\n<\/div>"],"html":true,"width":680,"align":"left","headerStyle":{"font-weight":"normal"}}],"resizable":true,"filterable":true,"searchable":true,"defaultPageSize":10,"showPageSizeOptions":false,"pageSizeOptions":[10,25,50,100],"paginationType":"numbers","showPagination":true,"showPageInfo":true,"minRows":1,"showSortable":true,"width":"1200px","height":"auto","theme":{"color":"#333333","backgroundColor":"#FFFFFF","stripedColor":"rgba(128,128,128,0.05)","style":{"fontFamily":"system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif"},"headerStyle":{"borderTopStyle":"solid","borderTopWidth":"2px","borderTopColor":"#D3D3D3","borderBottomStyle":"solid","borderBottomWidth":"2px","borderBottomColor":"#D3D3D3"}},"elementId":"vzyonjxzbt","dataKey":"99a6763f177e731173d74adf9d8f0842"},"children":[]},"class":"reactR_markup"},"evals":["tag.attribs.columns.0.style","tag.attribs.columns.1.style","tag.attribs.columns.2.style","tag.attribs.columns.3.style"],"jsHooks":[]}</script>
</div>
</div>
<div class="container">
    <footer class="site-footer">

      <center>
<p class="powered-by">
¬© 2011-2023. All thoughts and opinions here are my own. The icon was designed by <a href="https://www.linkedin.com/in/mauricio-guzman-6529b551/">Mauricio Guzm√°n</a> and is inspired by <a href="https://en.wikipedia.org/wiki/Huichol">Huichol culture</a>; it represents my community building interests.
</p>

<p class="powered-by copyright-license-text">
This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">CC BY NC SA 4.0</a>
</p>

<p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">
      <img src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" alt="CC icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/by.svg" alt="CC by icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/nc.svg" alt="CC NC icon" width="32px">
      <img src="https://mirrors.creativecommons.org/presskit/icons/sa.svg" alt="CC SA icon" width="32px">
    </a>
</p>

<div id="clustrmaps-widget"></div><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=300&t=m&d=Zo-deincaJGsqhqjIWsujtdoNOMJsnnZLefKmxDKVv8&co=124b73&cmo=dd53ff&cmn=4ec724'></script>

<p class="powered-by">
Published with <a href="https://bookdown.org/" target="_blank" rel="noopener">Bookdown</a>
</p>

</center>



<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-3CBVR6GQ07"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-3CBVR6GQ07');
</script>

<!-- Go to www.addthis.com/dashboard to customize your tools -->
<script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4ef23e545d41b87c"></script>

<script type="text/javascript">
var addthis_config = {
	data_ga_property: 'G-3CBVR6GQ07',
	data_ga_social : true,
	data_track_addressbar: true
};
var addthis_share = {
	// ... other options
	url_transforms : {
		shorten: {
			twitter: 'bitly'
		}
	},
	shorteners : {
		bitly : {}
	}
}
</script>

</footer>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="r-session-information.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
