<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Leo | Data Science guidance session logs</title>
  <meta name="description" content="6 Leo | Data Science guidance session logs" />
  <meta name="generator" content="bookdown 0.35 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Leo | Data Science guidance session logs" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Leo | Data Science guidance session logs" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="josh.html"/>
<link rel="next" href="louise.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.2/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#dsgs-guide-team-members"><i class="fa fa-check"></i>DSgs-guide team members</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#graphs"><i class="fa fa-check"></i>Graphs</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#r-session-information"><i class="fa fa-check"></i>R session information</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="abby.html"><a href="abby.html"><i class="fa fa-check"></i><b>1</b> Abby</a></li>
<li class="chapter" data-level="2" data-path="arta.html"><a href="arta.html"><i class="fa fa-check"></i><b>2</b> Arta</a></li>
<li class="chapter" data-level="3" data-path="cynthia.html"><a href="cynthia.html"><i class="fa fa-check"></i><b>3</b> Cynthia</a></li>
<li class="chapter" data-level="4" data-path="hedia.html"><a href="hedia.html"><i class="fa fa-check"></i><b>4</b> Hedia</a></li>
<li class="chapter" data-level="5" data-path="josh.html"><a href="josh.html"><i class="fa fa-check"></i><b>5</b> Josh</a></li>
<li class="chapter" data-level="6" data-path="leo.html"><a href="leo.html"><i class="fa fa-check"></i><b>6</b> Leo</a></li>
<li class="chapter" data-level="7" data-path="louise.html"><a href="louise.html"><i class="fa fa-check"></i><b>7</b> Louise</a></li>
<li class="chapter" data-level="8" data-path="maddy.html"><a href="maddy.html"><i class="fa fa-check"></i><b>8</b> Maddy</a></li>
<li class="chapter" data-level="9" data-path="nick.html"><a href="nick.html"><i class="fa fa-check"></i><b>9</b> Nick</a></li>
<li class="chapter" data-level="10" data-path="ryan.html"><a href="ryan.html"><i class="fa fa-check"></i><b>10</b> Ryan</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data Science guidance session logs</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="leo" class="section level1 hasAnchor" number="6">
<h1><span class="header-section-number">6</span> Leo<a href="leo.html#leo" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<table>
<colgroup>
<col width="19%" />
<col width="57%" />
<col width="23%" />
</colgroup>
<thead>
<tr class="header">
<th>Date</th>
<th>Help request</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2023-09-22</td>
<td>snRNA-seq training</td>
<td>Resolved some questions related to <a href="https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq" class="uri">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq</a> and made two pull requests to fix typos.</td>
</tr>
<tr class="even">
<td>2023-09-22</td>
<td>How to interpret terms on an interaction model</td>
<td>We went over the math derivations for interpreting beta1, beta2, and beta3 on an interaction model where X1 is a continuous variable, X2 is a discrete variable with 2 options and we model <code>Y ~ X1 * X2</code>. We also used <code>ExploreModelMatrix</code>.</td>
</tr>
<tr class="odd">
<td>2023-09-21</td>
<td>Guidance on expression heatmaps</td>
<td>I explained the benefits of centering and scaling and referred to both the short video on this topic <a href="https://youtu.be/c8ffeFVUzk8" class="uri">https://youtu.be/c8ffeFVUzk8</a> as well as the material we developed for the CSHL 2023 course <a href="https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html" class="uri">https://lcolladotor.github.io/cshl_rstats_genome_scale_2023/expression-heatmaps-with-complexheatmap.html</a>.</td>
</tr>
<tr class="even">
<td>2023-09-14</td>
<td>snRNA-seq training</td>
<td>Continued reviewing material from <a href="https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq" class="uri">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq</a></td>
</tr>
<tr class="odd">
<td>2023-09-13</td>
<td>Social media ideas</td>
<td>Discussed a few ideas about LIBD and social media</td>
</tr>
<tr class="even">
<td>2023-08-21</td>
<td>snRNA-seq training</td>
<td>Answered questions related to material at <a href="https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq" class="uri">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq</a></td>
</tr>
<tr class="odd">
<td>2023-08-18</td>
<td>ranger_metrics</td>
<td>We talked about the first scripts in <a href="https://github.com/LieberInstitute/ranger_metrics" class="uri">https://github.com/LieberInstitute/ranger_metrics</a></td>
</tr>
<tr class="even">
<td>2023-08-14</td>
<td>Getting started with snRNA-seq</td>
<td>We talked about material at <a href="https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq" class="uri">https://github.com/ComunidadBioInfo/cdsb2021_scRNAseq</a></td>
</tr>
<tr class="odd">
<td>2023-08-10</td>
<td>Introduction to what we do as a team and future plans</td>
<td>We talked about how I became an Investigator at LIBD, the overall type of work we do as a team, some of our main collaborations, and overall plans for applying for funding in the future. In particular, I highlighted how I want to train team members on how to write an NIH biosketch, though not all are going to be “key personnel”. I sent a link to the DSgs, team meeting schedule, rstats club schedule, and our <a href="http://lcolladotor.github.io/talk/teamtldr2022/" class="uri">http://lcolladotor.github.io/talk/teamtldr2022/</a> summary from April 20, 2022.</td>
</tr>
<tr class="even">
<td>2023-07-27</td>
<td>Guidance on reference data for deconvolution and snRNA-seq analysis</td>
<td>For deconvolution, we talked about finding an appropriate reference dataset and we talked about an expert we can contact to ask which dataset(s) might be the most relevant. For snRNA-seq data analysis, I mentioned OSCA (Orchestrating Single Cell Analysis with Bioconductor) and how Louise could help guide this person on our code based off OSCA through a series of DSgs sessions.</td>
</tr>
<tr class="odd">
<td>2023-07-27</td>
<td>Feedback on an R package + GO/KEGG enrichment analyses</td>
<td>We transferred <a href="https://github.com/LieberInstitute/qsvaRPkgDemo" class="uri">https://github.com/LieberInstitute/qsvaRPkgDemo</a> to the LieberInstitute account. I asked to include some citations for sources (like the image source) and some other small edits. For GO / KEGG enrichment with clusterProfiler I asked to not filter results (pvalueCutoff / qvalueCutoff parameters), so we can see more results prior to choosing which ones to plot.</td>
</tr>
<tr class="even">
<td>2023-07-26</td>
<td>Evaluate a potential external contractor</td>
<td>We discussed the current status of the team, the project, what skills are required, boundaries in terms of time and who would be involved (and who wouldn’t). We ended the session with me relaying a list of resources to check and read more carefully. Then in about 2 weeks we’ll meet again to determine the next steps.</td>
</tr>
<tr class="odd">
<td>2023-07-26</td>
<td>How to best document what skills I can perform or need to learn</td>
<td>We made a Google sheet with columns such as <code>study_type</code>, <code>skill_name</code>, <code>learning_priority</code>, etc. I also worked on it after the DSgs session to help complete the current information. See <a href="https://neuralplasticity.slack.com/archives/C04Q8MX99FY/p1690397027111449">Slack</a> for more details.</td>
</tr>
<tr class="even">
<td>2023-07-26</td>
<td>Need help debugging iSEE app errors/warnings</td>
<td>We resolved this issue by making sure that the variable names for which they wanted to specify the correct colors were actually part of the colData(SE). See <a href="https://jhu-genomics.slack.com/archives/C01KSLPCGAC/p1689858182290369">Slack</a> for the actual code links. I also asked them to make sure to use soft links / full paths instead of using setwd() for files outside the project + to keep all related files at JHPCE instead of on a laptop, to make it easier to rule out any sources of differences (aka for reproducibility purposes).</td>
</tr>
<tr class="odd">
<td>2023-07-20</td>
<td>Input on a course design</td>
<td>We discussed what would be the appropriate material for us to teach in a course. We will re-use some of our materials from <a href="https://github.com/lcolladotor/cshl_rstats_genome_scale_2023" class="uri">https://github.com/lcolladotor/cshl_rstats_genome_scale_2023</a>.</td>
</tr>
<tr class="even">
<td>2023-07-20</td>
<td>Feedback on responding to reviewers</td>
<td>We went over the proposed response to reviewers, clarified some sentences, and requested co-authors to approve the response to the reviewers.</td>
</tr>
<tr class="odd">
<td>2023-06-29</td>
<td>How to make a project documentation website</td>
<td>We adapted code from <a href="https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R" class="uri">https://github.com/LieberInstitute/spatialDLPFC/blob/main/code/update_website.R</a> and <a href="https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd" class="uri">https://github.com/LieberInstitute/spatialDLPFC/blob/main/README.Rmd</a> in order to make the files at <a href="https://github.com/LieberInstitute/septum_lateral" class="uri">https://github.com/LieberInstitute/septum_lateral</a>. See <a href="https://github.com/LieberInstitute/septum_lateral/commits/main" class="uri">https://github.com/LieberInstitute/septum_lateral/commits/main</a> from <a href="https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50" class="uri">https://github.com/LieberInstitute/septum_lateral/commit/dda5822c0f74c1d143e317c211650a042c274f50</a> to <a href="https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c" class="uri">https://github.com/LieberInstitute/septum_lateral/commit/230568af106ac9c75cd0045e76b762cdcc31cf4c</a> as well as related commits on the <code>gh-pages</code> branch.</td>
</tr>
<tr class="even">
<td>2023-06-28</td>
<td>clusterProfiler, set.seed, and BiocParallel questions</td>
<td>Summarized at <a href="https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659" class="uri">https://jhu-genomics.slack.com/archives/C018ZDB8QLU/p1687982732826659</a>. This involved explaining why we don’t use as a background universe all genes in the annotation in gene ontology / KEGG pathway enrichment analyses. Instead, we use those that pass the expression filters and for which we computed differential expression statistics.</td>
</tr>
<tr class="odd">
<td>2023-06-28</td>
<td>Resolving a permissions issue</td>
<td>We resolved <a href="https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199" class="uri">https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687803353217199</a> at <a href="https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&amp;cid=C02RG5VGL6R" class="uri">https://jhu-genomics.slack.com/archives/C02RG5VGL6R/p1687984850054149?thread_ts=1687803353.217199&amp;cid=C02RG5VGL6R</a>, which was due to a user incorrectly having a Linux group assigned to files at JHPCE when they are not part of the user group. Uploads through Cyberduck can trigger this. The solution is to run <a href="https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example" class="uri">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example</a> again on a qrsh session.</td>
</tr>
<tr class="even">
<td>2023-06-27</td>
<td>How to make dynamic error messages</td>
<td>We worked on qsvaR debugging an issue raised by Renee and Louise. We then improved the error messages on qsvaR as noted at <a href="https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400" class="uri">https://github.com/LieberInstitute/qsvaR/commit/feb9a9e4f8a499baba76271de37ca39a9969f400</a>.</td>
</tr>
<tr class="odd">
<td>2023-06-23</td>
<td>How to make a new Globus endpoint for a publication.</td>
<td>Using the septum lateral data, I trained Ryan on how to make this type of Globus endpoint, how to set up the right permissions, how to edit and update <a href="http://research.libd.org/globus/" class="uri">http://research.libd.org/globus/</a> (see <a href="https://github.com/LieberInstitute/globus/blob/b10e0fb89c40489d23028e8ed3f7812cdea76a6b/index.Rmd#L57">this</a> for details), and how to document the Globus information on a manuscript.</td>
</tr>
<tr class="even">
<td>2023-06-22</td>
<td>How to filter an SCE object prior to making an iSEE app</td>
<td>Using the lateral septum snRNA-seq data, I trained Heena on how to filter a SingleCellExperiment object to reduce the required memory from 7GB to less than 1GB in order to be able to deploy an iSEE app at shinyapps.io, whose computers have a maximum of 8 GB of RAM. See <a href="https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98" class="uri">https://github.com/LieberInstitute/septum_lateral/commit/e6c5d0099e128b909dcf467d46b4f7f172820f98</a> and related commits.</td>
</tr>
<tr class="odd">
<td>2023-06-21</td>
<td>How to debug an R package function from a user’s report</td>
<td>We worked with the script where Renee had noticed issues with qsvaR. We adapted it to work on qsvaR local updates through devtools::load_all(). We also resolved <a href="https://github.com/LieberInstitute/qsvaR/issues/29" class="uri">https://github.com/LieberInstitute/qsvaR/issues/29</a> and <a href="https://github.com/LieberInstitute/qsvaR/issues/28" class="uri">https://github.com/LieberInstitute/qsvaR/issues/28</a>.</td>
</tr>
<tr class="even">
<td>2023-06-08</td>
<td>Options for cell type specific eQTLs</td>
<td>Discussed how to best pseudobulk the data prior to doing this type of analysis. In particular, how to adapt <a href="https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148" class="uri">https://github.com/LieberInstitute/spatialLIBD/blob/edc8b72505af097895dcbf35887df28da8122e3c/R/registration_pseudobulk.R#L135-L148</a> and related code, as well as plots to make such as boxplots of the min_ncells by cell type (without any filtering) and cutoffs found by <code>isOutlier()</code> (different cutoff for each cell type) + PCA plots colored by min_ncells.</td>
</tr>
<tr class="odd">
<td>2023-06-08</td>
<td>Overview of what we do as a team and outreach options</td>
<td>Discussed a few different areas of growth, including a change in direction related to our outreach activities.</td>
</tr>
<tr class="even">
<td>2023-03-13</td>
<td>Updating spatialLIBD::read10xVisiumAnalysis</td>
<td>See <a href="https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099" class="uri">https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678716129409099</a> for more. Led to <a href="https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6" class="uri">https://github.com/LieberInstitute/spatialLIBD/commit/97b852686515a271f4db3e185a9a9180a09a5db6</a> and related commits. spatialLIBD version 1.11.10 now supports spaceranger version 2023.0208.0 output files.</td>
</tr>
<tr class="odd">
<td>2023-03-13</td>
<td>Verify GitHub ssh key setup and team overview</td>
<td>We used <a href="https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection" class="uri">https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection</a> and also went over the JHPCE and R config files from the team website. We also talked about the DSgs and how it would be beneficial for Kinnary to meet 1 on 1 with all DSgs guides to build familiarity and make it easier to ask for help in the future.</td>
</tr>
<tr class="even">
<td>2023-03-10</td>
<td>Debugging spatialLIBD::read10xVisiumAnalysis</td>
<td>See <a href="https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179" class="uri">https://jhu-genomics.slack.com/archives/C02TV4F1MCM/p1678466684480179</a> for more details. Heena is working with spaceranger output files from 2 different versions, and in particular, version 2023.0208.0 (internal 10x Genomics version) changed their output files structure</td>
</tr>
<tr class="odd">
<td>2023-03-09</td>
<td>Overview on “R/Bioconductor-powered Team Data Science”</td>
<td>Described what we do, the DSgs system, potential for visitors at our team meeting, and research ideas.</td>
</tr>
<tr class="even">
<td>2023-03-02</td>
<td>Help resolve JHPCE permission issues on the lifespan_hpc_spatial project</td>
<td>The solution involved asking BitSupport to login as a specific user and run <a href="https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example" class="uri">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#spatialteam-example</a>. Knowing the exact file paths that triggered the permission issues are important to help check with <code>ls -lh</code> and <code>nfs4_getfacl</code> under a <code>qrsh</code> session.</td>
</tr>
<tr class="odd">
<td>2023-03-02</td>
<td>Debug spaceranger 2023.0208.0 (likely an alpha for 2.0.1) with SpatialExperiment 1.8.0</td>
<td>See the <a href="https://community-bioc.slack.com/archives/CR1BLV821/p1677783580326559">Bioconductor Slack message</a> I wrote for more details.</td>
</tr>
<tr class="even">
<td>2023-03-02</td>
<td>How to write a tweetorial</td>
<td>End result: <a href="https://twitter.com/HeenaDivecha/status/1631360771342098433" class="uri">https://twitter.com/HeenaDivecha/status/1631360771342098433</a></td>
</tr>
<tr class="odd">
<td>2023-03-02</td>
<td>read10xVisium() in version 1.7.1 (and now 1.8.0) creates a dangerous silent error by mismatching tissue positions across multiple samples</td>
<td>See <a href="https://github.com/drighelli/SpatialExperiment/issues/135" class="uri">https://github.com/drighelli/SpatialExperiment/issues/135</a> and the pull request at <a href="https://github.com/drighelli/SpatialExperiment/pull/136" class="uri">https://github.com/drighelli/SpatialExperiment/pull/136</a> that resolves this issue.</td>
</tr>
<tr class="even">
<td>2023-02-28</td>
<td>How to write a tweetorial</td>
<td>No public result just yet.</td>
</tr>
<tr class="odd">
<td>2023-01-11</td>
<td>Getting started with snRNA-seq</td>
<td>See <a href="https://jhu-genomics.slack.com/archives/C0498763CUC/p1673451228034439">Slack notes</a> for further details. They involved <a href="https://github.com/LieberInstitute/template_project" class="uri">https://github.com/LieberInstitute/template_project</a>, <a href="https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h" class="uri">https://drive.google.com/drive/folders/1cn5d-Ey7-kkMiex8-74qxvxtCQT6o72h</a>, and other specific instructions for setting up a project.</td>
</tr>
<tr class="even">
<td>2022-11-01</td>
<td>Introduction to LIBD and our team</td>
<td><a href="https://lcolladotor.github.io/bioc_team_ds" class="uri">https://lcolladotor.github.io/bioc_team_ds</a> for my team website, <a href="https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing" class="uri">https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing</a> for the LIBD rstats club schedule, <a href="https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials" class="uri">https://lcolladotor.github.io/bioc_team_ds/rbioconductor-data-science-bootcamps.html#bootcamp-source-materials</a> could be of interest also, <a href="https://github.com/LieberInstitute/template_project" class="uri">https://github.com/LieberInstitute/template_project</a> for a template for organizing projects, learn about the DSgs <a href="https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs" class="uri">https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Y2GXx-zMJqs</a>, <a href="https://rstudio.github.io/reticulate/" class="uri">https://rstudio.github.io/reticulate/</a> for using Python from R</td>
</tr>
<tr class="odd">
<td>2022-09-29</td>
<td>Overview on using GitHub for Matlab projects</td>
<td>We talked about <a href="https://github.com/LieberInstitute/template_project" class="uri">https://github.com/LieberInstitute/template_project</a>, <a href="https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/" class="uri">https://hopstat.wordpress.com/2013/11/05/array-and-sequential-cluster-jobs/</a>, <a href="http://research.libd.org/sgejobs/reference/job_single.html" class="uri">http://research.libd.org/sgejobs/reference/job_single.html</a>, <a href="https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7" class="uri">https://github.com/LieberInstitute/template_project/blob/3987e7f307611b2bcf657d1aa6930e76c4cc2b9a/code/02_explore_data/01_ggpairs.sh#L7</a>, <a href="https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1" class="uri">https://lcolladotor.github.io/bioc_team_ds/organizing-your-work.html#.YzWxq-zMKX1</a>, <a href="https://happygitwithr.com/" class="uri">https://happygitwithr.com/</a>, <a href="https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent" class="uri">https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent</a>.</td>
</tr>
<tr class="even">
<td>2022-09-23</td>
<td>Introduction to DSgs, R, Bioconductor, JHPCE</td>
<td>LIBD rstats club schedule <a href="https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing" class="uri">https://docs.google.com/spreadsheets/d/1is8dZSd0FZ9Qi1Zvq1uRhm-P1McnJRd_zxdAfCRoMfA/edit?usp=sharing</a>. Bioconductor overview video <a href="https://youtu.be/c0Ch_sXiGDQ" class="uri">https://youtu.be/c0Ch_sXiGDQ</a> and related notes in case you want to skim them <a href="https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing" class="uri">https://docs.google.com/document/d/1VZ4dOesHjbWhrvSIXO8Ne1Qk4ZdGEhAySNviZOHPqjI/edit?usp=sharing</a> (we typically have this for each video entry). R and RStudio intro <a href="https://youtu.be/XTuJ8vGnzBU" class="uri">https://youtu.be/XTuJ8vGnzBU</a>. Example vignette <a href="https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html" class="uri">https://bioconductor.org/packages/release/data/experiment/vignettes/spatialLIBD/inst/doc/spatialLIBD.html</a> from <a href="https://bioconductor.org/packages/spatialLIBD/" class="uri">https://bioconductor.org/packages/spatialLIBD/</a>. Data Science guidance sessions <a href="https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0" class="uri">https://lcolladotor.github.io/bioc_team_ds/data-science-guidance-sessions.html#.Yy3s3ezMKX0</a>. JHPCE new user request <a href="https://jhpce.jhu.edu/register/user/" class="uri">https://jhpce.jhu.edu/register/user/</a>. Making your personal website with postcards <a href="https://youtu.be/9LUCZAx3SlU" class="uri">https://youtu.be/9LUCZAx3SlU</a>. Examples include <a href="https://lahuuki.github.io/" class="uri">https://lahuuki.github.io/</a> by Louise, <a href="https://jtleek.com/" class="uri">https://jtleek.com/</a>, <a href="http://aejaffe.com/" class="uri">http://aejaffe.com/</a>.</td>
</tr>
<tr class="odd">
<td>2022-09-06</td>
<td>How to organize OneDrive files</td>
<td>Related to how we organize files for a publication.</td>
</tr>
<tr class="even">
<td>2022-08-24</td>
<td>How can I interpret the y-axis from <code>plotExplanatoryVariables()</code>?</td>
<td>Density is related to the probability density function, see <a href="https://en.wikipedia.org/wiki/Kernel_density_estimation%20for">wiki</a> more details. <a href="https://stats.stackexchange.com/questions/4220/can-a-probability-distribution-value-exceeding-1-be-ok">This link</a> is also useful for understanding the y-axis in more detail. Violin plots are just density plots rotated 90 degrees and mirrored. They are compared against boxplots frequently, which while easier to understand, are not good for non-unimodal data. See <a href="https://mode.com/blog/violin-plot-examples/#:~:text=What%20is%20a%20violin%20plot%3F,the%20density%20of%20each%20variable">this link</a> for more. I like <code>plotExplanatoryVariables()</code> because it’s fast to run and gives a general idea of things. See <a href="https://bioconductor.org/packages/release/bioc/vignettes/variancePartition/inst/doc/variancePartition.pdf">Figure 1b</a> from <code>variancePartition</code> for a violin plot of similar data. <code>variancePartition</code> can handle mixed effects models (random effects) and in that scenario is more precise, but will also take longer to compute.</td>
</tr>
<tr class="odd">
<td>2022-08-17</td>
<td>DE analysis introduction</td>
<td>Check <a href="https://bioconductor.org/packages/devel/bioc/vignettes/EnhancedVolcano/inst/doc/EnhancedVolcano.html">EnchancedVolcano</a> which shows a quick DE guide and finishes with some Volcano plots, which can be good enough for a grant application. If you want to use limma, you could follow <a href="http://research.libd.org/SPEAQeasy-example/de_analysis_speaqeasy.html#16_Modeling">the SPEAQeasy example</a> or <a href="http://research.libd.org/SPEAQeasy-example/bootcamp_intro#5_Day_3">our DE bootcamp</a> a bit. Or dive deeper into <a href="http://bioconductor.org/packages/release/workflows/vignettes/RNAseq123/inst/doc/limmaWorkflow.html">limma’s documentation</a>.</td>
</tr>
<tr class="even">
<td>2022-07-19</td>
<td>Help with spatialLIBD colors setup</td>
<td>Note that <code>spe$var_colors</code> works for <code>spe$var</code> where the <code>var</code> part has to be the same in both variable names. Also <code>spe$var_colors</code> should be a named vector such that <code>unique(names(spe$var_colors))</code> matches <code>unique(spe$var)</code>.</td>
</tr>
<tr class="odd">
<td>2022-07-15</td>
<td>R, RStudio, git, SSH key setup</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-07-12</td>
<td>LIBD Data Science</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2022-07-06</td>
<td>Data science coordination</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-06-03</td>
<td>Tools and resources related to recount2 and recount3</td>
<td>Talked about recount2, recount3, snaptron, snapcount, recount.bwtool, recount-brain, recount predictions, megadepth, the recount3 study explorer. I also sent them all the links to them, though you can find many of them at <a href="https://jhubiostatistics.shinyapps.io/recount/" class="uri">https://jhubiostatistics.shinyapps.io/recount/</a>, <a href="https://jhubiostatistics.shinyapps.io/recount-brain/" class="uri">https://jhubiostatistics.shinyapps.io/recount-brain/</a>, and <a href="http://rna.recount.bio/" class="uri">http://rna.recount.bio/</a>.</td>
</tr>
<tr class="odd">
<td>2022-05-27</td>
<td>Debug <code>spatialLIBD::sig_genes_extract_all()</code>.</td>
<td>This had already been fixed, but Abby had an older version of spatialLIBD installed. We also went over some other steps needed to deploy an <code>spatialLIBD</code> app with pseudo-bulked data and modeling results.</td>
</tr>
<tr class="even">
<td>2022-05-27</td>
<td>How to debug an R package code</td>
<td>Resulted in a fix for <a href="https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1137544893">this issue</a>.</td>
</tr>
<tr class="odd">
<td>2022-05-27</td>
<td>Data science organization.</td>
<td>Brainstorming and planning session.</td>
</tr>
<tr class="even">
<td>2022-05-25</td>
<td>MatLab questions.</td>
<td>Resolved via email.</td>
</tr>
<tr class="odd">
<td>2022-05-20</td>
<td>Brainomics meeting with collaborators</td>
<td>Review of statistical models as well as access to data</td>
</tr>
<tr class="even">
<td>2022-05-18</td>
<td>Discussion on how to recruit members</td>
<td>Twitter, LinkedIn, jobRxiv</td>
</tr>
<tr class="odd">
<td>2022-05-17</td>
<td>BrainSeq Phase II development browser</td>
<td>Clarified what the data is showing and explained a bit the goals of the LIBD data portal that Geo Pertea is building.</td>
</tr>
<tr class="even">
<td>2022-05-06</td>
<td>Career development</td>
<td>PhD programs and other options.</td>
</tr>
<tr class="odd">
<td>2022-05-03</td>
<td>How to make a tweetorial for a pre-print</td>
<td>See <a href="https://twitter.com/lahuuki/status/1521581339903873024">this Twitter thread</a> for the result.</td>
</tr>
<tr class="even">
<td>2022-04-29</td>
<td>File organization for a paper submission</td>
<td>Provided guidance on how to share images and how to organize supplementary files. Resulting in this <a href="https://doi.org/10.6084/m9.figshare.19729567.v1">Figshare</a> zip file.</td>
</tr>
<tr class="odd">
<td>2022-04-28</td>
<td>In promptu meeting</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-04-27</td>
<td>Grant proposal feedback</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2022-04-21</td>
<td>CV feedback</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-04-20</td>
<td>Cover letter feedback</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2022-04-15</td>
<td>Debugged code from <a href="https://github.com/LieberInstitute/spatialLIBD/issues/33" class="uri">https://github.com/LieberInstitute/spatialLIBD/issues/33</a></td>
<td>See <a href="https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012" class="uri">https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1100169012</a>.</td>
</tr>
<tr class="even">
<td>2022-04-15</td>
<td>Brainomics meeting with collaborators</td>
<td>Clarified a few questions about LIBD’s SPEAQeasy RSE objects and a problem with some colData() that was missing, that we’ll provide early next week. Send links to our deconvolution work with Louise A Huuki-Myers.</td>
</tr>
<tr class="odd">
<td>2022-04-08</td>
<td>How to delete large files from GitHub history</td>
<td>If the situation had been simple, that is, that the latest commit was the one that commited the large files, we could have used the solution described at <a href="http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/" class="uri">http://lcolladotor.github.io/2020/03/18/you-just-committed-a-large-file-and-can-t-push-to-github/</a>. Instead, we had to use <a href="https://rtyley.github.io/bfg-repo-cleaner/" class="uri">https://rtyley.github.io/bfg-repo-cleaner/</a>.</td>
</tr>
<tr class="even">
<td>2022-04-08</td>
<td>macOS and JHPCE laptop set</td>
<td>JHPCE ssh keys (to avoid typing passwords when using qrsh), .Rprofile (both), .bashrc, .zshrc (laptop), iTerm2 (laptop), Textmate (laptop), remote port fowading for rmate, xquartz (laptop) installation and configuration. There are several LIBD rstats club videos on these topics already, but we just did all the setup fast. In particular, the .bashrc configuration was important to resolve a umask setting problem that was interfering with the permissions on the Visium IF AD project.</td>
</tr>
<tr class="odd">
<td>2022-04-06</td>
<td>Debugging JHPCE configuration files</td>
<td>See <a href="https://neuralplasticity.slack.com/archives/C010NFSQS48/p1649249130880349">this Slack thread</a>. Ultimately, this situation involved teaching Tony to share his code.</td>
</tr>
<tr class="even">
<td>2022-03-18</td>
<td>The authenticity of host ‘github.com’</td>
<td>I explained how the first time that you ssh into a new computer (which happens also with git pull/git push), you need to accept that the new host is authentic, which will then edit your ~/.ssh/known_hosts file.</td>
</tr>
<tr class="odd">
<td>2022-03-16</td>
<td>Plan how to make spatial registration R functions</td>
<td>This is challenging since it involves putting together code from multiple scripts/projects. We summarized the steps to do <a href="https://github.com/LieberInstitute/spatialLIBD/issues/33#issuecomment-1069493294">on this GitHub issue thread</a>.</td>
</tr>
<tr class="even">
<td>2022-03-15</td>
<td>Clarifying what role deconvolution can play on LCM-seq</td>
<td>Louise and I discussed the <a href="https://doi.org/10.1038/s41593-020-0604-z">Dentate Gyrus, Jaffe at al, 2020</a> paper which used LCM-seq, what deconvolution using Bisque can do, and what role it can play at different analytical stages.</td>
</tr>
<tr class="odd">
<td>2022-03-11</td>
<td>How to log in to JHU-controlled laptop</td>
<td>We had some trouble logging in because the WiFi wasn’t working without logging in. We solved this by using an ethernet cable at LIBD (which is part of the JHU network). Once we logged in once, then we could use the WiFi.</td>
</tr>
<tr class="even">
<td>2022-03-04</td>
<td>How to guide others learning DS</td>
<td>I described different strateguies for helping others navigate the world of data science. In particular, how weekly coding sessions can be really useful.</td>
</tr>
<tr class="odd">
<td>2022-03-04</td>
<td>“Megadepth usage to add a new annotation to hg38 file”</td>
<td>Created an R script that reads in the transcripts FASTA file from GENCODE and uses BSgenome from Bioconductor to edit it and save it as a larger .fasta.gz file. Nick Eagles helped us by pointing us to <a href="http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation" class="uri">http://research.libd.org/SPEAQeasy/annotation.html#custom-annotation</a>.</td>
</tr>
<tr class="even">
<td>2022-03-03</td>
<td>How to build a cov_rse object using qSVA v1</td>
<td>Linked to <a href="https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13" class="uri">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L13</a>, <code>/dcs04/lieber/lcolladotor/qSVA_LIBD3080/degradation_experiments/*/bed</code>, <a href="https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855" class="uri">https://github.com/LieberInstitute/SPEAQeasy/blob/master/main.nf#L1774-L1855</a>, <a href="https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh" class="uri">https://github.com/LieberInstitute/RNAseq-pipeline/blob/master/sh/step5-coverage.sh</a>, <a href="https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50" class="uri">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L3-L50</a>, <a href="https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5" class="uri">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R#L5</a>, <a href="https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R" class="uri">https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_bwtool.R</a>, <a href="https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R" class="uri">https://github.com/LieberInstitute/megadepth/blob/master/R/read_coverage.R</a>, <a href="https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253" class="uri">https://github.com/LieberInstitute/recount.bwtool/blob/master/R/coverage_matrix_bwtool.R#L227-L253</a>, <a href="https://github.com/CRG-Barcelona/bwtool/wiki#installation" class="uri">https://github.com/CRG-Barcelona/bwtool/wiki#installation</a>, <a href="http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html" class="uri">http://lcolladotor.github.io/derfinder/reference/getRegionCoverage.html</a>, and <a href="http://research.libd.org/qsvaR/" class="uri">http://research.libd.org/qsvaR/</a>. Andrew Jaffe linked <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/degradation_mat_CMC_ribozero.rda</code>, <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/run_de_analysis_CMC.R</code>, <code>/users/ajaffe/Lieber/Projects/RNAseq/SzControl_DE_paper/rdas/geneLevel_CMC_qSVsAndMod.rda</code> and <code>/users/ajaffe/Lieber/Projects/RNAseq/CMC/</code>.</td>
</tr>
<tr class="odd">
<td>2022-03-03</td>
<td>Debradation matrix for LIBD riboZ</td>
<td>Linked to <a href="https://github.com/LieberInstitute/qsvaR" class="uri">https://github.com/LieberInstitute/qsvaR</a>, <a href="http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0" class="uri">http://research.libd.org/rstatsclub/2018/12/11/quality-surrogate-variable-analysis/#.YiFawRDMKX0</a>, <a href="https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R" class="uri">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/quantify_top1000.R</a>, <a href="https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R" class="uri">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R</a>, <a href="https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28" class="uri">https://github.com/LieberInstitute/qsva_brain/blob/master/brainseq_phase2_qsv/make_qSVs.R#L28</a>. Made public the file <code>/dcl01/ajaffe/data/lab/qsva_brain/brainseq_phase2_qsv/rdas/degradation_rse_phase2_usingJoint_justFirst.rda</code> and other ones. See <a href="https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5" class="uri">https://github.com/LieberInstitute/qsva_brain/commit/cba8fa5e886be9242dcbaae7854d1eed0f69cef5</a> for details.</td>
</tr>
<tr class="even">
<td>2022-02-24</td>
<td>WGBS data analysis overview</td>
<td>Using BiocMAP <a href="http://research.libd.org/BiocMAP/" class="uri">http://research.libd.org/BiocMAP/</a> which Nick is finishing up, you could process the data and have R objects you could load and analyze. So, the data processing would be “easy” in a sense. Next, you might want to check the bsseq documentation <a href="https://bioconductor.org/packages/bsseq/" class="uri">https://bioconductor.org/packages/bsseq/</a> if you are interested in finding differentially methylated regions (DMRs) and in any case, because BiocMAP provides that type of R object as noted at <a href="http://research.libd.org/BiocMAP/outputs.html" class="uri">http://research.libd.org/BiocMAP/outputs.html</a>. <a href="https://jhubiostatistics.shinyapps.io/wgbsExprs/" class="uri">https://jhubiostatistics.shinyapps.io/wgbsExprs/</a> and the companion paper is the last WGBS data analysis project I did myself with code at <a href="https://github.com/LieberInstitute/brain-epigenomics" class="uri">https://github.com/LieberInstitute/brain-epigenomics</a>. Kira Perzel-Mandell, who worked with Shizhong, wrote <a href="https://github.com/LieberInstitute/wgbs_meqtl_analysis" class="uri">https://github.com/LieberInstitute/wgbs_meqtl_analysis</a> (link to paper there) and did other projects. If you are plotting methylation levels at some locations, then that’s very direct. If you are trying to find DMRs, it can take a bit of time to get used to the large objects and the related complications (once you filter the data, it’ll be more manageable). If you go into CpH instead of CpG, then memory constraints become harder to deal with.</td>
</tr>
<tr class="odd">
<td>2022-02-18</td>
<td>Ideas for possible future analyses</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-02-16</td>
<td>JHPCE array jobs and fasthplus</td>
<td>Related to JHPCE <a href="https://github.com/ntdyjack/fasthplus" class="uri">https://github.com/ntdyjack/fasthplus</a> and <a href="http://research.libd.org/sgejobs/" class="uri">http://research.libd.org/sgejobs/</a></td>
</tr>
<tr class="odd">
<td>2022-02-15</td>
<td>snRNA-seq and deconvolution overview</td>
<td>Brainstormed several ideas related to deconvolution of bulk data as well as uses of snRNA-seq. Refered to OSCA <a href="https://bioconductor.org/books/release/OSCA/" class="uri">https://bioconductor.org/books/release/OSCA/</a> at time.</td>
</tr>
<tr class="even">
<td>2021-12-06</td>
<td>JHPCE file permissions and ACLs</td>
<td>Motivated /dcs04/lieber/lcolladotor/_jhpce_org_LIBD001/update_permissions_spatialteam.sh</td>
</tr>
<tr class="odd">
<td>2021-10-20</td>
<td>Transfering files between JHPCE disks</td>
<td>Explained some scripts such as /dcs04/lieber/lcolladotor/BrainSEQ_LIBD001/_jhpce_org</td>
</tr>
<tr class="even">
<td>2021-09-27</td>
<td>Presentation preparation feedback</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-08-06</td>
<td>How to write a Twitter thread for a paper</td>
<td>We revised key features of Brenda Pardo’s Twitter thread <a href="https://twitter.com/PardoBree/status/1388253938391175173?s=20" class="uri">https://twitter.com/PardoBree/status/1388253938391175173?s=20</a> and some features that you would want your Twitter thread to have.</td>
</tr>
<tr class="even">
<td>2021-08-03</td>
<td>Refining an abstract submission</td>
<td>We talked about all the details that go into writing an abstract for a conference. These details can take longer to complete than initially anticipated.</td>
</tr>
<tr class="odd">
<td>2021-07-30</td>
<td>Git branches</td>
<td>We revised how to make a new branch in Git working with <a href="https://github.com/lmweber/OSTA-book" class="uri">https://github.com/lmweber/OSTA-book</a></td>
</tr>
<tr class="even">
<td>2021-07-28</td>
<td>DNA genotyping overview</td>
<td>We discussed the current DNA genotype pipeline and potential changes others might want.</td>
</tr>
<tr class="odd">
<td>2021-07-20</td>
<td>Overview of the RNA-seq universe</td>
<td>From SPEAQeasy, to qSVA, deconvolution, TWAS, etc. It might be best to give a talk about all these topics to whoever is interested at LIBD since we know about all these pieces but others might not.</td>
</tr>
<tr class="even">
<td>2021-07-13</td>
<td>Setting up a team at LIBD</td>
<td>We talked about <a href="https://lcolladotor.github.io/bioc_team_ds/" class="uri">https://lcolladotor.github.io/bioc_team_ds/</a> among other topics.</td>
</tr>
<tr class="odd">
<td>2021-07-01</td>
<td>How to set up a GitHub actions-powered website</td>
<td>This has changed over time. The most recent setup (as of 2021-08-06) is at <a href="https://github.com/lmweber/OSTA-book" class="uri">https://github.com/lmweber/OSTA-book</a></td>
</tr>
<tr class="even">
<td>2021-06-29</td>
<td>Planning session for getting up to speed with SpatialExperiment, spatialLIBD, here and ggplot2</td>
<td>We talked about the set of videos and resources publicly available to get to learn about R, Bioconductor and these packages. It was a lot of information and I’m sure that we’ll have more sessions later on to clarify questions as they arise.</td>
</tr>
<tr class="odd">
<td>2021-06-29</td>
<td>How to choose and add a licence to a GitHub repo</td>
<td>We talked about <a href="https://choosealicense.com/" class="uri">https://choosealicense.com/</a>, <a href="https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository" class="uri">https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository</a>, <a href="https://creativecommons.org/choose/" class="uri">https://creativecommons.org/choose/</a>.</td>
</tr>
<tr class="even">
<td>2021-06-29</td>
<td>Designing an MS-level class</td>
<td>We discussed the posibility of proposing a class that has a mix of biology lectures and some practical coding sessions. The coding sessions could be based after the CDSB 2021 mini courses series.</td>
</tr>
<tr class="odd">
<td>2021-06-21</td>
<td>Locating public data at JHPCE + bamtofastq workflow</td>
<td>We talked about the public data from Synapse and worked on locating some of it at JHPCE. I explained also how we have used bamtofastq when others don’t share their FASTQ files. Basically, we use bamtofastq to reconstruct the original FASTQ files (you might have to append the unmapped FASTQs if they are provided by the original authors) and then run FASTQ-processing tools like our SPEAQeasy pipeline.</td>
</tr>
<tr class="even">
<td>2021-06-17</td>
<td>How to transfer recount3 data</td>
<td>We met to figure out how to transfer the data for this collaboration, which ended up being quite tricky and involved using an SSH tunnel for rsync. Basically, a command like this one: <code>rsync -avzh -e 'ssh -o ProxyJump=user@server_somewhere_for_the_tunnel' *_files_of_interest user@final_server:/final_path/</code></td>
</tr>
<tr class="odd">
<td>2021-06-17</td>
<td>How to get started with R</td>
<td>We discussed the different possible goals and ways to get to them and determined a path forward that relies on the rest of the DSgs system as well as existing LIBD rstats club videos to get started. It’ll be supported by a meeting with Leo and others involved in the spatial transcriptomics projects from the R side.</td>
</tr>
<tr class="even">
<td>2021-06-17</td>
<td>Brainstorming session on collaboration</td>
<td>I showed the initial version of the BrainSeq Phase II developmental expression explorer developed by Bill Ulrich and Aaron Goldman. We discussed other possibilities for the collaboration including helping set up SPEAQeasy in their servers.</td>
</tr>
<tr class="odd">
<td>2021-06-02</td>
<td>How to prepare for a LIBD seminar part 4</td>
<td>We discussed the changes that we should make and the ones we shouldn’t based on the feedback received from the practice talk with Keri’s group. The feedback was excellent and very useful, however, we discussed which suggestions were maybe complicating the presentation too much and how to best incorporate them.</td>
</tr>
<tr class="even">
<td>2021-05-26</td>
<td>How to prepare for a LIBD seminar part 3</td>
<td>Similar to part 2. The presentation was looking very polished by this meeting!</td>
</tr>
<tr class="odd">
<td>2021-05-24</td>
<td>Grant application review</td>
<td>Helped review the grant application and revise the text.</td>
</tr>
<tr class="even">
<td>2021-05-21</td>
<td>PredictDB weights</td>
<td>Further updates on asking for help with this complicated project. Suggested reaching out to other LIBD members who have tried running the software before.</td>
</tr>
<tr class="odd">
<td>2021-05-20</td>
<td>Help preparing a figure for a grant application</td>
<td>Discussed with Josh what figures we needed to prepare and how to generate them. This also involved base_size = 30 with ggplot2. Otherwise the font was too small to read it on the grant.</td>
</tr>
<tr class="even">
<td>2021-05-20</td>
<td>How to prepare for a LIBD seminar part 2</td>
<td>Now that a first major draft of the presentation was complete, we revised it and proposed some changes to it. Some of them involved remaking the figures using base_size = 20 or even 30 with ggplot2. For future work, it might be best to save the images with that font size from the beginning instead of having to update them later.</td>
</tr>
<tr class="odd">
<td>2021-05-10</td>
<td>Setting up covr using GitHub actions in an R package</td>
<td>We went over the details of the GitHub actions workflow created by biocthis, the parts that you need to run from biocthis::use_bioc_pkg_templates(), and resolving issues prompted by R CMD check + BiocCheck</td>
</tr>
<tr class="even">
<td>2021-05-07</td>
<td>How to get started with computational biology</td>
<td>I gave an overview of the resources and materials we have created as well as the LIBD bootcamps that we have publicly available to help others get started with computational biology. This is likely the first of several meetings.</td>
</tr>
<tr class="odd">
<td>2021-05-07</td>
<td>How to insert a figure using the Oxford Bioinfomratics Overleaf LaTeX template</td>
<td>We spent a significant amount troubleshooting the problem here (nearly two hours?). In the end, using the insert image “here” LaTeX syntax resolved the problem.</td>
</tr>
<tr class="even">
<td>2021-05-06</td>
<td>PredictDB weights</td>
<td>General advice on how to ask for help from open source projects</td>
</tr>
<tr class="odd">
<td>2021-05-06</td>
<td>How to prepare for a LIBD seminar</td>
<td>Initial planning session for the LIBD seminar. We talked about the different slides we planned to show and the flow of the presentation. We also scheduled 3 practice talks with different groups of people.</td>
</tr>
<tr class="even">
<td>2021-05-03</td>
<td>TWAS prep</td>
<td>Overview of the TWAS pipeline</td>
</tr>
<tr class="odd">
<td>2021-04-08</td>
<td>General guidance on how to learn how to do differential expression analyses</td>
<td>Created a learning plan involving several team members that is now documented at a channel in JHU Genomics Slack. That way everyone can keep track of what others are teaching the person who asked the question, and we can have a common communication channel among everyone involved. Sessions include learning how to interact with JHPCE through RStudio, how to run SPEAQeasy, key R/Bioconductor packages (limma, SummarizedExperiment), and explore possible visualizations of interest.</td>
</tr>
<tr class="even">
<td>2021-03-29</td>
<td>spatialLIBD support</td>
<td>Discussion about features that spatialLIBD could support to make it easier to use for newer data.</td>
</tr>
<tr class="odd">
<td>2021-03-25</td>
<td>Creating a documentation website using bookdown</td>
<td>Help with understanding <a href="https://bookdown.org/yihui/bookdown/">bookdown</a> and ideas on how to organize a documentation website (chapters and information to consider).</td>
</tr>
<tr class="even">
<td>2021-03-25</td>
<td>Clarification of LIBD support</td>
<td>Discussed the research project proposal and how we can help.</td>
</tr>
<tr class="odd">
<td>2021-03-25</td>
<td>General advice</td>
<td>General advice. Received feedback regarding file backups.</td>
</tr>
<tr class="even">
<td>2021-03-24</td>
<td>Download SRA data and run SPEAQeasy</td>
<td>I identified that <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829" class="uri">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829</a> contains both miRNA and RNA-seq data. Using data from the SRA Run selector at <a href="https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&amp;o=acc_s%3Aa" class="uri">https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&amp;o=acc_s%3Aa</a> I was able to filter the data to the RNA-seq samples and subset the input samples.manifest file for SPEAQeasy.</td>
</tr>
<tr class="odd">
<td>2021-03-24</td>
<td>Explain the feasibility of an on-demand end user interface for DEG and eQTLs</td>
<td>To identify DEGs or eQTLs, the user has to choose what covariates to adjust for. Some studies might require more complicated models than others, as they might be from more than one brain region or present some batch effect. In addition, in newer projects we are considering adjusting for cell type fractions derived from deconvolution. Even if all the covariates were fixed across datasets, in DEGs we have typically adjusted for qSVs which are computed in a dataset-dependent fashion and in essence are PCs derived from degradation-associated genomic regions. Similarly, for eQTLs we adjust for expression PCs. In both we adjust for ancestry PCs. Computing PCs is input-dependent: you won’t get the same values or PCs if you change which samples you are using. So, computing DEGs and eQTLs is not as simple as say a map-reduce problem where you compute the same thing for every input sample, then aggregate the per-sample results. We have steps that depend on the whole sample set. Then there’s also steps for filtering expression features based on some minimum expression cutoffs. Plus, differences between say polyA and RiboZero data, which don’t make sense to process together. For these reasons, I don’t think that an on-demand computation pipeline is feasible based on a user selecting a set of input samples. Constructing RNA-seq objects (RangedSummarizedExperiment objects) as well as DNA genotyping VCF files based on a set of input samples is feasible and that’s what others are building. After that, I think that it’s best to adapt code we have (or even use newer tools) after carefully exploring the data.</td>
</tr>
<tr class="even">
<td>2021-03-11</td>
<td>BSP2 eQTLs with a subset of SNPs</td>
<td>Adapted code and shared it at <a href="https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock" class="uri">https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock</a></td>
</tr>
<tr class="odd">
<td>2021-03-10</td>
<td>How to write a small methods paper</td>
<td>Created GitHub issues at <a href="https://github.com/LieberInstitute/VisiumImgProcessing/issues" class="uri">https://github.com/LieberInstitute/VisiumImgProcessing/issues</a></td>
</tr>
<tr class="even">
<td>2021-02-05</td>
<td>BSP2 LIBD eQTLs request</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-01-29</td>
<td>Help determining what data is available and what can be used for an analysis</td>
<td>Use of galaxybrain for identifying samples with DNAm, DNA genotype and RNA-seq data</td>
</tr>
<tr class="even">
<td>2021-01-26</td>
<td>Guidance on how to download data from a study in GEO</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-01-14</td>
<td>biocthis macOS GHA bug</td>
<td><a href="https://support.bioconductor.org/p/p134069/#p134141">BioC Support website answer</a></td>
</tr>
<tr class="even">
<td>2021-01-11</td>
<td>Questions related to accessing results from the DG paper by Andrew Jaffe</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2020-11-25</td>
<td>Accesing CMC RNA-seq data</td>
<td>Started PsychENCODE access request process</td>
</tr>
<tr class="even">
<td>2020-11-11</td>
<td>Differences between recount2’s “bp_length” and SPEAQeasy’s “Length”</td>
<td><a href="https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1605123296024100">Check Slack helpdesk channel</a></td>
</tr>
<tr class="odd">
<td>2020-11-11</td>
<td>GitHub actions workflow issue with macOS on R-devel</td>
<td><a href="https://github.com/lcolladotor/biocthis/issues/15">GitHub issue at biocthis</a></td>
</tr>
<tr class="even">
<td>2020-11-03</td>
<td>Fixed two bugs on recount3 related to locate_url() for the BigWig files.</td>
<td>Resulted in recount3 versions 1.1.1 and 1.1.2 on bioc-devel (and 1.0.2 on bioc-release)</td>
</tr>
<tr class="odd">
<td>2020-11-03</td>
<td>Fixed a bug on biocthis for release branches</td>
<td><a href="https://github.com/lcolladotor/biocthis/issues/13">GitHub issue at biocthis</a></td>
</tr>
<tr class="even">
<td>2020-11-03</td>
<td>Issue with knitcitations</td>
<td><a href="https://github.com/lcolladotor/biocthis/issues/14">GitHub issue at biocthis</a></td>
</tr>
<tr class="odd">
<td>2020-10-30</td>
<td>Issue with JHPCE file permissions</td>
<td>Explained the <code>chgrp</code> and <code>umask</code> commands plus <code>~/.bashrc</code> file configuration. I also mentioned why we don’t use <code>chmod 777</code> at JHPCE. Details on <a href="https://jhu-genomics.slack.com/archives/CR9NYA0BF/p1604086263148400?thread_ts=1603826219.094600&amp;cid=CR9NYA0BF">Slack</a> (private channel).</td>
</tr>
<tr class="even">
<td>2020-10-30</td>
<td>recount3 GTEx data access</td>
<td><a href="https://support.bioconductor.org/p/p132487/#p132521">BioC Support website answer</a></td>
</tr>
<tr class="odd">
<td>2020-10-28</td>
<td>exceRpt pipeline counts</td>
<td><a href="https://support.bioconductor.org/p/p132450/#p132464">BioC Support website answer</a></td>
</tr>
<tr class="even">
<td>2020-10-21</td>
<td>Ballgown stattest</td>
<td><a href="https://support.bioconductor.org/p/134914/#134915">BioC Support website answer</a></td>
</tr>
</tbody>
</table>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="josh.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="louise.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
