<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>3 Leo | Data Science guidance session logs</title>
  <meta name="description" content="3 Leo | Data Science guidance session logs" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="3 Leo | Data Science guidance session logs" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="3 Leo | Data Science guidance session logs" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="josh.html"/>
<link rel="next" href="louise.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#dsgs-guide-team-members"><i class="fa fa-check"></i>DSgs-guide team members</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#graphs"><i class="fa fa-check"></i>Graphs</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#r-session-information"><i class="fa fa-check"></i>R session information</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="arta.html"><a href="arta.html"><i class="fa fa-check"></i><b>1</b> Arta</a></li>
<li class="chapter" data-level="2" data-path="josh.html"><a href="josh.html"><i class="fa fa-check"></i><b>2</b> Josh</a></li>
<li class="chapter" data-level="3" data-path="leo.html"><a href="leo.html"><i class="fa fa-check"></i><b>3</b> Leo</a></li>
<li class="chapter" data-level="4" data-path="louise.html"><a href="louise.html"><i class="fa fa-check"></i><b>4</b> Louise</a></li>
<li class="chapter" data-level="5" data-path="maddy.html"><a href="maddy.html"><i class="fa fa-check"></i><b>5</b> Maddy</a></li>
<li class="chapter" data-level="6" data-path="nick.html"><a href="nick.html"><i class="fa fa-check"></i><b>6</b> Nick</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data Science guidance session logs</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="leo" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Leo</h1>
<table>
<colgroup>
<col width="19%" />
<col width="57%" />
<col width="23%" />
</colgroup>
<thead>
<tr class="header">
<th>Date</th>
<th>Help request</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2020-10-21</td>
<td>Ballgown stattest</td>
<td><a href="https://support.bioconductor.org/p/134914/#134915">BioC Support website answer</a></td>
</tr>
<tr class="even">
<td>2020-10-28</td>
<td>exceRpt pipeline counts</td>
<td><a href="https://support.bioconductor.org/p/p132450/#p132464">BioC Support website answer</a></td>
</tr>
<tr class="odd">
<td>2020-10-30</td>
<td>recount3 GTEx data access</td>
<td><a href="https://support.bioconductor.org/p/p132487/#p132521">BioC Support website answer</a></td>
</tr>
<tr class="even">
<td>2020-10-30</td>
<td>Issue with JHPCE file permissions</td>
<td>Explained the <code>chgrp</code> and <code>umask</code> commands plus <code>~/.bashrc</code> file configuration. I also mentioned why we don’t use <code>chmod 777</code> at JHPCE. Details on <a href="https://jhu-genomics.slack.com/archives/CR9NYA0BF/p1604086263148400?thread_ts=1603826219.094600&amp;cid=CR9NYA0BF">Slack</a> (private channel).</td>
</tr>
<tr class="odd">
<td>2020-11-03</td>
<td>Issue with knitcitations</td>
<td><a href="https://github.com/lcolladotor/biocthis/issues/14">GitHub issue at biocthis</a></td>
</tr>
<tr class="even">
<td>2020-11-03</td>
<td>Fixed a bug on biocthis for release branches</td>
<td><a href="https://github.com/lcolladotor/biocthis/issues/13">GitHub issue at biocthis</a></td>
</tr>
<tr class="odd">
<td>2020-11-03</td>
<td>Fixed two bugs on recount3 related to locate_url() for the BigWig files.</td>
<td>Resulted in recount3 versions 1.1.1 and 1.1.2 on bioc-devel (and 1.0.2 on bioc-release)</td>
</tr>
<tr class="even">
<td>2020-11-11</td>
<td>GitHub actions workflow issue with macOS on R-devel</td>
<td><a href="https://github.com/lcolladotor/biocthis/issues/15">GitHub issue at biocthis</a></td>
</tr>
<tr class="odd">
<td>2020-11-11</td>
<td>Differences between recount2’s “bp_length” and SPEAQeasy’s “Length”</td>
<td><a href="https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1605123296024100">Check Slack helpdesk channel</a></td>
</tr>
<tr class="even">
<td>2020-11-25</td>
<td>Accesing CMC RNA-seq data</td>
<td>Started PsychENCODE access request process</td>
</tr>
<tr class="odd">
<td>2021-01-11</td>
<td>Questions related to accessing results from the DG paper by Andrew Jaffe</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-01-14</td>
<td>biocthis macOS GHA bug</td>
<td><a href="https://support.bioconductor.org/p/p134069/#p134141">BioC Support website answer</a></td>
</tr>
<tr class="odd">
<td>2021-01-26</td>
<td>Guidance on how to download data from a study in GEO</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-01-29</td>
<td>Help determining what data is available and what can be used for an analysis</td>
<td>Use of galaxybrain for identifying samples with DNAm, DNA genotype and RNA-seq data</td>
</tr>
<tr class="odd">
<td>2021-02-05</td>
<td>BSP2 LIBD eQTLs request</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-03-10</td>
<td>How to write a small methods paper</td>
<td>Created GitHub issues at <a href="https://github.com/LieberInstitute/VisiumImgProcessing/issues" class="uri">https://github.com/LieberInstitute/VisiumImgProcessing/issues</a></td>
</tr>
<tr class="odd">
<td>2021-03-11</td>
<td>BSP2 eQTLs with a subset of SNPs</td>
<td>Adapted code and shared it at <a href="https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock" class="uri">https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock</a></td>
</tr>
<tr class="even">
<td>2021-03-24</td>
<td>Download SRA data and run SPEAQeasy</td>
<td>I identified that <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829" class="uri">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829</a> contains both miRNA and RNA-seq data. Using data from the SRA Run selector at <a href="https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&amp;o=acc_s%3Aa" class="uri">https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&amp;o=acc_s%3Aa</a> I was able to filter the data to the RNA-seq samples and subset the input samples.manifest file for SPEAQeasy.</td>
</tr>
<tr class="odd">
<td>2021-03-24</td>
<td>Explain the feasibility of an on-demand end user interface for DEG and eQTLs</td>
<td>To identify DEGs or eQTLs, the user has to choose what covariates to adjust for. Some studies might require more complicated models than others, as they might be from more than one brain region or present some batch effect. In addition, in newer projects we are considering adjusting for cell type fractions derived from deconvolution. Even if all the covariates were fixed across datasets, in DEGs we have typically adjusted for qSVs which are computed in a dataset-dependent fashion and in essence are PCs derived from degradation-associated genomic regions. Similarly, for eQTLs we adjust for expression PCs. In both we adjust for ancestry PCs. Computing PCs is input-dependent: you won’t get the same values or PCs if you change which samples you are using. So, computing DEGs and eQTLs is not as simple as say a map-reduce problem where you compute the same thing for every input sample, then aggregate the per-sample results. We have steps that depend on the whole sample set. Then there’s also steps for filtering expression features based on some minimum expression cutoffs. Plus, differences between say polyA and RiboZero data, which don’t make sense to process together. For these reasons, I don’t think that an on-demand computation pipeline is feasible based on a user selecting a set of input samples. Constructing RNA-seq objects (RangedSummarizedExperiment objects) as well as DNA genotyping VCF files based on a set of input samples is feasible and that’s what others are building. After that, I think that it’s best to adapt code we have (or even use newer tools) after carefully exploring the data.</td>
</tr>
<tr class="even">
<td>2021-03-25</td>
<td>Creating a documentation website using bookdown</td>
<td>Help with understanding <a href="https://bookdown.org/yihui/bookdown/">bookdown</a> and ideas on how to organize a documentation website (chapters and information to consider).</td>
</tr>
<tr class="odd">
<td>2021-03-25</td>
<td>Clarification of LIBD support</td>
<td>Discussed the research project proposal and how we can help.</td>
</tr>
<tr class="even">
<td>2021-03-25</td>
<td>General advice</td>
<td>General advice. Received feedback regarding file backups.</td>
</tr>
<tr class="odd">
<td>2021-03-29</td>
<td>spatialLIBD support</td>
<td>Discussion about features that spatialLIBD could support to make it easier to use for newer data.</td>
</tr>
<tr class="even">
<td>2021-04-08</td>
<td>General guidance on how to learn how to do differential expression analyses</td>
<td>Created a learning plan involving several team members that is now documented at a channel in JHU Genomics Slack. That way everyone can keep track of what others are teaching the person who asked the question, and we can have a common communication channel among everyone involved. Sessions include learning how to interact with JHPCE through RStudio, how to run SPEAQeasy, key R/Bioconductor packages (limma, SummarizedExperiment), and explore possible visualizations of interest.</td>
</tr>
<tr class="odd">
<td>2021-05-03</td>
<td>TWAS prep</td>
<td>Overview of the TWAS pipeline</td>
</tr>
<tr class="even">
<td>2021-05-06</td>
<td>How to prepare for a LIBD seminar</td>
<td>Initial planning session for the LIBD seminar. We talked about the different slides we planned to show and the flow of the presentation. We also scheduled 3 practice talks with different groups of people.</td>
</tr>
<tr class="odd">
<td>2021-05-06</td>
<td>PredictDB weights</td>
<td>General advice on how to ask for help from open source projects</td>
</tr>
<tr class="even">
<td>2021-05-07</td>
<td>How to get started with computational biology</td>
<td>I gave an overview of the resources and materials we have created as well as the LIBD bootcamps that we have publicly available to help others get started with computational biology. This is likely the first of several meetings.</td>
</tr>
<tr class="odd">
<td>2021-05-07</td>
<td>How to insert a figure using the Oxford Bioinfomratics Overleaf LaTeX template</td>
<td>We spent a significant amount troubleshooting the problem here (nearly two hours?). In the end, using the insert image “here” LaTeX syntax resolved the problem.</td>
</tr>
<tr class="even">
<td>2021-05-10</td>
<td>Setting up covr using GitHub actions in an R package</td>
<td>We went over the details of the GitHub actions workflow created by biocthis, the parts that you need to run from biocthis::use_bioc_pkg_templates(), and resolving issues prompted by R CMD check + BiocCheck</td>
</tr>
<tr class="odd">
<td>2021-05-20</td>
<td>How to prepare for a LIBD seminar part 2</td>
<td>Now that a first major draft of the presentation was complete, we revised it and proposed some changes to it. Some of them involved remaking the figures using base_size = 20 or even 30 with ggplot2. For future work, it might be best to save the images with that font size from the beginning instead of having to update them later.</td>
</tr>
<tr class="even">
<td>2021-05-20</td>
<td>Help preparing a figure for a grant application</td>
<td>Discussed with Josh what figures we needed to prepare and how to generate them. This also involved base_size = 30 with ggplot2. Otherwise the font was too small to read it on the grant.</td>
</tr>
<tr class="odd">
<td>2021-05-21</td>
<td>PredictDB weights</td>
<td>Further updates on asking for help with this complicated project. Suggested reaching out to other LIBD members who have tried running the software before.</td>
</tr>
<tr class="even">
<td>2021-05-24</td>
<td>Grant application review</td>
<td>Helped review the grant application and revise the text.</td>
</tr>
<tr class="odd">
<td>2021-05-26</td>
<td>How to prepare for a LIBD seminar part 3</td>
<td>Similar to part 2. The presentation was looking very polished by this meeting!</td>
</tr>
<tr class="even">
<td>2021-06-02</td>
<td>How to prepare for a LIBD seminar part 4</td>
<td>We discussed the changes that we should make and the ones we shouldn’t based on the feedback received from the practice talk with Keri’s group. The feedback was excellent and very useful, however, we discussed which suggestions were maybe complicating the presentation too much and how to best incorporate them.</td>
</tr>
<tr class="odd">
<td>2021-06-17</td>
<td>Brainstorming session on collaboration</td>
<td>I showed the initial version of the BrainSeq Phase II developmental expression explorer developed by Bill Ulrich and Aaron Goldman. We discussed other possibilities for the collaboration including helping set up SPEAQeasy in their servers.</td>
</tr>
<tr class="even">
<td>2021-06-17</td>
<td>How to transfer recount3 data</td>
<td>We met to figure out how to transfer the data for this collaboration, which ended up being quite tricky and involved using an SSH tunnel for rsync. Basically, a command like this one: <code>rsync -avzh -e 'ssh -o ProxyJump=user@server_somewhere_for_the_tunnel' *_files_of_interest user@final_server:/final_path/</code></td>
</tr>
<tr class="odd">
<td>2021-06-17</td>
<td>How to get started with R</td>
<td>We discussed the different possible goals and ways to get to them and determined a path forward that relies on the rest of the DSgs system as well as existing LIBD rstats club videos to get started. It’ll be supported by a meeting with Leo and others involved in the spatial transcriptomics projects from the R side.</td>
</tr>
<tr class="even">
<td>2021-06-21</td>
<td>Locating public data at JHPCE + bamtofastq workflow</td>
<td>We talked about the public data from Synapse and worked on locating some of it at JHPCE. I explained also how we have used bamtofastq when others don’t share their FASTQ files. Basically, we use bamtofastq to reconstruct the original FASTQ files (you might have to append the unmapped FASTQs if they are provided by the original authors) and then run FASTQ-processing tools like our SPEAQeasy pipeline.</td>
</tr>
</tbody>
</table>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="josh.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="louise.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
