<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4 Louise | Data Science guidance session logs</title>
  <meta name="description" content="4 Louise | Data Science guidance session logs" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="4 Louise | Data Science guidance session logs" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4 Louise | Data Science guidance session logs" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="leo.html"/>
<link rel="next" href="maddy.html"/>
<script src="libs/header-attrs-2.12/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="index.html#overview">Overview<span></span></a>
<ul>
<li><a href="index.html#dsgs-guide-team-members">DSgs-guide team members<span></span></a></li>
<li><a href="index.html#graphs">Graphs<span></span></a></li>
<li><a href="index.html#r-session-information">R session information<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="arta.html"><a href="arta.html"><i class="fa fa-check"></i><b>1</b> Arta<span></span></a></li>
<li class="chapter" data-level="2" data-path="josh.html"><a href="josh.html"><i class="fa fa-check"></i><b>2</b> Josh<span></span></a></li>
<li class="chapter" data-level="3" data-path="leo.html"><a href="leo.html"><i class="fa fa-check"></i><b>3</b> Leo<span></span></a></li>
<li class="chapter" data-level="4" data-path="louise.html"><a href="louise.html"><i class="fa fa-check"></i><b>4</b> Louise<span></span></a></li>
<li class="chapter" data-level="5" data-path="maddy.html"><a href="maddy.html"><i class="fa fa-check"></i><b>5</b> Maddy<span></span></a></li>
<li class="chapter" data-level="6" data-path="nick.html"><a href="nick.html"><i class="fa fa-check"></i><b>6</b> Nick<span></span></a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data Science guidance session logs</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="louise" class="section level1 hasAnchor" number="4">
<h1><span class="header-section-number">4</span> Louise<a href="louise.html#louise" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<table>
<colgroup>
<col width="19%" />
<col width="57%" />
<col width="23%" />
</colgroup>
<thead>
<tr class="header">
<th>Date</th>
<th>Help request</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2022-03-04</td>
<td>Understand the covariates used in scRNA-seq cell type analysis for a project we are doing with striatal organoids</td>
<td>We went over the files (data and script) for the deconvolution analysis done for the habenula in BSPIII. We also discussed the theroy/methodology for deconvolution.</td>
</tr>
<tr class="even">
<td>2022-02-24</td>
<td>I would like to learn how you ran tensorQTL on the MDD data, maybe with using the GPU nodes for it.</td>
<td>Geo was interested int eh workflow I developed for tensorQTL. I went over the requirements and input formats that tensorqtl requirers, and how I had previously made them from our dataset. Also discussed the differnces between the functions and their outputs.</td>
</tr>
<tr class="odd">
<td>2022-02-24</td>
<td>Help building for loops to analyze WGCNA ouput data</td>
<td>We discussed how to build and save data with a for loop and with purrr::map. Went over how to define a function in map to do nested loops</td>
</tr>
<tr class="even">
<td>2022-02-15</td>
<td>Access to the gene info in the rowData of the Summarized Experiment objects to annotate eqtl results</td>
<td>We went over how to acess the rowData, and what the rowData inthe MDD project contains. Ended up adding columns to the eqtl risk snp subsets to help with analysis</td>
</tr>
<tr class="odd">
<td>2022-01-20</td>
<td>Troubleshooting ExprssionSet conversion</td>
<td>Fixed row names to both be ensembl IDs, discussed simplifing cell types in sce, converting sce arrarys to matrix</td>
</tr>
<tr class="even">
<td>2022-01-19</td>
<td>Help using BisqueRNA, getting snRNA-seq into correct input format, chosing refrence vs. marker based deconvolution</td>
<td>I shared some detials of our deconvolution approach, we walked through the SCE to Expression set converson, and discussed some issues that may come up</td>
</tr>
<tr class="odd">
<td>2021-11-03</td>
<td>Debugging help with rse_tx object</td>
<td>We discussed how to check for duplicate rownames, and stratedgies for variable naming + working with rse objects</td>
</tr>
<tr class="even">
<td>2021-10-29</td>
<td>qc general scripts, rnaseq scripts general, ggplot histogram with breaks</td>
<td>We found a R function that helped produce the plot she was looking for</td>
</tr>
<tr class="odd">
<td>2021-08-26</td>
<td>using map to cbind a list of six to its corresponding list of lists. The second is a list of 6, each a list of 3.</td>
<td>Did some troubleshooting on nested map functions, when to use ~.x and when to use function() syntax</td>
</tr>
<tr class="even">
<td>2021-08-23</td>
<td>Using purrr and map to loop over datasets and apply qsva</td>
<td>Went over basic, complex, and nested map() usage and how to apply it to Josh’s current work</td>
</tr>
<tr class="odd">
<td>2021-08-20</td>
<td>Help with JHPCE login issues</td>
<td>We were able to update the keys for the cluster, but the ssh key login was still broken</td>
</tr>
<tr class="even">
<td>2021-06-30</td>
<td>Debugging ggplot code</td>
<td>We reviewed her code and I gave some pointer on what bugs to look out for with certain errors in ggplot</td>
</tr>
<tr class="odd">
<td>2021-06-17</td>
<td>Manhattan plot making and tstatistic</td>
<td>Pointed her to some oacjages for creating TWAS plots whe was interested in and we walked through some code I used to compare T-stats from different DE models</td>
</tr>
<tr class="even">
<td>2021-06-03</td>
<td>questions about expression_cut and pca plots</td>
<td>We walked tought script from getRPKM to expression cut to filter rse objects, I also pointed Brenda towards some reasources to get started with ggplot</td>
</tr>
<tr class="odd">
<td>2021-05-28</td>
<td>Differential Expression help (continuation of 5/20)</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-05-20</td>
<td>Making plots for DE studies</td>
<td>We reviewed the model that Sreya was evaluating for DE. As we as troublshooting the plot functions from the SPEAQeasy bootcamp code. I also advised taht she could create a volcano plot for her results.</td>
</tr>
<tr class="odd">
<td>2021-05-12</td>
<td>Making plots for DE studies</td>
<td>We went over some plots we have made for DE previously, (quality control &amp; PC scatter plots, t-stat scatter between models, venn diagrams, and boxplots). Talked about some things to watch out for in her data (small samples size). We are planning to meet again when her data is ready.</td>
</tr>
<tr class="even">
<td>2021-04-19</td>
<td>JHPCE set up on mac</td>
<td>We got ssh key login working, added config files to JHPCE, went though a “hello world” example of using files on JHPCE on RStudio</td>
</tr>
<tr class="odd">
<td>2021-04-15</td>
<td>JHPCE set up on mac</td>
<td>Henry needed to reset his JHPCE password as he had forgotten it, but we went through RStudio set up and got cyberduck installed, we will set up a meeting next week for ssh keys and configuration</td>
</tr>
<tr class="even">
<td>2021-03-26</td>
<td>Need help understanding the sce - single cell object</td>
<td>Went over basic functions of SummarizedExperiment and SingleCellExperiment, what a Sparse matrix is - how to convert to non-sparse. Steps to take to use getRPKM on SingeCellExperient.</td>
</tr>
<tr class="odd">
<td>2021-03-19</td>
<td>Change point size in vis_clust_grid</td>
<td>We walked througth the SpatialLIBD code to find where the argument has to be changed</td>
</tr>
<tr class="even">
<td>2021-03-05</td>
<td>Maker finding for cell types in single cell data</td>
<td>I showed them the tools I developed in DeconvoBuddies pacakge, and provided links to example code where I had applied these funcitons in previous work</td>
</tr>
<tr class="odd">
<td>2021-02-09</td>
<td>Visualizing RPKM valus for genes in Humna and Primarte data</td>
<td>We spent time doing basic Rstudio setup and troubleshooting, and got the data downloaded</td>
</tr>
<tr class="even">
<td>2021-02-04</td>
<td>Work on habenula deconvolution and debug the deconvolution package</td>
<td>We went over applying DeconvoBuddies functions to hebenula data</td>
</tr>
<tr class="odd">
<td>2021-02-04</td>
<td>Github ssh key set up</td>
<td>We got the ssh keys working for Brenda’s github</td>
</tr>
<tr class="even">
<td>2021-01-27</td>
<td>group_by bug for get_mean_ratio</td>
<td>We are going to try an iterative strategy to keep tables smaller</td>
</tr>
<tr class="odd">
<td>2020-12-08</td>
<td>Resolving issues in phenotype data between genotype, expression, and LIBDpheno</td>
<td>Need to resolve swap bugs. Plan to create a table of phenotype data (Dx, age, sex, race) for each BrNum to mach genotype data</td>
</tr>
<tr class="even">
<td>2020-12-07</td>
<td>jhpce set up on mac</td>
<td>Went through the setting up passwordless login for jhpce using ssh keys, and coppied the configuration files from <a href="https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee" class="uri">https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee</a></td>
</tr>
<tr class="odd">
<td>2020-11-18</td>
<td>Help clustering sce data</td>
<td>We debugged some code and ultimetly decided that it might be best for Abby to write new code and use the tsne function to cluster and plot the data</td>
</tr>
<tr class="even">
<td>2020-11-13</td>
<td>Using reshape package to transform table for plotting</td>
<td>I recomended using dcast to do a “long” to “wide” data transformation</td>
</tr>
<tr class="odd">
<td>2020-11-12</td>
<td>Help adapting ggplot script for TWAS Z-score</td>
<td>We talked about how the data should be structured before using ggplot (transforming some data and adding catagorical columns)</td>
</tr>
<tr class="even">
<td>2020-11-05</td>
<td>LC repo organization</td>
<td>We discussed organizing by analysis (a directory for each one, then prehaps sub- directory for data &amp; plots). Create a readMe for each sub-folder. Use gitignore to keep repo from getting too big.</td>
</tr>
<tr class="odd">
<td>2020-10-27</td>
<td>Plotly help if possible? I’m trying to export an html file that allows the user to look at each snp/gene on the plot, and I am almost successful. Leo recommended that I talk to you on this matter.</td>
<td>Needed to use highlight_key and highlight functions to select points between plots. Used rstats example as guide <a href="https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14" class="uri">https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14</a></td>
</tr>
<tr class="even">
<td>2020-10-21</td>
<td>Question: Subset center regions of a set of different length sequences</td>
<td><a href="https://support.bioconductor.org/p/134532/#134925">BioC Support website answer</a></td>
</tr>
<tr class="odd">
<td>2020-10-20</td>
<td>Help with spatstat</td>
<td>spatstat package <a href="https://spatstat.org/releasenotes/spatstat-1.39-0.html" class="uri">https://spatstat.org/releasenotes/spatstat-1.39-0.html</a>, L function <a href="https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest" class="uri">https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest</a>, Kmark <a href="https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark" class="uri">https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark</a></td>
</tr>
</tbody>
</table>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="leo.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="maddy.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
