<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Maddy | Data Science guidance session logs</title>
  <meta name="description" content="6 Maddy | Data Science guidance session logs" />
  <meta name="generator" content="bookdown 0.25 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Maddy | Data Science guidance session logs" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Maddy | Data Science guidance session logs" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="louise.html"/>
<link rel="next" href="nick.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotly-main-2.5.1/plotly-latest.min.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="index.html#overview">Overview<span></span></a>
<ul>
<li><a href="index.html#dsgs-guide-team-members">DSgs-guide team members<span></span></a></li>
<li><a href="index.html#graphs">Graphs<span></span></a></li>
<li><a href="index.html#r-session-information">R session information<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="abby.html"><a href="abby.html"><i class="fa fa-check"></i><b>1</b> Abby<span></span></a></li>
<li class="chapter" data-level="2" data-path="arta.html"><a href="arta.html"><i class="fa fa-check"></i><b>2</b> Arta<span></span></a></li>
<li class="chapter" data-level="3" data-path="josh.html"><a href="josh.html"><i class="fa fa-check"></i><b>3</b> Josh<span></span></a></li>
<li class="chapter" data-level="4" data-path="leo.html"><a href="leo.html"><i class="fa fa-check"></i><b>4</b> Leo<span></span></a></li>
<li class="chapter" data-level="5" data-path="louise.html"><a href="louise.html"><i class="fa fa-check"></i><b>5</b> Louise<span></span></a></li>
<li class="chapter" data-level="6" data-path="maddy.html"><a href="maddy.html"><i class="fa fa-check"></i><b>6</b> Maddy<span></span></a></li>
<li class="chapter" data-level="7" data-path="nick.html"><a href="nick.html"><i class="fa fa-check"></i><b>7</b> Nick<span></span></a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data Science guidance session logs</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="maddy" class="section level1 hasAnchor" number="6">
<h1><span class="header-section-number">6</span> Maddy<a href="maddy.html#maddy" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<table>
<colgroup>
<col width="19%" />
<col width="57%" />
<col width="23%" />
</colgroup>
<thead>
<tr class="header">
<th>Date</th>
<th>Help request</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2022-04-06</td>
<td>applying size threshold to vistoseg outputs to get rid of smaller regions that are not nuclei</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-04-05</td>
<td>directed the person to the VistoSeg website for steps to run VistoSeg</td>
<td><a href="https://github.com/LieberInstitute/VistoSeg/issues/11" class="uri">https://github.com/LieberInstitute/VistoSeg/issues/11</a></td>
</tr>
<tr class="odd">
<td>2022-03-22</td>
<td>Help run trial shiny app deployment</td>
<td>Showed her app.R, deploy.R scripts and places where to edit, showed her how to navigate through the shinyapp portal to looks for logs when deploying the app. EXplained her few erros she ran through when deploying for the 1st time.</td>
</tr>
<tr class="even">
<td>2022-03-09</td>
<td>VisiumIF loupe alignment</td>
<td>The contrast adjusted images on MAC::preview should be saved and uploaded back to JHPCE to run spaceranger. Spaceranger needs the exact same image that was used in the loupe alignment (even if it is changed from raw image on preview)</td>
</tr>
<tr class="odd">
<td>2022-03-09</td>
<td>Masking of blood vessels from PNN channel in PNN project</td>
<td>Explained how the image data can be manipulated when claudin5 channel and WFA channel are merged and then the blood vessels are segmented and masked. The correct way is to segment blood vessels from claudin channel and mask the segmented objects in WFA channel.</td>
</tr>
<tr class="even">
<td>2022-03-09</td>
<td>How are InForm tiles arranged</td>
<td>The indices listed in the tiles’s filename is the x,y coordinates of the starting topleft pixel of the given tile. These indices never start with [0,0] as these tiles are picked from the ROI in the phenochart and not the entire imaged area. Once stored in a directory these tiles are not arranged in chronological order.</td>
</tr>
<tr class="odd">
<td>2022-02-25</td>
<td>Data type change of image data when loading into Python</td>
<td>Used KJ’s code from his rstats club presentation and described how the images shoukd be read using numpy array as uint32 but not floating <a href="https://colab.research.google.com/drive/1A4cavWcbgZF1PFQT87s5s_ATyZrN3IxL?usp=sharing" class="uri">https://colab.research.google.com/drive/1A4cavWcbgZF1PFQT87s5s_ATyZrN3IxL?usp=sharing</a></td>
</tr>
<tr class="even">
<td>2022-02-23</td>
<td>contrast adjustment, intensity normalization, channel/gene description in PNN images</td>
<td>used HALO to show how contrast can be adjusted when changing min max values in the image</td>
</tr>
<tr class="odd">
<td>2022-02-17</td>
<td>rename files on JHPCE through command line</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-02-16</td>
<td>Provide comparison table for different metrics obtained from image meta data in MATLAB, for Visium IF workflow</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2022-02-15</td>
<td>Explain Tangram, scRNAseq count matrix, spe object</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-02-14</td>
<td>Explain the concept of Squidpy paper</td>
<td><a href="https://www.nature.com/articles/s41592-021-01358-2" class="uri">https://www.nature.com/articles/s41592-021-01358-2</a></td>
</tr>
<tr class="odd">
<td>2022-02-11</td>
<td>tiling and metadata changes in PNN project</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-02-11</td>
<td>explaination on Image metadata, bit depth</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2022-02-10</td>
<td>Need for rotation of visium array images and its spatial coordinnnate in spe object</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2022-01-25</td>
<td>VistoSegIF pipeline</td>
<td>explained the functions of this pipline as these are not yet documented</td>
</tr>
<tr class="odd">
<td>2022-01-21</td>
<td>Transfer data from JHPCE to Neural_Plasticity</td>
<td>command to use <code>rsync -avzh --progress userid@jhpce-transfer01:/source directory/ /destination directory/</code></td>
</tr>
<tr class="even">
<td>2022-01-21</td>
<td>Transfer data Neural_Plasticity to JHPCE</td>
<td>use rsync, prepend spaces in the paths with <code>\</code> to avoid issues</td>
</tr>
<tr class="odd">
<td>2022-01-21</td>
<td>JHPCE directory organization, repo linking, making commits</td>
<td><a href="https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/43106c664883fd5df301e44cca35f2e8f2f289ad" class="uri">https://github.com/LieberInstitute/spatialDLPFC_SCZ/commit/43106c664883fd5df301e44cca35f2e8f2f289ad</a></td>
</tr>
<tr class="even">
<td>2022-01-21</td>
<td>opening python, R, MATLAB console interactive session on JHPCE</td>
<td>explain how to use IDE’s on local machine to edits scripts on JHPCE through cyberduck</td>
</tr>
<tr class="odd">
<td>2022-01-20</td>
<td>glitch in CAPTure tutorial about input data types to a function</td>
<td><a href="https://github.com/LieberInstitute/CaPTure/issues/2" class="uri">https://github.com/LieberInstitute/CaPTure/issues/2</a></td>
</tr>
<tr class="even">
<td>2021-12-09</td>
<td>Removing smaller objects from VistoSeg segmentations</td>
<td>refining segmentations in matlab by applying size thresholds</td>
</tr>
<tr class="odd">
<td>2021-12-08</td>
<td>applying kmeans to CA3 mouse RNAscope data to obtain gene expression levels</td>
<td>Showed code how to scale data based on <a href="https://uc-r.github.io/kmeans_clustering" class="uri">https://uc-r.github.io/kmeans_clustering</a> and optimise kmeans for the present dataset</td>
</tr>
<tr class="even">
<td>2021-11-18</td>
<td>running loupe browser for HPC images</td>
<td>Showed the steps on one sample HPC image and what outputs to save.</td>
</tr>
<tr class="odd">
<td>2021-11-18</td>
<td>countSpots from VistoSeg throwing errors for one DLPFC sample from round4</td>
<td>While debugging, its seen that VistoSeg when splitting the raw image missed a part of fiducial frame for capture area D1 of that sample. This happens when the center of the visium slide does not align with the center of the image, which has ti be adjusted while imaging. This image when run through spaceranger provided negative coordinates for the missing fiducial frame part of the image, but MATLAB will not accept negative indices for a image, which was throwing errors when recontructing spot grind in countSpots function.</td>
</tr>
<tr class="even">
<td>2021-11-18</td>
<td>Resolve issues with comments</td>
<td>Showed how to close issues on git with detailed comments of the final results</td>
</tr>
<tr class="odd">
<td>2021-11-16</td>
<td>batch processing - dotdotdot, on local machine</td>
<td>made custom Matlab script to batch process her 30 RNAscope images on her local machine</td>
</tr>
<tr class="even">
<td>2021-11-15</td>
<td>error in transferring Visuim HPC data to JHPCE</td>
<td>The user directories were cleared during the software updates for LIBD server, emailed bill to create one for me and Heena to use rsync on Neural_Plasiticity. The filenames Heena used for ‘source’ in rync did not exist at the source directory which was throwing errors.</td>
</tr>
<tr class="odd">
<td>2021-11-04</td>
<td>using awk</td>
<td>to pick filename from textfile for a array job</td>
</tr>
<tr class="even">
<td>2021-10-14</td>
<td>running spotspotcheck for SFN</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-10-07</td>
<td>doubts on VistoSeg</td>
<td>Explaination of why an index label M=3 was chosen for refineVNS in section 2.2.</td>
</tr>
<tr class="even">
<td>2021-09-20</td>
<td>issues with regioprops function in dotdotdot</td>
<td><a href="https://github.com/LieberInstitute/dotdotdot/issues/14" class="uri">https://github.com/LieberInstitute/dotdotdot/issues/14</a></td>
</tr>
<tr class="odd">
<td>2021-09-14</td>
<td>Fiducial frame enhancement for Visium IF images using preview on MAC</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-09-14</td>
<td>running Loupe browser for mannual alignment</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-09-09</td>
<td>Creating cyberduck bookmarks for JHPCE paths</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-09-09</td>
<td>writing shell script for countNuclei function in VistoSeg</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-09-09</td>
<td>Explaining Github issues, history and version control</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-09-09</td>
<td>Explaining SPE (Spatial Experiment) SCE (Single Cell Experiment) objects</td>
<td><a href="https://files.slack.com/files-pri/T029RHR07-F024D7ELKC2/image_from_ios.jpg" class="uri">https://files.slack.com/files-pri/T029RHR07-F024D7ELKC2/image_from_ios.jpg</a></td>
</tr>
<tr class="odd">
<td>2021-09-09</td>
<td>syntax error in shell script</td>
<td>used back quote instead of single quote when assigning string to variables</td>
</tr>
<tr class="even">
<td>2021-09-08</td>
<td>CellProfiler H&amp;E segmentations</td>
<td>We decided to run CellProfiler on a while capture area image of DLPFC sample to look at the segmentations as it was doing good job on the cropped regions of a DLPFC sample</td>
</tr>
<tr class="odd">
<td>2021-09-07</td>
<td>Trouble with implementing RNAscope function for humans</td>
<td><a href="https://github.com/LieberInstitute/dotdotdot/issues/13" class="uri">https://github.com/LieberInstitute/dotdotdot/issues/13</a></td>
</tr>
<tr class="even">
<td>2021-09-02</td>
<td>Error running VistoSeg::spospotcheck (Error in spotspotcheck&gt;start (line 60))</td>
<td>The H&amp;E image that he was segmenting and the H&amp;E image for which he gets the ‘json’ and ‘csv’ are not the same images or at least not the same resolution which is causing the error.</td>
</tr>
<tr class="odd">
<td>2021-09-02</td>
<td>Error while running rnascope_mouse - line 159</td>
<td><a href="https://github.com/LieberInstitute/dotdotdot/issues/12" class="uri">https://github.com/LieberInstitute/dotdotdot/issues/12</a></td>
</tr>
<tr class="even">
<td>2021-08-30</td>
<td>dynamically name structures in matlab based on user inputs</td>
<td>used the command ‘eval’</td>
</tr>
<tr class="odd">
<td>2021-08-30</td>
<td>parameter tuning for dotdotdot to split the nuclei clusters</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-08-30</td>
<td>improve lipofuscin masking in dotdotdot</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-08-23</td>
<td>Explain how to merge 2 data tables in R</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-08-23</td>
<td>Explain public keys and the authorized_keys file on JHPCE, removed duplicate public keys for the local machine to resolve ssh key login</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-08-21</td>
<td>resolving issues with multiple keys in the authorized_keys file</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-08-18</td>
<td>explanation of the methods and algorithm of a paper we are reviewing together</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-08-11</td>
<td>steps to resolve image segmentation of LC heated samples round 2</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-07-20</td>
<td>Optimizing DAPI segmentation in dotdotdot</td>
<td><a href="https://github.com/LieberInstitute/dotdotdot/issues/11" class="uri">https://github.com/LieberInstitute/dotdotdot/issues/11</a></td>
</tr>
<tr class="odd">
<td>2021-07-15</td>
<td>Issues with SUFI</td>
<td>Explain how some scripts in SUFI run and the bash scripts to use in place of python scripts for batch processing</td>
</tr>
<tr class="even">
<td>2021-07-15</td>
<td>Explain how awk works</td>
<td>showed a example of how AWK extracts text lines from a text file</td>
</tr>
<tr class="odd">
<td>2021-07-14</td>
<td>Resolve errors produced by dotdotdot on SUFI output</td>
<td>Referred to Vijay’s notes</td>
</tr>
<tr class="even">
<td>2021-07-14</td>
<td>R Code to create list of files to be processed through VisiumLIBD</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-07-14</td>
<td>Setup aliases in bashrc</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-07-09</td>
<td>recreating spatstat plots with more features</td>
<td>used ggplot</td>
</tr>
<tr class="odd">
<td>2021-07-09</td>
<td>Helping understand errors from VisiumLIBD</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-07-09</td>
<td>Helping understand errors from VisiumLIBD</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-06-24</td>
<td>making a list of filenames in a specific folder and subfolder in R</td>
<td>used Sys.glob and write.table to write the list to txt file</td>
</tr>
<tr class="even">
<td>2021-06-24</td>
<td>Rectified issues for batch running emma’s RNAscope data</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-06-24</td>
<td>create shell script for batch running images on VisiumLIBD</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-06-01</td>
<td>troubleshoot SUFI output and figured it was not compatible with dotdotdot</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-04-15</td>
<td>running and trouble shooting VisiumIF image splitting code</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-04-09</td>
<td>Resolving issues with running dotdotdot on 20X RNAscope images</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2021-04-09</td>
<td>runnning Hiplex registration</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-04-01</td>
<td>creating logs for issues from running SUFI (spectral unmixing of fluorescent biological images)</td>
<td>explaination of nurolucida system, how number of cores and RAM matter for running SUFI</td>
</tr>
<tr class="odd">
<td>2021-03-30</td>
<td>steps to troubleshoot issues with stitched multi tile RNAscope images</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-03-24</td>
<td>running nucleiCount for spatial DLPFC samples</td>
<td>explaination of scale_factors.json, tissuepositions_list.csv from spaceranger and the spotspotcheck package</td>
</tr>
<tr class="odd">
<td>2021-03-23</td>
<td>extracting p-values from linear mixed effect model for Calcium imaging data</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2021-03-17</td>
<td>visium image segmentation</td>
<td>explaination of the functions VNS.m and refineVNS.m</td>
</tr>
<tr class="odd">
<td>2021-03-17</td>
<td>troubleshooting memory usage for VNS function from VisiumLIBD</td>
<td>when running VNS.m for several files on same matlab session the data from previous sessions is saved in matlab cache memory automatically. This will cause memory issues with subsequent runs, its important to clear variables after every run.</td>
</tr>
<tr class="even">
<td>2021-03-12</td>
<td>split histology images into respective capture areas</td>
<td>ran her through an example, explained her the code on github repo (split.m is the main script)</td>
</tr>
<tr class="odd">
<td>2021-03-12</td>
<td>x-quartz installation</td>
<td>edited the config file for setting up x-quartz</td>
</tr>
<tr class="even">
<td>2021-03-06</td>
<td>resolve figure generation error in human rnascope script from dotdotdot</td>
<td>fixed the github repo</td>
</tr>
<tr class="odd">
<td>2021-03-05</td>
<td>image analysis on QuPath</td>
<td>explore loading/saving/exporting options on Qupath</td>
</tr>
<tr class="even">
<td>2021-02-26</td>
<td>data transfer from LIBD server to JHPCE</td>
<td>use rsync</td>
</tr>
<tr class="odd">
<td>2021-02-26</td>
<td>Submitting/debugging bash job for dotdotdot on cluster</td>
<td>Debugging through the input/output file in logs</td>
</tr>
<tr class="even">
<td>2021-02-19</td>
<td>explaination of Owner/Group/Other permissions for files on JHPCe</td>
<td>running chmod</td>
</tr>
<tr class="odd">
<td>2021-02-12</td>
<td>download repository from github to cluster and run dotdotdot on cluster</td>
<td>Used Amy’s rstats blogpost on git</td>
</tr>
<tr class="even">
<td>2021-02-05</td>
<td>JHPCE ssh keypair set up</td>
<td>We got Heena’s MAC registered for passwordless login to JHPCE</td>
</tr>
<tr class="odd">
<td>2021-02-05</td>
<td>dotdotdot parameter tuning for Ema’s rat RNAscope data</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2020-12-03</td>
<td>run dotdotdot pipeline local computer and cluster</td>
<td>got through the Github vignettes thoroughly</td>
</tr>
<tr class="odd">
<td>2020-12-03</td>
<td>how to open matlab (GUI and command line) on JHPCE</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2020-12-01</td>
<td>how to connect to LIBD servers on windows and mac</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2020-11-19</td>
<td>data transfer from LIBD server to JHPCE</td>
<td>to use rsync or scp, you need to have user account created on LIBD server (Bill can help create it)</td>
</tr>
<tr class="even">
<td>2020-11-19</td>
<td>download matlab on local computer through JHU software portal</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2020-11-04</td>
<td>3D volume extraction in Matlab</td>
<td><a href="https://www.mathworks.com/matlabcentral/answers/636365-how-can-i-measure-volume-from-a-3d-image?s_tid=srchtitle" class="uri">https://www.mathworks.com/matlabcentral/answers/636365-how-can-i-measure-volume-from-a-3d-image?s_tid=srchtitle</a></td>
</tr>
<tr class="even">
<td>2020-10-28</td>
<td>customize dotdotdot to his data</td>
<td>NA</td>
</tr>
</tbody>
</table>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="louise.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="nick.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
