<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Nick | Data Science guidance session logs</title>
  <meta name="description" content="6 Nick | Data Science guidance session logs" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Nick | Data Science guidance session logs" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Nick | Data Science guidance session logs" />
  
  
  

<meta name="author" content="Leonardo Collado-Torres" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="maddy.html"/>

<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#dsgs-guide-team-members"><i class="fa fa-check"></i>DSgs-guide team members</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#graphs"><i class="fa fa-check"></i>Graphs</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#r-session-information"><i class="fa fa-check"></i>R session information</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="arta.html"><a href="arta.html"><i class="fa fa-check"></i><b>1</b> Arta</a></li>
<li class="chapter" data-level="2" data-path="josh.html"><a href="josh.html"><i class="fa fa-check"></i><b>2</b> Josh</a></li>
<li class="chapter" data-level="3" data-path="leo.html"><a href="leo.html"><i class="fa fa-check"></i><b>3</b> Leo</a></li>
<li class="chapter" data-level="4" data-path="louise.html"><a href="louise.html"><i class="fa fa-check"></i><b>4</b> Louise</a></li>
<li class="chapter" data-level="5" data-path="maddy.html"><a href="maddy.html"><i class="fa fa-check"></i><b>5</b> Maddy</a></li>
<li class="chapter" data-level="6" data-path="nick.html"><a href="nick.html"><i class="fa fa-check"></i><b>6</b> Nick</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data Science guidance session logs</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="nick" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Nick</h1>
<table>
<colgroup>
<col width="19%" />
<col width="57%" />
<col width="23%" />
</colgroup>
<thead>
<tr class="header">
<th>Date</th>
<th>Help request</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>2022-02-07</td>
<td>Debugging seg fault with tensorqtl module</td>
<td>We tried to discover the source of a seg fault that would occur when she imported tensorqtl in python with the tensorqtl module. Another user and I weren’t able to replicate the issue. We later discovered that there were lines in her .bashrc that might’ve interfered with the module to produce the issue.</td>
</tr>
<tr class="even">
<td>2022-02-04</td>
<td>Using tensorqtl with a GPU</td>
<td>I described how to qrsh/sub jobs with the GPU node, including how to automatically set up ‘CUDA_VISIBLE_DEVICES’ to avoid contention for resources, and how to use the tensorqtl module to ensure GPUs were used when importing pytorch</td>
</tr>
<tr class="odd">
<td>2022-02-04</td>
<td>X11 and SSH issues</td>
<td>After ensuring our apparent Rstudio set-ups on our windows machines matched, she still wasn’t able to forward X11 while using Rstudio. Regarding a different issue, we discovered that a particular machine required additional lines in the ‘~/.ssh/config’ to properly enable RSA keys.</td>
</tr>
<tr class="even">
<td>2022-01-21</td>
<td>accessing the ‘gpu’ node at JHPCE correctly</td>
<td>I described how it’s accessed as a queue, but some manual work is left to the user (checking which GPUs are open and setting ‘CUDA_VISIBLE_DEVICES’)</td>
</tr>
<tr class="odd">
<td>2021-12-15</td>
<td>creating a ‘samples.manifest’ and choosing trimming settings</td>
<td>I described how I normally deduce sample IDs from FASTQ filenames when creating a ‘samples.manifest’ file for SPEAQeasy, described the difference between ‘–trim_mode’ settings that SPEAQeasy supports, and showed how to check what samples were trimmed.</td>
</tr>
<tr class="even">
<td>2021-11-17</td>
<td>using ‘list.files’ in R</td>
<td>I described using regular expressions in R to match files in a directory. We encountered a weird case where ‘list.files’ seemed to return different numbers of matched files when run repeatedly. After ensuring we were in the right directory, the issue went away, but it was unclear what caused the original problem.</td>
</tr>
<tr class="odd">
<td>2021-10-06</td>
<td>help with git</td>
<td>I explained how I get out of the situation where files that are too big get accidentally committed (using ‘git log’, ‘git reset –soft’, and other details)</td>
</tr>
<tr class="even">
<td>2021-10-04</td>
<td>debugging SPEAQeasy errors</td>
<td>We worked through an error message from the ‘VariantsMerge’ process, uncovering (arguably) a bug in SPEAQeasy, and I advised how to temporarily circumvent the problem. I also suggested a command for renaming a set of files that initially broke SPEAQeasy’s assumptions.</td>
</tr>
<tr class="odd">
<td>2021-09-22</td>
<td>help with ‘umask’ and git</td>
<td>I talked about how Linux does not allow creation of files w/ execution permissions set, regardless of ‘umask’ settings such as in the ‘.bashrc’. I also explained how I use branching with git and other tips for managing a repo that others also develop on.</td>
</tr>
<tr class="even">
<td>2021-09-10</td>
<td>merging FASTQ files automatically via SPEAQeasy</td>
<td>We downloaded a metadata table from SRA and parsed it through R to programatically create a “samples.manifest” file for use in SPEAQeasy, that merged samples that were each split into multiple FASTQ files.</td>
</tr>
<tr class="odd">
<td>2021-09-03</td>
<td>identifying SPEAQeasy error</td>
<td>SPEAQeasy would appear to run into “file not found” errors for just a handful of random samples in a large dataset. I eventually traced the error to a disk mounting issue on a particular compute node, which was resolved through bitsupport.</td>
</tr>
<tr class="even">
<td>2021-08-26</td>
<td>resuming SPEAQeasy after an issue in the counts step</td>
<td>I mentioned that she encountered a bug that was fixed in a more recent version of SPEAQeasy, but suggested she manually add a line of code temporarily so that she could resume the run before updating</td>
</tr>
<tr class="odd">
<td>2021-08-25</td>
<td>downloading FASTQ files from SRA</td>
<td>I referred her to a script I had written in the past that contains the required commands and configuration (with comments) for using the SRA toolkit</td>
</tr>
<tr class="even">
<td>2021-08-06</td>
<td>Forming a data frame from data in multiple files</td>
<td>I showed a fairly standard method for concatenating tabular data from multiple files into a single matrix, and saving the resulting R object</td>
</tr>
<tr class="odd">
<td>2021-07-14</td>
<td>getting SPEAQeasy running</td>
<td>In a series of email exchanges we diagnosed subtle errors resulting from bad FASTQ files and unintended options in the main script. Her feedback helped me improve some existing documentation.</td>
</tr>
<tr class="even">
<td>2021-07-14</td>
<td>debugging SPEAQeasy errors</td>
<td>I described correct use of the “–input” argument, and we also found that many of the input FASTQs were empty, leading to a somewhat vague error message.</td>
</tr>
<tr class="odd">
<td>2021-07-13</td>
<td>formatting a LaTeX document and bookdown website</td>
<td>I attempted some formatting tricks to eliminate unwanted whitespace but was ultimately unsuccessful. We suspected the LaTeX template required the whitespace to exist. We also tried to fix a bug with the header of a bookdown RMD not rendering as expected. It seemed that re-writing the header (without making any visible changes) fixed the issue, though we weren’t sure of the source of the initial problem.</td>
</tr>
<tr class="even">
<td>2021-07-02</td>
<td>Help indexing a possibly corrupt VCF file</td>
<td>We confirmed that an existing VCF was corrupt and found that tabix expected bgzip-compressed VCF input rather than compressed BCF input.</td>
</tr>
<tr class="odd">
<td>2021-06-30</td>
<td>Specifying output logs on SGE</td>
<td>I briefly mentioned the “#$ -o” and “#$ -e” arguments, and how several logs will generate in your home directory without those arguments and without “#$ -cwd”, when “qsub”ing a job at JHPCE.</td>
</tr>
<tr class="even">
<td>2021-06-16</td>
<td>addressing SPEAQeasy error</td>
<td>we talked about how to use the “–strand” option, the source of the error that was encountered</td>
</tr>
<tr class="odd">
<td>2021-05-19</td>
<td>Making a module</td>
<td>We went through the creation of an LIBD JHPCE module, covering the documentation process, lua syntax, and how to test a module before pushing.</td>
</tr>
<tr class="even">
<td>2021-05-19</td>
<td>Configuring a parallel series of bash scripts</td>
<td>We adapted a series of bash scripts requiring parallel functionality for use at JHPCE. This involved correctly setting some path variables and requesting multiple cores through SGE for some scripts.</td>
</tr>
<tr class="odd">
<td>2021-05-13</td>
<td>Setting up a SPEAQeasy run</td>
<td>I described how I typically construct a “samples.manifest” file and configure SPEAQeasy for a particular experiment.</td>
</tr>
<tr class="even">
<td>2021-05-03</td>
<td>Running SPEAQeasy- follow up</td>
<td>We observed an issue where log parsing in SPEAQeasy behaved differently for Sreya and I due to a difference in default python version loaded at JHPCE. I fixed the hisat2/2.2.1 module to ensure consistent python version use.</td>
</tr>
<tr class="odd">
<td>2021-04-28</td>
<td>Constructing an array job from an existing sequential shell script</td>
<td>I showed how I typically produce an array job shell script for the situation where I have many samples of data to process the same way (SIMD). We talked about using a text file where each line contains one sample name, and referencing that in the array script.</td>
</tr>
<tr class="even">
<td>2021-04-26</td>
<td>Running SPEAQeasy- follow up</td>
<td>We discussed how to handle strandness issues, and how different inferred strand patterns within a group of samples sequenced together can indicate problems with the input data. I described how to use the ‘–force_strand’ option with SPEAQeasy</td>
</tr>
<tr class="odd">
<td>2021-04-22</td>
<td>Running SPEAQeasy</td>
<td>We went through cloning SPEAQeasy, creating a “samples.manifest” file with R, and other relevant details for running SPEAQeasy at JHPCE. I also helped Henry set up git at JHPCE</td>
</tr>
<tr class="even">
<td>2021-04-22</td>
<td>Running SPEAQeasy- follow up</td>
<td>We troubleshooted strange errors from the main SPEAQeasy output, and saw that I was able to run the main script (“run_pipeline_jhpce.sh”) but Sreya encountered errors with an identical script. We suspected permissions issues in her working directory, which she emailed bitsupport about.</td>
</tr>
<tr class="odd">
<td>2021-04-19</td>
<td>Running SPEAQeasy</td>
<td>Although we encountered permissions issues, I showed how to create a “samples.manifest” file and configure SPEAQeasy for a particular set of samples.</td>
</tr>
<tr class="even">
<td>2021-04-12</td>
<td>Running the old RNA-seq pipeline on some SRA data</td>
<td>I showed my typical process for creating a “samples.manifest” file for a given dataset, and how to submit a pipeline run. We continued to debug some minor issues over slack</td>
</tr>
<tr class="odd">
<td>2021-04-07</td>
<td>Nextflow overview for work with imputation pipeline</td>
<td>I covered various nextflow concepts, as well as my typical approach in creating and running a nextflow project at JHPCE</td>
</tr>
<tr class="even">
<td>2021-03-22</td>
<td>Using an LIBD module</td>
<td>I described the need for the “module use” command and modifying the .bashrc to easily use LIBD modules</td>
</tr>
<tr class="odd">
<td>2021-03-18</td>
<td>Download SRA data and run SPEAQeasy</td>
<td>I fetched the SRA files, created a manifest and execution script for SPEAQeasy, but had an issue running “fastq-dump”. I decided to hand off the request, partially complete.</td>
</tr>
<tr class="even">
<td>2021-03-10</td>
<td>qSVA explanation and code review</td>
<td>Josh walked me through some code performing qSVA for a recent project, and I offered some tips involving code organization/ readibility.</td>
</tr>
<tr class="odd">
<td>2021-03-03</td>
<td>Advice for writing papers</td>
<td>We discussed how Sciwheel can be adapted for different journals, and provided general advice for getting started and formatting a paper for a particular journal.</td>
</tr>
<tr class="even">
<td>2021-03-02</td>
<td>Clarification about how recent CMC data was processed with SPEAQeasy</td>
<td>I explained that SPEAQeasy was used for the recent hg38 processing, and an older version of a similar pipeline was used for the original hg19 processing. We discussed our new method for retrieving gene symbols (“org.*.eg.db” packages) and some general code advice.</td>
</tr>
<tr class="odd">
<td>2021-03-02</td>
<td>Our group’s set-up and approach for writing papers with google docs, and managing citations</td>
<td>I went through the setup and example usage of Sciwheel and cross reference, and gave general advice (it’s often easiest to start with the “Methods” section, etc)</td>
</tr>
<tr class="even">
<td>2021-02-25</td>
<td>I need help in: downloading data from SRA to JHPCE, running SPEAQeasy to create rse object (one session will likely not be enough).</td>
<td>I explained how to use SPEAQeasy at JHPCE, and we tried an experimental method for downloading SRA data. The method didn’t work, and we plan to meet again after I verify a more typical method works.</td>
</tr>
<tr class="odd">
<td>2021-02-23</td>
<td>I need help with setting up Mobaexterm. Unable to open new terminals and folder permission issues</td>
<td>I hadn’t encountered the issue with new local terminals failing to open. I planned to follow up with better specific advice via email, and offer to explain this advice in a future DSGS.</td>
</tr>
<tr class="even">
<td>2021-01-06</td>
<td>Correlation values from labeledHeatmap</td>
<td><a href="https://support.bioconductor.org/p/p133591/#p133927">BioC Support website answer</a></td>
</tr>
<tr class="odd">
<td>2020-12-09</td>
<td>Help with importing HDF5 file</td>
<td>Used ‘h5ls’ command from shell to verify contents of file; eventually found we were looking at the wrong file (error message was cryptic about issue). <a href="https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1607034309074800" class="uri">https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1607034309074800</a></td>
</tr>
<tr class="even">
<td>2020-12-02</td>
<td>Matching Variants Between Two VCF Objects</td>
<td><a href="https://support.bioconductor.org/p/p133110/#p133319">BioC Support website answer</a></td>
</tr>
<tr class="odd">
<td>2020-11-20</td>
<td>ChipQC ‘names’ attribute [9] must be the same length as the vector [2]’</td>
<td><a href="https://support.bioconductor.org/p/134569/#p133028">BioC Support website answer</a></td>
</tr>
<tr class="even">
<td>2020-11-11</td>
<td>How does one modify a single value of a field in a seqinfo object?</td>
<td><a href="https://support.bioconductor.org/p/134139/#p132829">BioC Support website answer</a></td>
</tr>
<tr class="odd">
<td>2020-11-04</td>
<td>Extracting Sequences from Fasta Files with Rsamtools</td>
<td><a href="https://support.bioconductor.org/p/131600/#p132617">BioC Support website answer</a></td>
</tr>
<tr class="even">
<td>2020-11-02</td>
<td>Help with SpaceRanger</td>
<td>NA</td>
</tr>
<tr class="odd">
<td>2020-10-28</td>
<td>Changing Row Names of MultiAssayExperiment Assay</td>
<td><a href="https://support.bioconductor.org/p/p132406/#p132459">BioC Support website answer</a></td>
</tr>
<tr class="even">
<td>2020-10-28</td>
<td>issue running spaceranger</td>
<td>initial issue seemed to do with the spaceranger command arguments; we are continuining to investigate possible module issues or insufficient code in the <a href="http://research.libd.org/rstatsclub/2020/10/20/using-space-ranger-at-jhpce/">blog post</a></td>
</tr>
<tr class="odd">
<td>2020-10-26</td>
<td>Help running speakeasy on MARCC and JHPCE please</td>
<td>NA</td>
</tr>
<tr class="even">
<td>2020-10-21</td>
<td>Question: Cannot Install gmapR Bioconductor Package</td>
<td><a href="https://support.bioconductor.org/p/131285/#134921">BioC Support website answer</a></td>
</tr>
</tbody>
</table>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="maddy.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
