# Leo

| Date | Help request | Notes |
| ---- | ------------ | ----- |
| 2020-10-21 | Ballgown stattest | [BioC Support website answer](https://support.bioconductor.org/p/134914/#134915) |
| 2020-10-28 | exceRpt pipeline counts | [BioC Support website answer](https://support.bioconductor.org/p/p132450/#p132464) |
| 2020-10-30 | recount3 GTEx data access | [BioC Support website answer](https://support.bioconductor.org/p/p132487/#p132521) |
| 2020-10-30 | Issue with JHPCE file permissions | Explained the `chgrp` and `umask` commands plus `~/.bashrc` file configuration. I also mentioned why we don't use `chmod 777` at JHPCE. Details on [Slack](https://jhu-genomics.slack.com/archives/CR9NYA0BF/p1604086263148400?thread_ts=1603826219.094600&cid=CR9NYA0BF) (private channel). |
| 2020-11-03 | Issue with knitcitations | [GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/14) |
| 2020-11-03 | Fixed a bug on biocthis for release branches | [GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/13) |
| 2020-11-03 | Fixed two bugs on recount3 related to locate_url() for the BigWig files. | Resulted in recount3 versions 1.1.1 and 1.1.2 on bioc-devel (and 1.0.2 on bioc-release) |
| 2020-11-11 | GitHub actions workflow issue with macOS on R-devel | [GitHub issue at biocthis](https://github.com/lcolladotor/biocthis/issues/15) |
| 2020-11-11 | Differences between recount2's "bp_length" and SPEAQeasy's "Length" | [Check Slack helpdesk channel](https://jhu-genomics.slack.com/archives/CUYHWV7AA/p1605123296024100) |
| 2020-11-25 | Accesing CMC RNA-seq data | Started PsychENCODE access request process |
| 2021-01-11 | Questions related to accessing results from the DG paper by Andrew Jaffe | NA |
| 2021-01-14 | biocthis macOS GHA bug | [BioC Support website answer](https://support.bioconductor.org/p/p134069/#p134141) |
| 2021-01-26 | Guidance on how to download data from a study in GEO | NA |
| 2021-01-29 | Help determining what data is available and what can be used for an analysis | Use of galaxybrain for identifying samples with DNAm, DNA genotype and RNA-seq data |
| 2021-02-05 | BSP2 LIBD eQTLs request | NA |
| 2021-03-10 | How to write a small methods paper | Created GitHub issues at https://github.com/LieberInstitute/VisiumImgProcessing/issues |
| 2021-03-11 | BSP2 eQTLs with a subset of SNPs | Adapted code and shared it at https://github.com/LieberInstitute/brainseq_phase2/tree/master/eQTL_full/subset_hancock |
| 2021-03-24 | Download SRA data and run SPEAQeasy | I identified that https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE154829 contains both miRNA and RNA-seq data. Using data from the SRA Run selector at https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP273137&o=acc_s%3Aa I was able to filter the data to the RNA-seq samples and subset the input samples.manifest file for SPEAQeasy. |
| 2021-03-24 | Explain the feasibility of an on-demand end user interface for DEG and eQTLs | To identify DEGs or eQTLs, the user has to choose what covariates to adjust for. Some studies might require more complicated models than others, as they might be from more than one brain region or present some batch effect. In addition, in newer projects we are considering adjusting for cell type fractions derived from deconvolution. Even if all the covariates were fixed across datasets, in DEGs we have typically adjusted for qSVs which are computed in a dataset-dependent fashion and in essence are PCs derived from degradation-associated genomic regions. Similarly, for eQTLs we adjust for expression PCs. In both we adjust for ancestry PCs. Computing PCs is input-dependent: you won't get the same values or PCs if you change which samples you are using. So, computing DEGs and eQTLs is not as simple as say a map-reduce problem where you compute the same thing for every input sample, then aggregate the per-sample results. We have steps that depend on the whole sample set. Then there's also steps for filtering expression features based on some minimum expression cutoffs. Plus, differences between say polyA and RiboZero data, which don't make sense to process together. For these reasons, I don't think that an on-demand computation pipeline is feasible based on a user selecting a set of input samples. Constructing RNA-seq objects (RangedSummarizedExperiment objects) as well as DNA genotyping VCF files based on a set of input samples is feasible and that's what others are building. After that, I think that it's best to adapt code we have (or even use newer tools) after carefully exploring the data. |
| 2021-03-25 | Creating a documentation website using bookdown | Help with understanding [bookdown](https://bookdown.org/yihui/bookdown/) and ideas on how to organize a documentation website (chapters and information to consider). |
| 2021-03-25 | Clarification of LIBD support | Discussed the research project proposal and how we can help. |
| 2021-03-25 | General advice | General advice. Received feedback regarding file backups. |
| 2021-03-29 | spatialLIBD support | Discussion about features that spatialLIBD could support to make it easier to use for newer data. |
| 2021-04-08 | General guidance on how to learn how to do differential expression analyses | Created a learning plan involving several team members that is now documented at a channel in JHU Genomics Slack. That way everyone can keep track of what others are teaching the person who asked the question, and we can have a common communication channel among everyone involved. Sessions include learning how to interact with JHPCE through RStudio, how to run SPEAQeasy, key R/Bioconductor packages (limma, SummarizedExperiment), and explore possible visualizations of interest. |

