# Louise

| Date | Help request | Notes |
| ---- | ------------ | ----- |
| 2022-10-27 | I have this single nucleus RNA sequencing experiment with the mouse LS that Matt analyzed and we have a final single cell experiment, but I don't know how to do anything with this data set and want to learn how to plot heatmaps and possibly other things for this data set. Can we maybe schedule an hour to start this learning process and then I can schedule future time as needed? | Lionel walked me through some plots they need for the LS paper: marker genes heatmap and expression violin plots. I pointed him to some code from DLPFC sn that should be helpful (he needs access to this repo). He also needs to get setup with JHPCE on his new laptop, I pointed him to the team laptop setup docs. He also is new to ggplot, so I pointed him to the visulization chapeter of R for data scientist. This will probaly require some follow up meetings |
| 2022-10-26 | Check the DEG heatmaps | Daianna has added a new color scheme to the heatmaps that makes the differnces between positive and negative more clear, how ever I think the darkest red being close to 0 is misleading, so shes going to fix that. I also recomended she simplify the annotaions on the combined heatmap so its easier to find and interpret patterns. |
| 2022-10-17 | Review TMM paper | We reviewed the reasons or correcting counts and why thats important for DE analysis. We went over the math behind TMM, and then how thoes ideas are used in edgeR. I pointed Bukola to a stats quest video with a more detatiled explination of how the edgeR correction works. |
| 2022-10-14 | csvs that I want to merge together | Sang Ho showed me the csv files that he exported from different moduled on HALO. They seem to have the same rows, so copying the needed column in to a differnt csv is easy in excel (where Sang Ho is working) but I showed him how to check that things are really aligned with some excel functions. We also looked at the data in prism, which I had to quickly learned about. I advised that he should show the data in boxplots not as a barplot to better display that data, and went over the checks for running an anova. |
| 2022-10-11 | GO and KEGG boxplots with the modifications you suggested | Daianna showed me her dot plots with the correct 0.05 q val filter, they looked good, ready for discussion. One tweak , is hshe had a combined up + down expresion col for some plots which was redundant and should be removed. She had also made some heatmaps that didn;t show very much difference between case control, htey probaly need to be normalized by the genes overall expression, but I need to check prevous work to remeber how to best do that |
| 2022-10-10 | QC approach for hebenula | We went throught boxplots Bukola had made for the data, I suggested she add geom_jitter and how to make that work with geom_repel to help show more details about the data. We also discussed direct vs. standard evaluation of variable and how that makes writing funcitons with tidyverse tricky, but how she can use aes_() to do stadard eval with ggplot |
| 2022-10-05 | How to check technical effects for RNAseq data | I walked Bukola through our process for buld data QC, which involves making many exploratory plots of QC metrics and know groups (like batch, or case/control) to understand patterns and identify problematics samples. Then later implementing cutoff values either based on stats (like a 3 MAD cut off) or knowlege of ressonable values. Bukola will spend some time plotting QC values, adapting code from smoking mouse, then we will meet again to interpret these plots. |
| 2022-10-05 | Reviewing GO results and dotplots | Daianna and I went over the dot plots she made for GO analysis from the multiple DE test she ran. Her plots look good, but we identified that she needed to use the default q cutoff to get useful results. She will rerun the GO analysis and replot the results. |
| 2022-09-30 | Question on the feasibility of some approach with HALO (for the AD project) | Sang Ho walked me through an analysis they want to do for the AD project, it involves annotating HALO data. We talked thought a strategy, this may be a bit computationally tricky to implement but it seems doable. |
| 2022-09-30 | Working on making a violin plot for just one gene. So by brain region rather than gene type | We adapted my_plotExpression() (a function I wrote) to work for Bukola's data. This required some debugging as subsetting to just the one gene in question altered how some internal dataframes looked. I also gave Bukola some pointers about saving plots using ggplot. |
| 2022-08-22 | DE analysis introduction | I went over the DE pipeline we use, noting that the model development is the most demanding and non-static part. We talked specifics for their dataset which has some experimental design to consider as its organids and is diffrent from patient case control. I posinted Geo to scripts where we do QC, expression cutoff, and my run_DE script |
| 2022-08-18 | Git push help | Heena saw old commits pop up on slacks git notifications when she pushed commits she had locally on her laptop, but they didn't reappear in the git history. I think she did everything right and there might be something strange with the behavior of the slack message (as thats the only place things seemd wrong). We did some detective work between the files and the commits, everything seems good. Heena was able to then pull to the cluster with out conflicts |
| 2022-08-17 | check the models and the results I have obtained so far from the DEA | Had to work over slack again as Daianna was having trouble with zoom. Working with the feedback from the previous meetings, the experimetn has been narrowed down to 5 DE runs, and then comparing some pairs of the outputs. We were unsure if we should include mitoRate as a variable in the adult model as it correlates highly with totalAssignedGene, but decided to include it for now to match the model used in the pup DEA. I also sent her some code I have used to make scatter plots of t-stats to help compare DE results. |
| 2022-08-10 | Check the model design of the smoking mouse project | We intended to meet on zoom, but Daianna's connection was bad so we used Slack to chat. Daianna has made many plots and visuals about the many DE analysis for smoking mouse. I gave her some feedback about orgaizing her slides so the figures can practically be used inform DE models, right now everything is a bit scattered as shes planning on running 7 analysis so there is a lot to consider. We will need to meet again to finish this. |
| 2022-08-03 | Review results of Varaince Parition for smoking mouse | Daianna showed me her varianceParition plots, and we talked bout how to design the DE anlaysis based on what she's oberved in the mouse data. I advised to keep things simpler (ie not create too many groups) but rather use models to control for vaiables adding variance. We will meet next to talk model design for this work. |
| 2022-07-15 | QC update for smoking mouse | Daianna shared her updated QC plots, adn the one sample she planned to drop based on fastCellQC. I advised that she dig deeper in to how that QC finstion works, and what the cutoff values are, as well as exmaming mito vs. ribo precentages, and a two samples that don't seem to group with others of the same phenotype / |
| 2022-07-07 | Subsetting eQTL results with in risk loci | We discussed a strategy to subset the 102 loci in question, and adapted code to subset the FDR signif pairs to work with the location info. |
| 2022-07-07 | help row annotation colors of a pheatmap | I explained how to get pheatmap to use the same color scheme for column and row annotations by making the annotaion tables have the same colname, and suppying a color pallet |
| 2022-07-05 | QC update for smoking mouse | Daianna shared the updates she made to the QC plots based on our previous discussion, specifically separating brain and blood samples. |
| 2022-06-28 | Review progress on smoking mouse DE project | Daianna walked me through her progress in the DE analysis. We went over QC plots and discussed experimental design. She will check in with Kristen about how to organize the brain vs. blood samples. Currently she dropped the blood sampeles as they looked very different in several QC metrics. But I advised that they just be handled seperatly as it makes senese they look different as they are different tissue, and the signal is probaly biological rather than technical |
| 2022-06-17 | JHPCE Github setup | We fixed Sanjana's ssh set up and got the qrsh command working. We got git configurated for her locally and on JHPCE, as well as set up tokens. Created a test repo, and made commits from her laptop and JHPCE as an example |
| 2022-06-15 | Revisited the sc analysis and could use some help | We checked Sreya's code against the OSCA DE example run. She was worried that there weren't may significant genes, but her results seemed to be in the same range (10s of genes) as the example. We discussed how maybe there were QC or demographic variables that could have been added to the model, but the data she has doesn't have much to work with. We will revisit when I have maybe done more work with sn DE analysis. |
| 2022-06-15 | Github setup | We went over github basics and motivation behind version control. Quickly discovered that Sanjana's qrsh command is not working, requiring a password. Ended up using the session to troubleshoot that, with no success :( planning on circling back/re-setting up ssh for her. Will meet later about git setup |
| 2022-06-03 | I might need a review of how to go about selecting marker genes for the spatialDLPFC data. Also, I'm curious what code you have for making heatmaps showing marker gene expression across cell types. | I pointed Nick to a few scripts used in marker finding for the Tran Maynard dataset, using a few functions from DeconvoBuddies. For marker heat maps we went over how I hace built annotation tables, and a few tricks for optimizing pheatmaps for maker plots. |
| 2022-06-02 | Single cell rna seq analysis | We converted a downloaded dataset from tsv files to a sce object. Went over a strategy for performing DE on single cell data, using OSCA as a reference |
| 2022-06-01 | I have a mouse single-cell RNAseq data that I am aiming to identify cell types in different clusters. | We spent the session getting Sanjana set up better to work on JHPCE, setting up ssh keys, cyberduck, RStudio, and config files. Plan to meet again to set up git + start single cell work |
| 2022-05-18 | Color bar for heatmap | We both leaned how to use the image function to create a annotation bar for heatmaps based on a custom color pallet |
| 2022-05-09 | Cross Reference help | We went over how to create a new type of refrence (Supp Table/Supp Figure). Also walked though the full process of creating a label and then refrencing it in from the text + considreations for formatting |
| 2022-05-05 | Need some help to make a plot | We went thourgh some plotting code Sreya had adapted to make a manhattan plot, and went over which aestetics to change to get the desired plot |
| 2022-04-27 | Help with purrr loops for WGCNA funtions | I helped come up with a stratagedy to loop over 100+ files for an analysis and exreact a portion of the results in to one R object |
| 2022-04-07 | writing papers in google docs and setting up links for figures | We installed and configured Cross Refrence, then went over how to use the lable vs. refrence links + pro tips on how to avoid errors |
| 2022-04-06 | I'd like to consult with Louise about how to make a spatial plot in R using a data set from the HALO FISH-IF module. | We want over some plotting basics with ggplot2, and how to use geom_rect to recreate some spatial plots that a halo module produces in R. |
| 2022-03-24 | Advice on how to increase code coverate with testthat | We applied some test I used for the TREG package to help test more parts of the functions, like checking for errors w/ bad input, warning messages, and qualities of outputs. We discussed possible ways to use testthat on a ggplot, but that might need more thought. |
| 2022-03-15 | TODO | TODO |
| 2022-03-04 | Understand the covariates used in scRNA-seq cell type analysis for a project we are doing with striatal organoids | We went over the files (data and script) for the deconvolution analysis done for the habenula in BSPIII. We also discussed the theroy/methodology for deconvolution. |
| 2022-02-24 | I would like to learn how you ran tensorQTL on the MDD data, maybe with using the GPU nodes for it. | Geo was interested int eh workflow I developed for tensorQTL. I went over the requirements and input formats that tensorqtl requirers, and how I had previously made them from our dataset. Also discussed the differnces between the functions and their outputs. |
| 2022-02-24 | Help building for loops to analyze WGCNA ouput data | We discussed how to build and save data with a for loop and with purrr::map. Went over how to define a function in map to do nested loops |
| 2022-02-15 | Access to the gene info in the rowData of the Summarized Experiment objects to annotate eqtl results | We went over how to acess the rowData, and what the rowData inthe MDD project contains. Ended up adding columns to the eqtl risk snp subsets to help with analysis |
| 2022-01-20 | Troubleshooting ExprssionSet conversion | Fixed row names to both be ensembl IDs, discussed simplifing cell types in sce, converting sce arrarys to matrix |
| 2022-01-19 | Help using BisqueRNA, getting snRNA-seq into correct input format, chosing refrence vs. marker based deconvolution | I shared some detials of our deconvolution approach, we walked through the SCE to Expression set converson, and discussed some issues that may come up |
| 2021-11-03 | Debugging help with rse_tx object | We discussed how to check for duplicate rownames, and stratedgies for variable naming + working with rse objects |
| 2021-10-29 | qc general scripts, rnaseq scripts general, ggplot histogram with breaks | We found a R function that helped produce the plot she was looking for |
| 2021-08-26 | using map to cbind a list of six to its corresponding list of lists. The second is a list of 6, each a list of 3. | Did some troubleshooting on nested map functions, when to use ~.x and when to use function() syntax |
| 2021-08-23 | Using purrr and map to loop over datasets and apply qsva | Went over basic, complex, and nested map() usage and how to apply it to Josh's current work |
| 2021-08-20 | Help with JHPCE login issues | We were able to update the keys for the cluster, but the ssh key login was still broken |
| 2021-06-30 | Debugging ggplot code | We reviewed her code and I gave some pointer on what bugs to look out for with certain errors in ggplot |
| 2021-06-17 | Manhattan plot making and tstatistic | Pointed her to some oacjages for creating TWAS plots whe was interested in and we walked through some code I used to compare T-stats from different DE models |
| 2021-06-03 | questions about expression_cut and pca plots | We walked tought script from getRPKM to expression cut to filter rse objects, I also pointed Brenda towards some reasources to get started with ggplot |
| 2021-05-28 | Differential Expression help (continuation of 5/20) | NA |
| 2021-05-20 | Making plots for DE studies | We reviewed the model that Sreya was evaluating for DE. As we as troublshooting the plot functions from the SPEAQeasy bootcamp code. I also advised taht she could create a volcano plot for her results. |
| 2021-05-12 | Making plots for DE studies | We went over some plots we have made for DE previously, (quality control & PC scatter plots, t-stat scatter between models, venn diagrams, and boxplots). Talked about some things to watch out for in her data (small samples size). We are planning to meet again when her data is ready. |
| 2021-04-19 | JHPCE set up on mac | We got ssh key login working, added config files to JHPCE, went though a "hello world" example of using files on JHPCE on RStudio |
| 2021-04-15 | JHPCE set up on mac | Henry needed to reset his JHPCE password as he had forgotten it, but we went through RStudio set up and got cyberduck installed, we will set up a meeting next week for ssh keys and configuration |
| 2021-03-26 | Need help understanding the sce - single cell object | Went over basic functions of SummarizedExperiment and SingleCellExperiment, what a Sparse matrix is - how to convert to non-sparse. Steps to take to use getRPKM on SingeCellExperient. |
| 2021-03-19 | Change point size in vis_clust_grid | We walked througth the SpatialLIBD code to find where the argument has to be changed |
| 2021-03-05 | Maker finding for cell types in single cell data | I showed them the tools I developed in DeconvoBuddies pacakge, and provided links to example code where I had applied these funcitons in previous work |
| 2021-02-09 | Visualizing RPKM valus for genes in Humna and Primarte data | We spent time doing basic Rstudio setup and troubleshooting, and got the data downloaded |
| 2021-02-04 | Work on habenula deconvolution and debug the deconvolution package | We went over applying DeconvoBuddies functions to hebenula data |
| 2021-02-04 | Github ssh key set up | We got the ssh keys working for Brenda's github |
| 2021-01-27 | group_by bug for get_mean_ratio | We are going to try an iterative strategy to keep tables smaller |
| 2020-12-08 | Resolving issues in phenotype data between genotype, expression, and LIBDpheno | Need to resolve swap bugs. Plan to create a table of phenotype data (Dx, age, sex, race) for each BrNum to mach genotype data |
| 2020-12-07 | jhpce set up on mac | Went through the setting up passwordless login for jhpce using ssh keys, and coppied the configuration files from https://gist.github.com/lcolladotor/3b1063db30a206835ee7e086c8f037ee |
| 2020-11-18 | Help clustering sce data | We debugged some code and ultimetly decided that it might be best for Abby to write new code and use the tsne function to cluster and plot the data |
| 2020-11-13 | Using reshape package to transform table for plotting | I recomended using dcast to do a "long" to "wide" data transformation |
| 2020-11-12 | Help adapting ggplot script for TWAS Z-score | We talked about how the data should be structured before using ggplot (transforming some data and adding catagorical columns) |
| 2020-11-05 | LC repo organization | We discussed organizing by analysis (a directory for each one, then prehaps sub- directory for data & plots). Create a readMe for each sub-folder. Use gitignore to keep repo from getting too big. |
| 2020-10-27 | Plotly help if possible? I'm trying to export an html file that allows the user to look at each snp/gene on the plot, and I am almost successful. Leo recommended that I talk to you on this matter. | Needed to use highlight_key and highlight functions to select points between plots. Used rstats example as guide https://gist.github.com/lcolladotor/350ac8843e153d135f64880a1d029b14 |
| 2020-10-21 | Question: Subset center regions of a set of different length sequences | [BioC Support website answer](https://support.bioconductor.org/p/134532/#134925) |
| 2020-10-20 | Help with spatstat | spatstat package https://spatstat.org/releasenotes/spatstat-1.39-0.html, L function https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lest, Kmark https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kmark |

